{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "e578ba7a-6cc5-4cb7-9bd8-d63dbaba38af",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os, sys, glob\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "import getpass\n",
    "import subprocess\n",
    "import time\n",
    "import importlib\n",
    "from shutil import copy2\n",
    "\n",
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "\n",
    "### Path to this cloned GitHub repo:\n",
    "#SCRIPT_DIR = os.path.dirname(__file__)  # edit this to the GitHub repo path. Throws an error by default.\n",
    "\n",
    "##### EDIT ######\n",
    "SCRIPT_DIR = os.path.dirname('/home/ubuntu/data/heme_binder_diffusion_edit/scripts')  # edit this to the GitHub repo path. Throws an error by default.\n",
    "\n",
    "          \n",
    "assert os.path.exists(SCRIPT_DIR)\n",
    "sys.path.append(SCRIPT_DIR+\"/scripts/utils\")\n",
    "import utils"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d25115bb",
   "metadata": {},
   "source": [
    "# De novo heme binding protein design pipeline using RFdiffusionAA\n",
    "#### And other ligand binders too, I guess\n",
    "Indrek Kalvet, PhD (Institute for Protein Design, University of Washington), ikalvet@uw.edu\n",
    "\n",
    "As implemented in the publication \"<i>Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom\"\n",
    "\n",
    "Link:\n",
    "\n",
    "This notebook illustrates the design of heme-binding proteins, starting from minimal information (heme + substrate + CYS motif). It should work with minor modifications also for any other ligand.\n",
    "\n",
    "The pipeline consists of 7 steps:<br>\n",
    "\n",
    "    0) The protein backbones are generated with RFdiffusionAA\n",
    "    1) Sequence is designed with proteinMPNN (without the ligand)\n",
    "    2) Structures are predicted with AlphaFold2\n",
    "    3) Ligand binding site is designed with LigandMPNN/FastRelax, or Rosetta FastDesign\n",
    "    4) Sequences surrounding the ligand pocket are diversified with LigandMPNN\n",
    "    5) Final designed sequences are predicted with AlphaFold2\n",
    "    6) Alphafold2-predicted models are relaxed with the ligand and analyzed\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "89b27018",
   "metadata": {},
   "source": [
    "## Paths of important Python scripts and programs\n",
    "\n",
    "#### RFdiffusionAA:\n",
    "Download RFdiffusionAA from here: https://github.com/baker-laboratory/rf_diffusion_all_atom\n",
    "\n",
    "#### RFjoint inpainting (proteininpainting)\n",
    "(Optional) Download RFjoint Inpainting here: https://github.com/RosettaCommons/RFDesign\n",
    "\n",
    "Inpainting is used to further resample/diversify diffusion outputs, and it may also increase AF2 success rates.\n",
    "<br>\n",
    "\n",
    "#### AlphaFold2 and LigandMPNN\n",
    "Other dependencies (ligandMPNN and AlphaFold2) are downloaded as submodules with this GitHub repository.\n",
    "\n",
    "<br>\n",
    "<br>\n",
    "<i>After these evironments are set up and repositories are downloaded please provide paths to the inference scripts in the cell below:</i>\n",
    "\n",
    "<br>\n",
    "<br>\n",
    "\n",
    "#### Python or Apptainer image\n",
    "Please define in the cell below any Python executables or Apptainer image paths that are able to run the different scripts.<br>\n",
    "\n",
    "This pipeline is tested to run based on two different Conda environments: `diffusion` and `mlfold`.<br>\n",
    "Both of these can be set up based on the yml files provided by this repository.<br>\n",
    "`mlfold` is used for AlphaFold2, and `diffusion` is used for everything else.\n",
    "\n",
    "<br>\n",
    "\n",
    "Alternatively, if your system is using Apptainers, you can set them up based on the same yml files, or do it separately. For RFdiffusionAA you can download this Apptainer image:\n",
    "http://files.ipd.uw.edu/pub/RF-All-Atom/containers/rf_se3_diffusion.sif\n",
    "\n",
    "If you are setting the environments or containers up without the provided YML files, then the minimum requirements for the different types are:<br>\n",
    "`af2` Python needs to include jax=0.4.25 and jaxlib=0.4.23<br>\n",
    "`proteinMPNN` Python needs to include pytorch and prody<br>\n",
    "`general` Python needs to include pytorch, pyrosetta, prody<br>\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "b49d9e31",
   "metadata": {},
   "outputs": [],
   "source": [
    "##### EDIT ######\n",
    "diffusion_script = \"docker run -i --rm --privileged --gpus 1 -v /home/ubuntu/data/heme_binder_diffusion_edit/input:/input -v /home/ubuntu/data/heme_binder_diffusion_edit/output:/output rfdiffusion_aa apptainer run --bind /input:/input --bind /output:/output --nv rf_se3_diffusion.sif -u run_inference.py\"  # edit this\n",
    "inpaint_script = \"PATH/TO/RFDesign/inpainting/inpaint.py\"  # edit this if needed\n",
    "\n",
    "##### Download and plant them from Github in lib folders #####\n",
    "# proteinMPNN_script = f\"docker run -i --rm --gpus 1 -v /home/jonas/Data/git/heme_binder_diffusion_edit/input:/input -v /home/jonas/Data/git/heme_binder_diffusion_edit/output:/output ligandmpnn python3 run.py\"  # from submodule\n",
    "proteinMPNN_script = f\"{SCRIPT_DIR}/lib/LigandMPNN/run.py\"\n",
    "AF2_script = f\"{SCRIPT_DIR}/scripts/af2/af2.py\"  # from submodule\n",
    "# Setup conda environment af2, proteinMPNN, general\n",
    "\n",
    "\n",
    "### Python and/or Apptainer executables needed for running the jobs\n",
    "### Please provide paths to executables that are able to run the different tasks.\n",
    "### They can all be the same if you have an environment with all of the ncessary Python modules in one\n",
    "\n",
    "# If your added Apptainer does not execute scripts directly,\n",
    "# try adding 'apptainer run' or 'apptainer run --nv' (for GPU) in front of the command\n",
    "\n",
    "\n",
    "##### EDIT ######\n",
    "CONDAPATH = \"/home/ubuntu/anaconda3\"   # edit this depending on where your Conda environments live\n",
    "PYTHON = {\"diffusion\": f\"{CONDAPATH}/envs/diffusion/bin/python\",  # NOT USED - hopefully\n",
    "          \"af2\": f\"{CONDAPATH}/envs/mlfold/bin/python\",\n",
    "          \"proteinMPNN\": f\"{CONDAPATH}/envs/SE3nv_ln_pr/bin/python\",\n",
    "          \"general\": f\"{CONDAPATH}/envs/SE3nv_ln_pr/bin/python\"}  # Must be able to run LigandMPNN too..."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "74d4990c-bff7-44dd-b675-c0379d9a45af",
   "metadata": {},
   "source": [
    "## Project description and paths"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "480ed4b3-6af8-4556-85e5-bd20edd78cc0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Working directory: /home/ubuntu/data/heme_binder_diffusion_edit/output\n"
     ]
    }
   ],
   "source": [
    "username = getpass.getuser()  # your username on the running system\n",
    "EMAIL = f\"{username}@uw.edu\"  # edit based on your organization. For Slurm job notifications.\n",
    "\n",
    "PROJECT = \"example_Heme_diffusion\"\n",
    "\n",
    "### Path where the jobs will be run and outputs dumped\n",
    "WDIR = \"/home/ubuntu/data/heme_binder_diffusion_edit/output\"\n",
    "\n",
    "if not os.path.exists(WDIR):\n",
    "    os.makedirs(WDIR, exist_ok=True)\n",
    "\n",
    "print(f\"Working directory: {WDIR}\")\n",
    "\n",
    "USE_GPU_for_AF2 = True"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "204f8e5c-0235-478b-a123-cc896fa9734d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Ligand information\n",
    "params = [f\"{SCRIPT_DIR}/theozyme/HBA/HBA.params\"]  # Rosetta params file(s)\n",
    "LIGAND = \"HBA\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c5fca3ef-ba68-4896-8a98-dcc881832174",
   "metadata": {},
   "source": [
    "## 0: Setting up diffusion run"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "7b9faacb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found 1 PDB files\n"
     ]
    }
   ],
   "source": [
    "# Using example PDB file with ligand HBA and protein 7o2g backbone.\n",
    "## Note: the repository also contains additional HBA conformers with 7o2g and P450 motifs\n",
    "## in the same directory as a ZIP file.\n",
    "\n",
    "diffusion_inputs = glob.glob(f\"{SCRIPT_DIR}/input/*.pdb\")\n",
    "print(f\"Found {len(diffusion_inputs)} PDB files\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "01405838-201f-4e1e-940d-ea1d05fdaba5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Estimated time to produce 240 designs = 2800 minutes\n",
      "Wrote config file to /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/config.yaml\n"
     ]
    }
   ],
   "source": [
    "## Setting up general settings for diffusion\n",
    "\n",
    "DIFFUSION_DIR = f\"{WDIR}/0_diffusion\"\n",
    "if not os.path.exists(DIFFUSION_DIR):\n",
    "    os.makedirs(DIFFUSION_DIR, exist_ok=False)\n",
    "\n",
    "os.chdir(DIFFUSION_DIR)\n",
    "\n",
    "N_designs = 240\n",
    "T_steps = 200\n",
    "\n",
    "## Edit this config based on motif residues, etc...\n",
    "config = f\"\"\"\n",
    "defaults:\n",
    "  - aa\n",
    "\n",
    "diffuser:\n",
    "  T: {T_steps}\n",
    "\n",
    "inference:\n",
    "  num_designs: {N_designs}\n",
    "  model_runner: NRBStyleSelfCond\n",
    "  ligand: '{LIGAND}'\n",
    "\n",
    "model:\n",
    "  freeze_track_motif: True\n",
    "\n",
    "contigmap:\n",
    "  contigs: [\"30-110,A15-15,30-110\"]\n",
    "  inpaint_str: null\n",
    "  length: \"100-140\"\n",
    "\n",
    "potentials:\n",
    "  guiding_potentials: [\"type:ligand_ncontacts,weight:1\"] \n",
    "  guide_scale: 2\n",
    "  guide_decay: cubic\n",
    "\"\"\"\n",
    "\n",
    "estimated_time = 3.5 * T_steps * N_designs  # assuming 3.5 seconds per timestep on A4000 GPU\n",
    "\n",
    "print(f\"Estimated time to produce {N_designs} designs = {estimated_time/60:.0f} minutes\")\n",
    "with open(\"config.yaml\", \"w\") as file:\n",
    "    file.write(config)\n",
    "print(f\"Wrote config file to {os.path.realpath('config.yaml')}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "c8a89617-a8a7-45d5-abd5-dc85fa32bc17",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7o2g_HBA\n",
      "An example diffusion command that was generated:\n",
      "   #!/bin/bash\n",
      "docker run -i --rm --privileged --gpus 1 -v /home/ubuntu/data/heme_binder_diffusion_edit/input:/input -v /home/ubuntu/data/heme_binder_diffusion_edit/output:/output rfdiffusion_aa apptainer run --bind /input:/input --bind /output:/output --nv rf_se3_diffusion.sif -u run_inference.py --config-dir=/output/0_diffusion --config-name=config.yaml inference.input_pdb=/input/7o2g_HBA.pdb inference.output_prefix='/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif' > /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/output.log\n"
     ]
    }
   ],
   "source": [
    "## Setting up diffusion commands based on the input PDB file(s)\n",
    "## Diffusion jobs are run in separate directories for each input PDB\n",
    "\n",
    "commands_diffusion = []\n",
    "cmds_filename = \"commands_diffusion\"\n",
    "diffusion_rundirs = []\n",
    "###### Original\n",
    "\"\"\"\n",
    "with open(cmds_filename, \"w\") as file:\n",
    "    for p in diffusion_inputs:\n",
    "        pdbname = os.path.basename(p).replace(\".pdb\", \"\")\n",
    "        os.makedirs(pdbname, exist_ok=True)\n",
    "        cmd = f\"cd {pdbname} ; {PYTHON['diffusion']} {diffusion_script} --config-dir=../ \"\\\n",
    "              f\"--config-name=config.yaml inference.input_pdb={p} \"\\\n",
    "              f\"inference.output_prefix='./out/{pdbname}_dif' > output.log ; cd ..\\n\"\n",
    "        commands_diffusion.append(cmd)\n",
    "        diffusion_rundirs.append(pdbname)\n",
    "        file.write(cmd)\n",
    "\"\"\"\n",
    "with open(cmds_filename, \"w\") as file:\n",
    "    for p in diffusion_inputs:\n",
    "        pdbname = os.path.basename(p).replace(\".pdb\", \"\")\n",
    "        print(pdbname)\n",
    "        os.makedirs(pdbname, exist_ok=True)\n",
    "        cmd = f\"#!/bin/bash\\n{diffusion_script} --config-dir=/output/0_diffusion \"\\\n",
    "              f\"--config-name=config.yaml inference.input_pdb=/input/{pdbname}.pdb \"\\\n",
    "              f\"inference.output_prefix='/output/0_diffusion/{pdbname}/out/{pdbname}_dif' > {DIFFUSION_DIR}/{pdbname}/output.log\"\n",
    "        commands_diffusion.append(cmd)\n",
    "        diffusion_rundirs.append(pdbname)\n",
    "        file.write(cmd)\n",
    "\n",
    "print(f\"An example diffusion command that was generated:\\n   {cmd}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "b7196909-d565-4c86-8a20-d39d68bc1aeb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/.done\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/commands_diffusion\n",
      "Done file found - skipping\n"
     ]
    }
   ],
   "source": [
    "## Creating a Slurm submit script\n",
    "## adjust time depending on number of designs and available hardware\n",
    "submit_script = \"submit_diffusion.sh\"\n",
    "\n",
    "####### Original\n",
    "\"\"\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"diffusion_example\", gpu=True, gres=\"gpu:a4000:1\",\n",
    "                                 mem=\"8g\", N_cores=2, time=\"1:00:00\", email=EMAIL,\n",
    "                                 array=len(commands_diffusion), array_commandfile=cmds_filename)\n",
    "print(f\"Writing diffusion submission script to {submit_script}\")\n",
    "print(f\"{len(commands_diffusion)} diffusion jobs to run\")\n",
    "\n",
    "if not os.path.exists(DIFFUSION_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n",
    "\"\"\"\n",
    "\n",
    "file_path = DIFFUSION_DIR+\"/\"+cmds_filename\n",
    "print(DIFFUSION_DIR+\"/.done\")\n",
    "print(file_path)\n",
    "\n",
    "import stat\n",
    "# Get the current permissions of the file\n",
    "current_permissions = os.stat(file_path).st_mode\n",
    "# Add execute permission for the owner\n",
    "os.chmod(file_path, current_permissions | stat.S_IXUSR)\n",
    "\n",
    "if not os.path.exists(DIFFUSION_DIR+\"/.done\"):\n",
    "    print(\"Running...\")\n",
    "    # p = subprocess.Popen(\"bash \" + file_path, shell=True)\n",
    "    # p = subprocess.run(['bash'], stdin=file_path, shell=True)\n",
    "    # with open(file_path, 'r') as file:\n",
    "    #     subprocess.run(['bash'], stdin=file, shell=True)\n",
    "    # subprocess.run(['bash', file_path], shell=True)\n",
    "    p = subprocess.Popen([\"bash\", file_path], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n",
    "    print(output)\n",
    "    print(err)\n",
    "    print(\"Done\")\n",
    "else:\n",
    "    print(\"Done file found - skipping\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "86ed7eb2-c045-49a0-9264-292a6ef4dc27",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with diffusion and happy with the outputs then mark it as done\n",
    "DIFFUSION_DIR = f\"{WDIR}/0_diffusion\"\n",
    "os.chdir(DIFFUSION_DIR)\n",
    "\n",
    "if not os.path.exists(DIFFUSION_DIR+\"/.done\"):\n",
    "    with open(f\"{DIFFUSION_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a474a099-60e1-493d-85f2-12b8808a0b11",
   "metadata": {},
   "source": [
    "### Analyzing diffusion outputs\n",
    "The purpose of this step is to identify diffused backbones that meet certain quality criteria. These scaffolds should be relatively globular (measured by radius of gyration (rog), and longest helix). They should not have clashes between the ligand and the backbone, the ligand should not be too exposed (measured by relative SASA). The termini should not be too close to the ligand (term_mindist), and the backbone should not be too loopy. In the example below we are also looking for backbones that leave some part of the ligand more exposed."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "06c85384-baa4-41c2-bc21-42587839a44d",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_75.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_77.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_65.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_200.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_165.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_88.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_1.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_57.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_139.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_76.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_12.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_43.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_19.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_42.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_193.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_67.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_46.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_127.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_177.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_21.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_11.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_126.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_92.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_80.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_108.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_31.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_6.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_154.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_23.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_171.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_138.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_107.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_45.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_117.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_40.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_86.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_4.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_147.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_89.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_30.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_201.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_225.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_168.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_164.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_49.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_24.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_227.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_124.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_232.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_105.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_97.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_192.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_178.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_98.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_231.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_99.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_131.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_111.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_66.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_35.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_228.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_70.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_179.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_199.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_47.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_142.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_114.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_74.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_239.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_216.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_72.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_25.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_194.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_120.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_113.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_236.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_33.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_212.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_206.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_69.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_198.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_34.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_48.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_129.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_191.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_238.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_73.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_10.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_52.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_121.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_18.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_61.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_2.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_205.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_109.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_210.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_15.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_149.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_96.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_5.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_173.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_132.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_144.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_209.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_81.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_130.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_71.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_220.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_7.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_101.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_145.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_187.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_222.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_87.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_116.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_160.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_174.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_17.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_36.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_28.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_102.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_83.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_29.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_170.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_64.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_20.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_51.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_182.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_213.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_196.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_214.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_223.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_100.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_176.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_93.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_215.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_95.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_146.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_26.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_78.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_134.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_32.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_141.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_180.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_184.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_183.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_128.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152.pdb\n",
      "--ref /home/ubuntu/data/heme_binder_diffusion_edit/input/*.pdb\n",
      "--params /home/ubuntu/data/heme_binder_diffusion_edit/theozyme/HBA/HBA.params\n",
      "--term_limit 15.0\n",
      "--SASA_limit 0.3\n",
      "--loop_limit 0.4\n",
      "--ref_catres A15\n",
      "--rethread True\n",
      "--fix True\n",
      "--exclude_clash_atoms O1 O2 O3 O4 C5 C10\n",
      "--ligand_exposed_atoms C45 C46 C47\n",
      "--exposed_atom_SASA 10.0\n",
      "--longest_helix 30\n",
      "--rog 30.0\n",
      "--traj 5/30\n",
      "--analyze False\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/diffusion_analysis/process_diffusion_outputs.py --pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_75.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_77.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_65.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_200.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_165.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_88.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_1.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_57.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_139.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_76.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_12.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_43.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_19.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_42.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_193.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_67.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_46.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_127.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_177.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_21.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_11.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_126.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_92.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_80.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_108.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_31.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_6.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_154.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_23.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_171.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_138.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_107.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_45.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_117.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_40.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_86.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_4.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_147.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_89.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_30.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_201.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_225.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_168.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_164.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_49.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_24.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_227.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_124.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_232.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_105.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_97.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_192.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_178.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_98.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_231.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_99.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_131.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_111.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_66.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_35.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_228.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_70.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_179.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_199.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_47.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_142.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_114.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_74.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_239.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_216.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_72.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_25.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_194.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_120.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_113.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_236.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_33.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_212.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_206.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_69.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_198.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_34.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_48.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_129.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_191.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_238.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_73.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_10.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_52.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_121.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_18.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_61.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_2.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_205.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_109.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_210.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_15.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_149.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_96.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_5.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_173.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_132.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_144.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_209.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_81.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_130.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_71.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_220.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_7.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_101.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_145.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_187.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_222.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_87.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_116.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_160.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_174.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_17.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_36.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_28.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_102.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_83.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_29.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_170.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_64.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_20.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_51.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_182.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_213.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_196.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_214.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_223.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_100.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_176.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_93.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_215.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_95.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_146.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_26.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_78.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_134.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_32.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_141.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_180.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_184.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_183.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_128.pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152.pdb --ref /home/ubuntu/data/heme_binder_diffusion_edit/input/*.pdb --params /home/ubuntu/data/heme_binder_diffusion_edit/theozyme/HBA/HBA.params --term_limit 15.0 --SASA_limit 0.3 --loop_limit 0.4 --ref_catres A15 --rethread --fix --exclude_clash_atoms O1 O2 O3 O4 C5 C10 --ligand_exposed_atoms C45 C46 C47 --exposed_atom_SASA 10.0 --longest_helix 30 --rog 30.0 --traj 5/30\n",
      "-extra_res_fa /home/ubuntu/data/heme_binder_diffusion_edit/theozyme/HBA/HBA.params\n",
      "┌──────────────────────────────────────────────────────────────────────────────┐\n",
      "│                                 PyRosetta-4                                  │\n",
      "│              Created in JHU by Sergey Lyskov and PyRosetta Team              │\n",
      "│              (C) Copyright Rosetta Commons Member Institutions               │\n",
      "│                                                                              │\n",
      "│ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE │\n",
      "│              See LICENSE.md or email license@uw.edu for details              │\n",
      "└──────────────────────────────────────────────────────���───────────────────────┘\n",
      "PyRosetta-4 2024 [Rosetta PyRosetta4.Release.python39.ubuntu.cxx11thread.serialization 2024.10+release.2c36cbc7108d85646ca5b8ddc89c29ac1ccde88e 2024-03-01T16:53:36] retrieved from: http://www.pyrosetta.org\n",
      "240 designs to analyze.\n",
      "Using 3 processes\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_75.pdb: terminus too close to ligand: 12.58\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_75_traj1.pdb: terminus too close to ligand: 12.74\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_75_traj2.pdb: terminus too close to ligand: 12.79\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_75_traj3.pdb: terminus too close to ligand: 12.68\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_75_traj4.pdb: terminus too close to ligand: 12.68\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41.pdb: ligand is too close to the backbone 2.35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41_traj1.pdb: ligand is too close to the backbone 2.38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41_traj2.pdb: fixing CYS94 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41_traj2.pdb: ligand too exposed, L_SASA = 0.456\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41_traj3.pdb: fixing CYS94 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41_traj3.pdb: ligand too exposed, L_SASA = 0.456\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_41_traj4.pdb: ligand is too close to the backbone 2.35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_200.pdb: ligand is too close to the backbone 2.37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_200_traj1.pdb: ligand is too close to the backbone 2.37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_200_traj2.pdb: ligand is too close to the backbone 2.37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_200_traj3.pdb: ligand is too close to the backbone 2.39\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_200_traj4.pdb: ligand is too close to the backbone 2.42\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3.pdb: fixing CYS49 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3.pdb: ligand too exposed, L_SASA = 0.387\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3_traj1.pdb: fixing CYS49 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3_traj1.pdb: ligand too exposed, L_SASA = 0.364\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3_traj2.pdb: fixing CYS49 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3_traj2.pdb: ligand too exposed, L_SASA = 0.367\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3_traj3.pdb: fixing CYS49 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3_traj3.pdb: ligand too exposed, L_SASA = 0.374\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3_traj4.pdb: fixing CYS49 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_3_traj4.pdb: ligand too exposed, L_SASA = 0.377\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38.pdb: fixing CYS71 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38.pdb: ligand too exposed, L_SASA = 0.403\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38_traj1.pdb: fixing CYS71 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38_traj1.pdb: ligand too exposed, L_SASA = 0.385\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38_traj2.pdb: fixing CYS71 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38_traj2.pdb: ligand too exposed, L_SASA = 0.381\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38_traj3.pdb: fixing CYS71 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38_traj3.pdb: ligand too exposed, L_SASA = 0.379\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38_traj4.pdb: fixing CYS71 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_38_traj4.pdb: ligand too exposed, L_SASA = 0.376\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104.pdb: fixing CYS92 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104.pdb: ligand too exposed, L_SASA = 0.309\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104_traj1.pdb: fixing CYS92 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104_traj1.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 6.466\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104_traj2.pdb: fixing CYS92 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104_traj2.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 6.743\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104_traj3.pdb: fixing CYS92 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104_traj3.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 7.862\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104_traj4.pdb: fixing CYS92 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_104_traj4.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 7.177\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115.pdb: protein too loopy: 0.402\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115_traj1.pdb: protein too loopy: 0.402\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115_traj2.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115_traj2.pdb: ligand too exposed, L_SASA = 0.454\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115_traj3.pdb: ligand is too close to the backbone 2.48\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115_traj4.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_115_traj4.pdb: ligand too exposed, L_SASA = 0.436\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_12.pdb: protein too loopy: 0.429\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_12_traj1.pdb: protein too loopy: 0.429\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_12_traj2.pdb: protein too loopy: 0.429\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_12_traj3.pdb: protein too loopy: 0.429\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_12_traj4.pdb: protein too loopy: 0.445\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9.pdb: ligand too exposed, L_SASA = 0.525\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9_traj1.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9_traj1.pdb: ligand too exposed, L_SASA = 0.492\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9_traj2.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9_traj2.pdb: ligand too exposed, L_SASA = 0.476\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9_traj3.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9_traj3.pdb: ligand too exposed, L_SASA = 0.470\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9_traj4.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_9_traj4.pdb: ligand too exposed, L_SASA = 0.471\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234.pdb: fixing CYS56 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234.pdb: ligand too exposed, L_SASA = 0.522\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234_traj1.pdb: fixing CYS56 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234_traj1.pdb: ligand too exposed, L_SASA = 0.514\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234_traj2.pdb: fixing CYS56 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234_traj2.pdb: ligand too exposed, L_SASA = 0.497\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234_traj3.pdb: fixing CYS56 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234_traj3.pdb: ligand too exposed, L_SASA = 0.500\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234_traj4.pdb: fixing CYS56 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_234_traj4.pdb: ligand too exposed, L_SASA = 0.517\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_88.pdb: ligand is too close to the backbone 2.20\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_88_traj1.pdb: ligand is too close to the backbone 2.22\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_88_traj2.pdb: ligand is too close to the backbone 2.20\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_88_traj3.pdb: ligand is too close to the backbone 2.23\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_88_traj4.pdb: ligand is too close to the backbone 2.43\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_57.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_57.pdb: ligand too exposed, L_SASA = 0.303\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_57_traj1.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_57_traj2.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_57_traj3.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_57_traj4.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_76.pdb: terminus too close to ligand: 14.41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_76_traj1.pdb: terminus too close to ligand: 14.80\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_76_traj2.pdb: terminus too close to ligand: 14.63\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_76_traj3.pdb: terminus too close to ligand: 14.63\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_76_traj4.pdb: terminus too close to ligand: 14.79\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 9.371\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106_traj1.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106_traj1.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 7.315\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106_traj2.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106_traj2.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 6.358\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106_traj3.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106_traj3.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 4.987\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106_traj4.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_106_traj4.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 4.115\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_19.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_19_traj1.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_19_traj2.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_19_traj3.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_19_traj4.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56.pdb: fixing CYS60 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56.pdb: ligand too exposed, L_SASA = 0.390\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56_traj1.pdb: fixing CYS60 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56_traj1.pdb: ligand too exposed, L_SASA = 0.364\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56_traj2.pdb: fixing CYS60 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56_traj2.pdb: ligand too exposed, L_SASA = 0.368\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56_traj3.pdb: fixing CYS60 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137.pdb: ligand too exposed, L_SASA = 0.610\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137_traj1.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137_traj1.pdb: ligand too exposed, L_SASA = 0.557\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137_traj2.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137_traj2.pdb: ligand too exposed, L_SASA = 0.535\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137_traj3.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137_traj3.pdb: ligand too exposed, L_SASA = 0.500\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_137_traj4.pdb: terminus too close to ligand: 14.63\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_77.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_77_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_77_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_77_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_77_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_65.pdb: radius of gyration too high: 30.5\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_65_traj1.pdb: radius of gyration too high: 30.5\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_65_traj2.pdb: radius of gyration too high: 30.4\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_65_traj3.pdb: radius of gyration too high: 30.4\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_65_traj4.pdb: radius of gyration too high: 30.2\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_165.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_165_traj1.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_165_traj2.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_165_traj3.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_165_traj4.pdb: fixing CYS49 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_165_traj4.pdb: ligand too exposed, L_SASA = 0.348\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175.pdb: ligand too exposed, L_SASA = 0.380\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175_traj1.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175_traj1.pdb: ligand too exposed, L_SASA = 0.348\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175_traj2.pdb: protein too loopy: 0.419\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175_traj3.pdb: protein too loopy: 0.410\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_175_traj4.pdb: protein too loopy: 0.410\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_1.pdb: fixing CYS64 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_1_traj1.pdb: fixing CYS64 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_1_traj2.pdb: fixing CYS64 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_1_traj3.pdb: fixing CYS64 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_1_traj4.pdb: fixing CYS64 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_139.pdb: terminus too close to ligand: 12.05\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_139_traj1.pdb: terminus too close to ligand: 12.22\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_139_traj2.pdb: terminus too close to ligand: 13.09\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_139_traj3.pdb: terminus too close to ligand: 12.59\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_139_traj4.pdb: terminus too close to ligand: 12.95\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112.pdb: ligand too exposed, L_SASA = 0.336\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112_traj1.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112_traj1.pdb: ligand too exposed, L_SASA = 0.331\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112_traj2.pdb: ligand is too close to the backbone 2.28\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112_traj3.pdb: ligand is too close to the backbone 2.34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_112_traj4.pdb: ligand is too close to the backbone 1.99\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_43.pdb: fixing CYS80 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_43_traj1.pdb: fixing CYS80 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_43_traj2.pdb: fixing CYS80 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_43_traj3.pdb: fixing CYS80 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_43_traj4.pdb: fixing CYS80 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_42.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_42_traj1.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_42_traj2.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_42_traj3.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_42_traj4.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63.pdb: protein too loopy: 0.434\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161.pdb: ligand too exposed, L_SASA = 0.379\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161_traj1.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161_traj1.pdb: ligand too exposed, L_SASA = 0.380\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161_traj2.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161_traj2.pdb: ligand too exposed, L_SASA = 0.396\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161_traj3.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161_traj3.pdb: ligand too exposed, L_SASA = 0.401\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161_traj4.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_161_traj4.pdb: ligand too exposed, L_SASA = 0.379\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136.pdb: ligand too exposed, L_SASA = 0.368\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136_traj1.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136_traj1.pdb: ligand too exposed, L_SASA = 0.333\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136_traj2.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136_traj2.pdb: ligand too exposed, L_SASA = 0.327\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136_traj3.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136_traj3.pdb: ligand too exposed, L_SASA = 0.313\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136_traj4.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_136_traj4.pdb: ligand too exposed, L_SASA = 0.310\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186.pdb: ligand too exposed, L_SASA = 0.347\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186_traj1.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186_traj1.pdb: ligand too exposed, L_SASA = 0.311\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186_traj2.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186_traj2.pdb: ligand too exposed, L_SASA = 0.310\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186_traj3.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186_traj3.pdb: ligand too exposed, L_SASA = 0.310\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186_traj4.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_186_traj4.pdb: ligand too exposed, L_SASA = 0.313\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_127.pdb: protein too loopy: 0.469\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_127_traj1.pdb: protein too loopy: 0.477\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_127_traj2.pdb: protein too loopy: 0.477\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_127_traj3.pdb: protein too loopy: 0.477\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_127_traj4.pdb: protein too loopy: 0.477\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_21.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_21_traj1.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_21_traj2.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_21_traj3.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_21_traj4.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_11.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_11_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_11_traj2.pdb: terminus too close to ligand: 10.25\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_11_traj3.pdb: terminus too close to ligand: 9.90\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_11_traj4.pdb: terminus too close to ligand: 9.74\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_126.pdb: ligand is too close to the backbone 2.47\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_126_traj1.pdb: ligand is too close to the backbone 2.44\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_126_traj2.pdb: ligand is too close to the backbone 2.22\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_126_traj3.pdb: ligand is too close to the backbone 2.16\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_126_traj4.pdb: ligand is too close to the backbone 2.15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219.pdb: ligand too exposed, L_SASA = 0.314\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219_traj1.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219_traj1.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 7.858\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219_traj2.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219_traj2.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 8.756\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219_traj3.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219_traj3.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 9.285\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_219_traj4.pdb: terminus too close to ligand: 14.87\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63_traj1.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63_traj1.pdb: ligand too exposed, L_SASA = 0.367\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63_traj2.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63_traj2.pdb: ligand too exposed, L_SASA = 0.377\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63_traj3.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63_traj3.pdb: ligand too exposed, L_SASA = 0.379\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63_traj4.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_63_traj4.pdb: ligand too exposed, L_SASA = 0.378\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_67.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_67_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_67_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_67_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_67_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_46.pdb: ligand is too close to the backbone 2.08\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_46_traj1.pdb: ligand is too close to the backbone 2.02\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_46_traj2.pdb: ligand is too close to the backbone 2.04\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_46_traj3.pdb: ligand is too close to the backbone 2.06\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_46_traj4.pdb: ligand is too close to the backbone 2.23\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119.pdb: ligand too exposed, L_SASA = 0.363\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119_traj1.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119_traj1.pdb: ligand too exposed, L_SASA = 0.346\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119_traj2.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119_traj2.pdb: ligand too exposed, L_SASA = 0.346\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119_traj3.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119_traj3.pdb: ligand too exposed, L_SASA = 0.343\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119_traj4.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_119_traj4.pdb: ligand too exposed, L_SASA = 0.350\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226.pdb: ligand too exposed, L_SASA = 0.313\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226_traj1.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226_traj2.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226_traj2.pdb: ligand too exposed, L_SASA = 0.301\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226_traj3.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_226_traj4.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_80.pdb: terminus too close to ligand: 14.05\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_80_traj1.pdb: terminus too close to ligand: 14.06\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_80_traj2.pdb: terminus too close to ligand: 14.29\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_80_traj3.pdb: terminus too close to ligand: 14.32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_80_traj4.pdb: terminus too close to ligand: 14.58\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203.pdb: ligand too exposed, L_SASA = 0.350\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203_traj1.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203_traj1.pdb: ligand too exposed, L_SASA = 0.318\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203_traj2.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203_traj2.pdb: ligand too exposed, L_SASA = 0.314\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203_traj3.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203_traj3.pdb: ligand too exposed, L_SASA = 0.317\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203_traj4.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_203_traj4.pdb: ligand too exposed, L_SASA = 0.320\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60.pdb: ligand too exposed, L_SASA = 0.587\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60_traj1.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60_traj1.pdb: ligand too exposed, L_SASA = 0.581\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60_traj2.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60_traj2.pdb: ligand too exposed, L_SASA = 0.579\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60_traj3.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_60_traj4.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56_traj3.pdb: ligand too exposed, L_SASA = 0.372\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56_traj4.pdb: fixing CYS60 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_56_traj4.pdb: ligand too exposed, L_SASA = 0.343\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68.pdb: ligand too exposed, L_SASA = 0.421\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68_traj1.pdb: ligand is too close to the backbone 2.43\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68_traj2.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68_traj2.pdb: ligand too exposed, L_SASA = 0.388\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68_traj3.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68_traj3.pdb: ligand too exposed, L_SASA = 0.388\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68_traj4.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_68_traj4.pdb: ligand too exposed, L_SASA = 0.401\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_193.pdb: ligand is too close to the backbone 2.38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_193_traj1.pdb: ligand is too close to the backbone 2.29\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_193_traj2.pdb: ligand is too close to the backbone 2.24\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_193_traj3.pdb: ligand is too close to the backbone 2.10\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_193_traj4.pdb: ligand is too close to the backbone 1.95\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_177.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_177_traj1.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_177_traj2.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_177_traj3.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_177_traj4.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_92.pdb: fixing CYS37 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_92.pdb: ligand too exposed, L_SASA = 0.446\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_92_traj1.pdb: ligand is too close to the backbone 2.21\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_92_traj2.pdb: ligand is too close to the backbone 2.17\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_92_traj3.pdb: ligand is too close to the backbone 2.04\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_92_traj4.pdb: ligand is too close to the backbone 2.06\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44.pdb: ligand too exposed, L_SASA = 0.453\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44_traj1.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44_traj1.pdb: ligand too exposed, L_SASA = 0.448\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44_traj2.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44_traj2.pdb: ligand too exposed, L_SASA = 0.438\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44_traj3.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44_traj3.pdb: ligand too exposed, L_SASA = 0.426\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44_traj4.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_44_traj4.pdb: ligand too exposed, L_SASA = 0.409\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_108.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_108_traj1.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_108_traj2.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_108_traj3.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_108_traj4.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_154.pdb: terminus too close to ligand: 14.04\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_154_traj1.pdb: terminus too close to ligand: 13.72\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_154_traj2.pdb: terminus too close to ligand: 13.86\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_154_traj3.pdb: terminus too close to ligand: 13.88\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_154_traj4.pdb: terminus too close to ligand: 14.00\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39.pdb: ligand too exposed, L_SASA = 0.598\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39_traj1.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39_traj1.pdb: ligand too exposed, L_SASA = 0.564\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39_traj2.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39_traj2.pdb: ligand too exposed, L_SASA = 0.534\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39_traj3.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39_traj3.pdb: ligand too exposed, L_SASA = 0.528\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39_traj4.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_39_traj4.pdb: ligand too exposed, L_SASA = 0.529\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_45.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53.pdb: fixing CYS97 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53.pdb: ligand too exposed, L_SASA = 0.310\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53_traj1.pdb: fixing CYS97 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53_traj1.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 8.130\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53_traj2.pdb: terminus too close to ligand: 14.96\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53_traj3.pdb: terminus too close to ligand: 14.69\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_53_traj4.pdb: terminus too close to ligand: 14.46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_31.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_31_traj1.pdb: longest helix too long: 39\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_31_traj2.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_31_traj3.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_31_traj4.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_6.pdb: terminus too close to ligand: 10.18\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_6_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_6_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_6_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_6_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_23.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_23_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_23_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_23_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_23_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_171.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_171_traj1.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_171_traj2.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_171_traj3.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_171_traj4.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_138.pdb: ligand is too close to the backbone 2.44\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_138_traj1.pdb: ligand is too close to the backbone 2.41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_138_traj2.pdb: ligand is too close to the backbone 2.20\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_138_traj3.pdb: ligand is too close to the backbone 2.02\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_138_traj4.pdb: ligand is too close to the backbone 2.02\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_107.pdb: terminus too close to ligand: 5.27\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_107_traj1.pdb: terminus too close to ligand: 6.12\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_107_traj2.pdb: terminus too close to ligand: 7.39\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_107_traj3.pdb: terminus too close to ligand: 7.76\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_107_traj4.pdb: terminus too close to ligand: 7.87\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_86.pdb: terminus too close to ligand: 13.63\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_86_traj1.pdb: terminus too close to ligand: 13.35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_86_traj2.pdb: terminus too close to ligand: 13.21\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_86_traj3.pdb: terminus too close to ligand: 13.29\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_86_traj4.pdb: terminus too close to ligand: 13.28\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_147.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_147_traj1.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_147_traj2.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_147_traj3.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_147_traj4.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_30.pdb: terminus too close to ligand: 11.68\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_30_traj1.pdb: terminus too close to ligand: 10.59\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_30_traj2.pdb: terminus too close to ligand: 10.53\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_30_traj3.pdb: terminus too close to ligand: 10.89\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_30_traj4.pdb: terminus too close to ligand: 10.73\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_225.pdb: ligand is too close to the backbone 2.36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_225_traj1.pdb: ligand is too close to the backbone 2.43\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_225_traj2.pdb: protein too loopy: 0.407\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_225_traj3.pdb: protein too loopy: 0.417\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_225_traj4.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_225_traj4.pdb: ligand too exposed, L_SASA = 0.302\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_45_traj1.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_45_traj2.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_45_traj3.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_45_traj4.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151.pdb: ligand too exposed, L_SASA = 0.345\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151_traj1.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151_traj1.pdb: ligand too exposed, L_SASA = 0.332\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151_traj2.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151_traj2.pdb: ligand too exposed, L_SASA = 0.316\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151_traj3.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151_traj3.pdb: ligand too exposed, L_SASA = 0.316\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151_traj4.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_151_traj4.pdb: ligand too exposed, L_SASA = 0.314\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217.pdb: fixing CYS100 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217.pdb: ligand too exposed, L_SASA = 0.342\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217_traj1.pdb: fixing CYS100 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217_traj1.pdb: ligand too exposed, L_SASA = 0.318\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217_traj2.pdb: fixing CYS100 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217_traj2.pdb: ligand too exposed, L_SASA = 0.322\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217_traj3.pdb: fixing CYS100 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217_traj3.pdb: ligand too exposed, L_SASA = 0.327\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217_traj4.pdb: fixing CYS100 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_217_traj4.pdb: ligand too exposed, L_SASA = 0.328\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_168.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_168_traj1.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_168_traj2.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_168_traj3.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_168_traj4.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_164.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_164_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_164_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_164_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_164_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221.pdb: fixing CYS37 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221.pdb: ligand too exposed, L_SASA = 0.307\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221_traj1.pdb: fixing CYS37 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221_traj1.pdb: ligand too exposed, L_SASA = 0.312\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221_traj2.pdb: fixing CYS37 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221_traj2.pdb: ligand too exposed, L_SASA = 0.322\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221_traj3.pdb: fixing CYS37 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221_traj3.pdb: ligand too exposed, L_SASA = 0.308\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221_traj4.pdb: fixing CYS37 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_221_traj4.pdb: ligand too exposed, L_SASA = 0.306\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135.pdb: ligand too exposed, L_SASA = 0.413\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135_traj1.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135_traj1.pdb: ligand too exposed, L_SASA = 0.383\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135_traj2.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135_traj2.pdb: ligand too exposed, L_SASA = 0.352\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135_traj3.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135_traj3.pdb: ligand too exposed, L_SASA = 0.350\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135_traj4.pdb: fixing CYS77 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_135_traj4.pdb: ligand too exposed, L_SASA = 0.353\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233.pdb: ligand too exposed, L_SASA = 0.471\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233_traj1.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233_traj1.pdb: ligand too exposed, L_SASA = 0.415\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233_traj2.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185.pdb: ligand too exposed, L_SASA = 0.333\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185_traj1.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185_traj2.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185_traj3.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185_traj4.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_185_traj4.pdb: ligand too exposed, L_SASA = 0.303\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_117.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_117_traj1.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_117_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_117_traj3.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_117_traj4.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_40.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_40_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_40_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_40_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_40_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_4.pdb: terminus too close to ligand: 10.65\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_4_traj1.pdb: terminus too close to ligand: 11.44\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_4_traj2.pdb: terminus too close to ligand: 11.51\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_4_traj3.pdb: ligand is too close to the backbone 2.48\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_4_traj4.pdb: terminus too close to ligand: 11.03\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_89.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_89_traj1.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_89_traj2.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_89_traj3.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_89_traj4.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_201.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_201_traj1.pdb: ligand is too close to the backbone 2.21\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_201_traj2.pdb: ligand is too close to the backbone 2.20\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_201_traj3.pdb: ligand is too close to the backbone 2.12\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_201_traj4.pdb: ligand is too close to the backbone 1.95\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224.pdb: fixing CYS46 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224.pdb: ligand too exposed, L_SASA = 0.557\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224_traj1.pdb: fixing CYS46 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224_traj1.pdb: ligand too exposed, L_SASA = 0.548\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224_traj2.pdb: fixing CYS46 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224_traj2.pdb: ligand too exposed, L_SASA = 0.556\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224_traj3.pdb: fixing CYS46 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224_traj3.pdb: ligand too exposed, L_SASA = 0.552\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224_traj4.pdb: fixing CYS46 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_224_traj4.pdb: ligand too exposed, L_SASA = 0.540\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_49.pdb: terminus too close to ligand: 11.30\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_49_traj1.pdb: terminus too close to ligand: 11.38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_49_traj2.pdb: terminus too close to ligand: 11.47\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_49_traj3.pdb: terminus too close to ligand: 11.53\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_49_traj4.pdb: terminus too close to ligand: 11.32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_227.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_227_traj1.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_227_traj2.pdb: ligand is too close to the backbone 2.48\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_227_traj3.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_227_traj4.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79.pdb: ligand too exposed, L_SASA = 0.461\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79_traj1.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79_traj1.pdb: ligand too exposed, L_SASA = 0.442\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79_traj2.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79_traj2.pdb: ligand too exposed, L_SASA = 0.445\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79_traj3.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79_traj3.pdb: ligand too exposed, L_SASA = 0.439\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155.pdb: ligand too exposed, L_SASA = 0.453\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155_traj1.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155_traj1.pdb: ligand too exposed, L_SASA = 0.435\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155_traj2.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155_traj2.pdb: ligand too exposed, L_SASA = 0.428\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155_traj3.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155_traj3.pdb: ligand too exposed, L_SASA = 0.426\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155_traj4.pdb: fixing CYS72 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_155_traj4.pdb: ligand too exposed, L_SASA = 0.421\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_24.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_24_traj1.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_24_traj2.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_24_traj3.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_24_traj4.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_124.pdb: terminus too close to ligand: 12.99\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_124_traj1.pdb: terminus too close to ligand: 13.19\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_124_traj2.pdb: terminus too close to ligand: 13.30\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_124_traj3.pdb: terminus too close to ligand: 13.56\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_124_traj4.pdb: terminus too close to ligand: 13.60\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90.pdb: fixing CYS76 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90.pdb: ligand too exposed, L_SASA = 0.436\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90_traj1.pdb: fixing CYS76 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90_traj1.pdb: ligand too exposed, L_SASA = 0.409\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90_traj2.pdb: fixing CYS76 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90_traj2.pdb: ligand too exposed, L_SASA = 0.407\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90_traj3.pdb: fixing CYS76 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90_traj3.pdb: ligand too exposed, L_SASA = 0.397\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90_traj4.pdb: fixing CYS76 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_90_traj4.pdb: ligand too exposed, L_SASA = 0.392\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50.pdb: ligand too exposed, L_SASA = 0.537\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50_traj1.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50_traj1.pdb: ligand too exposed, L_SASA = 0.474\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50_traj2.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50_traj2.pdb: ligand too exposed, L_SASA = 0.464\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50_traj3.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50_traj3.pdb: ligand too exposed, L_SASA = 0.456\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50_traj4.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_50_traj4.pdb: ligand too exposed, L_SASA = 0.459\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218.pdb: ligand too exposed, L_SASA = 0.463\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218_traj1.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218_traj1.pdb: ligand too exposed, L_SASA = 0.460\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218_traj2.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218_traj2.pdb: ligand too exposed, L_SASA = 0.453\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218_traj3.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218_traj4.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_218_traj4.pdb: ligand too exposed, L_SASA = 0.439\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84.pdb: fixing CYS100 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84.pdb: ligand too exposed, L_SASA = 0.365\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84_traj1.pdb: fixing CYS100 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84_traj1.pdb: ligand too exposed, L_SASA = 0.329\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84_traj2.pdb: ligand is too close to the backbone 2.47\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84_traj3.pdb: ligand is too close to the backbone 2.45\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_84_traj4.pdb: ligand is too close to the backbone 2.43\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55.pdb: fixing CYS96 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55.pdb: ligand too exposed, L_SASA = 0.345\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55_traj1.pdb: fixing CYS96 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55_traj1.pdb: ligand too exposed, L_SASA = 0.328\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233_traj2.pdb: ligand too exposed, L_SASA = 0.405\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233_traj3.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233_traj3.pdb: ligand too exposed, L_SASA = 0.402\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233_traj4.pdb: fixing CYS43 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_233_traj4.pdb: ligand too exposed, L_SASA = 0.388\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150.pdb: fixing CYS52 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150.pdb: ligand too exposed, L_SASA = 0.389\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150_traj1.pdb: fixing CYS52 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150_traj1.pdb: ligand too exposed, L_SASA = 0.348\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150_traj2.pdb: fixing CYS52 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150_traj2.pdb: ligand too exposed, L_SASA = 0.306\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150_traj3.pdb: fixing CYS52 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150_traj3.pdb: ligand too exposed, L_SASA = 0.301\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150_traj4.pdb: fixing CYS52 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_150_traj4.pdb: ligand too exposed, L_SASA = 0.308\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_97.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_97_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_97_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_97_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_97_traj4.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_192.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_192_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_192_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_192_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_192_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230.pdb: ligand too exposed, L_SASA = 0.420\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230_traj1.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230_traj1.pdb: ligand too exposed, L_SASA = 0.422\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230_traj2.pdb: protein too loopy: 0.416\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_230_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181.pdb: ligand too exposed, L_SASA = 0.355\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181_traj1.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181_traj1.pdb: ligand too exposed, L_SASA = 0.338\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181_traj2.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181_traj2.pdb: ligand too exposed, L_SASA = 0.318\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181_traj3.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181_traj3.pdb: ligand too exposed, L_SASA = 0.317\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181_traj4.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_181_traj4.pdb: ligand too exposed, L_SASA = 0.319\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163.pdb: ligand is too close to the backbone 2.19\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163_traj1.pdb: fixing CYS79 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163_traj1.pdb: ligand too exposed, L_SASA = 0.410\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163_traj2.pdb: fixing CYS79 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163_traj2.pdb: ligand too exposed, L_SASA = 0.406\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163_traj3.pdb: ligand is too close to the backbone 2.40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_163_traj4.pdb: ligand is too close to the backbone 2.43\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_99.pdb: terminus too close to ligand: 14.25\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_99_traj1.pdb: terminus too close to ligand: 14.36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_99_traj2.pdb: terminus too close to ligand: 14.71\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_99_traj3.pdb: terminus too close to ligand: 14.27\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_99_traj4.pdb: terminus too close to ligand: 14.88\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110.pdb: ligand too exposed, L_SASA = 0.423\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110_traj1.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110_traj1.pdb: ligand too exposed, L_SASA = 0.409\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110_traj2.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_79_traj4.pdb: ligand is too close to the backbone 2.48\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_232.pdb: radius of gyration too high: 33.9\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_232_traj1.pdb: radius of gyration too high: 33.5\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_232_traj2.pdb: radius of gyration too high: 33.0\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_232_traj3.pdb: radius of gyration too high: 33.1\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_232_traj4.pdb: radius of gyration too high: 33.2\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_105.pdb: fixing CYS86 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_105_traj1.pdb: fixing CYS86 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_105_traj2.pdb: fixing CYS86 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_105_traj3.pdb: fixing CYS86 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_105_traj4.pdb: fixing CYS86 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_178.pdb: ligand is too close to the backbone 2.26\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_178_traj1.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_178_traj1.pdb: ligand too exposed, L_SASA = 0.359\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_178_traj2.pdb: ligand is too close to the backbone 2.44\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_178_traj3.pdb: ligand is too close to the backbone 1.80\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_178_traj4.pdb: ligand is too close to the backbone 1.68\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157.pdb: ligand too exposed, L_SASA = 0.329\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157_traj1.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157_traj1.pdb: ligand too exposed, L_SASA = 0.317\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157_traj2.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157_traj2.pdb: ligand too exposed, L_SASA = 0.326\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157_traj3.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157_traj3.pdb: ligand too exposed, L_SASA = 0.313\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157_traj4.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_157_traj4.pdb: ligand too exposed, L_SASA = 0.311\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_98.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_98_traj1.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_98_traj2.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_98_traj3.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_98_traj4.pdb: fixing CYS42 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_98_traj4.pdb: ligand too exposed, L_SASA = 0.330\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_231.pdb: terminus too close to ligand: 14.30\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_231_traj1.pdb: terminus too close to ligand: 14.52\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_231_traj2.pdb: terminus too close to ligand: 14.38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_231_traj3.pdb: terminus too close to ligand: 14.47\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_231_traj4.pdb: terminus too close to ligand: 14.58\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_131.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_131_traj1.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_131_traj2.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_131_traj3.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_131_traj4.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58_traj1.pdb: protein too loopy: 0.406\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58_traj2.pdb: fixing CYS84 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58_traj2.pdb: ligand too exposed, L_SASA = 0.490\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58_traj3.pdb: fixing CYS84 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58_traj3.pdb: ligand too exposed, L_SASA = 0.488\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58_traj4.pdb: fixing CYS84 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_58_traj4.pdb: ligand too exposed, L_SASA = 0.470\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172.pdb: ligand too exposed, L_SASA = 0.427\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172_traj1.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172_traj1.pdb: ligand too exposed, L_SASA = 0.388\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172_traj2.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172_traj2.pdb: ligand too exposed, L_SASA = 0.399\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172_traj3.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172_traj3.pdb: ligand too exposed, L_SASA = 0.398\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55_traj2.pdb: fixing CYS96 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55_traj2.pdb: ligand too exposed, L_SASA = 0.312\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55_traj3.pdb: fixing CYS96 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55_traj3.pdb: ligand too exposed, L_SASA = 0.310\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55_traj4.pdb: fixing CYS96 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_55_traj4.pdb: ligand too exposed, L_SASA = 0.302\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197.pdb: ligand too exposed, L_SASA = 0.462\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197_traj1.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197_traj1.pdb: ligand too exposed, L_SASA = 0.436\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197_traj2.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197_traj2.pdb: ligand too exposed, L_SASA = 0.436\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197_traj3.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197_traj3.pdb: ligand too exposed, L_SASA = 0.437\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197_traj4.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_197_traj4.pdb: ligand too exposed, L_SASA = 0.424\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_111.pdb: protein too loopy: 0.412\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_111_traj1.pdb: protein too loopy: 0.404\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_111_traj2.pdb: protein too loopy: 0.419\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_111_traj3.pdb: protein too loopy: 0.404\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_111_traj4.pdb: protein too loopy: 0.412\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_66.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_66_traj1.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_66_traj2.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_66_traj3.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_66_traj4.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_35.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_35_traj1.pdb: radius of gyration too high: 33.1\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_35_traj2.pdb: radius of gyration too high: 32.8\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_35_traj3.pdb: radius of gyration too high: 32.7\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_35_traj4.pdb: radius of gyration too high: 32.7\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133.pdb: ligand too exposed, L_SASA = 0.359\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133_traj1.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133_traj1.pdb: ligand too exposed, L_SASA = 0.329\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133_traj2.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133_traj2.pdb: ligand too exposed, L_SASA = 0.329\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133_traj3.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133_traj3.pdb: ligand too exposed, L_SASA = 0.322\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133_traj4.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_133_traj4.pdb: ligand too exposed, L_SASA = 0.319\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188.pdb: fixing CYS67 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188.pdb: ligand too exposed, L_SASA = 0.354\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188_traj1.pdb: fixing CYS67 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188_traj1.pdb: ligand too exposed, L_SASA = 0.346\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188_traj2.pdb: fixing CYS67 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188_traj2.pdb: ligand too exposed, L_SASA = 0.354\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188_traj3.pdb: fixing CYS67 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188_traj3.pdb: ligand too exposed, L_SASA = 0.332\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188_traj4.pdb: fixing CYS67 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_188_traj4.pdb: ligand too exposed, L_SASA = 0.321\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_114.pdb: ligand is too close to the backbone 2.42\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_114_traj1.pdb: protein too loopy: 0.443\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_114_traj2.pdb: protein too loopy: 0.406\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_114_traj3.pdb: protein too loopy: 0.443\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_114_traj4.pdb: protein too loopy: 0.481\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62.pdb: ligand too exposed, L_SASA = 0.468\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62_traj1.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62_traj1.pdb: ligand too exposed, L_SASA = 0.425\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172_traj4.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_172_traj4.pdb: ligand too exposed, L_SASA = 0.391\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_179.pdb: longest helix too long: 41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_179_traj1.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_179_traj2.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_179_traj3.pdb: longest helix too long: 41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_179_traj4.pdb: longest helix too long: 41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_199.pdb: terminus too close to ligand: 13.05\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_199_traj1.pdb: terminus too close to ligand: 12.81\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_199_traj2.pdb: terminus too close to ligand: 12.81\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_199_traj3.pdb: terminus too close to ligand: 13.01\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_199_traj4.pdb: terminus too close to ligand: 13.12\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_142.pdb: ligand is too close to the backbone 2.20\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_142_traj1.pdb: ligand is too close to the backbone 2.34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_142_traj2.pdb: ligand is too close to the backbone 2.20\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_142_traj3.pdb: ligand is too close to the backbone 2.40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_142_traj4.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_142_traj4.pdb: ligand too exposed, L_SASA = 0.463\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166.pdb: ligand too exposed, L_SASA = 0.429\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166_traj1.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166_traj1.pdb: ligand too exposed, L_SASA = 0.408\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166_traj2.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166_traj2.pdb: ligand too exposed, L_SASA = 0.413\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166_traj3.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166_traj3.pdb: ligand too exposed, L_SASA = 0.411\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_166_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167.pdb: fixing CYS84 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167.pdb: ligand too exposed, L_SASA = 0.600\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167_traj1.pdb: fixing CYS84 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167_traj1.pdb: ligand too exposed, L_SASA = 0.579\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167_traj2.pdb: fixing CYS84 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167_traj2.pdb: ligand too exposed, L_SASA = 0.576\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167_traj3.pdb: fixing CYS84 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167_traj3.pdb: ligand too exposed, L_SASA = 0.563\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167_traj4.pdb: fixing CYS84 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_167_traj4.pdb: ligand too exposed, L_SASA = 0.578\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235.pdb: protein too loopy: 0.406\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235_traj1.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235_traj1.pdb: ligand too exposed, L_SASA = 0.338\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235_traj2.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235_traj2.pdb: ligand too exposed, L_SASA = 0.338\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235_traj3.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235_traj3.pdb: ligand too exposed, L_SASA = 0.336\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235_traj4.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_235_traj4.pdb: ligand too exposed, L_SASA = 0.335\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_194.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_194_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_194_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_194_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_194_traj4.pdb: protein too loopy: 0.402\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_120.pdb: radius of gyration too high: 32.2\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_120_traj1.pdb: radius of gyration too high: 32.1\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_120_traj2.pdb: radius of gyration too high: 32.0\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_120_traj3.pdb: radius of gyration too high: 31.5\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_120_traj4.pdb: radius of gyration too high: 31.5\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_33.pdb: terminus too close to ligand: 10.90\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_33_traj1.pdb: terminus too close to ligand: 10.44\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_33_traj2.pdb: terminus too close to ligand: 10.74\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110_traj2.pdb: ligand too exposed, L_SASA = 0.406\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110_traj3.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110_traj3.pdb: ligand too exposed, L_SASA = 0.403\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110_traj4.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_110_traj4.pdb: ligand too exposed, L_SASA = 0.402\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122.pdb: fixing CYS83 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122.pdb: ligand too exposed, L_SASA = 0.423\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122_traj1.pdb: fixing CYS83 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122_traj1.pdb: ligand too exposed, L_SASA = 0.429\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122_traj2.pdb: fixing CYS83 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122_traj2.pdb: ligand too exposed, L_SASA = 0.428\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122_traj3.pdb: fixing CYS83 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122_traj3.pdb: ligand too exposed, L_SASA = 0.424\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122_traj4.pdb: fixing CYS83 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_122_traj4.pdb: ligand too exposed, L_SASA = 0.419\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_228.pdb: terminus too close to ligand: 14.31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_228_traj1.pdb: terminus too close to ligand: 14.07\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_228_traj2.pdb: terminus too close to ligand: 14.33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_228_traj3.pdb: terminus too close to ligand: 14.48\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_228_traj4.pdb: terminus too close to ligand: 14.54\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_70.pdb: ligand is too close to the backbone 2.24\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_70_traj1.pdb: ligand is too close to the backbone 2.06\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_70_traj2.pdb: ligand is too close to the backbone 2.05\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_70_traj3.pdb: ligand is too close to the backbone 2.04\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_70_traj4.pdb: ligand is too close to the backbone 2.01\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_47.pdb: ligand is too close to the backbone 1.78\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_47_traj1.pdb: ligand is too close to the backbone 1.65\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_47_traj2.pdb: ligand is too close to the backbone 1.65\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_47_traj3.pdb: ligand is too close to the backbone 1.71\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_47_traj4.pdb: ligand is too close to the backbone 1.89\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207.pdb: ligand too exposed, L_SASA = 0.362\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207_traj1.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207_traj1.pdb: ligand too exposed, L_SASA = 0.328\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207_traj2.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207_traj2.pdb: ligand too exposed, L_SASA = 0.317\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207_traj3.pdb: ligand is too close to the backbone 2.49\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_207_traj4.pdb: ligand is too close to the backbone 2.46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_74.pdb: ligand is too close to the backbone 1.97\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_74_traj1.pdb: ligand is too close to the backbone 2.04\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_74_traj2.pdb: ligand is too close to the backbone 2.22\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_74_traj3.pdb: ligand is too close to the backbone 2.20\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_74_traj4.pdb: ligand is too close to the backbone 2.24\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_216.pdb: fixing CYS97 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_216_traj1.pdb: fixing CYS97 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_216_traj2.pdb: fixing CYS97 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_216_traj3.pdb: fixing CYS97 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_216_traj4.pdb: fixing CYS97 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_25.pdb: ligand is too close to the backbone 2.17\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_25_traj1.pdb: ligand is too close to the backbone 2.26\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_25_traj2.pdb: ligand is too close to the backbone 2.33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_25_traj3.pdb: ligand is too close to the backbone 2.37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_25_traj4.pdb: fixing CYS52 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_236.pdb: terminus too close to ligand: 13.23\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_236_traj1.pdb: terminus too close to ligand: 13.13\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_236_traj2.pdb: terminus too close to ligand: 13.38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_236_traj3.pdb: terminus too close to ligand: 13.34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_236_traj4.pdb: terminus too close to ligand: 13.14\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62_traj2.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62_traj2.pdb: ligand too exposed, L_SASA = 0.422\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62_traj3.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62_traj3.pdb: ligand too exposed, L_SASA = 0.411\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62_traj4.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_62_traj4.pdb: ligand too exposed, L_SASA = 0.421\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_239.pdb: terminus too close to ligand: 13.99\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_239_traj1.pdb: terminus too close to ligand: 13.88\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_239_traj2.pdb: terminus too close to ligand: 14.00\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_239_traj3.pdb: terminus too close to ligand: 13.85\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_239_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237.pdb: ligand is too close to the backbone 2.22\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237_traj1.pdb: ligand is too close to the backbone 2.19\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237_traj2.pdb: ligand is too close to the backbone 2.41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237_traj3.pdb: fixing CYS56 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237_traj3.pdb: ligand too exposed, L_SASA = 0.366\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237_traj4.pdb: fixing CYS56 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_237_traj4.pdb: ligand too exposed, L_SASA = 0.360\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_72.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_72_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_72_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_72_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_72_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85.pdb: ligand too exposed, L_SASA = 0.389\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85_traj1.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85_traj1.pdb: ligand too exposed, L_SASA = 0.377\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85_traj2.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85_traj2.pdb: ligand too exposed, L_SASA = 0.383\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85_traj3.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85_traj3.pdb: ligand too exposed, L_SASA = 0.394\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85_traj4.pdb: fixing CYS54 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_85_traj4.pdb: ligand too exposed, L_SASA = 0.419\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_113.pdb: terminus too close to ligand: 13.06\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_113_traj1.pdb: ligand is too close to the backbone 2.47\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_113_traj2.pdb: ligand is too close to the backbone 2.46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_113_traj3.pdb: ligand is too close to the backbone 2.37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_113_traj4.pdb: terminus too close to ligand: 12.59\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_212.pdb: radius of gyration too high: 37.5\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_212_traj1.pdb: radius of gyration too high: 36.3\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_212_traj2.pdb: radius of gyration too high: 36.0\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_212_traj3.pdb: longest helix too long: 37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_212_traj4.pdb: longest helix too long: 37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190.pdb: ligand too exposed, L_SASA = 0.526\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190_traj1.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190_traj1.pdb: ligand too exposed, L_SASA = 0.470\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190_traj2.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190_traj2.pdb: ligand too exposed, L_SASA = 0.453\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190_traj3.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190_traj3.pdb: ligand too exposed, L_SASA = 0.448\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190_traj4.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_190_traj4.pdb: ligand too exposed, L_SASA = 0.453\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_198.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_198_traj1.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_198_traj2.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_198_traj3.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_198_traj4.pdb: longest helix too long: 31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_33_traj3.pdb: terminus too close to ligand: 10.80\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_33_traj4.pdb: terminus too close to ligand: 10.74\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_206.pdb: longest helix too long: 46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_206_traj1.pdb: longest helix too long: 46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_206_traj2.pdb: longest helix too long: 46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_206_traj3.pdb: longest helix too long: 46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_206_traj4.pdb: longest helix too long: 46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22.pdb: ligand too exposed, L_SASA = 0.454\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22_traj1.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22_traj1.pdb: ligand too exposed, L_SASA = 0.431\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22_traj2.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22_traj2.pdb: ligand too exposed, L_SASA = 0.431\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22_traj3.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22_traj3.pdb: ligand too exposed, L_SASA = 0.447\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22_traj4.pdb: fixing CYS34 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_22_traj4.pdb: ligand too exposed, L_SASA = 0.451\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_34.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_34_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_34_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_34_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_34_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_48.pdb: longest helix too long: 38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_48_traj1.pdb: longest helix too long: 38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_48_traj2.pdb: longest helix too long: 37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_48_traj3.pdb: longest helix too long: 37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_48_traj4.pdb: longest helix too long: 38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 3.838\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143_traj1.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143_traj1.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 2.799\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143_traj3.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143_traj3.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 1.801\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143_traj4.pdb: fixing CYS65 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_143_traj4.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 3.173\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_238.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_238_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_238_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_238_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_238_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_73.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_73_traj1.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_73_traj2.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_73_traj3.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_73_traj4.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_121.pdb: ligand is too close to the backbone 2.30\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_121_traj1.pdb: ligand is too close to the backbone 1.96\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_121_traj2.pdb: ligand is too close to the backbone 1.94\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_121_traj3.pdb: ligand is too close to the backbone 1.90\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_121_traj4.pdb: ligand is too close to the backbone 1.94\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204.pdb: ligand too exposed, L_SASA = 0.359\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204_traj1.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204_traj1.pdb: ligand too exposed, L_SASA = 0.348\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204_traj2.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204_traj2.pdb: ligand too exposed, L_SASA = 0.352\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204_traj3.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204_traj3.pdb: ligand too exposed, L_SASA = 0.347\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27.pdb: ligand too exposed, L_SASA = 0.402\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27_traj1.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27_traj1.pdb: ligand too exposed, L_SASA = 0.385\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27_traj2.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27_traj2.pdb: ligand too exposed, L_SASA = 0.383\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27_traj3.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27_traj3.pdb: ligand too exposed, L_SASA = 0.371\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27_traj4.pdb: fixing CYS75 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_27_traj4.pdb: ligand too exposed, L_SASA = 0.361\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_69.pdb: ligand is too close to the backbone 2.28\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_69_traj1.pdb: ligand is too close to the backbone 2.40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_69_traj2.pdb: fixing CYS95 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_69_traj3.pdb: fixing CYS95 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_69_traj4.pdb: fixing CYS95 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_129.pdb: ligand is too close to the backbone 1.76\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_129_traj1.pdb: ligand is too close to the backbone 2.00\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_129_traj2.pdb: ligand is too close to the backbone 2.11\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_129_traj3.pdb: ligand is too close to the backbone 2.16\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_129_traj4.pdb: ligand is too close to the backbone 2.11\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14.pdb: ligand too exposed, L_SASA = 0.413\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14_traj1.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14_traj1.pdb: ligand too exposed, L_SASA = 0.376\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14_traj2.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14_traj2.pdb: ligand too exposed, L_SASA = 0.375\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14_traj3.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14_traj3.pdb: ligand too exposed, L_SASA = 0.372\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14_traj4.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_14_traj4.pdb: ligand too exposed, L_SASA = 0.369\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_10.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_10_traj1.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_10_traj2.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_10_traj3.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_10_traj4.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_52.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_52.pdb: ligand too exposed, L_SASA = 0.305\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_52_traj1.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_52_traj2.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_52_traj3.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_52_traj4.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_18.pdb: ligand is too close to the backbone 2.35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_18_traj1.pdb: ligand is too close to the backbone 1.48\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_18_traj2.pdb: ligand is too close to the backbone 1.40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_18_traj3.pdb: ligand is too close to the backbone 1.40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_18_traj4.pdb: ligand is too close to the backbone 1.58\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_2.pdb: terminus too close to ligand: 14.80\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_2_traj1.pdb: terminus too close to ligand: 14.08\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_2_traj2.pdb: terminus too close to ligand: 13.54\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_2_traj3.pdb: terminus too close to ligand: 13.60\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_2_traj4.pdb: terminus too close to ligand: 13.57\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_210.pdb: protein too loopy: 0.447\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_210_traj1.pdb: protein too loopy: 0.439\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_210_traj2.pdb: protein too loopy: 0.439\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_210_traj3.pdb: protein too loopy: 0.439\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_210_traj4.pdb: protein too loopy: 0.455\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_149.pdb: fixing CYS33 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_149.pdb: ligand too exposed, L_SASA = 0.320\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_149_traj1.pdb: ligand is too close to the backbone 2.22\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153.pdb: ligand too exposed, L_SASA = 0.401\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153_traj1.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153_traj1.pdb: ligand too exposed, L_SASA = 0.399\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153_traj2.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153_traj2.pdb: ligand too exposed, L_SASA = 0.403\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153_traj3.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153_traj3.pdb: ligand too exposed, L_SASA = 0.391\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_153_traj4.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_191.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_191_traj1.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_191_traj2.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_191_traj3.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_191_traj4.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208.pdb: ligand too exposed, L_SASA = 0.592\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208_traj1.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208_traj1.pdb: ligand too exposed, L_SASA = 0.598\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208_traj2.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208_traj2.pdb: ligand too exposed, L_SASA = 0.602\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208_traj3.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208_traj3.pdb: ligand too exposed, L_SASA = 0.605\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208_traj4.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_208_traj4.pdb: ligand too exposed, L_SASA = 0.602\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91.pdb: ligand too exposed, L_SASA = 0.324\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91_traj1.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91_traj1.pdb: ligand too exposed, L_SASA = 0.308\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91_traj2.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91_traj3.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_91_traj4.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_61.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_61_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_61_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_61_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_61_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_205.pdb: protein too loopy: 0.422\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_205_traj1.pdb: protein too loopy: 0.422\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_205_traj2.pdb: protein too loopy: 0.412\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_205_traj3.pdb: protein too loopy: 0.422\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_205_traj4.pdb: protein too loopy: 0.422\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_109.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_109_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_109_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_109_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_109_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 9.606\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158_traj1.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158_traj1.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 9.692\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158_traj2.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158_traj3.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_158_traj4.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_144.pdb: protein too loopy: 0.427\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_144_traj1.pdb: protein too loopy: 0.435\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_144_traj2.pdb: protein too loopy: 0.435\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_144_traj3.pdb: protein too loopy: 0.419\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_144_traj4.pdb: protein too loopy: 0.444\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_81.pdb: protein too loopy: 0.425\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_81_traj1.pdb: protein too loopy: 0.410\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_81_traj2.pdb: protein too loopy: 0.403\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204_traj4.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_204_traj4.pdb: ligand too exposed, L_SASA = 0.348\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_15.pdb: protein too loopy: 0.409\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_15_traj1.pdb: protein too loopy: 0.417\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_15_traj2.pdb: protein too loopy: 0.409\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_15_traj3.pdb: protein too loopy: 0.433\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_15_traj4.pdb: protein too loopy: 0.433\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_96.pdb: terminus too close to ligand: 13.99\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_96_traj1.pdb: terminus too close to ligand: 13.85\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_96_traj2.pdb: terminus too close to ligand: 14.01\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_96_traj3.pdb: terminus too close to ligand: 14.14\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_96_traj4.pdb: ligand is too close to the backbone 2.49\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_173.pdb: fixing CYS64 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_173_traj1.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_173_traj2.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_173_traj3.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_173_traj4.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_209.pdb: ligand is too close to the backbone 2.34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_209_traj1.pdb: ligand is too close to the backbone 2.32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_209_traj2.pdb: ligand is too close to the backbone 2.34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_209_traj3.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_209_traj4.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_71.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_71_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_71_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_71_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_71_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54.pdb: ligand too exposed, L_SASA = 0.538\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54_traj1.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54_traj1.pdb: ligand too exposed, L_SASA = 0.529\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54_traj2.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54_traj2.pdb: ligand too exposed, L_SASA = 0.545\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54_traj3.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54_traj3.pdb: ligand too exposed, L_SASA = 0.543\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54_traj4.pdb: fixing CYS58 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_54_traj4.pdb: ligand too exposed, L_SASA = 0.546\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_101.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_101_traj1.pdb: terminus too close to ligand: 10.33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_101_traj2.pdb: terminus too close to ligand: 10.21\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_101_traj3.pdb: terminus too close to ligand: 10.05\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_101_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_187.pdb: terminus too close to ligand: 12.56\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_187_traj1.pdb: terminus too close to ligand: 12.91\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_187_traj2.pdb: terminus too close to ligand: 12.75\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_187_traj3.pdb: terminus too close to ligand: 13.51\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_187_traj4.pdb: terminus too close to ligand: 12.51\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_116.pdb: terminus too close to ligand: 12.15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_116_traj1.pdb: terminus too close to ligand: 12.19\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_116_traj2.pdb: longest helix too long: 38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_116_traj3.pdb: terminus too close to ligand: 12.38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_116_traj4.pdb: terminus too close to ligand: 12.50\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_160.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_160_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_160_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_160_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_160_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_17.pdb: terminus too close to ligand: 13.90\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_17_traj1.pdb: terminus too close to ligand: 13.80\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_149_traj2.pdb: ligand is too close to the backbone 2.20\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_149_traj3.pdb: ligand is too close to the backbone 2.36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_149_traj4.pdb: ligand is too close to the backbone 2.42\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_5.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_5_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_5_traj2.pdb: protein too loopy: 0.401\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_5_traj3.pdb: protein too loopy: 0.401\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_5_traj4.pdb: protein too loopy: 0.416\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_132.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_132.pdb: ligand too exposed, L_SASA = 0.321\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_132_traj1.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_132_traj2.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_132_traj3.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_132_traj4.pdb: fixing CYS35 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59.pdb: fixing CYS32 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59.pdb: ligand too exposed, L_SASA = 0.559\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59_traj1.pdb: fixing CYS32 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59_traj1.pdb: ligand too exposed, L_SASA = 0.540\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59_traj2.pdb: fixing CYS32 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59_traj2.pdb: ligand too exposed, L_SASA = 0.541\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59_traj3.pdb: fixing CYS32 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59_traj3.pdb: ligand too exposed, L_SASA = 0.537\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59_traj4.pdb: fixing CYS32 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_59_traj4.pdb: ligand too exposed, L_SASA = 0.534\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_220.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_220_traj1.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_220_traj2.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_220_traj3.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_220_traj4.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_87.pdb: ligand is too close to the backbone 2.28\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_87_traj1.pdb: ligand is too close to the backbone 2.06\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_87_traj2.pdb: ligand is too close to the backbone 2.08\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_87_traj3.pdb: ligand is too close to the backbone 2.21\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_87_traj4.pdb: ligand is too close to the backbone 2.24\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125.pdb: ligand too exposed, L_SASA = 0.383\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125_traj1.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125_traj1.pdb: ligand too exposed, L_SASA = 0.364\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125_traj2.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125_traj2.pdb: ligand too exposed, L_SASA = 0.358\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125_traj3.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125_traj3.pdb: ligand too exposed, L_SASA = 0.359\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125_traj4.pdb: fixing CYS40 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_125_traj4.pdb: ligand too exposed, L_SASA = 0.365\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123.pdb: ligand too exposed, L_SASA = 0.378\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123_traj1.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123_traj1.pdb: ligand too exposed, L_SASA = 0.378\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123_traj2.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123_traj2.pdb: ligand too exposed, L_SASA = 0.368\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123_traj3.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123_traj3.pdb: ligand too exposed, L_SASA = 0.362\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123_traj4.pdb: fixing CYS53 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_123_traj4.pdb: ligand too exposed, L_SASA = 0.353\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_83.pdb: terminus too close to ligand: 13.17\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_83_traj1.pdb: terminus too close to ligand: 13.41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_83_traj2.pdb: terminus too close to ligand: 13.40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_83_traj3.pdb: terminus too close to ligand: 13.24\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_83_traj4.pdb: terminus too close to ligand: 13.14\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_81_traj3.pdb: protein too loopy: 0.403\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_81_traj4.pdb: protein too loopy: 0.425\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_130.pdb: fixing CYS99 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_130_traj1.pdb: fixing CYS99 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_130_traj2.pdb: fixing CYS99 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_130_traj3.pdb: fixing CYS99 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_130_traj4.pdb: fixing CYS99 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_7.pdb: fixing CYS85 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_7.pdb: ligand too exposed, L_SASA = 0.392\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_7_traj1.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_7_traj2.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_7_traj3.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_7_traj4.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_145.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_145_traj1.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_145_traj2.pdb: longest helix too long: 32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_145_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_145_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_222.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_222.pdb: ligand too exposed, L_SASA = 0.309\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_222_traj1.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_222_traj2.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_222_traj3.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_222_traj4.pdb: fixing CYS70 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_174.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_174_traj1.pdb: terminus too close to ligand: 13.83\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_174_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_174_traj3.pdb: terminus too close to ligand: 13.67\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_174_traj4.pdb: terminus too close to ligand: 13.58\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_36.pdb: terminus too close to ligand: 12.13\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_36_traj1.pdb: terminus too close to ligand: 12.57\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_36_traj2.pdb: terminus too close to ligand: 12.64\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_36_traj3.pdb: terminus too close to ligand: 13.12\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_36_traj4.pdb: terminus too close to ligand: 13.16\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_102.pdb: fixing CYS94 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_102.pdb: ligand too exposed, L_SASA = 0.303\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_102_traj1.pdb: ligand is too close to the backbone 2.44\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_102_traj2.pdb: ligand is too close to the backbone 2.05\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_102_traj3.pdb: ligand is too close to the backbone 2.04\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_102_traj4.pdb: ligand is too close to the backbone 2.05\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_29.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_29_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_29_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_29_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_29_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229.pdb: ligand too exposed, L_SASA = 0.427\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229_traj1.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229_traj1.pdb: ligand too exposed, L_SASA = 0.407\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229_traj2.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229_traj2.pdb: ligand too exposed, L_SASA = 0.403\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229_traj3.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229_traj3.pdb: ligand too exposed, L_SASA = 0.391\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229_traj4.pdb: fixing CYS69 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_229_traj4.pdb: ligand too exposed, L_SASA = 0.374\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_64.pdb: fixing CYS90 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_64_traj1.pdb: fixing CYS90 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_64_traj2.pdb: fixing CYS90 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_64_traj3.pdb: fixing CYS90 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_17_traj2.pdb: terminus too close to ligand: 13.60\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_17_traj3.pdb: terminus too close to ligand: 13.74\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_17_traj4.pdb: terminus too close to ligand: 13.81\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_28.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_28_traj1.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_28_traj2.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_28_traj3.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_28_traj4.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156.pdb: ligand too exposed, L_SASA = 0.460\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156_traj1.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156_traj1.pdb: ligand too exposed, L_SASA = 0.439\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156_traj2.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156_traj2.pdb: ligand too exposed, L_SASA = 0.453\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156_traj3.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156_traj3.pdb: ligand too exposed, L_SASA = 0.459\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156_traj4.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_156_traj4.pdb: ligand too exposed, L_SASA = 0.457\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_170.pdb: terminus too close to ligand: 13.01\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_170_traj1.pdb: terminus too close to ligand: 13.32\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_170_traj2.pdb: terminus too close to ligand: 13.49\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_170_traj3.pdb: terminus too close to ligand: 14.00\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_170_traj4.pdb: terminus too close to ligand: 14.46\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202.pdb: fixing CYS50 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202.pdb: ligand too exposed, L_SASA = 0.399\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202_traj1.pdb: fixing CYS50 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202_traj1.pdb: ligand too exposed, L_SASA = 0.382\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202_traj2.pdb: fixing CYS50 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202_traj2.pdb: ligand too exposed, L_SASA = 0.376\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202_traj3.pdb: fixing CYS50 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202_traj3.pdb: ligand too exposed, L_SASA = 0.363\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202_traj4.pdb: fixing CYS50 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_202_traj4.pdb: ligand too exposed, L_SASA = 0.349\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103.pdb: fixing CYS93 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103.pdb: ligand too exposed, L_SASA = 0.318\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103_traj1.pdb: fixing CYS93 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103_traj1.pdb: ligand too exposed, L_SASA = 0.322\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103_traj2.pdb: fixing CYS93 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103_traj2.pdb: ligand too exposed, L_SASA = 0.347\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103_traj3.pdb: fixing CYS93 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103_traj3.pdb: ligand too exposed, L_SASA = 0.347\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103_traj4.pdb: fixing CYS93 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_103_traj4.pdb: ligand too exposed, L_SASA = 0.333\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211.pdb: ligand too exposed, L_SASA = 0.369\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211_traj1.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211_traj1.pdb: ligand too exposed, L_SASA = 0.347\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211_traj2.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211_traj2.pdb: ligand too exposed, L_SASA = 0.341\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211_traj3.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211_traj3.pdb: ligand too exposed, L_SASA = 0.335\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211_traj4.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_211_traj4.pdb: ligand too exposed, L_SASA = 0.340\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_223.pdb: ligand is too close to the backbone 2.36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_223_traj1.pdb: ligand is too close to the backbone 2.47\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_223_traj2.pdb: ligand is too close to the backbone 2.49\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_223_traj3.pdb: ligand is too close to the backbone 2.48\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_223_traj4.pdb: terminus too close to ligand: 13.90\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_93.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94.pdb: ligand too exposed, L_SASA = 0.328\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94_traj1.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94_traj1.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 6.021\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94_traj2.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94_traj2.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 5.791\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94_traj3.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94_traj3.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 5.907\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94_traj4.pdb: fixing CYS74 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_94_traj4.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 5.412\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195.pdb: ligand too exposed, L_SASA = 0.468\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195_traj1.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195_traj1.pdb: ligand too exposed, L_SASA = 0.388\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195_traj2.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195_traj2.pdb: ligand too exposed, L_SASA = 0.390\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195_traj3.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195_traj3.pdb: ligand too exposed, L_SASA = 0.395\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195_traj4.pdb: fixing CYS31 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_195_traj4.pdb: ligand too exposed, L_SASA = 0.384\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_20.pdb: terminus too close to ligand: 8.56\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_20_traj1.pdb: terminus too close to ligand: 8.84\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_20_traj2.pdb: terminus too close to ligand: 9.12\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_20_traj3.pdb: terminus too close to ligand: 9.79\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_20_traj4.pdb: terminus too close to ligand: 11.13\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169.pdb: fixing CYS42 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169.pdb: ligand too exposed, L_SASA = 0.392\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169_traj1.pdb: fixing CYS42 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169_traj1.pdb: ligand too exposed, L_SASA = 0.362\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169_traj2.pdb: fixing CYS42 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169_traj2.pdb: ligand too exposed, L_SASA = 0.352\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169_traj3.pdb: fixing CYS42 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169_traj3.pdb: ligand too exposed, L_SASA = 0.335\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169_traj4.pdb: fixing CYS42 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_169_traj4.pdb: ligand too exposed, L_SASA = 0.326\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_182.pdb: fixing CYS36 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_182.pdb: ligand too exposed, L_SASA = 0.324\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_182_traj1.pdb: terminus too close to ligand: 14.63\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_182_traj2.pdb: terminus too close to ligand: 14.41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_182_traj3.pdb: terminus too close to ligand: 14.61\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_182_traj4.pdb: longest helix too long: 38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_196.pdb: terminus too close to ligand: 11.63\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_196_traj1.pdb: terminus too close to ligand: 11.51\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_196_traj2.pdb: terminus too close to ligand: 12.06\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_196_traj3.pdb: terminus too close to ligand: 12.17\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_196_traj4.pdb: terminus too close to ligand: 12.47\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_176.pdb: terminus too close to ligand: 11.95\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_176_traj1.pdb: terminus too close to ligand: 11.82\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_176_traj2.pdb: terminus too close to ligand: 11.55\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_176_traj3.pdb: terminus too close to ligand: 11.66\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_176_traj4.pdb: protein too loopy: 0.420\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_215.pdb: terminus too close to ligand: 11.16\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_215_traj1.pdb: terminus too close to ligand: 11.40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_215_traj2.pdb: terminus too close to ligand: 11.50\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_215_traj3.pdb: terminus too close to ligand: 10.97\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_215_traj4.pdb: terminus too close to ligand: 10.44\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140.pdb: fixing CYS73 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140.pdb: ligand too exposed, L_SASA = 0.373\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_64_traj4.pdb: fixing CYS90 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_51.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_51.pdb: ligand too exposed, L_SASA = 0.414\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_51_traj1.pdb: ligand is too close to the backbone 2.09\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_51_traj2.pdb: ligand is too close to the backbone 2.07\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_51_traj3.pdb: ligand is too close to the backbone 2.01\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_51_traj4.pdb: ligand is too close to the backbone 2.03\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37.pdb: ligand too exposed, L_SASA = 0.423\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37_traj1.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37_traj1.pdb: ligand too exposed, L_SASA = 0.412\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37_traj2.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37_traj2.pdb: ligand too exposed, L_SASA = 0.411\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37_traj3.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37_traj3.pdb: ligand too exposed, L_SASA = 0.412\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37_traj4.pdb: fixing CYS66 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_37_traj4.pdb: ligand too exposed, L_SASA = 0.415\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_213.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_213_traj1.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_213_traj2.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_213_traj3.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_213_traj4.pdb: longest helix too long: 40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_214.pdb: longest helix too long: 45\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_214_traj1.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_214_traj2.pdb: longest helix too long: 34\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_214_traj3.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_214_traj4.pdb: longest helix too long: 33\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_100.pdb: fixing CYS95 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_100_traj1.pdb: fixing CYS95 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_100_traj2.pdb: fixing CYS95 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_100_traj3.pdb: fixing CYS95 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_100_traj4.pdb: fixing CYS95 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16.pdb: terminus too close to ligand: 13.19\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16_traj1.pdb: terminus too close to ligand: 14.53\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16_traj2.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16_traj2.pdb: ligand too exposed, L_SASA = 0.444\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16_traj3.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16_traj3.pdb: ligand too exposed, L_SASA = 0.446\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16_traj4.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_16_traj4.pdb: ligand too exposed, L_SASA = 0.448\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159.pdb: ligand too exposed, L_SASA = 0.368\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159_traj1.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159_traj1.pdb: ligand too exposed, L_SASA = 0.356\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159_traj2.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159_traj2.pdb: ligand too exposed, L_SASA = 0.350\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159_traj3.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159_traj3.pdb: ligand too exposed, L_SASA = 0.347\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159_traj4.pdb: fixing CYS61 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_159_traj4.pdb: ligand too exposed, L_SASA = 0.346\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82.pdb: ligand too exposed, L_SASA = 0.409\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82_traj1.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82_traj1.pdb: ligand too exposed, L_SASA = 0.369\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82_traj2.pdb: fixing CYS51 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82_traj2.pdb: ligand too exposed, L_SASA = 0.371\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_82_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_134.pdb: protein too loopy: 0.402\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_134_traj1.pdb: catalytic residue between loops/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140_traj1.pdb: fixing CYS73 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140_traj1.pdb: ligand too exposed, L_SASA = 0.347\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140_traj2.pdb: ligand is too close to the backbone 2.22\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140_traj3.pdb: ligand is too close to the backbone 2.38\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_140_traj4.pdb: ligand is too close to the backbone 2.40\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_146.pdb: longest helix too long: 41\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_146_traj1.pdb: longest helix too long: 35\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_146_traj2.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_146_traj3.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_146_traj4.pdb: longest helix too long: 36\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162.pdb: ligand too exposed, L_SASA = 0.549\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162_traj1.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162_traj1.pdb: ligand too exposed, L_SASA = 0.522\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162_traj2.pdb: protein too loopy: 0.412\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162_traj3.pdb: protein too loopy: 0.402\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162_traj4.pdb: fixing CYS41 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_162_traj4.pdb: ligand too exposed, L_SASA = 0.493\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148.pdb: ligand too exposed, L_SASA = 0.341\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148_traj1.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148_traj1.pdb: ligand too exposed, L_SASA = 0.322\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148_traj2.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148_traj3.pdb: terminus too close to ligand: 14.74\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_148_traj4.pdb: fixing CYS91 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189.pdb: ligand too exposed, L_SASA = 0.386\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189_traj1.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189_traj1.pdb: ligand too exposed, L_SASA = 0.335\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189_traj2.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189_traj2.pdb: ligand too exposed, L_SASA = 0.341\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189_traj3.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189_traj3.pdb: ligand too exposed, L_SASA = 0.349\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189_traj4.pdb: fixing CYS87 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_189_traj4.pdb: ligand too exposed, L_SASA = 0.346\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_184.pdb: terminus too close to ligand: 9.37\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_184_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_184_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_184_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_184_traj4.pdb: terminus too close to ligand: 10.28\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118.pdb: ligand too exposed, L_SASA = 0.379\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118_traj1.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118_traj1.pdb: ligand too exposed, L_SASA = 0.320\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118_traj2.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118_traj2.pdb: ligand too exposed, L_SASA = 0.318\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118_traj3.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118_traj3.pdb: ligand too exposed, L_SASA = 0.316\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118_traj4.pdb: fixing CYS62 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_118_traj4.pdb: ligand too exposed, L_SASA = 0.307\n",
      "\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_134_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_134_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_134_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_32.pdb: terminus too close to ligand: 13.28\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_32_traj1.pdb: terminus too close to ligand: 13.11\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_32_traj2.pdb: terminus too close to ligand: 13.07\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_32_traj3.pdb: terminus too close to ligand: 12.94\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_32_traj4.pdb: terminus too close to ligand: 12.62\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0.pdb: ligand too exposed, L_SASA = 0.320\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0_traj1.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0_traj1.pdb: ligand too exposed, L_SASA = 0.305\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0_traj2.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0_traj2.pdb: ligand too exposed, L_SASA = 0.302\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0_traj3.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0_traj3.pdb: ligand too exposed, L_SASA = 0.307\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0_traj4.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_0_traj4.pdb: ligand too exposed, L_SASA = 0.306\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_183.pdb: terminus too close to ligand: 12.56\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_183_traj1.pdb: terminus too close to ligand: 12.60\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_183_traj2.pdb: terminus too close to ligand: 12.89\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_183_traj3.pdb: terminus too close to ligand: 12.49\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_183_traj4.pdb: terminus too close to ligand: 12.60\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_128.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_128_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_128_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_128_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_128_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_93_traj1.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_93_traj2.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_93_traj3.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_93_traj4.pdb: catalytic residue between loops\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_95.pdb: terminus too close to ligand: 14.89\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_95_traj1.pdb: terminus too close to ligand: 14.89\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_95_traj2.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_95_traj3.pdb: terminus too close to ligand: 14.89\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_95_traj4.pdb: fixing CYS44 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13.pdb: ligand too exposed, L_SASA = 0.522\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13_traj1.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13_traj1.pdb: ligand too exposed, L_SASA = 0.503\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13_traj2.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13_traj2.pdb: ligand too exposed, L_SASA = 0.490\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13_traj3.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13_traj3.pdb: ligand too exposed, L_SASA = 0.499\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13_traj4.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_13_traj4.pdb: ligand too exposed, L_SASA = 0.495\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_26.pdb: terminus too close to ligand: 12.50\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_26_traj1.pdb: terminus too close to ligand: 12.31\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_26_traj2.pdb: terminus too close to ligand: 12.11\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_26_traj3.pdb: terminus too close to ligand: 11.89\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_26_traj4.pdb: terminus too close to ligand: 12.12\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_78.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_78_traj1.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_78_traj2.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_78_traj3.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_78_traj4.pdb: fixing CYS55 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_141.pdb: fixing CYS59 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_141.pdb: ligand too exposed, L_SASA = 0.407\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_141_traj1.pdb: ligand is too close to the backbone 2.26\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_141_traj2.pdb: ligand is too close to the backbone 2.19\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_141_traj3.pdb: ligand is too close to the backbone 1.99\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_141_traj4.pdb: ligand is too close to the backbone 2.09\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_180.pdb: ligand is too close to the backbone 1.87\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_180_traj1.pdb: ligand is too close to the backbone 1.64\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_180_traj2.pdb: ligand is too close to the backbone 1.78\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_180_traj3.pdb: ligand is too close to the backbone 1.69\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_180_traj4.pdb: ligand is too close to the backbone 1.66\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8.pdb: ligand too exposed, L_SASA = 0.570\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8_traj1.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8_traj1.pdb: ligand too exposed, L_SASA = 0.554\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8_traj2.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8_traj2.pdb: ligand too exposed, L_SASA = 0.544\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8_traj3.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8_traj3.pdb: ligand too exposed, L_SASA = 0.555\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8_traj4.pdb: fixing CYS68 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_8_traj4.pdb: ligand too exposed, L_SASA = 0.567\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 7.076\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152_traj1.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152_traj1.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 7.803\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152_traj2.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152_traj2.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 7.918\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152_traj3.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152_traj3.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 6.897\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152_traj4.pdb: fixing CYS57 with reference CYS15\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/7o2g_HBA/out/7o2g_HBA_dif_152_traj4.pdb: ligand atoms ['C45', 'C46', 'C47'] too buried: 7.277\n",
      "Sorting the matches took 391.829 seconds.\n",
      "     chainbreak  ...                                        description\n",
      "865    3.954522  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "385    3.942352  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "1105    3.91809  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "145    3.890351  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "980    3.904485  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "...         ...  ...                                                ...\n",
      "754    3.835683  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "274    3.825872  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "994     3.82827  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "514    3.821624  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "1077   3.859685  ...  /home/ubuntu/data/heme_binder_diffusion_edit/o...\n",
      "\n",
      "[1200 rows x 12 columns]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "### Analyzing diffusion outputs for clashes, ligand burial and scaffold quality\n",
    "## If it's running too slowly consider increasing --nproc\n",
    "\n",
    "analysis_script = f\"{SCRIPT_DIR}/scripts/diffusion_analysis/process_diffusion_outputs.py\"\n",
    "\n",
    "diffusion_outputs = []\n",
    "for d in diffusion_rundirs:\n",
    "    diffusion_outputs += glob.glob(f\"{DIFFUSION_DIR}/{d}/out/*.pdb\")\n",
    "\n",
    "# print(diffusion_outputs)\n",
    "\n",
    "# By default I don't use the --analyze flag. As a result the backbones are filtered as the script runs.\n",
    "# You can set --analyze to True to calculate all scores for all backbones.\n",
    "# This will slow the analysis down, but you can then filter the backbones separately afterwards.\n",
    "dif_analysis_cmd_dict = {\"--pdb\": \" \".join(diffusion_outputs),\n",
    "                        \"--ref\": f\"{SCRIPT_DIR}/input/*.pdb\",\n",
    "                        \"--params\": \" \".join(params),\n",
    "                        \"--term_limit\": \"15.0\",\n",
    "                        \"--SASA_limit\": \"0.3\",  # Highest allowed relative SASA of ligand\n",
    "                        \"--loop_limit\": \"0.4\",  # Fraction of backbone that can be loopy\n",
    "                        \"--ref_catres\": \"A15\",  # Position of CYS in diffusion input\n",
    "                        \"--rethread\": True,\n",
    "                        \"--fix\": True,\n",
    "                        \"--exclude_clash_atoms\": \"O1 O2 O3 O4 C5 C10\",  # Ligand atoms excluded from clashchecking because they are flexible\n",
    "                        \"--ligand_exposed_atoms\": \"C45 C46 C47\",  # Ligand atoms that need to be more exposed\n",
    "                        \"--exposed_atom_SASA\": \"10.0\",  # minimum absolute SASA for exposed ligand atoms\n",
    "                        \"--longest_helix\": \"30\",\n",
    "                        \"--rog\": \"30.0\",\n",
    "                        \"--partial\": None,\n",
    "                        \"--outdir\": None,\n",
    "                        \"--traj\": \"5/30\",  # Also random 5 models are taken from the last 30 steps of the diffusion trajectory\n",
    "                        \"--trb\": None,\n",
    "                        \"--analyze\": False,\n",
    "                        # \"--nproc\": \"1\",\n",
    "}\n",
    "\n",
    "# analysis_command = f\"{PYTHON['general']} {analysis_script}\"\n",
    "analysis_command = f\"{PYTHON['general']} {analysis_script}\"\n",
    "for k, val in dif_analysis_cmd_dict.items():\n",
    "    if val is not None:\n",
    "        if isinstance(val, list):\n",
    "            analysis_command += f\" {k}\"\n",
    "            analysis_command += \" \" + \" \".join(val)\n",
    "        elif isinstance(val, bool):\n",
    "            if val == True:\n",
    "                analysis_command += f\" {k}\"\n",
    "        else:\n",
    "            analysis_command += f\" {k} {val}\"\n",
    "        print(k, val)\n",
    "\n",
    "\n",
    "print(analysis_command)\n",
    "\n",
    "## Analyzing locally\n",
    "p = subprocess.Popen(analysis_command, shell=True)\n",
    "(output, err) = p.communicate()\n",
    "\n",
    "diffused_backbones_good = glob.glob(f\"{DIFFUSION_DIR}/filtered_structures/*.pdb\")\n",
    "\n",
    "dif_analysis_df = pd.read_csv(f\"{DIFFUSION_DIR}/diffusion_analysis.sc\", header=0, sep=\"\\s+\")\n",
    "\n",
    "\n",
    "\n",
    "####### Original\n",
    "\"\"\"\n",
    "if len(diffusion_outputs) < 100:\n",
    "    ## Analyzing locally\n",
    "    p = subprocess.Popen(analysis_command, shell=True)\n",
    "    (output, err) = p.communicate()\n",
    "else:\n",
    "    ## Too many structures to analyze.\n",
    "    ## Running the analysis as a SLURM job.\n",
    "    submit_script = \"submit_diffusion_analysis.sh\"\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"diffusion_analysis\",\n",
    "                                     mem=\"8g\", N_cores=dif_analysis_cmd_dict[\"--nproc\"], time=\"0:20:00\", email=EMAIL,\n",
    "                                     command=analysis_command, outfile_name=\"output_analysis\")\n",
    "\n",
    "diffused_backbones_good = glob.glob(f\"{DIFFUSION_DIR}/filtered_structures/*.pdb\")\n",
    "\n",
    "dif_analysis_df = pd.read_csv(f\"{DIFFUSION_DIR}/diffusion_analysis.sc\", header=0, sep=\"\\s+\")\n",
    "\"\"\"\n",
    "print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "c10b4314",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABKUAAASmCAYAAAD/KRjlAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjguMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy81sbWrAAAACXBIWXMAAA9hAAAPYQGoP6dpAAEAAElEQVR4nOzdeVwW9fr/8fctOwgkmNygpJZoJmqmHXM5iRumuWWlX83UsnMslw6pxzSzsAXMcilNz6lMTCVs0TbLxFTKg5aSlpqZFm4JUh5CVLpBnN8f/ZjTLajcCPfN8no+HvPImbnumeszwXyY6575jMUwDEMAAAAAAACAE9VydQIAAAAAAACoeShKAQAAAAAAwOkoSgEAAAAAAMDpKEoBAAAAAADA6ShKAQAAAAAAwOkoSgEAAAAAAMDpKEoBAAAAAADA6ShKAQAAAAAAwOkoSgEAAAAAAMDpKEqhWkhISJDFYtGOHTvKbZuxsbGyWCyVJp/yEBUVpcjISFenAQCVQtG5+tChQ65OpVRWrVqlFi1ayMfHRxaLRbt27XJ1SgBQZVS1c35pffzxx4qNjS3TZ0eNGqXatWuXaz5RUVGKioqyW2axWMqcI6o/ilLARTzwwAPaunWrq9MAAEC//PKL7r33Xl133XVat26dtm7dqqZNm7o6LQCAi3388ceaOXOmq9O4pK1bt+qBBx5wdRqopNxdnQBQWTVo0EANGjRwyb7Pnj0rX19fl+wbAFD5/PDDDyooKNDw4cPVpUuXS8bShwAAKpNbbrnF1SmgEuNOKVQZ33//vYYOHaqQkBB5eXnpmmuu0YgRI2Sz2cyY3NxcPfTQQ6pbt66Cg4M1aNAgHT9+3G47q1atUnR0tEJDQ+Xj46PmzZtr6tSpOnPmjF1cSY/vNWrUSH379tW6det00003ycfHR9dff71ef/31EnPOzs7Wfffdp6CgIPn5+alfv3766aef7GKKHqn7/PPP1bFjR/n6+ur++++XJJ06dUqTJ09W48aN5enpqfr16ysmJqZYri+//LJuvfVW1atXT35+fmrZsqVmz56tgoKCyx7XNWvWyNfXVw888IDOnTt32XgAqM5ef/11tW7dWt7e3goKCtIdd9yhffv2FYv74IMP1KFDB/n6+srf3189e/YsdndtUT+yc+dODRo0SAEBAQoMDNTw4cP1yy+/lDqnUaNGqXPnzpKkIUOGyGKxmI9GFD16sXv3bkVHR8vf31/du3eXJCUnJ2vAgAFq0KCBvL291aRJE40ZM0a//vprsX2Upo8FgOqmNOf8ovPswYMH1adPH9WuXVvh4eGaNGlSsXPksWPHdNddd8nf319XXXWV7rnnHm3fvl0Wi0UJCQl2sTt27FD//v0VFBQkb29vtWnTRm+99ZZdzNmzZ81rgaIc27VrpzfffNPM7eWXX5b0xyNyRZOjjyiWpm35+fl65plndP3118vLy0tXX3217rvvvlL1Z39+fM8wDPXp00fBwcE6cuSIXVtbtGih5s2bF7vWQfVGUQpVwjfffKObb75Z27Zt01NPPaVPPvlE8fHxstlsys/PN+MeeOABeXh4KDExUbNnz9bmzZs1fPhwu20dOHBAffr00ZIlS7Ru3TrFxMTorbfeUr9+/Uqdy6RJk/TII4/o/fffV6tWrTR69Gh9/vnnxWJHjx6tWrVqKTExUfPnz9dXX32lqKgo/fbbb3ZxGRkZGj58uIYNG6aPP/5YY8eO1dmzZ9WlSxctW7ZMDz/8sD755BM9+uijSkhIUP/+/WUYhvn5H3/8UcOGDdPy5cv10UcfafTo0Xr++ec1ZsyYS7Zl3rx5uvvuu/XYY4/ptddek7s7N08CqLni4+M1evRotWjRQqtXr9aLL76ob7/9Vh06dNCBAwfMuMTERA0YMEABAQF68803tWTJEmVnZysqKkpbtmwptt077rhDTZo00TvvvKPY2Fi999576tWrV6m+OJCkGTNmmBcdcXFx2rp1qxYtWmSuz8/PV//+/dWtWze9//775mMcP/74ozp06KDFixdr/fr1euKJJ/Tll1+qc+fOdvsubR8LANVJac/5klRQUKD+/fure/fuev/993X//fdr3rx5eu6558yYM2fOqGvXrtq0aZOee+45vfXWWwoJCdGQIUOK7XvTpk3q1KmTfvvtN/3rX//S+++/rxtvvFFDhgyxK15NnDhRixcv1sMPP6x169Zp+fLluvvuu3Xy5ElJf/QPd911l6Q/HpErmkJDQ0t9HErTtvPnz2vAgAGaNWuWhg0bprVr12rWrFlKTk5WVFSU8vLySr0/i8Wi5cuXy9fXV4MHDzb7o7Fjxyo9PV1vvfWW/Pz8Sr09VAMGUAV069bNuOqqq4ysrKwS1y9dutSQZIwdO9Zu+ezZsw1JRkZGRomfO3/+vFFQUGCkpKQYkoxvvvnGXPfkk08aF/6KNGzY0PD29jYOHz5sLsvLyzOCgoKMMWPGFMvnjjvusPv8f/7zH0OS8cwzz5jLunTpYkgyPvvsM7vY+Ph4o1atWsb27dvtlr/zzjuGJOPjjz8usU2FhYVGQUGB8cYbbxhubm7Gf//7X7t9tWjRwigsLDTGjx9veHp6GitWrChxOwBQ3RWdq9PT043s7GzDx8fH6NOnj13MkSNHDC8vL2PYsGGGYfxxjg0LCzNatmxpFBYWmnG5ublGvXr1jI4dO5rLivqRRx55xG6bK1euNCQ5dP7dtGmTIcl4++237ZaPHDnSkGS8/vrrl/x8UX93+PBhQ5Lx/vvvm+su18cCQHVQlnO+YfzvPPvWW2/Zxfbp08do1qyZOf/yyy8bkoxPPvnELm7MmDGGJGPp0qXmsuuvv95o06aNUVBQYBfbt29fIzQ01OxfIiMjjYEDB16yXePGjSt2zVJapW3bm2++aUgy3n33Xbu47du3G5KMRYsWmcu6dOlidOnSxS5OkvHkk0/aLduyZYvh7u5uxMTEGK+//rohyXjttdfK1A5UbdwphUrv7NmzSklJ0eDBg3X11VdfMrZ///52861atZIkHT582Fz2008/adiwYbJarXJzc5OHh4c5PkdJj2hc6MYbb9Q111xjznt7e6tp06Z2+yhyzz332M137NhRDRs21KZNm+yW16lTR926dbNb9tFHHykyMlI33nijzp07Z069evWSxWLR5s2bzdidO3eqf//+Cg4ONts0YsQIFRYW6ocffrDb7u+//66BAwdq5cqVWr9+fbEcAaAm2rp1q/Ly8jRq1Ci75eHh4erWrZs+++wzSdL+/ft1/Phx3XvvvapV639/RtWuXVt33nmntm3bprNnz9pt48Lz7ODBg+Xu7l6sL7gSd955Z7FlWVlZevDBBxUeHi53d3d5eHioYcOGkv7X3znSxwJAdVHac34Ri8VS7KmKVq1a2f39n5KSIn9/f9122212cUOHDrWbP3jwoL7//nuzb/jz3/l9+vRRRkaG9u/fL0n6y1/+ok8++URTp07V5s2bHbojqbRK07aPPvpIV111lfr162eX74033iir1Wp3XVJanTp10rPPPqv58+froYce0vDhwzV69OgrbQ6qIJ7VQaWXnZ2twsLCUg06HhwcbDfv5eUlSeYJ/PTp0/rrX/8qb29vPfPMM2ratKl8fX119OhRDRo0qFQn+gv3UbSfkj5rtVpLXFZ0y22Rkm6xPXHihA4ePCgPD48S8ygaE+TIkSP661//qmbNmunFF19Uo0aN5O3tra+++krjxo0rlldWVpaOHj2qHj16qGPHjhdvKADUIEXn5ZLOx2FhYUpOTi5V3Pnz55WdnW030PiFfYG7u7uCg4OL9QVl5evrq4CAALtl58+fV3R0tI4fP64ZM2aoZcuW8vPz0/nz53XLLbeYfYMjfSwAVBelPecX8fX1lbe3t90yLy8v/f7773bbDAkJKba9C5edOHFCkjR58mRNnjy5xPyK/s5/6aWX1KBBA61atUrPPfecvL291atXLz3//POKiIi4XDNLpTRtO3HihH777Td5enpeMl9H3XPPPZoxY4ZsNpv++c9/lmkbqPooSqHSCwoKkpubm44dO3bF29q4caOOHz+uzZs327296MIxnspLZmZmicuaNGlit+zCAdUlqW7duvLx8bnoIOp169aVJL333ns6c+aMVq9ebX4DLkm7du0q8XPXXHON5s6dqzvuuEODBg3S22+/XawjAoCapugLh4yMjGLrjh8/bp5zLxdXq1Yt1alTx255Zmam6tevb86fO3dOJ0+eLPFLjrIoqQ/Zs2ePvvnmGyUkJGjkyJHm8oMHD9rFlWcfCwBVRWnP+Y5u86uvviq2/MLrgaJtT5s2TYMGDSpxW82aNZMk+fn5aebMmZo5c6ZOnDhh3jXVr18/ff/99w7nWFZFL5Fat25diev9/f0d3mZhYaHuuece1alTR15eXho9erT+85//XLTwheqLx/dQ6fn4+KhLly56++23y1yFL1L0h3vRHVRF/v3vf1/Rdi9m5cqVdvOpqak6fPiw+dakS+nbt69+/PFHBQcHq127dsWmRo0aSSq5TYZh6NVXX73otqOjo/Xpp5/q888/V9++fXnDBYAar0OHDvLx8dGKFSvslh87dkwbN24032jXrFkz1a9fX4mJiXYvnDhz5ozeffdd8418f3ZhX/DWW2/p3LlzpeoLyqq0/V159rEAUFWU9pzviC5duig3N1effPKJ3fKkpCS7+WbNmikiIkLffPNNiX/jt2vXrsQiT0hIiEaNGqWhQ4dq//795qPiFz4ZUhH69u2rkydPqrCwsMR8i4pojnjyySf1xRdfaOXKlVq1apW++eYb7paqobhTClXC3Llz1blzZ7Vv315Tp05VkyZNdOLECX3wwQcOFZQ6duyoOnXq6MEHH9STTz4pDw8PrVy5Ut98802F5L1jxw498MADuvvuu3X06FFNnz5d9evX19ixYy/72ZiYGL377ru69dZb9cgjj6hVq1Y6f/68jhw5ovXr12vSpElq3769evbsKU9PTw0dOlRTpkzR77//rsWLFys7O/uS2+/cubM+++wz3XbbbYqOjtbHH3+swMDA8mo6AFQpV111lWbMmKHHHntMI0aM0NChQ3Xy5EnNnDlT3t7eevLJJyVJtWrV0uzZs3XPPfeob9++GjNmjGw2m55//nn99ttvmjVrVrFtr169Wu7u7urZs6f27t2rGTNmqHXr1ho8eHCFtef666/Xddddp6lTp8owDAUFBenDDz8s9kiKdPk+tizfgANAZVbac74jRo4cqXnz5mn48OF65pln1KRJE33yySf69NNPJcluHMJ///vf6t27t3r16qVRo0apfv36+u9//6t9+/bp66+/1ttvvy1Jat++vfr27atWrVqpTp062rdvn5YvX273BUjLli0lSc8995x69+4tNzc3tWrVqlzvOPq///s/rVy5Un369NE//vEP/eUvf5GHh4eOHTumTZs2acCAAbrjjjtKvb3k5GTFx8drxowZZgEwPj5ekydPVlRUlEPbQtXHnVKoElq3bq2vvvpKbdu21bRp03Tbbbfp0UcflZeXl0Mn3ODgYK1du1a+vr4aPny47r//ftWuXVurVq2qkLyXLFmi/Px8/d///Z8efvhhtWvXTps3b1ZQUNBlP+vn56cvvvhCo0aN0iuvvKLbb79dgwcPNp8tL7pT6vrrr9e7776r7OxsDRo0SBMmTNCNN96ol1566bL7aNeunVJSUvTTTz+pW7dufEsOoEabNm2aXnvtNX3zzTcaOHCgxo8frxYtWig1NdVu7I5hw4bpvffe08mTJzVkyBDdd999CggI0KZNm9S5c+di2129erW+//57DRo0SE888YT69eun9evXV+gjCh4eHvrwww/VtGlTjRkzRkOHDlVWVpY2bNhQLLa8+lgAqEpKe84vLT8/P23cuFFRUVGaMmWK7rzzTh05ckSLFi2S9EchrEjXrl311Vdf6aqrrlJMTIx69Oihhx56SBs2bFCPHj3MuG7duumDDz7Qfffdp+joaM2ePVsjRozQhx9+aMYMGzZMDzzwgBYtWqQOHTro5ptv1vHjx8t+YErg5uamDz74QI899phWr16tO+64QwMHDtSsWbPk7e1tFsZKIyMjQ8OHD1dUVJSeeOIJc/nEiRPVr18/3X///Tp06FC55o/KzWL8+d5zAAAAlIvY2FjNnDlTv/zyS5nGJwEAVH1xcXF6/PHHdeTIEV4qAZSAx/cAAAAAALhCCxculPTHkwwFBQXauHGjXnrpJQ0fPpyCFHARFKUAAABc5Pz58zp//vwlY9zd+XMNAKoCX19fzZs3T4cOHZLNZtM111yjRx99VI8//rhT86BvQVXC43sAAAAuUvSI36Wkp6eb4wgCAHA5o0aN0rJlyy4ZQxkAlQVFKQAAABc5fvz4ZQekLe+3KAEAqrdDhw5d9gVG7dq1c1I2wKVRlAIAAAAAAIDT1XJ1AgAAAAAAAKh5quToZufPn9fx48fl7+8vi8Xi6nQAoEoxDEO5ubkKCwtTrVoV/93E4sWLtXjxYh06dEiS1KJFCz3xxBPq3bu3mc/MmTP1yiuvKDs7W+3bt9fLL7+sFi1amNuw2WyaPHmy3nzzTeXl5al79+5atGiRQ2+yoe8AgLJzdt9RGdBvAEDZlbbfqJKP7x07dkzh4eGuTgMAqrSjR4865fXEH374odzc3NSkSRNJ0rJly/T8889r586datGihZ577jk9++yzSkhIUNOmTfXMM8/o888/1/79++Xv7y9Jeuihh/Thhx8qISFBwcHBmjRpkv773/8qLS1Nbm5upcqDvgMArpyz+o7KgH4DAK7c5fqNKlmUysnJ0VVXXaWjR48qICDA1ekAQJVy6tQphYeH67ffflNgYKBLcggKCtLzzz+v+++/X2FhYYqJidGjjz4q6Y+7okJCQvTcc89pzJgxysnJ0dVXX63ly5dryJAhkv4YHDo8PFwff/yxevXqVap90ncAQNlVhr7D2eg3AKDsSttvVMnH94punw0ICKCDAIAycsWjCIWFhXr77bd15swZdejQQenp6crMzFR0dLQZ4+XlpS5duig1NVVjxoxRWlqaCgoK7GLCwsIUGRmp1NTUUhel6DsA4MrVpMfY6DcA4Mpdrt+okkUpAEDVsnv3bnXo0EG///67ateurTVr1uiGG25QamqqJCkkJMQuPiQkRIcPH5YkZWZmytPTU3Xq1CkWk5mZedF92mw22Ww2c/7UqVPl1RwAAAAA5aBmjFIIAHCpZs2aadeuXdq2bZseeughjRw5Ut999525/sJvUAzDuOy3KpeLiY+PV2BgoDkxLggAAABQuVCUAgBUOE9PTzVp0kTt2rVTfHy8WrdurRdffFFWq1WSit3xlJWVZd49ZbValZ+fr+zs7IvGlGTatGnKyckxp6NHj5ZzqwAAAABcCYpSAACnMwxDNptNjRs3ltVqVXJysrkuPz9fKSkp6tixoySpbdu28vDwsIvJyMjQnj17zJiSeHl5meOAMB4IAAAAUPkwphQAoEI99thj6t27t8LDw5Wbm6ukpCRt3rxZ69atk8ViUUxMjOLi4hQREaGIiAjFxcXJ19dXw4YNkyQFBgZq9OjRmjRpkoKDgxUUFKTJkyerZcuW6tGjh4tbBwAAAKCsKEoBACrUiRMndO+99yojI0OBgYFq1aqV1q1bp549e0qSpkyZory8PI0dO1bZ2dlq37691q9fL39/f3Mb8+bNk7u7uwYPHqy8vDx1795dCQkJcnNzc1WzAAAAAFwhi2EYhquTcNSpU6cUGBionJwcHscAAAfV1HNoTW03AJSHmngOrYltBoDyUtpzKGNKAQAAAAAAwOl4fM/JGk1d67J9H5p1u8v2DQCAI+gvAQBVBX0WUHbcKQUAAAAAAACnc7go9fPPP2v48OEKDg6Wr6+vbrzxRqWlpZnrDcNQbGyswsLC5OPjo6ioKO3du9duGzabTRMmTFDdunXl5+en/v3769ixY1feGgAAAAAAAFQJDhWlsrOz1alTJ3l4eOiTTz7Rd999pzlz5uiqq64yY2bPnq25c+dq4cKF2r59u6xWq3r27Knc3FwzJiYmRmvWrFFSUpK2bNmi06dPq2/fviosLCy3hgEAAAAAAKDycmhMqeeee07h4eFaunSpuaxRo0bmvw3D0Pz58zV9+nQNGjRIkrRs2TKFhIQoMTFRY8aMUU5OjpYsWaLly5erR48ekqQVK1YoPDxcGzZsUK9evcqhWQAAAAAAAKjMHLpT6oMPPlC7du109913q169emrTpo1effVVc316eroyMzMVHR1tLvPy8lKXLl2UmpoqSUpLS1NBQYFdTFhYmCIjI80YAAAAAAAAVG8OFaV++uknLV68WBEREfr000/14IMP6uGHH9Ybb7whScrMzJQkhYSE2H0uJCTEXJeZmSlPT0/VqVPnojEXstlsOnXqlN0EAAAAAACAqsuhx/fOnz+vdu3aKS4uTpLUpk0b7d27V4sXL9aIESPMOIvFYvc5wzCKLbvQpWLi4+M1c+ZMR1IFAAAAAABAJebQnVKhoaG64YYb7JY1b95cR44ckSRZrVZJKnbHU1ZWlnn3lNVqVX5+vrKzsy8ac6Fp06YpJyfHnI4ePepI2gAAAAAAAKhkHCpKderUSfv377db9sMPP6hhw4aSpMaNG8tqtSo5Odlcn5+fr5SUFHXs2FGS1LZtW3l4eNjFZGRkaM+ePWbMhby8vBQQEGA3AQAAAAAAoOpy6PG9Rx55RB07dlRcXJwGDx6sr776Sq+88opeeeUVSX88thcTE6O4uDhFREQoIiJCcXFx8vX11bBhwyRJgYGBGj16tCZNmqTg4GAFBQVp8uTJatmypfk2PgAAAAAAAFRvDhWlbr75Zq1Zs0bTpk3TU089pcaNG2v+/Pm65557zJgpU6YoLy9PY8eOVXZ2ttq3b6/169fL39/fjJk3b57c3d01ePBg5eXlqXv37kpISJCbm1v5tQwAAAAAAACVlkNFKUnq27ev+vbte9H1FotFsbGxio2NvWiMt7e3FixYoAULFji6ewAAAAAAAFQDDo0pBQAAAAAAAJQHilIAAAAAAABwOopSAAAAAAAAcDqKUgAAAAAAAHA6ilIAAAAAAABwOopSAAAAAAAAcDqKUgAAAAAAAHA6ilIAAAAAAABwOndXJwAAACqnRlPXujoFAAAAVGPcKQUAAAAAAACnoygFAAAAAAAAp6MoBQCoMPHx8br55pvl7++vevXqaeDAgdq/f79dzKhRo2SxWOymW265xS7GZrNpwoQJqlu3rvz8/NS/f38dO3bMmU0BAAAAUM4oSgEAKkxKSorGjRunbdu2KTk5WefOnVN0dLTOnDljF3fbbbcpIyPDnD7++GO79TExMVqzZo2SkpK0ZcsWnT59Wn379lVhYaEzmwMAAACgHDHQOQCgwqxbt85ufunSpapXr57S0tJ06623msu9vLxktVpL3EZOTo6WLFmi5cuXq0ePHpKkFStWKDw8XBs2bFCvXr0qrgEAAAAAKgx3SgEAnCYnJ0eSFBQUZLd88+bNqlevnpo2baq//e1vysrKMtelpaWpoKBA0dHR5rKwsDBFRkYqNTX1ovuy2Ww6deqU3QQAAACg8qAoBQBwCsMwNHHiRHXu3FmRkZHm8t69e2vlypXauHGj5syZo+3bt6tbt26y2WySpMzMTHl6eqpOnTp22wsJCVFmZuZF9xcfH6/AwEBzCg8Pr5iGAQAAACgTHt8DADjF+PHj9e2332rLli12y4cMGWL+OzIyUu3atVPDhg21du1aDRo06KLbMwxDFovlouunTZumiRMnmvOnTp2iMAUAAABUItwpBQCocBMmTNAHH3ygTZs2qUGDBpeMDQ0NVcOGDXXgwAFJktVqVX5+vrKzs+3isrKyFBISctHteHl5KSAgwG4CAAAAUHlQlAIAVBjDMDR+/HitXr1aGzduVOPGjS/7mZMnT+ro0aMKDQ2VJLVt21YeHh5KTk42YzIyMrRnzx517NixwnIHAAAAULEoSgEAKsy4ceO0YsUKJSYmyt/fX5mZmcrMzFReXp4k6fTp05o8ebK2bt2qQ4cOafPmzerXr5/q1q2rO+64Q5IUGBio0aNHa9KkSfrss8+0c+dODR8+XC1btjTfxgcAqD4WL16sVq1amXe5dujQQZ988om53jAMxcbGKiwsTD4+PoqKitLevXvttmGz2TRhwgTVrVtXfn5+6t+/v44dO+bspgAALoOiFACgwixevFg5OTmKiopSaGioOa1atUqS5Obmpt27d2vAgAFq2rSpRo4cqaZNm2rr1q3y9/c3tzNv3jwNHDhQgwcPVqdOneTr66sPP/xQbm5urmoaAKCCNGjQQLNmzdKOHTu0Y8cOdevWTQMGDDALT7Nnz9bcuXO1cOFCbd++XVarVT179lRubq65jZiYGK1Zs0ZJSUnasmWLTp8+rb59+6qwsNBVzQIAlICBzgEAFcYwjEuu9/Hx0aeffnrZ7Xh7e2vBggVasGBBeaUGAKik+vXrZzf/7LPPavHixdq2bZtuuOEGzZ8/X9OnTzdfhrFs2TKFhIQoMTFRY8aMUU5OjpYsWaLly5ebd9SuWLFC4eHh2rBhg3r16uX0NgEASsadUgAAAAAqpcLCQiUlJenMmTPq0KGD0tPTlZmZqejoaDPGy8tLXbp0UWpqqiQpLS1NBQUFdjFhYWGKjIw0Y0pis9l06tQpuwkAULEcKkrFxsbKYrHYTVar1VzP890AAAAArtTu3btVu3ZteXl56cEHH9SaNWt0ww03KDMzU5KKvX01JCTEXJeZmSlPT0/VqVPnojEliY+PV2BgoDmFh4eXc6sAABdy+E6pFi1aKCMjw5x2795truP5bgAAAABXqlmzZtq1a5e2bdumhx56SCNHjtR3331nrrdYLHbxhmEUW3ahy8VMmzZNOTk55nT06NErawQA4LIcHlPK3d3d7u6oIoZh8Hw3AAAAgCvm6empJk2aSJLatWun7du368UXX9Sjjz4q6Y+7oUJDQ834rKws8+4pq9Wq/Px8ZWdn290tlZWVpY4dO150n15eXvLy8qqI5gAALsLhO6UOHDigsLAwNW7cWP/3f/+nn376SZJ4vhsAAABAhTAMQzabTY0bN5bValVycrK5Lj8/XykpKWbBqW3btvLw8LCLycjI0J49ey5ZlAIAOJ9Dd0q1b99eb7zxhpo2baoTJ07omWeeUceOHbV3795LPt99+PBhSVf2fPfMmTMdSRUAAABAFfTYY4+pd+/eCg8PV25urpKSkrR582atW7dOFotFMTExiouLU0REhCIiIhQXFydfX18NGzZMkhQYGKjRo0dr0qRJCg4OVlBQkCZPnqyWLVuaT2sAACoHh4pSvXv3Nv/dsmVLdejQQdddd52WLVumW265RVLFPd89ceJEc/7UqVMMPAgAAABUQydOnNC9996rjIwMBQYGqlWrVlq3bp169uwpSZoyZYry8vI0duxYZWdnq3379lq/fr38/f3NbcybN0/u7u4aPHiw8vLy1L17dyUkJMjNzc1VzQIAlMDhMaX+zM/PTy1bttSBAwc0cOBASTzfDQAAAKDslixZcsn1FotFsbGxio2NvWiMt7e3FixYoAULFpRzdgCA8uTwmFJ/ZrPZtG/fPoWGhvJ8NwAAAAAAAErNoTulJk+erH79+umaa65RVlaWnnnmGZ06dUojR47k+W4AAAAAAACUmkNFqWPHjmno0KH69ddfdfXVV+uWW27Rtm3b1LBhQ0k83w0AAAAAAIDScagolZSUdMn1PN8NAAAAAACA0riiMaUAAAAAAACAsqAoBQAAAAAAAKejKAUAAAAAAACnoygFAAAAAAAAp6MoBQAAAAAAAKejKAUAAAAAAACnoygFAAAAAAAAp6MoBQAAAAAAAKejKAUAAAAAAACnc3d1Aq7SaOpaV6cAAAAAAABQY3GnFAAAAAAAAJyOohQAAAAAAACcjqIUAKDCxMfH6+abb5a/v7/q1aungQMHav/+/XYxhmEoNjZWYWFh8vHxUVRUlPbu3WsXY7PZNGHCBNWtW1d+fn7q37+/jh075symAAAAAChnFKUAABUmJSVF48aN07Zt25ScnKxz584pOjpaZ86cMWNmz56tuXPnauHChdq+fbusVqt69uyp3NxcMyYmJkZr1qxRUlKStmzZotOnT6tv374qLCx0RbMAAAAAlIMaO9A5AKDirVu3zm5+6dKlqlevntLS0nTrrbfKMAzNnz9f06dP16BBgyRJy5YtU0hIiBITEzVmzBjl5ORoyZIlWr58uXr06CFJWrFihcLDw7Vhwwb16tXL6e0CAAAAcOW4UwoA4DQ5OTmSpKCgIElSenq6MjMzFR0dbcZ4eXmpS5cuSk1NlSSlpaWpoKDALiYsLEyRkZFmDAAAAICqhzulAABOYRiGJk6cqM6dOysyMlKSlJmZKUkKCQmxiw0JCdHhw4fNGE9PT9WpU6dYTNHnS2Kz2WSz2cz5U6dOlUs7AAAAAJQP7pQCADjF+PHj9e233+rNN98sts5isdjNG4ZRbNmFLhcTHx+vwMBAcwoPDy9b4gAAAAAqBEUpAECFmzBhgj744ANt2rRJDRo0MJdbrVZJKnbHU1ZWlnn3lNVqVX5+vrKzsy8aU5Jp06YpJyfHnI4ePVpezQEAAABQDnh8DwBQYQzD0IQJE7RmzRpt3rxZjRs3tlvfuHFjWa1WJScnq02bNpKk/Px8paSk6LnnnpMktW3bVh4eHkpOTtbgwYMlSRkZGdqzZ49mz5590X17eXnJy8urglqG6qzR1LUu2e+hWbe7ZL8AAACuQlEKAFBhxo0bp8TERL3//vvy9/c374gKDAyUj4+PLBaLYmJiFBcXp4iICEVERCguLk6+vr4aNmyYGTt69GhNmjRJwcHBCgoK0uTJk9WyZUvzbXwAAAAAqh6KUgCACrN48WJJUlRUlN3ypUuXatSoUZKkKVOmKC8vT2PHjlV2drbat2+v9evXy9/f34yfN2+e3N3dNXjwYOXl5al79+5KSEiQm5ubs5oCAAAAoJxd0ZhS8fHx5rfcRQzDUGxsrMLCwuTj46OoqCjt3bvX7nM2m00TJkxQ3bp15efnp/79++vYsWNXkgoAoBIyDKPEqaggJf0xyHlsbKwyMjL0+++/KyUlxXw7XxFvb28tWLBAJ0+e1NmzZ/Xhhx8ycDkAAABQxZW5KLV9+3a98soratWqld3y2bNna+7cuVq4cKG2b98uq9Wqnj17Kjc314yJiYnRmjVrlJSUpC1btuj06dPq27evCgsLy94SAAAAAAAAVBllenzv9OnTuueee/Tqq6/qmWeeMZcbhqH58+dr+vTpGjRokCRp2bJlCgkJUWJiosaMGaOcnBwtWbJEy5cvN8cCWbFihcLDw7Vhwwb16tWrHJoFAAAAAED1xss5UNWV6U6pcePG6fbbby82wGx6eroyMzMVHR1tLvPy8lKXLl2UmpoqSUpLS1NBQYFdTFhYmCIjI82YC9lsNp06dcpuAgAAAAAAQNXl8J1SSUlJ+vrrr7V9+/Zi64reqhQSEmK3PCQkRIcPHzZjPD09VadOnWIxRZ+/UHx8vGbOnOloqgAAAAAAAKikHLpT6ujRo/rHP/6hFStWyNvb+6JxFovFbt4wjGLLLnSpmGnTpiknJ8ecjh496kjaAAAAAAAAqGQcKkqlpaUpKytLbdu2lbu7u9zd3ZWSkqKXXnpJ7u7u5h1SF97xlJWVZa6zWq3Kz89Xdnb2RWMu5OXlpYCAALsJAAAAAAAAVZdDRanu3btr9+7d2rVrlzm1a9dO99xzj3bt2qVrr71WVqtVycnJ5mfy8/OVkpKijh07SpLatm0rDw8Pu5iMjAzt2bPHjAEAAAAAAED15tCYUv7+/oqMjLRb5ufnp+DgYHN5TEyM4uLiFBERoYiICMXFxcnX11fDhg2TJAUGBmr06NGaNGmSgoODFRQUpMmTJ6tly5bFBk4HAAAAAABA9eTwQOeXM2XKFOXl5Wns2LHKzs5W+/bttX79evn7+5sx8+bNk7u7uwYPHqy8vDx1795dCQkJcnNzK+90AAAAAADVXKOpa12dAoAyuOKi1ObNm+3mLRaLYmNjFRsbe9HPeHt7a8GCBVqwYMGV7h4AAAAAAABVkENjSgEAAAAAAADlgaIUAAAAAAAAnI6iFAAAAAAAAJyOohQAAAAAAACcjqIUAAAAAAAAnI6iFAAAAAAAAJyOohQAAAAAAACcjqIUAAAAAAAAnI6iFAAAAAAAAJyOohQAAACASiM+Pl4333yz/P39Va9ePQ0cOFD79++3izEMQ7GxsQoLC5OPj4+ioqK0d+9euxibzaYJEyaobt268vPzU//+/XXs2DFnNgUAcBkUpQAAAABUGikpKRo3bpy2bdum5ORknTt3TtHR0Tpz5owZM3v2bM2dO1cLFy7U9u3bZbVa1bNnT+Xm5poxMTExWrNmjZKSkrRlyxadPn1affv2VWFhoSuaBQAogburEwAAAACAIuvWrbObX7p0qerVq6e0tDTdeuutMgxD8+fP1/Tp0zVo0CBJ0rJlyxQSEqLExESNGTNGOTk5WrJkiZYvX64ePXpIklasWKHw8HBt2LBBvXr1cnq7AADFcacUAAAAgEorJydHkhQUFCRJSk9PV2ZmpqKjo80YLy8vdenSRampqZKktLQ0FRQU2MWEhYUpMjLSjLmQzWbTqVOn7CYAQMWiKAUAAACgUjIMQxMnTlTnzp0VGRkpScrMzJQkhYSE2MWGhISY6zIzM+Xp6ak6depcNOZC8fHxCgwMNKfw8PDybg4A4AIUpQAAAABUSuPHj9e3336rN998s9g6i8ViN28YRrFlF7pUzLRp05STk2NOR48eLXviAIBSoSgFAKhQn3/+ufr166ewsDBZLBa99957dutHjRoli8ViN91yyy12MbxBCQBqngkTJuiDDz7Qpk2b1KBBA3O51WqVpGJ3PGVlZZl3T1mtVuXn5ys7O/uiMRfy8vJSQECA3QQAqFgUpQAAFerMmTNq3bq1Fi5ceNGY2267TRkZGeb08ccf263nDUoAUHMYhqHx48dr9erV2rhxoxo3bmy3vnHjxrJarUpOTjaX5efnKyUlRR07dpQktW3bVh4eHnYxGRkZ2rNnjxkDAHA93r4HAKhQvXv3Vu/evS8Z4+XlZX7zfSHeoAQANcu4ceOUmJio999/X/7+/uYdUYGBgfLx8ZHFYlFMTIzi4uIUERGhiIgIxcXFydfXV8OGDTNjR48erUmTJik4OFhBQUGaPHmyWrZsafYlAADXoygFAHC5zZs3q169errqqqvUpUsXPfvss6pXr56ky79BiaIUAFQvixcvliRFRUXZLV+6dKlGjRolSZoyZYry8vI0duxYZWdnq3379lq/fr38/f3N+Hnz5snd3V2DBw9WXl6eunfvroSEBLm5uTmrKQCAy6AoBQBwqd69e+vuu+9Ww4YNlZ6erhkzZqhbt25KS0uTl5dXmd6gJP0xDpXNZjPnebU3AFQNhmFcNsZisSg2NlaxsbEXjfH29taCBQu0YMGCcswOAFCeKEoBAFxqyJAh5r8jIyPVrl07NWzYUGvXrtWgQYMu+rnLvWUpPj5eM2fOLNdcAQAAAJQfBjoHAFQqoaGhatiwoQ4cOCCpbG9Qkni1NwAAAFDZOVSUWrx4sVq1amW+IrVDhw765JNPzPWGYSg2NlZhYWHy8fFRVFSU9u7da7cNXusNALiUkydP6ujRowoNDZVU9jco8WpvAAAAoHJzqCjVoEEDzZo1Szt27NCOHTvUrVs3DRgwwCw8zZ49W3PnztXChQu1fft2Wa1W9ezZU7m5ueY2eK03ANQsp0+f1q5du7Rr1y5JUnp6unbt2qUjR47o9OnTmjx5srZu3apDhw5p8+bN6tevn+rWras77rhDkv0blD777DPt3LlTw4cP5w1KAAAAQBXn0JhS/fr1s5t/9tlntXjxYm3btk033HCD5s+fr+nTp5tjgCxbtkwhISFKTEzUmDFjeK03ANRAO3bsUNeuXc35iRMnSpJGjhypxYsXa/fu3XrjjTf022+/KTQ0VF27dtWqVat4gxIAAABQzZV5oPPCwkK9/fbbOnPmjDp06KD09HRlZmbavbLby8tLXbp0UWpqqsaMGcNrvQGgBoqKirrkm5Q+/fTTy26DNygBAAAA1Y/DRandu3erQ4cO+v3331W7dm2tWbNGN9xwg1JTUyWp2KCzISEhOnz4sCTxWm8AAAAAAABIKsPb95o1a6Zdu3Zp27ZteuihhzRy5Eh999135voLX899uVd2lyYmPj5egYGB5hQeHu5o2gAAAAAAAKhEHC5KeXp6qkmTJmrXrp3i4+PVunVrvfjii7JarZJU7I6nP7+ym9d6AwAAAAAAQCpDUepChmHIZrOpcePGslqtdq/szs/PV0pKivnKbl7rDQAAAAAAAMnBMaUee+wx9e7dW+Hh4crNzVVSUpI2b96sdevWyWKxKCYmRnFxcYqIiFBERITi4uLk6+urYcOGSbJ/rXdwcLCCgoI0efJkXusNAAAAAABQwzhUlDpx4oTuvfdeZWRkKDAwUK1atdK6devUs2dPSdKUKVOUl5ensWPHKjs7W+3bt9f69et5rTcAAAAAAADsOFSUWrJkySXXWywWxcbGKjY29qIxvNYbAAAAAAAAVzymFAAAAAAAAOAoilIAAAAAAABwOopSAAAAAAAAcDqKUgAAAAAAAHA6ilIAAAAAAABwOopSAAAAAAAAcDqKUgAAAAAAAHA6ilIAAAAAAABwOopSAAAAAAAAcDqKUgAAAAAAAHA6d1cnAAAAANdqNHWtS/Z7aNbtLtkvAACoHLhTCgAAAAAAAE5HUQoAAAAAAABOR1EKAAAAAAAATkdRCgAAAAAAAE5HUQoAAAAAAABOx9v3AAAAAABAqbnqra0Sb26tbrhTCgAAAAAAAE5HUQoAUKE+//xz9evXT2FhYbJYLHrvvffs1huGodjYWIWFhcnHx0dRUVHau3evXYzNZtOECRNUt25d+fn5qX///jp27JgTWwEAAACgvFGUAgBUqDNnzqh169ZauHBhietnz56tuXPnauHChdq+fbusVqt69uyp3NxcMyYmJkZr1qxRUlKStmzZotOnT6tv374qLCx0VjMAAAAAlDPGlAIAVKjevXurd+/eJa4zDEPz58/X9OnTNWjQIEnSsmXLFBISosTERI0ZM0Y5OTlasmSJli9frh49ekiSVqxYofDwcG3YsEG9evVyWluAiuTK8TkAAABcgTulAAAuk56erszMTEVHR5vLvLy81KVLF6WmpkqS0tLSVFBQYBcTFhamyMhIM6YkNptNp06dspsAAAAAVB4UpQAALpOZmSlJCgkJsVseEhJirsvMzJSnp6fq1Klz0ZiSxMfHKzAw0JzCw8PLOXsAAAAAV8KholR8fLxuvvlm+fv7q169eho4cKD2799vF8OAtQAAR1ksFrt5wzCKLbvQ5WKmTZumnJwcczp69Gi55AoAAACgfDhUlEpJSdG4ceO0bds2JScn69y5c4qOjtaZM2fMGAasBQCUltVqlaRidzxlZWWZd09ZrVbl5+crOzv7ojEl8fLyUkBAgN0EAAAAoPJwqCi1bt06jRo1Si1atFDr1q21dOlSHTlyRGlpaZKKD1gbGRmpZcuW6ezZs0pMTJQkc8DaOXPmqEePHmrTpo1WrFih3bt3a8OGDeXfQgBApdW4cWNZrVYlJyeby/Lz85WSkqKOHTtKktq2bSsPDw+7mIyMDO3Zs8eMAQAAAFD1XNGYUjk5OZKkoKAgSRU3YC2D1QJA1XX69Gnt2rVLu3btkvRHX7Fr1y4dOXJEFotFMTExiouL05o1a7Rnzx6NGjVKvr6+GjZsmCQpMDBQo0eP1qRJk/TZZ59p586dGj58uFq2bGm+jQ8AAABA1eNe1g8ahqGJEyeqc+fOioyMlHTpAWsPHz5sxjg6YG18fLxmzpxZ1lQBAC60Y8cOde3a1ZyfOHGiJGnkyJFKSEjQlClTlJeXp7Fjxyo7O1vt27fX+vXr5e/vb35m3rx5cnd31+DBg5WXl6fu3bsrISFBbm5uTm8PAAAAgPJR5qLU+PHj9e2332rLli3F1pX3gLXTpk0zL2Ik6dSpU7xFCQCqiKioKBmGcdH1FotFsbGxio2NvWiMt7e3FixYoAULFlRAhgAAAABcoUyP702YMEEffPCBNm3apAYNGpjLK2rAWgarBQAAAAAAqF4cKkoZhqHx48dr9erV2rhxoxo3bmy3ngFrAQAAAAAAUBoOPb43btw4JSYm6v3335e/v795R1RgYKB8fHzsBqyNiIhQRESE4uLiLjpgbXBwsIKCgjR58mQGrAUAAAAAAKhBHLpTavHixcrJyVFUVJRCQ0PNadWqVWbMlClTFBMTo7Fjx6pdu3b6+eefSxywduDAgRo8eLA6deokX19fffjhhwxYCwAAAECff/65+vXrp7CwMFksFr333nt26w3DUGxsrMLCwuTj46OoqCjt3bvXLsZms2nChAmqW7eu/Pz81L9/fx07dsyJrQAAXI7Dj++VNI0aNcqMKRqwNiMjQ7///rtSUlLMt/MVKRqw9uTJkzp79qw+/PBDBi4HAAAAIEk6c+aMWrdurYULF5a4fvbs2Zo7d64WLlyo7du3y2q1qmfPnsrNzTVjYmJitGbNGiUlJWnLli06ffq0+vbtq8LCQmc1AwBwGWV++x4AAAAAVITevXurd+/eJa4zDEPz58/X9OnTNWjQIEnSsmXLFBISosTERI0ZM0Y5OTlasmSJli9fbg4RsmLFCoWHh2vDhg3q1auX09oCALi4Mr19DwAAAABcIT09XZmZmYqOjjaXeXl5qUuXLkpNTZUkpaWlqaCgwC4mLCxMkZGRZgwAwPW4UwoAgEqu0dS1rk4BqBCu/Nk+NOt2l+0bV6boZUshISF2y0NCQnT48GEzxtPTU3Xq1CkWU/T5C9lsNtlsNnP+1KlT5Zk2AKAE3CkFAAAAoMqxWCx284ZhFFt2oUvFxMfHKzAw0JwY8xYAKh5FKQAAAABVhtVqlaRidzxlZWWZd09ZrVbl5+crOzv7ojEXmjZtmnJycszp6NGjFZA9AODPKEoBAAAAqDIaN24sq9Wq5ORkc1l+fr5SUlLUsWNHSVLbtm3l4eFhF5ORkaE9e/aYMRfy8vJSQECA3QQAqFiMKQUAAACgUjl9+rQOHjxozqenp2vXrl0KCgrSNddco5iYGMXFxSkiIkIRERGKi4uTr6+vhg0bJkkKDAzU6NGjNWnSJAUHBysoKEiTJ09Wy5YtzbfxAQBcj6IUAAAAgEplx44d6tq1qzk/ceJESdLIkSOVkJCgKVOmKC8vT2PHjlV2drbat2+v9evXy9/f3/zMvHnz5O7ursGDBysvL0/du3dXQkKC3NzcnN4eAEDJKEoBAAAAqFSioqJkGMZF11ssFsXGxio2NvaiMd7e3lqwYIEWLFhQARkCAMoDY0oBAAAAAADA6ShKAQAAAAAAwOkoSgEAAAAAAMDpKEoBAAAAAADA6ShKAQAAAAAAwOkoSgEAAAAAAMDpKEoBAAAAAADA6ShKAQAAAAAAwOkoSgEAAAAAAMDpKEoBAAAAAADA6ShKAQAAAAAAwOkoSgEAXCo2NlYWi8Vuslqt5nrDMBQbG6uwsDD5+PgoKipKe/fudWHGAAAAAMoDRSkAgMu1aNFCGRkZ5rR7925z3ezZszV37lwtXLhQ27dvl9VqVc+ePZWbm+vCjAEAAABcKYeLUp9//rn69eunsLAwWSwWvffee3brS/ONts1m04QJE1S3bl35+fmpf//+Onbs2BU1BABQdbm7u8tqtZrT1VdfLemPPmX+/PmaPn26Bg0apMjISC1btkxnz55VYmKii7MGAAAAcCXcHf3AmTNn1Lp1a91333268847i60v+kY7ISFBTZs21TPPPKOePXtq//798vf3lyTFxMToww8/VFJSkoKDgzVp0iT17dtXaWlpcnNzu/JWAQCqlAMHDigsLExeXl5q37694uLidO211yo9PV2ZmZmKjo42Y728vNSlSxelpqZqzJgxF92mzWaTzWYz50+dOlWhbQBQtTSautZl+z4063aX7RsAgMrE4aJU79691bt37xLXXfiNtiQtW7ZMISEhSkxM1JgxY5STk6MlS5Zo+fLl6tGjhyRpxYoVCg8P14YNG9SrV68raA4AoKpp37693njjDTVt2lQnTpzQM888o44dO2rv3r3KzMyUJIWEhNh9JiQkRIcPH77kduPj4zVz5swKyxsAAADAlXG4KHUppflGOy0tTQUFBXYxYWFhioyMVGpqaolFKb7tBoDq689fdLRs2VIdOnTQddddp2XLlumWW26RJFksFrvPGIZRbNmFpk2bpokTJ5rzp06dUnh4eDlmDgDApbnqjjzuxgNQVZRrUao032hnZmbK09NTderUKRZT9PkL8W13+aBTBFAV+Pn5qWXLljpw4IAGDhwo6Y++IzQ01IzJysoq1tdcyMvLS15eXhWZKgAAAIArUCFv3yvLN9qXipk2bZpycnLM6ejRo+WWKwCgcrHZbNq3b59CQ0PVuHFjWa1WJScnm+vz8/OVkpKijh07ujBLAAAAAFeqXItSVqtVkord8fTnb7StVqvy8/OVnZ190ZgLeXl5KSAgwG4CAFQPkydPVkpKitLT0/Xll1/qrrvu0qlTpzRy5EhZLBbFxMQoLi5Oa9as0Z49ezRq1Cj5+vpq2LBhrk4dAAAAwBUo16JUab7Rbtu2rTw8POxiMjIytGfPHr71BoAa6NixYxo6dKiaNWumQYMGydPTU9u2bVPDhg0lSVOmTFFMTIzGjh2rdu3a6eeff9b69evNN7oCAAAAqJocHlPq9OnTOnjwoDmfnp6uXbt2KSgoSNdcc435jXZERIQiIiIUFxdn9412YGCgRo8erUmTJik4OFhBQUGaPHmyWrZsab6NDwBQcyQlJV1yvcViUWxsrGJjY52TEAAAAACncLgotWPHDnXt2tWcL3qz0ciRI5WQkKApU6YoLy9PY8eOVXZ2ttq3b1/sG+158+bJ3d1dgwcPVl5enrp3766EhAS5ubmVQ5MAAAAAAABQ2TlclIqKipJhGBddX5pvtL29vbVgwQItWLDA0d0DAAAAAACgGqiQt+8BAAAAAAAAl0JRCgAAAAAAAE7n8ON7AAAAAAAArtBo6lqX7PfQrNtdst/qjjulAAAAAAAA4HQUpQAAAAAAAOB0FKUAAAAAAADgdBSlAAAAAAAA4HQUpQAAAAAAAOB0FKUAAAAAAADgdBSlAAAAAAAA4HQUpQAAAAAAAOB0FKUAAAAAAADgdBSlAAAAAAAA4HTurk4AAAAAAACgMms0da3L9n1o1u0u23dFoygFAAAAACgXrrxwB1D18PgeAAAAAAAAnI6iFAAAAAAAAJyOohQAAAAAAACcjjGlAAAAACdy1Zg71XmgXNhjXCcAVQV3SgEAAAAAAMDpuFMKFY5XZwIAAAAAgAtxpxQAAAAAAACczqV3Si1atEjPP/+8MjIy1KJFC82fP19//etfXZkSAKASc2W/wfgcAFA1cc0BAJWXy4pSq1atUkxMjBYtWqROnTrp3//+t3r37q3vvvtO11xzjavSAgBUUvQbAABH0XcAqA6q85A4Lnt8b+7cuRo9erQeeOABNW/eXPPnz1d4eLgWL17sqpQAAJUY/QYAwFH0HQBQubnkTqn8/HylpaVp6tSpdsujo6OVmprqipRQTVXnijJQk9BvAAAcRd8BAJWfS4pSv/76qwoLCxUSEmK3PCQkRJmZmcXibTabbDabOZ+TkyNJOnXqVJlzOG87W+bPAqVxzSNvu2S/e2b2csl+a6rIJz912b7L+v+66NxpGEZ5plOhHO03pPLvO+g3AFR1V/K3c03oO7jmAIDiynoOLG2/4dKBzi0Wi928YRjFlklSfHy8Zs6cWWx5eHh4heUGVFWB812dAZzlSv9f5+bmKjAwsFxycZbS9hsSfQcAXKg8/kaozn0H/QYAFFfR1xwuKUrVrVtXbm5uxb6hyMrKKvZNhiRNmzZNEydONOfPnz+v//73vwoODr7oxUhJTp06pfDwcB09elQBAQFlb0AlQpuqBtpUNdSUNhmGodzcXIWFhbk4u9JztN+Qyq/vuFLV8eeqonHMHMPxcgzHy3FFx+y7776r1n1Hefcb/Kz9gePwPxyLP3Ac/qc6H4vSXnO4pCjl6emptm3bKjk5WXfccYe5PDk5WQMGDCgW7+XlJS8vL7tlV111VZn3HxAQUO3+h9OmqoE2VQ01oU1V7VtuR/sNqfz7jitVHX+uKhrHzDEcL8dwvBxXv3591arlsvckOczV1xxF+Fn7A8fhfzgWf+A4/E91PRalueZw2eN7EydO1L333qt27dqpQ4cOeuWVV3TkyBE9+OCDrkoJAFCJ0W8AABxF3wEAlZvLilJDhgzRyZMn9dRTTykjI0ORkZH6+OOP1bBhQ1elBACoxOg3AACOou8AgMrNpQOdjx07VmPHjnXa/ry8vPTkk08Wuy23KqNNVQNtqhpoU+Xn7H6jPFS3/wfOwDFzDMfLMRwvx1X1Y+aqvqOqH7fywnH4H47FHzgO/8OxkCxGVXqvKwAAAAAAAKqFqjNSIQAAAAAAAKoNilIAAAAAAABwOopSAAAAAAAAcDqKUgAAAAAAAHC6alOUWrx4sVq1aqWAgAAFBASoQ4cO+uSTTy75mZUrV6p169by9fVVaGio7rvvPp08edJJGV9eWdr08ssvq3nz5vLx8VGzZs30xhtvOCnbsomPj5fFYlFMTMwl41JSUtS2bVt5e3vr2muv1b/+9S/nJFgGpWlTRkaGhg0bpmbNmqlWrVqXbb+rlaZNq1evVs+ePXX11VebP6+ffvqp85J0UGnatGXLFnXq1EnBwcHy8fHR9ddfr3nz5jkvSQeV9vepyH/+8x+5u7vrxhtvrNC8aoJFixapcePG8vb2Vtu2bfXFF19cNLaq/f5XFEeOWVU7v1QER45XVTt3VQRHjtef1eTzoiPHbPPmzbJYLMWm77//3okZV25l/Rmsyj7//HP169dPYWFhslgseu+99+zWG4ah2NhYhYWFycfHR1FRUdq7d69rkq1A8fHxuvnmm+Xv76969epp4MCB2r9/v11MTTgWl7uWrQnH4GJK+pu9Jh+PalOUatCggWbNmqUdO3Zox44d6tatmwYMGHDR/5FbtmzRiBEjNHr0aO3du1dvv/22tm/frgceeMDJmV+co21avHixpk2bptjYWO3du1czZ87UuHHj9OGHHzo589LZvn27XnnlFbVq1eqScenp6erTp4/++te/aufOnXrsscf08MMP691333VSpqVX2jbZbDZdffXVmj59ulq3bu2k7MqmtG36/PPP1bNnT3388cdKS0tT165d1a9fP+3cudNJmZZeadvk5+en8ePH6/PPP9e+ffv0+OOP6/HHH9crr7zipExLr7RtKpKTk6MRI0aoe/fuFZxZ9bdq1SrFxMRo+vTp2rlzp/7617+qd+/eOnLkSInxVen3v6I4esyq0vmlIjh6vKrSuasiOHq8itTk82JZj9n+/fuVkZFhThEREU7KuHIr6/Gs6s6cOaPWrVtr4cKFJa6fPXu25s6dq4ULF2r79u2yWq3q2bOncnNznZxpxUpJSdG4ceO0bds2JScn69y5c4qOjtaZM2fMmJpwLC53LVsTjkFJLvY3e009HpIkoxqrU6eO8dprr5W47vnnnzeuvfZau2UvvfSS0aBBA2ekVmaXalOHDh2MyZMn2y37xz/+YXTq1MkZqTkkNzfXiIiIMJKTk40uXboY//jHPy4aO2XKFOP666+3WzZmzBjjlltuqeAsHeNIm/7MkVhnK2ubitxwww3GzJkzKya5MrrSNt1xxx3G8OHDKya5MipLm4YMGWI8/vjjxpNPPmm0bt26wnOszv7yl78YDz74oN2y66+/3pg6deplP1uZf/8r0pUcsyKV8fxSUcrjeFXGc1dFKevxqsnnRUeP2aZNmwxJRnZ2thOyq3rK43e2qpNkrFmzxpw/f/68YbVajVmzZpnLfv/9dyMwMND417/+5YIMnScrK8uQZKSkpBiGUbOPRdG1bE09Bhf7m72mHo8i1eZOqT8rLCxUUlKSzpw5ow4dOpQY07FjRx07dkwff/yxDMPQiRMn9M477+j22293cralU5o22Ww2eXt72y3z8fHRV199pYKCAmekWWrjxo3T7bffrh49elw2duvWrYqOjrZb1qtXL+3YsaNStcuRNlUVV9Km8+fPKzc3V0FBQRWQWdldSZt27typ1NRUdenSpQIyKztH27R06VL9+OOPevLJJys4s+ovPz9faWlpxc5R0dHRSk1NdVFWlVt5HLPKen6pCOVxvCrruasilPV41eTz4pX8jLVp00ahoaHq3r27Nm3aVJFpVhn0CyVLT09XZmam3XHx8vJSly5dqv1xycnJkSSzz6qJx+LCa9maeAyki//NXlOPRxF3VydQnnbv3q0OHTro999/V+3atbVmzRrdcMMNJcZ27NhRK1eu1JAhQ/T777/r3Llz6t+/vxYsWODkrC/NkTb16tVLr732mgYOHKibbrpJaWlpev3111VQUKBff/1VoaGhTs6+ZElJSfr666+1ffv2UsVnZmYqJCTEbllISIjOnTtXadrlaJuqgitt05w5c3TmzBkNHjy4nDMru7K2qUGDBvrll1907tw5xcbGVqrHfB1t04EDBzR16lR98cUXcnevVl2AS/z6668qLCws8RyVmZnpoqwqt/I4ZpXx/FJRruR4VeZzV0Upy/Gq6efFshyz0NBQvfLKK2rbtq1sNpuWL1+u7t27a/Pmzbr11ludkXalRb9QsqK2l3RcDh8+7IqUnMIwDE2cOFGdO3dWZGSkpJp1LC52LVtUaKkJx6DIpf5mr0k/EyWpVj1vs2bNtGvXLv3222969913NXLkSKWkpJRYxPnuu+/08MMP64knnlCvXr2UkZGhf/7zn3rwwQe1ZMkSF2RfMkfaNGPGDGVmZuqWW26RYRgKCQnRqFGjNHv2bLm5ubkg++KOHj2qf/zjH1q/fn2xu7ouxWKx2M0bhlHiclcoa5sqsytt05tvvqnY2Fi9//77qlevXgVk6LgradMXX3yh06dPa9u2bZo6daqaNGmioUOHVlCmpedomwoLCzVs2DDNnDlTTZs2dUKGNUdJ56jKcH6qzMp6zCrj+cUZynK8Kuu5yxlKe7w4L/6PIz9jzZo1U7Nmzcz5Dh066OjRo3rhhRdqfFGqCP1CyWracRk/fry+/fZbbdmypdi6mnAsLnYtW6QmHAOp9H+z15TjUYwrnhl0lu7duxt///vfS1w3fPhw46677rJb9sUXXxiSjOPHjzsjvTK5VJuK5OfnG0ePHjXOnTtnLFq0yPD39zcKCwudlOGlrVmzxpBkuLm5mZMkw2KxGG5ubsa5c+eKfeavf/2r8fDDD9stW716teHu7m7k5+c7K/WLKkub/qwyjilzJW1KSkoyfHx8jI8++siJGV/elf5/KvL0008bTZs2reBsS8fRNmVnZxeLt1gs5rLPPvvMRS2pumw2m+Hm5masXr3abvnDDz9s3HrrrZf9fGX8/a9oV3LMKuv5pSJd6c9Ykcp07qpIjh4vzovl9zP2zDPPFBsDtCYqr+NZ1emCMaV+/PFHQ5Lx9ddf28X179/fGDFihJOzc47x48cbDRo0MH766Se75TXxWBQpupatacfgcn+zHzx4sEYdjwtVyzGlihiGIZvNVuK6s2fPqlYt++YX3U1k/P+7cCqjS7WpiIeHhxo0aCA3NzclJSWpb9++xdrqKt27d9fu3bu1a9cuc2rXrp3uuece7dq1q8Q7ujp06KDk5GS7ZevXr1e7du3k4eHhrNQvqixtquzK2qY333xTo0aNUmJiYqUbn628/j+V5nfQWRxtU0BAQLH4Bx980PwWq3379i5qSdXl6emptm3bFjtHJScnq2PHji7KqnIr6zGrzOeXilReP2OV6dxVkRw9XpwXy+9nbOfOnZViSAVXo18oWePGjWW1Wu2OS35+vlJSUqrdcTEMQ+PHj9fq1au1ceNGNW7c2G59TToWFyrqi2raMbjc3+zXXnttjToexbiwIFaupk2bZnz++edGenq68e233xqPPfaYUatWLWP9+vWGYRjG1KlTjXvvvdeMX7p0qeHu7m4sWrTI+PHHH40tW7YY7dq1M/7yl7+4qgnFONqm/fv3G8uXLzd++OEH48svvzSGDBliBAUFGenp6S5qQelceKfAhe366aefDF9fX+ORRx4xvvvuO2PJkiWGh4eH8c4777gg29K5XJsMwzB27txp7Ny502jbtq0xbNgwY+fOncbevXudnGnpXa5NiYmJhru7u/Hyyy8bGRkZ5vTbb7+5INvSuVybFi5caHzwwQfGDz/8YPzwww/G66+/bgQEBBjTp093QbalU5qfvT+riW+ZKm9JSUmGh4eHsWTJEuO7774zYmJiDD8/P+PQoUOGYVSP3//y5ugxq4rnl/Lk6PGqiueu8lSW38k/q4nnRUeP2bx584w1a9YYP/zwg7Fnzx5j6tSphiTj3XffdVUTKpXLHc/qKjc31+zfJBlz5841du7caRw+fNgwDMOYNWuWERgYaKxevdrYvXu3MXToUCM0NNQ4deqUizMvXw899JARGBhobN682a7POnv2rBlTE47F5a5la8IxuJQL/2avycej2owpdeLECd17773KyMhQYGCgWrVqpXXr1qlnz56SpIyMDB05csSMHzVqlHJzc7Vw4UJNmjRJV111lbp166bnnnvOVU0oxtE2FRYWas6cOdq/f788PDzUtWtXpaamqlGjRi5qQdlc2K7GjRvr448/1iOPPKKXX35ZYWFheumll3TnnXe6MEvHXNgm6Y831hRJS0tTYmKiGjZsqEOHDjk5u7K5sE3//ve/de7cOY0bN07jxo0zl48cOVIJCQkuyNBxF7bp/PnzmjZtmtLT0+Xu7q7rrrtOs2bN0pgxY1yYpWNK+tlD+RoyZIhOnjypp556ShkZGYqMjNTHH3+shg0bSqqev/9XytFjVh3OL1fC0eNVHc5dV6Isv5M1naPHLD8/X5MnT9bPP/8sHx8ftWjRQmvXrlWfPn1c1YRK5XLHs7rasWOHunbtas5PnDhR0v/O1VOmTFFeXp7Gjh2r7OxstW/fXuvXr5e/v7+rUq4QixcvliRFRUXZLV+6dKlGjRolSTXiWFzuWrYmHANH1OTjYTGMSvysGgAAAAAAAKqlyjHQEAAAAAAAAGoUilIAAAAAAABwOopSAAAAAAAAcDqKUgAAAAAAAHA6ilIAAAAAAABwOopSAAAAAAAAcDqKUgAAAAAAAHA6ilKo1g4dOiSLxaJdu3Zd0XaioqIUExPj9P2W1ahRozRw4ECX7BsAqjpHz/kVLTMzUz179pSfn5+uuuoqV6cDANVWZTv/V4TyuE5ISEiw649iY2N14403XtE2UXO5uzoBoCpYvXq1PDw8XJ0GAKAGmjdvnjIyMrRr1y4FBga6Oh0AQCVx6NAhNW7cWDt37nRpUWjy5MmaMGGCy/aPqo2iFFAKQUFBFb4PwzBUWFgod3d+LQEA//Pjjz+qbdu2ioiIuGhMQUEBX54AAFyidu3aql27tqvTQBXF43uoFs6fP6/nnntOTZo0kZeXl6655ho9++yz5vqffvpJXbt2la+vr1q3bq2tW7ea606ePKmhQ4eqQYMG8vX1VcuWLfXmm2/abf/CW3kbNWqkuLg43X///fL399c111yjV155pVhe33//vTp27Chvb2+1aNFCmzdvNtdt3rxZFotFn376qdq1aycvLy998cUXMgxDs2fP1rXXXisfHx+1bt1a77zzjvm5wsJCjR49Wo0bN5aPj4+aNWumF1988ZLHJy0tTfXq1bM7JgCAy8vOztaIESNUp04d+fr6qnfv3jpw4IBdzLvvvqsWLVrIy8tLjRo10pw5c+zWN2rUSE8//bSGDRum2rVrKywsTAsWLCjV/hs1aqR3331Xb7zxhiwWi0aNGiVJslgs+te//qUBAwbIz89PzzzzTKn7h9dff93MNzQ0VOPHjy/bwQGAauxy5/+iR9g+/fRTNW/eXLVr19Ztt92mjIwMM+bcuXN6+OGHddVVVyk4OFiPPvqoRo4caff43OX+9s/OztY999yjq6++Wj4+PoqIiNDSpUslSY0bN5YktWnTRhaLRVFRUaVu3wsvvKDQ0FAFBwdr3LhxKigoMNfl5+drypQpql+/vvz8/NS+fXu765gL/fnxvd9//10tWrTQ3//+d3N9enq6AgMD9eqrr5Y6P9QgBlANTJkyxahTp46RkJBgHDx40Pjiiy+MV1991UhPTzckGddff73x0UcfGfv37zfuuusuo2HDhkZBQYFhGIZx7Ngx4/nnnzd27txp/Pjjj8ZLL71kuLm5Gdu2bTO336VLF+Mf//iHOd+wYUMjKCjIePnll40DBw4Y8fHxRq1atYx9+/YZhmGY+23QoIHxzjvvGN99953xwAMPGP7+/savv/5qGIZhbNq0yZBktGrVyli/fr1x8OBB49dffzUee+wx4/rrrzfWrVtn/Pjjj8bSpUsNLy8vY/PmzYZhGEZ+fr7xxBNPGF999ZXx008/GStWrDB8fX2NVatWmfmNHDnSGDBggLmfwMBAY9GiRRX5vwAAqo0/n/P79+9vNG/e3Pj888+NXbt2Gb169TKaNGli5OfnG4ZhGDt27DBq1aplPPXUU8b+/fuNpUuXGj4+PsbSpUvN7TVs2NDw9/c34uPjjf3795v9zPr16y+bS1ZWlnHbbbcZgwcPNjIyMozffvvNMAzDkGTUq1fPWLJkifHjjz8ahw4dKlX/sGjRIsPb29uYP3++sX//fuOrr74y5s2bV27HDgCqMkfO/0uXLjU8PDyMHj16GNu3bzfS0tKM5s2bG8OGDTO398wzzxhBQUHG6tWrjX379hkPPvigERAQYP6dbhjGZf/2HzdunHHjjTca27dvN9LT043k5GTjgw8+MAzDML766itDkrFhwwYjIyPDOHny5GXbOHLkSCMgIMB48MEHjX379hkffvih4evra7zyyitmzLBhw4yOHTsan3/+uXHw4EHj+eefN7y8vIwffvjBbHtgYKAZ/+STTxqtW7c253fu3Gl4enoaa9asMc6dO2d06tTJrs3An1GUQpV36tQpw8vLy3j11VeLrSsqDr322mvmsr179xqSzAJSSfr06WNMmjTJnC+pKDV8+HBz/vz580a9evWMxYsX2+131qxZZkxBQYHRoEED47nnnjMM439Fqffee8+MOX36tOHt7W2kpqba5TN69Ghj6NChF8137Nixxp133mnOFxWl3nvvPcPf399ITEy86GcBAPaKzvk//PCDIcn4z3/+Y6779ddfDR8fH+Ott94yDOOPP9x79uxp9/l//vOfxg033GDON2zY0LjtttvsYoYMGWL07t27VPkMGDDAGDlypN0ySUZMTMxlP3th/xAWFmZMnz69VPsFgJrGkfP/0qVLDUnGwYMHzZiXX37ZCAkJMedDQkKM559/3pw/d+6ccc0115gFmtL87d+vXz/jvvvuKzHfomuOnTt3lrqNI0eONBo2bGicO3fOXHb33XcbQ4YMMQzDMA4ePGhYLBbj559/tvtc9+7djWnTppltv1RRyjAMY/bs2UbdunWNCRMmGFar1fjll19KnSNqFgavQZW3b98+2Ww2de/e/aIxrVq1Mv8dGhoqScrKytL111+vwsJCzZo1S6tWrdLPP/8sm80mm80mPz+/S+73z9u0WCyyWq3Kysqyi+nQoYP5b3d3d7Vr10779u2zi2nXrp357++++06///67evbsaReTn5+vNm3amPP/+te/9Nprr+nw4cPKy8tTfn5+scENv/zyS3300Ud6++23dccdd1yyLQCA4vbt2yd3d3e1b9/eXBYcHKxmzZqZ5/J9+/ZpwIABdp/r1KmT5s+fr8LCQrm5uUmy7w+K5ufPn39F+f25/yhyqf4hKytLx48fv2R/CQAo3flfknx9fXXdddeZ86Ghoeb1QE5Ojk6cOKG//OUv5no3Nze1bdtW58+fl1S6v/0feugh3Xnnnfr6668VHR2tgQMHqmPHjlfUvhYtWpj9U1Heu3fvliR9/fXXMgxDTZs2tfuMzWZTcHBwqfcxadIkvf/++1qwYIE++eQT1a1b94pyRvVFUQpVno+Pz2Vj/jz4q8VikSSzM5gzZ47mzZun+fPnq2XLlvLz81NMTIzy8/NLvc2i7RZt81KK9l/kz8Wvos+vXbtW9evXt4vz8vKSJL311lt65JFHNGfOHHXo0EH+/v56/vnn9eWXX9rFX3fddQoODtbrr7+u22+/XZ6enpfNDQDwP4ZhXHR50bn8z/++3OcudOHnHHXhlyeX6x9K018CAEp3/pdKvh648LOX6iNK87d/7969dfjwYa1du1YbNmxQ9+7dNW7cOL3wwgsOtup/LnUdc/78ebm5uSktLc2ucCXJocHMs7KytH//frm5uenAgQO67bbbypwvqjcGOkeVFxERIR8fH3322Wdl+vwXX3yhAQMGaPjw4WrdurWuvfbaYoPYltW2bdvMf587d05paWm6/vrrLxp/ww03yMvLS0eOHFGTJk3spvDwcDPfjh07auzYsWrTpo2aNGmiH3/8sdi26tatq40bN+rHH3/UkCFD7AYvBABc3g033KBz587ZFf1PnjypH374Qc2bNzdjtmzZYve51NRUNW3a1O6P+T/3B0Xzl+oPyuJy/YO/v78aNWpU5v4SAGqK0pz/LycwMFAhISH66quvzGWFhYXauXOn3X4u97e/JF199dUaNWqUVqxYofnz55svWCr60rmwsPCK2vtnbdq0UWFhobKysorlZLVaS72d+++/X5GRkXrjjTc0ZcoUfffdd+WWI6oX7pRCleft7a1HH31UU6ZMkaenpzp16qRffvlFe/fuLdUjCk2aNNG7776r1NRU1alTR3PnzlVmZmapO5xLefnllxUREaHmzZtr3rx5ys7O1v3333/ReH9/f02ePFmPPPKIzp8/r86dO+vUqVNKTU1V7dq1NXLkSDVp0kRvvPGGPv30UzVu3FjLly/X9u3bzbdv/Fm9evW0ceNGde3aVUOHDlVSUpLc3fm1B4DSiIiI0IABA/S3v/1N//73v+Xv76+pU6eqfv365iN7kyZN0s0336ynn35aQ4YM0datW7Vw4UItWrTIblv/+c9/NHv2bA0cOFDJycl6++23tXbt2nLNtzT9Q2xsrB588EHVq1dPvXv3Vm5urv7zn/9owoQJ5ZoLAFRlpTn/l8aECRMUHx+vJk2a6Prrr9eCBQuUnZ1t3j1Vmr/9n3jiCbVt21YtWrSQzWbTRx99ZF6n1KtXTz4+Plq3bp0aNGggb29vBQYGXlHbmzZtqnvuuUcjRozQnDlz1KZNG/3666/auHGjWrZsqT59+lx2Gy+//LK2bt2qb7/9VuHh4frkk090zz336Msvv+TpDRTDnVKoFmbMmKFJkybpiSeeUPPmzTVkyJBi4ztd6rM33XSTevXqpaioKFmtVrvXtF6JWbNm6bnnnlPr1q31xRdf6P3337/s89RPP/20nnjiCcXHx6t58+bq1auXPvzwQ/Oi4sEHH9SgQYM0ZMgQtW/fXidPntTYsWMvuj2r1aqNGzdq9+7duueee8r1mxQAqO6WLl2qtm3bqm/fvurQoYMMw9DHH39sPvpw00036a233lJSUpIiIyP1xBNP6KmnntKoUaPstjNp0iSlpaWpTZs2evrppzVnzhz16tWrXHMtTf8wcuRIzZ8/X4sWLVKLFi3Ut2/fcrs7GACqk8ud/0vj0Ucf1dChQzVixAh16NBBtWvXVq9eveTt7W3GXO5vf09PT02bNk2tWrXSrbfeKjc3NyUlJUn6Y8zal156Sf/+978VFhbmUMHscm0fMWKEJk2apGbNmql///768ssv7e7eupjvv/9e//znP7Vo0SIz/uWXX9Zvv/2mGTNmlEt+qF4sRmkHPgAAAIDDGjVqpJiYGMXExLg6FQCAC50/f17NmzfX4MGD9fTTT7s6HaBS4DkeAAAAAADK2eHDh7V+/Xp16dJFNptNCxcuVHp6uoYNG+bq1IBKg8f3AAAAXGTlypWqXbt2iVOLFi1cnR4A4ArUqlVLCQkJuvnmm9WpUyft3r1bGzZsKJexay/mYn1K7dq19cUXX1TYfoGy4vE9AAAAF8nNzdWJEydKXOfh4aGGDRs6OSMAQFV28ODBi66rX7++fHx8nJgNcHkUpQAAAAAAAOB0PL4HAAAAAAAAp6MoBQAAAAAAAKejKAUAAAAAAACnoygFAAAAAAAAp6MoBQAAAAAAAKejKAUAAAAAAACnoygFAAAAAAAAp6MoBQAAAAAAAKejKAUAAAAAAACnoygFAAAAAAAAp6MoBQAAAAAAAKejKAUAAAAAAACnoygFAAAAAAAAp6MoBZdKSEiQxWLRoUOHXJ1KuTl+/LhiY2O1a9cuV6dySYsWLVJCQkKx5YcOHZLFYim2btWqVWrRooV8fHxksVgqffsAAACA8vLFF19o8ODBql+/vjw9PRUYGKiOHTtq8eLFOnPmTLH4goICWa1WWSwWvfPOOy7I2PUaNWqkUaNGmfObN2+WxWLR5s2bXZYTKh+KUkA5O378uGbOnFnpizYXK0qFhoZq69atuv32281lv/zyi+69915dd911WrdunbZu3aqmTZs6MVsAAADANZ588kndeuut+vnnn/X0008rOTlZSUlJ6t69u2JjY/X4448X+8xHH32kEydOSJKWLFni7JQrpZtuuklbt27VTTfd5OpUUIm4uzoBAJWLl5eXbrnlFrtlP/zwgwoKCjR8+HB16dLFRZkBQNV39uxZ+fr6ujoNAEApnD17VmvXrtVTTz2l0aNH69VXX5XFYjHX9+7dW1OmTNHWrVuLfXbJkiXy9PRUly5dtH79eh07dkwNGjRwZvqVTkBAQLHrDIA7pVDpvP7662rdurW8vb0VFBSkO+64Q/v27SsW98EHH6hDhw7y9fWVv7+/evbsWaxDiI2NlcVi0c6dOzVo0CAFBAQoMDBQw4cP1y+//OJQXgcPHtR9992niIgI+fr6qn79+urXr592795txmzevFk333yzJOm+++6TxWKRxWJRbGxsqfZRdEtrYmKiHn30UYWGhqp27drq16+fTpw4odzcXP39739X3bp1VbduXd133306ffq03TZ+//13TZs2TY0bN5anp6fq16+vcePG6bfffjNjGjVqpL179yolJcXMsVGjRpKKP743atQode7cWZI0ZMgQWSwWRUVFOXTsAKAmKuqDvv76a911112qU6eOrrvuulKdpyXJZrNp0qRJslqt8vX11a233qq0tLRij0MAAK7cxc7ZTz31lOrUqaOXXnrJriBVxN/fX9HR0XbLjh8/rnXr1qlfv3765z//qfPnz5f4hMLlFF0bvPnmm5o+fbrCwsIUEBCgHj16aP/+/cXiS3MdNWrUKNWuXVsHDx5Unz59VLt2bYWHh2vSpEmy2Wx2sTNnzlT79u0VFBSkgIAA3XTTTVqyZIkMw7CLKygo0JQpU8z+qnPnzvrqq68u2h4e38OfUZRCpRIfH6/Ro0erRYsWWr16tV588UV9++236tChgw4cOGDGJSYmasCAAQoICNCbb76pJUuWKDs7W1FRUdqyZUux7d5xxx1q0qSJ3nnnHcXGxuq9995Tr169VFBQUOrcjh8/ruDgYM2aNUvr1q3Tyy+/LHd3d7Vv397sFG666SYtXbpUkvT4449r69at2rp1qx544AGHjsNjjz2mrKwsJSQkaM6cOdq8ebOGDh2qO++8U4GBgXrzzTc1ZcoULV++XI899pj5OcMwNHDgQL3wwgu69957tXbtWk2cOFHLli1Tt27dzI5mzZo1uvbaa9WmTRszxzVr1pSYy4wZM/Tyyy9LkuLi4rR161YtWrTIofYAQE02aNAgNWnSRG+//bb+9a9/leo8Lf3x5cb8+fN133336f3339edd96pO+64o1jxCgBQfv58zp41a5b27Nmj6Ohoh+5yTUhIUGFhoe6//3716NFDDRs21Ouvv16smFNajz32mA4fPqzXXntNr7zyig4cOKB+/fqpsLDQjCntdZT0RxGpf//+6t69u95//33df//9mjdvnp577jm7uEOHDmnMmDF66623tHr1ag0aNEgTJkzQ008/bRf3t7/9TS+88IJGjBhh9leDBg1SdnZ2mdqLGsYAXGjp0qWGJCM9Pd3Izs42fHx8jD59+tjFHDlyxPDy8jKGDRtmGIZhFBYWGmFhYUbLli2NwsJCMy43N9eoV6+e0bFjR3PZk08+aUgyHnnkEbttrly50pBkrFixosy5nzt3zsjPzzciIiLstr99+3ZDkrF06VKHt7lp0yZDktGvXz+75TExMYYk4+GHH7ZbPnDgQCMoKMicX7dunSHJmD17tl3cqlWrDEnGK6+8Yi5r0aKF0aVLl2I5pKenF8u/KK+3337b4TYBQE1V1Ac98cQT5rLSnqf37t1rSDIeffRRu7g333zTkGSMHDmywvMHgJqkpHP2tm3bDEnG1KlTS72d8+fPG02aNDHq169vnDt3zm7bn332mUM5Ff0NfuH10VtvvWVIMrZu3WoYhlHq6yjDMIyRI0cakoy33nrLLrZPnz5Gs2bNLppLYWGhUVBQYDz11FNGcHCwcf78ecMwDGPfvn2XvN76c39V1J5NmzaV+hig+uNOKVQaW7duVV5eXrFHEsLDw9WtWzd99tlnkqT9+/fr+PHjuvfee1Wr1v9+hGvXrq0777xT27Zt09mzZ+22cc8999jNDx48WO7u7tq0aVOp8zt37pzi4uJ0ww03yNPTU+7u7vL09NSBAwdKfLzwSvTt29duvnnz5pJkN/h40fL//ve/5iN8GzdulKRix/Duu++Wn5+feQwBAM5z5513mv8u7Xk6JSVF0h/91Z/dddddcndnSFAAqCh/PmeXRUpKig4ePKiRI0fKzc1N0v+G9Xj99dfLtM3+/fvbzbdq1UqSdPjwYUmlv44qYrFY1K9fv2LbLNpekY0bN6pHjx4KDAyUm5ubPDw89MQTT+jkyZPKysqSJPN66mLXW8DlUJRCpXHy5ElJf7z97UJhYWHm+svFnT9/vtitolar1W7e3d1dwcHB5rZKY+LEiZoxY4YGDhyoDz/8UF9++aW2b9+u1q1bKy8vr9TbKY2goCC7eU9Pz0su//333yX9cWzc3d119dVX28VZLBZZrVaH2gsAKB9/7q9Ke54u+m9ISIhdXFH/BQCoGH8+Z19zzTWSpPT09FJ/vuhNe0WPW//2228KDAxU586d9e6775bpEewLz/teXl6SZF6DlPY6qoivr6+8vb2LbbPomkKSvvrqK3OsrFdffVX/+c9/tH37dk2fPr3EfV/segu4HEqXqDSKTloZGRnF1h0/flx169YtVVytWrVUp04du+WZmZmqX7++OX/u3DmdPHnSoRPlihUrNGLECMXFxdkt//XXX3XVVVeVejsVKTg4WOfOndMvv/xid8FjGIYyMzPNQdgBAM7z54FxS3ueLuqfTpw4UWL/BQCoGH8+Z4eGhqply5Zav359qd6empOTo3fffVeSLvp3d2JiosaOHVt+Cav011GOSEpKkoeHhz766CO7AtZ7771X4r4vdr0FXA53SqHS6NChg3x8fLRixQq75ceOHdPGjRvVvXt3SVKzZs1Uv359JSYm2g0WeObMGb377rvmG/n+bOXKlXbzb731ls6dO+fQW+QsFov5rUSRtWvX6ueff7ZbduE3F85UdIwuPIbvvvuuzpw5Y66X/sjTFTkCQE1W2vP0rbfeKklatWqVXdw777yjc+fOOSFTAID0x0t/srOz9fDDD5c4UPnp06e1fv16SX8UnPLy8vT0009r06ZNxaa6deuW+RG+SyntdZQjLBaL3N3dzUcQpT+ub5YvX24XV3Q9dbHrLeByuFMKlcZVV12lGTNm6LHHHtOIESM0dOhQnTx5UjNnzpS3t7eefPJJSVKtWrU0e/Zs3XPPPerbt6/GjBkjm82m559/Xr/99ptmzZpVbNurV6+Wu7u7evbsqb1792rGjBlq3bp1sbE6LqVv375KSEjQ9ddfr1atWiktLU3PP/+8GjRoYBd33XXXycfHRytXrlTz5s1Vu3ZthYWFKSws7MoOUCn07NlTvXr10qOPPqpTp06pU6dO+vbbb/Xkk0+qTZs2uvfee83Yli1bKikpSatWrdK1114rb29vtWzZssJzBICarLTn6RYtWmjo0KGaM2eO3Nzc1K1bN+3du1dz5sxRYGCg3ZiKAICKc/fdd2vGjBl6+umn9f3332v06NG67rrrdPbsWX355Zf697//rSFDhig6OlpLlixRnTp1NHny5GKPx0nSiBEjNHfuXH3zzTdq3bp1ueVY2usoR9x+++2aO3euhg0bpr///e86efKkXnjhhWJf0jdv3lzDhw/X/Pnz5eHhoR49emjPnj164YUXFBAQUF5NRDXGXzSoVKZNm6bXXntN33zzjQYOHKjx48erRYsWSk1NVUREhBk3bNgwvffeezp58qSGDBmi++67TwEBAdq0aZM6d+5cbLurV6/W999/r0GDBumJJ55Qv379tH79enNMptJ48cUXNXz4cMXHx6tfv3764IMPtHr1al133XV2cb6+vnr99dd18uRJRUdH6+abb9Yrr7xS9oPiAIvFovfee08TJ07U0qVL1adPH/O14xs3brTrRGbOnKkuXbrob3/7m/7yl78UG+wQAFD+HDlPL126VP/4xz+0ZMkS9evXT0lJSXrrrbckqdI8Ng4ANcFTTz2llJQUhYaGavr06erRo4eGDBmiTz/9VBMnTtRTTz2lb7/9VmlpaRo5cmSJBSlJ+vvf/y7pf+NOlafSXkeVVrdu3fT6669r9+7d6tevn6ZPn6677rpLU6dOLRa7ZMkSTZw4UQkJCerfv7/eeustvfvuu8WGVCny50ckAYtR0j2IQDURGxurmTNn6pdffinTs9QAAFQmqamp6tSpk1auXKlhw4a5Oh0AAErt/fff18CBA7V7925FRka6Oh1UEjy+BwAAUAklJydr69atatu2rXx8fPTNN99o1qxZioiI0KBBg1ydHgAApWKz2fTFF19o4cKFuvrqq9WkSRNXp4RKhKIUarzz58/r/Pnzl4xxd7+yXxXDMFRYWHjJGDc3N25lBQCYAgICtH79es2fP1+5ubmqW7euevfurfj4+Is+GgIAqPxq2rVBRkaG+vTpo2bNmmnlypX0YbDD43uo8Yoe8buU9PR0NWrUqMz72Lx5s7p27XrJmKVLl2rUqFFl3gcAAACAyo9rA+B/KEqhxjt+/LiOHz9+yZhWrVo5NCj6hXJzc7V///5LxjRu3FjBwcFl3gcAAACAyo9rA+B/KEoBAAAAAADA6Wq5OgEAAAAAAADUPBSlAAAAAAAA4HRV8u1758+f1/Hjx+Xv719t3kgAAM5iGIZyc3MVFhamWrVqzncT9B0AUHY1se+g3wCAsittv1Eli1LHjx9XeHi4q9MAgCrt6NGjatCggavTcBr6DgC4cjWp76DfAIArd7l+o0oWpfz9/SX90biAgAAXZwMAVcupU6cUHh5unktrCvoOACi7mth30G8AQNmVtt+okkWpottnAwIC6CAAoIxq2qMI9B0AcOVqUt9BvwEAV+5y/UbNeCAcAAAAAAAAlQpFKQAAAAAAADgdRSkAAAAAAAA4HUUpAAAAAAAAOB1FKQAAAAAAADgdRSkAAAAAAAA4HUUpAAAAAAAAOJ27qxNwlUZT17pkv4dm3e6S/QKAqyxevFiLFy/WoUOHJEktWrTQE088od69e0uSDMPQzJkz9corryg7O1vt27fXyy+/rBYtWpjbsNlsmjx5st58803l5eWpe/fuWrRokRo0aOCKJjkdfRYAAJWXq/ppib4aVR93SgEAKlSDBg00a9Ys7dixQzt27FC3bt00YMAA7d27V5I0e/ZszZ07VwsXLtT27dtltVrVs2dP5ebmmtuIiYnRmjVrlJSUpC1btuj06dPq27evCgsLXdUsAAAAAFeIohQAoEL169dPffr0UdOmTdW0aVM9++yzql27trZt2ybDMDR//nxNnz5dgwYNUmRkpJYtW6azZ88qMTFRkpSTk6MlS5Zozpw56tGjh9q0aaMVK1Zo9+7d2rBhg4tbBwAAAKCsKEoBAJymsLBQSUlJOnPmjDp06KD09HRlZmYqOjrajPHy8lKXLl2UmpoqSUpLS1NBQYFdTFhYmCIjI82YkthsNp06dcpuAgAAAFB5UJQCAFS43bt3q3bt2vLy8tKDDz6oNWvW6IYbblBmZqYkKSQkxC4+JCTEXJeZmSlPT0/VqVPnojEliY+PV2BgoDmFh4eXc6sAAAAAXAmKUgCACtesWTPt2rVL27Zt00MPPaSRI0fqu+++M9dbLBa7eMMwii270OVipk2bppycHHM6evTolTUCAAAAQLmiKAUAqHCenp5q0qSJ2rVrp/j4eLVu3VovvviirFarJBW74ykrK8u8e8pqtSo/P1/Z2dkXjSmJl5eXAgIC7CYAAAAAlQdFKQCA0xmGIZvNpsaNG8tqtSo5Odlcl5+fr5SUFHXs2FGS1LZtW3l4eNjFZGRkaM+ePWYMAAAAgKrH3dUJAACqt8cee0y9e/dWeHi4cnNzlZSUpM2bN2vdunWyWCyKiYlRXFycIiIiFBERobi4OPn6+mrYsGGSpMDAQI0ePVqTJk1ScHCwgoKCNHnyZLVs2VI9evRwcesAAAAAlBVFKQBAhTpx4oTuvfdeZWRkKDAwUK1atdK6devUs2dPSdKUKVOUl5ensWPHKjs7W+3bt9f69evl7+9vbmPevHlyd3fX4MGDlZeXp+7duyshIUFubm6uahYAAACAK0RRCgBQoZYsWXLJ9RaLRbGxsYqNjb1ojLe3txYsWKAFCxaUc3YAAAAAXIUxpQAAAAAAAOB0FKUAAAAAAADgdBSlAAAAAAAA4HQUpQAAAAAAAOB0FKUAAAAAAADgdBSlAAAAAAAA4HQUpQAAAAAAAOB0FKUAAAAAAADgdO6uTgAAAAAAADiu0dS1LtnvoVm3u2S/qH64UwoAAAAAAABO51BR6ty5c3r88cfVuHFj+fj46Nprr9VTTz2l8+fPmzGGYSg2NlZhYWHy8fFRVFSU9u7da7cdm82mCRMmqG7duvLz81P//v117Nix8mkRAAAAAAAAKj2HilLPPfec/vWvf2nhwoXat2+fZs+ereeff14LFiwwY2bPnq25c+dq4cKF2r59u6xWq3r27Knc3FwzJiYmRmvWrFFSUpK2bNmi06dPq2/fviosLCy/lgEAAAAAAKDScmhMqa1bt2rAgAG6/fY/nh9t1KiR3nzzTe3YsUPSH3dJzZ8/X9OnT9egQYMkScuWLVNISIgSExM1ZswY5eTkaMmSJVq+fLl69OghSVqxYoXCw8O1YcMG9erVqzzbBwAAAAAAgErIoTulOnfurM8++0w//PCDJOmbb77Rli1b1KdPH0lSenq6MjMzFR0dbX7Gy8tLXbp0UWpqqiQpLS1NBQUFdjFhYWGKjIw0YwAAAAAAAFC9OXSn1KOPPqqcnBxdf/31cnNzU2FhoZ599lkNHTpUkpSZmSlJCgkJsftcSEiIDh8+bMZ4enqqTp06xWKKPn8hm80mm81mzp86dcqRtAEAAAAAAFDJOHSn1KpVq7RixQolJibq66+/1rJly/TCCy9o2bJldnEWi8Vu3jCMYssudKmY+Ph4BQYGmlN4eLgjaQMAAAAAAKCScago9c9//lNTp07V//3f/6lly5a699579cgjjyg+Pl6SZLVaJanYHU9ZWVnm3VNWq1X5+fnKzs6+aMyFpk2bppycHHM6evSoI2kDAAAAAACgknGoKHX27FnVqmX/ETc3N50/f16S1LhxY1mtViUnJ5vr8/PzlZKSoo4dO0qS2rZtKw8PD7uYjIwM7dmzx4y5kJeXlwICAuwmAAAAAAAAVF0OjSnVr18/Pfvss7rmmmvUokUL7dy5U3PnztX9998v6Y/H9mJiYhQXF6eIiAhFREQoLi5Ovr6+GjZsmCQpMDBQo0eP1qRJkxQcHKygoCBNnjxZLVu2NN/GBwAAAAAAgOrNoaLUggULNGPGDI0dO1ZZWVkKCwvTmDFj9MQTT5gxU6ZMUV5ensaOHavs7Gy1b99e69evl7+/vxkzb948ubu7a/DgwcrLy1P37t2VkJAgNze38msZAAAAAAAAKi2HilL+/v6aP3++5s+ff9EYi8Wi2NhYxcbGXjTG29tbCxYs0IIFCxzZPQAAAAAAAKoJh8aUAgAAAAAAAMoDRSkAQIWJj4/XzTffLH9/f9WrV08DBw7U/v377WJGjRoli8ViN91yyy12MTabTRMmTFDdunXl5+en/v3769ixY85sCgDAiT7//HP169dPYWFhslgseu+99+zW03cAQPVAUQoAUGFSUlI0btw4bdu2TcnJyTp37pyio6N15swZu7jbbrtNGRkZ5vTxxx/brY+JidGaNWuUlJSkLVu26PTp0+rbt68KCwud2RwAgJOcOXNGrVu31sKFCy8aQ98BAFWfQ2NKAQDgiHXr1tnNL126VPXq1VNaWppuvfVWc7mXl5esVmuJ28jJydGSJUu0fPly8y2tK1asUHh4uDZs2KBevXpVXAMAAC7Ru3dv9e7d+5Ix9B0AUPVxpxQAwGlycnIkSUFBQXbLN2/erHr16qlp06b629/+pqysLHNdWlqaCgoKFB0dbS4LCwtTZGSkUlNTnZM4AKDSKe++w2az6dSpU3YTAKBiUZQCADiFYRiaOHGiOnfurMjISHN57969tXLlSm3cuFFz5szR9u3b1a1bN9lsNklSZmamPD09VadOHbvthYSEKDMz86L74+ICAKqviug74uPjFRgYaE7h4eEV3g4AqOl4fA8A4BTjx4/Xt99+qy1bttgtHzJkiPnvyMhItWvXTg0bNtTatWs1aNCgi27PMAxZLJaLro+Pj9fMmTOvPHEAQKVTEX3HtGnTNHHiRHP+1KlTFKYAoIJxpxQAoMJNmDBBH3zwgTZt2qQGDRpcMjY0NFQNGzbUgQMHJElWq1X5+fnKzs62i8vKylJISMhFtzNt2jTl5OSY09GjR6+8IQCASqk8+g4vLy8FBATYTQCAikVRCgBQYQzD0Pjx47V69Wpt3LhRjRs3vuxnTp48qaNHjyo0NFSS1LZtW3l4eCg5OdmMycjI0J49e9SxY8eLboeLCwCoOcqr7wAAOBeP7wEAKsy4ceOUmJio999/X/7+/uY4HoGBgfLx8dHp06cVGxurO++8U6GhoTp06JAee+wx1a1bV3fccYcZO3r0aE2aNEnBwcEKCgrS5MmT1bJlS/ONSgCA6uX06dM6ePCgOZ+enq5du3YpKChIQUFB9B0AUE1QlAIAVJjFixdLkqKiouyWL126VKNGjZKbm5t2796tN954Q7/99ptCQ0PVtWtXrVq1Sv7+/mb8vHnz5O7ursGDBysvL0/du3dXQkKC3NzcnNkcAICT7NixQ127djXni8Z6GjlypBYvXkzfAQDVBEUpAECFMQzjkut9fHz06aefXnY73t7eWrBggRYsWFBeqQEAKrGoqKhL9iH0HQBQPTCmFAAAAAAAAJyOohQAAAAAAACcjsf3AABAiRpNXeuyfR+adbvL9g0AAADn4E4pAAAAAAAAOB1FKQAAAAAAADgdRSkAAAAAAAA4HUUpAAAAAAAAOB1FKQAAAAAAADgdRSkAAAAAAAA4HUUpAAAAAAAAOB1FKQAAAAAAADgdRSkAAAAAAAA4HUUpAAAAAAAAOB1FKQAAAAAAADgdRSkAAAAAAAA4HUUpAAAAAAAAOB1FKQAAAAAAADidu6sTAAAAAADgSjSautbVKQAoA+6UAgAAAAAAgNNRlAIAAAAAAIDTUZQCAAAAAACA01GUAgAAAAAAgNM5XJT6+eefNXz4cAUHB8vX11c33nij0tLSzPWGYSg2NlZhYWHy8fFRVFSU9u7da7cNm82mCRMmqG7duvLz81P//v117NixK28NAAAAAAAAqgSHilLZ2dnq1KmTPDw89Mknn+i7777TnDlzdNVVV5kxs2fP1ty5c7Vw4UJt375dVqtVPXv2VG5urhkTExOjNWvWKCkpSVu2bNHp06fVt29fFRYWllvDAACuFx8fr5tvvln+/v6qV6+eBg4cqP3799vF8GUGAAAAUDO5OxL83HPPKTw8XEuXLjWXNWrUyPy3YRiaP3++pk+frkGDBkmSli1bppCQECUmJmrMmDHKycnRkiVLtHz5cvXo0UOStGLFCoWHh2vDhg3q1atXOTQLAFAZpKSkaNy4cbr55pt17tw5TZ8+XdHR0fruu+/k5+cn6X9fZiQkJKhp06Z65pln1LNnT+3fv1/+/v6S/vgy48MPP1RSUpKCg4M1adIk9e3bV2lpaXJzc3NlEwEAAGqcRlPXumzfh2bd7rJ9o/w5dKfUBx98oHbt2unuu+9WvXr11KZNG7366qvm+vT0dGVmZio6Otpc5uXlpS5duig1NVWSlJaWpoKCAruYsLAwRUZGmjEAgOph3bp1GjVqlFq0aKHWrVtr6dKlOnLkiPnY94VfZkRGRmrZsmU6e/asEhMTJcn8MmPOnDnq0aOH2rRpoxUrVmj37t3asGGDK5sHAAAA4Ao4VJT66aeftHjxYkVEROjTTz/Vgw8+qIcfflhvvPGGJCkzM1OSFBISYve5kJAQc11mZqY8PT1Vp06di8ZcyGaz6dSpU3YTAKDqycnJkSQFBQVJ4ssM/D/27j2sqjrt//hnK7g9BCSobEhEarBSyEwbFW2kVIzxUGOTlpU6Y42NhyI1R3Oatk0DZnlosOyxIbXUbGZKc0anxFQcI0tJS60xS1QsGH4aASqBh+/vjx7X4xZQQPYBeL+ua10X67vutfb9XWvvtdj3XgcAAAA0ZNW6fO/s2bPq1q2bkpOTJUldunTR3r17tXDhQo0cOdKKs9lsLvMZY8q1XehiMSkpKZo5c2Z1UgUA+BhjjCZNmqTevXsrJiZG0sV/zDh06JAVU90fM6Qff9AoLS21xvlBAwAAAPAt1TpTKiwsTB07dnRpu/7663X48GFJksPhkKRyXxLy8/OtLxwOh0NlZWUqKCioNOZC06dPV2FhoTXk5ORUJ20AgA+YMGGCPvvsM73xxhvlptX2jxnSjz9oBAUFWUNERETNEgcAAADgFtUqSvXq1avcU5O+/PJLRUZGSpKioqLkcDiUnp5uTS8rK1NGRobi4uIkSV27dpW/v79LTG5urvbs2WPFXMhutyswMNBlAADUHRMnTtSaNWu0adMmtW3b1mp3148ZEj9oAAAAAL6uWkWpxx57TNu2bVNycrK++uorrVixQosWLdL48eMl/fhLd1JSkpKTk7Vq1Srt2bNHo0ePVvPmzTVixAhJUlBQkMaMGaPJkyfr/fff186dO3X//fcrNjbWehofAKB+MMZowoQJevvtt7Vx40ZFRUW5THfXjxkSP2gAAAAAvq5a95S6+eabtWrVKk2fPl1PP/20oqKiNH/+fN13331WzNSpU1VSUqJx48apoKBA3bt31/r1663HekvSvHnz5Ofnp2HDhqmkpER9+/bVkiVLeKw3ANQz48eP14oVK/TOO+8oICDAOiMqKChIzZo1c/kxIzo6WtHR0UpOTq70x4yQkBAFBwdrypQp/JgBAAAA1HHVKkpJ0qBBgzRo0KBKp9tsNjmdTjmdzkpjmjZtqtTUVKWmplb35QEAdcjChQslSfHx8S7tixcv1ujRoyXxYwYAAADQUFW7KAUAQFUZYy4Zw48ZAAAAQMNUrXtKAQAAAAAAALWBohQAAAAAAAA8jqIUAAAAAAAAPI6iFAAAAAAAADyOohQAAAAAAAA8jqIUAAAAAAAAPI6iFAAAAAAAADyOohQAAAAAAAA8zs/bCQAAAAAAAFRF+2lrvfK6B2cN9Mrr1necKQUAAAAAAACPoygFAAAAAAAAj6MoBQAAAMCnbNmyRYMHD1Z4eLhsNptWr17tMt0YI6fTqfDwcDVr1kzx8fHau3evS0xpaakmTpyoVq1aqUWLFhoyZIiOHDniwV4AAC6FohQAAAAAn3LixAl17txZCxYsqHD67NmzNXfuXC1YsEDbt2+Xw+FQ//79VVxcbMUkJSVp1apVWrlypbZu3arjx49r0KBBOnPmjKe6AQC4BG50DgAAAMCnJCYmKjExscJpxhjNnz9fM2bM0NChQyVJS5cuVWhoqFasWKGxY8eqsLBQaWlpev3119WvXz9J0rJlyxQREaENGzZowIABHusLAKBynCkFAAAAoM7Izs5WXl6eEhISrDa73a4+ffooMzNTkpSVlaVTp065xISHhysmJsaKAQB4H2dKAQAAAKgz8vLyJEmhoaEu7aGhoTp06JAV06RJE7Vs2bJczLn5L1RaWqrS0lJrvKioqDbTBgBUgDOlAAAAANQ5NpvNZdwYU67tQheLSUlJUVBQkDVERETUWq4AgIpRlAIAAABQZzgcDkkqd8ZTfn6+dfaUw+FQWVmZCgoKKo250PTp01VYWGgNOTk5bsgeAHA+ilIAAAAA6oyoqCg5HA6lp6dbbWVlZcrIyFBcXJwkqWvXrvL393eJyc3N1Z49e6yYC9ntdgUGBroMAAD34p5SAAAAAHzK8ePH9dVXX1nj2dnZ2rVrl4KDg9WuXTslJSUpOTlZ0dHRio6OVnJyspo3b64RI0ZIkoKCgjRmzBhNnjxZISEhCg4O1pQpUxQbG2s9jQ8A4H0UpQAAAAD4lB07dujWW2+1xidNmiRJGjVqlJYsWaKpU6eqpKRE48aNU0FBgbp3767169crICDAmmfevHny8/PTsGHDVFJSor59+2rJkiVq3Lixx/sDAKgYRSkAAAAAPiU+Pl7GmEqn22w2OZ1OOZ3OSmOaNm2q1NRUpaamuiFDAEBt4J5SAAAAAAAA8DiKUgAAt9qyZYsGDx6s8PBw2Ww2rV692mX66NGjZbPZXIYePXq4xJSWlmrixIlq1aqVWrRooSFDhujIkSMe7AUAAACA2kZRCgDgVidOnFDnzp21YMGCSmNuv/125ebmWsO6detcpiclJWnVqlVauXKltm7dquPHj2vQoEE6c+aMu9MHAAAA4CbcUwoA4FaJiYlKTEy8aIzdbpfD4ahwWmFhodLS0vT6669bT0xatmyZIiIitGHDBg0YMKDWcwYAAADgfpwpBQDwus2bN6tNmzbq0KGDHnroIeXn51vTsrKydOrUKSUkJFht4eHhiomJUWZmpjfSBQAAAFALOFMKAOBViYmJuvvuuxUZGans7Gw9+eSTuu2225SVlSW73a68vDw1adJELVu2dJkvNDRUeXl5lS63tLRUpaWl1nhRUZHb+gAAAACg+ihKAQC8avjw4dbfMTEx6tatmyIjI7V27VoNHTq00vmMMbLZbJVOT0lJ0cyZM2s1VwAAAAC1h8v3AAA+JSwsTJGRkdq/f78kyeFwqKysTAUFBS5x+fn5Cg0NrXQ506dPV2FhoTXk5OS4NW8AAAAA1UNRCgDgU44dO6acnByFhYVJkrp27Sp/f3+lp6dbMbm5udqzZ4/i4uIqXY7dbldgYKDLAAAAAMB3cPkeAMCtjh8/rq+++soaz87O1q5duxQcHKzg4GA5nU7dddddCgsL08GDB/XEE0+oVatW+sUvfiFJCgoK0pgxYzR58mSFhIQoODhYU6ZMUWxsrPU0PgAAAAB1D0UpAIBb7dixQ7feeqs1PmnSJEnSqFGjtHDhQu3evVuvvfaavv/+e4WFhenWW2/Vm2++qYCAAGueefPmyc/PT8OGDVNJSYn69u2rJUuWqHHjxh7vDwAAABqe9tPWeu21D84a6LXXdjeKUgAAt4qPj5cxptLp77333iWX0bRpU6Wmpio1NbU2UwMAAADgRZd1T6mUlBTZbDYlJSVZbcYYOZ1OhYeHq1mzZoqPj9fevXtd5istLdXEiRPVqlUrtWjRQkOGDNGRI0cuJxUAAAAAAADUITUuSm3fvl2LFi3SDTfc4NI+e/ZszZ07VwsWLND27dvlcDjUv39/FRcXWzFJSUlatWqVVq5cqa1bt+r48eMaNGiQzpw5U/OeAAAAAAAAoM6oUVHq+PHjuu+++/TKK6+oZcuWVrsxRvPnz9eMGTM0dOhQxcTEaOnSpTp58qRWrFghSSosLFRaWprmzJmjfv36qUuXLlq2bJl2796tDRs21E6vAAAAAAAA4NNqVJQaP368Bg4cWO6pR9nZ2crLy1NCQoLVZrfb1adPH2VmZkqSsrKydOrUKZeY8PBwxcTEWDEXKi0tVVFRkcsAAAAAAACAuqvaNzpfuXKlPvnkE23fvr3ctLy8PElSaGioS3toaKgOHTpkxTRp0sTlDKtzMefmv1BKSopmzpxZ3VQBAAAAAADgo6p1plROTo4effRRLVu2TE2bNq00zmazuYwbY8q1XehiMdOnT1dhYaE15OTkVCdtAAAAAAAA+JhqFaWysrKUn5+vrl27ys/PT35+fsrIyNCf//xn+fn5WWdIXXjGU35+vjXN4XCorKxMBQUFlcZcyG63KzAw0GUAAAAAAABA3VWtolTfvn21e/du7dq1yxq6deum++67T7t27dLVV18th8Oh9PR0a56ysjJlZGQoLi5OktS1a1f5+/u7xOTm5mrPnj1WDAAAAAAAAOq3at1TKiAgQDExMS5tLVq0UEhIiNWelJSk5ORkRUdHKzo6WsnJyWrevLlGjBghSQoKCtKYMWM0efJkhYSEKDg4WFOmTFFsbGy5G6cDAAAAAACgfqr2jc4vZerUqSopKdG4ceNUUFCg7t27a/369QoICLBi5s2bJz8/Pw0bNkwlJSXq27evlixZosaNG9d2OgAAAAAAAPBBl12U2rx5s8u4zWaT0+mU0+msdJ6mTZsqNTVVqampl/vyAAAAAAAAqIOqdU8pAAAAAAAAoDZQlAIAAAAAAIDHUZQCAAAAAACAx1GUAgAAAAAAgMdRlAIAAAAAAIDHUZQCAAAAAACAx1GUAgAAAAAAgMdRlAIAAAAAAIDHUZQCAAAAAACAx/l5OwEAAOqC9tPWejsFAAAAoF7hTCkAAAAAAAB4HEUpAAAAAAAAeBxFKQCAW23ZskWDBw9WeHi4bDabVq9e7TLdGCOn06nw8HA1a9ZM8fHx2rt3r0tMaWmpJk6cqFatWqlFixYaMmSIjhw54sFeAAAAAKhtFKUAAG514sQJde7cWQsWLKhw+uzZszV37lwtWLBA27dvl8PhUP/+/VVcXGzFJCUladWqVVq5cqW2bt2q48ePa9CgQTpz5oynugEAAACglnGjcwCAWyUmJioxMbHCacYYzZ8/XzNmzNDQoUMlSUuXLlVoaKhWrFihsWPHqrCwUGlpaXr99dfVr18/SdKyZcsUERGhDRs2aMCAAR7rCwAAAIDaw5lSAACvyc7OVl5enhISEqw2u92uPn36KDMzU5KUlZWlU6dOucSEh4crJibGiqlIaWmpioqKXAYAAAAAvoMzpQAAXpOXlydJCg0NdWkPDQ3VoUOHrJgmTZqoZcuW5WLOzV+RlJQUzZw5s5Yzhqe0n7bWa699cNZAr702AABAQ8KZUgAAr7PZbC7jxphybRe6VMz06dNVWFhoDTk5ObWSKwAAAIDaQVEKAOA1DodDksqd8ZSfn2+dPeVwOFRWVqaCgoJKYypit9sVGBjoMgAAAADwHRSlAABeExUVJYfDofT0dKutrKxMGRkZiouLkyR17dpV/v7+LjG5ubnas2ePFQMAAACg7uGeUgAAtzp+/Li++uorazw7O1u7du1ScHCw2rVrp6SkJCUnJys6OlrR0dFKTk5W8+bNNWLECElSUFCQxowZo8mTJyskJETBwcGaMmWKYmNjrafxAQAAAKh7KEoBANxqx44duvXWW63xSZMmSZJGjRqlJUuWaOrUqSopKdG4ceNUUFCg7t27a/369QoICLDmmTdvnvz8/DRs2DCVlJSob9++WrJkiRo3buzx/gAAAACoHRSlAABuFR8fL2NMpdNtNpucTqecTmelMU2bNlVqaqpSU1PdkCEAAAAAb+CeUgAAAADqFKfTKZvN5jKce3iG9OMTWp1Op8LDw9WsWTPFx8dr7969XswYAFARilIAAAAA6pxOnTopNzfXGnbv3m1Nmz17tubOnasFCxZo+/btcjgc6t+/v4qLi72YMQDgQhSlAAAAANQ5fn5+cjgc1tC6dWtJP54lNX/+fM2YMUNDhw5VTEyMli5dqpMnT2rFihVezhoAcD6KUgAAAADqnP379ys8PFxRUVG65557dODAAUk/PuU1Ly9PCQkJVqzdblefPn2UmZlZ6fJKS0tVVFTkMgAA3IsbnQMAAJyn/bS1Xnndg7MGeuV1gbqoe/fueu2119ShQwf997//1TPPPKO4uDjt3btXeXl5kqTQ0FCXeUJDQ3Xo0KFKl5mSkqKZM2e6NW8AgCvOlAIAAABQpyQmJuquu+5SbGys+vXrp7VrfywmL1261Iqx2Wwu8xhjyrWdb/r06SosLLSGnJwc9yQPALBQlAIAAABQp7Vo0UKxsbHav3+/9RS+c2dMnZOfn1/u7Knz2e12BQYGugwAAPeiKAUAAACgTistLdUXX3yhsLAwRUVFyeFwKD093ZpeVlamjIwMxcXFeTFLAMCFuKcUAAAAgDplypQpGjx4sNq1a6f8/Hw988wzKioq0qhRo2Sz2ZSUlKTk5GRFR0crOjpaycnJat68uUaMGOHt1AEA56EoBQAAAKBOOXLkiO69914dPXpUrVu3Vo8ePbRt2zZFRkZKkqZOnaqSkhKNGzdOBQUF6t69u9avX6+AgAAvZw4AOB9FKQAAAAB1ysqVKy863Wazyel0yul0eiYhAECNVKsolZKSorffflv/+c9/1KxZM8XFxenZZ5/Vtddea8UYYzRz5kwtWrTI+lXixRdfVKdOnayY0tJSTZkyRW+88YZKSkrUt29fvfTSS2rbtm3t9QwAAAAA4FHtp631dgoA6pBq3eg8IyND48eP17Zt25Senq7Tp08rISFBJ06csGJmz56tuXPnasGCBdq+fbscDof69++v4uJiKyYpKUmrVq3SypUrtXXrVh0/flyDBg3SmTNnaq9nAAAAAAAA8FnVOlPq3XffdRlfvHix2rRpo6ysLP3sZz+TMUbz58/XjBkzNHToUEnS0qVLFRoaqhUrVmjs2LEqLCxUWlqaXn/9dfXr10+StGzZMkVERGjDhg0aMGBALXUNAAAAAAAAvqpaZ0pdqLCwUJIUHBwsScrOzlZeXp4SEhKsGLvdrj59+igzM1OSlJWVpVOnTrnEhIeHKyYmxooBAAAAAABA/VbjG50bYzRp0iT17t1bMTExkqS8vDxJUmhoqEtsaGioDh06ZMU0adJELVu2LBdzbv4LlZaWqrS01BovKiqqadoAAAAAAADwATU+U2rChAn67LPP9MYbb5SbZrPZXMaNMeXaLnSxmJSUFAUFBVlDRERETdMGAAAAAACAD6hRUWrixIlas2aNNm3a5PLEPIfDIUnlznjKz8+3zp5yOBwqKytTQUFBpTEXmj59ugoLC60hJyenJmkDAAAAAADAR1SrKGWM0YQJE/T2229r48aNioqKcpkeFRUlh8Oh9PR0q62srEwZGRmKi4uTJHXt2lX+/v4uMbm5udqzZ48VcyG73a7AwECXAQAAAAAAAHVXte4pNX78eK1YsULvvPOOAgICrDOigoKC1KxZM9lsNiUlJSk5OVnR0dGKjo5WcnKymjdvrhEjRlixY8aM0eTJkxUSEqLg4GBNmTJFsbGx1tP4AAAAAAAAUL9Vqyi1cOFCSVJ8fLxL++LFizV69GhJ0tSpU1VSUqJx48apoKBA3bt31/r16xUQEGDFz5s3T35+fho2bJhKSkrUt29fLVmyRI0bN7683gAAAAAAAKBOqFZRyhhzyRibzSan0ymn01lpTNOmTZWamqrU1NTqvDwAAAAAAADqiRo/fQ8AAAAAAACoqWqdKQUAQG1zOp2aOXOmS1toaKh130JjjGbOnKlFixZZl4W/+OKL6tSpkzfSBQAAADyq/bS1Xnvtg7MGunX5nCkFAPC6Tp06KTc31xp2795tTZs9e7bmzp2rBQsWaPv27XI4HOrfv7+Ki4u9mDEAAACAy0VRCgDgdX5+fnI4HNbQunVrST+eJTV//nzNmDFDQ4cOVUxMjJYuXaqTJ09qxYoVXs4aAAAAwOWgKAUA8Lr9+/crPDxcUVFRuueee3TgwAFJUnZ2tvLy8pSQkGDF2u129enTR5mZmRddZmlpqYqKilwGAAAAAL6De0oBALyqe/fueu2119ShQwf997//1TPPPKO4uDjt3bvXuq9UaGioyzyhoaE6dOjQRZebkpJS7l5VgC+rz/eLAAAAqAhnSgEAvCoxMVF33XWXYmNj1a9fP61d++MX86VLl1oxNpvNZR5jTLm2C02fPl2FhYXWkJOTU/vJAwAAAKgxilIAAJ/SokULxcbGav/+/XI4HJJknTF1Tn5+frmzpy5kt9sVGBjoMgAAAADwHRSlAAA+pbS0VF988YXCwsIUFRUlh8Oh9PR0a3pZWZkyMjIUFxfnxSwBAAAAXC7uKQUA8KopU6Zo8ODBateunfLz8/XMM8+oqKhIo0aNks1mU1JSkpKTkxUdHa3o6GglJyerefPmGjFihLdTBwAAAHAZKEoBALzqyJEjuvfee3X06FG1bt1aPXr00LZt2xQZGSlJmjp1qkpKSjRu3DgVFBSoe/fuWr9+vQICArycOQAAAIDLQVEKAOBVK1euvOh0m80mp9Mpp9PpmYQAAAAAeARFKQC4DDzCHQAAAABqhhudAwAAAAAAwOMoSgEAAAAAAMDjKEoBAAAAAADA4yhKAQAAAAAAwOMoSgEAAAAAAMDjKEoBAAAAAADA4yhKAQAAAAAAwOMoSgEAAAAAAMDjKEoBAAAAAADA4yhKAQAAAAAAwOMoSgEAAAAAAMDjKEoBAAAAAADA4/y8nQAAAAAAoPa0n7bW2ykAQJVwphQAAAAAAAA8jqIUAAAAAAAAPI6iFAAAAAAAADyOohQAAAAAAAA8jqIUAAAAAAAAPI6n7wEAAACAG/AUPAC4OIpSAAAA8ApvfmE/OGug114bAAD8iMv3AAAAAAAA4HGcKQUAANDAcYkRAADwBq+eKfXSSy8pKipKTZs2VdeuXfXvf//bm+kAAHwcxw0AQHVx7AAA3+W1otSbb76ppKQkzZgxQzt37tQtt9yixMREHT582FspAQB8GMcNAEB1cewAAN/mtcv35s6dqzFjxujBBx+UJM2fP1/vvfeeFi5cqJSUFG+lBQDwURw3ANQX3rpcsiHe3J1jBwD4Nq8UpcrKypSVlaVp06a5tCckJCgzM7NcfGlpqUpLS63xwsJCSVJRUVGNczhberLG816Oy8kZgO/x1r5Eqvn+5Nx8xpjaTMetqnvckGr/2OHNbQ2g9rV77G/eTsHjLuf/0IZw7KhP3zkAoLa4+zuHV4pSR48e1ZkzZxQaGurSHhoaqry8vHLxKSkpmjlzZrn2iIgIt+XoLkHzvZ0BgPricvcnxcXFCgoKqpVc3K26xw2pfh07AKA21Mb/ofX52MFxAwDKc/d3Dq8+fc9ms7mMG2PKtUnS9OnTNWnSJGv87Nmz+u677xQSElJh/KUUFRUpIiJCOTk5CgwMrH7idRB9ps/1WUPs9+X02Rij4uJihYeHuyk796nqcUOq/WOHJzTE93JtYv1dHtbf5anv668hHDvq4nHjYur6e7Ku5y/V/T6Qv3fV9fyretzwSlGqVatWaty4cblfKPLz88v9kiFJdrtddrvdpe3KK6+87DwCAwPr5Ma9HPS5YWiIfZYaZr9r2ue68iv3OdU9bkjuO3Z4QkN8L9cm1t/lYf1dnvq8/ur7saMuHzcupq6/J+t6/lLd7wP5e1ddzr8qxw2vPH2vSZMm6tq1q9LT013a09PTFRcX542UAAA+jOMGAKC6OHYAgO/z2uV7kyZN0gMPPKBu3bqpZ8+eWrRokQ4fPqyHH37YWykBAHwYxw0AQHVx7AAA3+a1otTw4cN17NgxPf3008rNzVVMTIzWrVunyMhIt7+23W7XU089Ve703PqMPjcMDbHPUsPsd0PsszePG57SELdrbWL9XR7W3+Vh/fmmhnDsqExdf0/W9fylut8H8veuup5/VdlMXXquKwAAAAAAAOoFr9xTCgAAAAAAAA0bRSkAAAAAAAB4HEUpAAAAAAAAeBxFKQAAAAAAAHhcvSxKvfTSS4qKilLTpk3VtWtX/fvf/75ofEZGhrp27aqmTZvq6quv1ssvv+yhTGtXdfr99ttvq3///mrdurUCAwPVs2dPvffeex7MtnZUd1uf88EHH8jPz0833nijexN0g+r2ubS0VDNmzFBkZKTsdruuueYavfrqqx7KtnZUt8/Lly9X586d1bx5c4WFhelXv/qVjh075qFsL9+WLVs0ePBghYeHy2azafXq1Zecp77sxxqClJQU3XzzzQoICFCbNm105513at++fS4xxhg5nU6Fh4erWbNmio+P1969e72UsW+pyvobPXq0bDaby9CjRw8vZexbFi5cqBtuuEGBgYHW8f9f//qXNZ333sVdav3x3oO7XOq9d6HNmzeXey/abDb95z//cYl766231LFjR9ntdnXs2FGrVq3yifwr+izZbDZ16tTJilmyZEmFMT/88INb+nC+lJQU2Ww2JSUlXTSuKv+feWobnK8q+Vfl+6K3tkFV8ve1z8D5qpK/r38GalO9K0q9+eabSkpK0owZM7Rz507dcsstSkxM1OHDhyuMz87O1s9//nPdcsst2rlzp5544gk98sgjeuuttzyc+eWpbr+3bNmi/v37a926dcrKytKtt96qwYMHa+fOnR7OvOaq2+dzCgsLNXLkSPXt29dDmdaemvR52LBhev/995WWlqZ9+/bpjTfe0HXXXefBrC9Pdfu8detWjRw5UmPGjNHevXv1t7/9Tdu3b9eDDz7o4cxr7sSJE+rcubMWLFhQpfj6sh9rKDIyMjR+/Hht27ZN6enpOn36tBISEnTixAkrZvbs2Zo7d64WLFig7du3y+FwqH///iouLvZi5r6hKutPkm6//Xbl5uZaw7p167yUsW9p27atZs2apR07dmjHjh267bbbdMcdd1iFJ957F3ep9Sfx3oN7VOW9V5F9+/a5vB+jo6OtaR9++KGGDx+uBx54QJ9++qkeeOABDRs2TB999JHX83/hhRdc8s7JyVFwcLDuvvtul7jAwECXuNzcXDVt2rTW8z/f9u3btWjRIt1www0XjavK/2ee3AbVzb+q3xc9vQ2qmv85vvIZOKeq+fvyZ6DWmXrmpz/9qXn44Ydd2q677jozbdq0CuOnTp1qrrvuOpe2sWPHmh49ergtR3eobr8r0rFjRzNz5szaTs1tatrn4cOHm9///vfmqaeeMp07d3ZjhrWvun3+17/+ZYKCgsyxY8c8kZ5bVLfPzz33nLn66qtd2v785z+btm3bui1Hd5JkVq1addGY+rIfa6jy8/ONJJORkWGMMebs2bPG4XCYWbNmWTE//PCDCQoKMi+//LK30vRZF64/Y4wZNWqUueOOO7yXVB3TsmVL85e//IX3Xg2dW3/G8N6DZ53/3rvQpk2bjCRTUFBQ6fzDhg0zt99+u0vbgAEDzD333FObaVbqYvlfaNWqVcZms5mDBw9abYsXLzZBQUFuyq5ixcXFJjo62qSnp5s+ffqYRx99tNLYqvx/5ultUJ38K3Lh90VPb4Pq5O+Ln4HLWf++8hlwh3p1plRZWZmysrKUkJDg0p6QkKDMzMwK5/nwww/LxQ8YMEA7duzQqVOn3JZrbapJvy909uxZFRcXKzg42B0p1rqa9nnx4sX6+uuv9dRTT7k7xVpXkz6vWbNG3bp10+zZs3XVVVepQ4cOmjJlikpKSjyR8mWrSZ/j4uJ05MgRrVu3TsYY/fe//9Xf//53DRw40BMpe0V92I81ZIWFhZJk7X+zs7OVl5fnsk3tdrv69OlT5X16Q3Lh+jtn8+bNatOmjTp06KCHHnpI+fn53kjPp505c0YrV67UiRMn1LNnT9571XTh+juH9x7crbL3XkW6dOmisLAw9e3bV5s2bXKZVtn/D+7+vFcn/3PS0tLUr18/RUZGurQfP35ckZGRatu2rQYNGuT2qz7Gjx+vgQMHql+/fpeMrcr/Z57eBtXJ/0KVfV/05DaoSf6+9Bm4nPXvK58Bd/DzdgK16ejRozpz5oxCQ0Nd2kNDQ5WXl1fhPHl5eRXGnz59WkePHlVYWJjb8q0tNen3hebMmaMTJ05o2LBh7kix1tWkz/v379e0adP073//W35+de+tX5M+HzhwQFu3blXTpk21atUqHT16VOPGjdN3331XJ+4rVZM+x8XFafny5Ro+fLh++OEHnT59WkOGDFFqaqonUvaK+rAfa6iMMZo0aZJ69+6tmJgYSbLe2xVt00OHDnk8R19W0fqTpMTERN19992KjIxUdna2nnzySd12223KysqS3W73Ysa+Yffu3erZs6d++OEHXXHFFVq1apU6duxo/RPOe+/iKlt/Eu89uNfF3nsXCgsL06JFi9S1a1eVlpbq9ddfV9++fbV582b97Gc/k1T5/w9V/f7gzvzPl5ubq3/9619asWKFS/t1112nJUuWKDY2VkVFRXrhhRfUq1cvffrppy6XaNWWlStX6pNPPtH27durFF+V/888uQ2qm/+FKvq+6MltUN38fe0zcDnr31c+A+5S976ZV4HNZnMZN8aUa7tUfEXtvq66/T7njTfekNPp1DvvvKM2bdq4Kz23qGqfz5w5oxEjRmjmzJnq0KGDp9Jzi+ps57Nnz8pms2n58uUKCgqSJM2dO1e//OUv9eKLL6pZs2Zuz7c2VKfPn3/+uR555BH94Q9/0IABA5Sbm6vHH39cDz/8sNLS0jyRrlfUl/1YQzNhwgR99tln2rp1a7lpNd2nNySVrb/hw4dbf8fExKhbt26KjIzU2rVrNXToUE+n6XOuvfZa7dq1S99//73eeustjRo1ShkZGdZ03nsXV9n669ixI+89uNXF3nsVxV577bXWeM+ePZWTk6Pnn3/e+kIuefbzXp38z7dkyRJdeeWVuvPOO13ae/To4fIggV69eummm25Samqq/vznP9dq7jk5OXr00Ue1fv36at2vpyr/n3liG9Q0/3Mq+77oqW1Qk/x96TNwuevfFz4D7lSvilKtWrVS48aNy1U28/Pzy1VAz3E4HBXG+/n5KSQkxG251qaa9PucN998U2PGjNHf/va3Gp1G6C3V7XNxcbF27NihnTt3asKECZJ+LNgYY+Tn56f169frtttu80juNVWT7RwWFqarrrrKKkhJ0vXXXy9jjI4cOeLzFfSa9DklJUW9evXS448/Lkm64YYb1KJFC91yyy165pln6uVZQ/VhP9YQTZw4UWvWrNGWLVvUtm1bq93hcEj68de789+vVdmnNySVrb+KhIWFKTIyUvv37/dQdr6tSZMm+slPfiJJ6tatm7Zv364XXnhBv/vd7yTx3ruUytbf//zP/5SL5b2H2lSd915FevTooWXLllnjlf3/4K7Pe03yN8bo1Vdf1QMPPKAmTZpcdPmNGjXSzTff7JbPW1ZWlvLz89W1a1er7cyZM9qyZYsWLFig0tJSNW7c2GWeqvx/5qltUJP8z6nO90V3bYPLyf983voMXE7+vvIZcKd6dU+pJk2aqGvXrkpPT3dpT09PV1xcXIXz9OzZs1z8+vXr1a1bN/n7+7st19pUk35LP1a8R48erRUrVtS5++1Ut8+BgYHavXu3du3aZQ0PP/yw9YtN9+7dPZV6jdVkO/fq1Uvffvutjh8/brV9+eWXatSo0SW/xPmCmvT55MmTatTIddd2bid/7tep+qY+7McaEmOMJkyYoLffflsbN25UVFSUy/SoqCg5HA6XbVpWVqaMjIyL7tMbikutv4ocO3ZMOTk59bIoXRuMMSotLeW9V0Pn1l9FeO/BnS723qvIzp07Xd6Llf3/4KnPe1Xyz8jI0FdffaUxY8ZUaXm7du1yy+etb9++5b5LdOvWTffdd5927dpVYUGhKv+feWob1CR/qfrfF921DWqa/4W89Rm4nPx95TPgVp64m7onrVy50vj7+5u0tDTz+eefm6SkJNOiRQvrLvXTpk0zDzzwgBV/4MAB07x5c/PYY4+Zzz//3KSlpRl/f3/z97//3VtdqJHq9nvFihXGz8/PvPjiiyY3N9cavv/+e291odqq2+cL1cWn71W3z8XFxaZt27bml7/8pdm7d6/JyMgw0dHR5sEHH/RWF6qtun1evHix8fPzMy+99JL5+uuvzdatW023bt3MT3/6U291odqKi4vNzp07zc6dO40kM3fuXLNz505z6NAhY0z93Y81FL/97W9NUFCQ2bx5s8v+9+TJk1bMrFmzTFBQkHn77bfN7t27zb333mvCwsJMUVGRFzP3DZdaf8XFxWby5MkmMzPTZGdnm02bNpmePXuaq666ivVnjJk+fbrZsmWLyc7ONp999pl54oknTKNGjcz69euNMbz3LuVi64/3HtzpUp/dC/83mDdvnlm1apX58ssvzZ49e8y0adOMJPPWW29ZMR988IFp3LixmTVrlvniiy/MrFmzjJ+fn9m2bZvX8z/n/vvvN927d69wmU6n07z77rvm66+/Njt37jS/+tWvjJ+fn/noo49qPf+KXPj0tJr8f+bJbVDd/KvyfdGb2+BS+fvaZ6C6+Z/jy5+B2lLvilLGGPPiiy+ayMhI06RJE3PTTTeVe0x0nz59XOI3b95sunTpYpo0aWLat29vFi5c6OGMa0d1+t2nTx8jqdwwatQozyd+Gaq7rc9XF4tSxlS/z1988YXp16+fadasmWnbtq2ZNGmSy5ffuqC6ff7zn/9sOnbsaJo1a2bCwsLMfffdZ44cOeLhrGvu3CNsK/t81uf9WENQ0baVZBYvXmzFnD171jz11FPG4XAYu91ufvazn5ndu3d7L2kfcqn1d/LkSZOQkGBat25t/P39Tbt27cyoUaPM4cOHvZu4j/j1r39t7U9bt25t+vbta30pNIb33qVcbP3x3oM7Xeqze+H/Bs8++6y55pprTNOmTU3Lli1N7969zdq1a8st929/+5u59tprjb+/v7nuuutcvrB7M39jjPn+++9Ns2bNzKJFiypcZlJSkmnXrp21zISEBJOZmemW/CtyYVGhpv+feWobXOhS+Vfl+6I3t8Gl8ve1z8CFqvL+8fXPQG2xGVNPr2cBAAAAAACAz6pX95QCAAAAAABA3UBRCgAAAAAAAB5HUQoAAAAAAAAeR1EKAAAAAAAAHkdRCgAAAAAAAB5HUQoAAAAAAAAeR1EKAAAAAAAAHkdRCgAAAAAAAB5HUQo+Lz4+XklJSd5Oo95YsmSJrrzySpe2RYsWKSIiQo0aNdL8+fO9khcAAACAuuPgwYOy2WzatWtXheNAVVCUAmpZRUUfb2nfvn25ItPw4cP15ZdfWuNFRUWaMGGCfve73+mbb77Rb37zGw9nCQANS1lZmbdTAADUgp07d+ruu+9WaGiomjZtqg4dOuihhx5y+V/7nISEBDVu3Fjbtm3zQqaeERERodzcXMXExHg7FdQhFKWABqZZs2Zq06aNNX748GGdOnVKAwcOVFhYmJo3b+7F7ACg/omPj9eECRM0adIktWrVSv3791dGRoZ++tOfym63KywsTNOmTdPp06eteYqLi3XfffepRYsWCgsL07x58zhzGAB8RFlZmf75z3+qR48eKi0t1fLly/XFF1/o9ddfV1BQkJ588kmX+MOHD+vDDz/UhAkTlJaW5qWs3a9x48ZyOBzy8/PzdiqoQyhKoU4pKCjQyJEj1bJlSzVv3lyJiYnav3+/S8xbb72lTp06yW63q3379pozZ47L9Pbt2+uPf/yjRowYoSuuuELh4eFKTU2tcg5z585VbGysWrRooYiICI0bN07Hjx+XJG3evFm/+tWvVFhYKJvNJpvNJqfTeclltm/fXs8884xGjhypK664QpGRkXrnnXf0//7f/9Mdd9yhK664QrGxsdqxY0eV+xofH69Dhw7pscces3KRXM/kWrJkiWJjYyVJV199tWw2mw4ePFjldQEAqJqlS5fKz89PH3zwgZKTk/Xzn/9cN998sz799FMtXLhQaWlpeuaZZ6z4SZMm6YMPPtCaNWuUnp6uf//73/rkk0+82AMAaLgu/HGhV69e+tWvfqWf//znWrNmjfr166eoqCh1795dzz//vP7nf/7HZf7Fixdr0KBB+u1vf6s333xTJ06cqNZrP/LII5o6daqCg4PlcDjKfb84fPiw9Z0hMDBQw4YN03//+19rutPp1I033qjXX39d7du3V1BQkO655x4VFxdbMe+++6569+6tK6+8UiEhIRo0aJC+/vprl9f5+OOP1aVLFzVt2lTdunXTzp07XaZz+R5qgqIU6pTRo0drx44dWrNmjT788EMZY/Tzn/9cp06dkiRlZWVp2LBhuueee7R79245nU49+eSTWrJkictynnvuOd1www365JNPNH36dD322GNKT0+vUg6NGjXSn//8Z+3Zs0dLly7Vxo0bNXXqVElSXFyc5s+fr8DAQOXm5io3N1dTpkyp0nLnzZunXr16aefOnRo4cKAeeOABjRw5Uvfff78++eQT/eQnP9HIkSNljKlSX99++221bdtWTz/9tJXLhYYPH64NGzZI+vEgk5ubq4iIiCrlCwCoup/85CeaPXu2rr32Wq1bt04RERFasGCBrrvuOt15552aOXOm5syZo7Nnz6q4uFhLly7V888/r759+yomJkaLFy/WmTNnvN0NAGiwzv9xYejQoTp69Kj1HeBC59/KwxijxYsX6/7779d1112nDh066K9//Wu1X7tFixb66KOPNHv2bD399NPWdxdjjO6880599913ysjIUHp6ur7++msNHz7cZRlff/21Vq9erX/+85/65z//qYyMDM2aNcuafuLECU2aNEnbt2/X+++/r0aNGukXv/iFzp49a00fNGiQrr32WmVlZcnpdFb5ew5wUQbwcX369DGPPvqo+fLLL40k88EHH1jTjh49apo1a2b++te/GmOMGTFihOnfv7/L/I8//rjp2LGjNR4ZGWluv/12l5jhw4ebxMTEGuX317/+1YSEhFjjixcvNkFBQdVaRmRkpLn//vut8dzcXCPJPPnkk1bbhx9+aCSZ3NxcY0zV+zpv3jyXmAvz27lzp5FksrOzq5UzAKBq+vTpYx588EFr/Be/+IUZPXq0S8yuXbuMJHPo0CGXv8/XpUsX8+ijj3oiZQDAefr06WNuvPFGa/zZZ581ksx33313yXnXr19vWrdubU6dOmWMMWbevHmmV69e1Xrt3r17u7TdfPPN5ne/+521/MaNG5vDhw9b0/fu3WskmY8//tgYY8xTTz1lmjdvboqKiqyYxx9/3HTv3r3S183PzzeSzO7du40xxvzP//yPCQ4ONidOnLBiFi5caCSZnTt3GmOMyc7OdhkHqoIzpVBnfPHFF/Lz81P37t2ttpCQEF177bX64osvrJhevXq5zNerVy/t37/f5Rfmnj17usT07NnTWsalbNq0Sf3799dVV12lgIAAjRw5UseOHavWabgVueGGG6y/Q0NDJcm6tO78tvz8fElV7ysAwPtatGhh/W2MsS6pPr9Nkmw2m8vfFcUAADyvW7du1t/V2R+npaVp+PDh1n2W7r33Xn300Ufat29flZdx/vcESQoLC3P5ThAREeFytUPHjh115ZVXuny/ad++vQICAipchvTjmVQjRozQ1VdfrcDAQEVFRUn68dLAc6/TuXNnl/vPXvidCqgJilKoMyrb+Z//z/3F/tG/lAvnq8ihQ4f085//XDExMXrrrbeUlZWlF198UZKsSwhryt/fv1wuFbWdO4X2cvoKAPCejh07KjMz02WfnZmZqYCAAF111VW65ppr5O/vr48//tiaXlRUVO4eigAAzzn/x4UOHTpIkv7zn/9cdJ7vvvtOq1ev1ksvvSQ/Pz/5+fnpqquu0unTp/Xqq69W+bXP/04g/fi94GLfCSpqv9gyJGnw4ME6duyYXnnlFX300Uf66KOPJP3fE2P5ngF3oSiFOqNjx446ffq0tYOUpGPHjunLL7/U9ddfb8Vs3brVZb7MzEx16NBBjRs3ttoufBTrtm3bdN11110yhx07duj06dOaM2eOevTooQ4dOujbb791iWnSpIlHzlSqSl89lQsAoOrGjRunnJwcTZw4Uf/5z3/0zjvv6KmnntKkSZPUqFEjBQQEaNSoUXr88ce1adMm7d27V7/+9a/VqFGjKv2AAgBwr4SEBLVq1UqzZ8+ucPr3338vSVq+fLnatm2rTz/9VLt27bKG+fPna+nSpS5PXa2pjh076vDhw8rJybHaPv/8cxUWFlrfkS7l2LFj+uKLL/T73/9effv21fXXX6+CgoJyr/Ppp5+qpKTEarvwOxVQExSlUGdER0frjjvu0EMPPaStW7fq008/1f3336+rrrpKd9xxhyRp8uTJev/99/XHP/5RX375pZYuXaoFCxaUuwnfBx98oNmzZ+vLL7/Uiy++qL/97W969NFHL5nDNddco9OnTys1NVUHDhzQ66+/rpdfftklpn379jp+/Ljef/99HT16VCdPnqy9lXCeqvS1ffv22rJli7755hsdPXrULXkAAKrnqquu0rp16/Txxx+rc+fOevjhhzVmzBj9/ve/t2Lmzp2rnj17atCgQerXr5969eql66+/Xk2bNvVi5gAA6cezpv7yl79o7dq1GjJkiDZs2KCDBw9qx44dmjp1qh5++GFJP16698tf/lIxMTEuw69//Wt9//33Wrt27WXn0q9fP91www2677779Mknn+jjjz/WyJEj1adPH5dLDi+mZcuWCgkJ0aJFi/TVV19p48aNmjRpkkvMiBEj1KhRI40ZM0aff/651q1bp+eff/6y8wcoSqFOWbx4sbp27apBgwapZ8+eMsZo3bp11umoN910k/76179q5cqViomJ0R/+8Ac9/fTTGj16tMtyJk+erKysLHXp0kV//OMfNWfOHA0YMOCSr3/jjTdq7ty5evbZZxUTE6Ply5crJSXFJSYuLk4PP/ywhg8frtatW1f6C8rlqkpfn376aR08eFDXXHONWrdu7ZY8AAAXt3nzZs2fP9+lrU+fPvr4449VWlqq3NxczZo1y7rfiCQFBARo+fLlOnHihHJzc/Wb3/xG+/bt009+8hMPZw8AqMgdd9yhzMxM+fv7a8SIEbruuut07733qrCwUM8884yysrL06aef6q677io3b0BAgBISEpSWlnbZedhsNq1evVotW7bUz372M/Xr109XX3213nzzzSovo1GjRlq5cqWysrIUExOjxx57TM8995xLzBVXXKF//OMf+vzzz9WlSxfNmDFDzz777GXnD9gMF4eigWnfvr2SkpKUlJTk7VQAAKjQzp079Z///Ec//elPVVhYqKefflqbN2/WV199pVatWnk7PQAAytm3b5+uu+467d+/nx9RUGV+lw4BAACApz3//PPat2+fmjRpoq5du+rf//43BSkAgE/67rvv9Pe//12BgYEuTwIELoWiFHCe5cuXa+zYsRVOi4yM1N69e6u9zH//+99KTEysdPrx48ervUwAQP3WpUsXZWVleTsNAIAbHT58WB07dqx0+ueff6527dp5MKOaGzNmjLKysrRw4ULZ7XZvp4M6hMv3gPMUFxfrv//9b4XT/P39FRkZWe1llpSU6Jtvvql0Oqe2AgAAAA3P6dOndfDgwUqnt2/f3uV+g0B9RFEKAAAAAAAAHsfT9wAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxFKXQYCxZskQ2m00HDx6UJI0ePVrt27f3aA4HDx6UzWbTkiVLrDan0ymbzVat5Zw8eVJOp1ObN2+u3QQBoI7IzMyU0+nU999/7+1UvMJms8npdNbqMtu3b6/Ro0db45s3b5bNZqv2seall15yOc4BAP7PRx99pF/84hdq166d7Ha7QkND1bNnT02ePLnC+EmTJslms2nQoEGVLjMnJ0fjxo1Thw4d1KxZMwUHBys2NlYPPfSQcnJyKpxnzZo1stlsCgkJUWlpaa30DagJilJosJ588kmtWrXK22nowQcf1IcffliteU6ePKmZM2dSlALQYGVmZmrmzJkNtij14Ycf6sEHH3Tra9x000368MMPddNNN1VrPopSAFCxtWvXKi4uTkVFRZo9e7bWr1+vF154Qb169dKbb75ZLv7UqVNatmyZJOndd9/VN998Uy7myJEjuummm5Senq5JkyZp3bp1evXVV3Xvvfdq+/btOnDgQIW5pKWlSZK+++47rV69uvY6CVSTn7cTALzlmmuu8XYKkqS2bduqbdu23k4DABq8kydPqnnz5t5Oo0p69Ojh9tcIDAz0yOsAQEMxe/ZsRUVF6b333pOf3/99Fb/nnns0e/bscvHvvPOO/t//+38aOHCg1q5dq6VLl+qJJ55wiXnllVd09OhRffzxx4qKirLa77zzTj3xxBM6e/ZsueXm5eVp3bp1uu2225SZmam0tDQNHz68FnsKVB1nSqHBqujyve+//15jxoxRcHCwrrjiCg0cOFAHDhyo0WUS3377rYYNG6aAgAAFBQVp+PDhysvLKxdX0eV7GzduVHx8vEJCQtSsWTO1a9dOd911l06ePKmDBw+qdevWkqSZM2fKZrPJZrO5XHIBAPWZ0+nU448/LkmKioqy9oPnzh5988031bNnT7Vo0UJXXHGFBgwYoJ07d7osY/To0briiiu0e/duJSQkKCAgQH379pX046VxEyZM0OLFi3XttdeqWbNm6tatm7Zt2yZjjJ577jlFRUXpiiuu0G233aavvvqqWvmfu5x848aNeuihhxQSEqLAwECNHDlSJ06cUF5enoYNG6Yrr7xSYWFhmjJlik6dOuWyjAuPS+eWuWnTJv32t79Vq1atFBISoqFDh+rbb791mffUqVOaOnWqHA6Hmjdvrt69e+vjjz8ul2dFl+8dOHBA99xzj8LDw63LTvr27atdu3ZJ+vESwL179yojI8PaLp6+VB4AfNWxY8fUqlUrl4LUOY0alf9qnpaWpiZNmmjx4sWKiIjQ4sWLZYwpt8xGjRqpTZs2Fb5mRctdunSpTp8+rccee0xDhw7V+++/r0OHDtWwV8DloSgF/K+zZ89q8ODBWrFihX73u99p1apV6t69u26//fZqL6ukpET9+vXT+vXrlZKSor/97W9yOBxV+gXi4MGDGjhwoJo0aaJXX31V7777rmbNmqUWLVqorKxMYWFhevfddyVJY8aM0YcffqgPP/xQTz75ZLXzBIC66MEHH9TEiRMlSW+//ba1H7zpppuUnJyse++9Vx07dtRf//pXvf766youLtYtt9yizz//3GU5ZWVlGjJkiG677Ta98847mjlzpjXtn//8p/7yl79o1qxZeuONN1RcXKyBAwdq8uTJ+uCDD7RgwQItWrRIn3/+ue66665yXxKq2o+goCCtXLlSv//977VixQo99NBDGjhwoDp37qy///3vGjVqlObMmaPU1NQqL9Pf318rVqzQ7NmztXnzZt1///0uMQ899JCef/55jRw5Uu+8847uuusuDR06VAUFBZdc/s9//nNlZWVp9uzZSk9P18KFC9WlSxfrMspVq1bp6quvVpcuXazt4guXygOAL+jZs6c++ugjPfLII/roo4/K/eBwviNHjmj9+vW644471Lp1a40aNUpfffWVtmzZUm6ZZ8+e1dChQ/Xee++pqKjoknm8+uqrCgsLU2Jion7961/r7NmzXHYN7zFAA7F48WIjyWRnZxtjjBk1apSJjIy0pq9du9ZIMgsXLnSZLyUlxUgyTz31VJVfa+HChUaSeeedd1zaH3roISPJLF682Gp76qmnzPkfxb///e9Gktm1a1ely/9//+//VTsnAKhPnnvuOZd9ujHGHD582Pj5+ZmJEye6xBYXFxuHw2GGDRtmtY0aNcpIMq+++mq5ZUsyDofDHD9+3GpbvXq1kWRuvPFGc/bsWat9/vz5RpL57LPPqpz7uePRhXneeeedRpKZO3euS/uNN95obrrppnI5nn8MOLfMcePGucTNnj3bSDK5ubnGGGO++OILI8k89thjLnHLly83ksyoUaOstk2bNhlJZtOmTcYYY44ePWokmfnz51+0f506dTJ9+vS5aAwANERHjx41vXv3NpKMJOPv72/i4uJMSkqKKS4udol9+umnjSTz7rvvGmOMOXDggLHZbOaBBx5wiTt79qwZO3asadSokZFkbDabuf76681jjz3mcow8Z8uWLUaSmTZtmjV/VFSUiYyMdDm+AZ7CmVLA/8rIyJAkDRs2zKX93nvvrfayNm3apICAAA0ZMsSlfcSIEZec98Ybb1STJk30m9/8RkuXLq305oQAAFfvvfeeTp8+rZEjR+r06dPW0LRpU/Xp06fCh0PcddddFS7r1ltvVYsWLazx66+/XpKUmJjocsn1ufaaXPZw4ZOUzi1r4MCB5dqruvwLjzs33HCDS36bNm2SJN13330uccOGDavwcpLzBQcH65prrtFzzz2nuXPnaufOnRXeqwQAULGQkBD9+9//1vbt2zVr1izdcccd+vLLLzV9+nTFxsbq6NGjkiRjjHXJXv/+/SX9eLl6fHy83nrrLZezoWw2m15++WUdOHBAL730kn71q1/p1KlTmjdvnjp16mR9xznn3A3Of/3rX1vzjx49WocOHdL777/vidUAuKAoBfyvY8eOyc/PT8HBwS7toaGhNVpWRfM5HI5LznvNNddow4YNatOmjcaPH69rrrlG11xzjV544YVq5wEADcl///tfSdLNN98sf39/l+HNN9+0/tk/p3nz5goMDKxwWRceC5o0aXLR9h9++KHa+VbnNaq6/JCQEJdxu90u6cfLyqUfj09S+eORn59fuXkvZLPZ9P7772vAgAGaPXu2brrpJrVu3VqPPPKIiouLq5QfAEDq1q2bfve73+lvf/ubvv32Wz322GM6ePCgdbPzjRs3Kjs7W3fffbeKior0/fff6/vvv9ewYcN08uRJvfHGG+WWGRkZqd/+9rdKS0vT/v379eabb+qHH36w7sEoScXFxfrb3/6mn/70p2rdurW13F/84hey2WxWwQrwJJ6+B/yvkJAQnT59Wt99953LF4KKbk5elWVVdNPYqi7rlltu0S233KIzZ85ox44dSk1NVVJSkkJDQ3XPPfdUOx8AaAhatWolSfr73/+uyMjIS8Zf+JCJhuBc4SkvL09XXXWV1X769GmrYHUxkZGR1peWL7/8Un/961/ldDpVVlaml19+2T1JA0A95u/vr6eeekrz5s3Tnj17JP3f2Uxz587V3Llzy82TlpamsWPHXnS5w4YNU0pKirVMSXrjjTd08uRJffzxx2rZsmW5eVatWqWCgoIKpwHuwplSwP/q06ePpB+f2nS+lStXVntZt956q4qLi7VmzRqX9hUrVlRrOY0bN1b37t314osvSpI++eQTSeV/+QaAhqai/eCAAQPk5+enr7/+Wt26datwaOji4+MlScuXL3dp/+tf/6rTp09Xa1kdOnTQ73//e8XGxlrHJ+nHbcPxCQDKy83NrbD9iy++kCSFh4eroKBAq1atUq9evbRp06Zyw3333aft27dbxabKlnn8+HHl5OQoPDzcaktLS1NAQIDef//9cst97rnnVFpaWu74ALgbZ0oB/+v2229Xr169NHnyZBUVFalr16768MMP9dprr0mq+HGqlRk5cqTmzZunkSNH6k9/+pOio6O1bt06vffee5ec9+WXX9bGjRs1cOBAtWvXTj/88INeffVVSVK/fv0kSQEBAYqMjNQ777yjvn37Kjg4WK1ateKx2wAajNjYWEnSCy+8oFGjRsnf31/XXnutnn76ac2YMUMHDhzQ7bffrpYtW+q///2vPv74Y7Vo0cLlCXsN0fXXX6/7779f8+fPl7+/v/r166c9e/bo+eefr/RSxnM+++wzTZgwQXfffbeio6PVpEkTbdy4UZ999pmmTZtmxcXGxmrlypV68803dfXVV6tp06bW9gKAhmzAgAFq27atBg8erOuuu05nz57Vrl27NGfOHF1xxRV69NFHtXz5cv3www965JFHrB8SzhcSEqLly5crLS1N8+bN05/+9Cd98MEHGj58uG688UY1a9ZM2dnZWrBggY4dO6bnnntOkrRnzx59/PHH+u1vf6vbbrut3HJ79eqlOXPmKC0tTRMmTHD3qgAsFKWA/9WoUSP94x//0OTJkzVr1iyVlZWpV69eWrZsmXr06KErr7yyystq3ry5Nm7cqEcffVTTpk2TzWZTQkKCVq5cqbi4uIvOe+ONN2r9+vV66qmnlJeXpyuuuEIxMTFas2aNEhISrLi0tDQ9/vjjGjJkiEpLSzVq1Cge5QqgwYiPj9f06dO1dOlSvfLKKzp79qw2bdqk6dOnq2PHjnrhhRf0xhtvqLS0VA6HQzfffLMefvhhb6ftE9LS0hQaGqolS5boz3/+s2688Ua99dZbl7w83OFw6JprrtFLL72knJwc2Ww2XX311ZozZ44mTpxoxc2cOVO5ubl66KGHVFxcrMjISB08eNDNvQIA3/f73/9e77zzjubNm6fc3FyVlpYqLCxM/fr10/Tp03X99ddr5MiRatOmje68884KlxEbG6sePXpo2bJlevbZZ/XAAw9I+vHqjueee06FhYUKDg5W165dtW7dOiUmJkr6v0sCK7vsz9/fX6NHj9asWbP0ySef6Kabbqr9FQBUwGaMMd5OAvBlK1as0H333acPPvjgkgUlAAAAAABQNRSlgPO88cYb+uabbxQbG6tGjRpp27Zteu6559SlS5dyj1MFAAAAAAA1x+V7wHkCAgK0cuVKPfPMMzpx4oTCwsI0evRoPfPMM1bMpW4E26hRo2rdfwoAUD8YY3TmzJmLxjRu3LhBPvUPAACgIpwpBVTDwYMHFRUVddGYp556Sk6n0zMJAQB8xpIlS/SrX/3qojGbNm2q8Ma1AAAADRFFKaAaysrK9Nlnn100Jjw83OXRqwCAhuHYsWPKzs6+aMy1116rgIAAD2UEAADg2yhKAQAAAAAAwOO48Q0AAAAAAAA8rk7e6Pzs2bP69ttvFRAQwM1CAaCajDEqLi5WeHh4g7opP8cOAKi5hnjs4LgBADVX1eNGnSxKffvtt4qIiPB2GgBQp+Xk5Kht27beTsNjOHYAwOVrSMcOjhsAcPkuddyok0WpczcIzcnJUWBgoJezAYC6paioSBEREQ3uZsscOwCg5hrisYPjBgDUXFWPG3WyKHXu9NnAwEAOEABQQw3tUgSOHQBw+RrSsYPjBgBcvksdNxrGBeEAAAAAAADwKRSlAAAAAAAA4HEUpQAAbpOSkqKbb75ZAQEBatOmje68807t27fPJcYYI6fTqfDwcDVr1kzx8fHau3evS0xpaakmTpyoVq1aqUWLFhoyZIiOHDniya4AAAAAqGUUpQAAbpORkaHx48dr27ZtSk9P1+nTp5WQkKATJ05YMbNnz9bcuXO1YMECbd++XQ6HQ/3791dxcbEVk5SUpFWrVmnlypXaunWrjh8/rkGDBunMmTPe6BYAwM22bNmiwYMHKzw8XDabTatXr3aZzg8aAFA/UJQCALjNu+++q9GjR6tTp07q3LmzFi9erMOHDysrK0vSj18q5s+frxkzZmjo0KGKiYnR0qVLdfLkSa1YsUKSVFhYqLS0NM2ZM0f9+vVTly5dtGzZMu3evVsbNmzwZvcAAG5y4sQJde7cWQsWLKhwOj9oAED9QFEKAOAxhYWFkqTg4GBJUnZ2tvLy8pSQkGDF2O129enTR5mZmZKkrKwsnTp1yiUmPDxcMTExVkxFSktLVVRU5DIAAOqGxMREPfPMMxo6dGi5afygAQD1B0UpAIBHGGM0adIk9e7dWzExMZKkvLw8SVJoaKhLbGhoqDUtLy9PTZo0UcuWLSuNqUhKSoqCgoKsISIioja7AwDwEnf9oMGPGQDgeX7eTgBA7Wo/ba3XXvvgrIFee234vgkTJuizzz7T1q1by02z2Wwu48aYcm0XulTM9OnTNWnSJGu8qKiIwhSqxFv7UfahQNVc7AeNQ4cOWTHV/UEjJSVFM2fOdEPGqO/4/xuoOc6UAgC43cSJE7VmzRpt2rRJbdu2tdodDocklfuCkJ+fb33ZcDgcKisrU0FBQaUxFbHb7QoMDHQZAAD1R23/oDF9+nQVFhZaQ05OTq3lCgCoGGdKAQDcxhijiRMnatWqVdq8ebOioqJcpkdFRcnhcCg9PV1dunSRJJWVlSkjI0PPPvusJKlr167y9/dXenq6hg0bJknKzc3Vnj17NHv2bM92CB7jzV+dAfi283/QCAsLs9or+0Hj/LOl8vPzFRcXV+Fy7Xa77Ha7GzMHAFyIM6UAAG4zfvx4LVu2TCtWrFBAQIDy8vKUl5enkpISST/+yp2UlKTk5GStWrVKe/bs0ejRo9W8eXONGDFCkhQUFKQxY8Zo8uTJev/997Vz507df//9io2NVb9+/bzZPQCAF5z/g8Y5537QOFdwOv8HjXPO/aBRWVEKAOB5nCkFAHCbhQsXSpLi4+Nd2hcvXqzRo0dLkqZOnaqSkhKNGzdOBQUF6t69u9avX6+AgAArft68efLz89OwYcNUUlKivn37asmSJWrcuLGnugIA8KDjx4/rq6++ssazs7O1a9cuBQcHq127dtYPGtHR0YqOjlZycnKlP2iEhIQoODhYU6ZM4QcNAPAxFKUAAG5jjLlkjM1mk9PplNPprDSmadOmSk1NVWpqai1mBwDwVTt27NCtt95qjZ97cMWoUaO0ZMkSftAAgHqCohQAAAAAnxIfH3/RHzb4QQMX4l6EQN3EPaUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBx1SpKpaSk6Oabb1ZAQIDatGmjO++8U/v27XOJMcbI6XQqPDxczZo1U3x8vPbu3esSU1paqokTJ6pVq1Zq0aKFhgwZoiNHjlx+bwAAAAAAAFAnVKsolZGRofHjx2vbtm1KT0/X6dOnlZCQoBMnTlgxs2fP1ty5c7VgwQJt375dDodD/fv3V3FxsRWTlJSkVatWaeXKldq6dauOHz+uQYMG6cyZM7XXMwAAAAAAAPgsv+oEv/vuuy7jixcvVps2bZSVlaWf/exnMsZo/vz5mjFjhoYOHSpJWrp0qUJDQ7VixQqNHTtWhYWFSktL0+uvv65+/fpJkpYtW6aIiAht2LBBAwYMqKWuAQAAAAAAwFdd1j2lCgsLJUnBwcGSpOzsbOXl5SkhIcGKsdvt6tOnjzIzMyVJWVlZOnXqlEtMeHi4YmJirJgLlZaWqqioyGUAAAAAAABA3VXjopQxRpMmTVLv3r0VExMjScrLy5MkhYaGusSGhoZa0/Ly8tSkSRO1bNmy0pgLpaSkKCgoyBoiIiJqmjYAAAAAAAB8QI2LUhMmTNBnn32mN954o9w0m83mMm6MKdd2oYvFTJ8+XYWFhdaQk5NT07QBAAAAAADgA2pUlJo4caLWrFmjTZs2qW3btla7w+GQpHJnPOXn51tnTzkcDpWVlamgoKDSmAvZ7XYFBga6DAAAAAAAAKi7qlWUMsZowoQJevvtt7Vx40ZFRUW5TI+KipLD4VB6errVVlZWpoyMDMXFxUmSunbtKn9/f5eY3Nxc7dmzx4oBAAAAAABA/Vatp++NHz9eK1as0DvvvKOAgADrjKigoCA1a9ZMNptNSUlJSk5OVnR0tKKjo5WcnKzmzZtrxIgRVuyYMWM0efJkhYSEKDg4WFOmTFFsbKz1ND4AAAAAAADUb9UqSi1cuFCSFB8f79K+ePFijR49WpI0depUlZSUaNy4cSooKFD37t21fv16BQQEWPHz5s2Tn5+fhg0bppKSEvXt21dLlixR48aNL683AAAAAAAAqBOqVZQyxlwyxmazyel0yul0VhrTtGlTpaamKjU1tTovDwAAAAAAgHqixk/fAwAAAAAAAGqKohQAAAAAAAA8jqIUAAAAAAAAPK5a95QCAAAAAKAy7aet9XYKAOoQzpQCAAAAAACAx1GUAgAAAAAAgMdRlAIAAAAAAIDHcU8pAAAAAHXK6dOn5XQ6tXz5cuXl5SksLEyjR4/W73//ezVq9OPv7sYYzZw5U4sWLVJBQYG6d++uF198UZ06dfJy9kDt8dY9vA7OGuiV10X9w5lSAAAAAOqUZ599Vi+//LIWLFigL774QrNnz9Zzzz2n1NRUK2b27NmaO3euFixYoO3bt8vhcKh///4qLi72YuYAgPNRlAIAAABQp3z44Ye64447NHDgQLVv316//OUvlZCQoB07dkj68Syp+fPna8aMGRo6dKhiYmK0dOlSnTx5UitWrPBy9gCAc7h8D0Ct8eYjgDmFGACAhqN37956+eWX9eWXX6pDhw769NNPtXXrVs2fP1+SlJ2drby8PCUkJFjz2O129enTR5mZmRo7dqyXMgcAnI+iFAAAAIA65Xe/+50KCwt13XXXqXHjxjpz5oz+9Kc/6d5775Uk5eXlSZJCQ0Nd5gsNDdWhQ4cqXGZpaalKS0ut8aKiIjdlDwA4h8v3AAAAANQpb775ppYtW6YVK1bok08+0dKlS/X8889r6dKlLnE2m81l3BhTru2clJQUBQUFWUNERITb8gcA/IiiFAAAAIA65fHHH9e0adN0zz33KDY2Vg888IAee+wxpaSkSJIcDoek/ztj6pz8/PxyZ0+dM336dBUWFlpDTk6OezsBAKAoBQAAAKBuOXnypBo1cv0q07hxY509e1aSFBUVJYfDofT0dGt6WVmZMjIyFBcXV+Ey7Xa7AgMDXQYAgHtxTykAAAAAdcrgwYP1pz/9Se3atVOnTp20c+dOzZ07V7/+9a8l/XjZXlJSkpKTkxUdHa3o6GglJyerefPmGjFihJezBwCcQ1EKAAAAQJ2SmpqqJ598UuPGjVN+fr7Cw8M1duxY/eEPf7Bipk6dqpKSEo0bN04FBQXq3r271q9fr4CAAC9mDgA4H5fvAQDcasuWLRo8eLDCw8Nls9m0evVql+mjR4+WzWZzGXr06OESU1paqokTJ6pVq1Zq0aKFhgwZoiNHjniwFwAAXxIQEKD58+fr0KFDKikp0ddff61nnnlGTZo0sWJsNpucTqdyc3P1ww8/KCMjQzExMV7MGgBwIYpSAAC3OnHihDp37qwFCxZUGnP77bcrNzfXGtatW+cyPSkpSatWrdLKlSu1detWHT9+XIMGDdKZM2fcnT4AAAAAN+HyPQCAWyUmJioxMfGiMXa73XpS0oUKCwuVlpam119/Xf369ZMkLVu2TBEREdqwYYMGDBhQ6zkDAAAAcD/OlAIAeN3mzZvVpk0bdejQQQ899JDy8/OtaVlZWTp16pQSEhKstvDwcMXExCgzM7PSZZaWlqqoqMhlAAAAAOA7KEoBALwqMTFRy5cv18aNGzVnzhxt375dt912m0pLSyVJeXl5atKkiVq2bOkyX2hoqPLy8ipdbkpKioKCgqwhIiLCrf0AAAAAUD1cvgcA8Krhw4dbf8fExKhbt26KjIzU2rVrNXTo0ErnM8bIZrNVOn369OmaNGmSNV5UVERhCgAAAPAhFKUAAD4lLCxMkZGR2r9/vyTJ4XCorKxMBQUFLmdL5efnKy4urtLl2O122e12t+dbn7WfttbbKQAAAKAe4/I9AIBPOXbsmHJychQWFiZJ6tq1q/z9/ZWenm7F5Obmas+ePRctSgEAAADwbZwpBQBwq+PHj+urr76yxrOzs7Vr1y4FBwcrODhYTqdTd911l8LCwnTw4EE98cQTatWqlX7xi19IkoKCgjRmzBhNnjxZISEhCg4O1pQpUxQbG2s9jQ8AAABA3UNRCgDgVjt27NCtt95qjZ+7z9OoUaO0cOFC7d69W6+99pq+//57hYWF6dZbb9Wbb76pgIAAa5558+bJz89Pw4YNU0lJifr27aslS5aocePGHu8PAAAAgNpBUQoA4Fbx8fEyxlQ6/b333rvkMpo2barU1FSlpqbWZmoAAAAAvIh7SgEAAAAAAMDjKEoBAAAAAADA4yhKAQAAAAAAwOMoSgEAAAAAAMDjKEoBAAAAAADA4yhKAQAAAAAAwOMoSgEAAAAAAMDjql2U2rJliwYPHqzw8HDZbDatXr3aZfro0aNls9lchh49erjElJaWauLEiWrVqpVatGihIUOG6MiRI5fVEQAAAAAAANQd1S5KnThxQp07d9aCBQsqjbn99tuVm5trDevWrXOZnpSUpFWrVmnlypXaunWrjh8/rkGDBunMmTPV7wEAAAAAAADqHL/qzpCYmKjExMSLxtjtdjkcjgqnFRYWKi0tTa+//rr69esnSVq2bJkiIiK0YcMGDRgwoLopAQAAAAAAoI5xyz2lNm/erDZt2qhDhw566KGHlJ+fb03LysrSqVOnlJCQYLWFh4crJiZGmZmZFS6vtLRURUVFLgMAAAAAAADqrlovSiUmJmr58uXauHGj5syZo+3bt+u2225TaWmpJCkvL09NmjRRy5YtXeYLDQ1VXl5ehctMSUlRUFCQNURERNR22gAAAAAAAPCgal++dynDhw+3/o6JiVG3bt0UGRmptWvXaujQoZXOZ4yRzWarcNr06dM1adIka7yoqIjCFAAAAAAAQB3mlsv3zhcWFqbIyEjt379fkuRwOFRWVqaCggKXuPz8fIWGhla4DLvdrsDAQJcBAAAAAAAAdZfbi1LHjh1TTk6OwsLCJEldu3aVv7+/0tPTrZjc3Fzt2bNHcXFx7k4HAAAAAAAAPqDal+8dP35cX331lTWenZ2tXbt2KTg4WMHBwXI6nbrrrrsUFhamgwcP6oknnlCrVq30i1/8QpIUFBSkMWPGaPLkyQoJCVFwcLCmTJmi2NhY62l8AAAAAAAAqN+qXZTasWOHbr31Vmv83L2eRo0apYULF2r37t167bXX9P333yssLEy33nqr3nzzTQUEBFjzzJs3T35+fho2bJhKSkrUt29fLVmyRI0bN66FLgEAAAAAAMDXVbsoFR8fL2NMpdPfe++9Sy6jadOmSk1NVWpqanVfHgAAAAAAAPWA2+8pBQAAAAAAAFyIohQAAAAAAAA8jqIUAAAAAAAAPI6iFAAAAIA655tvvtH999+vkJAQNW/eXDfeeKOysrKs6cYYOZ1OhYeHq1mzZoqPj9fevXu9mDEA4EIUpQAAAADUKQUFBerVq5f8/f31r3/9S59//rnmzJmjK6+80oqZPXu25s6dqwULFmj79u1yOBzq37+/iouLvZc4AMBFtZ++BwAAAADe9OyzzyoiIkKLFy+22tq3b2/9bYzR/PnzNWPGDA0dOlSStHTpUoWGhmrFihUaO3asp1MGAFSAohTgJu2nrfV2CgAAAPXSmjVrNGDAAN19993KyMjQVVddpXHjxumhhx6SJGVnZysvL08JCQnWPHa7XX369FFmZmaFRanS0lKVlpZa40VFRe7vCAA0cFy+BwAAAKBOOXDggBYuXKjo6Gi99957evjhh/XII4/otddekyTl5eVJkkJDQ13mCw0NtaZdKCUlRUFBQdYQERHh3k4AAChKAQAAAKhbzp49q5tuuknJycnq0qWLxo4dq4ceekgLFy50ibPZbC7jxphybedMnz5dhYWF1pCTk+O2/AEAP6IoBQAAAKBOCQsLU8eOHV3arr/+eh0+fFiS5HA4JKncWVH5+fnlzp46x263KzAw0GUAALgXRSkAAAAAdUqvXr20b98+l7Yvv/xSkZGRkqSoqCg5HA6lp6db08vKypSRkaG4uDiP5goAqBw3OgdQL3jrxvIHZw30yusCANCQPfbYY4qLi1NycrKGDRumjz/+WIsWLdKiRYsk/XjZXlJSkpKTkxUdHa3o6GglJyerefPmGjFihJezBwCcQ1EKAAAAQJ1y8803a9WqVZo+fbqefvppRUVFaf78+brvvvusmKlTp6qkpETjxo1TQUGBunfvrvXr1ysgIMCLmQMAzkdRCgAAAECdM2jQIA0aNKjS6TabTU6nU06n03NJAQ2Et65SkLhSob7hnlIAAAAAAADwOIpSAAAAAAAA8DiKUgAAAAAAAPA4ilIAAAAAAADwOIpSAAAAAAAA8DiKUgAAAAAAAPA4ilIAAAAAAADwOD9vJwAAAACp/bS1Xnvtg7MGeu21AQBAw8WZUgAAAAAAAPA4ilIAALfasmWLBg8erPDwcNlsNq1evdplujFGTqdT4eHhatasmeLj47V3716XmNLSUk2cOFGtWrVSixYtNGTIEB05csSDvQAAAABQ27h8DwDgVidOnFDnzp31q1/9SnfddVe56bNnz9bcuXO1ZMkSdejQQc8884z69++vffv2KSAgQJKUlJSkf/zjH1q5cqVCQkI0efJkDRo0SFlZWWrcuLGnuwQAgE/z5uXAAFAdFKUAAG6VmJioxMTECqcZYzR//nzNmDFDQ4cOlSQtXbpUoaGhWrFihcaOHavCwkKlpaXp9ddfV79+/SRJy5YtU0REhDZs2KABAwZ4rC8AAAAAag+X7wEAvCY7O1t5eXlKSEiw2ux2u/r06aPMzExJUlZWlk6dOuUSEx4erpiYGCumIqWlpSoqKnIZAAAAAPgOilIAAK/Jy8uTJIWGhrq0h4aGWtPy8vLUpEkTtWzZstKYiqSkpCgoKMgaIiIiajl7AAAAAJeDohQAwOtsNpvLuDGmXNuFLhUzffp0FRYWWkNOTk6t5AoAAACgdlCUAgB4jcPhkKRyZzzl5+dbZ085HA6VlZWpoKCg0piK2O12BQYGugwAAAAAfAdFKQCA10RFRcnhcCg9Pd1qKysrU0ZGhuLi4iRJXbt2lb+/v0tMbm6u9uzZY8UAAAAAqHt4+h4AwK2OHz+ur776yhrPzs7Wrl27FBwcrHbt2ikpKUnJycmKjo5WdHS0kpOT1bx5c40YMUKSFBQUpDFjxmjy5MkKCQlRcHCwpkyZotjYWOtpfAAAAADqHopSAAC32rFjh2699VZrfNKkSZKkUaNGacmSJZo6dapKSko0btw4FRQUqHv37lq/fr0CAgKseebNmyc/Pz8NGzZMJSUl6tu3r5YsWaLGjRt7vD8AAAAAagdFKQCAW8XHx8sYU+l0m80mp9Mpp9NZaUzTpk2Vmpqq1NRUN2QIAAAAwBu4pxQAAAAAAAA8rtpFqS1btmjw4MEKDw+XzWbT6tWrXaYbY+R0OhUeHq5mzZopPj5ee/fudYkpLS3VxIkT1apVK7Vo0UJDhgzRkSNHLqsjAAAAAAAAqDuqXZQ6ceKEOnfurAULFlQ4ffbs2Zo7d64WLFig7du3y+FwqH///iouLrZikpKStGrVKq1cuVJbt27V8ePHNWjQIJ05c6bmPQEAAAAAAECdUe17SiUmJioxMbHCacYYzZ8/XzNmzNDQoUMlSUuXLlVoaKhWrFihsWPHqrCwUGlpaXr99detpyYtW7ZMERER2rBhgwYMGHAZ3QEAAAAAAEBdUKv3lMrOzlZeXp4SEhKsNrvdrj59+igzM1OSlJWVpVOnTrnEhIeHKyYmxoq5UGlpqYqKilwGAAAAAAAA1F21WpTKy8uTJIWGhrq0h4aGWtPy8vLUpEkTtWzZstKYC6WkpCgoKMgaIiIiajNtAAAAAAAAeJhbnr5ns9lcxo0x5doudLGY6dOnq7Cw0BpycnJqLVcAAAAAAAB4Xq0WpRwOhySVO+MpPz/fOnvK4XCorKxMBQUFlcZcyG63KzAw0GUAAAAAAABA3VWrRamoqCg5HA6lp6dbbWVlZcrIyFBcXJwkqWvXrvL393eJyc3N1Z49e6wYAAAAAAAA1G/Vfvre8ePH9dVXX1nj2dnZ2rVrl4KDg9WuXTslJSUpOTlZ0dHRio6OVnJyspo3b64RI0ZIkoKCgjRmzBhNnjxZISEhCg4O1pQpUxQbG2s9jQ8AAAAAAAD1W7XPlNqxY4e6dOmiLl26SJImTZqkLl266A9/+IMkaerUqUpKStK4cePUrVs3ffPNN1q/fr0CAgKsZcybN0933nmnhg0bpl69eql58+b6xz/+ocaNG9dStwAAAAA0FCkpKbLZbEpKSrLajDFyOp0KDw9Xs2bNFB8fr71793ovSQBAOdU+Uyo+Pl7GmEqn22w2OZ1OOZ3OSmOaNm2q1NRUpaamVvflAQAAAMCyfft2LVq0SDfccINL++zZszV37lwtWbJEHTp00DPPPKP+/ftr3759Lj+YAwC8xy1P3wMAAAAAdzt+/Ljuu+8+vfLKK2rZsqXVbozR/PnzNWPGDA0dOlQxMTFaunSpTp48qRUrVngxYwDA+ShKAQAAAKiTxo8fr4EDB5a7N212drby8vKUkJBgtdntdvXp00eZmZkVLqu0tFRFRUUuAwDAvap9+R4AAAAAeNvKlSv1ySefaPv27eWm5eXlSZJCQ0Nd2kNDQ3Xo0KEKl5eSkqKZM2fWfqIAgEpxphQAAACAOiUnJ0ePPvqoli1bpqZNm1YaZ7PZXMaNMeXazpk+fboKCwutIScnp1ZzBgCUx5lSAAAAAOqUrKws5efnq2vXrlbbmTNntGXLFi1YsED79u2T9OMZU2FhYVZMfn5+ubOnzrHb7bLb7e5NHADggjOlAAAAANQpffv21e7du7Vr1y5r6Natm+677z7t2rVLV199tRwOh9LT0615ysrKlJGRobi4OC9mDgA4H2dKAQAAAKhTAgICFBMT49LWokULhYSEWO1JSUlKTk5WdHS0oqOjlZycrObNm2vEiBHeSBkAUAGKUgAAAADqnalTp6qkpETjxo1TQUGBunfvrvXr1ysgIMDbqQEA/hdFKQAAAAB13ubNm13GbTabnE6nnE6nV/IBAFwa95QCAAAAAACAx3GmFOq19tPWejsFAAAAAABQAc6UAgAAAAAAgMdRlAIAAAAAAIDHUZQCAAAAAACAx1GUAgAAAAAAgMdRlAIAAAAAAIDHUZQCAAAAAACAx1GUAgAAAAAAgMdRlAIAAAAAAIDH+Xk7AQAAAAAAAF/Wftpar732wVkDvfba7saZUgAAAAAAAPA4ilIAAAAAAADwOIpSAAAAAAAA8DiKUgAAAAAAAPA4ilIAAAAAAADwOIpSAAAAAAAA8DiKUgAAAAAAAPA4ilIAAAAAAADwOIpSAAAAAAAA8DiKUgAAr3I6nbLZbC6Dw+Gwphtj5HQ6FR4ermbNmik+Pl579+71YsYAAAAAagNFKQCA13Xq1Em5ubnWsHv3bmva7NmzNXfuXC1YsEDbt2+Xw+FQ//79VVxc7MWMAQAAAFwuilIAAK/z8/OTw+GwhtatW0v68Syp+fPna8aMGRo6dKhiYmK0dOlSnTx5UitWrPBy1gAAAAAuB0UpAIDX7d+/X+Hh4YqKitI999yjAwcOSJKys7OVl5enhIQEK9Zut6tPnz7KzMy86DJLS0tVVFTkMgAAAADwHRSlAABe1b17d7322mt677339MorrygvL09xcXE6duyY8vLyJEmhoaEu84SGhlrTKpOSkqKgoCBriIiIcFsfAAAAAFQfRSkAgFclJibqrrvuUmxsrPr166e1a9dKkpYuXWrF2Gw2l3mMMeXaLjR9+nQVFhZaQ05OTu0nDwAAAKDGar0oxVOUAACXo0WLFoqNjdX+/fut48eFZ0Xl5+eXO3vqQna7XYGBgS4DAAAAAN/hljOleIoSAKCmSktL9cUXXygsLExRUVFyOBxKT0+3ppeVlSkjI0NxcXFezBIAAADA5fJzy0L/9ylKF7rwKUrSj5dnhIaGasWKFRo7dqw70gEA+LApU6Zo8ODBateunfLz8/XMM8+oqKhIo0aNks1mU1JSkpKTkxUdHa3o6GglJyerefPmGjFihLdTBwAAAHAZ3FKUOvcUJbvdru7duys5OVlXX331JZ+iVFlRqrS0VKWlpdY4T1CqW9pPW+vtFAD4sCNHjujee+/V0aNH1bp1a/Xo0UPbtm1TZGSkJGnq1KkqKSnRuHHjVFBQoO7du2v9+vUKCAjwcuYAAAAALketX77njqco8QQlAKi/Vq5cqW+//VZlZWX65ptv9NZbb6ljx47WdJvNJqfTqdzcXP3www/KyMhQTEyMFzMGAHhbSkqKbr75ZgUEBKhNmza68847tW/fPpcY7mULAL6v1otS7niKEk9QAgAAAHBORkaGxo8fr23btik9PV2nT59WQkKCTpw4YcVwL1sA8H1uuXzvfOc/RenOO++U9ONTlMLCwqyYSz1FyW63y263uztVAAAAAHXAu+++6zK+ePFitWnTRllZWfrZz37GvWwBoI5we1Hq3FOUbrnlFpenKHXp0kXS/z1F6dlnn3V3KgAA1Encmw8ALq6wsFCSFBwcLEk1vpctAMCzar0oxVOUAAAAAHiKMUaTJk1S7969rXsOXuxetocOHapwOTxcCQA8r9aLUjxFCQAAAICnTJgwQZ999pm2bt1ablp17mWbkpKimTNn1mpunOkKABdX60WplStXXnT6uacoOZ3O2n5pAAAAAA3IxIkTtWbNGm3ZskVt27a12h0Oh6Tq3ct2+vTpmjRpkjVeVFTEU78BwM1q/el7AAAAAOBOxhhNmDBBb7/9tjZu3KioqCiX6effy/acc/eyjYuLq3CZdrtdgYGBLgMAwL3cfqNzAAAAAKhN48eP14oVK/TOO+8oICDAuodUUFCQmjVrxr1sAaCOoCgFAAAAoE5ZuHChJCk+Pt6lffHixRo9erQk7mULAHUBRSkAAAAAdYox5pIx3MsWAHwfRakGhKd/AAAAAAAAX8GNzgEAAAAAAOBxnCkFAAAAAADgo7x51dPBWQPdunzOlAIAAAAAAIDHcaYUAAAAAACoE7hXcv3CmVIAAAAAAADwOIpSAAAAAAAA8DiKUgAAAAAAAPA47ikFAJehPj8JAwAAAADciTOlAAAAAAAA4HEUpQAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxFKUAAAAAAADgcRSlAAAAAAAA4HEUpQAAAAAAAOBxft5OAAAAAN7Vftpar7zuwVkDvfK6AADAN3CmFAAAAAAAADyOohQAAAAAAAA8jsv3AAAA4BXeumxQ4tJBAAB8AWdKAQAAAAAAwOMoSgEAAAAAAMDjKEoBAAAAAADA4yhKAQAAAAAAwOO40bmHefOGngAAAAAAAL6CM6UAAAAAAADgcZwpBQAAgAbHm2evH5w10GuvDQCAL+FMKQAAAAAAAHgcZ0oBAFAF3BMQAAAAqF0UpQAAAAAP8laRm8sGAQC+xqtFqZdeeknPPfeccnNz1alTJ82fP1+33HKLR16bX7wBoO7x5nEDAFA3cewAAN/ltXtKvfnmm0pKStKMGTO0c+dO3XLLLUpMTNThw4e9lRIAwIdx3AAAVBfHDgDwbV4rSs2dO1djxozRgw8+qOuvv17z589XRESEFi5c6K2UAAA+jOMGAKC6OHYAgG/zyuV7ZWVlysrK0rRp01zaExISlJmZWS6+tLRUpaWl1nhhYaEkqaioqMY5nC09WeN5AcAX1HQfeG4+Y0xtpuNW1T1uSLV/7OC4AaCuu5z/nRvCsYPvHABQnru/c3ilKHX06FGdOXNGoaGhLu2hoaHKy8srF5+SkqKZM2eWa4+IiHBbjgDg64LmX978xcXFCgoKqpVc3K26xw2JYwcAXOhyjxtS/T52cNwAgPLc/Z3Dqzc6t9lsLuPGmHJtkjR9+nRNmjTJGj979qy+++47hYSEVBhf24qKihQREaGcnBwFBga6/fV8TUPuf0Puu9Sw+1+f+26MUXFxscLDw72dSrVV9bgh1c6xo66+D8jb8+pq7uTteXU194Zw7PD2d47aUFffX7WN9cA6OIf14L11UNXjhleKUq1atVLjxo3L/UKRn59f7pcMSbLb7bLb7S5tV155pTtTrFBgYGCDfSNLDbv/DbnvUsPuf33te135lfuc6h43pNo9dtTV9wF5e15dzZ28Pa8u5l7fjx2+8p2jNtTF95c7sB5YB+ewHryzDqpy3PDKjc6bNGmirl27Kj093aU9PT1dcXFx3kgJAODDOG4AAKqLYwcA+D6vXb43adIkPfDAA+rWrZt69uypRYsW6fDhw3r44Ye9lRIAwIdx3AAAVBfHDgDwbV4rSg0fPlzHjh3T008/rdzcXMXExGjdunWKjIz0VkqVstvteuqpp8qdzttQNOT+N+S+Sw27/w25777KG8eNuvo+IG/Pq6u5k7fn1eXc66K69J2jNvD++hHrgXVwDuvB99eBzdSl57oCAAAAAACgXvDKPaUAAAAAAADQsFGUAgAAAAAAgMdRlAIAAAAAAIDHUZQCAAAAAACAx1GUkrRlyxYNHjxY4eHhstlsWr169UXjN2/eLJvNVm74z3/+45mEa1FKSopuvvlmBQQEqE2bNrrzzju1b9++S86XkZGhrl27qmnTprr66qv18ssveyDb2lWTvtenbb9w4ULdcMMNCgwMVGBgoHr27Kl//etfF52nPmx3qfp9r0/bHZfmdDrLbWuHw+HttCp0qeOXMUZOp1Ph4eFq1qyZ4uPjtXfvXu8ke55L5T169Ohy26BHjx7eSfY8VTlu+OI6r0revrrOL7W/9sX1LV06b19d3/BNtbXvKS0t1cSJE9WqVSu1aNFCQ4YM0ZEjRzzZlctSG/uDur4OLpSSkiKbzaakpCSrrSGsh0v9r9YQ1oEkffPNN7r//vsVEhKi5s2b68Ybb1RWVpY1va6sB4pSkk6cOKHOnTtrwYIF1Zpv3759ys3NtYbo6Gg3Zeg+GRkZGj9+vLZt26b09HSdPn1aCQkJOnHiRKXzZGdn6+c//7luueUW7dy5U0888YQeeeQRvfXWWx7M/PLVpO/n1Idt37ZtW82aNUs7duzQjh07dNttt+mOO+6o9J/5+rLdper3/Zz6sN1RNZ06dXLZ1rt37/Z2ShW61PFr9uzZmjt3rhYsWKDt27fL4XCof//+Ki4u9nCmrqpy3L399ttdtsG6des8mGHFqnLc8MV1XtXjnS+u80vtr31xfVclb8k31zd8U23te5KSkrRq1SqtXLlSW7du1fHjxzVo0CCdOXPGG92qttrYH9T1dXC+7du3a9GiRbrhhhtc2hvKerjY/2oNYR0UFBSoV69e8vf317/+9S99/vnnmjNnjq688korps6sBwMXksyqVasuGrNp0yYjyRQUFHgkJ0/Kz883kkxGRkalMVOnTjXXXXedS9vYsWNNjx493J2eW1Wl7/V52xtjTMuWLc1f/vKXCqfV1+1+zsX6Xt+3O1w99dRTpnPnzt5Oo9ouPH6dPXvWOBwOM2vWLKvthx9+MEFBQebll1/2QoYVq+i4O2rUKHPHHXd4JZ/quPC4UVfWeUXHu7qyzo35v/11XVnf55x/nKlL6xu+pyb7nu+//974+/ublStXWjHffPONadSokXn33Xc924FaVJ39QX1aB8XFxSY6Otqkp6ebPn36mEcffdQY03DeCxf7X62hrIPf/e53pnfv3pVOr0vrgTOlLkOXLl0UFhamvn37atOmTd5Op1YUFhZKkoKDgyuN+fDDD5WQkODSNmDAAO3YsUOnTp1ya37uVJW+n1Pftv2ZM2e0cuVKnThxQj179qwwpr5u96r0/Zz6tt1Ruf379ys8PFxRUVG65557dODAAW+nVG3Z2dnKy8tz+dza7Xb16dNHmZmZXsysajZv3qw2bdqoQ4cOeuihh5Sfn+/tlMq58LhRV9Z5Zcc7X1/nF+6v68r6ruw44+vrG76rJvuerKwsnTp1yiUmPDxcMTExPvV5qaqa7A/q0zoYP368Bg4cqH79+rm0N6T1UNn/ag1lHaxZs0bdunXT3XffrTZt2qhLly565ZVXrOl1aT1QlKqBsLAwLVq0SG+99ZbefvttXXvtterbt6+2bNni7dQuizFGkyZNUu/evRUTE1NpXF5enkJDQ13aQkNDdfr0aR09etTdabpFVfte37b97t27dcUVV8hut+vhhx/WqlWr1LFjxwpj69t2r07f69t2x8V1795dr732mt577z298sorysvLU1xcnI4dO+bt1KolLy9Pkir83J6b5qsSExO1fPlybdy4UXPmzNH27dt12223qbS01NupWSo6btSFdV7Z8c6X13ll+2tfX98XO8748vqGb6vpvicvL09NmjRRy5YtK42pCy5nf1Bf1sHKlSv1ySefKCUlpdy0hrIeLva/WkNZBwcOHNDChQsVHR2t9957Tw8//LAeeeQRvfbaa5Lq1nvBz2OvVI9ce+21uvbaa63xnj17KicnR88//7x+9rOfeTGzyzNhwgR99tln2rp16yVjbTaby7gxpsL2uqKqfa9v2/7aa6/Vrl279P333+utt97SqFGjlJGRUWlxpj5t9+r0vb5td1xcYmKi9XdsbKx69uypa665RkuXLtWkSZO8mFnNVPS59fXP7PDhw62/Y2Ji1K1bN0VGRmrt2rUaOnSoFzP7Pxc7bvjyOq8sb19e55Xtr8/x1fV9seOML69v+Lba3vf4yuelqtyxP6hL6yAnJ0ePPvqo1q9fr6ZNm1YaV9/Xw8X+Vzv30Ij6vg7Onj2rbt26KTk5WdKPV3Ts3btXCxcu1MiRI624urAeOFOqlvTo0UP79+/3dho1NnHiRK1Zs0abNm1S27ZtLxrrcDjKVU7z8/Pl5+enkJAQd6bpFtXpe0Xq8rZv0qSJfvKTn6hbt25KSUlR586d9cILL1QYW9+2e3X6XpG6vN1RPS1atFBsbGyd297nnkJT0ef2wl/NfF1YWJgiIyN9ZhtUdtzw9XVeneOdL63zyvbXvr6+q3Oc8aX1Dd91Ofseh8OhsrIyFRQUVBpTF1zO/qA+rIOsrCzl5+era9eu8vPzk5+fnzIyMvTnP/9Zfn5+Vj/q+3q40Pn/qzWU90JYWFi5H9Ovv/56HT58WFLd2i9QlKolO3fuVFhYmLfTqDZjjCZMmKC3335bGzduVFRU1CXn6dmzp9LT013a1q9fr27dusnf399dqda6mvS9InV121fEGFPppQP1ZbtX5mJ9r0h92u64uNLSUn3xxRd1bntHRUXJ4XC4fG7LysqUkZGhuLg4L2ZWfceOHVNOTo7Xt8Gljhu+us5rcrzzlXVekXP7a19d35W52HHGl9c3vK829j1du3aVv7+/S0xubq727Nnjk5+XqqrO/qA+rIO+fftq9+7d2rVrlzV069ZN9913n3bt2qWrr766QayHC53/v1pDeS/06tVL+/btc2n78ssvFRkZKamO7Rc8cTd1X1dcXGx27txpdu7caSSZuXPnmp07d5pDhw4ZY4yZNm2aeeCBB6z4efPmmVWrVpkvv/zS7Nmzx0ybNs1IMm+99Za3ulBjv/3tb01QUJDZvHmzyc3NtYaTJ09aMRf2/8CBA6Z58+bmscceM59//rlJS0sz/v7+5u9//7s3ulBjNel7fdr206dPN1u2bDHZ2dnms88+M0888YRp1KiRWb9+vTGm/m53Y6rf9/q03XFpkydPNps3bzYHDhww27ZtM4MGDTIBAQHm4MGD3k6tnEsdv2bNmmWCgoLM22+/bXbv3m3uvfdeExYWZoqKinw27+LiYjN58mSTmZlpsrOzzaZNm0zPnj3NVVdd5fW8q3Lc8MV1fqm8fXmdX2p/7Yvr+1J5+/L6hm+qrX3Pww8/bNq2bWs2bNhgPvnkE3PbbbeZzp07m9OnT3ujW9VWG/uDur4OKnL+0/eMaRjr4VL/qzWEdfDxxx8bPz8/86c//cns37/fLF++3DRv3twsW7bMiqkr64GilPm/x71fOIwaNcoY8+Nje/v06WPFP/vss+aaa64xTZs2NS1btjS9e/c2a9eu9U7yl6mifksyixcvtmIu7L8xxmzevNl06dLFNGnSxLRv394sXLjQs4nXgpr0vT5t+1//+tcmMjLSNGnSxLRu3dr07dvXOqgbU3+3uzHV73t92u64tOHDh5uwsDDj7+9vwsPDzdChQ83evXu9nVaFLnX8Onv2rHnqqaeMw+Ewdrvd/OxnPzO7d+/2btLm4nmfPHnSJCQkmNatWxt/f3/Trl07M2rUKHP48GFvp12l44YvrvNL5e3L6/xS+2tfXN/GXDxvX17f8E21te8pKSkxEyZMMMHBwaZZs2Zm0KBBdep9Vxv7g7q+DipyYVGqIayHS/2v1hDWgTHG/OMf/zAxMTHGbreb6667zixatMhlel1ZDzZj/vdOxQAAAAAAAICHcE8pAAAAAAAAeBxFKQAAAAAAAHgcRSkAAAAAAAB4HEUpAAAAAAAAeBxFKQAAAAAAAHgcRSkAAAAAAAB4HEUpAAAAAAAAeBxFKQAAAAAAAHgcRSk0WPHx8UpKSpIktW/fXvPnz3fbax08eFA2m027du2SJG3evFk2m03ff/+9214TAFC/1NaxY/To0brzzjut8fOPhwAAAJ5EUQqQtH37dv3mN7/x2OvFxcUpNzdXQUFBl4ylgAUANVPfii3VOXZUx9tvv60//vGPVYqtb+sUALwpPz9fY8eOVbt27WS32+VwODRgwAB9+OGHLnGZmZlq3Lixbr/99gqX89Zbb6l79+4KCgpSQECAOnXqpMmTJ1cYm5CQoMaNG2vbtm213h+gJihKAZJat26t5s2be+z1mjRpIofDIZvN5rHXBABUX1lZmbdTsLjr2BEcHKyAgIBaXSYA4NLuuusuffrpp1q6dKm+/PJLrVmzRvHx8fruu+9c4l599VVNnDhRW7du1eHDh12mbdiwQffcc49++ctf6uOPP1ZWVpb+9Kc/VXj8Onz4sD788ENNmDBBaWlpbu0bUFUUpQCVv3zvP//5j3r37q2mTZuqY8eO2rBhg2w2m1avXl2l5X388cfq0qWLmjZtqm7dumnnzp0u0y88++nQoUMaPHiwWrZsqRYtWqhTp05at26dDh48qFtvvVWS1LJlS9lsNo0ePboWegwA9dvo0aOVkZGhF154QTabTTabTQf/f3t3Hp/Tmf9//H0TbolEELIgYqm11tZSSyWqidpGLbVNlenyVVsFrVJaYaZSau0orU6oTindqLE16ZC0miJiLQYdQdpKM9ZEEJJcvz/6c7e3iAp37jvi9Xw8zqM517nOdT7napyT+3Nf5zrHjunAgQPq3LmzPD095efnp4EDB+rUqVO2/UJCQjRixAiNGTNGFSpUUGhoqO2a/eWXX6pp06Zyd3fXI488otTUVG3YsEH16tVTmTJl1L9/f128ePGW4gsJCdHIkSMVHh6ucuXKyc/PT4sWLVJGRob+8pe/yMvLSzVr1tSGDRts+1x/73j//fdVtmxZffnll6pXr548PT312GOP6eTJk7Z9srOzNWbMGJUtW1Y+Pj4aN26cjDG5Yvn96KcFCxaoVq1aKlWqlPz8/NS7d++b9ikAIP/OnTunLVu2aPr06Wrfvr2CgoLUokULTZgwQV26dLHVy8jI0Mcff6yhQ4eqa9euev/99+3aWbt2rdq2bauXXnpJderUUe3atfX444/r73//e65jLlmyRF27dtXQoUO1cuVKZWRkFPRpAn+IpBRwnZycHD3++OPy8PDQtm3btGjRIk2cOPGW98/IyFDXrl1Vp04dJSYmKiIiQi+++OJN9xk+fLgyMzP19ddfa9++fZo+fbo8PT0VGBiozz77TJJ06NAhnTx5UvPmzbuj8wOAe8G8efPUqlUrPffcczp58qROnjypEiVKKDg4WE2aNNGOHTu0ceNG/fLLL+rTp4/dvkuXLpWbm5u+/fZbvfvuu7byiIgIzZ8/X/Hx8UpOTlafPn00d+5cLV++XOvWrVNMTMwNPwTkZenSpapQoYK2b9+ukSNHaujQoXriiSfUunVr7dy5Ux07dtTAgQNvmui6ePGiZs6cqX/+85/6+uuvdeLECbt7zqxZs7R48WJFRUVpy5YtOnPmjFatWpVnezt27NALL7ygqVOn6tChQ9q4caPatWuXZ58GBgbe8vkCAH7j6ekpT09PrV69WpmZmXnWW7lyperUqaM6deroySef1JIlS+y+XPD399f+/fv1/fff3/R4xhgtWbJETz75pOrWravatWvr448/dtj5ALfNAPeo4OBgM2rUKGOMMUFBQWbOnDnGGGM2bNhg3NzczMmTJ211Y2JijCSzatWqP2z33XffNeXLlzcZGRm2soULFxpJZteuXcYYYzZv3mwkmbNnzxpjjGnYsKGJiIi4YXvX1wUA3JrfX+eNMebVV181YWFhdnWSk5ONJHPo0CHbPk2aNLGrc+06/NVXX9nKIiMjjSTz3//+11Y2ZMgQ07Fjx1uOrW3btrb1rKwsU7p0aTNw4EBb2cmTJ40k891339nFce1+sGTJEiPJ/PDDD7Z93n77bePn52dbDwgIMG+88YZt/erVq6ZKlSqme/fudrFc66fPPvvMlClTxqSlpeUZ9+/7FABw+z799FNTrlw5U6pUKdO6dWszYcIEs2fPHrs6rVu3NnPnzjXG/HoNr1ChgomJibFtv3DhguncubORZIKCgkzfvn1NVFSUuXz5sl070dHRpmLFiubq1avGGGPmzJlj2rRpU8BnCPwxRkoB1zl06JACAwPl7+9vK2vRosUt73/w4EE1btzYbo6qVq1a3XSfF154QX/729/Upk0bTZ48WXv37s1/4ACAm0pMTNTmzZtt3057enqqbt26kqT//ve/tnrNmjW74f6NGjWy/ezn5ycPDw/VqFHDriw1NfWW4/l9e8WLF5ePj48aNmxo156km7bp4eGhmjVr2tYDAgJs9c+fP6+TJ0/a3YPc3NzyPD9JCg0NVVBQkGrUqKGBAwdq2bJlt/xIIgAgf3r16qWff/5Za9asUceOHRUbG6sHHnjA9ojeoUOHtH37dvXr10/Sr9fwvn37avHixbY2SpcurXXr1umHH37QpEmT5OnpqbFjx6pFixZ2uYCV9gAATvJJREFU1++oqCj17dtXbm5ukqT+/ftr27ZtOnTokPNOGLgBklLAdYwxdzSJrLluro5b8eyzz+ro0aMaOHCg9u3bp2bNmuXrERAAwB/LyclRt27dtHv3brvlyJEjtkfUpF//wL+REiVK2H62WCx269fKcnJybjmeG+1//TGuxZ2fNm7nPnSNl5eXdu7cqY8++kgBAQF67bXX1LhxY94ACwAFpFSpUgoNDdVrr72m+Ph4DR48WJMnT5b0ayIpKytLlStXlpubm9zc3LRw4UJ9/vnnOnv2rF07NWvW1LPPPqt//OMf2rlzpw4cOKCVK1dKks6cOaPVq1drwYIFtnYqV66srKwsuwQX4AokpYDr1K1bVydOnNAvv/xiK0tISLjl/evXr689e/bo0qVLtrJbeeVqYGCgnn/+eX3++ecaO3as3nvvPUm/vm1J+nWyWgDArStZsqTdtfOBBx7Q/v37Va1aNd133312S16JqLuZt7e3AgIC7O5BWVlZSkxMvOl+bm5uevTRRzVjxgzt3btXx44d06ZNmyTl7lMAgGPVr19fGRkZysrK0gcffKBZs2bZfZGyZ88eBQUFadmyZXm2Ua1aNXl4eNgmMl+2bJmqVKmiPXv22LU1d+5cLV26VFlZWc46PSAXklLAdUJDQ1WzZk0NGjRIe/fu1bfffmub6PxWRlANGDBAxYoV0zPPPKMDBw5o/fr1mjlz5k33CQ8P15dffqmkpCTt3LlTmzZtUr169SRJQUFBslgsWrt2rf73v//pwoULd36SAHAPqFatmrZt26Zjx47p1KlTGj58uM6cOaP+/ftr+/btOnr0qKKjo/X0008X2UTLqFGj9MYbb2jVqlX6z3/+o2HDht101NPatWv11ltvaffu3Tp+/Lg++OAD5eTkqE6dOpJy92l+RoYBAH5z+vRpPfLII/rwww+1d+9eJSUl6ZNPPtGMGTPUvXt3rV27VmfPntUzzzyjBg0a2C29e/dWVFSUpF9fwjFu3DjFxsYqKSlJu3bt0tNPP62rV68qNDRU0q8jrnr37p2rnaefflrnzp3TunXrXNkVuMeRlAKuU7x4ca1evVoXLlxQ8+bN9eyzz2rSpEmSfh1e+0c8PT31r3/9SwcOHFDTpk01ceJETZ8+/ab7ZGdna/jw4apXr54ee+wx1alTRwsWLJAkVa5cWVOmTNH48ePl5+enESNG3PlJAsA94MUXX1Tx4sVVv359VaxYUVeuXNG3336r7OxsdezYUQ0aNNCoUaPk7e2tYsWK5p9EY8eO1VNPPaXBgwerVatW8vLyUo8ePfKsX7ZsWX3++ed65JFHVK9ePb3zzjv66KOPdP/990vK3acnTpxw1qkAQJHi6empli1bas6cOWrXrp0aNGigV199Vc8995zmz5+vqKgoPfroo/L29s61b69evbR7927t3LlTwcHBOnr0qJ566inVrVtXnTp1UkpKiqKjo21vA9+zZ4969eqVqx0vLy+FhYXZElyAK1jMnUw8ANwjvv32W7Vt21Y//PCD3YSyAAAAAADg9pCUAm5g1apV8vT0VK1atfTDDz9o1KhRKleunLZs2eLq0AAAAAAAKBKK5lh14A6lp6dr2LBhqlu3rgYPHqzmzZvriy++kCRNmzbN7nXiv186derk4sgBAK524sSJPO8Tnp6ePPIGAADw/zFSCsinM2fO6MyZMzfc5u7ursqVKzs5IgBAYZKVlaVjx47lub1atWpyc3NzXkAAAACFFEkpAAAAAAAAOB2P7wEAAAAAAMDpSEoBAAAAAADA6UhKAQAAAAAAwOlISgEAAAAAAMDpSEoBAAAAAADA6UhKAQAAAAAAwOlISgEAAAAAAMDpSEoBAAAAAADA6UhKAQAAAAAAwOlISgEAAAAAAMDpSEoBAAAAAADA6UhKAQAAAAAAwOlISgEAAAAAAMDpSEqhSNm2bZt69OihqlWrymq1ys/PT61atdLYsWNvWH/MmDGyWCzq2rVrnm0mJydr2LBhql27ttzd3VW+fHk1bNhQzz33nJKTk2+4z5o1a2SxWOTj46PMzEyHnNvtCAkJUUhIiMuOD6Do4Ppa9FgsFkVERBRI2wsWLND7779fIG0DAICig6QUiox169apdevWSktL04wZMxQdHa158+apTZs2WrlyZa76V69e1YcffihJ2rhxo3766adcdX788Uc98MADiomJ0ZgxY7R+/XotXrxY/fv3V0JCgo4ePXrDWKKioiRJZ86c0erVqx13kgDgAlxfkV8kpQAAwK2wGGOMq4MAHCE4OFg//fST/vOf/8jNzc1uW05OjooVs8/Bfvrpp3riiSfUpUsXrVu3Tq+//rpeeeUVuzqTJ0/W1KlTdfToUVWvXj3XMW/UbkpKigIDA9WuXTvFx8fr4YcfVnR0tEPO8eLFi/Lw8Ljl+tdGScXGxjrk+ADuTffC9fVeZLFYNHny5AIZLdWgQQNVqFCB+w8AALgpRkqhyDh9+rQqVKiQ6wOTpFwfbKRfv20vWbKklixZosDAQC1ZskTX52hPnz6tYsWKydfX94bHvFG7S5cuVVZWlkaPHq2ePXvq3//+t44fP57v8xk8eLA8PT21b98+hYWFycvLSx06dJAkXblyRX/7299Ut25dWa1WVaxYUX/5y1/0v//9L9/HAYA/UtSur5K0Y8cO/elPf1L58uVVqlQpNW3aVB9//LFt+6lTpxQYGKjWrVvr6tWrtvIDBw6odOnSGjhwoK0sJCREDRo00DfffKOHHnpI7u7uqly5sl599VVlZ2fbHffMmTMaNmyYKleurJIlS6pGjRqaOHFirkcRP/nkE7Vs2VLe3t7y8PBQjRo19PTTT9vVSUtL04svvqjq1aurZMmSqly5ssLDw5WRkZGr3nPPPScfHx95enrqscce0+HDh2+r36ZMmaKWLVuqfPnyKlOmjB544AFFRUXZ/f+tVq2a9u/fr7i4OFksFlksFlWrVs22/cSJE3ryySfl6+srq9WqevXqadasWcrJybHVOXbsmCwWi958801Nnz5d1apVk7u7u0JCQnT48GFdvXpV48ePV6VKleTt7a0ePXooNTXVLtZNmzYpJCREPj4+cnd3V9WqVdWrVy9dvHjxts4dAAAUAAMUEc8++6yRZEaOHGm2bt1qrly5kmfd5ORkU6xYMfPEE08YY4yZNGmSkWRiY2Pt6n344YdGkgkLCzMbN24058+f/8M4ateubQICAkxWVpb56quvjCQTERGR7/MZNGiQKVGihKlWrZqJjIw0//73v82XX35psrOzzWOPPWZKly5tpkyZYmJiYsw//vEPU7lyZVO/fn1z8eJFWxvBwcEmODg438cGgN8ratfXTZs2mZIlS5qHH37YrFy50mzcuNEMHjzYSDJLliyx1duyZYtxc3Mzo0ePNsYYk5GRYerXr2/q1q1rLly4YKsXHBxsfHx8TKVKlcxbb71lvvzyS/PCCy8YSWb48OG2epcuXTKNGjUypUuXNjNnzjTR0dHm1VdfNW5ubqZz5862evHx8cZisZh+/fqZ9evXm02bNpklS5aYgQMH2upkZGSYJk2amAoVKpjZs2ebr776ysybN894e3ubRx55xOTk5BhjjMnJyTHt27c3VqvVvP766yY6OtpMnjzZ1KhRw0gykydPzlffDR482ERFRZmYmBgTExNj/vrXvxp3d3czZcoUW52dO3eaGjVqmKZNm5rvvvvOfPfdd2bnzp3GGGNSU1NN5cqVTcWKFc0777xjNm7caEaMGGEkmaFDh9raSEpKMpJMUFCQ6datm1m7dq358MMPjZ+fn6ldu7YZOHCgefrpp82GDRvMO++8Yzw9PU23bt3s9i9VqpQJDQ01q1evNrGxsWbZsmVm4MCB5uzZs/k6ZwAAUHBISqHIOHXqlGnbtq2RZCSZEiVKmNatW5vIyEiTnp5uV3fq1KlGktm4caMxxpijR48ai8Vi9we/Mb/+MT9kyBBTrFgxI8lYLBZTr149M3r0aJOUlJQrhq+//tpIMuPHj7ftX716dRMUFGT7gHCrBg0aZCSZxYsX25V/9NFHRpL57LPP7MoTEhKMJLNgwQJbGUkpAI5Q1K6vdevWNU2bNjVXr161K+/atasJCAgw2dnZtrLp06cbSWbVqlVm0KBBxt3d3ezdu9duv+DgYCPJfPHFF3blzz33nClWrJg5fvy4McaYd955x0gyH3/8sV29a8eIjo42xhgzc+ZMI8mcO3cuz3OIjIw0xYoVMwkJCXbln376qZFk1q9fb4wxZsOGDUaSmTdvnl29119//baSUr+XnZ1trl69aqZOnWp8fHzs/j/cf//9N7z/jB8/3kgy27ZtsysfOnSosVgs5tChQ8aY35JSjRs3tvv/MXfuXCPJ/OlPf7LbPzw83EiyJTev9cPu3btv+/wAAEDBIymFIichIcG88cYbpnfv3qZChQpGkqlWrZr53//+Z4z57YNMYGCg3R+67du3Nx4eHjf8tv7YsWNmwYIF5umnnzb33XefkWQ8PDxyffN/LZF0+PBhW9mUKVOMJBMTE5Ov87jW1vXx/PnPfzZly5Y1V65cMVevXrVb/P39TZ8+fWx1SUoBcKSicH09cuSIkWRmzpyZ6xq6YMECI8kcOHDAVj8nJ8d06dLFlCpVykgy//jHP3K1GRwcbLy8vHKVb9682Ugy//znP40xxvTp08eULl06VxLtl19+MZLMyy+/bIwxJi4uzjaKbOXKlebHH3/M1XabNm1Mo0aNcp1Denq6sVgsZty4ccYYY8aNG2ckmVOnTtntfy3pk9+k1L///W/ToUMHU6ZMGVuS8tqSkpJiq5dXUqpFixamfv36ucq3bdtmJJmFCxfaxTdhwgS7el9++aWRZN5991278nfffddIMvv27TPGGPPDDz+YkiVLmhYtWpj333/f/Pe//83XeQIAAOdgTikUOc2aNdPLL7+sTz75RD///LNGjx6tY8eOacaMGZJ+nWMiKSlJTzzxhNLS0nTu3DmdO3dOffr00cWLF/XRRx/lajMoKEhDhw5VVFSUjhw5opUrV+ry5ct66aWXbHXS09P1ySefqEWLFqpYsaKt3R49eshisdjeGJUfHh4eKlOmjF3ZL7/8onPnzqlkyZIqUaKE3ZKSkqJTp07l+zgAcCuKwvX1l19+kSS9+OKLua6hw4YNkyS766jFYtHgwYN1+fJl+fv7280l9Xt+fn65yvz9/SX9On/Wtf/6+/vLYrHY1fP19ZWbm5utXrt27bR69WplZWXpqaeeUpUqVdSgQQO7/vvll1+0d+/eXOfg5eUlY4ztHE6fPi03Nzf5+PjcMLb82L59u8LCwiRJ7733nr799lslJCRo4sSJkqRLly79YRunT59WQEBArvJKlSrZtv9e+fLl7dZLlix50/LLly9LkmrWrKmvvvpKvr6+Gj58uGrWrKmaNWtq3rx5fxgjAABwntwzlgJFSIkSJTR58mTNmTNH33//vaTfXic+e/ZszZ49O9c+UVFRGjJkyE3b7dOnjyIjI21tStJHH32kixcvavv27SpXrlyufVatWqWzZ8/ecFterv/gIkkVKlSQj4+PNm7ceMN9vLy8brl9ALhdd+v1tUKFCpKkCRMmqGfPnjesU6dOHdvPJ0+e1PDhw9WkSRPt379fL774ot56661c+1xLdv1eSkqKJNkSQj4+Ptq2bZuMMXbX99TUVGVlZdlik6Tu3bure/fuyszM1NatWxUZGakBAwaoWrVqatWqlSpUqCB3d3ctXrz4pufp4+OjrKwsnT592i4xdS22/FixYoVKlCihtWvXqlSpUrby1atX33IbPj4+OnnyZK7yn3/+2S5uR3j44Yf18MMPKzs7Wzt27NDf//53hYeHy8/PT/369XPYcQAAwO0jKYUi4+TJkzf89vXgwYOSfv0W9uzZs1q1apXatGmjv/3tb7nq/uMf/9CyZcv0/fffq0GDBnm2eeHCBSUnJ9u+2ZV+/bDl5eWl1atX53pr1I4dO/TSSy9p2bJlGjFixB2dZ9euXbVixQplZ2erZcuWd9QWANyKonR9rVOnjmrVqqU9e/Zo2rRpN62bnZ2t/v37y2KxaMOGDVq2bJlefPFFhYSE5Epopaena82aNfrTn/5kK1u+fLmKFSumdu3aSZI6dOigjz/+WKtXr1aPHj1s9T744APb9utZrVYFBwerbNmy+vLLL7Vr1y61atVKXbt21bRp0+Tj46Pq1avneQ7t27fXjBkztGzZMr3wwgt2seWXxWKRm5ubihcvbiu7dOmS/vnPf94w7huNnOrQoYMiIyO1c+dOPfDAA7byDz74QBaLRe3bt893XH+kePHiatmyperWratly5Zp586dJKUAACgkSEqhyOjYsaOqVKmibt26qW7dusrJydHu3bs1a9YseXp6atSoUVq2bJkuX76sF154QSEhIbna8PHx0bJlyxQVFaU5c+bo9ddf17fffqu+ffuqSZMmcnd3V1JSkubPn6/Tp0/rzTfflCR9//332r59u4YOHapHHnkkV7tt2rTRrFmzFBUVdcdJqX79+mnZsmXq3LmzRo0apRYtWqhEiRL68ccftXnzZnXv3t3uww4A3Kmidn1999131alTJ3Xs2FGDBw9W5cqVdebMGR08eFA7d+7UJ598IkmaPHmyvvnmG0VHR8vf319jx45VXFycnnnmGTVt2tQuGeTj46OhQ4fqxIkTql27ttavX6/33ntPQ4cOVdWqVSVJTz31lN5++20NGjRIx44dU8OGDbVlyxZNmzZNnTt31qOPPipJeu211/Tjjz+qQ4cOqlKlis6dO6d58+apRIkSCg4OliSFh4frs88+U7t27TR69Gg1atRIOTk5OnHihKKjozV27Fi1bNlSYWFhateuncaNG6eMjAw1a9ZM33777Q0TSX+kS5cumj17tgYMGKD/+7//0+nTpzVz5kxZrdZcdRs2bKgVK1Zo5cqVqlGjhkqVKqWGDRtq9OjR+uCDD9SlSxdNnTpVQUFBWrdunRYsWKChQ4eqdu3a+Y7rRt555x1t2rRJXbp0UdWqVXX58mXbqLJr/QwAAAoBV09qBTjKypUrzYABA0ytWrWMp6enKVGihKlataoZOHCgbdLaJk2aGF9fX5OZmZlnOw899JCpUKGCyczMNFu3bjXDhw83jRs3NuXLlzfFixc3FStWNI899pjtzUbG/PbWn5u95efaG4cSExNv6XwGDRpkSpcufcNtV69eNTNnzjSNGzc2pUqVMp6enqZu3bpmyJAh5siRI7Z6THQOwBGK2vXVGGP27Nlj+vTpY3x9fU2JEiWMv7+/eeSRR8w777xjjDEmOjraFCtWLNdE4KdPnzZVq1Y1zZs3t51rcHCwuf/++01sbKxp1qyZsVqtJiAgwLzyyiu53vB3+vRp8/zzz5uAgADj5uZmgoKCzIQJE8zly5dtddauXWs6depkKleubEqWLGl8fX1N586dzTfffGPX1oULF8ykSZNMnTp1TMmSJY23t7dp2LChGT16tN2k4+fOnTNPP/20KVu2rPHw8DChoaHmP//5z21NdL548WJTp04dY7VaTY0aNUxkZKSJiooykuzemnjs2DETFhZmvLy8jCQTFBRk23b8+HEzYMAA4+PjY0qUKGHq1Klj3nzzTbvJ8a9NdP7mm2/aHf/a5PGffPKJXfmSJUuMJNvbCL/77jvTo0cPExQUZKxWq/Hx8THBwcFmzZo1+TpfAABQsCzGGOOqhBgAAMDdLiQkRKdOnbKbBwsAAAB/jLfvAQAAAAAAwOmYUwpwspycHOXk5Ny0jpsb/zQBIL+4vt6+rKysm24vVqxYrknmAQAA7hR/XQBO9vTTT6tEiRI3XQAA+eeq62tsbOxd/ejesWPH/rDfpk6d6uowAQBAEcScUoCTHTt2TKdOnbppnWbNmjkpGgAoOri+3p4rV65o7969N61TqVIlVapUyUkRAQCAewVJKQAAAAAAADgdj+8BAAAAAADA6e7K2T5zcnL0888/y8vLSxaLxdXhAMBdxRij9PR0VapU6Z6auJh7BwDcvnv13gEAKFh3ZVLq559/VmBgoKvDAIC7WnJysqpUqeLqMJyGewcA3Ll77d4BAChYd2VSysvLS9KvN8UyZcq4OBoAuLukpaUpMDDQdi29V3DvAIDbd6/eOwAABeuuTEpde+yiTJkyfLAAgNt0rz3Cxr0DAO7cvXbvAAAULB4IBwAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDT5Tsp9fXXX6tbt26qVKmSLBaLVq9ebbfdGKOIiAhVqlRJ7u7uCgkJ0f79++3qZGZmauTIkapQoYJKly6tP/3pT/rxxx/v6EQAAAAAAABw98h3UiojI0ONGzfW/Pnzb7h9xowZmj17tubPn6+EhAT5+/srNDRU6enptjrh4eFatWqVVqxYoS1btujChQvq2rWrsrOzb/9MAAAAAAAAcNdwy+8OnTp1UqdOnW64zRijuXPnauLEierZs6ckaenSpfLz89Py5cs1ZMgQnT9/XlFRUfrnP/+pRx99VJL04YcfKjAwUF999ZU6dux4B6cD2Ks2fp3Ljn3sjS4uOzYAAPnhqvsl90oAAO5tDp1TKikpSSkpKQoLC7OVWa1WBQcHKz4+XpKUmJioq1ev2tWpVKmSGjRoYKtzvczMTKWlpdktAAAAAAAAuHs5NCmVkpIiSfLz87Mr9/Pzs21LSUlRyZIlVa5cuTzrXC8yMlLe3t62JTAw0JFhAwAAAAAAwMkK5O17FovFbt0Yk6vsejerM2HCBJ0/f962JCcnOyxWAAAAAAAAOJ9Dk1L+/v6SlGvEU2pqqm30lL+/v65cuaKzZ8/mWed6VqtVZcqUsVsAAAAAAABw93JoUqp69ery9/dXTEyMrezKlSuKi4tT69atJUkPPvigSpQoYVfn5MmT+v777211AAAAAAAAULTl++17Fy5c0A8//GBbT0pK0u7du1W+fHlVrVpV4eHhmjZtmmrVqqVatWpp2rRp8vDw0IABAyRJ3t7eeuaZZzR27Fj5+PiofPnyevHFF9WwYUPb2/gAAAAAAABQtOU7KbVjxw61b9/etj5mzBhJ0qBBg/T+++9r3LhxunTpkoYNG6azZ8+qZcuWio6OlpeXl22fOXPmyM3NTX369NGlS5fUoUMHvf/++ypevLgDTgkAAAAAAACFncUYY1wdRH6lpaXJ29tb58+fZ34p3FS18etcduxjb3Rx2bGBm7lXr6H36nkDt8JV90vulXcPrqEAgIKQ75FSQH65MjEEoPBbuHChFi5cqGPHjkmS7r//fr322mvq1KmTpF/fzjplyhQtWrTINgL37bff1v333+/CqAEAAADcKYdOdA4AQH5VqVJFb7zxhnbs2KEdO3bokUceUffu3bV//35J0owZMzR79mzNnz9fCQkJ8vf3V2hoqNLT010cOQAAAIA7QVIKAOBS3bp1U+fOnVW7dm3Vrl1br7/+ujw9PbV161YZYzR37lxNnDhRPXv2VIMGDbR06VJdvHhRy5cvd3XoAAAAAO4ASSkAQKGRnZ2tFStWKCMjQ61atVJSUpJSUlIUFhZmq2O1WhUcHKz4+PibtpWZmam0tDS7BQAAAEDhQVIKAOBy+/btk6enp6xWq55//nmtWrVK9evXV0pKiiTJz8/Prr6fn59tW14iIyPl7e1tWwIDAwssfgAAAAD5R1IKAOByderU0e7du7V161YNHTpUgwYN0oEDB2zbLRaLXX1jTK6y602YMEHnz5+3LcnJyQUSOwAAAIDbw9v3AAAuV7JkSd13332SpGbNmikhIUHz5s3Tyy+/LElKSUlRQECArX5qamqu0VPXs1qtslqtBRc0AAAAgDvCSCkAQKFjjFFmZqaqV68uf39/xcTE2LZduXJFcXFxat26tQsjBAAAAHCnGCkFAHCpV155RZ06dVJgYKDS09O1YsUKxcbGauPGjbJYLAoPD9e0adNUq1Yt1apVS9OmTZOHh4cGDBjg6tABAAAA3AGSUgAAl/rll180cOBAnTx5Ut7e3mrUqJE2btyo0NBQSdK4ceN06dIlDRs2TGfPnlXLli0VHR0tLy8vF0cOAAAA4E6QlAIAuFRUVNRNt1ssFkVERCgiIsI5AQEAAABwCuaUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNO5uToAAACAwqTa+HUuOe6xN7q45LgAAACuwkgpAAAAAAAAOB0jpe4hrvrmFwAAAAAA4HqMlAIAAAAAAIDTMVIKKCDMSQIAAAAAQN4YKQUAAAAAAACnc3hSKisrS5MmTVL16tXl7u6uGjVqaOrUqcrJybHVMcYoIiJClSpVkru7u0JCQrR//35HhwIAAAAAAIBCyuFJqenTp+udd97R/PnzdfDgQc2YMUNvvvmm/v73v9vqzJgxQ7Nnz9b8+fOVkJAgf39/hYaGKj093dHhAAAAAAAAoBByeFLqu+++U/fu3dWlSxdVq1ZNvXv3VlhYmHbs2CHp11FSc+fO1cSJE9WzZ081aNBAS5cu1cWLF7V8+XJHhwMAKOQiIyPVvHlzeXl5ydfXV48//rgOHTpkV2fw4MGyWCx2y0MPPeSiiAEAAAA4gsOTUm3bttW///1vHT58WJK0Z88ebdmyRZ07d5YkJSUlKSUlRWFhYbZ9rFargoODFR8f7+hwAACFXFxcnIYPH66tW7cqJiZGWVlZCgsLU0ZGhl29xx57TCdPnrQt69evd1HEAAAAABzB4W/fe/nll3X+/HnVrVtXxYsXV3Z2tl5//XX1799fkpSSkiJJ8vPzs9vPz89Px48fv2GbmZmZyszMtK2npaU5OmwAgIts3LjRbn3JkiXy9fVVYmKi2rVrZyu3Wq3y9/d3dngAAAAACojDR0qtXLlSH374oZYvX66dO3dq6dKlmjlzppYuXWpXz2Kx2K0bY3KVXRMZGSlvb2/bEhgY6OiwAQCFxPnz5yVJ5cuXtyuPjY2Vr6+vateureeee06pqamuCA8AAACAgzh8pNRLL72k8ePHq1+/fpKkhg0b6vjx44qMjNSgQYNs33KnpKQoICDAtl9qamqu0VPXTJgwQWPGjLGtp6WlkZgCgCLIGKMxY8aobdu2atCgga28U6dOeuKJJxQUFKSkpCS9+uqreuSRR5SYmCir1XrDthhlCwAAABRuDk9KXbx4UcWK2Q/AKl68uHJyciRJ1atXl7+/v2JiYtS0aVNJ0pUrVxQXF6fp06ffsE2r1Zrnhw4AQNExYsQI7d27V1u2bLEr79u3r+3nBg0aqFmzZgoKCtK6devUs2fPG7YVGRmpKVOmFGi8AAAAAG6fwx/f69atm15//XWtW7dOx44d06pVqzR79mz16NFD0q+P7YWHh2vatGlatWqVvv/+ew0ePFgeHh4aMGCAo8MBANwlRo4cqTVr1mjz5s2qUqXKTesGBAQoKChIR44cybPOhAkTdP78eduSnJzs6JABAAAA3AGHj5T6+9//rldffVXDhg1TamqqKlWqpCFDhui1116z1Rk3bpwuXbqkYcOG6ezZs2rZsqWio6Pl5eXl6HAAAIWcMUYjR47UqlWrFBsbq+rVq//hPqdPn1ZycrLdY+DXY5QtAAAAULg5PCnl5eWluXPnau7cuXnWsVgsioiIUEREhKMPDwC4ywwfPlzLly/XF198IS8vL9tbWr29veXu7q4LFy4oIiJCvXr1UkBAgI4dO6ZXXnlFFSpUsI3CBQAAAHD3cXhSCgCA/Fi4cKEkKSQkxK58yZIlGjx4sIoXL659+/bpgw8+0Llz5xQQEKD27dtr5cqVjLAFAAAA7mIkpQAALmWMuel2d3d3ffnll06KBgAAAICzOHyicwAAAAAAAOCPkJQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTubk6AAAAgOtVG7/O1SHACVz5//nYG11cdmwAAPArRkoBAAAAAADA6UhKAQAAAAAAwOlISgEAAAAAAMDpSEoBAAAAAADA6UhKAQAAAAAAwOlISgEAAAAAAMDpSEoBAAAAAADA6UhKAQAAAAAAwOlISgEAXCoyMlLNmzeXl5eXfH199fjjj+vQoUN2dYwxioiIUKVKleTu7q6QkBDt37/fRREDAAAAcASSUgAAl4qLi9Pw4cO1detWxcTEKCsrS2FhYcrIyLDVmTFjhmbPnq358+crISFB/v7+Cg0NVXp6ugsjBwAAAHAn3FwdAADg3rZx40a79SVLlsjX11eJiYlq166djDGaO3euJk6cqJ49e0qSli5dKj8/Py1fvlxDhgxxRdgAAAAA7hAjpQAAhcr58+clSeXLl5ckJSUlKSUlRWFhYbY6VqtVwcHBio+Pz7OdzMxMpaWl2S0AAAAACg+SUgCAQsMYozFjxqht27Zq0KCBJCklJUWS5OfnZ1fXz8/Ptu1GIiMj5e3tbVsCAwMLLnAAAAAA+UZSCgBQaIwYMUJ79+7VRx99lGubxWKxWzfG5Cr7vQkTJuj8+fO2JTk52eHxAgAAALh9BZKU+umnn/Tkk0/Kx8dHHh4eatKkiRITE23beYsSAOB6I0eO1Jo1a7R582ZVqVLFVu7v7y9JuUZFpaam5ho99XtWq1VlypSxWwAAAAAUHg5PSp09e1Zt2rRRiRIltGHDBh04cECzZs1S2bJlbXV4ixIA4BpjjEaMGKHPP/9cmzZtUvXq1e22V69eXf7+/oqJibGVXblyRXFxcWrdurWzwwUAAADgIA5/+9706dMVGBioJUuW2MqqVatm+5m3KAEAfm/48OFavny5vvjiC3l5edlGRHl7e8vd3V0Wi0Xh4eGaNm2aatWqpVq1amnatGny8PDQgAEDXBw9AAAAgNvl8JFSa9asUbNmzfTEE0/I19dXTZs21XvvvWfbfrtvUQIAFE0LFy7U+fPnFRISooCAANuycuVKW51x48YpPDxcw4YNU7NmzfTTTz8pOjpaXl5eLowcAAAAwJ1w+Eipo0ePauHChRozZoxeeeUVbd++XS+88IKsVqueeuqpm75F6fjx4zdsMzMzU5mZmbZ1XusNAEWHMeYP61gsFkVERCgiIqLgAwIAAADgFA5PSuXk5KhZs2aaNm2aJKlp06bav3+/Fi5cqKeeespWLz9vUYqMjNSUKVMcHSoAAHeFauPXuToEAAAAwOEc/vheQECA6tevb1dWr149nThxQtLtvUWJ13oDAAAAAAAULQ5PSrVp00aHDh2yKzt8+LCCgoIk3d5blHitNwAAAAAAQNHi8Mf3Ro8erdatW2vatGnq06ePtm/frkWLFmnRokWSxFuUAAAAAAAA4PikVPPmzbVq1SpNmDBBU6dOVfXq1TV37lz9+c9/ttUZN26cLl26pGHDhuns2bNq2bIlb1ECAAAAAAC4hzg8KSVJXbt2VdeuXfPczluUAAAAAAAA7m0On1MKAAAAAAAA+CMkpQAAAAAAAOB0BfL4HgDXqTZ+ncuOfeyNLi47NgAAAADg7sJIKQAAAAAAADgdSSkAAAAAAAA4HUkpAAAAAAAAOB1JKQAAAAAAADgdSSkAAAAAAAA4HUkpAAAAAAAAOB1JKQAAAAAAADgdSSkAAAAAAAA4HUkpAAAAAAAAOJ2bqwO411Qbv87VIQAAAAAAALgcI6UAAAAAAADgdCSlAAAAAAAA4HQkpQAALvf111+rW7duqlSpkiwWi1avXm23ffDgwbJYLHbLQw895JpgAQAAADgESSkAgMtlZGSocePGmj9/fp51HnvsMZ08edK2rF+/3okRAgAAAHA0JjoHALhcp06d1KlTp5vWsVqt8vf3d1JEAAAAAAoaI6UAAHeF2NhY+fr6qnbt2nruueeUmpp60/qZmZlKS0uzWwAAAAAUHiSlAACFXqdOnbRs2TJt2rRJs2bNUkJCgh555BFlZmbmuU9kZKS8vb1tS2BgoBMjBgAAAPBHeHwPAFDo9e3b1/ZzgwYN1KxZMwUFBWndunXq2bPnDfeZMGGCxowZY1tPS0sjMQUAAAAUIiSlAAB3nYCAAAUFBenIkSN51rFarbJarU6MCgAAAEB+8PgeAOCuc/r0aSUnJysgIMDVoQAAAAC4TYyUAgC43IULF/TDDz/Y1pOSkrR7926VL19e5cuXV0REhHr16qWAgAAdO3ZMr7zyiipUqKAePXq4MGoAAAAAd4KkFADA5Xbs2KH27dvb1q/NBTVo0CAtXLhQ+/bt0wcffKBz584pICBA7du318qVK+Xl5eWqkAEAAADcIZJSAACXCwkJkTEmz+1ffvmlE6MBAAAA4AzMKQUAAAAAAACnIykFAAAAAAAApyMpBQAAAAAAAKcjKQUAAAAAAACnK/CkVGRkpCwWi8LDw21lxhhFRESoUqVKcnd3V0hIiPbv31/QoQAAAAAAAKCQKNCkVEJCghYtWqRGjRrZlc+YMUOzZ8/W/PnzlZCQIH9/f4WGhio9Pb0gwwEAAAAAAEAhUWBJqQsXLujPf/6z3nvvPZUrV85WbozR3LlzNXHiRPXs2VMNGjTQ0qVLdfHiRS1fvrygwgEAAAAAAEAhUmBJqeHDh6tLly569NFH7cqTkpKUkpKisLAwW5nValVwcLDi4+MLKhwAAAAAAAAUIm4F0eiKFSu0c+dOJSQk5NqWkpIiSfLz87Mr9/Pz0/Hjx2/YXmZmpjIzM23raWlpDowWAAAAAAAAzubwkVLJyckaNWqUPvzwQ5UqVSrPehaLxW7dGJOr7JrIyEh5e3vblsDAQIfGDAAAAAAAAOdyeFIqMTFRqampevDBB+Xm5iY3NzfFxcXprbfekpubm22E1LURU9ekpqbmGj11zYQJE3T+/Hnbkpyc7OiwAQAAAAAA4EQOf3yvQ4cO2rdvn13ZX/7yF9WtW1cvv/yyatSoIX9/f8XExKhp06aSpCtXriguLk7Tp0+/YZtWq1VWq9XRoQIAAAAAAMBFHJ6U8vLyUoMGDezKSpcuLR8fH1t5eHi4pk2bplq1aqlWrVqaNm2aPDw8NGDAAEeHAwAAAAAAgEKoQCY6/yPjxo3TpUuXNGzYMJ09e1YtW7ZUdHS0vLy8XBEOAAAAAAAAnMwpSanY2Fi7dYvFooiICEVERDjj8AAAAAAAAChkHD7ROQAAAAAAAPBHSEoBAAAAAADA6UhKAQAAAAAAwOlISgEAAAAAAMDpSEoBAAAAAADA6UhKAQAAAAAAwOlISgEAXO7rr79Wt27dVKlSJVksFq1evdpuuzFGERERqlSpktzd3RUSEqL9+/e7JlgAAAAADkFSCgDgchkZGWrcuLHmz59/w+0zZszQ7NmzNX/+fCUkJMjf31+hoaFKT093cqQAAAAAHMXN1QEAANCpUyd16tTphtuMMZo7d64mTpyonj17SpKWLl0qPz8/LV++XEOGDHFmqAAAAAAchJFSAIBCLSkpSSkpKQoLC7OVWa1WBQcHKz4+3oWRAQAAALgTjJQCUCRUG7/OJcc99kYXlxz3XpKSkiJJ8vPzsyv38/PT8ePH89wvMzNTmZmZtvW0tLSCCRAAAADAbSEpBQC4K1gsFrt1Y0yust+LjIzUlClTHHZ8VyU+AQAAgKKKx/cAAIWav7+/pN9GTF2Tmpqaa/TU702YMEHnz5+3LcnJyQUaJwAAAID8ISkFACjUqlevLn9/f8XExNjKrly5ori4OLVu3TrP/axWq8qUKWO3AAAAACg8eHwPAOByFy5c0A8//GBbT0pK0u7du1W+fHlVrVpV4eHhmjZtmmrVqqVatWpp2rRp8vDw0IABA1wYNQAAAIA7QVIKAOByO3bsUPv27W3rY8aMkSQNGjRI77//vsaNG6dLly5p2LBhOnv2rFq2bKno6Gh5eXm5KmQAAAAAd4ikFADA5UJCQmSMyXO7xWJRRESEIiIinBcUAAAAgALFnFIAAAAAAABwOpJSAAAAAAAAcDqSUgAAAAAAAHA6klIAAAAAAABwOpJSAAAAAAAAcDqSUgAAAAAAAHA6klIAAAAAAABwOjdXBwAAAAA4W7Xx61x27GNvdHHZsQEAKEwYKQUAAAAAAACnIykFAAAAAAAApyMpBQAAAAAAAKcjKQUAAAAAAACnIykFAAAAAAAAp3N4UioyMlLNmzeXl5eXfH199fjjj+vQoUN2dYwxioiIUKVKleTu7q6QkBDt37/f0aEAAAAAAACgkHJ4UiouLk7Dhw/X1q1bFRMTo6ysLIWFhSkjI8NWZ8aMGZo9e7bmz5+vhIQE+fv7KzQ0VOnp6Y4OBwAAAAAAAIWQm6Mb3Lhxo936kiVL5Ovrq8TERLVr107GGM2dO1cTJ05Uz549JUlLly6Vn5+fli9friFDhjg6JAAAAAAAABQyBT6n1Pnz5yVJ5cuXlyQlJSUpJSVFYWFhtjpWq1XBwcGKj48v6HAAAAAAAABQCDh8pNTvGWM0ZswYtW3bVg0aNJAkpaSkSJL8/Pzs6vr5+en48eM3bCczM1OZmZm29bS0tAKKGAAAAAAAAM5QoEmpESNGaO/evdqyZUuubRaLxW7dGJOr7JrIyEhNmTLFobFVG7/Ooe0BAAAAAADg1hXY43sjR47UmjVrtHnzZlWpUsVW7u/vL+m3EVPXpKam5ho9dc2ECRN0/vx525KcnFxQYQMAAAAAAMAJHD5SyhijkSNHatWqVYqNjVX16tXttlevXl3+/v6KiYlR06ZNJUlXrlxRXFycpk+ffsM2rVarrFaro0MFAAAoNBjFDQAA7jUOT0oNHz5cy5cv1xdffCEvLy/biChvb2+5u7vLYrEoPDxc06ZNU61atVSrVi1NmzZNHh4eGjBggKPDAQAAAAAAQCHk8Mf3Fi5cqPPnzyskJEQBAQG2ZeXKlbY648aNU3h4uIYNG6ZmzZrpp59+UnR0tLy8vBwdDgCgCIiIiJDFYrFbrj0ODgAAAODuVCCP7/0Ri8WiiIgIRUREOPrwAIAi6v7779dXX31lWy9evLgLowEAAABwpwr07XsAADiKm5sbo6MAAACAIqTA3r4HAIAjHTlyRJUqVVL16tXVr18/HT169Kb1MzMzlZaWZrcAAAAAKDwYKQUAd8CVb8s69kYXlx3b2Vq2bKkPPvhAtWvX1i+//KK//e1vat26tfbv3y8fH58b7hMZGakpU6Y4OVIAAAAAt4qRUgCAQq9Tp07q1auXGjZsqEcffVTr1v2aDFy6dGme+0yYMEHnz5+3LcnJyc4KFwAAAMAtYKQUAIdx5agh3FtKly6thg0b6siRI3nWsVqtslqtTowKAAAAQH4wUgoAcNfJzMzUwYMHFRAQ4OpQAAAAANwmklIAgELvxRdfVFxcnJKSkrRt2zb17t1baWlpGjRokKtDAwAAAHCbeHwPAFDo/fjjj+rfv79OnTqlihUr6qGHHtLWrVsVFBTk6tAAAAAA3CaSUgCAQm/FihWuDgEAAACAg5GUAgAAAJzIVS8GOfZGF5ccFwCAvDCnFAAAAAAAAJyOpBQAAAAAAACcjqQUAAAAAAAAnI6kFAAAAAAAAJyOpBQAAAAAAACcjqQUAAAAAAAAnI6kFAAAAAAAAJyOpBQAAAAAAACcjqQUAAAAAAAAnI6kFAAAAAAAAJyOpBQAAAAAAACcjqQUAAAAAAAAnI6kFAAAAAAAAJyOpBQAAAAAAACcjqQUAAAAAAAAnI6kFAAAAAAAAJyOpBQAAAAAAACcjqQUAAAAAAAAnI6kFAAAAAAAAJyOpBQAAAAAAACczqVJqQULFqh69eoqVaqUHnzwQX3zzTeuDAcAUMhx3wAAAACKDpclpVauXKnw8HBNnDhRu3bt0sMPP6xOnTrpxIkTrgoJAFCIcd8AAAAAihaXJaVmz56tZ555Rs8++6zq1aunuXPnKjAwUAsXLnRVSACAQoz7BgAAAFC0uCQpdeXKFSUmJiosLMyuPCwsTPHx8a4ICQBQiHHfAAAAAIoeN1cc9NSpU8rOzpafn59duZ+fn1JSUnLVz8zMVGZmpm39/PnzkqS0tLTbjiEn8+Jt7wsAhcHtXgOv7WeMcWQ4BSq/9w3J8fcO7hsA7nZ38rfz3XjvAAAUfi5JSl1jsVjs1o0xucokKTIyUlOmTMlVHhgYWGCxAUBh5z33zvZPT0+Xt7e3Q2Jxllu9b0jcOwDgend635DuznsHAKDwcklSqkKFCipevHiub7dTU1NzfQsuSRMmTNCYMWNs6zk5OTpz5ox8fHzy/DBSmKWlpSkwMFDJyckqU6aMq8NxKfriN/TFr+iH3xRUXxhjlJ6erkqVKjmszYKW3/uGlPe9o0SJEqpatSq/Y3ng32De6Ju80Td5Kyp9czfeOwAAhZ9LklIlS5bUgw8+qJiYGPXo0cNWHhMTo+7du+eqb7VaZbVa7crKli1b0GEWuDJlytzVf5w4En3xG/riV/TDbwqiL+62b7nze9+Q8r53XHsEhd+xm6N/8kbf5I2+yVtR6Ju77d4BACj8XPb43pgxYzRw4EA1a9ZMrVq10qJFi3TixAk9//zzrgoJAFCIcd8AAAAAihaXJaX69u2r06dPa+rUqTp58qQaNGig9evXKygoyFUhAQAKMe4bAAAAQNHi0onOhw0bpmHDhrkyBJewWq2aPHlyrsdK7kX0xW/oi1/RD7+hL3JzxH2Dfr05+idv9E3e6Ju80TcAAOTNYnivKwAAAAAAAJysmKsDAAAAAAAAwL2HpBQAAAAAAACcjqQUAAAAAAAAnI6kVAFZsGCBqlevrlKlSunBBx/UN998k2fdzz//XKGhoapYsaLKlCmjVq1a6csvv3RitAUrP32xZcsWtWnTRj4+PnJ3d1fdunU1Z84cJ0ZbsPLTF7/37bffys3NTU2aNCnYAJ0kP/0QGxsri8WSa/nPf/7jxIgLTn5/JzIzMzVx4kQFBQXJarWqZs2aWrx4sZOiLRpu999hUfL111+rW7duqlSpkiwWi1avXm233RijiIgIVapUSe7u7goJCdH+/ftdE6yTRUZGqnnz5vLy8pKvr68ef/xxHTp0yK7Ovdo/CxcuVKNGjVSmTBnb3ysbNmywbb9X++VGIiMjZbFYFB4ebiujfwAAyI2kVAFYuXKlwsPDNXHiRO3atUsPP/ywOnXqpBMnTtyw/tdff63Q0FCtX79eiYmJat++vbp166Zdu3Y5OXLHy29flC5dWiNGjNDXX3+tgwcPatKkSZo0aZIWLVrk5MgdL799cc358+f11FNPqUOHDk6KtGDdbj8cOnRIJ0+etC21atVyUsQF53b6ok+fPvr3v/+tqKgoHTp0SB999JHq1q3rxKjvbrf7+1fUZGRkqHHjxpo/f/4Nt8+YMUOzZ8/W/PnzlZCQIH9/f4WGhio9Pd3JkTpfXFychg8frq1btyomJkZZWVkKCwtTRkaGrc692j9VqlTRG2+8oR07dmjHjh165JFH1L17d1ti5V7tl+slJCRo0aJFatSokV05/QMAwA0YOFyLFi3M888/b1dWt25dM378+Ftuo379+mbKlCmODs3pHNEXPXr0ME8++aSjQ3O62+2Lvn37mkmTJpnJkyebxo0bF2CEzpHffti8ebORZM6ePeuE6Jwrv32xYcMG4+3tbU6fPu2M8IokR1yTihpJZtWqVbb1nJwc4+/vb9544w1b2eXLl423t7d55513XBCha6WmphpJJi4uzhhD/1yvXLly5h//+Af98v+lp6ebWrVqmZiYGBMcHGxGjRpljOH3BgCAvDBSysGuXLmixMREhYWF2ZWHhYUpPj7+ltrIyclRenq6ypcvXxAhOo0j+mLXrl2Kj49XcHBwQYToNLfbF0uWLNF///tfTZ48uaBDdIo7+Z1o2rSpAgIC1KFDB23evLkgw3SK2+mLNWvWqFmzZpoxY4YqV66s2rVr68UXX9SlS5ecEfJdzxHXpHtBUlKSUlJS7PrJarUqODj4nuyn8+fPS5Ltnkz//Co7O1srVqxQRkaGWrVqRb/8f8OHD1eXLl306KOP2pXTPwAA3JibqwMoak6dOqXs7Gz5+fnZlfv5+SklJeWW2pg1a5YyMjLUp0+fggjRae6kL6pUqaL//e9/ysrKUkREhJ599tmCDLXA3U5fHDlyROPHj9c333wjN7ei8U/1dvohICBAixYt0oMPPqjMzEz985//VIcOHRQbG6t27do5I+wCcTt9cfToUW3ZskWlSpXSqlWrdOrUKQ0bNkxnzpxhXqlb4Ijr873gWl/cqJ+OHz/uipBcxhijMWPGqG3btmrQoIEk+mffvn1q1aqVLl++LE9PT61atUr169e3JVbu1X6RpBUrVmjnzp1KSEjIte1e/70BACAvReOTbiFksVjs1o0xucpu5KOPPlJERIS++OIL+fr6FlR4TnU7ffHNN9/owoUL2rp1q8aPH6/77rtP/fv3L8gwneJW+yI7O1sDBgzQlClTVLt2bWeF5zT5+Z2oU6eO6tSpY1tv1aqVkpOTNXPmzLs6KXVNfvoiJydHFotFy5Ytk7e3tyRp9uzZ6t27t95++225u7sXeLxFwe1en+819JM0YsQI7d27V1u2bMm17V7tnzp16mj37t06d+6cPvvsMw0aNEhxcXG27fdqvyQnJ2vUqFGKjo5WqVKl8qx3r/YPAAB54fE9B6tQoYKKFy+e61v31NTUXN+OXW/lypV65pln9PHHH+ca9n03upO+qF69uho2bKjnnntOo0ePVkRERAFGWvDy2xfp6enasWOHRowYITc3N7m5uWnq1Knas2eP3NzctGnTJmeF7lB38jvxew899JCOHDni6PCc6nb6IiAgQJUrV7YlpCSpXr16Msboxx9/LNB4iwJH/f4Vdf7+/pJ0z/fTyJEjtWbNGm3evFlVqlSxld/r/VOyZEndd999atasmSIjI9W4cWPNmzfvnu+XxMREpaam6sEHH7Tdt+Pi4vTWW2/Jzc3N1gf3av8AAJAXklIOVrJkST344IOKiYmxK4+JiVHr1q3z3O+jjz7S4MGDtXz5cnXp0qWgw3SK2+2L6xljlJmZ6ejwnCq/fVGmTBnt27dPu3fvti3PP/+87Rvqli1bOit0h3LU78SuXbsUEBDg6PCc6nb6ok2bNvr555914cIFW9nhw4dVrFgxuw/NuDFH/f4VddWrV5e/v79dP125ckVxcXH3RD8ZYzRixAh9/vnn2rRpk6pXr263/V7vn+tdu0ff6/3SoUOHXPftZs2a6c9//rN2796tGjVq3NP9AwBAnlwwuXqRt2LFClOiRAkTFRVlDhw4YMLDw03p0qXNsWPHjDHGjB8/3gwcONBWf/ny5cbNzc28/fbb5uTJk7bl3LlzrjoFh8lvX8yfP9+sWbPGHD582Bw+fNgsXrzYlClTxkycONFVp+Aw+e2L6xWVt+/ltx/mzJljVq1aZQ4fPmy+//57M378eCPJfPbZZ646BYfJb1+kp6ebKlWqmN69e5v9+/ebuLg4U6tWLfPss8+66hTuOn/U5/eK9PR0s2vXLrNr1y4jycyePdvs2rXLHD9+3BhjzBtvvGG8vb3N559/bvbt22f69+9vAgICTFpamosjL3hDhw413t7eJjY21u6efPHiRVude7V/JkyYYL7++muTlJRk9u7da1555RVTrFgxEx0dbYy5d/slL79/+54x9A8AADdCUqqAvP322yYoKMiULFnSPPDAA7ZXSRtjzKBBg0xwcLBtPTg42EjKtQwaNMj5gReA/PTFW2+9Ze6//37j4eFhypQpY5o2bWoWLFhgsrOzXRC54+WnL65XVJJSxuSvH6ZPn25q1qxpSpUqZcqVK2fatm1r1q1b54KoC0Z+fycOHjxoHn30UePu7m6qVKlixowZY/dhGX/sZn1+r9i8efNN7zs5OTlm8uTJxt/f31itVtOuXTuzb98+1wbtJDfqF0lmyZIltjr3av88/fTTtn87FStWNB06dLAlpIy5d/slL9cnpegfAABysxhjjLNHZwEAAAAAAODexpxSAAAAAAAAcDqSUgAAAAAAAHA6klIAAAAAAABwOpJSAAAAAAAAcDqSUgAAAAAAAHA6klIAAAAAAABwOpJSAAAAAAAAcDqSUgAAAAAAAHA6klJAERESEqLw8HBXhwEAKIRiY2NlsVh07tw5V4cCAABgQ1IK94TU1FQNGTJEVatWldVqlb+/vzp27KjvvvvOrl58fLyKFy+uxx577IbtfPbZZ2rZsqW8vb3l5eWl+++/X2PHjr1h3bCwMBUvXlxbt251+PkAwN2Ea3DRwxchAADAEUhK4Z7Qq1cv7dmzR0uXLtXhw4e1Zs0ahYSE6MyZM3b1Fi9erJEjR2rLli06ceKE3bavvvpK/fr1U+/evbV9+3YlJibq9ddf15UrV3Id78SJE/ruu+80YsQIRUVF3VHsN2ofAO4md/M1GAAAAAXIAEXc2bNnjSQTGxt703oXLlwwXl5e5j//+Y/p27evmTJlit32UaNGmZCQkFs6ZkREhOnXr585ePCg8fLyMhcuXLjleIODg83w4cPN6NGjjY+Pj2nXrp0xxpj9+/ebTp06mdKlSxtfX1/z5JNPmv/97392+40aNeqWjwMAznC3XYNzcnLM9OnTTfXq1U2pUqVMo0aNzCeffGLb1qFDB9OxY0eTk5NjO7/AwEDzyiuvGGOM2bx5s5Fk1q5daxo1amSsVqtp0aKF2bt3r91xPv30U1O/fn1TsmRJExQUZGbOnGm3/e233zb33XefsVqtxtfX1/Tq1euWYrxm3bp1platWqZUqVImJCTELFmyxEgyZ8+e/cM+OHXqlOnXr5+pXLmycXd3Nw0aNDDLly+3bR80aJCRZLckJSUZY4yJjY01zZs3NyVLljT+/v7m5ZdfNlevXrXtGxwcbEaMGGFGjRplypYta3x9fc27775rLly4YAYPHmw8PT1NjRo1zPr16237nDlzxgwYMMBUqFDBlCpVytx3331m8eLFf3geAACg8GOkFIo8T09PeXp6avXq1crMzMyz3sqVK1WnTh3VqVNHTz75pJYsWSJjjG27v7+/9u/fr++///6mxzPGaMmSJXryySdVt25d1a5dWx9//HG+Yl66dKnc3Nz07bff6t1339XJkycVHBysJk2aaMeOHdq4caN++eUX9enTJ1/tAoCz3W3X4EmTJmnJkiVauHCh9u/fr9GjR+vJJ59UXFycLBaLli5dqu3bt+utt96SJD3//PPy8/NTRESEXTsvvfSSZs6cqYSEBPn6+upPf/qTrl69KklKTExUnz591K9fP+3bt08RERF69dVX9f7770uSduzYoRdeeEFTp07VoUOHtHHjRrVr1+6WYpSk5ORk9ezZU507d9bu3bv17LPPavz48bfcB5cvX9aDDz6otWvX6vvvv9f//d//aeDAgdq2bZskad68eWrVqpWee+45nTx5UidPnlRgYKB++uknde7cWc2bN9eePXu0cOFCRUVF6W9/+5td+0uXLlWFChW0fft2jRw5UkOHDtUTTzyh1q1ba+fOnerYsaMGDhyoixcvSpJeffVVHThwQBs2bNDBgwe1cOFCVahQ4ZbPBwAAFGKuzYkBzvHpp5+acuXKmVKlSpnWrVubCRMmmD179tjVad26tZk7d64xxpirV6+aChUqmJiYGNv2CxcumM6dOxtJJigoyPTt29dERUWZy5cv27UTHR1tKlasaPtmeM6cOaZNmza3HGtwcLBp0qSJXdmrr75qwsLC7MqSk5ONJHPo0CHbfoyUAlAY3S3X4AsXLphSpUqZ+Ph4u/JnnnnG9O/f37b+8ccfG6vVaiZMmGA8PDxs12FjfhsptWLFClvZ6dOnjbu7u1m5cqUxxpgBAwaY0NBQu2O89NJLpn79+sYYYz777DNTpkwZk5aWdlsxTpgwwdSrV882mssYY15++eVbHil1I507dzZjx461rd/onvPKK6+YOnXq2B337bffNp6eniY7O9u2X9u2bW3bs7KyTOnSpc3AgQNtZSdPnjSSzHfffWeMMaZbt27mL3/5y23FDQAACjdGSuGe0KtXL/38889as2aNOnbsqNjYWD3wwAO2b6UPHTqk7du3q1+/fpIkNzc39e3bV4sXL7a1Ubp0aa1bt04//PCDJk2aJE9PT40dO1YtWrSwfZsrSVFRUerbt6/c3NwkSf3799e2bdt06NChW463WbNmduuJiYnavHmzbcSBp6en6tatK0n673//e1t9AgDOcrdcgw8cOKDLly8rNDTU7nr7wQcf2F1rn3jiCfXs2VORkZGaNWuWateunautVq1a2X4uX7686tSpo4MHD0qSDh48qDZt2tjVb9OmjY4cOaLs7GyFhoYqKChINWrU0MCBA7Vs2TLbOd5KjAcPHtRDDz0ki8Vyw3j+SHZ2tl5//XU1atRIPj4+8vT0VHR0dK55vq538OBBtWrVyu64bdq00YULF/Tjjz/ayho1amT7uXjx4vLx8VHDhg1tZX5+fpJ+nSBfkoYOHaoVK1aoSZMmGjdunOLj42/5XAAAQOHm5uoAAGcpVaqUQkNDFRoaqtdee03PPvusJk+erMGDBysqKkpZWVmqXLmyrb4xRiVKlNDZs2dVrlw5W3nNmjVVs2ZNPfvss5o4caJq166tlStX6i9/+YvOnDmj1atX6+rVq1q4cKFtn+zsbC1evFjTp0+/pVhLly5tt56Tk6Nu3brdcP+AgID8dgUAON3dcA3OycmRJK1bt84uFkmyWq22ny9evKjExEQVL15cR44cueU+uJasMcbYJW6ulV3j5eWlnTt3KjY2VtHR0XrttdcUERGhhISEW4rx923djlmzZmnOnDmaO3euGjZsqNKlSys8PPwPX7xxs/P6fXmJEiXs6lgsFruya3WvnWunTp10/PhxrVu3Tl999ZU6dOig4cOHa+bMmbd/kgAAoFBgpBTuWfXr11dGRoaysrL0wQcfaNasWdq9e7dt2bNnj4KCgrRs2bI826hWrZo8PDyUkZEhSVq2bJmqVKmiPXv22LU1d+5cLV26VFlZWbcV6wMPPKD9+/erWrVquu++++yW6xNYAHA3KIzX4Pr168tqterEiRO5rrWBgYG2emPHjlWxYsW0YcMGvfXWW9q0aVOutrZu3Wr7+ezZszp8+LBthGv9+vW1ZcsWu/rx8fGqXbu2ihcvLunX0WKPPvqoZsyYob179+rYsWPatGnTLcVYv359u+NfH88f+eabb9S9e3c9+eSTaty4sWrUqJEr+VayZEllZ2fn6r/4+Hi7pFh8fLy8vLxyJdDyq2LFiho8eLA+/PBDzZ07V4sWLbqj9gAAQOHASCkUeadPn9YTTzyhp59+Wo0aNZKXl5d27NihGTNmqHv37lq7dq3Onj2rZ555Rt7e3nb79u7dW1FRURoxYoQiIiJ08eJFde7cWUFBQTp37pzeeustXb16VaGhoZJ+fWykd+/eatCggV07QUFBevnll7Vu3Tp179493+cwfPhwvffee+rfv79eeuklVahQQT/88INWrFih9957z/YhBgAKm7vpGuzl5aUXX3xRo0ePVk5Ojtq2bau0tDTFx8fL09NTgwYN0rp167R48WJ99913euCBBzR+/HgNGjRIe/futRvRNXXqVPn4+MjPz08TJ05UhQoV9Pjjj0v6NanVvHlz/fWvf1Xfvn313Xffaf78+VqwYIEkae3atTp69KjatWuncuXKaf369crJyVGdOnVuKcbnn39es2bN0pgxYzRkyBAlJibaHpW8Fffdd58+++wzxcfHq1y5cpo9e7ZSUlJUr149W51q1app27ZtOnbsmDw9PVW+fHkNGzZMc+fO1ciRIzVixAgdOnRIkydP1pgxY1Ss2O1/D/raa6/pwQcf1P3336/MzEytXbvWLhYAAHAXc910VoBzXL582YwfP9488MADxtvb23h4eJg6deqYSZMmmYsXL5quXbuazp0733DfxMREI8kkJiaaTZs2mV69epnAwEBTsmRJ4+fnZx577DHzzTffGGOM2bFjh5Fktm/ffsO2unXrZrp16/aH8eY1Yfnhw4dNjx49TNmyZY27u7upW7euCQ8Pt00oy0TnAAqju+0anJOTY+bNm2fq1KljSpQoYSpWrGg6duxo4uLiTGpqqvHz8zPTpk2z1b969app0aKF6dOnjzHmt4nO//Wvf5n777/flCxZ0jRv3tzs3r3b7jiffvqpqV+/vilRooSpWrWqefPNN23bvvnmGxMcHGzKlStn3N3dTaNGjWyTpP9RjNf861//Mvfdd5+xWq3m4YcfNosXL77lic5Pnz5tunfvbjw9PY2vr6+ZNGmSeeqpp0z37t1tdQ4dOmQeeugh4+7ubiSZpKQkY4wxsbGxpnnz5qZkyZLG39/fvPzyy7ZJ54258b0qKCjIzJkzx65Mklm1apUxxpi//vWvpl69esbd3d2UL1/edO/e3Rw9evQPzwMAABR+FmPucOIBAAAASJJiY2PVvn17nT17VmXLlnV1OAAAAIUac0oBAAAAAADA6UhKAU504sQJu1d4X7/80eu2AQC3j2vwrzp16pRnH0ybNs3V4QEAgHsIj+8BTpSVlaVjx47lub1atWpyc+P9AwBQELgG/+qnn37SpUuXbritfPnyKl++vJMjAgAA9yqSUgAAAAAAAHA6Ht8DAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNORlAIAAAAAAIDTkZQCAAAAAACA05GUAgAAAAAAgNP9PzHCIOYZu76PAAAAAElFTkSuQmCC",
      "text/plain": [
       "<Figure size 1200x1200 with 11 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "## Visualizing the distributions of diffusion analysis metrics\n",
    "## Plotting design scores\n",
    "plt.figure(figsize=(12, 12))\n",
    "for i,k in enumerate(dif_analysis_df.keys()):\n",
    "    if k in [\"description\"]:\n",
    "        continue\n",
    "    plt.subplot(4, 3, i+1)\n",
    "    plt.hist(dif_analysis_df[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4d6136b4",
   "metadata": {},
   "source": [
    "It is highly advised that you manually inspect the filtered diffusion outputs before continuing with the pipeline.\n",
    "While the filters attempt to pick out the most offending designs then nothing beats your own intuition and judgement."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9561c720",
   "metadata": {},
   "source": [
    "If you would like to perform RFjoint Inpainting on the diffusion outputs, please go to the [inpainting section](#inpainting)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f93a2f55",
   "metadata": {},
   "source": [
    "## 1: Running ProteinMPNN on diffused backbones\n",
    "\n",
    "We are first trying to just design a sequence on the backbone, without considering the ligand.\n",
    "The goal is to first find backbones that fold well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "7235734c-e319-48cd-80e2-f6c5e0cacfae",
   "metadata": {},
   "outputs": [],
   "source": [
    "diffused_backbones_good = glob.glob(f\"{DIFFUSION_DIR}/filtered_structures/*.pdb\")\n",
    "assert len(diffused_backbones_good) > 0, \"No good backbones found!\"\n",
    "\n",
    "os.chdir(WDIR)\n",
    "\n",
    "MPNN_DIR = f\"{WDIR}/1_proteinmpnn\"\n",
    "os.makedirs(MPNN_DIR, exist_ok=True)\n",
    "os.chdir(MPNN_DIR)\n",
    "\n",
    "\n",
    "### Parsing diffusion output TRB files to extract fixed motif residues\n",
    "## These residues will not be redesigned with proteinMPNN\n",
    "mask_json_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/make_maskdict_from_trb.py --out masked_pos.jsonl --trb\"\n",
    "for d in diffused_backbones_good:\n",
    "    mask_json_cmd += \" \" + d.replace(\".pdb\", \".trb\")\n",
    "\n",
    "p = subprocess.Popen(mask_json_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n",
    "\n",
    "assert os.path.exists(\"masked_pos.jsonl\"), \"Failed to create masked positions JSONL file\"\n",
    "\n",
    "# Read the file contents\n",
    "with open(\"masked_pos.jsonl\", 'r') as file:\n",
    "    content = file.read()\n",
    "\n",
    "# Replace all occurrences of \"AAA\" with \"BBB\"\n",
    "updated_content = content.replace(WDIR, WDIR)\n",
    "\n",
    "# Write the updated content back to the file\n",
    "with open(\"masked_pos.jsonl\", 'w') as file:\n",
    "    file.write(updated_content)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "c2ae0e8c-2f68-4fbe-91d0-f861d0595ba0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Example MPNN command:\n",
      "#!/bin/bash\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_25_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_201.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_173.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.1 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_25_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_201.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_173.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.2 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_25_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_201.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_173.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97_traj4.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj2.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj3.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/run.py --model_type protein_mpnn --ligand_mpnn_use_atom_context 0 --fixed_residues_multi masked_pos.jsonl --out_folder . --number_of_batches 5 --temperature 0.3 --omit_AA CM --pdb_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj1.pdb --checkpoint_protein_mpnn /home/ubuntu/data/heme_binder_diffusion_edit/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\n",
      "\n",
      "Running...\n",
      "b\"Designing protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_25_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119']\\nThese residues will be fixed:  ['A52']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_201.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A53']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A44']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A31']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A44']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A36']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_173.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A31']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A36']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_25_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119']\\nThese residues will be fixed:  ['A52']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_201.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A53']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A44']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A31']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A44']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A36']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_173.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A31']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A36']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_25_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119']\\nThese residues will be fixed:  ['A52']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_201.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A53']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_91_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A44']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A31']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_95_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A44']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A36']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_216.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A97']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_227_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_73_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_173.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_209_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A31']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_220_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_28.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121']\\nThese residues will be fixed:  ['A57']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_132_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136']\\nThese residues will be fixed:  ['A35']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_147_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A51']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_52_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_226_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A66']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_89_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A43']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_64.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A90']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_177_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126']\\nThese residues will be fixed:  ['A77']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_1_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135']\\nThese residues will be fixed:  ['A64']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_191_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A68']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_97_traj4.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127']\\nThese residues will be fixed:  ['A36']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_100.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138', 'A139']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_148_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_43_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A80']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_158_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132']\\nThese residues will be fixed:  ['A91']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_222_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137', 'A138']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_108_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123']\\nThese residues will be fixed:  ['A58']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_105_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128']\\nThese residues will be fixed:  ['A86']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_130_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A99']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_78.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113']\\nThese residues will be fixed:  ['A55']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_24_traj2.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133']\\nThese residues will be fixed:  ['A40']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_185_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_69_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131']\\nThese residues will be fixed:  ['A95']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_45_traj3.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A72', 'A73', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134']\\nThese residues will be fixed:  ['A74']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_131_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A71', 'A72', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105']\\nThese residues will be fixed:  ['A70']\\nDesigning protein from this path: /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/7o2g_HBA_dif_57_traj1.pdb\\nThese residues will be redesigned:  ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13', 'A14', 'A15', 'A16', 'A17', 'A18', 'A19', 'A20', 'A21', 'A22', 'A23', 'A24', 'A25', 'A26', 'A27', 'A28', 'A29', 'A30', 'A31', 'A32', 'A33', 'A34', 'A35', 'A36', 'A37', 'A38', 'A39', 'A40', 'A41', 'A42', 'A43', 'A44', 'A45', 'A46', 'A47', 'A48', 'A49', 'A50', 'A51', 'A52', 'A53', 'A54', 'A55', 'A56', 'A57', 'A58', 'A59', 'A60', 'A61', 'A62', 'A63', 'A64', 'A65', 'A66', 'A67', 'A68', 'A69', 'A70', 'A71', 'A73', 'A74', 'A75', 'A76', 'A77', 'A78', 'A79', 'A80', 'A81', 'A82', 'A83', 'A84', 'A85', 'A86', 'A87', 'A88', 'A89', 'A90', 'A91', 'A92', 'A93', 'A94', 'A95', 'A96', 'A97', 'A98', 'A99', 'A100', 'A101', 'A102', 'A103', 'A104', 'A105', 'A106', 'A107', 'A108', 'A109', 'A110', 'A111', 'A112', 'A113', 'A114', 'A115', 'A116', 'A117', 'A118', 'A119', 'A120', 'A121', 'A122', 'A123', 'A124', 'A125', 'A126', 'A127', 'A128', 'A129', 'A130', 'A131', 'A132', 'A133', 'A134', 'A135', 'A136', 'A137']\\nThese residues will be fixed:  ['A72']\\n\"\n",
      "b''\n",
      "Done\n"
     ]
    }
   ],
   "source": [
    "\n",
    "### Setting up proteinMPNN run commands\n",
    "## We're doing design with 3 temperatures, and 5 sequences each.\n",
    "## This usually gives decent success with designable backbones.\n",
    "## For more complicated cases consider doing >100 sequences.\n",
    "\n",
    "MPNN_temperatures = [0.1, 0.2, 0.3]\n",
    "MPNN_outputs_per_temperature = 5\n",
    "MPNN_omit_AAs = \"CM\"\n",
    "\n",
    "\n",
    "commands_mpnn = \"#!/bin/bash\\n\"\n",
    "cmds_filename_mpnn = \"commands_mpnn\"\n",
    "\n",
    "with open(cmds_filename_mpnn, \"w\") as file:\n",
    "    for T in MPNN_temperatures:\n",
    "        for f in diffused_backbones_good:\n",
    "            commands_mpnn += f\"{PYTHON['proteinMPNN']} {proteinMPNN_script} \"\n",
    "            commands_mpnn += f\"--model_type protein_mpnn --ligand_mpnn_use_atom_context 0 \"\n",
    "            commands_mpnn += \"--fixed_residues_multi masked_pos.jsonl --out_folder . \"\n",
    "            commands_mpnn += f\"--number_of_batches {MPNN_outputs_per_temperature} --temperature {T} \"\n",
    "            commands_mpnn += f\"--omit_AA {MPNN_omit_AAs} --pdb_path {f} \"\n",
    "            commands_mpnn += f\"--checkpoint_protein_mpnn {SCRIPT_DIR}/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\\n\"\n",
    "    file.write(commands_mpnn)\n",
    "\n",
    "print(\"Example MPNN command:\")\n",
    "print(commands_mpnn)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "41acc375-69a3-46eb-b30e-0434b60b8888",
   "metadata": {},
   "outputs": [],
   "source": [
    "if not os.path.exists(MPNN_DIR+\"/.done\"):\n",
    "    print(\"Running...\")\n",
    "    p = subprocess.Popen([\"bash\", cmds_filename_mpnn], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n",
    "    print(output)\n",
    "    print(err)\n",
    "    print(\"Done\")\n",
    "else:\n",
    "    print(\"Done file found - skipping\")\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "1f793273",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with diffusion and happy with the outputs then mark it as done\n",
    "MPNN_DIR = f\"{WDIR}/1_proteinmpnn\"\n",
    "os.chdir(MPNN_DIR)\n",
    "\n",
    "if not os.path.exists(MPNN_DIR+\"/.done\"):\n",
    "    with open(f\"{MPNN_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "97333709",
   "metadata": {
    "scrolled": true
   },
   "source": [
    "## 2: Running AlphaFold2\n",
    "Performing AF2 single sequence predictions.<br>\n",
    "By default only using AF2 model 4 with 3 recycles. For more complicated folds 10+ recycles might be necessary."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "065029b2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "A total on 705 sequences will be predicted.\n",
      "0 duplicate sequences removed.\n",
      "100 sequences of 133 length.\n",
      "30 sequences of 133 length.\n",
      "25 sequences of 121 length.\n",
      "25 sequences of 123 length.\n",
      "25 sequences of 112 length.\n",
      "50 sequences of 131 length.\n",
      "25 sequences of 105 length.\n",
      "35 sequences of 127 length.\n",
      "25 sequences of 139 length.\n",
      "20 sequences of 107 length.\n",
      "35 sequences of 138 length.\n",
      "50 sequences of 128 length.\n",
      "20 sequences of 136 length.\n",
      "30 sequences of 135 length.\n",
      "20 sequences of 137 length.\n",
      "25 sequences of 134 length.\n",
      "35 sequences of 113 length.\n",
      "85 sequences of 120 length.\n",
      "25 sequences of 126 length.\n",
      "5 sequences of 119 length.\n",
      "15 sequences of 132 length.\n"
     ]
    }
   ],
   "source": [
    "os.chdir(WDIR)\n",
    "\n",
    "AF2_DIR = f\"{WDIR}/2_af2\"\n",
    "os.makedirs(AF2_DIR, exist_ok=True)\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "### First collecting MPNN outputs and creating FASTA files for AF2 input\n",
    "mpnn_fasta = utils.parse_fasta_files(glob.glob(f\"{MPNN_DIR}/seqs/*.fa\"))\n",
    "mpnn_fasta = {k: seq.strip() for k, seq in mpnn_fasta.items() if \"model_path\" not in k}  # excluding the diffused poly-A sequence\n",
    "# Giving sequences unique names based on input PDB name, temperature, and sequence identifier\n",
    "mpnn_fasta = {k.split(\",\")[0]+\"_\"+k.split(\",\")[2].replace(\" T=\", \"T\")+\"_0_\"+k.split(\",\")[1].replace(\" id=\", \"\"): seq for k, seq in mpnn_fasta.items()}\n",
    "\n",
    "print(f\"A total on {len(mpnn_fasta)} sequences will be predicted.\")\n",
    "\n",
    "## Splitting the MPNN sequences based on length\n",
    "## and grouping them in smaller batches for each AF2 job\n",
    "## Use group size of >40 when running on GPU. Also depends on how many sequences and resources you have.\n",
    "\n",
    "SEQUENCES_PER_AF2_JOB = 5  # CPU\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    SEQUENCES_PER_AF2_JOB = 100  # GPU\n",
    "mpnn_fasta_split = utils.split_fasta_based_on_length(mpnn_fasta, SEQUENCES_PER_AF2_JOB, write_files=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "acf52a6b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AF2 command:\n",
      "\n",
      "#!/bin/sh\n",
      "export CUDA_HOME=/usr/local/cuda-12\n",
      "export PATH=$CUDA_HOME/bin:$PATH\n",
      "export LD_LIBRARY_PATH=$CUDA_HOME/lib64:$LD_LIBRARY_PATH\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 134aa_0.fasta --scorefile 134aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 132aa_0.fasta --scorefile 132aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 120aa_0.fasta --scorefile 120aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 131aa_0.fasta --scorefile 131aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 121aa_0.fasta --scorefile 121aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 127aa_0.fasta --scorefile 127aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 133aa_1.fasta --scorefile 133aa_1.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 113aa_0.fasta --scorefile 113aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 133aa_0.fasta --scorefile 133aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 128aa_0.fasta --scorefile 128aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 123aa_0.fasta --scorefile 123aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 126aa_0.fasta --scorefile 126aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 107aa_0.fasta --scorefile 107aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 139aa_0.fasta --scorefile 139aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 136aa_0.fasta --scorefile 136aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 135aa_0.fasta --scorefile 135aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 138aa_0.fasta --scorefile 138aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 112aa_0.fasta --scorefile 112aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 137aa_0.fasta --scorefile 137aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 105aa_0.fasta --scorefile 105aa_0.csv\n",
      "/home/ubuntu/anaconda3/envs/mlfold/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/af2/af2.py --af-nrecycles 3 --af-models 4 --fasta 119aa_0.fasta --scorefile 119aa_0.csv\n",
      "\n",
      "b'Setting up models took 5.986 seconds.\\n/tmp/2ioflgu4\\nSequence 0 completed in 123.1 sec with 1 models; lDDT=78.417\\nSequence 1 completed in 9.1 sec with 1 models; lDDT=55.068\\nSequence 2 completed in 9.1 sec with 1 models; lDDT=64.651\\nSequence 3 completed in 9.1 sec with 1 models; lDDT=55.308\\nSequence 4 completed in 9.6 sec with 1 models; lDDT=62.807\\nSequence 5 completed in 9.2 sec with 1 models; lDDT=58.530\\nSequence 6 completed in 9.2 sec with 1 models; lDDT=53.186\\nSequence 7 completed in 9.2 sec with 1 models; lDDT=52.607\\nSequence 8 completed in 9.1 sec with 1 models; lDDT=41.859\\nSequence 9 completed in 9.2 sec with 1 models; lDDT=53.140\\nSequence 10 completed in 9.2 sec with 1 models; lDDT=75.186\\nSequence 11 completed in 9.1 sec with 1 models; lDDT=52.148\\nSequence 12 completed in 9.2 sec with 1 models; lDDT=70.419\\nSequence 13 completed in 9.2 sec with 1 models; lDDT=47.960\\nSequence 14 completed in 9.1 sec with 1 models; lDDT=52.288\\nSequence 15 completed in 9.1 sec with 1 models; lDDT=60.764\\nSequence 16 completed in 9.2 sec with 1 models; lDDT=67.317\\nSequence 17 completed in 9.2 sec with 1 models; lDDT=73.521\\nSequence 18 completed in 10.1 sec with 1 models; lDDT=60.116\\nSequence 19 completed in 9.3 sec with 1 models; lDDT=55.625\\nSequence 20 completed in 9.2 sec with 1 models; lDDT=58.658\\nSequence 21 completed in 9.3 sec with 1 models; lDDT=79.818\\nSequence 22 completed in 9.2 sec with 1 models; lDDT=58.817\\nSequence 23 completed in 9.3 sec with 1 models; lDDT=69.443\\nSequence 24 completed in 9.2 sec with 1 models; lDDT=62.338\\nDone with 25 sequences. 345.063 sec.\\nSetting up models took 3.378 seconds.\\n/tmp/kchomtaa\\nSequence 0 completed in 102.3 sec with 1 models; lDDT=60.004\\nSequence 1 completed in 8.8 sec with 1 models; lDDT=63.125\\nSequence 2 completed in 8.9 sec with 1 models; lDDT=65.058\\nSequence 3 completed in 9.2 sec with 1 models; lDDT=87.254\\nSequence 4 completed in 8.8 sec with 1 models; lDDT=69.564\\nSequence 5 completed in 8.8 sec with 1 models; lDDT=49.963\\nSequence 6 completed in 8.8 sec with 1 models; lDDT=67.465\\nSequence 7 completed in 8.9 sec with 1 models; lDDT=82.749\\nSequence 8 completed in 8.7 sec with 1 models; lDDT=63.523\\nSequence 9 completed in 8.7 sec with 1 models; lDDT=80.985\\nSequence 10 completed in 8.7 sec with 1 models; lDDT=67.141\\nSequence 11 completed in 8.8 sec with 1 models; lDDT=56.049\\nSequence 12 completed in 8.8 sec with 1 models; lDDT=50.873\\nSequence 13 completed in 8.9 sec with 1 models; lDDT=64.318\\nSequence 14 completed in 8.8 sec with 1 models; lDDT=62.831\\nDone with 15 sequences. 225.964 sec.\\nSetting up models took 3.386 seconds.\\n/tmp/vudcddk8\\nSequence 0 completed in 120.3 sec with 1 models; lDDT=79.559\\nSequence 1 completed in 7.8 sec with 1 models; lDDT=50.375\\nSequence 2 completed in 7.3 sec with 1 models; lDDT=73.047\\nSequence 3 completed in 7.4 sec with 1 models; lDDT=82.462\\nSequence 4 completed in 7.3 sec with 1 models; lDDT=61.979\\nSequence 5 completed in 7.1 sec with 1 models; lDDT=61.075\\nSequence 6 completed in 7.1 sec with 1 models; lDDT=70.718\\nSequence 7 completed in 7.2 sec with 1 models; lDDT=67.063\\nSequence 8 completed in 7.2 sec with 1 models; lDDT=69.542\\nSequence 9 completed in 7.1 sec with 1 models; lDDT=65.413\\nSequence 10 completed in 7.3 sec with 1 models; lDDT=81.863\\nSequence 11 completed in 7.1 sec with 1 models; lDDT=50.188\\nSequence 12 completed in 7.1 sec with 1 models; lDDT=55.514\\nSequence 13 completed in 7.2 sec with 1 models; lDDT=62.193\\nSequence 14 completed in 8.0 sec with 1 models; lDDT=61.675\\nSequence 15 completed in 7.3 sec with 1 models; lDDT=58.470\\nSequence 16 completed in 7.2 sec with 1 models; lDDT=84.398\\nSequence 17 completed in 7.2 sec with 1 models; lDDT=52.196\\nSequence 18 completed in 7.3 sec with 1 models; lDDT=46.265\\nSequence 19 completed in 7.2 sec with 1 models; lDDT=66.309\\nSequence 20 completed in 7.2 sec with 1 models; lDDT=71.314\\nSequence 21 completed in 7.3 sec with 1 models; lDDT=79.477\\nSequence 22 completed in 7.2 sec with 1 models; lDDT=59.906\\nSequence 23 completed in 7.3 sec with 1 models; lDDT=62.194\\nSequence 24 completed in 7.2 sec with 1 models; lDDT=71.505\\nSequence 25 completed in 7.2 sec with 1 models; lDDT=62.392\\nSequence 26 completed in 7.3 sec with 1 models; lDDT=66.221\\nSequence 27 completed in 8.1 sec with 1 models; lDDT=45.505\\nSequence 28 completed in 7.4 sec with 1 models; lDDT=67.412\\nSequence 29 completed in 7.2 sec with 1 models; lDDT=66.017\\nSequence 30 completed in 7.2 sec with 1 models; lDDT=80.546\\nSequence 31 completed in 7.2 sec with 1 models; lDDT=71.847\\nSequence 32 completed in 7.3 sec with 1 models; lDDT=78.910\\nSequence 33 completed in 7.3 sec with 1 models; lDDT=79.581\\nSequence 34 completed in 7.1 sec with 1 models; lDDT=68.516\\nSequence 35 completed in 7.4 sec with 1 models; lDDT=60.115\\nSequence 36 completed in 7.2 sec with 1 models; lDDT=60.174\\nSequence 37 completed in 7.2 sec with 1 models; lDDT=60.769\\nSequence 38 completed in 7.3 sec with 1 models; lDDT=81.439\\nSequence 39 completed in 7.4 sec with 1 models; lDDT=78.112\\nSequence 40 completed in 8.1 sec with 1 models; lDDT=73.262\\nSequence 41 completed in 7.3 sec with 1 models; lDDT=68.495\\nSequence 42 completed in 7.3 sec with 1 models; lDDT=74.265\\nSequence 43 completed in 7.4 sec with 1 models; lDDT=53.533\\nSequence 44 completed in 7.4 sec with 1 models; lDDT=66.230\\nSequence 45 completed in 7.2 sec with 1 models; lDDT=56.471\\nSequence 46 completed in 7.2 sec with 1 models; lDDT=59.627\\nSequence 47 completed in 7.2 sec with 1 models; lDDT=62.074\\nSequence 48 completed in 7.3 sec with 1 models; lDDT=85.199\\nSequence 49 completed in 7.2 sec with 1 models; lDDT=77.361\\nSequence 50 completed in 7.3 sec with 1 models; lDDT=62.975\\nSequence 51 completed in 7.2 sec with 1 models; lDDT=61.050\\nSequence 52 completed in 7.1 sec with 1 models; lDDT=58.443\\nSequence 53 completed in 8.1 sec with 1 models; lDDT=50.774\\nSequence 54 completed in 7.2 sec with 1 models; lDDT=75.918\\nSequence 55 completed in 7.3 sec with 1 models; lDDT=55.094\\nSequence 56 completed in 7.3 sec with 1 models; lDDT=71.986\\nSequence 57 completed in 7.4 sec with 1 models; lDDT=66.980\\nSequence 58 completed in 7.3 sec with 1 models; lDDT=72.261\\nSequence 59 completed in 7.2 sec with 1 models; lDDT=60.702\\nSequence 60 completed in 7.4 sec with 1 models; lDDT=65.441\\nSequence 61 completed in 7.3 sec with 1 models; lDDT=68.559\\nSequence 62 completed in 7.2 sec with 1 models; lDDT=65.532\\nSequence 63 completed in 7.2 sec with 1 models; lDDT=60.052\\nSequence 64 completed in 7.3 sec with 1 models; lDDT=78.246\\nSequence 65 completed in 7.2 sec with 1 models; lDDT=71.618\\nSequence 66 completed in 7.2 sec with 1 models; lDDT=76.585\\nSequence 67 completed in 8.0 sec with 1 models; lDDT=72.618\\nSequence 68 completed in 7.3 sec with 1 models; lDDT=71.817\\nSequence 69 completed in 7.3 sec with 1 models; lDDT=76.059\\nSequence 70 completed in 7.2 sec with 1 models; lDDT=58.510\\nSequence 71 completed in 7.3 sec with 1 models; lDDT=88.846\\nSequence 72 completed in 7.2 sec with 1 models; lDDT=54.911\\nSequence 73 completed in 7.2 sec with 1 models; lDDT=51.774\\nSequence 74 completed in 7.3 sec with 1 models; lDDT=57.560\\nSequence 75 completed in 7.2 sec with 1 models; lDDT=69.625\\nSequence 76 completed in 7.2 sec with 1 models; lDDT=47.352\\nSequence 77 completed in 7.2 sec with 1 models; lDDT=85.423\\nSequence 78 completed in 7.1 sec with 1 models; lDDT=57.858\\nSequence 79 completed in 8.1 sec with 1 models; lDDT=74.503\\nSequence 80 completed in 7.4 sec with 1 models; lDDT=57.688\\nSequence 81 completed in 7.3 sec with 1 models; lDDT=83.528\\nSequence 82 completed in 7.3 sec with 1 models; lDDT=80.250\\nSequence 83 completed in 7.2 sec with 1 models; lDDT=54.359\\nSequence 84 completed in 7.3 sec with 1 models; lDDT=63.313\\nDone with 85 sequences. 734.240 sec.\\nSetting up models took 3.369 seconds.\\n/tmp/apf0kwzk\\nSequence 0 completed in 121.5 sec with 1 models; lDDT=53.540\\nSequence 1 completed in 9.0 sec with 1 models; lDDT=70.867\\nSequence 2 completed in 8.8 sec with 1 models; lDDT=76.718\\nSequence 3 completed in 8.7 sec with 1 models; lDDT=59.400\\nSequence 4 completed in 9.4 sec with 1 models; lDDT=90.981\\nSequence 5 completed in 8.9 sec with 1 models; lDDT=54.257\\nSequence 6 completed in 8.8 sec with 1 models; lDDT=75.091\\nSequence 7 completed in 8.9 sec with 1 models; lDDT=80.908\\nSequence 8 completed in 8.8 sec with 1 models; lDDT=63.742\\nSequence 9 completed in 8.9 sec with 1 models; lDDT=77.297\\nSequence 10 completed in 8.9 sec with 1 models; lDDT=74.178\\nSequence 11 completed in 8.9 sec with 1 models; lDDT=64.999\\nSequence 12 completed in 8.7 sec with 1 models; lDDT=55.640\\nSequence 13 completed in 8.9 sec with 1 models; lDDT=76.839\\nSequence 14 completed in 8.9 sec with 1 models; lDDT=82.883\\nSequence 15 completed in 8.8 sec with 1 models; lDDT=70.316\\nSequence 16 completed in 8.8 sec with 1 models; lDDT=77.627\\nSequence 17 completed in 8.8 sec with 1 models; lDDT=84.528\\nSequence 18 completed in 9.7 sec with 1 models; lDDT=68.735\\nSequence 19 completed in 8.8 sec with 1 models; lDDT=75.538\\nSequence 20 completed in 9.0 sec with 1 models; lDDT=86.934\\nSequence 21 completed in 9.0 sec with 1 models; lDDT=83.701\\nSequence 22 completed in 8.9 sec with 1 models; lDDT=87.186\\nSequence 23 completed in 9.0 sec with 1 models; lDDT=88.671\\nSequence 24 completed in 9.0 sec with 1 models; lDDT=76.522\\nSequence 25 completed in 8.8 sec with 1 models; lDDT=75.479\\nSequence 26 completed in 8.8 sec with 1 models; lDDT=63.653\\nSequence 27 completed in 8.9 sec with 1 models; lDDT=77.517\\nSequence 28 completed in 9.0 sec with 1 models; lDDT=84.804\\nSequence 29 completed in 8.9 sec with 1 models; lDDT=70.530\\nSequence 30 completed in 8.7 sec with 1 models; lDDT=57.861\\nSequence 31 completed in 9.7 sec with 1 models; lDDT=75.404\\nSequence 32 completed in 9.0 sec with 1 models; lDDT=80.354\\nSequence 33 completed in 8.9 sec with 1 models; lDDT=75.578\\nSequence 34 completed in 8.8 sec with 1 models; lDDT=60.570\\nSequence 35 completed in 9.0 sec with 1 models; lDDT=71.691\\nSequence 36 completed in 8.9 sec with 1 models; lDDT=79.398\\nSequence 37 completed in 8.8 sec with 1 models; lDDT=60.422\\nSequence 38 completed in 8.9 sec with 1 models; lDDT=66.758\\nSequence 39 completed in 8.9 sec with 1 models; lDDT=59.157\\nSequence 40 completed in 8.8 sec with 1 models; lDDT=80.251\\nSequence 41 completed in 8.8 sec with 1 models; lDDT=69.890\\nSequence 42 completed in 8.7 sec with 1 models; lDDT=57.226\\nSequence 43 completed in 8.8 sec with 1 models; lDDT=60.567\\nSequence 44 completed in 8.8 sec with 1 models; lDDT=54.665\\nSequence 45 completed in 9.8 sec with 1 models; lDDT=76.231\\nSequence 46 completed in 8.9 sec with 1 models; lDDT=80.471\\nSequence 47 completed in 9.0 sec with 1 models; lDDT=69.254\\nSequence 48 completed in 8.9 sec with 1 models; lDDT=77.808\\nSequence 49 completed in 8.8 sec with 1 models; lDDT=86.520\\nDone with 50 sequences. 559.036 sec.\\nSetting up models took 3.372 seconds.\\n/tmp/6189m8_j\\nSequence 0 completed in 140.0 sec with 1 models; lDDT=79.098\\nSequence 1 completed in 7.7 sec with 1 models; lDDT=67.280\\nSequence 2 completed in 7.6 sec with 1 models; lDDT=65.927\\nSequence 3 completed in 7.5 sec with 1 models; lDDT=73.029\\nSequence 4 completed in 7.5 sec with 1 models; lDDT=87.859\\nSequence 5 completed in 7.6 sec with 1 models; lDDT=47.698\\nSequence 6 completed in 8.1 sec with 1 models; lDDT=51.919\\nSequence 7 completed in 7.6 sec with 1 models; lDDT=78.216\\nSequence 8 completed in 7.7 sec with 1 models; lDDT=56.310\\nSequence 9 completed in 7.6 sec with 1 models; lDDT=79.281\\nSequence 10 completed in 7.5 sec with 1 models; lDDT=75.741\\nSequence 11 completed in 7.6 sec with 1 models; lDDT=87.868\\nSequence 12 completed in 7.6 sec with 1 models; lDDT=83.617\\nSequence 13 completed in 7.5 sec with 1 models; lDDT=58.745\\nSequence 14 completed in 7.6 sec with 1 models; lDDT=75.432\\nSequence 15 completed in 7.6 sec with 1 models; lDDT=85.324\\nSequence 16 completed in 7.6 sec with 1 models; lDDT=49.256\\nSequence 17 completed in 7.5 sec with 1 models; lDDT=60.036\\nSequence 18 completed in 7.6 sec with 1 models; lDDT=51.178\\nSequence 19 completed in 7.6 sec with 1 models; lDDT=50.114\\nSequence 20 completed in 8.5 sec with 1 models; lDDT=79.844\\nSequence 21 completed in 7.7 sec with 1 models; lDDT=71.036\\nSequence 22 completed in 7.6 sec with 1 models; lDDT=71.068\\nSequence 23 completed in 7.6 sec with 1 models; lDDT=57.246\\nSequence 24 completed in 7.6 sec with 1 models; lDDT=49.956\\nDone with 25 sequences. 323.496 sec.\\nSetting up models took 3.380 seconds.\\n/tmp/j5q_84r7\\nSequence 0 completed in 146.5 sec with 1 models; lDDT=82.078\\nSequence 1 completed in 8.1 sec with 1 models; lDDT=87.086\\nSequence 2 completed in 8.1 sec with 1 models; lDDT=89.911\\nSequence 3 completed in 8.1 sec with 1 models; lDDT=86.113\\nSequence 4 completed in 8.1 sec with 1 models; lDDT=87.322\\nSequence 5 completed in 8.2 sec with 1 models; lDDT=82.298\\nSequence 6 completed in 8.6 sec with 1 models; lDDT=80.943\\nSequence 7 completed in 8.2 sec with 1 models; lDDT=88.749\\nSequence 8 completed in 8.1 sec with 1 models; lDDT=69.056\\nSequence 9 completed in 8.2 sec with 1 models; lDDT=72.828\\nSequence 10 completed in 8.1 sec with 1 models; lDDT=79.088\\nSequence 11 completed in 8.1 sec with 1 models; lDDT=86.442\\nSequence 12 completed in 8.1 sec with 1 models; lDDT=90.771\\nSequence 13 completed in 8.1 sec with 1 models; lDDT=85.066\\nSequence 14 completed in 8.0 sec with 1 models; lDDT=83.346\\nSequence 15 completed in 8.1 sec with 1 models; lDDT=87.508\\nSequence 16 completed in 8.2 sec with 1 models; lDDT=88.070\\nSequence 17 completed in 8.0 sec with 1 models; lDDT=89.121\\nSequence 18 completed in 8.1 sec with 1 models; lDDT=88.368\\nSequence 19 completed in 8.1 sec with 1 models; lDDT=89.348\\nSequence 20 completed in 8.9 sec with 1 models; lDDT=85.970\\nSequence 21 completed in 8.1 sec with 1 models; lDDT=89.184\\nSequence 22 completed in 8.1 sec with 1 models; lDDT=88.626\\nSequence 23 completed in 8.1 sec with 1 models; lDDT=84.175\\nSequence 24 completed in 8.2 sec with 1 models; lDDT=90.284\\nSequence 25 completed in 8.1 sec with 1 models; lDDT=80.008\\nSequence 26 completed in 8.1 sec with 1 models; lDDT=66.035\\nSequence 27 completed in 8.0 sec with 1 models; lDDT=79.423\\nSequence 28 completed in 8.1 sec with 1 models; lDDT=77.455\\nSequence 29 completed in 8.0 sec with 1 models; lDDT=72.151\\nSequence 30 completed in 8.1 sec with 1 models; lDDT=68.107\\nSequence 31 completed in 8.0 sec with 1 models; lDDT=67.425\\nSequence 32 completed in 8.1 sec with 1 models; lDDT=60.456\\nSequence 33 completed in 8.1 sec with 1 models; lDDT=63.713\\nSequence 34 completed in 8.9 sec with 1 models; lDDT=63.902\\nDone with 35 sequences. 424.216 sec.\\nSetting up models took 3.323 seconds.\\n/tmp/954lfsoa\\nSequence 0 completed in 120.8 sec with 1 models; lDDT=80.714\\nSequence 1 completed in 9.0 sec with 1 models; lDDT=82.285\\nSequence 2 completed in 9.0 sec with 1 models; lDDT=82.729\\nSequence 3 completed in 9.0 sec with 1 models; lDDT=83.921\\nSequence 4 completed in 9.5 sec with 1 models; lDDT=89.540\\nSequence 5 completed in 9.0 sec with 1 models; lDDT=52.408\\nSequence 6 completed in 9.1 sec with 1 models; lDDT=69.936\\nSequence 7 completed in 9.1 sec with 1 models; lDDT=77.997\\nSequence 8 completed in 9.1 sec with 1 models; lDDT=71.649\\nSequence 9 completed in 9.0 sec with 1 models; lDDT=73.130\\nSequence 10 completed in 9.0 sec with 1 models; lDDT=59.773\\nSequence 11 completed in 9.0 sec with 1 models; lDDT=84.126\\nSequence 12 completed in 8.9 sec with 1 models; lDDT=85.290\\nSequence 13 completed in 8.9 sec with 1 models; lDDT=84.612\\nSequence 14 completed in 8.9 sec with 1 models; lDDT=85.850\\nSequence 15 completed in 8.9 sec with 1 models; lDDT=80.460\\nSequence 16 completed in 8.9 sec with 1 models; lDDT=68.405\\nSequence 17 completed in 8.9 sec with 1 models; lDDT=84.020\\nSequence 18 completed in 9.8 sec with 1 models; lDDT=86.729\\nSequence 19 completed in 9.0 sec with 1 models; lDDT=85.847\\nSequence 20 completed in 9.0 sec with 1 models; lDDT=51.836\\nSequence 21 completed in 9.0 sec with 1 models; lDDT=53.474\\nSequence 22 completed in 8.9 sec with 1 models; lDDT=42.790\\nSequence 23 completed in 8.9 sec with 1 models; lDDT=58.992\\nSequence 24 completed in 8.9 sec with 1 models; lDDT=44.634\\nSequence 25 completed in 8.8 sec with 1 models; lDDT=68.025\\nSequence 26 completed in 9.0 sec with 1 models; lDDT=60.243\\nSequence 27 completed in 9.0 sec with 1 models; lDDT=72.401\\nSequence 28 completed in 8.9 sec with 1 models; lDDT=83.878\\nSequence 29 completed in 8.9 sec with 1 models; lDDT=82.594\\nDone with 30 sequences. 381.909 sec.\\nSetting up models took 3.375 seconds.\\n/tmp/m5kk9tfk\\nSequence 0 completed in 136.7 sec with 1 models; lDDT=82.540\\nSequence 1 completed in 6.7 sec with 1 models; lDDT=79.985\\nSequence 2 completed in 6.7 sec with 1 models; lDDT=61.487\\nSequence 3 completed in 6.6 sec with 1 models; lDDT=44.014\\nSequence 4 completed in 7.1 sec with 1 models; lDDT=74.936\\nSequence 5 completed in 6.7 sec with 1 models; lDDT=72.944\\nSequence 6 completed in 6.7 sec with 1 models; lDDT=54.411\\nSequence 7 completed in 6.6 sec with 1 models; lDDT=48.526\\nSequence 8 completed in 6.6 sec with 1 models; lDDT=46.082\\nSequence 9 completed in 6.7 sec with 1 models; lDDT=42.861\\nSequence 10 completed in 6.7 sec with 1 models; lDDT=81.879\\nSequence 11 completed in 6.7 sec with 1 models; lDDT=81.773\\nSequence 12 completed in 6.7 sec with 1 models; lDDT=74.417\\nSequence 13 completed in 6.6 sec with 1 models; lDDT=77.604\\nSequence 14 completed in 6.6 sec with 1 models; lDDT=86.819\\nSequence 15 completed in 6.6 sec with 1 models; lDDT=44.353\\nSequence 16 completed in 6.7 sec with 1 models; lDDT=60.247\\nSequence 17 completed in 7.5 sec with 1 models; lDDT=77.313\\nSequence 18 completed in 6.7 sec with 1 models; lDDT=76.324\\nSequence 19 completed in 6.7 sec with 1 models; lDDT=66.833\\nSequence 20 completed in 6.7 sec with 1 models; lDDT=61.611\\nSequence 21 completed in 6.6 sec with 1 models; lDDT=62.046\\nSequence 22 completed in 6.7 sec with 1 models; lDDT=63.522\\nSequence 23 completed in 6.7 sec with 1 models; lDDT=75.480\\nSequence 24 completed in 6.6 sec with 1 models; lDDT=77.258\\nSequence 25 completed in 6.6 sec with 1 models; lDDT=66.403\\nSequence 26 completed in 6.7 sec with 1 models; lDDT=43.979\\nSequence 27 completed in 6.6 sec with 1 models; lDDT=48.338\\nSequence 28 completed in 6.7 sec with 1 models; lDDT=77.458\\nSequence 29 completed in 6.6 sec with 1 models; lDDT=83.039\\nSequence 30 completed in 6.6 sec with 1 models; lDDT=65.037\\nSequence 31 completed in 7.7 sec with 1 models; lDDT=73.479\\nSequence 32 completed in 6.8 sec with 1 models; lDDT=78.425\\nSequence 33 completed in 6.8 sec with 1 models; lDDT=80.413\\nSequence 34 completed in 6.8 sec with 1 models; lDDT=88.664\\nDone with 35 sequences. 366.027 sec.\\nSetting up models took 3.425 seconds.\\n/tmp/urm7w4ns\\nSequence 0 completed in 120.2 sec with 1 models; lDDT=57.243\\nSequence 1 completed in 8.9 sec with 1 models; lDDT=45.107\\nSequence 2 completed in 9.0 sec with 1 models; lDDT=46.214\\nSequence 3 completed in 9.3 sec with 1 models; lDDT=58.426\\nSequence 4 completed in 9.0 sec with 1 models; lDDT=45.350\\nSequence 5 completed in 9.0 sec with 1 models; lDDT=82.077\\nSequence 6 completed in 9.0 sec with 1 models; lDDT=68.017\\nSequence 7 completed in 9.1 sec with 1 models; lDDT=87.529\\nSequence 8 completed in 8.9 sec with 1 models; lDDT=78.105\\nSequence 9 completed in 8.9 sec with 1 models; lDDT=84.870\\nSequence 10 completed in 8.9 sec with 1 models; lDDT=79.722\\nSequence 11 completed in 9.0 sec with 1 models; lDDT=77.537\\nSequence 12 completed in 8.9 sec with 1 models; lDDT=79.531\\nSequence 13 completed in 8.9 sec with 1 models; lDDT=87.664\\nSequence 14 completed in 8.9 sec with 1 models; lDDT=77.730\\nSequence 15 completed in 8.9 sec with 1 models; lDDT=73.540\\nSequence 16 completed in 8.9 sec with 1 models; lDDT=60.335\\nSequence 17 completed in 9.8 sec with 1 models; lDDT=75.680\\nSequence 18 completed in 9.0 sec with 1 models; lDDT=78.474\\nSequence 19 completed in 8.9 sec with 1 models; lDDT=82.715\\nSequence 20 completed in 9.0 sec with 1 models; lDDT=81.479\\nSequence 21 completed in 9.0 sec with 1 models; lDDT=82.408\\nSequence 22 completed in 8.9 sec with 1 models; lDDT=77.568\\nSequence 23 completed in 8.9 sec with 1 models; lDDT=80.444\\nSequence 24 completed in 8.9 sec with 1 models; lDDT=78.303\\nSequence 25 completed in 9.0 sec with 1 models; lDDT=73.506\\nSequence 26 completed in 9.0 sec with 1 models; lDDT=78.278\\nSequence 27 completed in 9.0 sec with 1 models; lDDT=66.563\\nSequence 28 completed in 9.0 sec with 1 models; lDDT=74.977\\nSequence 29 completed in 8.9 sec with 1 models; lDDT=69.667\\nSequence 30 completed in 8.7 sec with 1 models; lDDT=62.627\\nSequence 31 completed in 9.7 sec with 1 models; lDDT=68.061\\nSequence 32 completed in 8.9 sec with 1 models; lDDT=72.410\\nSequence 33 completed in 8.9 sec with 1 models; lDDT=70.928\\nSequence 34 completed in 8.9 sec with 1 models; lDDT=64.066\\nSequence 35 completed in 8.9 sec with 1 models; lDDT=81.978\\nSequence 36 completed in 9.0 sec with 1 models; lDDT=85.689\\nSequence 37 completed in 9.0 sec with 1 models; lDDT=82.552\\nSequence 38 completed in 8.9 sec with 1 models; lDDT=78.703\\nSequence 39 completed in 8.9 sec with 1 models; lDDT=72.988\\nSequence 40 completed in 8.9 sec with 1 models; lDDT=80.772\\nSequence 41 completed in 8.9 sec with 1 models; lDDT=77.301\\nSequence 42 completed in 9.0 sec with 1 models; lDDT=59.043\\nSequence 43 completed in 8.9 sec with 1 models; lDDT=47.553\\nSequence 44 completed in 9.8 sec with 1 models; lDDT=61.927\\nSequence 45 completed in 8.9 sec with 1 models; lDDT=77.663\\nSequence 46 completed in 9.0 sec with 1 models; lDDT=73.523\\nSequence 47 completed in 8.9 sec with 1 models; lDDT=81.671\\nSequence 48 completed in 8.9 sec with 1 models; lDDT=66.995\\nSequence 49 completed in 8.9 sec with 1 models; lDDT=74.159\\nSequence 50 completed in 9.1 sec with 1 models; lDDT=70.113\\nSequence 51 completed in 9.1 sec with 1 models; lDDT=72.631\\nSequence 52 completed in 9.0 sec with 1 models; lDDT=69.883\\nSequence 53 completed in 9.1 sec with 1 models; lDDT=85.034\\nSequence 54 completed in 9.0 sec with 1 models; lDDT=77.651\\nSequence 55 completed in 8.9 sec with 1 models; lDDT=50.890\\nSequence 56 completed in 8.9 sec with 1 models; lDDT=56.180\\nSequence 57 completed in 8.9 sec with 1 models; lDDT=55.230\\nSequence 58 completed in 9.8 sec with 1 models; lDDT=51.153\\nSequence 59 completed in 8.9 sec with 1 models; lDDT=52.250\\nSequence 60 completed in 9.0 sec with 1 models; lDDT=68.492\\nSequence 61 completed in 8.9 sec with 1 models; lDDT=87.688\\nSequence 62 completed in 8.9 sec with 1 models; lDDT=72.445\\nSequence 63 completed in 8.8 sec with 1 models; lDDT=74.948\\nSequence 64 completed in 8.9 sec with 1 models; lDDT=70.371\\nSequence 65 completed in 8.9 sec with 1 models; lDDT=77.607\\nSequence 66 completed in 9.0 sec with 1 models; lDDT=71.685\\nSequence 67 completed in 9.0 sec with 1 models; lDDT=51.822\\nSequence 68 completed in 9.0 sec with 1 models; lDDT=78.739\\nSequence 69 completed in 8.9 sec with 1 models; lDDT=72.938\\nSequence 70 completed in 8.9 sec with 1 models; lDDT=75.303\\nSequence 71 completed in 8.8 sec with 1 models; lDDT=74.667\\nSequence 72 completed in 9.8 sec with 1 models; lDDT=77.683\\nSequence 73 completed in 9.0 sec with 1 models; lDDT=72.981\\nSequence 74 completed in 9.0 sec with 1 models; lDDT=69.702\\nSequence 75 completed in 8.9 sec with 1 models; lDDT=53.916\\nSequence 76 completed in 8.9 sec with 1 models; lDDT=56.575\\nSequence 77 completed in 8.9 sec with 1 models; lDDT=48.078\\nSequence 78 completed in 9.0 sec with 1 models; lDDT=56.371\\nSequence 79 completed in 9.0 sec with 1 models; lDDT=49.647\\nSequence 80 completed in 8.8 sec with 1 models; lDDT=78.980\\nSequence 81 completed in 8.9 sec with 1 models; lDDT=87.928\\nSequence 82 completed in 8.9 sec with 1 models; lDDT=89.145\\nSequence 83 completed in 8.9 sec with 1 models; lDDT=77.564\\nSequence 84 completed in 8.9 sec with 1 models; lDDT=85.856\\nSequence 85 completed in 8.8 sec with 1 models; lDDT=50.501\\nSequence 86 completed in 9.7 sec with 1 models; lDDT=43.545\\nSequence 87 completed in 8.9 sec with 1 models; lDDT=52.068\\nSequence 88 completed in 8.9 sec with 1 models; lDDT=44.999\\nSequence 89 completed in 9.0 sec with 1 models; lDDT=44.317\\nSequence 90 completed in 9.0 sec with 1 models; lDDT=65.660\\nSequence 91 completed in 9.0 sec with 1 models; lDDT=73.404\\nSequence 92 completed in 9.0 sec with 1 models; lDDT=74.002\\nSequence 93 completed in 9.0 sec with 1 models; lDDT=72.990\\nSequence 94 completed in 8.9 sec with 1 models; lDDT=70.874\\nSequence 95 completed in 8.9 sec with 1 models; lDDT=73.174\\nSequence 96 completed in 8.9 sec with 1 models; lDDT=70.917\\nSequence 97 completed in 8.8 sec with 1 models; lDDT=79.218\\nSequence 98 completed in 8.9 sec with 1 models; lDDT=82.370\\nSequence 99 completed in 8.8 sec with 1 models; lDDT=72.175\\nDone with 100 sequences. 1010.030 sec.\\nSetting up models took 3.386 seconds.\\n/tmp/r9t4q7x9\\nSequence 0 completed in 117.1 sec with 1 models; lDDT=65.194\\nSequence 1 completed in 8.1 sec with 1 models; lDDT=66.566\\nSequence 2 completed in 8.1 sec with 1 models; lDDT=62.977\\nSequence 3 completed in 8.1 sec with 1 models; lDDT=59.610\\nSequence 4 completed in 8.1 sec with 1 models; lDDT=48.429\\nSequence 5 completed in 8.0 sec with 1 models; lDDT=60.875\\nSequence 6 completed in 8.1 sec with 1 models; lDDT=71.437\\nSequence 7 completed in 8.1 sec with 1 models; lDDT=67.778\\nSequence 8 completed in 8.1 sec with 1 models; lDDT=76.121\\nSequence 9 completed in 8.0 sec with 1 models; lDDT=68.131\\nSequence 10 completed in 8.2 sec with 1 models; lDDT=74.130\\nSequence 11 completed in 8.1 sec with 1 models; lDDT=65.656\\nSequence 12 completed in 8.8 sec with 1 models; lDDT=57.756\\nSequence 13 completed in 8.1 sec with 1 models; lDDT=72.659\\nSequence 14 completed in 8.2 sec with 1 models; lDDT=61.029\\nSequence 15 completed in 8.2 sec with 1 models; lDDT=44.268\\nSequence 16 completed in 8.1 sec with 1 models; lDDT=60.655\\nSequence 17 completed in 8.0 sec with 1 models; lDDT=51.279\\nSequence 18 completed in 8.1 sec with 1 models; lDDT=48.836\\nSequence 19 completed in 8.0 sec with 1 models; lDDT=55.916\\nSequence 20 completed in 8.1 sec with 1 models; lDDT=51.618\\nSequence 21 completed in 8.0 sec with 1 models; lDDT=55.194\\nSequence 22 completed in 8.0 sec with 1 models; lDDT=59.285\\nSequence 23 completed in 8.1 sec with 1 models; lDDT=69.327\\nSequence 24 completed in 8.0 sec with 1 models; lDDT=55.518\\nSequence 25 completed in 8.0 sec with 1 models; lDDT=70.342\\nSequence 26 completed in 9.0 sec with 1 models; lDDT=81.450\\nSequence 27 completed in 8.1 sec with 1 models; lDDT=64.787\\nSequence 28 completed in 8.1 sec with 1 models; lDDT=78.406\\nSequence 29 completed in 8.1 sec with 1 models; lDDT=63.956\\nSequence 30 completed in 8.0 sec with 1 models; lDDT=44.666\\nSequence 31 completed in 8.1 sec with 1 models; lDDT=48.667\\nSequence 32 completed in 8.1 sec with 1 models; lDDT=53.198\\nSequence 33 completed in 8.0 sec with 1 models; lDDT=55.124\\nSequence 34 completed in 8.0 sec with 1 models; lDDT=53.411\\nSequence 35 completed in 8.0 sec with 1 models; lDDT=55.213\\nSequence 36 completed in 8.0 sec with 1 models; lDDT=51.856\\nSequence 37 completed in 8.0 sec with 1 models; lDDT=72.497\\nSequence 38 completed in 8.9 sec with 1 models; lDDT=77.104\\nSequence 39 completed in 8.1 sec with 1 models; lDDT=53.486\\nSequence 40 completed in 8.2 sec with 1 models; lDDT=51.133\\nSequence 41 completed in 8.1 sec with 1 models; lDDT=66.166\\nSequence 42 completed in 8.0 sec with 1 models; lDDT=55.613\\nSequence 43 completed in 8.2 sec with 1 models; lDDT=87.798\\nSequence 44 completed in 8.1 sec with 1 models; lDDT=72.406\\nSequence 45 completed in 8.1 sec with 1 models; lDDT=49.447\\nSequence 46 completed in 8.0 sec with 1 models; lDDT=59.629\\nSequence 47 completed in 8.1 sec with 1 models; lDDT=49.910\\nSequence 48 completed in 8.1 sec with 1 models; lDDT=49.942\\nSequence 49 completed in 8.1 sec with 1 models; lDDT=54.537\\nDone with 50 sequences. 515.312 sec.\\nSetting up models took 3.375 seconds.\\n/tmp/vvyosxau\\nSequence 0 completed in 142.2 sec with 1 models; lDDT=66.068\\nSequence 1 completed in 7.6 sec with 1 models; lDDT=70.567\\nSequence 2 completed in 7.7 sec with 1 models; lDDT=55.329\\nSequence 3 completed in 7.7 sec with 1 models; lDDT=79.684\\nSequence 4 completed in 7.6 sec with 1 models; lDDT=52.816\\nSequence 5 completed in 7.6 sec with 1 models; lDDT=71.987\\nSequence 6 completed in 8.2 sec with 1 models; lDDT=76.370\\nSequence 7 completed in 7.6 sec with 1 models; lDDT=59.917\\nSequence 8 completed in 7.8 sec with 1 models; lDDT=66.407\\nSequence 9 completed in 7.6 sec with 1 models; lDDT=59.292\\nSequence 10 completed in 7.6 sec with 1 models; lDDT=64.763\\nSequence 11 completed in 7.7 sec with 1 models; lDDT=77.047\\nSequence 12 completed in 7.7 sec with 1 models; lDDT=71.145\\nSequence 13 completed in 7.6 sec with 1 models; lDDT=52.374\\nSequence 14 completed in 7.6 sec with 1 models; lDDT=61.384\\nSequence 15 completed in 7.7 sec with 1 models; lDDT=69.190\\nSequence 16 completed in 7.6 sec with 1 models; lDDT=78.304\\nSequence 17 completed in 7.7 sec with 1 models; lDDT=86.759\\nSequence 18 completed in 7.7 sec with 1 models; lDDT=77.740\\nSequence 19 completed in 7.6 sec with 1 models; lDDT=69.219\\nSequence 20 completed in 8.4 sec with 1 models; lDDT=76.296\\nSequence 21 completed in 7.8 sec with 1 models; lDDT=69.447\\nSequence 22 completed in 7.7 sec with 1 models; lDDT=73.892\\nSequence 23 completed in 7.6 sec with 1 models; lDDT=62.351\\nSequence 24 completed in 7.6 sec with 1 models; lDDT=63.024\\nDone with 25 sequences. 327.410 sec.\\nSetting up models took 3.376 seconds.\\n/tmp/itm8cedi\\nSequence 0 completed in 142.3 sec with 1 models; lDDT=83.371\\nSequence 1 completed in 8.1 sec with 1 models; lDDT=61.838\\nSequence 2 completed in 8.0 sec with 1 models; lDDT=85.860\\nSequence 3 completed in 8.1 sec with 1 models; lDDT=85.759\\nSequence 4 completed in 8.1 sec with 1 models; lDDT=75.641\\nSequence 5 completed in 8.0 sec with 1 models; lDDT=85.004\\nSequence 6 completed in 8.5 sec with 1 models; lDDT=82.471\\nSequence 7 completed in 8.1 sec with 1 models; lDDT=82.111\\nSequence 8 completed in 8.0 sec with 1 models; lDDT=87.983\\nSequence 9 completed in 8.0 sec with 1 models; lDDT=74.242\\nSequence 10 completed in 8.0 sec with 1 models; lDDT=90.766\\nSequence 11 completed in 8.0 sec with 1 models; lDDT=84.347\\nSequence 12 completed in 8.1 sec with 1 models; lDDT=88.898\\nSequence 13 completed in 8.0 sec with 1 models; lDDT=83.465\\nSequence 14 completed in 8.0 sec with 1 models; lDDT=84.800\\nSequence 15 completed in 7.9 sec with 1 models; lDDT=69.892\\nSequence 16 completed in 7.9 sec with 1 models; lDDT=85.506\\nSequence 17 completed in 7.9 sec with 1 models; lDDT=85.724\\nSequence 18 completed in 8.0 sec with 1 models; lDDT=82.076\\nSequence 19 completed in 7.9 sec with 1 models; lDDT=84.428\\nSequence 20 completed in 8.9 sec with 1 models; lDDT=83.861\\nSequence 21 completed in 8.1 sec with 1 models; lDDT=81.773\\nSequence 22 completed in 8.1 sec with 1 models; lDDT=83.023\\nSequence 23 completed in 8.0 sec with 1 models; lDDT=63.619\\nSequence 24 completed in 8.0 sec with 1 models; lDDT=69.711\\nDone with 25 sequences. 335.825 sec.\\nSetting up models took 3.390 seconds.\\n/tmp/vc61_a2w\\nSequence 0 completed in 138.4 sec with 1 models; lDDT=62.586\\nSequence 1 completed in 6.3 sec with 1 models; lDDT=68.130\\nSequence 2 completed in 6.3 sec with 1 models; lDDT=67.451\\nSequence 3 completed in 6.3 sec with 1 models; lDDT=51.707\\nSequence 4 completed in 6.3 sec with 1 models; lDDT=78.548\\nSequence 5 completed in 6.3 sec with 1 models; lDDT=69.951\\nSequence 6 completed in 6.7 sec with 1 models; lDDT=59.291\\nSequence 7 completed in 6.4 sec with 1 models; lDDT=61.606\\nSequence 8 completed in 6.4 sec with 1 models; lDDT=75.843\\nSequence 9 completed in 6.2 sec with 1 models; lDDT=59.831\\nSequence 10 completed in 6.2 sec with 1 models; lDDT=68.612\\nSequence 11 completed in 6.3 sec with 1 models; lDDT=75.555\\nSequence 12 completed in 6.2 sec with 1 models; lDDT=58.073\\nSequence 13 completed in 6.3 sec with 1 models; lDDT=69.029\\nSequence 14 completed in 6.2 sec with 1 models; lDDT=56.661\\nSequence 15 completed in 6.2 sec with 1 models; lDDT=59.763\\nSequence 16 completed in 6.2 sec with 1 models; lDDT=60.836\\nSequence 17 completed in 6.2 sec with 1 models; lDDT=63.703\\nSequence 18 completed in 6.2 sec with 1 models; lDDT=50.119\\nSequence 19 completed in 6.2 sec with 1 models; lDDT=46.726\\nDone with 20 sequences. 257.740 sec.\\nSetting up models took 3.381 seconds.\\n/tmp/5h1lt51s\\nSequence 0 completed in 122.4 sec with 1 models; lDDT=86.065\\nSequence 1 completed in 9.6 sec with 1 models; lDDT=78.504\\nSequence 2 completed in 9.7 sec with 1 models; lDDT=37.174\\nSequence 3 completed in 9.7 sec with 1 models; lDDT=80.875\\nSequence 4 completed in 10.2 sec with 1 models; lDDT=72.284\\nSequence 5 completed in 9.7 sec with 1 models; lDDT=77.479\\nSequence 6 completed in 9.7 sec with 1 models; lDDT=40.415\\nSequence 7 completed in 9.7 sec with 1 models; lDDT=72.536\\nSequence 8 completed in 9.7 sec with 1 models; lDDT=44.175\\nSequence 9 completed in 9.7 sec with 1 models; lDDT=73.845\\nSequence 10 completed in 9.6 sec with 1 models; lDDT=62.531\\nSequence 11 completed in 9.6 sec with 1 models; lDDT=87.808\\nSequence 12 completed in 9.7 sec with 1 models; lDDT=86.707\\nSequence 13 completed in 9.6 sec with 1 models; lDDT=77.549\\nSequence 14 completed in 9.6 sec with 1 models; lDDT=73.414\\nSequence 15 completed in 9.6 sec with 1 models; lDDT=41.375\\nSequence 16 completed in 9.7 sec with 1 models; lDDT=72.775\\nSequence 17 completed in 9.7 sec with 1 models; lDDT=72.590\\nSequence 18 completed in 10.4 sec with 1 models; lDDT=67.886\\nSequence 19 completed in 9.4 sec with 1 models; lDDT=75.786\\nSequence 20 completed in 9.5 sec with 1 models; lDDT=69.093\\nSequence 21 completed in 9.5 sec with 1 models; lDDT=59.821\\nSequence 22 completed in 9.5 sec with 1 models; lDDT=81.104\\nSequence 23 completed in 9.5 sec with 1 models; lDDT=85.447\\nSequence 24 completed in 9.5 sec with 1 models; lDDT=67.170\\nDone with 25 sequences. 354.246 sec.\\nSetting up models took 3.251 seconds.\\n/tmp/3vpkbnv0\\nSequence 0 completed in 96.7 sec with 1 models; lDDT=62.663\\nSequence 1 completed in 9.0 sec with 1 models; lDDT=82.104\\nSequence 2 completed in 9.0 sec with 1 models; lDDT=72.602\\nSequence 3 completed in 9.3 sec with 1 models; lDDT=68.630\\nSequence 4 completed in 9.0 sec with 1 models; lDDT=69.101\\nSequence 5 completed in 9.0 sec with 1 models; lDDT=85.283\\nSequence 6 completed in 9.0 sec with 1 models; lDDT=79.614\\nSequence 7 completed in 9.0 sec with 1 models; lDDT=81.039\\nSequence 8 completed in 8.9 sec with 1 models; lDDT=74.455\\nSequence 9 completed in 8.9 sec with 1 models; lDDT=77.450\\nSequence 10 completed in 8.9 sec with 1 models; lDDT=56.067\\nSequence 11 completed in 8.9 sec with 1 models; lDDT=64.587\\nSequence 12 completed in 8.9 sec with 1 models; lDDT=76.209\\nSequence 13 completed in 9.0 sec with 1 models; lDDT=79.143\\nSequence 14 completed in 8.9 sec with 1 models; lDDT=81.595\\nSequence 15 completed in 8.9 sec with 1 models; lDDT=85.662\\nSequence 16 completed in 9.6 sec with 1 models; lDDT=91.177\\nSequence 17 completed in 9.0 sec with 1 models; lDDT=76.828\\nSequence 18 completed in 9.1 sec with 1 models; lDDT=72.615\\nSequence 19 completed in 9.0 sec with 1 models; lDDT=85.234\\nDone with 20 sequences. 268.134 sec.\\nSetting up models took 3.178 seconds.\\n/tmp/d3k3wds5\\nSequence 0 completed in 114.1 sec with 1 models; lDDT=70.703\\nSequence 1 completed in 9.0 sec with 1 models; lDDT=64.687\\nSequence 2 completed in 9.0 sec with 1 models; lDDT=80.893\\nSequence 3 completed in 9.0 sec with 1 models; lDDT=64.948\\nSequence 4 completed in 9.4 sec with 1 models; lDDT=50.190\\nSequence 5 completed in 9.1 sec with 1 models; lDDT=71.763\\nSequence 6 completed in 9.0 sec with 1 models; lDDT=82.059\\nSequence 7 completed in 9.0 sec with 1 models; lDDT=75.378\\nSequence 8 completed in 9.0 sec with 1 models; lDDT=71.335\\nSequence 9 completed in 9.0 sec with 1 models; lDDT=75.759\\nSequence 10 completed in 9.0 sec with 1 models; lDDT=51.222\\nSequence 11 completed in 9.0 sec with 1 models; lDDT=46.922\\nSequence 12 completed in 9.0 sec with 1 models; lDDT=54.181\\nSequence 13 completed in 9.2 sec with 1 models; lDDT=77.155\\nSequence 14 completed in 9.0 sec with 1 models; lDDT=61.572\\nSequence 15 completed in 9.0 sec with 1 models; lDDT=71.844\\nSequence 16 completed in 9.0 sec with 1 models; lDDT=69.362\\nSequence 17 completed in 9.0 sec with 1 models; lDDT=77.972\\nSequence 18 completed in 9.7 sec with 1 models; lDDT=64.755\\nSequence 19 completed in 9.0 sec with 1 models; lDDT=70.703\\nSequence 20 completed in 8.9 sec with 1 models; lDDT=63.777\\nSequence 21 completed in 9.1 sec with 1 models; lDDT=67.744\\nSequence 22 completed in 9.0 sec with 1 models; lDDT=59.511\\nSequence 23 completed in 9.0 sec with 1 models; lDDT=68.360\\nSequence 24 completed in 9.1 sec with 1 models; lDDT=74.203\\nSequence 25 completed in 9.0 sec with 1 models; lDDT=74.975\\nSequence 26 completed in 9.0 sec with 1 models; lDDT=60.532\\nSequence 27 completed in 9.0 sec with 1 models; lDDT=65.486\\nSequence 28 completed in 9.0 sec with 1 models; lDDT=78.870\\nSequence 29 completed in 9.0 sec with 1 models; lDDT=71.440\\nDone with 30 sequences. 376.623 sec.\\nSetting up models took 3.245 seconds.\\n/tmp/b2anzk7k\\nSequence 0 completed in 114.8 sec with 1 models; lDDT=47.480\\nSequence 1 completed in 9.5 sec with 1 models; lDDT=39.348\\nSequence 2 completed in 9.4 sec with 1 models; lDDT=47.235\\nSequence 3 completed in 9.5 sec with 1 models; lDDT=44.602\\nSequence 4 completed in 9.9 sec with 1 models; lDDT=44.005\\nSequence 5 completed in 9.5 sec with 1 models; lDDT=42.145\\nSequence 6 completed in 9.5 sec with 1 models; lDDT=50.705\\nSequence 7 completed in 9.4 sec with 1 models; lDDT=50.362\\nSequence 8 completed in 9.4 sec with 1 models; lDDT=49.451\\nSequence 9 completed in 9.4 sec with 1 models; lDDT=49.920\\nSequence 10 completed in 9.4 sec with 1 models; lDDT=45.724\\nSequence 11 completed in 9.4 sec with 1 models; lDDT=46.619\\nSequence 12 completed in 9.4 sec with 1 models; lDDT=45.362\\nSequence 13 completed in 9.4 sec with 1 models; lDDT=47.894\\nSequence 14 completed in 9.4 sec with 1 models; lDDT=51.643\\nSequence 15 completed in 9.4 sec with 1 models; lDDT=83.612\\nSequence 16 completed in 9.3 sec with 1 models; lDDT=50.166\\nSequence 17 completed in 9.3 sec with 1 models; lDDT=65.854\\nSequence 18 completed in 10.0 sec with 1 models; lDDT=54.986\\nSequence 19 completed in 9.4 sec with 1 models; lDDT=68.904\\nSequence 20 completed in 9.5 sec with 1 models; lDDT=58.385\\nSequence 21 completed in 9.5 sec with 1 models; lDDT=57.235\\nSequence 22 completed in 9.5 sec with 1 models; lDDT=51.205\\nSequence 23 completed in 9.5 sec with 1 models; lDDT=52.926\\nSequence 24 completed in 9.4 sec with 1 models; lDDT=59.252\\nSequence 25 completed in 9.4 sec with 1 models; lDDT=64.866\\nSequence 26 completed in 9.2 sec with 1 models; lDDT=50.499\\nSequence 27 completed in 9.3 sec with 1 models; lDDT=45.652\\nSequence 28 completed in 9.3 sec with 1 models; lDDT=53.419\\nSequence 29 completed in 9.4 sec with 1 models; lDDT=87.612\\nSequence 30 completed in 9.3 sec with 1 models; lDDT=48.160\\nSequence 31 completed in 10.1 sec with 1 models; lDDT=46.180\\nSequence 32 completed in 9.4 sec with 1 models; lDDT=46.099\\nSequence 33 completed in 9.5 sec with 1 models; lDDT=66.782\\nSequence 34 completed in 9.3 sec with 1 models; lDDT=51.525\\nDone with 35 sequences. 436.528 sec.\\nSetting up models took 3.224 seconds.\\n/tmp/05ghyu5n\\nSequence 0 completed in 109.5 sec with 1 models; lDDT=58.220\\nSequence 1 completed in 6.5 sec with 1 models; lDDT=56.895\\nSequence 2 completed in 6.5 sec with 1 models; lDDT=56.066\\nSequence 3 completed in 6.5 sec with 1 models; lDDT=62.622\\nSequence 4 completed in 6.5 sec with 1 models; lDDT=56.833\\nSequence 5 completed in 6.6 sec with 1 models; lDDT=46.587\\nSequence 6 completed in 6.5 sec with 1 models; lDDT=73.913\\nSequence 7 completed in 6.5 sec with 1 models; lDDT=66.555\\nSequence 8 completed in 6.5 sec with 1 models; lDDT=68.437\\nSequence 9 completed in 6.5 sec with 1 models; lDDT=64.729\\nSequence 10 completed in 6.5 sec with 1 models; lDDT=65.092\\nSequence 11 completed in 6.4 sec with 1 models; lDDT=61.805\\nSequence 12 completed in 6.5 sec with 1 models; lDDT=53.572\\nSequence 13 completed in 7.2 sec with 1 models; lDDT=54.659\\nSequence 14 completed in 6.5 sec with 1 models; lDDT=48.157\\nSequence 15 completed in 6.6 sec with 1 models; lDDT=47.684\\nSequence 16 completed in 6.6 sec with 1 models; lDDT=62.112\\nSequence 17 completed in 6.5 sec with 1 models; lDDT=48.570\\nSequence 18 completed in 6.5 sec with 1 models; lDDT=56.073\\nSequence 19 completed in 6.5 sec with 1 models; lDDT=77.682\\nSequence 20 completed in 6.6 sec with 1 models; lDDT=76.441\\nSequence 21 completed in 6.5 sec with 1 models; lDDT=49.797\\nSequence 22 completed in 6.4 sec with 1 models; lDDT=63.810\\nSequence 23 completed in 6.5 sec with 1 models; lDDT=52.516\\nSequence 24 completed in 6.5 sec with 1 models; lDDT=70.170\\nDone with 25 sequences. 266.630 sec.\\nSetting up models took 3.179 seconds.\\n/tmp/baxvf77l\\nSequence 0 completed in 111.8 sec with 1 models; lDDT=63.408\\nSequence 1 completed in 9.3 sec with 1 models; lDDT=69.697\\nSequence 2 completed in 9.2 sec with 1 models; lDDT=68.767\\nSequence 3 completed in 9.2 sec with 1 models; lDDT=79.229\\nSequence 4 completed in 9.7 sec with 1 models; lDDT=70.770\\nSequence 5 completed in 9.4 sec with 1 models; lDDT=80.679\\nSequence 6 completed in 9.3 sec with 1 models; lDDT=73.790\\nSequence 7 completed in 9.3 sec with 1 models; lDDT=81.962\\nSequence 8 completed in 9.2 sec with 1 models; lDDT=61.069\\nSequence 9 completed in 9.4 sec with 1 models; lDDT=81.271\\nSequence 10 completed in 9.3 sec with 1 models; lDDT=55.073\\nSequence 11 completed in 9.3 sec with 1 models; lDDT=57.512\\nSequence 12 completed in 9.4 sec with 1 models; lDDT=77.070\\nSequence 13 completed in 9.3 sec with 1 models; lDDT=76.187\\nSequence 14 completed in 9.3 sec with 1 models; lDDT=78.758\\nSequence 15 completed in 9.2 sec with 1 models; lDDT=49.397\\nSequence 16 completed in 9.3 sec with 1 models; lDDT=66.755\\nSequence 17 completed in 9.2 sec with 1 models; lDDT=73.844\\nSequence 18 completed in 10.1 sec with 1 models; lDDT=69.909\\nSequence 19 completed in 9.3 sec with 1 models; lDDT=72.496\\nDone with 20 sequences. 289.313 sec.\\nSetting up models took 3.209 seconds.\\n/tmp/hl074wv8\\nSequence 0 completed in 129.1 sec with 1 models; lDDT=60.918\\nSequence 1 completed in 5.9 sec with 1 models; lDDT=64.781\\nSequence 2 completed in 5.8 sec with 1 models; lDDT=64.627\\nSequence 3 completed in 5.8 sec with 1 models; lDDT=54.787\\nSequence 4 completed in 5.9 sec with 1 models; lDDT=66.306\\nSequence 5 completed in 5.9 sec with 1 models; lDDT=54.497\\nSequence 6 completed in 6.3 sec with 1 models; lDDT=78.568\\nSequence 7 completed in 5.9 sec with 1 models; lDDT=49.827\\nSequence 8 completed in 6.0 sec with 1 models; lDDT=80.182\\nSequence 9 completed in 5.9 sec with 1 models; lDDT=48.931\\nSequence 10 completed in 5.8 sec with 1 models; lDDT=71.591\\nSequence 11 completed in 5.8 sec with 1 models; lDDT=58.812\\nSequence 12 completed in 5.8 sec with 1 models; lDDT=75.826\\nSequence 13 completed in 5.8 sec with 1 models; lDDT=68.876\\nSequence 14 completed in 5.8 sec with 1 models; lDDT=57.037\\nSequence 15 completed in 5.8 sec with 1 models; lDDT=66.830\\nSequence 16 completed in 5.9 sec with 1 models; lDDT=58.159\\nSequence 17 completed in 5.8 sec with 1 models; lDDT=80.055\\nSequence 18 completed in 5.8 sec with 1 models; lDDT=72.597\\nSequence 19 completed in 5.8 sec with 1 models; lDDT=70.763\\nSequence 20 completed in 6.5 sec with 1 models; lDDT=60.722\\nSequence 21 completed in 5.9 sec with 1 models; lDDT=52.120\\nSequence 22 completed in 5.9 sec with 1 models; lDDT=75.001\\nSequence 23 completed in 5.9 sec with 1 models; lDDT=49.322\\nSequence 24 completed in 5.9 sec with 1 models; lDDT=59.653\\nDone with 25 sequences. 270.846 sec.\\nSetting up models took 3.207 seconds.\\n/tmp/2e68u43f\\nSequence 0 completed in 140.4 sec with 1 models; lDDT=87.807\\nSequence 1 completed in 7.2 sec with 1 models; lDDT=77.529\\nSequence 2 completed in 7.2 sec with 1 models; lDDT=84.865\\nSequence 3 completed in 7.1 sec with 1 models; lDDT=75.472\\nSequence 4 completed in 7.1 sec with 1 models; lDDT=72.191\\nDone with 5 sequences. 168.968 sec.\\n'\n",
      "b''\n",
      "Done\n",
      "\n"
     ]
    }
   ],
   "source": [
    "## Setting up AlphaFold2 run\n",
    "\n",
    "AF2_recycles = 3\n",
    "AF2_models = \"4\"  # add other models to this string if needed, i.e. \"3 4 5\"\n",
    "\n",
    "commands_af2 = \"\"\"\n",
    "#!/bin/sh\n",
    "export CUDA_HOME=/usr/local/cuda-12\n",
    "export PATH=$CUDA_HOME/bin:$PATH\n",
    "export LD_LIBRARY_PATH=$CUDA_HOME/lib64:$LD_LIBRARY_PATH\n",
    "\"\"\"\n",
    "cmds_filename_af2 = \"commands_af2\"\n",
    "with open(cmds_filename_af2, \"w\") as file:\n",
    "    for ff in glob.glob(\"*.fasta\"):\n",
    "        commands_af2 += (f\"{PYTHON['af2']} {AF2_script} \")\n",
    "        commands_af2 += (f\"--af-nrecycles {AF2_recycles} --af-models {AF2_models} \")\n",
    "        commands_af2 += (f\"--fasta {ff} --scorefile {ff.replace('.fasta', '.csv')}\\n\")\n",
    "    file.write(commands_af2)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    print(\"AF2 command:\")\n",
    "    print(commands_af2)\n",
    "    p = subprocess.Popen(['bash', cmds_filename_af2], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n",
    "    print(output)\n",
    "    print(err)\n",
    "    print('Done')\n",
    "else:\n",
    "    print(\"Skipping - Done\")\n",
    "\n",
    "##### Original\n",
    "\"\"\"\n",
    "commands_af2 = []\n",
    "cmds_filename_af2 = \"commands_af2\"\n",
    "with open(cmds_filename_af2, \"w\") as file:\n",
    "    for ff in glob.glob(\"*.fasta\"):\n",
    "        commands_af2.append(f\"{PYTHON['af2']} {AF2_script} \"\n",
    "                             f\"--af-nrecycles {AF2_recycles} --af-models {AF2_models} \"\n",
    "                             f\"--fasta {ff} --scorefile {ff.replace('.fasta', '.csv')}\\n\")\n",
    "        file.write(commands_af2[-1])\n",
    "\n",
    "print(\"Example AF2 command:\")\n",
    "print(commands_af2[-1])\n",
    "\n",
    "### Running AF2 with Slurm.\n",
    "### Running jobs on the CPU. It takes ~10 minutes per sequence\n",
    "### \n",
    "\n",
    "submit_script = \"submit_af2.sh\"\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"2_af2\", mem=\"6g\", \n",
    "                                     N_cores=2, gpu=True, gres=\"gpu:rtx2080:1\", time=\"00:12:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "else:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"2_af2\", mem=\"6g\", \n",
    "                                     N_cores=4, time=\"01:00:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n",
    "\"\"\"\n",
    "\n",
    "print()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "2a88b24e",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with diffusion and happy with the outputs then mark it as done\n",
    "AF2_DIR = f\"{WDIR}/2_af2\"\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    with open(f\"{AF2_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "242e9def",
   "metadata": {},
   "source": [
    "### Analyzing AlphaFold2 predictions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "30f38048",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/utils/analyze_af2.py --scorefile scores.csv --ref_path /home/ubuntu/data/heme_binder_diffusion_edit/output/0_diffusion/filtered_structures/ --mpnn --params /home/ubuntu/data/heme_binder_diffusion_edit/theozyme/HBA/HBA.params\n",
      "Only calculating RMSD for predictions with lDDT >= 0.0\n",
      "705 models with lDDT >= 0.0 to analyze.\n",
      "┌──────────────────────────────────────────────────────────────────────────────┐\n",
      "│                                 PyRosetta-4                                  │\n",
      "│              Created in JHU by Sergey Lyskov and PyRosetta Team              │\n",
      "│              (C) Copyright Rosetta Commons Member Institutions               │\n",
      "│                                                                              │\n",
      "│ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE │\n",
      "│              See LICENSE.md or email license@uw.edu for details              │\n",
      "└──────────────────────────────────────────────────���───────────────────────────┘\n",
      "PyRosetta-4 2024 [Rosetta PyRosetta4.Release.python39.ubuntu.cxx11thread.serialization 2024.10+release.2c36cbc7108d85646ca5b8ddc89c29ac1ccde88e 2024-03-01T16:53:36] retrieved from: http://www.pyrosetta.org\n",
      "7o2g_HBA_dif_131_traj4_T0.3_0_3_model_4.0_r3_af2: length = 105, lDDT = 64.63, rmsd = 8.538, \n",
      "7o2g_HBA_dif_131_traj4_T0.3_0_4_model_4.0_r3_af2: length = 105, lDDT = 54.79, rmsd = 7.060, \n",
      "7o2g_HBA_dif_131_traj4_T0.3_0_5_model_4.0_r3_af2: length = 105, lDDT = 66.31, rmsd = 3.206, \n",
      "7o2g_HBA_dif_131_T0.3_0_1_model_4.0_r3_af2: length = 105, lDDT = 54.50, rmsd = 11.145, \n",
      "7o2g_HBA_dif_131_T0.3_0_2_model_4.0_r3_af2: length = 105, lDDT = 78.57, rmsd = 8.487, \n",
      "7o2g_HBA_dif_131_traj1_T0.3_0_1_model_4.0_r3_af2: length = 105, lDDT = 71.59, rmsd = 2.982, \n",
      "7o2g_HBA_dif_131_traj1_T0.3_0_5_model_4.0_r3_af2: length = 105, lDDT = 57.04, rmsd = 10.379, \n",
      "7o2g_HBA_dif_131_traj3_T0.3_0_1_model_4.0_r3_af2: length = 105, lDDT = 66.83, rmsd = 5.054, \n",
      "7o2g_HBA_dif_131_traj3_T0.3_0_4_model_4.0_r3_af2: length = 105, lDDT = 72.60, rmsd = 7.931, \n",
      "7o2g_HBA_dif_131_traj2_T0.3_0_2_model_4.0_r3_af2: length = 105, lDDT = 52.12, rmsd = 12.889, \n",
      "7o2g_HBA_dif_131_traj2_T0.3_0_5_model_4.0_r3_af2: length = 105, lDDT = 59.65, rmsd = 2.944, \n",
      "7o2g_HBA_dif_52_traj3_T0.3_0_3_model_4.0_r3_af2: length = 107, lDDT = 67.45, rmsd = 8.456, \n",
      "7o2g_HBA_dif_52_traj4_T0.3_0_1_model_4.0_r3_af2: length = 107, lDDT = 69.95, rmsd = 8.207, \n",
      "7o2g_HBA_dif_52_traj4_T0.3_0_4_model_4.0_r3_af2: length = 107, lDDT = 75.84, rmsd = 3.138, \n",
      "7o2g_HBA_dif_52_traj2_T0.3_0_1_model_4.0_r3_af2: length = 107, lDDT = 68.61, rmsd = 10.166, \n",
      "7o2g_HBA_dif_52_traj2_T0.3_0_5_model_4.0_r3_af2: length = 107, lDDT = 56.66, rmsd = 12.229, \n",
      "7o2g_HBA_dif_52_traj1_T0.3_0_3_model_4.0_r3_af2: length = 107, lDDT = 63.70, rmsd = 10.119, \n",
      "7o2g_HBA_dif_52_traj1_T0.3_0_5_model_4.0_r3_af2: length = 107, lDDT = 46.73, rmsd = 5.238, \n",
      "7o2g_HBA_dif_220_traj3_T0.3_0_2_model_4.0_r3_af2: length = 112, lDDT = 56.90, rmsd = 9.506, \n",
      "7o2g_HBA_dif_220_traj3_T0.3_0_4_model_4.0_r3_af2: length = 112, lDDT = 62.62, rmsd = 8.963, \n",
      "7o2g_HBA_dif_220_traj2_T0.3_0_2_model_4.0_r3_af2: length = 112, lDDT = 73.91, rmsd = 11.442, \n",
      "7o2g_HBA_dif_220_traj2_T0.3_0_5_model_4.0_r3_af2: length = 112, lDDT = 64.73, rmsd = 3.736, \n",
      "7o2g_HBA_dif_220_T0.3_0_3_model_4.0_r3_af2: length = 112, lDDT = 53.57, rmsd = 11.836, \n",
      "7o2g_HBA_dif_220_traj1_T0.3_0_1_model_4.0_r3_af2: length = 112, lDDT = 47.68, rmsd = 6.183, \n",
      "7o2g_HBA_dif_220_traj1_T0.3_0_4_model_4.0_r3_af2: length = 112, lDDT = 56.07, rmsd = 11.050, \n",
      "7o2g_HBA_dif_220_traj4_T0.3_0_2_model_4.0_r3_af2: length = 112, lDDT = 49.80, rmsd = 13.645, \n",
      "7o2g_HBA_dif_220_traj4_T0.3_0_4_model_4.0_r3_af2: length = 112, lDDT = 52.52, rmsd = 9.521, \n",
      "7o2g_HBA_dif_78_traj2_T0.3_0_1_model_4.0_r3_af2: length = 113, lDDT = 82.54, rmsd = 2.507, \n",
      "7o2g_HBA_dif_78_traj2_T0.3_0_4_model_4.0_r3_af2: length = 113, lDDT = 44.01, rmsd = 12.192, \n",
      "7o2g_HBA_dif_78_traj1_T0.3_0_2_model_4.0_r3_af2: length = 113, lDDT = 54.41, rmsd = 12.648, \n",
      "7o2g_HBA_dif_78_traj1_T0.3_0_5_model_4.0_r3_af2: length = 113, lDDT = 42.86, rmsd = 9.453, \n",
      "7o2g_HBA_dif_209_traj3_T0.3_0_3_model_4.0_r3_af2: length = 113, lDDT = 74.42, rmsd = 3.799, \n",
      "7o2g_HBA_dif_78_traj3_T0.3_0_1_model_4.0_r3_af2: length = 113, lDDT = 44.35, rmsd = 11.901, \n",
      "7o2g_HBA_dif_78_traj3_T0.3_0_4_model_4.0_r3_af2: length = 113, lDDT = 76.32, rmsd = 2.743, \n",
      "7o2g_HBA_dif_78_traj4_T0.3_0_3_model_4.0_r3_af2: length = 113, lDDT = 63.52, rmsd = 11.897, \n",
      "7o2g_HBA_dif_78_T0.3_0_1_model_4.0_r3_af2: length = 113, lDDT = 66.40, rmsd = 2.042, \n",
      "7o2g_HBA_dif_78_T0.3_0_4_model_4.0_r3_af2: length = 113, lDDT = 77.46, rmsd = 2.414, \n",
      "7o2g_HBA_dif_209_traj4_T0.3_0_2_model_4.0_r3_af2: length = 113, lDDT = 73.48, rmsd = 7.438, \n",
      "7o2g_HBA_dif_209_traj4_T0.3_0_4_model_4.0_r3_af2: length = 113, lDDT = 80.41, rmsd = 2.623, \n",
      "7o2g_HBA_dif_25_traj4_T0.3_0_2_model_4.0_r3_af2: length = 119, lDDT = 77.53, rmsd = 2.174, \n",
      "7o2g_HBA_dif_185_traj3_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 79.56, rmsd = 2.409, \n",
      "7o2g_HBA_dif_185_traj3_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 82.46, rmsd = 2.737, \n",
      "7o2g_HBA_dif_185_traj2_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 61.08, rmsd = 12.379, \n",
      "7o2g_HBA_dif_185_traj2_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 67.06, rmsd = 3.378, \n",
      "7o2g_HBA_dif_185_traj2_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 65.41, rmsd = 8.534, \n",
      "7o2g_HBA_dif_64_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 62.19, rmsd = 8.754, \n",
      "7o2g_HBA_dif_226_traj1_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 52.20, rmsd = 10.346, \n",
      "7o2g_HBA_dif_226_traj4_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 71.31, rmsd = 1.960, \n",
      "7o2g_HBA_dif_226_traj4_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 71.50, rmsd = 10.163, \n",
      "7o2g_HBA_dif_191_traj1_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 45.51, rmsd = 15.365, \n",
      "7o2g_HBA_dif_191_traj1_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 66.02, rmsd = 9.548, \n",
      "7o2g_HBA_dif_201_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 78.91, rmsd = 2.098, \n",
      "7o2g_HBA_dif_191_traj4_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 60.17, rmsd = 11.733, \n",
      "7o2g_HBA_dif_191_traj2_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 73.26, rmsd = 8.538, \n",
      "7o2g_HBA_dif_191_traj2_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 66.23, rmsd = 13.299, \n",
      "7o2g_HBA_dif_64_traj2_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 62.07, rmsd = 12.431, \n",
      "7o2g_HBA_dif_226_traj3_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 62.98, rmsd = 8.630, \n",
      "7o2g_HBA_dif_226_traj3_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 75.92, rmsd = 8.568, \n",
      "7o2g_HBA_dif_191_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 66.98, rmsd = 7.291, \n",
      "7o2g_HBA_dif_191_traj3_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 68.56, rmsd = 7.288, \n",
      "7o2g_HBA_dif_191_traj3_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 65.53, rmsd = 8.355, \n",
      "7o2g_HBA_dif_185_traj1_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 76.58, rmsd = 1.134, \n",
      "7o2g_HBA_dif_64_traj4_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 58.51, rmsd = 7.899, \n",
      "7o2g_HBA_dif_64_traj4_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 88.85, rmsd = 1.421, \n",
      "7o2g_HBA_dif_64_traj1_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 69.62, rmsd = 5.700, \n",
      "7o2g_HBA_dif_64_traj1_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 85.42, rmsd = 1.864, \n",
      "7o2g_HBA_dif_64_traj3_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 83.53, rmsd = 1.900, \n",
      "7o2g_HBA_dif_28_T0.3_0_1_model_4.0_r3_af2: length = 121, lDDT = 79.10, rmsd = 3.417, \n",
      "7o2g_HBA_dif_28_T0.3_0_5_model_4.0_r3_af2: length = 121, lDDT = 87.86, rmsd = 1.264, \n",
      "7o2g_HBA_dif_28_traj3_T0.3_0_1_model_4.0_r3_af2: length = 121, lDDT = 47.70, rmsd = 12.134, \n",
      "7o2g_HBA_dif_28_traj3_T0.3_0_4_model_4.0_r3_af2: length = 121, lDDT = 56.31, rmsd = 6.696, \n",
      "7o2g_HBA_dif_28_traj2_T0.3_0_3_model_4.0_r3_af2: length = 121, lDDT = 83.62, rmsd = 1.889, \n",
      "7o2g_HBA_dif_28_traj4_T0.3_0_1_model_4.0_r3_af2: length = 121, lDDT = 85.32, rmsd = 1.077, \n",
      "7o2g_HBA_dif_28_traj4_T0.3_0_3_model_4.0_r3_af2: length = 121, lDDT = 60.04, rmsd = 14.229, \n",
      "7o2g_HBA_dif_28_traj4_T0.3_0_5_model_4.0_r3_af2: length = 121, lDDT = 50.11, rmsd = 14.684, \n",
      "7o2g_HBA_dif_28_traj1_T0.3_0_3_model_4.0_r3_af2: length = 121, lDDT = 71.07, rmsd = 5.875, \n",
      "7o2g_HBA_dif_108_traj4_T0.3_0_1_model_4.0_r3_af2: length = 123, lDDT = 66.07, rmsd = 4.685, \n",
      "7o2g_HBA_dif_108_traj4_T0.3_0_4_model_4.0_r3_af2: length = 123, lDDT = 79.68, rmsd = 2.933, \n",
      "7o2g_HBA_dif_108_traj1_T0.3_0_2_model_4.0_r3_af2: length = 123, lDDT = 76.37, rmsd = 2.689, \n",
      "7o2g_HBA_dif_108_traj1_T0.3_0_4_model_4.0_r3_af2: length = 123, lDDT = 66.41, rmsd = 3.037, \n",
      "7o2g_HBA_dif_108_T0.3_0_1_model_4.0_r3_af2: length = 123, lDDT = 64.76, rmsd = 15.353, \n",
      "7o2g_HBA_dif_108_T0.3_0_3_model_4.0_r3_af2: length = 123, lDDT = 71.14, rmsd = 5.016, \n",
      "7o2g_HBA_dif_108_traj2_T0.3_0_2_model_4.0_r3_af2: length = 123, lDDT = 78.30, rmsd = 3.235, \n",
      "7o2g_HBA_dif_108_traj2_T0.3_0_5_model_4.0_r3_af2: length = 123, lDDT = 69.22, rmsd = 3.723, \n",
      "7o2g_HBA_dif_108_traj3_T0.3_0_4_model_4.0_r3_af2: length = 123, lDDT = 62.35, rmsd = 11.660, \n",
      "7o2g_HBA_dif_177_traj3_T0.3_0_3_model_4.0_r3_af2: length = 126, lDDT = 85.86, rmsd = 1.184, \n",
      "7o2g_HBA_dif_177_T0.3_0_1_model_4.0_r3_af2: length = 126, lDDT = 85.00, rmsd = 3.477, \n",
      "7o2g_HBA_dif_177_T0.3_0_4_model_4.0_r3_af2: length = 126, lDDT = 87.98, rmsd = 1.427, \n",
      "7o2g_HBA_dif_177_traj1_T0.3_0_2_model_4.0_r3_af2: length = 126, lDDT = 84.35, rmsd = 8.371, \n",
      "7o2g_HBA_dif_131_traj4_T0.3_0_2_model_4.0_r3_af2: length = 105, lDDT = 64.78, rmsd = 3.127, \n",
      "7o2g_HBA_dif_131_T0.3_0_4_model_4.0_r3_af2: length = 105, lDDT = 80.18, rmsd = 2.281, \n",
      "7o2g_HBA_dif_131_T0.3_0_5_model_4.0_r3_af2: length = 105, lDDT = 48.93, rmsd = 11.140, \n",
      "7o2g_HBA_dif_131_traj1_T0.3_0_4_model_4.0_r3_af2: length = 105, lDDT = 68.88, rmsd = 5.294, \n",
      "7o2g_HBA_dif_131_traj3_T0.3_0_3_model_4.0_r3_af2: length = 105, lDDT = 80.06, rmsd = 11.969, \n",
      "7o2g_HBA_dif_131_traj3_T0.3_0_5_model_4.0_r3_af2: length = 105, lDDT = 70.76, rmsd = 3.075, \n",
      "7o2g_HBA_dif_131_traj2_T0.3_0_3_model_4.0_r3_af2: length = 105, lDDT = 75.00, rmsd = 2.462, \n",
      "7o2g_HBA_dif_52_traj3_T0.3_0_1_model_4.0_r3_af2: length = 107, lDDT = 62.59, rmsd = 8.217, \n",
      "7o2g_HBA_dif_52_traj3_T0.3_0_4_model_4.0_r3_af2: length = 107, lDDT = 51.71, rmsd = 9.914, \n",
      "7o2g_HBA_dif_52_traj4_T0.3_0_2_model_4.0_r3_af2: length = 107, lDDT = 59.29, rmsd = 8.251, \n",
      "7o2g_HBA_dif_52_traj4_T0.3_0_5_model_4.0_r3_af2: length = 107, lDDT = 59.83, rmsd = 5.307, \n",
      "7o2g_HBA_dif_52_traj2_T0.3_0_4_model_4.0_r3_af2: length = 107, lDDT = 69.03, rmsd = 11.067, \n",
      "7o2g_HBA_dif_52_traj1_T0.3_0_2_model_4.0_r3_af2: length = 107, lDDT = 60.84, rmsd = 13.356, \n",
      "7o2g_HBA_dif_220_traj3_T0.3_0_1_model_4.0_r3_af2: length = 112, lDDT = 58.22, rmsd = 9.717, \n",
      "7o2g_HBA_dif_220_traj3_T0.3_0_5_model_4.0_r3_af2: length = 112, lDDT = 56.83, rmsd = 14.619, \n",
      "7o2g_HBA_dif_220_traj2_T0.3_0_3_model_4.0_r3_af2: length = 112, lDDT = 66.56, rmsd = 2.552, \n",
      "7o2g_HBA_dif_220_T0.3_0_1_model_4.0_r3_af2: length = 112, lDDT = 65.09, rmsd = 11.630, \n",
      "7o2g_HBA_dif_220_T0.3_0_5_model_4.0_r3_af2: length = 112, lDDT = 48.16, rmsd = 12.720, \n",
      "7o2g_HBA_dif_220_traj1_T0.3_0_3_model_4.0_r3_af2: length = 112, lDDT = 48.57, rmsd = 10.934, \n",
      "7o2g_HBA_dif_220_traj4_T0.3_0_1_model_4.0_r3_af2: length = 112, lDDT = 76.44, rmsd = 13.005, \n",
      "7o2g_HBA_dif_220_traj4_T0.3_0_5_model_4.0_r3_af2: length = 112, lDDT = 70.17, rmsd = 10.760, \n",
      "7o2g_HBA_dif_78_traj2_T0.3_0_3_model_4.0_r3_af2: length = 113, lDDT = 61.49, rmsd = 11.619, \n",
      "7o2g_HBA_dif_78_traj2_T0.3_0_5_model_4.0_r3_af2: length = 113, lDDT = 74.94, rmsd = 3.180, \n",
      "7o2g_HBA_dif_78_traj1_T0.3_0_3_model_4.0_r3_af2: length = 113, lDDT = 48.53, rmsd = 12.231, \n",
      "7o2g_HBA_dif_209_traj3_T0.3_0_1_model_4.0_r3_af2: length = 113, lDDT = 81.88, rmsd = 1.153, \n",
      "7o2g_HBA_dif_209_traj3_T0.3_0_4_model_4.0_r3_af2: length = 113, lDDT = 77.60, rmsd = 13.053, \n",
      "7o2g_HBA_dif_78_traj3_T0.3_0_3_model_4.0_r3_af2: length = 113, lDDT = 77.31, rmsd = 3.004, \n",
      "7o2g_HBA_dif_78_traj4_T0.3_0_1_model_4.0_r3_af2: length = 113, lDDT = 61.61, rmsd = 12.284, \n",
      "7o2g_HBA_dif_78_traj4_T0.3_0_2_model_4.0_r3_af2: length = 113, lDDT = 62.05, rmsd = 7.178, \n",
      "7o2g_HBA_dif_78_traj4_T0.3_0_5_model_4.0_r3_af2: length = 113, lDDT = 77.26, rmsd = 3.015, \n",
      "7o2g_HBA_dif_78_T0.3_0_3_model_4.0_r3_af2: length = 113, lDDT = 48.34, rmsd = 14.034, \n",
      "7o2g_HBA_dif_209_traj4_T0.3_0_1_model_4.0_r3_af2: length = 113, lDDT = 65.04, rmsd = 4.529, \n",
      "7o2g_HBA_dif_209_traj4_T0.3_0_5_model_4.0_r3_af2: length = 113, lDDT = 88.66, rmsd = 1.650, \n",
      "7o2g_HBA_dif_25_traj4_T0.3_0_3_model_4.0_r3_af2: length = 119, lDDT = 84.86, rmsd = 1.056, \n",
      "7o2g_HBA_dif_25_traj4_T0.3_0_5_model_4.0_r3_af2: length = 119, lDDT = 72.19, rmsd = 9.663, \n",
      "7o2g_HBA_dif_185_traj3_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 50.38, rmsd = 11.475, \n",
      "7o2g_HBA_dif_185_traj3_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 61.98, rmsd = 9.515, \n",
      "7o2g_HBA_dif_185_traj2_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 69.54, rmsd = 3.944, \n",
      "7o2g_HBA_dif_64_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 50.19, rmsd = 13.022, \n",
      "7o2g_HBA_dif_64_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 55.51, rmsd = 7.049, \n",
      "7o2g_HBA_dif_226_traj1_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 84.40, rmsd = 2.804, \n",
      "7o2g_HBA_dif_226_traj1_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 66.31, rmsd = 7.528, \n",
      "7o2g_HBA_dif_226_traj4_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 79.48, rmsd = 2.005, \n",
      "7o2g_HBA_dif_226_traj4_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 62.19, rmsd = 3.714, \n",
      "7o2g_HBA_dif_191_traj1_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 62.39, rmsd = 5.281, \n",
      "7o2g_HBA_dif_191_traj1_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 67.41, rmsd = 5.949, \n",
      "7o2g_HBA_dif_201_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 71.85, rmsd = 6.306, \n",
      "7o2g_HBA_dif_201_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 68.52, rmsd = 2.795, \n",
      "7o2g_HBA_dif_191_traj4_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 81.44, rmsd = 6.197, \n",
      "7o2g_HBA_dif_191_traj2_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 68.50, rmsd = 14.312, \n",
      "7o2g_HBA_dif_191_traj2_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 53.53, rmsd = 11.147, \n",
      "7o2g_HBA_dif_64_traj2_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 59.63, rmsd = 11.702, \n",
      "7o2g_HBA_dif_64_traj2_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 77.36, rmsd = 1.589, \n",
      "7o2g_HBA_dif_226_traj3_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 61.05, rmsd = 12.443, \n",
      "7o2g_HBA_dif_191_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 55.09, rmsd = 8.789, \n",
      "7o2g_HBA_dif_191_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 72.26, rmsd = 3.008, \n",
      "7o2g_HBA_dif_191_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 60.70, rmsd = 7.641, \n",
      "7o2g_HBA_dif_191_traj3_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 60.05, rmsd = 5.781, \n",
      "7o2g_HBA_dif_185_traj1_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 71.62, rmsd = 2.060, \n",
      "7o2g_HBA_dif_185_traj1_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 76.06, rmsd = 3.607, \n",
      "7o2g_HBA_dif_64_traj4_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 51.77, rmsd = 11.388, \n",
      "7o2g_HBA_dif_64_traj4_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 57.56, rmsd = 8.828, \n",
      "7o2g_HBA_dif_64_traj1_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 57.86, rmsd = 6.538, \n",
      "7o2g_HBA_dif_64_traj3_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 80.25, rmsd = 2.447, \n",
      "7o2g_HBA_dif_64_traj3_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 54.36, rmsd = 12.807, \n",
      "7o2g_HBA_dif_28_T0.3_0_2_model_4.0_r3_af2: length = 121, lDDT = 67.28, rmsd = 4.247, \n",
      "7o2g_HBA_dif_28_T0.3_0_3_model_4.0_r3_af2: length = 121, lDDT = 65.93, rmsd = 4.257, \n",
      "7o2g_HBA_dif_28_traj3_T0.3_0_2_model_4.0_r3_af2: length = 121, lDDT = 51.92, rmsd = 11.917, \n",
      "7o2g_HBA_dif_28_traj2_T0.3_0_1_model_4.0_r3_af2: length = 121, lDDT = 75.74, rmsd = 3.099, \n",
      "7o2g_HBA_dif_28_traj2_T0.3_0_5_model_4.0_r3_af2: length = 121, lDDT = 75.43, rmsd = 8.681, \n",
      "7o2g_HBA_dif_28_traj4_T0.3_0_4_model_4.0_r3_af2: length = 121, lDDT = 51.18, rmsd = 11.843, \n",
      "7o2g_HBA_dif_28_traj1_T0.3_0_2_model_4.0_r3_af2: length = 121, lDDT = 71.04, rmsd = 9.461, \n",
      "7o2g_HBA_dif_28_traj1_T0.3_0_4_model_4.0_r3_af2: length = 121, lDDT = 57.25, rmsd = 9.012, \n",
      "7o2g_HBA_dif_108_traj4_T0.3_0_2_model_4.0_r3_af2: length = 123, lDDT = 70.57, rmsd = 4.275, \n",
      "7o2g_HBA_dif_108_traj1_T0.3_0_1_model_4.0_r3_af2: length = 123, lDDT = 71.99, rmsd = 6.488, \n",
      "7o2g_HBA_dif_108_traj1_T0.3_0_5_model_4.0_r3_af2: length = 123, lDDT = 59.29, rmsd = 4.726, \n",
      "7o2g_HBA_dif_108_T0.3_0_4_model_4.0_r3_af2: length = 123, lDDT = 52.37, rmsd = 12.287, \n",
      "7o2g_HBA_dif_108_traj2_T0.3_0_3_model_4.0_r3_af2: length = 123, lDDT = 86.76, rmsd = 5.735, \n",
      "7o2g_HBA_dif_108_traj2_T0.3_0_4_model_4.0_r3_af2: length = 123, lDDT = 77.74, rmsd = 2.909, \n",
      "7o2g_HBA_dif_108_traj3_T0.3_0_2_model_4.0_r3_af2: length = 123, lDDT = 69.45, rmsd = 3.791, \n",
      "7o2g_HBA_dif_108_traj3_T0.3_0_3_model_4.0_r3_af2: length = 123, lDDT = 73.89, rmsd = 4.015, \n",
      "7o2g_HBA_dif_177_traj3_T0.3_0_2_model_4.0_r3_af2: length = 126, lDDT = 61.84, rmsd = 11.368, \n",
      "7o2g_HBA_dif_177_traj3_T0.3_0_5_model_4.0_r3_af2: length = 126, lDDT = 75.64, rmsd = 3.306, \n",
      "7o2g_HBA_dif_177_T0.3_0_2_model_4.0_r3_af2: length = 126, lDDT = 82.47, rmsd = 1.090, \n",
      "7o2g_HBA_dif_177_T0.3_0_5_model_4.0_r3_af2: length = 126, lDDT = 74.24, rmsd = 4.761, \n",
      "7o2g_HBA_dif_177_traj1_T0.3_0_4_model_4.0_r3_af2: length = 126, lDDT = 83.47, rmsd = 2.321, \n",
      "7o2g_HBA_dif_177_traj2_T0.3_0_2_model_4.0_r3_af2: length = 126, lDDT = 85.51, rmsd = 2.758, \n",
      "7o2g_HBA_dif_177_traj2_T0.3_0_4_model_4.0_r3_af2: length = 126, lDDT = 82.08, rmsd = 1.545, \n",
      "7o2g_HBA_dif_177_traj4_T0.3_0_1_model_4.0_r3_af2: length = 126, lDDT = 83.86, rmsd = 2.815, \n",
      "7o2g_HBA_dif_131_traj4_T0.3_0_1_model_4.0_r3_af2: length = 105, lDDT = 60.92, rmsd = 4.991, \n",
      "7o2g_HBA_dif_131_T0.3_0_3_model_4.0_r3_af2: length = 105, lDDT = 49.83, rmsd = 12.473, \n",
      "7o2g_HBA_dif_131_traj1_T0.3_0_2_model_4.0_r3_af2: length = 105, lDDT = 58.81, rmsd = 7.942, \n",
      "7o2g_HBA_dif_131_traj1_T0.3_0_3_model_4.0_r3_af2: length = 105, lDDT = 75.83, rmsd = 9.406, \n",
      "7o2g_HBA_dif_131_traj3_T0.3_0_2_model_4.0_r3_af2: length = 105, lDDT = 58.16, rmsd = 9.079, \n",
      "7o2g_HBA_dif_131_traj2_T0.3_0_1_model_4.0_r3_af2: length = 105, lDDT = 60.72, rmsd = 8.995, \n",
      "7o2g_HBA_dif_131_traj2_T0.3_0_4_model_4.0_r3_af2: length = 105, lDDT = 49.32, rmsd = 14.284, \n",
      "7o2g_HBA_dif_52_traj3_T0.3_0_2_model_4.0_r3_af2: length = 107, lDDT = 68.13, rmsd = 5.290, \n",
      "7o2g_HBA_dif_52_traj3_T0.3_0_5_model_4.0_r3_af2: length = 107, lDDT = 78.55, rmsd = 5.766, \n",
      "7o2g_HBA_dif_52_traj4_T0.3_0_3_model_4.0_r3_af2: length = 107, lDDT = 61.61, rmsd = 4.070, \n",
      "7o2g_HBA_dif_52_traj2_T0.3_0_2_model_4.0_r3_af2: length = 107, lDDT = 75.56, rmsd = 11.988, \n",
      "7o2g_HBA_dif_52_traj2_T0.3_0_3_model_4.0_r3_af2: length = 107, lDDT = 58.07, rmsd = 10.976, \n",
      "7o2g_HBA_dif_52_traj1_T0.3_0_1_model_4.0_r3_af2: length = 107, lDDT = 59.76, rmsd = 11.062, \n",
      "7o2g_HBA_dif_52_traj1_T0.3_0_4_model_4.0_r3_af2: length = 107, lDDT = 50.12, rmsd = 14.380, \n",
      "7o2g_HBA_dif_220_traj3_T0.3_0_3_model_4.0_r3_af2: length = 112, lDDT = 56.07, rmsd = 10.491, \n",
      "7o2g_HBA_dif_220_traj2_T0.3_0_1_model_4.0_r3_af2: length = 112, lDDT = 46.59, rmsd = 2.624, \n",
      "7o2g_HBA_dif_220_traj2_T0.3_0_4_model_4.0_r3_af2: length = 112, lDDT = 68.44, rmsd = 9.735, \n",
      "7o2g_HBA_dif_220_T0.3_0_2_model_4.0_r3_af2: length = 112, lDDT = 61.80, rmsd = 10.029, \n",
      "7o2g_HBA_dif_220_T0.3_0_4_model_4.0_r3_af2: length = 112, lDDT = 54.66, rmsd = 13.128, \n",
      "7o2g_HBA_dif_220_traj1_T0.3_0_2_model_4.0_r3_af2: length = 112, lDDT = 62.11, rmsd = 12.460, \n",
      "7o2g_HBA_dif_220_traj1_T0.3_0_5_model_4.0_r3_af2: length = 112, lDDT = 77.68, rmsd = 9.552, \n",
      "7o2g_HBA_dif_220_traj4_T0.3_0_3_model_4.0_r3_af2: length = 112, lDDT = 63.81, rmsd = 14.897, \n",
      "7o2g_HBA_dif_78_traj2_T0.3_0_2_model_4.0_r3_af2: length = 113, lDDT = 79.98, rmsd = 1.863, \n",
      "7o2g_HBA_dif_78_traj1_T0.3_0_1_model_4.0_r3_af2: length = 113, lDDT = 72.94, rmsd = 3.253, \n",
      "7o2g_HBA_dif_78_traj1_T0.3_0_4_model_4.0_r3_af2: length = 113, lDDT = 46.08, rmsd = 9.811, \n",
      "7o2g_HBA_dif_209_traj3_T0.3_0_2_model_4.0_r3_af2: length = 113, lDDT = 81.77, rmsd = 2.256, \n",
      "7o2g_HBA_dif_209_traj3_T0.3_0_5_model_4.0_r3_af2: length = 113, lDDT = 86.82, rmsd = 2.928, \n",
      "7o2g_HBA_dif_78_traj3_T0.3_0_2_model_4.0_r3_af2: length = 113, lDDT = 60.25, rmsd = 2.132, \n",
      "7o2g_HBA_dif_78_traj3_T0.3_0_5_model_4.0_r3_af2: length = 113, lDDT = 66.83, rmsd = 6.351, \n",
      "7o2g_HBA_dif_78_traj4_T0.3_0_4_model_4.0_r3_af2: length = 113, lDDT = 75.48, rmsd = 3.660, \n",
      "7o2g_HBA_dif_78_T0.3_0_2_model_4.0_r3_af2: length = 113, lDDT = 43.98, rmsd = 11.035, \n",
      "7o2g_HBA_dif_78_T0.3_0_5_model_4.0_r3_af2: length = 113, lDDT = 83.04, rmsd = 2.478, \n",
      "7o2g_HBA_dif_209_traj4_T0.3_0_3_model_4.0_r3_af2: length = 113, lDDT = 78.42, rmsd = 11.335, \n",
      "7o2g_HBA_dif_25_traj4_T0.3_0_1_model_4.0_r3_af2: length = 119, lDDT = 87.81, rmsd = 1.614, \n",
      "7o2g_HBA_dif_25_traj4_T0.3_0_4_model_4.0_r3_af2: length = 119, lDDT = 75.47, rmsd = 3.696, \n",
      "7o2g_HBA_dif_185_traj3_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 73.05, rmsd = 2.895, \n",
      "7o2g_HBA_dif_185_traj2_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 70.72, rmsd = 2.006, \n",
      "7o2g_HBA_dif_64_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 81.86, rmsd = 1.904, \n",
      "7o2g_HBA_dif_64_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 61.67, rmsd = 10.046, \n",
      "7o2g_HBA_dif_226_traj1_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 58.47, rmsd = 13.236, \n",
      "7o2g_HBA_dif_226_traj1_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 46.27, rmsd = 10.580, \n",
      "7o2g_HBA_dif_226_traj4_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 59.91, rmsd = 3.219, \n",
      "7o2g_HBA_dif_191_traj1_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 66.22, rmsd = 18.835, \n",
      "7o2g_HBA_dif_201_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 80.55, rmsd = 2.295, \n",
      "7o2g_HBA_dif_201_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 79.58, rmsd = 2.082, \n",
      "7o2g_HBA_dif_191_traj4_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 60.12, rmsd = 12.718, \n",
      "7o2g_HBA_dif_191_traj4_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 60.77, rmsd = 13.639, \n",
      "7o2g_HBA_dif_191_traj4_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 78.11, rmsd = 1.886, \n",
      "7o2g_HBA_dif_191_traj2_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 74.27, rmsd = 8.945, \n",
      "7o2g_HBA_dif_64_traj2_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 56.47, rmsd = 8.594, \n",
      "7o2g_HBA_dif_64_traj2_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 85.20, rmsd = 1.539, \n",
      "7o2g_HBA_dif_226_traj3_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 58.44, rmsd = 8.603, \n",
      "7o2g_HBA_dif_226_traj3_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 50.77, rmsd = 8.841, \n",
      "7o2g_HBA_dif_191_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 71.99, rmsd = 3.284, \n",
      "7o2g_HBA_dif_191_traj3_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 65.44, rmsd = 5.290, \n",
      "7o2g_HBA_dif_191_traj3_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 78.25, rmsd = 6.941, \n",
      "7o2g_HBA_dif_185_traj1_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 72.62, rmsd = 2.604, \n",
      "7o2g_HBA_dif_185_traj1_T0.3_0_4_model_4.0_r3_af2: length = 120, lDDT = 71.82, rmsd = 4.947, \n",
      "7o2g_HBA_dif_64_traj4_T0.3_0_3_model_4.0_r3_af2: length = 120, lDDT = 54.91, rmsd = 12.081, \n",
      "7o2g_HBA_dif_64_traj1_T0.3_0_2_model_4.0_r3_af2: length = 120, lDDT = 47.35, rmsd = 13.876, \n",
      "7o2g_HBA_dif_64_traj1_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 74.50, rmsd = 11.370, \n",
      "7o2g_HBA_dif_64_traj3_T0.3_0_1_model_4.0_r3_af2: length = 120, lDDT = 57.69, rmsd = 7.438, \n",
      "7o2g_HBA_dif_64_traj3_T0.3_0_5_model_4.0_r3_af2: length = 120, lDDT = 63.31, rmsd = 12.526, \n",
      "7o2g_HBA_dif_28_T0.3_0_4_model_4.0_r3_af2: length = 121, lDDT = 73.03, rmsd = 5.043, \n",
      "7o2g_HBA_dif_28_traj3_T0.3_0_3_model_4.0_r3_af2: length = 121, lDDT = 78.22, rmsd = 1.965, \n",
      "7o2g_HBA_dif_28_traj3_T0.3_0_5_model_4.0_r3_af2: length = 121, lDDT = 79.28, rmsd = 2.953, \n",
      "7o2g_HBA_dif_28_traj2_T0.3_0_2_model_4.0_r3_af2: length = 121, lDDT = 87.87, rmsd = 0.772, \n",
      "7o2g_HBA_dif_28_traj2_T0.3_0_4_model_4.0_r3_af2: length = 121, lDDT = 58.74, rmsd = 7.885, \n",
      "7o2g_HBA_dif_28_traj4_T0.3_0_2_model_4.0_r3_af2: length = 121, lDDT = 49.26, rmsd = 7.268, \n",
      "7o2g_HBA_dif_28_traj1_T0.3_0_1_model_4.0_r3_af2: length = 121, lDDT = 79.84, rmsd = 1.240, \n",
      "7o2g_HBA_dif_28_traj1_T0.3_0_5_model_4.0_r3_af2: length = 121, lDDT = 49.96, rmsd = 14.620, \n",
      "7o2g_HBA_dif_108_traj4_T0.3_0_3_model_4.0_r3_af2: length = 123, lDDT = 55.33, rmsd = 14.214, \n",
      "7o2g_HBA_dif_108_traj4_T0.3_0_5_model_4.0_r3_af2: length = 123, lDDT = 52.82, rmsd = 6.309, \n",
      "7o2g_HBA_dif_108_traj1_T0.3_0_3_model_4.0_r3_af2: length = 123, lDDT = 59.92, rmsd = 5.593, \n",
      "7o2g_HBA_dif_108_T0.3_0_2_model_4.0_r3_af2: length = 123, lDDT = 77.05, rmsd = 14.037, \n",
      "7o2g_HBA_dif_108_T0.3_0_5_model_4.0_r3_af2: length = 123, lDDT = 61.38, rmsd = 4.569, \n",
      "7o2g_HBA_dif_108_traj2_T0.3_0_1_model_4.0_r3_af2: length = 123, lDDT = 69.19, rmsd = 2.158, \n",
      "7o2g_HBA_dif_108_traj3_T0.3_0_1_model_4.0_r3_af2: length = 123, lDDT = 76.30, rmsd = 2.795, \n",
      "7o2g_HBA_dif_108_traj3_T0.3_0_5_model_4.0_r3_af2: length = 123, lDDT = 63.02, rmsd = 6.925, \n",
      "7o2g_HBA_dif_177_traj3_T0.3_0_1_model_4.0_r3_af2: length = 126, lDDT = 83.37, rmsd = 2.647, \n",
      "7o2g_HBA_dif_177_traj3_T0.3_0_4_model_4.0_r3_af2: length = 126, lDDT = 85.76, rmsd = 1.121, \n",
      "7o2g_HBA_dif_177_T0.3_0_3_model_4.0_r3_af2: length = 126, lDDT = 82.11, rmsd = 2.375, \n",
      "7o2g_HBA_dif_177_traj1_T0.3_0_1_model_4.0_r3_af2: length = 126, lDDT = 90.77, rmsd = 2.115, \n",
      "7o2g_HBA_dif_177_traj1_T0.3_0_3_model_4.0_r3_af2: length = 126, lDDT = 88.90, rmsd = 1.865, \n",
      "7o2g_HBA_dif_177_traj1_T0.3_0_5_model_4.0_r3_af2: length = 126, lDDT = 84.80, rmsd = 2.082, \n",
      "7o2g_HBA_dif_177_traj2_T0.3_0_3_model_4.0_r3_af2: length = 126, lDDT = 85.72, rmsd = 2.078, \n",
      "7o2g_HBA_dif_177_traj4_T0.3_0_2_model_4.0_r3_af2: length = 126, lDDT = 81.77, rmsd = 1.106, \n",
      "7o2g_HBA_dif_73_traj3_T0.3_0_1_model_4.0_r3_af2: length = 127, lDDT = 82.08, rmsd = 3.002, \n",
      "7o2g_HBA_dif_73_traj3_T0.3_0_4_model_4.0_r3_af2: length = 127, lDDT = 86.11, rmsd = 2.145, \n",
      "7o2g_HBA_dif_177_traj2_T0.3_0_1_model_4.0_r3_af2: length = 126, lDDT = 69.89, rmsd = 3.999, \n",
      "7o2g_HBA_dif_177_traj2_T0.3_0_5_model_4.0_r3_af2: length = 126, lDDT = 84.43, rmsd = 1.320, \n",
      "7o2g_HBA_dif_177_traj4_T0.3_0_4_model_4.0_r3_af2: length = 126, lDDT = 63.62, rmsd = 15.798, \n",
      "7o2g_HBA_dif_73_traj3_T0.3_0_2_model_4.0_r3_af2: length = 127, lDDT = 87.09, rmsd = 2.046, \n",
      "7o2g_HBA_dif_73_traj3_T0.3_0_5_model_4.0_r3_af2: length = 127, lDDT = 87.32, rmsd = 1.733, \n",
      "7o2g_HBA_dif_73_T0.3_0_2_model_4.0_r3_af2: length = 127, lDDT = 80.94, rmsd = 1.898, \n",
      "7o2g_HBA_dif_73_traj2_T0.3_0_1_model_4.0_r3_af2: length = 127, lDDT = 79.09, rmsd = 2.304, \n",
      "7o2g_HBA_dif_73_traj2_T0.3_0_4_model_4.0_r3_af2: length = 127, lDDT = 85.07, rmsd = 1.826, \n",
      "7o2g_HBA_dif_73_traj4_T0.3_0_3_model_4.0_r3_af2: length = 127, lDDT = 89.12, rmsd = 2.682, \n",
      "7o2g_HBA_dif_73_traj1_T0.3_0_1_model_4.0_r3_af2: length = 127, lDDT = 85.97, rmsd = 1.968, \n",
      "7o2g_HBA_dif_73_traj1_T0.3_0_4_model_4.0_r3_af2: length = 127, lDDT = 84.17, rmsd = 1.956, \n",
      "7o2g_HBA_dif_97_traj4_T0.3_0_2_model_4.0_r3_af2: length = 127, lDDT = 66.03, rmsd = 4.265, \n",
      "7o2g_HBA_dif_97_traj4_T0.3_0_3_model_4.0_r3_af2: length = 127, lDDT = 79.42, rmsd = 2.653, \n",
      "7o2g_HBA_dif_97_T0.3_0_2_model_4.0_r3_af2: length = 127, lDDT = 67.42, rmsd = 7.034, \n",
      "7o2g_HBA_dif_97_T0.3_0_5_model_4.0_r3_af2: length = 127, lDDT = 63.90, rmsd = 8.935, \n",
      "7o2g_HBA_dif_43_traj3_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 66.57, rmsd = 4.392, \n",
      "7o2g_HBA_dif_43_traj3_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 48.43, rmsd = 10.546, \n",
      "7o2g_HBA_dif_43_traj4_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 76.12, rmsd = 2.866, \n",
      "7o2g_HBA_dif_43_traj1_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 65.66, rmsd = 3.170, \n",
      "7o2g_HBA_dif_43_traj1_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 72.66, rmsd = 3.020, \n",
      "7o2g_HBA_dif_105_traj2_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 60.66, rmsd = 6.285, \n",
      "7o2g_HBA_dif_105_traj2_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 55.92, rmsd = 12.941, \n",
      "7o2g_HBA_dif_105_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 59.28, rmsd = 8.690, \n",
      "7o2g_HBA_dif_105_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 55.52, rmsd = 11.341, \n",
      "7o2g_HBA_dif_43_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 64.79, rmsd = 5.617, \n",
      "7o2g_HBA_dif_105_traj1_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 48.67, rmsd = 10.352, \n",
      "7o2g_HBA_dif_105_traj3_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 55.21, rmsd = 9.505, \n",
      "7o2g_HBA_dif_105_traj3_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 53.49, rmsd = 3.248, \n",
      "7o2g_HBA_dif_43_traj2_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 51.13, rmsd = 9.051, \n",
      "7o2g_HBA_dif_43_traj2_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 87.80, rmsd = 2.158, \n",
      "7o2g_HBA_dif_105_traj4_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 49.45, rmsd = 11.821, \n",
      "7o2g_HBA_dif_105_traj4_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 54.54, rmsd = 10.315, \n",
      "7o2g_HBA_dif_130_traj2_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 70.87, rmsd = 3.601, \n",
      "7o2g_HBA_dif_130_traj2_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 90.98, rmsd = 0.944, \n",
      "7o2g_HBA_dif_69_traj4_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 80.91, rmsd = 3.078, \n",
      "7o2g_HBA_dif_69_traj2_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 65.00, rmsd = 2.108, \n",
      "7o2g_HBA_dif_148_traj4_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 70.32, rmsd = 3.257, \n",
      "7o2g_HBA_dif_148_traj4_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 84.53, rmsd = 4.012, \n",
      "7o2g_HBA_dif_148_traj4_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 75.54, rmsd = 11.169, \n",
      "7o2g_HBA_dif_130_traj4_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 87.19, rmsd = 1.216, \n",
      "7o2g_HBA_dif_130_traj1_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 63.65, rmsd = 7.665, \n",
      "7o2g_HBA_dif_148_traj2_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 57.86, rmsd = 10.640, \n",
      "7o2g_HBA_dif_148_traj2_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 75.40, rmsd = 10.929, \n",
      "7o2g_HBA_dif_148_traj2_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 60.57, rmsd = 10.493, \n",
      "7o2g_HBA_dif_130_traj3_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 66.76, rmsd = 9.956, \n",
      "7o2g_HBA_dif_69_traj3_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 57.23, rmsd = 6.016, \n",
      "7o2g_HBA_dif_130_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 76.23, rmsd = 3.268, \n",
      "7o2g_HBA_dif_130_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 77.81, rmsd = 3.176, \n",
      "7o2g_HBA_dif_158_traj2_T0.3_0_3_model_4.0_r3_af2: length = 132, lDDT = 65.06, rmsd = 4.011, \n",
      "7o2g_HBA_dif_158_traj3_T0.3_0_1_model_4.0_r3_af2: length = 132, lDDT = 49.96, rmsd = 11.185, \n",
      "7o2g_HBA_dif_158_traj3_T0.3_0_3_model_4.0_r3_af2: length = 132, lDDT = 82.75, rmsd = 2.568, \n",
      "7o2g_HBA_dif_158_traj4_T0.3_0_1_model_4.0_r3_af2: length = 132, lDDT = 67.14, rmsd = 4.814, \n",
      "7o2g_HBA_dif_158_traj4_T0.3_0_3_model_4.0_r3_af2: length = 132, lDDT = 50.87, rmsd = 11.317, \n",
      "7o2g_HBA_dif_216_traj3_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 57.24, rmsd = 9.292, \n",
      "7o2g_HBA_dif_216_traj3_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 58.43, rmsd = 12.282, \n",
      "7o2g_HBA_dif_24_traj4_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 68.02, rmsd = 4.643, \n",
      "7o2g_HBA_dif_24_traj4_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 84.87, rmsd = 3.816, \n",
      "7o2g_HBA_dif_147_traj2_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 77.54, rmsd = 2.874, \n",
      "7o2g_HBA_dif_147_traj2_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 77.73, rmsd = 4.552, \n",
      "7o2g_HBA_dif_24_traj2_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 75.68, rmsd = 2.759, \n",
      "7o2g_HBA_dif_24_traj2_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 82.72, rmsd = 1.888, \n",
      "7o2g_HBA_dif_147_traj4_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 77.57, rmsd = 1.742, \n",
      "7o2g_HBA_dif_227_traj3_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 73.51, rmsd = 4.220, \n",
      "7o2g_HBA_dif_227_traj3_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 74.98, rmsd = 4.030, \n",
      "7o2g_HBA_dif_89_traj3_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 72.41, rmsd = 2.185, \n",
      "7o2g_HBA_dif_24_traj3_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 81.98, rmsd = 1.520, \n",
      "7o2g_HBA_dif_24_traj3_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 85.69, rmsd = 3.029, \n",
      "7o2g_HBA_dif_95_traj2_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 80.77, rmsd = 1.939, \n",
      "7o2g_HBA_dif_95_traj2_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 47.55, rmsd = 8.810, \n",
      "7o2g_HBA_dif_89_traj2_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 81.67, rmsd = 2.844, \n",
      "7o2g_HBA_dif_89_traj2_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 74.16, rmsd = 6.334, \n",
      "7o2g_HBA_dif_227_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 69.88, rmsd = 6.917, \n",
      "7o2g_HBA_dif_216_traj2_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 56.18, rmsd = 9.455, \n",
      "7o2g_HBA_dif_216_traj2_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 55.23, rmsd = 16.004, \n",
      "7o2g_HBA_dif_89_traj4_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 68.49, rmsd = 2.734, \n",
      "7o2g_HBA_dif_89_traj4_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 74.95, rmsd = 3.377, \n",
      "7o2g_HBA_dif_95_traj4_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 51.82, rmsd = 11.171, \n",
      "7o2g_HBA_dif_89_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 75.30, rmsd = 2.351, \n",
      "7o2g_HBA_dif_89_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 72.98, rmsd = 2.130, \n",
      "7o2g_HBA_dif_216_traj4_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 56.58, rmsd = 10.259, \n",
      "7o2g_HBA_dif_147_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 78.98, rmsd = 3.495, \n",
      "7o2g_HBA_dif_147_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 77.56, rmsd = 4.845, \n",
      "7o2g_HBA_dif_216_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 43.55, rmsd = 15.389, \n",
      "7o2g_HBA_dif_227_traj4_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 65.66, rmsd = 14.237, \n",
      "7o2g_HBA_dif_227_traj4_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 72.99, rmsd = 3.702, \n",
      "7o2g_HBA_dif_89_traj1_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 73.17, rmsd = 4.265, \n",
      "7o2g_HBA_dif_89_traj1_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 82.37, rmsd = 3.221, \n",
      "7o2g_HBA_dif_89_traj1_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 72.17, rmsd = 6.681, \n",
      "7o2g_HBA_dif_147_traj3_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 83.92, rmsd = 4.713, \n",
      "7o2g_HBA_dif_177_traj4_T0.3_0_3_model_4.0_r3_af2: length = 126, lDDT = 83.02, rmsd = 3.058, \n",
      "7o2g_HBA_dif_177_traj4_T0.3_0_5_model_4.0_r3_af2: length = 126, lDDT = 69.71, rmsd = 2.127, \n",
      "7o2g_HBA_dif_73_traj3_T0.3_0_3_model_4.0_r3_af2: length = 127, lDDT = 89.91, rmsd = 2.132, \n",
      "7o2g_HBA_dif_73_T0.3_0_1_model_4.0_r3_af2: length = 127, lDDT = 82.30, rmsd = 1.576, \n",
      "7o2g_HBA_dif_73_T0.3_0_5_model_4.0_r3_af2: length = 127, lDDT = 72.83, rmsd = 3.168, \n",
      "7o2g_HBA_dif_73_traj2_T0.3_0_2_model_4.0_r3_af2: length = 127, lDDT = 86.44, rmsd = 2.525, \n",
      "7o2g_HBA_dif_73_traj2_T0.3_0_5_model_4.0_r3_af2: length = 127, lDDT = 83.35, rmsd = 2.959, \n",
      "7o2g_HBA_dif_73_traj4_T0.3_0_2_model_4.0_r3_af2: length = 127, lDDT = 88.07, rmsd = 1.704, \n",
      "7o2g_HBA_dif_73_traj4_T0.3_0_4_model_4.0_r3_af2: length = 127, lDDT = 88.37, rmsd = 1.507, \n",
      "7o2g_HBA_dif_73_traj1_T0.3_0_2_model_4.0_r3_af2: length = 127, lDDT = 89.18, rmsd = 2.661, \n",
      "7o2g_HBA_dif_97_traj4_T0.3_0_1_model_4.0_r3_af2: length = 127, lDDT = 80.01, rmsd = 2.375, \n",
      "7o2g_HBA_dif_97_traj4_T0.3_0_5_model_4.0_r3_af2: length = 127, lDDT = 72.15, rmsd = 2.798, \n",
      "7o2g_HBA_dif_97_T0.3_0_1_model_4.0_r3_af2: length = 127, lDDT = 68.11, rmsd = 7.511, \n",
      "7o2g_HBA_dif_97_T0.3_0_4_model_4.0_r3_af2: length = 127, lDDT = 63.71, rmsd = 2.671, \n",
      "7o2g_HBA_dif_43_traj3_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 62.98, rmsd = 3.630, \n",
      "7o2g_HBA_dif_43_traj4_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 60.88, rmsd = 4.102, \n",
      "7o2g_HBA_dif_43_traj4_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 67.78, rmsd = 2.817, \n",
      "7o2g_HBA_dif_43_traj1_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 74.13, rmsd = 3.029, \n",
      "7o2g_HBA_dif_43_traj1_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 61.03, rmsd = 5.135, \n",
      "7o2g_HBA_dif_105_traj2_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 51.28, rmsd = 12.922, \n",
      "7o2g_HBA_dif_105_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 55.19, rmsd = 11.378, \n",
      "7o2g_HBA_dif_43_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 70.34, rmsd = 3.590, \n",
      "7o2g_HBA_dif_43_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 78.41, rmsd = 5.752, \n",
      "7o2g_HBA_dif_105_traj1_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 44.67, rmsd = 14.784, \n",
      "7o2g_HBA_dif_105_traj1_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 55.12, rmsd = 11.563, \n",
      "7o2g_HBA_dif_105_traj1_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 53.41, rmsd = 9.403, \n",
      "7o2g_HBA_dif_105_traj3_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 77.10, rmsd = 2.440, \n",
      "7o2g_HBA_dif_43_traj2_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 55.61, rmsd = 5.462, \n",
      "7o2g_HBA_dif_105_traj4_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 59.63, rmsd = 13.211, \n",
      "7o2g_HBA_dif_105_traj4_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 49.91, rmsd = 13.940, \n",
      "7o2g_HBA_dif_130_traj2_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 53.54, rmsd = 9.591, \n",
      "7o2g_HBA_dif_130_traj2_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 59.40, rmsd = 9.539, \n",
      "7o2g_HBA_dif_69_traj4_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 54.26, rmsd = 4.352, \n",
      "7o2g_HBA_dif_69_traj4_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 63.74, rmsd = 2.432, \n",
      "7o2g_HBA_dif_69_traj2_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 55.64, rmsd = 8.762, \n",
      "7o2g_HBA_dif_69_traj2_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 76.84, rmsd = 8.885, \n",
      "7o2g_HBA_dif_148_traj4_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 77.63, rmsd = 4.391, \n",
      "7o2g_HBA_dif_130_traj4_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 86.93, rmsd = 1.400, \n",
      "7o2g_HBA_dif_130_traj4_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 76.52, rmsd = 3.460, \n",
      "7o2g_HBA_dif_130_traj1_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 77.52, rmsd = 4.988, \n",
      "7o2g_HBA_dif_130_traj1_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 84.80, rmsd = 1.409, \n",
      "7o2g_HBA_dif_148_traj2_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 80.35, rmsd = 11.303, \n",
      "7o2g_HBA_dif_130_traj3_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 79.40, rmsd = 1.686, \n",
      "7o2g_HBA_dif_130_traj3_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 59.16, rmsd = 12.565, \n",
      "7o2g_HBA_dif_69_traj3_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 69.89, rmsd = 2.986, \n",
      "7o2g_HBA_dif_69_traj3_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 54.66, rmsd = 12.992, \n",
      "7o2g_HBA_dif_130_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 69.25, rmsd = 17.504, \n",
      "7o2g_HBA_dif_158_traj2_T0.3_0_1_model_4.0_r3_af2: length = 132, lDDT = 60.00, rmsd = 14.937, \n",
      "7o2g_HBA_dif_158_traj2_T0.3_0_4_model_4.0_r3_af2: length = 132, lDDT = 87.25, rmsd = 1.842, \n",
      "7o2g_HBA_dif_158_traj3_T0.3_0_2_model_4.0_r3_af2: length = 132, lDDT = 67.47, rmsd = 11.130, \n",
      "7o2g_HBA_dif_158_traj3_T0.3_0_5_model_4.0_r3_af2: length = 132, lDDT = 80.98, rmsd = 9.272, \n",
      "7o2g_HBA_dif_158_traj4_T0.3_0_4_model_4.0_r3_af2: length = 132, lDDT = 64.32, rmsd = 7.567, \n",
      "7o2g_HBA_dif_216_traj3_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 46.21, rmsd = 15.236, \n",
      "7o2g_HBA_dif_24_traj4_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 82.08, rmsd = 2.318, \n",
      "7o2g_HBA_dif_24_traj4_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 78.11, rmsd = 8.112, \n",
      "7o2g_HBA_dif_147_traj2_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 79.53, rmsd = 3.470, \n",
      "7o2g_HBA_dif_24_traj2_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 73.54, rmsd = 3.107, \n",
      "7o2g_HBA_dif_24_traj2_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 78.47, rmsd = 2.380, \n",
      "7o2g_HBA_dif_147_traj4_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 82.41, rmsd = 2.145, \n",
      "7o2g_HBA_dif_147_traj4_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 78.30, rmsd = 2.514, \n",
      "7o2g_HBA_dif_227_traj3_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 66.56, rmsd = 3.562, \n",
      "7o2g_HBA_dif_227_traj3_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 69.67, rmsd = 6.856, \n",
      "7o2g_HBA_dif_89_traj3_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 68.06, rmsd = 4.918, \n",
      "7o2g_HBA_dif_89_traj3_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 70.93, rmsd = 3.742, \n",
      "7o2g_HBA_dif_24_traj3_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 82.55, rmsd = 1.740, \n",
      "7o2g_HBA_dif_24_traj3_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 72.99, rmsd = 9.906, \n",
      "7o2g_HBA_dif_95_traj2_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 59.04, rmsd = 13.778, \n",
      "7o2g_HBA_dif_89_traj2_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 77.66, rmsd = 2.472, \n",
      "7o2g_HBA_dif_89_traj2_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 73.52, rmsd = 1.999, \n",
      "7o2g_HBA_dif_227_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 70.11, rmsd = 3.563, \n",
      "7o2g_HBA_dif_227_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 85.03, rmsd = 4.484, \n",
      "7o2g_HBA_dif_216_traj2_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 50.89, rmsd = 13.839, \n",
      "7o2g_HBA_dif_216_traj2_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 52.25, rmsd = 10.904, \n",
      "7o2g_HBA_dif_89_traj4_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 72.44, rmsd = 5.979, \n",
      "7o2g_HBA_dif_95_traj4_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 77.61, rmsd = 5.682, \n",
      "7o2g_HBA_dif_95_traj4_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 78.74, rmsd = 3.893, \n",
      "7o2g_HBA_dif_89_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 77.68, rmsd = 2.158, \n",
      "7o2g_HBA_dif_216_traj4_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 53.92, rmsd = 9.260, \n",
      "7o2g_HBA_dif_216_traj4_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 56.37, rmsd = 6.380, \n",
      "7o2g_HBA_dif_147_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 87.93, rmsd = 2.822, \n",
      "7o2g_HBA_dif_147_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 85.86, rmsd = 1.994, \n",
      "7o2g_HBA_dif_216_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 52.07, rmsd = 13.216, \n",
      "7o2g_HBA_dif_216_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 45.00, rmsd = 15.036, \n",
      "7o2g_HBA_dif_227_traj4_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 73.40, rmsd = 4.380, \n",
      "7o2g_HBA_dif_227_traj4_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 70.87, rmsd = 3.543, \n",
      "7o2g_HBA_dif_89_traj1_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 79.22, rmsd = 3.665, \n",
      "7o2g_HBA_dif_147_traj3_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 82.28, rmsd = 2.676, \n",
      "7o2g_HBA_dif_147_traj3_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 89.54, rmsd = 2.579, \n",
      "7o2g_HBA_dif_227_traj1_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 71.65, rmsd = 6.382, \n",
      "7o2g_HBA_dif_73_T0.3_0_3_model_4.0_r3_af2: length = 127, lDDT = 88.75, rmsd = 1.297, \n",
      "7o2g_HBA_dif_73_T0.3_0_4_model_4.0_r3_af2: length = 127, lDDT = 69.06, rmsd = 4.993, \n",
      "7o2g_HBA_dif_73_traj2_T0.3_0_3_model_4.0_r3_af2: length = 127, lDDT = 90.77, rmsd = 0.944, \n",
      "7o2g_HBA_dif_73_traj4_T0.3_0_1_model_4.0_r3_af2: length = 127, lDDT = 87.51, rmsd = 3.194, \n",
      "7o2g_HBA_dif_73_traj4_T0.3_0_5_model_4.0_r3_af2: length = 127, lDDT = 89.35, rmsd = 1.965, \n",
      "7o2g_HBA_dif_73_traj1_T0.3_0_3_model_4.0_r3_af2: length = 127, lDDT = 88.63, rmsd = 1.287, \n",
      "7o2g_HBA_dif_73_traj1_T0.3_0_5_model_4.0_r3_af2: length = 127, lDDT = 90.28, rmsd = 3.771, \n",
      "7o2g_HBA_dif_97_traj4_T0.3_0_4_model_4.0_r3_af2: length = 127, lDDT = 77.45, rmsd = 2.970, \n",
      "7o2g_HBA_dif_97_T0.3_0_3_model_4.0_r3_af2: length = 127, lDDT = 60.46, rmsd = 3.458, \n",
      "7o2g_HBA_dif_43_traj3_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 65.19, rmsd = 2.567, \n",
      "7o2g_HBA_dif_43_traj3_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 59.61, rmsd = 5.440, \n",
      "7o2g_HBA_dif_43_traj4_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 71.44, rmsd = 1.805, \n",
      "7o2g_HBA_dif_43_traj4_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 68.13, rmsd = 2.665, \n",
      "7o2g_HBA_dif_43_traj1_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 57.76, rmsd = 6.935, \n",
      "7o2g_HBA_dif_105_traj2_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 44.27, rmsd = 14.142, \n",
      "7o2g_HBA_dif_105_traj2_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 48.84, rmsd = 9.009, \n",
      "7o2g_HBA_dif_105_T0.3_0_1_model_4.0_r3_af2: length = 128, lDDT = 51.62, rmsd = 12.046, \n",
      "7o2g_HBA_dif_105_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 69.33, rmsd = 15.671, \n",
      "7o2g_HBA_dif_43_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 81.45, rmsd = 3.250, \n",
      "7o2g_HBA_dif_43_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 63.96, rmsd = 4.081, \n",
      "7o2g_HBA_dif_105_traj1_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 53.20, rmsd = 9.618, \n",
      "7o2g_HBA_dif_105_traj3_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 51.86, rmsd = 12.117, \n",
      "7o2g_HBA_dif_105_traj3_T0.3_0_3_model_4.0_r3_af2: length = 128, lDDT = 72.50, rmsd = 10.601, \n",
      "7o2g_HBA_dif_43_traj2_T0.3_0_2_model_4.0_r3_af2: length = 128, lDDT = 66.17, rmsd = 3.204, \n",
      "7o2g_HBA_dif_43_traj2_T0.3_0_5_model_4.0_r3_af2: length = 128, lDDT = 72.41, rmsd = 7.573, \n",
      "7o2g_HBA_dif_105_traj4_T0.3_0_4_model_4.0_r3_af2: length = 128, lDDT = 49.94, rmsd = 8.858, \n",
      "7o2g_HBA_dif_130_traj2_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 76.72, rmsd = 10.909, \n",
      "7o2g_HBA_dif_69_traj4_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 75.09, rmsd = 2.723, \n",
      "7o2g_HBA_dif_69_traj4_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 77.30, rmsd = 2.407, \n",
      "7o2g_HBA_dif_69_traj2_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 74.18, rmsd = 7.344, \n",
      "7o2g_HBA_dif_69_traj2_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 82.88, rmsd = 1.915, \n",
      "7o2g_HBA_dif_148_traj4_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 68.73, rmsd = 10.281, \n",
      "7o2g_HBA_dif_130_traj4_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 83.70, rmsd = 2.917, \n",
      "7o2g_HBA_dif_130_traj4_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 88.67, rmsd = 1.470, \n",
      "7o2g_HBA_dif_130_traj1_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 75.48, rmsd = 2.637, \n",
      "7o2g_HBA_dif_130_traj1_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 70.53, rmsd = 2.464, \n",
      "7o2g_HBA_dif_148_traj2_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 75.58, rmsd = 11.843, \n",
      "7o2g_HBA_dif_130_traj3_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 71.69, rmsd = 3.215, \n",
      "7o2g_HBA_dif_130_traj3_T0.3_0_3_model_4.0_r3_af2: length = 131, lDDT = 60.42, rmsd = 9.095, \n",
      "7o2g_HBA_dif_69_traj3_T0.3_0_1_model_4.0_r3_af2: length = 131, lDDT = 80.25, rmsd = 2.204, \n",
      "7o2g_HBA_dif_69_traj3_T0.3_0_4_model_4.0_r3_af2: length = 131, lDDT = 60.57, rmsd = 9.403, \n",
      "7o2g_HBA_dif_130_T0.3_0_2_model_4.0_r3_af2: length = 131, lDDT = 80.47, rmsd = 3.644, \n",
      "7o2g_HBA_dif_130_T0.3_0_5_model_4.0_r3_af2: length = 131, lDDT = 86.52, rmsd = 3.090, \n",
      "7o2g_HBA_dif_158_traj2_T0.3_0_2_model_4.0_r3_af2: length = 132, lDDT = 63.12, rmsd = 4.267, \n",
      "7o2g_HBA_dif_158_traj2_T0.3_0_5_model_4.0_r3_af2: length = 132, lDDT = 69.56, rmsd = 3.466, \n",
      "7o2g_HBA_dif_158_traj3_T0.3_0_4_model_4.0_r3_af2: length = 132, lDDT = 63.52, rmsd = 13.999, \n",
      "7o2g_HBA_dif_158_traj4_T0.3_0_2_model_4.0_r3_af2: length = 132, lDDT = 56.05, rmsd = 5.989, \n",
      "7o2g_HBA_dif_158_traj4_T0.3_0_5_model_4.0_r3_af2: length = 132, lDDT = 62.83, rmsd = 15.430, \n",
      "7o2g_HBA_dif_216_traj3_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 45.11, rmsd = 13.299, \n",
      "7o2g_HBA_dif_216_traj3_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 45.35, rmsd = 14.832, \n",
      "7o2g_HBA_dif_24_traj4_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 87.53, rmsd = 3.204, \n",
      "7o2g_HBA_dif_147_traj2_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 79.72, rmsd = 5.535, \n",
      "7o2g_HBA_dif_147_traj2_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 87.66, rmsd = 2.585, \n",
      "7o2g_HBA_dif_24_traj2_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 60.34, rmsd = 5.645, \n",
      "7o2g_HBA_dif_147_traj4_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 81.48, rmsd = 1.821, \n",
      "7o2g_HBA_dif_147_traj4_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 80.44, rmsd = 3.477, \n",
      "7o2g_HBA_dif_227_traj3_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 78.28, rmsd = 4.209, \n",
      "7o2g_HBA_dif_89_traj3_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 62.63, rmsd = 6.995, \n",
      "7o2g_HBA_dif_89_traj3_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 64.07, rmsd = 8.057, \n",
      "7o2g_HBA_dif_24_traj3_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 78.70, rmsd = 1.745, \n",
      "7o2g_HBA_dif_95_traj2_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 77.30, rmsd = 12.260, \n",
      "7o2g_HBA_dif_95_traj2_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 61.93, rmsd = 7.441, \n",
      "7o2g_HBA_dif_89_traj2_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 67.00, rmsd = 5.182, \n",
      "7o2g_HBA_dif_227_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 72.63, rmsd = 4.138, \n",
      "7o2g_HBA_dif_227_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 77.65, rmsd = 4.710, \n",
      "7o2g_HBA_dif_216_traj2_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 51.15, rmsd = 8.336, \n",
      "7o2g_HBA_dif_89_traj4_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 87.69, rmsd = 2.440, \n",
      "7o2g_HBA_dif_89_traj4_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 70.37, rmsd = 9.965, \n",
      "7o2g_HBA_dif_95_traj4_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 71.69, rmsd = 2.763, \n",
      "7o2g_HBA_dif_95_traj4_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 72.94, rmsd = 4.823, \n",
      "7o2g_HBA_dif_89_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 74.67, rmsd = 2.917, \n",
      "7o2g_HBA_dif_89_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 69.70, rmsd = 5.102, \n",
      "7o2g_HBA_dif_216_traj4_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 48.08, rmsd = 14.647, \n",
      "7o2g_HBA_dif_216_traj4_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 49.65, rmsd = 10.020, \n",
      "7o2g_HBA_dif_147_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 89.14, rmsd = 1.538, \n",
      "7o2g_HBA_dif_216_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 50.50, rmsd = 17.224, \n",
      "7o2g_HBA_dif_216_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 44.32, rmsd = 15.628, \n",
      "7o2g_HBA_dif_227_traj4_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 74.00, rmsd = 5.691, \n",
      "7o2g_HBA_dif_89_traj1_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 70.92, rmsd = 5.854, \n",
      "7o2g_HBA_dif_147_traj3_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 80.71, rmsd = 3.575, \n",
      "7o2g_HBA_dif_147_traj3_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 82.73, rmsd = 2.933, \n",
      "7o2g_HBA_dif_227_traj1_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 52.41, rmsd = 10.646, \n",
      "7o2g_HBA_dif_227_traj1_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 78.00, rmsd = 2.270, \n",
      "7o2g_HBA_dif_227_traj1_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 73.13, rmsd = 9.828, \n",
      "7o2g_HBA_dif_147_traj1_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 85.29, rmsd = 3.665, \n",
      "7o2g_HBA_dif_24_traj1_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 68.41, rmsd = 7.699, \n",
      "7o2g_HBA_dif_24_traj1_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 85.85, rmsd = 3.600, \n",
      "7o2g_HBA_dif_216_traj1_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 42.79, rmsd = 11.675, \n",
      "7o2g_HBA_dif_24_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 68.03, rmsd = 10.951, \n",
      "7o2g_HBA_dif_147_traj1_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 84.13, rmsd = 2.958, \n",
      "7o2g_HBA_dif_147_traj1_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 85.85, rmsd = 2.461, \n",
      "7o2g_HBA_dif_24_traj1_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 86.73, rmsd = 3.015, \n",
      "7o2g_HBA_dif_216_traj1_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 53.47, rmsd = 9.744, \n",
      "7o2g_HBA_dif_216_traj1_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 44.63, rmsd = 8.977, \n",
      "7o2g_HBA_dif_24_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 72.40, rmsd = 9.440, \n",
      "7o2g_HBA_dif_24_T0.3_0_5_model_4.0_r3_af2: length = 133, lDDT = 82.59, rmsd = 1.665, \n",
      "7o2g_HBA_dif_45_traj1_T0.3_0_3_model_4.0_r3_af2: length = 134, lDDT = 64.65, rmsd = 2.570, \n",
      "7o2g_HBA_dif_45_traj2_T0.3_0_1_model_4.0_r3_af2: length = 134, lDDT = 58.53, rmsd = 9.080, \n",
      "7o2g_HBA_dif_45_traj2_T0.3_0_3_model_4.0_r3_af2: length = 134, lDDT = 52.61, rmsd = 16.684, \n",
      "7o2g_HBA_dif_45_traj2_T0.3_0_5_model_4.0_r3_af2: length = 134, lDDT = 53.14, rmsd = 15.941, \n",
      "7o2g_HBA_dif_45_traj3_T0.3_0_2_model_4.0_r3_af2: length = 134, lDDT = 52.15, rmsd = 17.261, \n",
      "7o2g_HBA_dif_45_traj4_T0.3_0_1_model_4.0_r3_af2: length = 134, lDDT = 60.76, rmsd = 11.679, \n",
      "7o2g_HBA_dif_45_traj4_T0.3_0_5_model_4.0_r3_af2: length = 134, lDDT = 55.62, rmsd = 9.374, \n",
      "7o2g_HBA_dif_45_T0.3_0_3_model_4.0_r3_af2: length = 134, lDDT = 58.82, rmsd = 14.816, \n",
      "7o2g_HBA_dif_45_T0.3_0_5_model_4.0_r3_af2: length = 134, lDDT = 62.34, rmsd = 6.346, \n",
      "7o2g_HBA_dif_1_traj4_T0.3_0_3_model_4.0_r3_af2: length = 135, lDDT = 80.89, rmsd = 3.693, \n",
      "7o2g_HBA_dif_1_traj1_T0.3_0_2_model_4.0_r3_af2: length = 135, lDDT = 82.06, rmsd = 4.646, \n",
      "7o2g_HBA_dif_173_T0.3_0_1_model_4.0_r3_af2: length = 135, lDDT = 51.22, rmsd = 5.148, \n",
      "7o2g_HBA_dif_173_T0.3_0_2_model_4.0_r3_af2: length = 135, lDDT = 46.92, rmsd = 8.188, \n",
      "7o2g_HBA_dif_173_T0.3_0_5_model_4.0_r3_af2: length = 135, lDDT = 61.57, rmsd = 6.379, \n",
      "7o2g_HBA_dif_1_T0.3_0_1_model_4.0_r3_af2: length = 135, lDDT = 71.84, rmsd = 4.436, \n",
      "7o2g_HBA_dif_1_T0.3_0_5_model_4.0_r3_af2: length = 135, lDDT = 70.70, rmsd = 16.824, \n",
      "7o2g_HBA_dif_1_traj3_T0.3_0_3_model_4.0_r3_af2: length = 135, lDDT = 59.51, rmsd = 7.966, \n",
      "7o2g_HBA_dif_1_traj2_T0.3_0_1_model_4.0_r3_af2: length = 135, lDDT = 74.97, rmsd = 15.149, \n",
      "7o2g_HBA_dif_1_traj2_T0.3_0_4_model_4.0_r3_af2: length = 135, lDDT = 78.87, rmsd = 17.343, \n",
      "7o2g_HBA_dif_132_traj1_T0.3_0_2_model_4.0_r3_af2: length = 136, lDDT = 82.10, rmsd = 3.467, \n",
      "7o2g_HBA_dif_132_traj1_T0.3_0_4_model_4.0_r3_af2: length = 136, lDDT = 68.63, rmsd = 4.247, \n",
      "7o2g_HBA_dif_132_traj2_T0.3_0_2_model_4.0_r3_af2: length = 136, lDDT = 79.61, rmsd = 2.441, \n",
      "7o2g_HBA_dif_132_traj2_T0.3_0_4_model_4.0_r3_af2: length = 136, lDDT = 74.45, rmsd = 1.929, \n",
      "7o2g_HBA_dif_132_traj3_T0.3_0_2_model_4.0_r3_af2: length = 136, lDDT = 64.59, rmsd = 9.362, \n",
      "7o2g_HBA_dif_132_traj3_T0.3_0_5_model_4.0_r3_af2: length = 136, lDDT = 81.59, rmsd = 2.561, \n",
      "7o2g_HBA_dif_132_traj4_T0.3_0_2_model_4.0_r3_af2: length = 136, lDDT = 91.18, rmsd = 0.985, \n",
      "7o2g_HBA_dif_132_traj4_T0.3_0_5_model_4.0_r3_af2: length = 136, lDDT = 85.23, rmsd = 2.358, \n",
      "7o2g_HBA_dif_57_traj3_T0.3_0_4_model_4.0_r3_af2: length = 137, lDDT = 79.23, rmsd = 2.544, \n",
      "7o2g_HBA_dif_57_traj2_T0.3_0_3_model_4.0_r3_af2: length = 137, lDDT = 81.96, rmsd = 2.792, \n",
      "7o2g_HBA_dif_57_traj2_T0.3_0_5_model_4.0_r3_af2: length = 137, lDDT = 81.27, rmsd = 2.293, \n",
      "7o2g_HBA_dif_57_traj1_T0.3_0_4_model_4.0_r3_af2: length = 137, lDDT = 76.19, rmsd = 2.521, \n",
      "7o2g_HBA_dif_57_traj4_T0.3_0_3_model_4.0_r3_af2: length = 137, lDDT = 73.84, rmsd = 4.366, \n",
      "7o2g_HBA_dif_222_traj2_T0.3_0_2_model_4.0_r3_af2: length = 138, lDDT = 39.35, rmsd = 17.095, \n",
      "7o2g_HBA_dif_222_traj2_T0.3_0_5_model_4.0_r3_af2: length = 138, lDDT = 44.01, rmsd = 13.841, \n",
      "7o2g_HBA_dif_222_traj1_T0.3_0_3_model_4.0_r3_af2: length = 138, lDDT = 50.36, rmsd = 16.418, \n",
      "7o2g_HBA_dif_222_traj4_T0.3_0_2_model_4.0_r3_af2: length = 138, lDDT = 46.62, rmsd = 15.842, \n",
      "7o2g_HBA_dif_91_traj2_T0.3_0_1_model_4.0_r3_af2: length = 138, lDDT = 83.61, rmsd = 2.669, \n",
      "7o2g_HBA_dif_91_traj2_T0.3_0_5_model_4.0_r3_af2: length = 138, lDDT = 68.90, rmsd = 11.906, \n",
      "7o2g_HBA_dif_222_traj3_T0.3_0_2_model_4.0_r3_af2: length = 138, lDDT = 57.23, rmsd = 17.276, \n",
      "7o2g_HBA_dif_222_traj3_T0.3_0_4_model_4.0_r3_af2: length = 138, lDDT = 52.93, rmsd = 11.047, \n",
      "7o2g_HBA_dif_91_traj3_T0.3_0_1_model_4.0_r3_af2: length = 138, lDDT = 64.87, rmsd = 5.499, \n",
      "7o2g_HBA_dif_91_traj3_T0.3_0_4_model_4.0_r3_af2: length = 138, lDDT = 53.42, rmsd = 12.231, \n",
      "7o2g_HBA_dif_91_traj4_T0.3_0_3_model_4.0_r3_af2: length = 138, lDDT = 46.10, rmsd = 9.115, \n",
      "7o2g_HBA_dif_100_traj4_T0.3_0_1_model_4.0_r3_af2: length = 139, lDDT = 86.06, rmsd = 1.149, \n",
      "7o2g_HBA_dif_100_traj4_T0.3_0_5_model_4.0_r3_af2: length = 139, lDDT = 72.28, rmsd = 3.788, \n",
      "7o2g_HBA_dif_100_T0.3_0_3_model_4.0_r3_af2: length = 139, lDDT = 72.54, rmsd = 2.920, \n",
      "7o2g_HBA_dif_100_T0.3_0_5_model_4.0_r3_af2: length = 139, lDDT = 73.84, rmsd = 3.816, \n",
      "7o2g_HBA_dif_100_traj3_T0.3_0_3_model_4.0_r3_af2: length = 139, lDDT = 86.71, rmsd = 1.046, \n",
      "7o2g_HBA_dif_100_traj1_T0.3_0_2_model_4.0_r3_af2: length = 139, lDDT = 72.78, rmsd = 3.881, \n",
      "7o2g_HBA_dif_100_traj2_T0.3_0_1_model_4.0_r3_af2: length = 139, lDDT = 69.09, rmsd = 4.997, \n",
      "7o2g_HBA_dif_100_traj2_T0.3_0_4_model_4.0_r3_af2: length = 139, lDDT = 85.45, rmsd = 2.525, \n",
      "7o2g_HBA_dif_227_traj1_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 69.94, rmsd = 14.282, \n",
      "7o2g_HBA_dif_147_traj1_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 59.77, rmsd = 3.223, \n",
      "7o2g_HBA_dif_147_traj1_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 84.61, rmsd = 2.549, \n",
      "7o2g_HBA_dif_24_traj1_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 80.46, rmsd = 4.285, \n",
      "7o2g_HBA_dif_24_traj1_T0.3_0_3_model_4.0_r3_af2: length = 133, lDDT = 84.02, rmsd = 1.424, \n",
      "7o2g_HBA_dif_216_traj1_T0.3_0_1_model_4.0_r3_af2: length = 133, lDDT = 51.84, rmsd = 12.815, \n",
      "7o2g_HBA_dif_216_traj1_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 58.99, rmsd = 12.534, \n",
      "7o2g_HBA_dif_24_T0.3_0_2_model_4.0_r3_af2: length = 133, lDDT = 60.24, rmsd = 10.145, \n",
      "7o2g_HBA_dif_45_traj1_T0.3_0_1_model_4.0_r3_af2: length = 134, lDDT = 78.42, rmsd = 6.220, \n",
      "7o2g_HBA_dif_45_traj1_T0.3_0_4_model_4.0_r3_af2: length = 134, lDDT = 55.31, rmsd = 15.459, \n",
      "7o2g_HBA_dif_45_traj2_T0.3_0_2_model_4.0_r3_af2: length = 134, lDDT = 53.19, rmsd = 19.060, \n",
      "7o2g_HBA_dif_45_traj3_T0.3_0_1_model_4.0_r3_af2: length = 134, lDDT = 75.19, rmsd = 3.043, \n",
      "7o2g_HBA_dif_45_traj3_T0.3_0_4_model_4.0_r3_af2: length = 134, lDDT = 47.96, rmsd = 16.469, \n",
      "7o2g_HBA_dif_45_traj4_T0.3_0_3_model_4.0_r3_af2: length = 134, lDDT = 73.52, rmsd = 1.980, \n",
      "7o2g_HBA_dif_45_T0.3_0_2_model_4.0_r3_af2: length = 134, lDDT = 79.82, rmsd = 8.830, \n",
      "7o2g_HBA_dif_1_traj4_T0.3_0_1_model_4.0_r3_af2: length = 135, lDDT = 70.70, rmsd = 5.243, \n",
      "7o2g_HBA_dif_1_traj4_T0.3_0_4_model_4.0_r3_af2: length = 135, lDDT = 64.95, rmsd = 4.921, \n",
      "7o2g_HBA_dif_1_traj1_T0.3_0_1_model_4.0_r3_af2: length = 135, lDDT = 71.76, rmsd = 14.792, \n",
      "7o2g_HBA_dif_1_traj1_T0.3_0_5_model_4.0_r3_af2: length = 135, lDDT = 75.76, rmsd = 2.650, \n",
      "7o2g_HBA_dif_173_T0.3_0_4_model_4.0_r3_af2: length = 135, lDDT = 77.16, rmsd = 10.174, \n",
      "7o2g_HBA_dif_1_T0.3_0_3_model_4.0_r3_af2: length = 135, lDDT = 77.97, rmsd = 1.815, \n",
      "7o2g_HBA_dif_1_traj3_T0.3_0_1_model_4.0_r3_af2: length = 135, lDDT = 63.78, rmsd = 2.831, \n",
      "7o2g_HBA_dif_1_traj3_T0.3_0_5_model_4.0_r3_af2: length = 135, lDDT = 74.20, rmsd = 7.308, \n",
      "7o2g_HBA_dif_1_traj2_T0.3_0_3_model_4.0_r3_af2: length = 135, lDDT = 65.49, rmsd = 17.344, \n",
      "7o2g_HBA_dif_1_traj2_T0.3_0_5_model_4.0_r3_af2: length = 135, lDDT = 71.44, rmsd = 15.761, \n",
      "7o2g_HBA_dif_132_traj1_T0.3_0_3_model_4.0_r3_af2: length = 136, lDDT = 72.60, rmsd = 2.815, \n",
      "7o2g_HBA_dif_132_traj2_T0.3_0_1_model_4.0_r3_af2: length = 136, lDDT = 85.28, rmsd = 1.773, \n",
      "7o2g_HBA_dif_132_traj2_T0.3_0_5_model_4.0_r3_af2: length = 136, lDDT = 77.45, rmsd = 2.092, \n",
      "7o2g_HBA_dif_132_traj3_T0.3_0_3_model_4.0_r3_af2: length = 136, lDDT = 76.21, rmsd = 9.190, \n",
      "7o2g_HBA_dif_132_traj4_T0.3_0_1_model_4.0_r3_af2: length = 136, lDDT = 85.66, rmsd = 3.718, \n",
      "7o2g_HBA_dif_132_traj4_T0.3_0_4_model_4.0_r3_af2: length = 136, lDDT = 72.61, rmsd = 2.404, \n",
      "7o2g_HBA_dif_57_traj3_T0.3_0_1_model_4.0_r3_af2: length = 137, lDDT = 63.41, rmsd = 5.065, \n",
      "7o2g_HBA_dif_57_traj3_T0.3_0_3_model_4.0_r3_af2: length = 137, lDDT = 68.77, rmsd = 3.281, \n",
      "7o2g_HBA_dif_57_traj2_T0.3_0_1_model_4.0_r3_af2: length = 137, lDDT = 80.68, rmsd = 2.809, \n",
      "7o2g_HBA_dif_57_traj2_T0.3_0_2_model_4.0_r3_af2: length = 137, lDDT = 73.79, rmsd = 3.066, \n",
      "7o2g_HBA_dif_57_traj1_T0.3_0_1_model_4.0_r3_af2: length = 137, lDDT = 55.07, rmsd = 9.320, \n",
      "7o2g_HBA_dif_57_traj1_T0.3_0_5_model_4.0_r3_af2: length = 137, lDDT = 78.76, rmsd = 1.847, \n",
      "7o2g_HBA_dif_57_traj4_T0.3_0_1_model_4.0_r3_af2: length = 137, lDDT = 49.40, rmsd = 10.653, \n",
      "7o2g_HBA_dif_57_traj4_T0.3_0_4_model_4.0_r3_af2: length = 137, lDDT = 69.91, rmsd = 3.065, \n",
      "7o2g_HBA_dif_57_traj4_T0.3_0_5_model_4.0_r3_af2: length = 137, lDDT = 72.50, rmsd = 2.594, \n",
      "7o2g_HBA_dif_222_traj2_T0.3_0_4_model_4.0_r3_af2: length = 138, lDDT = 44.60, rmsd = 15.219, \n",
      "7o2g_HBA_dif_222_traj1_T0.3_0_2_model_4.0_r3_af2: length = 138, lDDT = 50.70, rmsd = 8.265, \n",
      "7o2g_HBA_dif_222_traj1_T0.3_0_4_model_4.0_r3_af2: length = 138, lDDT = 49.45, rmsd = 15.830, \n",
      "7o2g_HBA_dif_222_traj4_T0.3_0_3_model_4.0_r3_af2: length = 138, lDDT = 45.36, rmsd = 16.664, \n",
      "7o2g_HBA_dif_222_traj4_T0.3_0_5_model_4.0_r3_af2: length = 138, lDDT = 51.64, rmsd = 11.378, \n",
      "7o2g_HBA_dif_91_traj2_T0.3_0_3_model_4.0_r3_af2: length = 138, lDDT = 65.85, rmsd = 10.050, \n",
      "7o2g_HBA_dif_222_traj3_T0.3_0_1_model_4.0_r3_af2: length = 138, lDDT = 58.38, rmsd = 12.155, \n",
      "7o2g_HBA_dif_222_traj3_T0.3_0_5_model_4.0_r3_af2: length = 138, lDDT = 59.25, rmsd = 13.298, \n",
      "7o2g_HBA_dif_91_traj3_T0.3_0_3_model_4.0_r3_af2: length = 138, lDDT = 45.65, rmsd = 12.068, \n",
      "7o2g_HBA_dif_91_traj3_T0.3_0_5_model_4.0_r3_af2: length = 138, lDDT = 87.61, rmsd = 2.416, \n",
      "7o2g_HBA_dif_91_traj4_T0.3_0_2_model_4.0_r3_af2: length = 138, lDDT = 46.18, rmsd = 10.401, \n",
      "7o2g_HBA_dif_91_traj4_T0.3_0_5_model_4.0_r3_af2: length = 138, lDDT = 51.52, rmsd = 14.706, \n",
      "7o2g_HBA_dif_100_traj4_T0.3_0_3_model_4.0_r3_af2: length = 139, lDDT = 37.17, rmsd = 13.974, \n",
      "7o2g_HBA_dif_100_T0.3_0_2_model_4.0_r3_af2: length = 139, lDDT = 40.41, rmsd = 15.372, \n",
      "7o2g_HBA_dif_100_traj3_T0.3_0_1_model_4.0_r3_af2: length = 139, lDDT = 62.53, rmsd = 7.414, \n",
      "7o2g_HBA_dif_100_traj3_T0.3_0_5_model_4.0_r3_af2: length = 139, lDDT = 73.41, rmsd = 2.673, \n",
      "7o2g_HBA_dif_100_traj1_T0.3_0_3_model_4.0_r3_af2: length = 139, lDDT = 72.59, rmsd = 2.965, \n",
      "7o2g_HBA_dif_100_traj1_T0.3_0_5_model_4.0_r3_af2: length = 139, lDDT = 75.79, rmsd = 1.906, \n",
      "7o2g_HBA_dif_100_traj2_T0.3_0_3_model_4.0_r3_af2: length = 139, lDDT = 81.10, rmsd = 1.956, \n",
      "7o2g_HBA_dif_24_T0.3_0_4_model_4.0_r3_af2: length = 133, lDDT = 83.88, rmsd = 3.976, \n",
      "7o2g_HBA_dif_45_traj1_T0.3_0_2_model_4.0_r3_af2: length = 134, lDDT = 55.07, rmsd = 7.218, \n",
      "7o2g_HBA_dif_45_traj1_T0.3_0_5_model_4.0_r3_af2: length = 134, lDDT = 62.81, rmsd = 11.411, \n",
      "7o2g_HBA_dif_45_traj2_T0.3_0_4_model_4.0_r3_af2: length = 134, lDDT = 41.86, rmsd = 18.362, \n",
      "7o2g_HBA_dif_45_traj3_T0.3_0_3_model_4.0_r3_af2: length = 134, lDDT = 70.42, rmsd = 2.435, \n",
      "7o2g_HBA_dif_45_traj3_T0.3_0_5_model_4.0_r3_af2: length = 134, lDDT = 52.29, rmsd = 17.305, \n",
      "7o2g_HBA_dif_45_traj4_T0.3_0_2_model_4.0_r3_af2: length = 134, lDDT = 67.32, rmsd = 14.640, \n",
      "7o2g_HBA_dif_45_traj4_T0.3_0_4_model_4.0_r3_af2: length = 134, lDDT = 60.12, rmsd = 9.411, \n",
      "7o2g_HBA_dif_45_T0.3_0_1_model_4.0_r3_af2: length = 134, lDDT = 58.66, rmsd = 14.709, \n",
      "7o2g_HBA_dif_45_T0.3_0_4_model_4.0_r3_af2: length = 134, lDDT = 69.44, rmsd = 15.253, \n",
      "7o2g_HBA_dif_1_traj4_T0.3_0_2_model_4.0_r3_af2: length = 135, lDDT = 64.69, rmsd = 5.186, \n",
      "7o2g_HBA_dif_1_traj4_T0.3_0_5_model_4.0_r3_af2: length = 135, lDDT = 50.19, rmsd = 8.642, \n",
      "7o2g_HBA_dif_1_traj1_T0.3_0_3_model_4.0_r3_af2: length = 135, lDDT = 75.38, rmsd = 7.929, \n",
      "7o2g_HBA_dif_1_traj1_T0.3_0_4_model_4.0_r3_af2: length = 135, lDDT = 71.33, rmsd = 3.517, \n",
      "7o2g_HBA_dif_173_T0.3_0_3_model_4.0_r3_af2: length = 135, lDDT = 54.18, rmsd = 7.936, \n",
      "7o2g_HBA_dif_1_T0.3_0_2_model_4.0_r3_af2: length = 135, lDDT = 69.36, rmsd = 15.902, \n",
      "7o2g_HBA_dif_1_T0.3_0_4_model_4.0_r3_af2: length = 135, lDDT = 64.75, rmsd = 19.236, \n",
      "7o2g_HBA_dif_1_traj3_T0.3_0_2_model_4.0_r3_af2: length = 135, lDDT = 67.74, rmsd = 2.345, \n",
      "7o2g_HBA_dif_1_traj3_T0.3_0_4_model_4.0_r3_af2: length = 135, lDDT = 68.36, rmsd = 3.987, \n",
      "7o2g_HBA_dif_1_traj2_T0.3_0_2_model_4.0_r3_af2: length = 135, lDDT = 60.53, rmsd = 16.937, \n",
      "7o2g_HBA_dif_132_traj1_T0.3_0_1_model_4.0_r3_af2: length = 136, lDDT = 62.66, rmsd = 15.618, \n",
      "7o2g_HBA_dif_132_traj1_T0.3_0_5_model_4.0_r3_af2: length = 136, lDDT = 69.10, rmsd = 11.949, \n",
      "7o2g_HBA_dif_132_traj2_T0.3_0_3_model_4.0_r3_af2: length = 136, lDDT = 81.04, rmsd = 3.652, \n",
      "7o2g_HBA_dif_132_traj3_T0.3_0_1_model_4.0_r3_af2: length = 136, lDDT = 56.07, rmsd = 10.573, \n",
      "7o2g_HBA_dif_132_traj3_T0.3_0_4_model_4.0_r3_af2: length = 136, lDDT = 79.14, rmsd = 2.601, \n",
      "7o2g_HBA_dif_132_traj4_T0.3_0_3_model_4.0_r3_af2: length = 136, lDDT = 76.83, rmsd = 5.549, \n",
      "7o2g_HBA_dif_57_traj3_T0.3_0_2_model_4.0_r3_af2: length = 137, lDDT = 69.70, rmsd = 2.937, \n",
      "7o2g_HBA_dif_57_traj3_T0.3_0_5_model_4.0_r3_af2: length = 137, lDDT = 70.77, rmsd = 6.645, \n",
      "7o2g_HBA_dif_57_traj2_T0.3_0_4_model_4.0_r3_af2: length = 137, lDDT = 61.07, rmsd = 6.625, \n",
      "7o2g_HBA_dif_57_traj1_T0.3_0_2_model_4.0_r3_af2: length = 137, lDDT = 57.51, rmsd = 8.070, \n",
      "7o2g_HBA_dif_57_traj1_T0.3_0_3_model_4.0_r3_af2: length = 137, lDDT = 77.07, rmsd = 4.172, \n",
      "7o2g_HBA_dif_57_traj4_T0.3_0_2_model_4.0_r3_af2: length = 137, lDDT = 66.75, rmsd = 2.794, \n",
      "7o2g_HBA_dif_222_traj2_T0.3_0_1_model_4.0_r3_af2: length = 138, lDDT = 47.48, rmsd = 13.450, \n",
      "7o2g_HBA_dif_222_traj2_T0.3_0_3_model_4.0_r3_af2: length = 138, lDDT = 47.23, rmsd = 16.041, \n",
      "7o2g_HBA_dif_222_traj1_T0.3_0_1_model_4.0_r3_af2: length = 138, lDDT = 42.15, rmsd = 10.061, \n",
      "7o2g_HBA_dif_222_traj1_T0.3_0_5_model_4.0_r3_af2: length = 138, lDDT = 49.92, rmsd = 8.768, \n",
      "7o2g_HBA_dif_222_traj4_T0.3_0_1_model_4.0_r3_af2: length = 138, lDDT = 45.72, rmsd = 17.945, \n",
      "7o2g_HBA_dif_222_traj4_T0.3_0_4_model_4.0_r3_af2: length = 138, lDDT = 47.89, rmsd = 15.943, \n",
      "7o2g_HBA_dif_91_traj2_T0.3_0_2_model_4.0_r3_af2: length = 138, lDDT = 50.17, rmsd = 6.159, \n",
      "7o2g_HBA_dif_91_traj2_T0.3_0_4_model_4.0_r3_af2: length = 138, lDDT = 54.99, rmsd = 10.425, \n",
      "7o2g_HBA_dif_222_traj3_T0.3_0_3_model_4.0_r3_af2: length = 138, lDDT = 51.20, rmsd = 9.642, \n",
      "7o2g_HBA_dif_91_traj3_T0.3_0_2_model_4.0_r3_af2: length = 138, lDDT = 50.50, rmsd = 15.482, \n",
      "7o2g_HBA_dif_91_traj4_T0.3_0_1_model_4.0_r3_af2: length = 138, lDDT = 48.16, rmsd = 17.800, \n",
      "7o2g_HBA_dif_91_traj4_T0.3_0_4_model_4.0_r3_af2: length = 138, lDDT = 66.78, rmsd = 12.207, \n",
      "7o2g_HBA_dif_100_traj4_T0.3_0_2_model_4.0_r3_af2: length = 139, lDDT = 78.50, rmsd = 1.182, \n",
      "7o2g_HBA_dif_100_traj4_T0.3_0_4_model_4.0_r3_af2: length = 139, lDDT = 80.88, rmsd = 3.336, \n",
      "7o2g_HBA_dif_100_T0.3_0_1_model_4.0_r3_af2: length = 139, lDDT = 77.48, rmsd = 2.756, \n",
      "7o2g_HBA_dif_100_T0.3_0_4_model_4.0_r3_af2: length = 139, lDDT = 44.17, rmsd = 13.422, \n",
      "7o2g_HBA_dif_100_traj3_T0.3_0_2_model_4.0_r3_af2: length = 139, lDDT = 87.81, rmsd = 1.067, \n",
      "7o2g_HBA_dif_100_traj3_T0.3_0_4_model_4.0_r3_af2: length = 139, lDDT = 77.55, rmsd = 2.739, \n",
      "7o2g_HBA_dif_100_traj1_T0.3_0_1_model_4.0_r3_af2: length = 139, lDDT = 41.38, rmsd = 10.699, \n",
      "7o2g_HBA_dif_100_traj1_T0.3_0_4_model_4.0_r3_af2: length = 139, lDDT = 67.89, rmsd = 4.946, \n",
      "7o2g_HBA_dif_100_traj2_T0.3_0_2_model_4.0_r3_af2: length = 139, lDDT = 59.82, rmsd = 3.272, \n",
      "7o2g_HBA_dif_100_traj2_T0.3_0_5_model_4.0_r3_af2: length = 139, lDDT = 67.17, rmsd = 11.823, \n",
      " ID    lDDT     rmsd rmsd_SR1 Name\n",
      "  0  60.918    4.991    5.131 7o2g_HBA_dif_131_traj4_T0.3_0_1\n",
      "  1  64.781    3.127    4.130 7o2g_HBA_dif_131_traj4_T0.3_0_2\n",
      "  2  64.627    8.538    6.441 7o2g_HBA_dif_131_traj4_T0.3_0_3\n",
      "  3  54.787    7.060    3.271 7o2g_HBA_dif_131_traj4_T0.3_0_4\n",
      "  4  66.306    3.206    4.251 7o2g_HBA_dif_131_traj4_T0.3_0_5\n",
      "  5  54.497   11.145   14.472 7o2g_HBA_dif_131_T0.3_0_1\n",
      "  6  78.568    8.487    4.441 7o2g_HBA_dif_131_T0.3_0_2\n",
      "  7  49.827   12.473    6.549 7o2g_HBA_dif_131_T0.3_0_3\n",
      "  8  80.182    2.281    6.453 7o2g_HBA_dif_131_T0.3_0_4\n",
      "  9  48.931   11.140    7.324 7o2g_HBA_dif_131_T0.3_0_5\n",
      " 10  71.591    2.982    2.279 7o2g_HBA_dif_131_traj1_T0.3_0_1\n",
      " 11  58.812    7.942    5.208 7o2g_HBA_dif_131_traj1_T0.3_0_2\n",
      " 12  75.826    9.406    6.271 7o2g_HBA_dif_131_traj1_T0.3_0_3\n",
      " 13  68.876    5.294    6.086 7o2g_HBA_dif_131_traj1_T0.3_0_4\n",
      " 14  57.037   10.379   11.341 7o2g_HBA_dif_131_traj1_T0.3_0_5\n",
      " 15  66.830    5.054    2.903 7o2g_HBA_dif_131_traj3_T0.3_0_1\n",
      " 16  58.159    9.079    3.609 7o2g_HBA_dif_131_traj3_T0.3_0_2\n",
      " 17  80.055   11.969   12.364 7o2g_HBA_dif_131_traj3_T0.3_0_3\n",
      " 18  72.597    7.931    8.844 7o2g_HBA_dif_131_traj3_T0.3_0_4\n",
      " 19  70.763    3.075    2.535 7o2g_HBA_dif_131_traj3_T0.3_0_5\n",
      " 20  60.722    8.995    2.809 7o2g_HBA_dif_131_traj2_T0.3_0_1\n",
      " 21  52.120   12.889   13.604 7o2g_HBA_dif_131_traj2_T0.3_0_2\n",
      " 22  75.001    2.462    2.855 7o2g_HBA_dif_131_traj2_T0.3_0_3\n",
      " 23  49.322   14.284    3.971 7o2g_HBA_dif_131_traj2_T0.3_0_4\n",
      " 24  59.653    2.944    4.337 7o2g_HBA_dif_131_traj2_T0.3_0_5\n",
      "  0  62.586    8.217    4.558 7o2g_HBA_dif_52_traj3_T0.3_0_1\n",
      "  1  68.130    5.290    5.536 7o2g_HBA_dif_52_traj3_T0.3_0_2\n",
      "  2  67.451    8.456    5.573 7o2g_HBA_dif_52_traj3_T0.3_0_3\n",
      "  3  51.707    9.914    5.371 7o2g_HBA_dif_52_traj3_T0.3_0_4\n",
      "  4  78.548    5.766    5.658 7o2g_HBA_dif_52_traj3_T0.3_0_5\n",
      "  5  69.951    8.207    4.582 7o2g_HBA_dif_52_traj4_T0.3_0_1\n",
      "  6  59.291    8.251    4.120 7o2g_HBA_dif_52_traj4_T0.3_0_2\n",
      "  7  61.606    4.070    4.877 7o2g_HBA_dif_52_traj4_T0.3_0_3\n",
      "  8  75.843    3.138    4.709 7o2g_HBA_dif_52_traj4_T0.3_0_4\n",
      "  9  59.831    5.307    4.105 7o2g_HBA_dif_52_traj4_T0.3_0_5\n",
      " 10  68.612   10.166    5.695 7o2g_HBA_dif_52_traj2_T0.3_0_1\n",
      " 11  75.555   11.988    8.033 7o2g_HBA_dif_52_traj2_T0.3_0_2\n",
      " 12  58.073   10.976    5.849 7o2g_HBA_dif_52_traj2_T0.3_0_3\n",
      " 13  69.029   11.067    7.409 7o2g_HBA_dif_52_traj2_T0.3_0_4\n",
      " 14  56.661   12.229    5.963 7o2g_HBA_dif_52_traj2_T0.3_0_5\n",
      " 15  59.763   11.062    4.577 7o2g_HBA_dif_52_traj1_T0.3_0_1\n",
      " 16  60.836   13.356    6.455 7o2g_HBA_dif_52_traj1_T0.3_0_2\n",
      " 17  63.703   10.119    6.304 7o2g_HBA_dif_52_traj1_T0.3_0_3\n",
      " 18  50.119   14.380    8.019 7o2g_HBA_dif_52_traj1_T0.3_0_4\n",
      " 19  46.726    5.238    4.758 7o2g_HBA_dif_52_traj1_T0.3_0_5\n",
      "  0  58.220    9.717    7.458 7o2g_HBA_dif_220_traj3_T0.3_0_1\n",
      "  1  56.895    9.506    6.453 7o2g_HBA_dif_220_traj3_T0.3_0_2\n",
      "  2  56.066   10.491    6.768 7o2g_HBA_dif_220_traj3_T0.3_0_3\n",
      "  3  62.622    8.963    9.402 7o2g_HBA_dif_220_traj3_T0.3_0_4\n",
      "  4  56.833   14.619    5.227 7o2g_HBA_dif_220_traj3_T0.3_0_5\n",
      "  5  46.587    2.624    1.908 7o2g_HBA_dif_220_traj2_T0.3_0_1\n",
      "  6  73.913   11.442    5.273 7o2g_HBA_dif_220_traj2_T0.3_0_2\n",
      "  7  66.555    2.552    2.732 7o2g_HBA_dif_220_traj2_T0.3_0_3\n",
      "  8  68.437    9.735    3.017 7o2g_HBA_dif_220_traj2_T0.3_0_4\n",
      "  9  64.729    3.736    4.719 7o2g_HBA_dif_220_traj2_T0.3_0_5\n",
      " 10  65.092   11.630   14.616 7o2g_HBA_dif_220_T0.3_0_1\n",
      " 11  61.805   10.029    8.820 7o2g_HBA_dif_220_T0.3_0_2\n",
      " 12  53.572   11.836    6.770 7o2g_HBA_dif_220_T0.3_0_3\n",
      " 13  54.659   13.128    4.001 7o2g_HBA_dif_220_T0.3_0_4\n",
      " 14  48.157   12.720    9.020 7o2g_HBA_dif_220_T0.3_0_5\n",
      " 15  47.684    6.183    5.055 7o2g_HBA_dif_220_traj1_T0.3_0_1\n",
      " 16  62.112   12.460   12.874 7o2g_HBA_dif_220_traj1_T0.3_0_2\n",
      " 17  48.570   10.934    7.331 7o2g_HBA_dif_220_traj1_T0.3_0_3\n",
      " 18  56.073   11.050    2.220 7o2g_HBA_dif_220_traj1_T0.3_0_4\n",
      " 19  77.682    9.552    7.904 7o2g_HBA_dif_220_traj1_T0.3_0_5\n",
      " 20  76.441   13.005    9.621 7o2g_HBA_dif_220_traj4_T0.3_0_1\n",
      " 21  49.797   13.645    7.493 7o2g_HBA_dif_220_traj4_T0.3_0_2\n",
      " 22  63.810   14.897   10.264 7o2g_HBA_dif_220_traj4_T0.3_0_3\n",
      " 23  52.516    9.521    6.938 7o2g_HBA_dif_220_traj4_T0.3_0_4\n",
      " 24  70.170   10.760    9.527 7o2g_HBA_dif_220_traj4_T0.3_0_5\n",
      "  0  82.540    2.507    2.239 7o2g_HBA_dif_78_traj2_T0.3_0_1\n",
      "  1  79.985    1.863    2.214 7o2g_HBA_dif_78_traj2_T0.3_0_2\n",
      "  2  61.487   11.619    5.195 7o2g_HBA_dif_78_traj2_T0.3_0_3\n",
      "  3  44.014   12.192   18.256 7o2g_HBA_dif_78_traj2_T0.3_0_4\n",
      "  4  74.936    3.180    5.288 7o2g_HBA_dif_78_traj2_T0.3_0_5\n",
      "  5  72.944    3.253    3.030 7o2g_HBA_dif_78_traj1_T0.3_0_1\n",
      "  6  54.411   12.648    4.761 7o2g_HBA_dif_78_traj1_T0.3_0_2\n",
      "  7  48.526   12.231    5.183 7o2g_HBA_dif_78_traj1_T0.3_0_3\n",
      "  8  46.082    9.811    7.009 7o2g_HBA_dif_78_traj1_T0.3_0_4\n",
      "  9  42.861    9.453    3.418 7o2g_HBA_dif_78_traj1_T0.3_0_5\n",
      " 10  81.879    1.153    1.609 7o2g_HBA_dif_209_traj3_T0.3_0_1\n",
      " 11  81.773    2.256    3.336 7o2g_HBA_dif_209_traj3_T0.3_0_2\n",
      " 12  74.417    3.799    2.585 7o2g_HBA_dif_209_traj3_T0.3_0_3\n",
      " 13  77.604   13.053   15.898 7o2g_HBA_dif_209_traj3_T0.3_0_4\n",
      " 14  86.819    2.928    2.312 7o2g_HBA_dif_209_traj3_T0.3_0_5\n",
      " 15  44.353   11.901    4.060 7o2g_HBA_dif_78_traj3_T0.3_0_1\n",
      " 16  60.247    2.132    3.742 7o2g_HBA_dif_78_traj3_T0.3_0_2\n",
      " 17  77.313    3.004    2.843 7o2g_HBA_dif_78_traj3_T0.3_0_3\n",
      " 18  76.324    2.743    3.187 7o2g_HBA_dif_78_traj3_T0.3_0_4\n",
      " 19  66.833    6.351    4.396 7o2g_HBA_dif_78_traj3_T0.3_0_5\n",
      " 20  61.611   12.284    5.355 7o2g_HBA_dif_78_traj4_T0.3_0_1\n",
      " 21  62.046    7.178    5.690 7o2g_HBA_dif_78_traj4_T0.3_0_2\n",
      " 22  63.522   11.897    6.110 7o2g_HBA_dif_78_traj4_T0.3_0_3\n",
      " 23  75.480    3.660    3.672 7o2g_HBA_dif_78_traj4_T0.3_0_4\n",
      " 24  77.258    3.015    3.329 7o2g_HBA_dif_78_traj4_T0.3_0_5\n",
      " 25  66.403    2.042    2.585 7o2g_HBA_dif_78_T0.3_0_1\n",
      " 26  43.979   11.035    6.921 7o2g_HBA_dif_78_T0.3_0_2\n",
      " 27  48.338   14.034   11.507 7o2g_HBA_dif_78_T0.3_0_3\n",
      " 28  77.458    2.414    2.916 7o2g_HBA_dif_78_T0.3_0_4\n",
      " 29  83.039    2.478    2.983 7o2g_HBA_dif_78_T0.3_0_5\n",
      " 30  65.037    4.529    4.310 7o2g_HBA_dif_209_traj4_T0.3_0_1\n",
      " 31  73.479    7.438    7.860 7o2g_HBA_dif_209_traj4_T0.3_0_2\n",
      " 32  78.425   11.335   12.884 7o2g_HBA_dif_209_traj4_T0.3_0_3\n",
      " 33  80.413    2.623    1.920 7o2g_HBA_dif_209_traj4_T0.3_0_4\n",
      " 34  88.664    1.650    1.950 7o2g_HBA_dif_209_traj4_T0.3_0_5\n",
      "  0  87.807    1.614    2.731 7o2g_HBA_dif_25_traj4_T0.3_0_1\n",
      "  1  77.529    2.174    3.065 7o2g_HBA_dif_25_traj4_T0.3_0_2\n",
      "  2  84.865    1.056    2.260 7o2g_HBA_dif_25_traj4_T0.3_0_3\n",
      "  3  75.472    3.696    2.948 7o2g_HBA_dif_25_traj4_T0.3_0_4\n",
      "  4  72.191    9.663    5.386 7o2g_HBA_dif_25_traj4_T0.3_0_5\n",
      "  0  79.559    2.409    3.086 7o2g_HBA_dif_185_traj3_T0.3_0_1\n",
      "  1  50.375   11.475    2.493 7o2g_HBA_dif_185_traj3_T0.3_0_2\n",
      "  2  73.047    2.895    3.801 7o2g_HBA_dif_185_traj3_T0.3_0_3\n",
      "  3  82.462    2.737    4.647 7o2g_HBA_dif_185_traj3_T0.3_0_4\n",
      "  4  61.979    9.515    8.445 7o2g_HBA_dif_185_traj3_T0.3_0_5\n",
      "  5  61.075   12.379    5.020 7o2g_HBA_dif_185_traj2_T0.3_0_1\n",
      "  6  70.718    2.006    2.556 7o2g_HBA_dif_185_traj2_T0.3_0_2\n",
      "  7  67.063    3.378    2.242 7o2g_HBA_dif_185_traj2_T0.3_0_3\n",
      "  8  69.542    3.944    2.341 7o2g_HBA_dif_185_traj2_T0.3_0_4\n",
      "  9  65.413    8.534    5.562 7o2g_HBA_dif_185_traj2_T0.3_0_5\n",
      " 10  81.863    1.904    4.185 7o2g_HBA_dif_64_T0.3_0_1\n",
      " 11  50.188   13.022    5.336 7o2g_HBA_dif_64_T0.3_0_2\n",
      " 12  55.514    7.049    4.516 7o2g_HBA_dif_64_T0.3_0_3\n",
      " 13  62.193    8.754    5.321 7o2g_HBA_dif_64_T0.3_0_4\n",
      " 14  61.675   10.046    6.416 7o2g_HBA_dif_64_T0.3_0_5\n",
      " 15  58.470   13.236    6.186 7o2g_HBA_dif_226_traj1_T0.3_0_1\n",
      " 16  84.398    2.804    3.600 7o2g_HBA_dif_226_traj1_T0.3_0_2\n",
      " 17  52.196   10.346    6.310 7o2g_HBA_dif_226_traj1_T0.3_0_3\n",
      " 18  46.265   10.580    8.726 7o2g_HBA_dif_226_traj1_T0.3_0_4\n",
      " 19  66.309    7.528    4.744 7o2g_HBA_dif_226_traj1_T0.3_0_5\n",
      " 20  71.314    1.960    1.724 7o2g_HBA_dif_226_traj4_T0.3_0_1\n",
      " 21  79.477    2.005    3.232 7o2g_HBA_dif_226_traj4_T0.3_0_2\n",
      " 22  59.906    3.219    3.147 7o2g_HBA_dif_226_traj4_T0.3_0_3\n",
      " 23  62.194    3.714    5.026 7o2g_HBA_dif_226_traj4_T0.3_0_4\n",
      " 24  71.505   10.163    7.181 7o2g_HBA_dif_226_traj4_T0.3_0_5\n",
      " 25  62.392    5.281    6.052 7o2g_HBA_dif_191_traj1_T0.3_0_1\n",
      " 26  66.221   18.835   13.773 7o2g_HBA_dif_191_traj1_T0.3_0_2\n",
      " 27  45.505   15.365   11.593 7o2g_HBA_dif_191_traj1_T0.3_0_3\n",
      " 28  67.412    5.949    8.114 7o2g_HBA_dif_191_traj1_T0.3_0_4\n",
      " 29  66.017    9.548    7.443 7o2g_HBA_dif_191_traj1_T0.3_0_5\n",
      " 30  80.546    2.295    1.844 7o2g_HBA_dif_201_T0.3_0_1\n",
      " 31  71.847    6.306    8.526 7o2g_HBA_dif_201_T0.3_0_2\n",
      " 32  78.910    2.098    1.686 7o2g_HBA_dif_201_T0.3_0_3\n",
      " 33  79.581    2.082    1.754 7o2g_HBA_dif_201_T0.3_0_4\n",
      " 34  68.516    2.795    4.041 7o2g_HBA_dif_201_T0.3_0_5\n",
      " 35  60.115   12.718    9.246 7o2g_HBA_dif_191_traj4_T0.3_0_1\n",
      " 36  60.174   11.733   14.819 7o2g_HBA_dif_191_traj4_T0.3_0_2\n",
      " 37  60.769   13.639   13.561 7o2g_HBA_dif_191_traj4_T0.3_0_3\n",
      " 38  81.439    6.197    7.855 7o2g_HBA_dif_191_traj4_T0.3_0_4\n",
      " 39  78.112    1.886    2.295 7o2g_HBA_dif_191_traj4_T0.3_0_5\n",
      " 40  73.262    8.538    9.764 7o2g_HBA_dif_191_traj2_T0.3_0_1\n",
      " 41  68.495   14.312    7.914 7o2g_HBA_dif_191_traj2_T0.3_0_2\n",
      " 42  74.265    8.945   11.145 7o2g_HBA_dif_191_traj2_T0.3_0_3\n",
      " 43  53.533   11.147   11.186 7o2g_HBA_dif_191_traj2_T0.3_0_4\n",
      " 44  66.230   13.299    8.781 7o2g_HBA_dif_191_traj2_T0.3_0_5\n",
      " 45  56.471    8.594    3.777 7o2g_HBA_dif_64_traj2_T0.3_0_1\n",
      " 46  59.627   11.702   10.590 7o2g_HBA_dif_64_traj2_T0.3_0_2\n",
      " 47  62.074   12.431    7.909 7o2g_HBA_dif_64_traj2_T0.3_0_3\n",
      " 48  85.199    1.539    2.862 7o2g_HBA_dif_64_traj2_T0.3_0_4\n",
      " 49  77.361    1.589    2.544 7o2g_HBA_dif_64_traj2_T0.3_0_5\n",
      " 50  62.975    8.630    2.218 7o2g_HBA_dif_226_traj3_T0.3_0_1\n",
      " 51  61.050   12.443    9.434 7o2g_HBA_dif_226_traj3_T0.3_0_2\n",
      " 52  58.443    8.603    3.346 7o2g_HBA_dif_226_traj3_T0.3_0_3\n",
      " 53  50.774    8.841    7.192 7o2g_HBA_dif_226_traj3_T0.3_0_4\n",
      " 54  75.918    8.568    4.480 7o2g_HBA_dif_226_traj3_T0.3_0_5\n",
      " 55  55.094    8.789    8.575 7o2g_HBA_dif_191_T0.3_0_1\n",
      " 56  71.986    3.284    5.690 7o2g_HBA_dif_191_T0.3_0_2\n",
      " 57  66.980    7.291   13.527 7o2g_HBA_dif_191_T0.3_0_3\n",
      " 58  72.261    3.008    7.011 7o2g_HBA_dif_191_T0.3_0_4\n",
      " 59  60.702    7.641   10.906 7o2g_HBA_dif_191_T0.3_0_5\n",
      " 60  65.441    5.290    9.268 7o2g_HBA_dif_191_traj3_T0.3_0_1\n",
      " 61  68.559    7.288    5.617 7o2g_HBA_dif_191_traj3_T0.3_0_2\n",
      " 62  65.532    8.355    6.979 7o2g_HBA_dif_191_traj3_T0.3_0_3\n",
      " 63  60.052    5.781    6.631 7o2g_HBA_dif_191_traj3_T0.3_0_4\n",
      " 64  78.246    6.941   10.251 7o2g_HBA_dif_191_traj3_T0.3_0_5\n",
      " 65  71.618    2.060    1.793 7o2g_HBA_dif_185_traj1_T0.3_0_1\n",
      " 66  76.585    1.134    1.243 7o2g_HBA_dif_185_traj1_T0.3_0_2\n",
      " 67  72.618    2.604    4.043 7o2g_HBA_dif_185_traj1_T0.3_0_3\n",
      " 68  71.817    4.947    2.762 7o2g_HBA_dif_185_traj1_T0.3_0_4\n",
      " 69  76.059    3.607    2.907 7o2g_HBA_dif_185_traj1_T0.3_0_5\n",
      " 70  58.510    7.899    4.681 7o2g_HBA_dif_64_traj4_T0.3_0_1\n",
      " 71  88.846    1.421    3.194 7o2g_HBA_dif_64_traj4_T0.3_0_2\n",
      " 72  54.911   12.081    6.396 7o2g_HBA_dif_64_traj4_T0.3_0_3\n",
      " 73  51.774   11.388    4.595 7o2g_HBA_dif_64_traj4_T0.3_0_4\n",
      " 74  57.560    8.828    8.727 7o2g_HBA_dif_64_traj4_T0.3_0_5\n",
      " 75  69.625    5.700    3.376 7o2g_HBA_dif_64_traj1_T0.3_0_1\n",
      " 76  47.352   13.876    9.434 7o2g_HBA_dif_64_traj1_T0.3_0_2\n",
      " 77  85.423    1.864    2.829 7o2g_HBA_dif_64_traj1_T0.3_0_3\n",
      " 78  57.858    6.538    4.527 7o2g_HBA_dif_64_traj1_T0.3_0_4\n",
      " 79  74.503   11.370    4.699 7o2g_HBA_dif_64_traj1_T0.3_0_5\n",
      " 80  57.688    7.438    5.077 7o2g_HBA_dif_64_traj3_T0.3_0_1\n",
      " 81  83.528    1.900    4.834 7o2g_HBA_dif_64_traj3_T0.3_0_2\n",
      " 82  80.250    2.447    3.907 7o2g_HBA_dif_64_traj3_T0.3_0_3\n",
      " 83  54.359   12.807    9.709 7o2g_HBA_dif_64_traj3_T0.3_0_4\n",
      " 84  63.313   12.526    6.343 7o2g_HBA_dif_64_traj3_T0.3_0_5\n",
      "  0  79.098    3.417    3.195 7o2g_HBA_dif_28_T0.3_0_1\n",
      "  1  67.280    4.247    2.102 7o2g_HBA_dif_28_T0.3_0_2\n",
      "  2  65.927    4.257    1.732 7o2g_HBA_dif_28_T0.3_0_3\n",
      "  3  73.029    5.043    3.998 7o2g_HBA_dif_28_T0.3_0_4\n",
      "  4  87.859    1.264    1.342 7o2g_HBA_dif_28_T0.3_0_5\n",
      "  5  47.698   12.134    9.041 7o2g_HBA_dif_28_traj3_T0.3_0_1\n",
      "  6  51.919   11.917    8.279 7o2g_HBA_dif_28_traj3_T0.3_0_2\n",
      "  7  78.216    1.965    1.264 7o2g_HBA_dif_28_traj3_T0.3_0_3\n",
      "  8  56.310    6.696    2.012 7o2g_HBA_dif_28_traj3_T0.3_0_4\n",
      "  9  79.281    2.953    3.634 7o2g_HBA_dif_28_traj3_T0.3_0_5\n",
      " 10  75.741    3.099    2.892 7o2g_HBA_dif_28_traj2_T0.3_0_1\n",
      " 11  87.868    0.772    1.387 7o2g_HBA_dif_28_traj2_T0.3_0_2\n",
      " 12  83.617    1.889    2.762 7o2g_HBA_dif_28_traj2_T0.3_0_3\n",
      " 13  58.745    7.885    3.056 7o2g_HBA_dif_28_traj2_T0.3_0_4\n",
      " 14  75.432    8.681    4.460 7o2g_HBA_dif_28_traj2_T0.3_0_5\n",
      " 15  85.324    1.077    1.811 7o2g_HBA_dif_28_traj4_T0.3_0_1\n",
      " 16  49.256    7.268    4.844 7o2g_HBA_dif_28_traj4_T0.3_0_2\n",
      " 17  60.036   14.229    4.562 7o2g_HBA_dif_28_traj4_T0.3_0_3\n",
      " 18  51.178   11.843    3.511 7o2g_HBA_dif_28_traj4_T0.3_0_4\n",
      " 19  50.114   14.684   11.464 7o2g_HBA_dif_28_traj4_T0.3_0_5\n",
      " 20  79.844    1.240    2.541 7o2g_HBA_dif_28_traj1_T0.3_0_1\n",
      " 21  71.036    9.461    7.955 7o2g_HBA_dif_28_traj1_T0.3_0_2\n",
      " 22  71.068    5.875    4.185 7o2g_HBA_dif_28_traj1_T0.3_0_3\n",
      " 23  57.246    9.012    3.580 7o2g_HBA_dif_28_traj1_T0.3_0_4\n",
      " 24  49.956   14.620    9.839 7o2g_HBA_dif_28_traj1_T0.3_0_5\n",
      "  0  66.068    4.685    3.923 7o2g_HBA_dif_108_traj4_T0.3_0_1\n",
      "  1  70.567    4.275    2.219 7o2g_HBA_dif_108_traj4_T0.3_0_2\n",
      "  2  55.329   14.214    4.622 7o2g_HBA_dif_108_traj4_T0.3_0_3\n",
      "  3  79.684    2.933    2.979 7o2g_HBA_dif_108_traj4_T0.3_0_4\n",
      "  4  52.816    6.309    3.257 7o2g_HBA_dif_108_traj4_T0.3_0_5\n",
      "  5  71.987    6.488    3.877 7o2g_HBA_dif_108_traj1_T0.3_0_1\n",
      "  6  76.370    2.689    2.114 7o2g_HBA_dif_108_traj1_T0.3_0_2\n",
      "  7  59.917    5.593    4.413 7o2g_HBA_dif_108_traj1_T0.3_0_3\n",
      "  8  66.407    3.037    1.924 7o2g_HBA_dif_108_traj1_T0.3_0_4\n",
      "  9  59.292    4.726    2.023 7o2g_HBA_dif_108_traj1_T0.3_0_5\n",
      " 10  64.763   15.353   10.183 7o2g_HBA_dif_108_T0.3_0_1\n",
      " 11  77.047   14.037   13.448 7o2g_HBA_dif_108_T0.3_0_2\n",
      " 12  71.145    5.016    2.859 7o2g_HBA_dif_108_T0.3_0_3\n",
      " 13  52.374   12.287   17.337 7o2g_HBA_dif_108_T0.3_0_4\n",
      " 14  61.384    4.569    2.257 7o2g_HBA_dif_108_T0.3_0_5\n",
      " 15  69.190    2.158    2.697 7o2g_HBA_dif_108_traj2_T0.3_0_1\n",
      " 16  78.304    3.235    1.617 7o2g_HBA_dif_108_traj2_T0.3_0_2\n",
      " 17  86.759    5.735    1.877 7o2g_HBA_dif_108_traj2_T0.3_0_3\n",
      " 18  77.740    2.909    3.442 7o2g_HBA_dif_108_traj2_T0.3_0_4\n",
      " 19  69.219    3.723    3.147 7o2g_HBA_dif_108_traj2_T0.3_0_5\n",
      " 20  76.296    2.795    2.556 7o2g_HBA_dif_108_traj3_T0.3_0_1\n",
      " 21  69.447    3.791    2.172 7o2g_HBA_dif_108_traj3_T0.3_0_2\n",
      " 22  73.892    4.015    3.304 7o2g_HBA_dif_108_traj3_T0.3_0_3\n",
      " 23  62.351   11.660    4.081 7o2g_HBA_dif_108_traj3_T0.3_0_4\n",
      " 24  63.024    6.925    3.349 7o2g_HBA_dif_108_traj3_T0.3_0_5\n",
      "  0  83.371    2.647    3.966 7o2g_HBA_dif_177_traj3_T0.3_0_1\n",
      "  1  61.838   11.368    7.663 7o2g_HBA_dif_177_traj3_T0.3_0_2\n",
      "  2  85.860    1.184    3.181 7o2g_HBA_dif_177_traj3_T0.3_0_3\n",
      "  3  85.759    1.121    2.466 7o2g_HBA_dif_177_traj3_T0.3_0_4\n",
      "  4  75.641    3.306    5.116 7o2g_HBA_dif_177_traj3_T0.3_0_5\n",
      "  5  85.004    3.477    5.355 7o2g_HBA_dif_177_T0.3_0_1\n",
      "  6  82.471    1.090    1.766 7o2g_HBA_dif_177_T0.3_0_2\n",
      "  7  82.111    2.375    4.760 7o2g_HBA_dif_177_T0.3_0_3\n",
      "  8  87.983    1.427    2.030 7o2g_HBA_dif_177_T0.3_0_4\n",
      "  9  74.242    4.761    3.927 7o2g_HBA_dif_177_T0.3_0_5\n",
      " 10  90.766    2.115    2.282 7o2g_HBA_dif_177_traj1_T0.3_0_1\n",
      " 11  84.347    8.371    5.257 7o2g_HBA_dif_177_traj1_T0.3_0_2\n",
      " 12  88.898    1.865    2.774 7o2g_HBA_dif_177_traj1_T0.3_0_3\n",
      " 13  83.465    2.321    5.148 7o2g_HBA_dif_177_traj1_T0.3_0_4\n",
      " 14  84.800    2.082    2.199 7o2g_HBA_dif_177_traj1_T0.3_0_5\n",
      " 15  69.892    3.999    4.981 7o2g_HBA_dif_177_traj2_T0.3_0_1\n",
      " 16  85.506    2.758    3.611 7o2g_HBA_dif_177_traj2_T0.3_0_2\n",
      " 17  85.724    2.078    3.734 7o2g_HBA_dif_177_traj2_T0.3_0_3\n",
      " 18  82.076    1.545    5.710 7o2g_HBA_dif_177_traj2_T0.3_0_4\n",
      " 19  84.428    1.320    2.412 7o2g_HBA_dif_177_traj2_T0.3_0_5\n",
      " 20  83.861    2.815    4.847 7o2g_HBA_dif_177_traj4_T0.3_0_1\n",
      " 21  81.773    1.106    0.828 7o2g_HBA_dif_177_traj4_T0.3_0_2\n",
      " 22  83.023    3.058    6.561 7o2g_HBA_dif_177_traj4_T0.3_0_3\n",
      " 23  63.619   15.798   11.173 7o2g_HBA_dif_177_traj4_T0.3_0_4\n",
      " 24  69.711    2.127    2.710 7o2g_HBA_dif_177_traj4_T0.3_0_5\n",
      "  0  82.078    3.002    9.448 7o2g_HBA_dif_73_traj3_T0.3_0_1\n",
      "  1  87.086    2.046    1.423 7o2g_HBA_dif_73_traj3_T0.3_0_2\n",
      "  2  89.911    2.132    1.257 7o2g_HBA_dif_73_traj3_T0.3_0_3\n",
      "  3  86.113    2.145    1.442 7o2g_HBA_dif_73_traj3_T0.3_0_4\n",
      "  4  87.322    1.733    1.375 7o2g_HBA_dif_73_traj3_T0.3_0_5\n",
      "  5  82.298    1.576    1.963 7o2g_HBA_dif_73_T0.3_0_1\n",
      "  6  80.943    1.898    2.707 7o2g_HBA_dif_73_T0.3_0_2\n",
      "  7  88.749    1.297    1.562 7o2g_HBA_dif_73_T0.3_0_3\n",
      "  8  69.056    4.993    3.767 7o2g_HBA_dif_73_T0.3_0_4\n",
      "  9  72.828    3.168    1.745 7o2g_HBA_dif_73_T0.3_0_5\n",
      " 10  79.088    2.304    2.527 7o2g_HBA_dif_73_traj2_T0.3_0_1\n",
      " 11  86.442    2.525    1.576 7o2g_HBA_dif_73_traj2_T0.3_0_2\n",
      " 12  90.771    0.944    1.167 7o2g_HBA_dif_73_traj2_T0.3_0_3\n",
      " 13  85.066    1.826    0.980 7o2g_HBA_dif_73_traj2_T0.3_0_4\n",
      " 14  83.346    2.959    1.284 7o2g_HBA_dif_73_traj2_T0.3_0_5\n",
      " 15  87.508    3.194    7.869 7o2g_HBA_dif_73_traj4_T0.3_0_1\n",
      " 16  88.070    1.704    2.317 7o2g_HBA_dif_73_traj4_T0.3_0_2\n",
      " 17  89.121    2.682    1.416 7o2g_HBA_dif_73_traj4_T0.3_0_3\n",
      " 18  88.368    1.507    2.423 7o2g_HBA_dif_73_traj4_T0.3_0_4\n",
      " 19  89.348    1.965    1.547 7o2g_HBA_dif_73_traj4_T0.3_0_5\n",
      " 20  85.970    1.968    0.687 7o2g_HBA_dif_73_traj1_T0.3_0_1\n",
      " 21  89.184    2.661    1.227 7o2g_HBA_dif_73_traj1_T0.3_0_2\n",
      " 22  88.626    1.287    1.275 7o2g_HBA_dif_73_traj1_T0.3_0_3\n",
      " 23  84.175    1.956    0.943 7o2g_HBA_dif_73_traj1_T0.3_0_4\n",
      " 24  90.284    3.771    9.555 7o2g_HBA_dif_73_traj1_T0.3_0_5\n",
      " 25  80.008    2.375    4.415 7o2g_HBA_dif_97_traj4_T0.3_0_1\n",
      " 26  66.035    4.265    6.350 7o2g_HBA_dif_97_traj4_T0.3_0_2\n",
      " 27  79.423    2.653    3.889 7o2g_HBA_dif_97_traj4_T0.3_0_3\n",
      " 28  77.455    2.970    5.863 7o2g_HBA_dif_97_traj4_T0.3_0_4\n",
      " 29  72.151    2.798    3.703 7o2g_HBA_dif_97_traj4_T0.3_0_5\n",
      " 30  68.107    7.511    3.326 7o2g_HBA_dif_97_T0.3_0_1\n",
      " 31  67.425    7.034    3.283 7o2g_HBA_dif_97_T0.3_0_2\n",
      " 32  60.456    3.458    3.126 7o2g_HBA_dif_97_T0.3_0_3\n",
      " 33  63.713    2.671    3.452 7o2g_HBA_dif_97_T0.3_0_4\n",
      " 34  63.902    8.935    5.565 7o2g_HBA_dif_97_T0.3_0_5\n",
      "  0  65.194    2.567    3.943 7o2g_HBA_dif_43_traj3_T0.3_0_1\n",
      "  1  66.566    4.392    6.319 7o2g_HBA_dif_43_traj3_T0.3_0_2\n",
      "  2  62.977    3.630    2.980 7o2g_HBA_dif_43_traj3_T0.3_0_3\n",
      "  3  59.610    5.440    5.562 7o2g_HBA_dif_43_traj3_T0.3_0_4\n",
      "  4  48.429   10.546    1.685 7o2g_HBA_dif_43_traj3_T0.3_0_5\n",
      "  5  60.875    4.102    3.402 7o2g_HBA_dif_43_traj4_T0.3_0_1\n",
      "  6  71.437    1.805    3.112 7o2g_HBA_dif_43_traj4_T0.3_0_2\n",
      "  7  67.778    2.817    1.812 7o2g_HBA_dif_43_traj4_T0.3_0_3\n",
      "  8  76.121    2.866    2.817 7o2g_HBA_dif_43_traj4_T0.3_0_4\n",
      "  9  68.131    2.665    3.911 7o2g_HBA_dif_43_traj4_T0.3_0_5\n",
      " 10  74.130    3.029    2.102 7o2g_HBA_dif_43_traj1_T0.3_0_1\n",
      " 11  65.656    3.170    4.448 7o2g_HBA_dif_43_traj1_T0.3_0_2\n",
      " 12  57.756    6.935    5.392 7o2g_HBA_dif_43_traj1_T0.3_0_3\n",
      " 13  72.659    3.020    2.804 7o2g_HBA_dif_43_traj1_T0.3_0_4\n",
      " 14  61.029    5.135    4.547 7o2g_HBA_dif_43_traj1_T0.3_0_5\n",
      " 15  44.268   14.142    9.547 7o2g_HBA_dif_105_traj2_T0.3_0_1\n",
      " 16  60.655    6.285    5.188 7o2g_HBA_dif_105_traj2_T0.3_0_2\n",
      " 17  51.279   12.922    8.239 7o2g_HBA_dif_105_traj2_T0.3_0_3\n",
      " 18  48.836    9.009    4.325 7o2g_HBA_dif_105_traj2_T0.3_0_4\n",
      " 19  55.916   12.941    4.381 7o2g_HBA_dif_105_traj2_T0.3_0_5\n",
      " 20  51.618   12.046   13.327 7o2g_HBA_dif_105_T0.3_0_1\n",
      " 21  55.194   11.378    8.180 7o2g_HBA_dif_105_T0.3_0_2\n",
      " 22  59.285    8.690    3.662 7o2g_HBA_dif_105_T0.3_0_3\n",
      " 23  69.327   15.671    5.167 7o2g_HBA_dif_105_T0.3_0_4\n",
      " 24  55.518   11.341    6.144 7o2g_HBA_dif_105_T0.3_0_5\n",
      " 25  70.342    3.590    6.167 7o2g_HBA_dif_43_T0.3_0_1\n",
      " 26  81.450    3.250    6.417 7o2g_HBA_dif_43_T0.3_0_2\n",
      " 27  64.787    5.617    4.628 7o2g_HBA_dif_43_T0.3_0_3\n",
      " 28  78.406    5.752    5.145 7o2g_HBA_dif_43_T0.3_0_4\n",
      " 29  63.956    4.081    2.664 7o2g_HBA_dif_43_T0.3_0_5\n",
      " 30  44.666   14.784    4.208 7o2g_HBA_dif_105_traj1_T0.3_0_1\n",
      " 31  48.667   10.352   18.908 7o2g_HBA_dif_105_traj1_T0.3_0_2\n",
      " 32  53.198    9.618    3.999 7o2g_HBA_dif_105_traj1_T0.3_0_3\n",
      " 33  55.124   11.563    3.704 7o2g_HBA_dif_105_traj1_T0.3_0_4\n",
      " 34  53.411    9.403    6.799 7o2g_HBA_dif_105_traj1_T0.3_0_5\n",
      " 35  55.213    9.505    5.473 7o2g_HBA_dif_105_traj3_T0.3_0_1\n",
      " 36  51.856   12.117    8.299 7o2g_HBA_dif_105_traj3_T0.3_0_2\n",
      " 37  72.497   10.601    3.440 7o2g_HBA_dif_105_traj3_T0.3_0_3\n",
      " 38  77.104    2.440    3.568 7o2g_HBA_dif_105_traj3_T0.3_0_4\n",
      " 39  53.486    3.248    1.531 7o2g_HBA_dif_105_traj3_T0.3_0_5\n",
      " 40  51.133    9.051   10.048 7o2g_HBA_dif_43_traj2_T0.3_0_1\n",
      " 41  66.166    3.204    3.660 7o2g_HBA_dif_43_traj2_T0.3_0_2\n",
      " 42  55.613    5.462    4.592 7o2g_HBA_dif_43_traj2_T0.3_0_3\n",
      " 43  87.798    2.158    2.526 7o2g_HBA_dif_43_traj2_T0.3_0_4\n",
      " 44  72.406    7.573    6.271 7o2g_HBA_dif_43_traj2_T0.3_0_5\n",
      " 45  49.447   11.821    2.753 7o2g_HBA_dif_105_traj4_T0.3_0_1\n",
      " 46  59.629   13.211    4.821 7o2g_HBA_dif_105_traj4_T0.3_0_2\n",
      " 47  49.910   13.940   10.913 7o2g_HBA_dif_105_traj4_T0.3_0_3\n",
      " 48  49.942    8.858    4.103 7o2g_HBA_dif_105_traj4_T0.3_0_4\n",
      " 49  54.537   10.315    9.080 7o2g_HBA_dif_105_traj4_T0.3_0_5\n",
      "  0  53.540    9.591    1.349 7o2g_HBA_dif_130_traj2_T0.3_0_1\n",
      "  1  70.867    3.601    2.446 7o2g_HBA_dif_130_traj2_T0.3_0_2\n",
      "  2  76.718   10.909    6.596 7o2g_HBA_dif_130_traj2_T0.3_0_3\n",
      "  3  59.400    9.539    6.680 7o2g_HBA_dif_130_traj2_T0.3_0_4\n",
      "  4  90.981    0.944    0.984 7o2g_HBA_dif_130_traj2_T0.3_0_5\n",
      "  5  54.257    4.352    4.933 7o2g_HBA_dif_69_traj4_T0.3_0_1\n",
      "  6  75.091    2.723    3.733 7o2g_HBA_dif_69_traj4_T0.3_0_2\n",
      "  7  80.908    3.078    2.978 7o2g_HBA_dif_69_traj4_T0.3_0_3\n",
      "  8  63.742    2.432    4.433 7o2g_HBA_dif_69_traj4_T0.3_0_4\n",
      "  9  77.297    2.407    3.671 7o2g_HBA_dif_69_traj4_T0.3_0_5\n",
      " 10  74.178    7.344    9.311 7o2g_HBA_dif_69_traj2_T0.3_0_1\n",
      " 11  64.999    2.108    4.041 7o2g_HBA_dif_69_traj2_T0.3_0_2\n",
      " 12  55.640    8.762   17.475 7o2g_HBA_dif_69_traj2_T0.3_0_3\n",
      " 13  76.839    8.885   20.323 7o2g_HBA_dif_69_traj2_T0.3_0_4\n",
      " 14  82.883    1.915    2.472 7o2g_HBA_dif_69_traj2_T0.3_0_5\n",
      " 15  70.316    3.257    2.480 7o2g_HBA_dif_148_traj4_T0.3_0_1\n",
      " 16  77.627    4.391    3.035 7o2g_HBA_dif_148_traj4_T0.3_0_2\n",
      " 17  84.528    4.012    7.525 7o2g_HBA_dif_148_traj4_T0.3_0_3\n",
      " 18  68.735   10.281    9.405 7o2g_HBA_dif_148_traj4_T0.3_0_4\n",
      " 19  75.538   11.169    7.932 7o2g_HBA_dif_148_traj4_T0.3_0_5\n",
      " 20  86.934    1.400    1.928 7o2g_HBA_dif_130_traj4_T0.3_0_1\n",
      " 21  83.701    2.917    2.727 7o2g_HBA_dif_130_traj4_T0.3_0_2\n",
      " 22  87.186    1.216    2.993 7o2g_HBA_dif_130_traj4_T0.3_0_3\n",
      " 23  88.671    1.470    3.534 7o2g_HBA_dif_130_traj4_T0.3_0_4\n",
      " 24  76.522    3.460    1.870 7o2g_HBA_dif_130_traj4_T0.3_0_5\n",
      " 25  75.479    2.637    3.196 7o2g_HBA_dif_130_traj1_T0.3_0_1\n",
      " 26  63.653    7.665   11.241 7o2g_HBA_dif_130_traj1_T0.3_0_2\n",
      " 27  77.517    4.988    6.709 7o2g_HBA_dif_130_traj1_T0.3_0_3\n",
      " 28  84.804    1.409    2.447 7o2g_HBA_dif_130_traj1_T0.3_0_4\n",
      " 29  70.530    2.464    3.235 7o2g_HBA_dif_130_traj1_T0.3_0_5\n",
      " 30  57.861   10.640    8.613 7o2g_HBA_dif_148_traj2_T0.3_0_1\n",
      " 31  75.404   10.929   14.642 7o2g_HBA_dif_148_traj2_T0.3_0_2\n",
      " 32  80.354   11.303   15.931 7o2g_HBA_dif_148_traj2_T0.3_0_3\n",
      " 33  75.578   11.843   13.604 7o2g_HBA_dif_148_traj2_T0.3_0_4\n",
      " 34  60.570   10.493    9.049 7o2g_HBA_dif_148_traj2_T0.3_0_5\n",
      " 35  71.691    3.215    2.399 7o2g_HBA_dif_130_traj3_T0.3_0_1\n",
      " 36  79.398    1.686    1.706 7o2g_HBA_dif_130_traj3_T0.3_0_2\n",
      " 37  60.422    9.095    3.566 7o2g_HBA_dif_130_traj3_T0.3_0_3\n",
      " 38  66.758    9.956    1.853 7o2g_HBA_dif_130_traj3_T0.3_0_4\n",
      " 39  59.157   12.565    1.878 7o2g_HBA_dif_130_traj3_T0.3_0_5\n",
      " 40  80.251    2.204    3.026 7o2g_HBA_dif_69_traj3_T0.3_0_1\n",
      " 41  69.890    2.986    3.663 7o2g_HBA_dif_69_traj3_T0.3_0_2\n",
      " 42  57.226    6.016    7.954 7o2g_HBA_dif_69_traj3_T0.3_0_3\n",
      " 43  60.567    9.403   13.196 7o2g_HBA_dif_69_traj3_T0.3_0_4\n",
      " 44  54.665   12.992    8.356 7o2g_HBA_dif_69_traj3_T0.3_0_5\n",
      " 45  76.231    3.268    3.598 7o2g_HBA_dif_130_T0.3_0_1\n",
      " 46  80.471    3.644    3.436 7o2g_HBA_dif_130_T0.3_0_2\n",
      " 47  69.254   17.504   13.623 7o2g_HBA_dif_130_T0.3_0_3\n",
      " 48  77.808    3.176    4.460 7o2g_HBA_dif_130_T0.3_0_4\n",
      " 49  86.520    3.090    1.516 7o2g_HBA_dif_130_T0.3_0_5\n",
      "  0  60.004   14.937   11.985 7o2g_HBA_dif_158_traj2_T0.3_0_1\n",
      "  1  63.125    4.267    6.097 7o2g_HBA_dif_158_traj2_T0.3_0_2\n",
      "  2  65.058    4.011    6.566 7o2g_HBA_dif_158_traj2_T0.3_0_3\n",
      "  3  87.254    1.842    2.227 7o2g_HBA_dif_158_traj2_T0.3_0_4\n",
      "  4  69.564    3.466    6.093 7o2g_HBA_dif_158_traj2_T0.3_0_5\n",
      "  5  49.963   11.185   13.620 7o2g_HBA_dif_158_traj3_T0.3_0_1\n",
      "  6  67.465   11.130    7.295 7o2g_HBA_dif_158_traj3_T0.3_0_2\n",
      "  7  82.749    2.568    5.388 7o2g_HBA_dif_158_traj3_T0.3_0_3\n",
      "  8  63.523   13.999    7.035 7o2g_HBA_dif_158_traj3_T0.3_0_4\n",
      "  9  80.985    9.272    7.863 7o2g_HBA_dif_158_traj3_T0.3_0_5\n",
      " 10  67.141    4.814    4.898 7o2g_HBA_dif_158_traj4_T0.3_0_1\n",
      " 11  56.049    5.989    7.574 7o2g_HBA_dif_158_traj4_T0.3_0_2\n",
      " 12  50.873   11.317    4.560 7o2g_HBA_dif_158_traj4_T0.3_0_3\n",
      " 13  64.318    7.567    9.007 7o2g_HBA_dif_158_traj4_T0.3_0_4\n",
      " 14  62.831   15.430    7.416 7o2g_HBA_dif_158_traj4_T0.3_0_5\n",
      "  0  57.243    9.292   13.830 7o2g_HBA_dif_216_traj3_T0.3_0_1\n",
      "  1  45.107   13.299   12.524 7o2g_HBA_dif_216_traj3_T0.3_0_2\n",
      "  2  46.214   15.236   12.148 7o2g_HBA_dif_216_traj3_T0.3_0_3\n",
      "  3  58.426   12.282   12.527 7o2g_HBA_dif_216_traj3_T0.3_0_4\n",
      "  4  45.350   14.832    5.234 7o2g_HBA_dif_216_traj3_T0.3_0_5\n",
      "  5  82.077    2.318    4.006 7o2g_HBA_dif_24_traj4_T0.3_0_1\n",
      "  6  68.017    4.643    5.637 7o2g_HBA_dif_24_traj4_T0.3_0_2\n",
      "  7  87.529    3.204    4.448 7o2g_HBA_dif_24_traj4_T0.3_0_3\n",
      "  8  78.105    8.112    3.896 7o2g_HBA_dif_24_traj4_T0.3_0_4\n",
      "  9  84.870    3.816    2.836 7o2g_HBA_dif_24_traj4_T0.3_0_5\n",
      " 10  79.722    5.535    3.101 7o2g_HBA_dif_147_traj2_T0.3_0_1\n",
      " 11  77.537    2.874    4.663 7o2g_HBA_dif_147_traj2_T0.3_0_2\n",
      " 12  79.531    3.470    3.883 7o2g_HBA_dif_147_traj2_T0.3_0_3\n",
      " 13  87.664    2.585    4.933 7o2g_HBA_dif_147_traj2_T0.3_0_4\n",
      " 14  77.730    4.552    8.732 7o2g_HBA_dif_147_traj2_T0.3_0_5\n",
      " 15  73.540    3.107    3.541 7o2g_HBA_dif_24_traj2_T0.3_0_1\n",
      " 16  60.335    5.645    7.118 7o2g_HBA_dif_24_traj2_T0.3_0_2\n",
      " 17  75.680    2.759    5.865 7o2g_HBA_dif_24_traj2_T0.3_0_3\n",
      " 18  78.474    2.380    3.590 7o2g_HBA_dif_24_traj2_T0.3_0_4\n",
      " 19  82.715    1.888    3.174 7o2g_HBA_dif_24_traj2_T0.3_0_5\n",
      " 20  81.479    1.821    2.328 7o2g_HBA_dif_147_traj4_T0.3_0_1\n",
      " 21  82.408    2.145    4.119 7o2g_HBA_dif_147_traj4_T0.3_0_2\n",
      " 22  77.568    1.742    1.432 7o2g_HBA_dif_147_traj4_T0.3_0_3\n",
      " 23  80.444    3.477    6.178 7o2g_HBA_dif_147_traj4_T0.3_0_4\n",
      " 24  78.303    2.514    3.540 7o2g_HBA_dif_147_traj4_T0.3_0_5\n",
      " 25  73.506    4.220    4.510 7o2g_HBA_dif_227_traj3_T0.3_0_1\n",
      " 26  78.278    4.209    3.027 7o2g_HBA_dif_227_traj3_T0.3_0_2\n",
      " 27  66.563    3.562    4.409 7o2g_HBA_dif_227_traj3_T0.3_0_3\n",
      " 28  74.977    4.030    3.844 7o2g_HBA_dif_227_traj3_T0.3_0_4\n",
      " 29  69.667    6.856    5.186 7o2g_HBA_dif_227_traj3_T0.3_0_5\n",
      " 30  62.627    6.995    4.635 7o2g_HBA_dif_89_traj3_T0.3_0_1\n",
      " 31  68.061    4.918    6.590 7o2g_HBA_dif_89_traj3_T0.3_0_2\n",
      " 32  72.410    2.185    4.044 7o2g_HBA_dif_89_traj3_T0.3_0_3\n",
      " 33  70.928    3.742    7.045 7o2g_HBA_dif_89_traj3_T0.3_0_4\n",
      " 34  64.066    8.057    7.062 7o2g_HBA_dif_89_traj3_T0.3_0_5\n",
      " 35  81.978    1.520    2.492 7o2g_HBA_dif_24_traj3_T0.3_0_1\n",
      " 36  85.689    3.029    3.210 7o2g_HBA_dif_24_traj3_T0.3_0_2\n",
      " 37  82.552    1.740    3.507 7o2g_HBA_dif_24_traj3_T0.3_0_3\n",
      " 38  78.703    1.745    3.578 7o2g_HBA_dif_24_traj3_T0.3_0_4\n",
      " 39  72.988    9.906   13.719 7o2g_HBA_dif_24_traj3_T0.3_0_5\n",
      " 40  80.772    1.939    1.927 7o2g_HBA_dif_95_traj2_T0.3_0_1\n",
      " 41  77.301   12.260   12.951 7o2g_HBA_dif_95_traj2_T0.3_0_2\n",
      " 42  59.043   13.778    3.592 7o2g_HBA_dif_95_traj2_T0.3_0_3\n",
      " 43  47.553    8.810    3.337 7o2g_HBA_dif_95_traj2_T0.3_0_4\n",
      " 44  61.927    7.441    5.785 7o2g_HBA_dif_95_traj2_T0.3_0_5\n",
      " 45  77.663    2.472    4.553 7o2g_HBA_dif_89_traj2_T0.3_0_1\n",
      " 46  73.523    1.999    2.283 7o2g_HBA_dif_89_traj2_T0.3_0_2\n",
      " 47  81.671    2.844    5.177 7o2g_HBA_dif_89_traj2_T0.3_0_3\n",
      " 48  66.995    5.182    3.234 7o2g_HBA_dif_89_traj2_T0.3_0_4\n",
      " 49  74.159    6.334    6.164 7o2g_HBA_dif_89_traj2_T0.3_0_5\n",
      " 50  70.113    3.563    4.228 7o2g_HBA_dif_227_T0.3_0_1\n",
      " 51  72.631    4.138    0.725 7o2g_HBA_dif_227_T0.3_0_2\n",
      " 52  69.883    6.917    2.406 7o2g_HBA_dif_227_T0.3_0_3\n",
      " 53  85.034    4.484    3.196 7o2g_HBA_dif_227_T0.3_0_4\n",
      " 54  77.651    4.710    4.383 7o2g_HBA_dif_227_T0.3_0_5\n",
      " 55  50.890   13.839    6.642 7o2g_HBA_dif_216_traj2_T0.3_0_1\n",
      " 56  56.180    9.455    3.695 7o2g_HBA_dif_216_traj2_T0.3_0_2\n",
      " 57  55.230   16.004   16.886 7o2g_HBA_dif_216_traj2_T0.3_0_3\n",
      " 58  51.153    8.336    9.353 7o2g_HBA_dif_216_traj2_T0.3_0_4\n",
      " 59  52.250   10.904    2.955 7o2g_HBA_dif_216_traj2_T0.3_0_5\n",
      " 60  68.492    2.734    1.481 7o2g_HBA_dif_89_traj4_T0.3_0_1\n",
      " 61  87.688    2.440    1.935 7o2g_HBA_dif_89_traj4_T0.3_0_2\n",
      " 62  72.445    5.979    4.715 7o2g_HBA_dif_89_traj4_T0.3_0_3\n",
      " 63  74.948    3.377    4.324 7o2g_HBA_dif_89_traj4_T0.3_0_4\n",
      " 64  70.371    9.965    8.593 7o2g_HBA_dif_89_traj4_T0.3_0_5\n",
      " 65  77.607    5.682    9.810 7o2g_HBA_dif_95_traj4_T0.3_0_1\n",
      " 66  71.685    2.763    3.265 7o2g_HBA_dif_95_traj4_T0.3_0_2\n",
      " 67  51.822   11.171   11.975 7o2g_HBA_dif_95_traj4_T0.3_0_3\n",
      " 68  78.739    3.893    6.054 7o2g_HBA_dif_95_traj4_T0.3_0_4\n",
      " 69  72.938    4.823    5.083 7o2g_HBA_dif_95_traj4_T0.3_0_5\n",
      " 70  75.303    2.351    2.647 7o2g_HBA_dif_89_T0.3_0_1\n",
      " 71  74.667    2.917    6.333 7o2g_HBA_dif_89_T0.3_0_2\n",
      " 72  77.683    2.158    3.180 7o2g_HBA_dif_89_T0.3_0_3\n",
      " 73  72.981    2.130    4.130 7o2g_HBA_dif_89_T0.3_0_4\n",
      " 74  69.702    5.102    5.582 7o2g_HBA_dif_89_T0.3_0_5\n",
      " 75  53.916    9.260    6.055 7o2g_HBA_dif_216_traj4_T0.3_0_1\n",
      " 76  56.575   10.259   11.621 7o2g_HBA_dif_216_traj4_T0.3_0_2\n",
      " 77  48.078   14.647   15.252 7o2g_HBA_dif_216_traj4_T0.3_0_3\n",
      " 78  56.371    6.380    4.421 7o2g_HBA_dif_216_traj4_T0.3_0_4\n",
      " 79  49.647   10.020    4.422 7o2g_HBA_dif_216_traj4_T0.3_0_5\n",
      " 80  78.980    3.495    4.771 7o2g_HBA_dif_147_T0.3_0_1\n",
      " 81  87.928    2.822    3.859 7o2g_HBA_dif_147_T0.3_0_2\n",
      " 82  89.145    1.538    1.191 7o2g_HBA_dif_147_T0.3_0_3\n",
      " 83  77.564    4.845    4.309 7o2g_HBA_dif_147_T0.3_0_4\n",
      " 84  85.856    1.994    3.176 7o2g_HBA_dif_147_T0.3_0_5\n",
      " 85  50.501   17.224   20.485 7o2g_HBA_dif_216_T0.3_0_1\n",
      " 86  43.545   15.389   12.398 7o2g_HBA_dif_216_T0.3_0_2\n",
      " 87  52.068   13.216    5.821 7o2g_HBA_dif_216_T0.3_0_3\n",
      " 88  44.999   15.036    7.902 7o2g_HBA_dif_216_T0.3_0_4\n",
      " 89  44.317   15.628   10.213 7o2g_HBA_dif_216_T0.3_0_5\n",
      " 90  65.660   14.237    4.509 7o2g_HBA_dif_227_traj4_T0.3_0_1\n",
      " 91  73.404    4.380    3.641 7o2g_HBA_dif_227_traj4_T0.3_0_2\n",
      " 92  74.002    5.691    4.119 7o2g_HBA_dif_227_traj4_T0.3_0_3\n",
      " 93  72.990    3.702    4.229 7o2g_HBA_dif_227_traj4_T0.3_0_4\n",
      " 94  70.874    3.543    2.766 7o2g_HBA_dif_227_traj4_T0.3_0_5\n",
      " 95  73.174    4.265    5.300 7o2g_HBA_dif_89_traj1_T0.3_0_1\n",
      " 96  70.917    5.854    3.144 7o2g_HBA_dif_89_traj1_T0.3_0_2\n",
      " 97  79.218    3.665    6.006 7o2g_HBA_dif_89_traj1_T0.3_0_3\n",
      " 98  82.370    3.221    6.002 7o2g_HBA_dif_89_traj1_T0.3_0_4\n",
      " 99  72.175    6.681    4.607 7o2g_HBA_dif_89_traj1_T0.3_0_5\n",
      "  0  80.714    3.575    4.331 7o2g_HBA_dif_147_traj3_T0.3_0_1\n",
      "  1  82.285    2.676    4.330 7o2g_HBA_dif_147_traj3_T0.3_0_2\n",
      "  2  82.729    2.933    4.585 7o2g_HBA_dif_147_traj3_T0.3_0_3\n",
      "  3  83.921    4.713    9.242 7o2g_HBA_dif_147_traj3_T0.3_0_4\n",
      "  4  89.540    2.579    5.016 7o2g_HBA_dif_147_traj3_T0.3_0_5\n",
      "  5  52.408   10.646   23.568 7o2g_HBA_dif_227_traj1_T0.3_0_1\n",
      "  6  69.936   14.282    4.806 7o2g_HBA_dif_227_traj1_T0.3_0_2\n",
      "  7  77.997    2.270    2.887 7o2g_HBA_dif_227_traj1_T0.3_0_3\n",
      "  8  71.649    6.382    2.917 7o2g_HBA_dif_227_traj1_T0.3_0_4\n",
      "  9  73.130    9.828    7.801 7o2g_HBA_dif_227_traj1_T0.3_0_5\n",
      " 10  59.773    3.223    2.039 7o2g_HBA_dif_147_traj1_T0.3_0_1\n",
      " 11  84.126    2.958    3.480 7o2g_HBA_dif_147_traj1_T0.3_0_2\n",
      " 12  85.290    3.665    4.301 7o2g_HBA_dif_147_traj1_T0.3_0_3\n",
      " 13  84.612    2.549    5.471 7o2g_HBA_dif_147_traj1_T0.3_0_4\n",
      " 14  85.850    2.461    3.048 7o2g_HBA_dif_147_traj1_T0.3_0_5\n",
      " 15  80.460    4.285    3.086 7o2g_HBA_dif_24_traj1_T0.3_0_1\n",
      " 16  68.405    7.699    6.205 7o2g_HBA_dif_24_traj1_T0.3_0_2\n",
      " 17  84.020    1.424    2.860 7o2g_HBA_dif_24_traj1_T0.3_0_3\n",
      " 18  86.729    3.015    4.073 7o2g_HBA_dif_24_traj1_T0.3_0_4\n",
      " 19  85.847    3.600    3.611 7o2g_HBA_dif_24_traj1_T0.3_0_5\n",
      " 20  51.836   12.815   13.355 7o2g_HBA_dif_216_traj1_T0.3_0_1\n",
      " 21  53.474    9.744    2.938 7o2g_HBA_dif_216_traj1_T0.3_0_2\n",
      " 22  42.790   11.675    9.667 7o2g_HBA_dif_216_traj1_T0.3_0_3\n",
      " 23  58.992   12.534    2.706 7o2g_HBA_dif_216_traj1_T0.3_0_4\n",
      " 24  44.634    8.977    5.251 7o2g_HBA_dif_216_traj1_T0.3_0_5\n",
      " 25  68.025   10.951    8.264 7o2g_HBA_dif_24_T0.3_0_1\n",
      " 26  60.243   10.145    9.140 7o2g_HBA_dif_24_T0.3_0_2\n",
      " 27  72.401    9.440    2.612 7o2g_HBA_dif_24_T0.3_0_3\n",
      " 28  83.878    3.976    3.493 7o2g_HBA_dif_24_T0.3_0_4\n",
      " 29  82.594    1.665    3.259 7o2g_HBA_dif_24_T0.3_0_5\n",
      "  0  78.417    6.220    2.563 7o2g_HBA_dif_45_traj1_T0.3_0_1\n",
      "  1  55.068    7.218    1.905 7o2g_HBA_dif_45_traj1_T0.3_0_2\n",
      "  2  64.651    2.570    1.637 7o2g_HBA_dif_45_traj1_T0.3_0_3\n",
      "  3  55.308   15.459    5.791 7o2g_HBA_dif_45_traj1_T0.3_0_4\n",
      "  4  62.807   11.411    5.124 7o2g_HBA_dif_45_traj1_T0.3_0_5\n",
      "  5  58.530    9.080    4.252 7o2g_HBA_dif_45_traj2_T0.3_0_1\n",
      "  6  53.186   19.060    9.979 7o2g_HBA_dif_45_traj2_T0.3_0_2\n",
      "  7  52.607   16.684    5.974 7o2g_HBA_dif_45_traj2_T0.3_0_3\n",
      "  8  41.859   18.362   10.081 7o2g_HBA_dif_45_traj2_T0.3_0_4\n",
      "  9  53.140   15.941   15.466 7o2g_HBA_dif_45_traj2_T0.3_0_5\n",
      " 10  75.186    3.043    1.410 7o2g_HBA_dif_45_traj3_T0.3_0_1\n",
      " 11  52.148   17.261    7.882 7o2g_HBA_dif_45_traj3_T0.3_0_2\n",
      " 12  70.419    2.435    2.158 7o2g_HBA_dif_45_traj3_T0.3_0_3\n",
      " 13  47.960   16.469    3.697 7o2g_HBA_dif_45_traj3_T0.3_0_4\n",
      " 14  52.288   17.305    9.307 7o2g_HBA_dif_45_traj3_T0.3_0_5\n",
      " 15  60.764   11.679    5.495 7o2g_HBA_dif_45_traj4_T0.3_0_1\n",
      " 16  67.317   14.640    4.556 7o2g_HBA_dif_45_traj4_T0.3_0_2\n",
      " 17  73.521    1.980    2.267 7o2g_HBA_dif_45_traj4_T0.3_0_3\n",
      " 18  60.116    9.411    2.372 7o2g_HBA_dif_45_traj4_T0.3_0_4\n",
      " 19  55.625    9.374    4.011 7o2g_HBA_dif_45_traj4_T0.3_0_5\n",
      " 20  58.658   14.709    6.461 7o2g_HBA_dif_45_T0.3_0_1\n",
      " 21  79.818    8.830    9.620 7o2g_HBA_dif_45_T0.3_0_2\n",
      " 22  58.817   14.816    9.624 7o2g_HBA_dif_45_T0.3_0_3\n",
      " 23  69.443   15.253   16.233 7o2g_HBA_dif_45_T0.3_0_4\n",
      " 24  62.338    6.346    1.300 7o2g_HBA_dif_45_T0.3_0_5\n",
      "  0  70.703    5.243    3.621 7o2g_HBA_dif_1_traj4_T0.3_0_1\n",
      "  1  64.687    5.186    6.993 7o2g_HBA_dif_1_traj4_T0.3_0_2\n",
      "  2  80.893    3.693    3.209 7o2g_HBA_dif_1_traj4_T0.3_0_3\n",
      "  3  64.948    4.921    3.365 7o2g_HBA_dif_1_traj4_T0.3_0_4\n",
      "  4  50.190    8.642    1.446 7o2g_HBA_dif_1_traj4_T0.3_0_5\n",
      "  5  71.763   14.792   14.237 7o2g_HBA_dif_1_traj1_T0.3_0_1\n",
      "  6  82.059    4.646    5.152 7o2g_HBA_dif_1_traj1_T0.3_0_2\n",
      "  7  75.378    7.929    6.774 7o2g_HBA_dif_1_traj1_T0.3_0_3\n",
      "  8  71.335    3.517    2.547 7o2g_HBA_dif_1_traj1_T0.3_0_4\n",
      "  9  75.759    2.650    4.968 7o2g_HBA_dif_1_traj1_T0.3_0_5\n",
      " 10  51.222    5.148    4.905 7o2g_HBA_dif_173_T0.3_0_1\n",
      " 11  46.922    8.188    8.065 7o2g_HBA_dif_173_T0.3_0_2\n",
      " 12  54.181    7.936   11.967 7o2g_HBA_dif_173_T0.3_0_3\n",
      " 13  77.155   10.174   10.812 7o2g_HBA_dif_173_T0.3_0_4\n",
      " 14  61.572    6.379    4.895 7o2g_HBA_dif_173_T0.3_0_5\n",
      " 15  71.844    4.436    1.708 7o2g_HBA_dif_1_T0.3_0_1\n",
      " 16  69.362   15.902   17.335 7o2g_HBA_dif_1_T0.3_0_2\n",
      " 17  77.972    1.815    3.509 7o2g_HBA_dif_1_T0.3_0_3\n",
      " 18  64.755   19.236   21.385 7o2g_HBA_dif_1_T0.3_0_4\n",
      " 19  70.703   16.824   17.383 7o2g_HBA_dif_1_T0.3_0_5\n",
      " 20  63.777    2.831    3.294 7o2g_HBA_dif_1_traj3_T0.3_0_1\n",
      " 21  67.744    2.345    3.900 7o2g_HBA_dif_1_traj3_T0.3_0_2\n",
      " 22  59.511    7.966    7.801 7o2g_HBA_dif_1_traj3_T0.3_0_3\n",
      " 23  68.360    3.987    1.847 7o2g_HBA_dif_1_traj3_T0.3_0_4\n",
      " 24  74.203    7.308    7.672 7o2g_HBA_dif_1_traj3_T0.3_0_5\n",
      " 25  74.975   15.149   15.919 7o2g_HBA_dif_1_traj2_T0.3_0_1\n",
      " 26  60.532   16.937   18.455 7o2g_HBA_dif_1_traj2_T0.3_0_2\n",
      " 27  65.486   17.344   18.429 7o2g_HBA_dif_1_traj2_T0.3_0_3\n",
      " 28  78.870   17.343   18.349 7o2g_HBA_dif_1_traj2_T0.3_0_4\n",
      " 29  71.440   15.761   17.930 7o2g_HBA_dif_1_traj2_T0.3_0_5\n",
      "  0  62.663   15.618    8.775 7o2g_HBA_dif_132_traj1_T0.3_0_1\n",
      "  1  82.104    3.467    2.648 7o2g_HBA_dif_132_traj1_T0.3_0_2\n",
      "  2  72.602    2.815    2.085 7o2g_HBA_dif_132_traj1_T0.3_0_3\n",
      "  3  68.630    4.247    1.910 7o2g_HBA_dif_132_traj1_T0.3_0_4\n",
      "  4  69.101   11.949    1.587 7o2g_HBA_dif_132_traj1_T0.3_0_5\n",
      "  5  85.283    1.773    2.093 7o2g_HBA_dif_132_traj2_T0.3_0_1\n",
      "  6  79.614    2.441    2.316 7o2g_HBA_dif_132_traj2_T0.3_0_2\n",
      "  7  81.039    3.652    3.445 7o2g_HBA_dif_132_traj2_T0.3_0_3\n",
      "  8  74.455    1.929    1.994 7o2g_HBA_dif_132_traj2_T0.3_0_4\n",
      "  9  77.450    2.092    1.896 7o2g_HBA_dif_132_traj2_T0.3_0_5\n",
      " 10  56.067   10.573    1.348 7o2g_HBA_dif_132_traj3_T0.3_0_1\n",
      " 11  64.587    9.362    1.424 7o2g_HBA_dif_132_traj3_T0.3_0_2\n",
      " 12  76.209    9.190    2.653 7o2g_HBA_dif_132_traj3_T0.3_0_3\n",
      " 13  79.143    2.601    1.650 7o2g_HBA_dif_132_traj3_T0.3_0_4\n",
      " 14  81.595    2.561    1.746 7o2g_HBA_dif_132_traj3_T0.3_0_5\n",
      " 15  85.662    3.718    3.856 7o2g_HBA_dif_132_traj4_T0.3_0_1\n",
      " 16  91.177    0.985    1.538 7o2g_HBA_dif_132_traj4_T0.3_0_2\n",
      " 17  76.828    5.549    3.604 7o2g_HBA_dif_132_traj4_T0.3_0_3\n",
      " 18  72.615    2.404    1.559 7o2g_HBA_dif_132_traj4_T0.3_0_4\n",
      " 19  85.234    2.358    3.899 7o2g_HBA_dif_132_traj4_T0.3_0_5\n",
      "  0  63.408    5.065    5.876 7o2g_HBA_dif_57_traj3_T0.3_0_1\n",
      "  1  69.697    2.937    4.360 7o2g_HBA_dif_57_traj3_T0.3_0_2\n",
      "  2  68.767    3.281    5.752 7o2g_HBA_dif_57_traj3_T0.3_0_3\n",
      "  3  79.229    2.544    4.817 7o2g_HBA_dif_57_traj3_T0.3_0_4\n",
      "  4  70.770    6.645   10.732 7o2g_HBA_dif_57_traj3_T0.3_0_5\n",
      "  5  80.679    2.809    7.037 7o2g_HBA_dif_57_traj2_T0.3_0_1\n",
      "  6  73.790    3.066    4.963 7o2g_HBA_dif_57_traj2_T0.3_0_2\n",
      "  7  81.962    2.792    4.982 7o2g_HBA_dif_57_traj2_T0.3_0_3\n",
      "  8  61.069    6.625   14.344 7o2g_HBA_dif_57_traj2_T0.3_0_4\n",
      "  9  81.271    2.293    4.007 7o2g_HBA_dif_57_traj2_T0.3_0_5\n",
      " 10  55.073    9.320    8.738 7o2g_HBA_dif_57_traj1_T0.3_0_1\n",
      " 11  57.512    8.070    9.193 7o2g_HBA_dif_57_traj1_T0.3_0_2\n",
      " 12  77.070    4.172    8.097 7o2g_HBA_dif_57_traj1_T0.3_0_3\n",
      " 13  76.187    2.521    4.160 7o2g_HBA_dif_57_traj1_T0.3_0_4\n",
      " 14  78.758    1.847    2.975 7o2g_HBA_dif_57_traj1_T0.3_0_5\n",
      " 15  49.397   10.653    4.713 7o2g_HBA_dif_57_traj4_T0.3_0_1\n",
      " 16  66.755    2.794    4.343 7o2g_HBA_dif_57_traj4_T0.3_0_2\n",
      " 17  73.844    4.366    8.053 7o2g_HBA_dif_57_traj4_T0.3_0_3\n",
      " 18  69.909    3.065    4.640 7o2g_HBA_dif_57_traj4_T0.3_0_4\n",
      " 19  72.496    2.594    2.622 7o2g_HBA_dif_57_traj4_T0.3_0_5\n",
      "  0  47.480   13.450   16.206 7o2g_HBA_dif_222_traj2_T0.3_0_1\n",
      "  1  39.348   17.095   11.030 7o2g_HBA_dif_222_traj2_T0.3_0_2\n",
      "  2  47.235   16.041   14.172 7o2g_HBA_dif_222_traj2_T0.3_0_3\n",
      "  3  44.602   15.219   15.296 7o2g_HBA_dif_222_traj2_T0.3_0_4\n",
      "  4  44.005   13.841    5.518 7o2g_HBA_dif_222_traj2_T0.3_0_5\n",
      "  5  42.145   10.061    7.312 7o2g_HBA_dif_222_traj1_T0.3_0_1\n",
      "  6  50.705    8.265    5.838 7o2g_HBA_dif_222_traj1_T0.3_0_2\n",
      "  7  50.362   16.418   16.027 7o2g_HBA_dif_222_traj1_T0.3_0_3\n",
      "  8  49.451   15.830   20.902 7o2g_HBA_dif_222_traj1_T0.3_0_4\n",
      "  9  49.920    8.768    7.194 7o2g_HBA_dif_222_traj1_T0.3_0_5\n",
      " 10  45.724   17.945    4.442 7o2g_HBA_dif_222_traj4_T0.3_0_1\n",
      " 11  46.619   15.842   14.680 7o2g_HBA_dif_222_traj4_T0.3_0_2\n",
      " 12  45.362   16.664   16.779 7o2g_HBA_dif_222_traj4_T0.3_0_3\n",
      " 13  47.894   15.943   13.591 7o2g_HBA_dif_222_traj4_T0.3_0_4\n",
      " 14  51.643   11.378   13.691 7o2g_HBA_dif_222_traj4_T0.3_0_5\n",
      " 15  83.612    2.669    4.611 7o2g_HBA_dif_91_traj2_T0.3_0_1\n",
      " 16  50.166    6.159    2.528 7o2g_HBA_dif_91_traj2_T0.3_0_2\n",
      " 17  65.854   10.050    5.352 7o2g_HBA_dif_91_traj2_T0.3_0_3\n",
      " 18  54.986   10.425    8.341 7o2g_HBA_dif_91_traj2_T0.3_0_4\n",
      " 19  68.904   11.906    4.912 7o2g_HBA_dif_91_traj2_T0.3_0_5\n",
      " 20  58.385   12.155    9.734 7o2g_HBA_dif_222_traj3_T0.3_0_1\n",
      " 21  57.235   17.276   14.757 7o2g_HBA_dif_222_traj3_T0.3_0_2\n",
      " 22  51.205    9.642    6.832 7o2g_HBA_dif_222_traj3_T0.3_0_3\n",
      " 23  52.926   11.047   11.077 7o2g_HBA_dif_222_traj3_T0.3_0_4\n",
      " 24  59.252   13.298    7.547 7o2g_HBA_dif_222_traj3_T0.3_0_5\n",
      " 25  64.866    5.499   12.831 7o2g_HBA_dif_91_traj3_T0.3_0_1\n",
      " 26  50.499   15.482   16.473 7o2g_HBA_dif_91_traj3_T0.3_0_2\n",
      " 27  45.652   12.068    3.617 7o2g_HBA_dif_91_traj3_T0.3_0_3\n",
      " 28  53.419   12.231    5.846 7o2g_HBA_dif_91_traj3_T0.3_0_4\n",
      " 29  87.612    2.416    5.706 7o2g_HBA_dif_91_traj3_T0.3_0_5\n",
      " 30  48.160   17.800    4.969 7o2g_HBA_dif_91_traj4_T0.3_0_1\n",
      " 31  46.180   10.401    2.584 7o2g_HBA_dif_91_traj4_T0.3_0_2\n",
      " 32  46.099    9.115    8.011 7o2g_HBA_dif_91_traj4_T0.3_0_3\n",
      " 33  66.782   12.207    2.979 7o2g_HBA_dif_91_traj4_T0.3_0_4\n",
      " 34  51.525   14.706    5.627 7o2g_HBA_dif_91_traj4_T0.3_0_5\n",
      "  0  86.065    1.149    1.400 7o2g_HBA_dif_100_traj4_T0.3_0_1\n",
      "  1  78.504    1.182    2.225 7o2g_HBA_dif_100_traj4_T0.3_0_2\n",
      "  2  37.174   13.974   23.517 7o2g_HBA_dif_100_traj4_T0.3_0_3\n",
      "  3  80.875    3.336    3.327 7o2g_HBA_dif_100_traj4_T0.3_0_4\n",
      "  4  72.284    3.788    3.732 7o2g_HBA_dif_100_traj4_T0.3_0_5\n",
      "  5  77.479    2.756    7.486 7o2g_HBA_dif_100_T0.3_0_1\n",
      "  6  40.415   15.372   26.752 7o2g_HBA_dif_100_T0.3_0_2\n",
      "  7  72.536    2.920    1.566 7o2g_HBA_dif_100_T0.3_0_3\n",
      "  8  44.175   13.422   26.084 7o2g_HBA_dif_100_T0.3_0_4\n",
      "  9  73.845    3.816    2.604 7o2g_HBA_dif_100_T0.3_0_5\n",
      " 10  62.531    7.414    5.264 7o2g_HBA_dif_100_traj3_T0.3_0_1\n",
      " 11  87.808    1.067    2.381 7o2g_HBA_dif_100_traj3_T0.3_0_2\n",
      " 12  86.707    1.046    2.483 7o2g_HBA_dif_100_traj3_T0.3_0_3\n",
      " 13  77.549    2.739    2.261 7o2g_HBA_dif_100_traj3_T0.3_0_4\n",
      " 14  73.414    2.673    3.535 7o2g_HBA_dif_100_traj3_T0.3_0_5\n",
      " 15  41.375   10.699    6.060 7o2g_HBA_dif_100_traj1_T0.3_0_1\n",
      " 16  72.775    3.881    3.954 7o2g_HBA_dif_100_traj1_T0.3_0_2\n",
      " 17  72.590    2.965    0.665 7o2g_HBA_dif_100_traj1_T0.3_0_3\n",
      " 18  67.886    4.946    1.705 7o2g_HBA_dif_100_traj1_T0.3_0_4\n",
      " 19  75.786    1.906    2.349 7o2g_HBA_dif_100_traj1_T0.3_0_5\n",
      " 20  69.093    4.997    3.444 7o2g_HBA_dif_100_traj2_T0.3_0_1\n",
      " 21  59.821    3.272    7.691 7o2g_HBA_dif_100_traj2_T0.3_0_2\n",
      " 22  81.104    1.956    4.185 7o2g_HBA_dif_100_traj2_T0.3_0_3\n",
      " 23  85.447    2.525    1.582 7o2g_HBA_dif_100_traj2_T0.3_0_4\n",
      " 24  67.170   11.823    3.016 7o2g_HBA_dif_100_traj2_T0.3_0_5\n",
      "None\n",
      "None\n",
      "Done\n"
     ]
    }
   ],
   "source": [
    "# Combining all CSV scorefiles into one\n",
    "AF2_DIR = f\"{WDIR}/2_af2\"\n",
    "DIFFUSION_DIR = f\"{WDIR}/0_diffusion\"\n",
    "\n",
    "os.system(\"head -n 1 $(ls *aa*.csv | shuf -n 1) > scores.csv ; for f in *aa*.csv ; do tail -n +2 ${f} >> scores.csv ; done\")\n",
    "assert os.path.exists(\"scores.csv\"), \"Could not combine scorefiles\"\n",
    "\n",
    "### Calculating the RMSDs of AF2 predictions relative to the diffusion outputs\n",
    "### Catalytic residue sidechain RMSDs are calculated in the reference PDB has REMARK 666 line present\n",
    "\n",
    "analysis_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/utils/analyze_af2.py --scorefile scores.csv \"\\\n",
    "               f\"--ref_path {DIFFUSION_DIR}/filtered_structures/ --mpnn --params {' '.join(params)}\"\n",
    "\n",
    "\n",
    "### Clean up score.csv for repeating header rows\n",
    "file_path = 'scores.csv'\n",
    "lines = []\n",
    "\n",
    "with open(file_path, 'r') as file:\n",
    "    lines = file.readlines()\n",
    "\n",
    "firstLine = True\n",
    "with open(file_path, 'w') as file:\n",
    "    for line in lines:\n",
    "        if firstLine:\n",
    "            firstLine = False\n",
    "            file.write(line)\n",
    "        else:\n",
    "            if \"ID,Name,Sequence,Model/Tag,Output_PDB,lDDT,Time\" in line:\n",
    "                pass\n",
    "            else:\n",
    "                file.write(line)\n",
    "\n",
    "print(analysis_cmd)\n",
    "\n",
    "# Analyzing locally\n",
    "p = subprocess.Popen(analysis_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n",
    "print(output)\n",
    "print(err)\n",
    "print('Done')\n",
    "\n",
    "# if len(glob.glob(f\"{AF2_DIR}/*.pdb\")) < 100:\n",
    "#     ## Analyzing locally\n",
    "#     p = subprocess.Popen(analysis_cmd, shell=True)\n",
    "#     (output, err) = p.communicate()\n",
    "# else:\n",
    "#     ## Running as a Slurm job\n",
    "#     submit_script = \"submit_af2_analysis.sh\"\n",
    "#     utils.create_slurm_submit_script(filename=submit_script, name=\"af2_analysis\",\n",
    "#                                      mem=\"16g\", N_cores=8, time=\"0:20:00\", email=EMAIL,\n",
    "#                                      command=analysis_cmd, outfile_name=\"output_analysis\")\n",
    "\n",
    "#     p = subprocess.Popen([\"sbatch\", submit_script])\n",
    "#     (output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "45c18f2e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "lDDT                     >=  85.000: 64 items left. 64 pass (9%).\n",
      "rmsd                     <=   1.500: 18 items left. 28 pass (4%).\n",
      "rmsd_SR1                 <=   1.000: 1 items left. 7 pass (1%).\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABKUAAAEiCAYAAAAoMGGMAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjguMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy81sbWrAAAACXBIWXMAAA9hAAAPYQGoP6dpAABDUUlEQVR4nO3dfVxUZf7/8fcoOAIh3s9AImLhNxMy09LQElMxUsswb7vRLNfWmyQ1k2w37NtC2qpsWra6rqLG6m6paWaKpZhf1yLSMnPVViwq5ss3lwCFBm/O7w8fnl8j3oAOM9y8no/HeTyc61znzOc6M5zL+ZzrOsdiGIYhAAAAAAAAwIPqeTsAAAAAAAAA1D0kpQAAAAAAAOBxJKUAAAAAAADgcSSlAAAAAAAA4HEkpQAAAAAAAOBxJKUAAAAAAADgcSSlAAAAAAAA4HEkpQAAAAAAAOBxJKUAAAAAAADgcSSlgAssX75cFotFx44dkySNHj1aFovFXAICAtSmTRvdf//9WrZsmZxOZ7l9xMTEmPXr1aunwMBA3XjjjRoyZIjefvttnT171qyblJTksv9LLTExMR46AgCAmqxNmzYaPXq0t8MAALjZ1ZzfT548qdmzZ6tjx45q1KiRAgMDdcMNN2jo0KHKzMw06+3YscPlt0f9+vXVokULDRw4UJ999lm5/R44cEDjx4/XnXfeqYCAAFksFu3YseMaW4i6yMfbAQA1gZ+fnz766CNJUmlpqXJzc7V582aNHTtWc+fO1QcffKBWrVq5bNO2bVu99dZbks51Bjk5OVq/fr2GDBmiu+66Sxs3blRQUJCefPJJ3XvvveZ2eXl5io+P16RJkzRy5EizvFGjRh5oKQAAAIDa4MyZM4qNjdX+/fv17LPP6o477pAkHTlyRBs3btTHH3+snj17umyTnJysXr166dSpU9q7d69mzZqlnj17at++fYqIiDDrffbZZ1q/fr06deqk3r17a+PGjR5tG2oPklJABdSrV0/dunVzKXvsscf0+OOPa8CAAXrooYe0Z88el/V+fn7ltnnyySe1bNkyjRkzRr/5zW+0Zs0atWrVyiWhdX6EVuvWrcttDwCo/kpKSuTv7+/tMAAAblbTzu87d+7U7t279de//lWPP/64Wd6vXz9NnDjRZfbGeREREeZvkLvuukuNGzfWqFGjtGrVKs2aNcus9+ijj2rUqFGSpLfffpukFK4a0/eAaxAbG6uxY8fqk08+0c6dOyu0zeOPP6777rtP//jHP/Ttt99WcYQAgKp0fgr2559/roceekhNmjTRDTfcoDZt2mjAgAF677331KlTJ/n5+al9+/Z67733JJ2bKt6+fXsFBATojjvuKDc14ujRoxo+fLhCQkJktVpls9nUu3dv7du3z6xz6tQpTZ8+XXa7Xf7+/urRo4c+/fRTTzYfAGqt2nB+P378uCQpODj4ouvr1btyOqBLly6SpP/93/+t9LZARfBNAq7R/fffL0kVTkqd38YwDH388cdVFRYAwIPi4+N144036h//+IfefPNNSdIXX3yhxMREPffcc1q7dq2CgoIUHx+vF198UX/5y1+UnJyst956S4WFhRowYIBKS0vN/d13333Kzs7WnDlzlJGRoUWLFqlTp076+eefzTpjx47VH//4Rz322GN69913NXjwYMXHx6ugoMDTzQeAWqsmn9+7dOkiX19fTZ48WW+99Zby8vIq3f6cnBxJUrt27Sq9LVARTN8DrlFYWJgk6ccff6zSbQAA1deoUaNcpjVMnjxZx48f1549e3T99ddLkkJCQnTrrbdqyZIl+uabb8wpIBaLRYMGDdK2bds0cOBAHT9+XIcOHVJqaqoeeeQRc5/x8fHmv//1r38pLS1NzzzzjObMmSNJ6tu3r2w2mx5++GFPNBkA6oSafH5v06aN3nzzTU2ePNl8v+DgYPXt21dPPvmk7rrrrnLbnD17VqdPnzbvKTV16lTdfPPNGjNmTKXeG6goRkoB18gwDI9sAwCovgYPHlyu7NZbbzV/sEhS+/btJZ17Quuv70lyvvz8lO6mTZvqhhtu0Kuvvqp58+Zp79695e77sX37dkkq9wNl6NCh8vHhmiMAuEtNP7+PGTNG33//vdLT0/X0008rNDRUq1atUs+ePfXqq6+Wqz9s2DD5+vrK399f3bt3V1FRkTZt2qTGjRtX+r2BiiApBVyj851MSEhIlW4DAKi+Lna/jqZNm7q8btCgwWXLf/nlF0nnrqx/+OGH6tevn+bMmaPbbrtNLVq00NNPP63i4mJJ//8+IXa73WVfPj4+atasmRtaBACQasf5PSgoSCNGjNCf/vQnffLJJ/ryyy9ls9k0c+ZMl2mDkjR79mxlZWUpMzNTM2fO1P/+7/9q0KBBcjqdV/XewJWQlAKu0YYNGySduzJSmW0sFovuvvvuKooKAOBJFovFrfsLCwvT0qVL5XA4dOjQIT3zzDN644039Oyzz0qS+cPE4XC4bHf69GnzBw0A4NrVxvN7hw4dNHz4cJ06dUqHDx92Wde2bVt16dJFd999t15++WW99NJL+uKLL7RgwQK3vDdwIZJSwDXIyMjQX/7yF0VHR6tHjx4V2mbZsmXavHmzRowYodatW1dxhACAmq5du3Z64YUXFBUVpc8//1zS/78Q8tZbb7nU/fvf/67Tp097OkQAwFWo6vP78ePHVVZWdtF1//rXvyRdeebG9OnTdeONN+qVV14xR3MB7sRNB4AKOHv2rPbs2SNJcjqd+u6777R582b9/e9/V/v27fX3v/+93DalpaXmNqWlpTp69KjWr1+v9957Tz179jSf3gEAwK99+eWXmjhxooYMGaKIiAg1aNBAH330kb788kvNmDFD0rn7lDzyyCNKTU2Vr6+v+vTpo6+++kp//OMf1ahRIy+3AABwMZ4+v2/fvl2TJ0/Www8/rOjoaDVr1kz5+fn629/+pg8++ECPPfaYWrVqddl9+Pr6Kjk5WUOHDtWf/vQnvfDCC5KkkpISvf/++5Jk/ubJzMzUTz/9pICAAMXFxVX28KCOIikFVEBpaanuvPNOSZKfn59atGihjh07asmSJXr44YfN+eK/dvToUXObgIAA2Ww23XbbbfrHP/6h+Ph41avHQEUAQHl2u1033HCD3njjDeXm5spisaht27aaO3euJk2aZNZbunSpbDabli9frtdee0233nqr3nnnHQ0fPtyL0QMALsXT5/du3bppzJgx2r59u1auXKmffvpJfn5+uvnmm7VgwQL99re/rdB+hgwZoq5du2revHmaNGmSgoKClJ+fryFDhrjUS0pKknRuiuKxY8cqFSvqLovBY8AAAAAAAADgYQzVAAAAAAAAgMcxfQ8AAAAAgBrkSjc9r1evHrcLQY3AtxQAAAAAgBrE19f3ssuYMWO8HSJQIYyUAgAAAACgBsnKyrrs+ubNm3soEuDacKNzAAAAAAAAeBzT9wAAAAAAAOBxNXL63tmzZ/Xjjz8qMDBQFovF2+EAQI1lGIaKi4sVEhJSq2+GSb8BANeOPgMAUFEV7TNqZFLqxx9/VGhoqLfDAIBaIzc3V61atfJ2GFWGfgMA3Ic+AwBQUVfqM2pkUiowMFDSucY1atTIy9EAQM1VVFSk0NBQ87xaW9FvAMC1o88AAFRURfuMGpmUOj+MtlGjRnQUAOAGtX16Av0GALgPfQYAoKKu1GfU3sngAAAAAAAAqLZISgEAAAAAAMDjSEoBAAAAAADA40hKAQAAAPCqlJQU3X777QoMDFTLli01aNAgHTp0yKXO6NGjZbFYXJZu3bq51HE6nZo0aZKaN2+ugIAA3X///fr+++892RQAQCWQlAIAAADgVZmZmZowYYL27NmjjIwMnT59WrGxsTp58qRLvXvvvVd5eXnm8v7777usT0hI0Lp167R69Wrt2rVLJ06c0IABA3TmzBlPNgcAUEE18ul7AAAAAGqPDz74wOX1smXL1LJlS2VnZ+vuu+82y61Wq+x2+0X3UVhYqKVLl2rlypXq06ePJGnVqlUKDQ3Vtm3b1K9fv6prAADgqjBSCgAAAEC1UlhYKElq2rSpS/mOHTvUsmVLtWvXTmPHjlV+fr65Ljs7W6dOnVJsbKxZFhISosjISO3evfuS7+V0OlVUVOSyAAA8g5FSANyuzYxNXnnfY6/098r7AlXJW39PEn9TALzDMAxNmTJFPXr0UGRkpFkeFxenIUOGKCwsTDk5Ofrd736ne+65R9nZ2bJarXI4HGrQoIGaNGnisj+bzSaHw3HJ90tJSdGsWbOqrD2eRJ8BoKYhKQUAAACg2pg4caK+/PJL7dq1y6V82LBh5r8jIyPVpUsXhYWFadOmTYqPj7/k/gzDkMViueT6xMRETZkyxXxdVFSk0NDQa2gBAKCimL4HAAAAoFqYNGmSNmzYoO3bt6tVq1aXrRscHKywsDAdOXJEkmS321VWVqaCggKXevn5+bLZbJfcj9VqVaNGjVwWAIBnkJQCAAAA4FWGYWjixIlau3atPvroI4WHh19xm+PHjys3N1fBwcGSpM6dO8vX11cZGRlmnby8PH311VeKjo6ustgBAFeP6XsAAAAAvGrChAlKT0/Xu+++q8DAQPMeUEFBQfLz89OJEyeUlJSkwYMHKzg4WMeOHdPzzz+v5s2b68EHHzTrPvHEE5o6daqaNWumpk2batq0aYqKijKfxgcAqF5ISgEAAADwqkWLFkmSYmJiXMqXLVum0aNHq379+tq/f79WrFihn3/+WcHBwerVq5fWrFmjwMBAs/78+fPl4+OjoUOHqrS0VL1799by5ctVv359TzYHAFBBJKUAAAAAeJVhGJdd7+fnpy1btlxxPw0bNtSCBQu0YMECd4UGAKhC3FMKAAAAAAAAHkdSCgAAAAAAAB5HUgoAAAAAAAAeR1IKAAAAAAAAHkdSCgAAAAAAAB5HUgoAAAAAAAAe5+PtAABUjTYzNnk7BAAAAAAALomRUgAAAAAAAPC4Sieldu7cqYEDByokJEQWi0Xr1693WW8YhpKSkhQSEiI/Pz/FxMTowIEDLnWcTqcmTZqk5s2bKyAgQPfff7++//77a2oIAAAAAAAAao5KJ6VOnjypjh07auHChRddP2fOHM2bN08LFy5UVlaW7Ha7+vbtq+LiYrNOQkKC1q1bp9WrV2vXrl06ceKEBgwYoDNnzlx9SwAAAAAAAFBjVPqeUnFxcYqLi7voOsMwlJqaqpkzZyo+Pl6SlJaWJpvNpvT0dI0bN06FhYVaunSpVq5cqT59+kiSVq1apdDQUG3btk39+vW7huYAAAAAAACgJnDrPaVycnLkcDgUGxtrllmtVvXs2VO7d++WJGVnZ+vUqVMudUJCQhQZGWnWAQDUDlea8j169GhZLBaXpVu3bi51mPINAAAA1E5uTUo5HA5Jks1mcym32WzmOofDoQYNGqhJkyaXrHMhp9OpoqIilwUAUP1dacq3JN17773Ky8szl/fff99lPVO+AQAAgNqp0tP3KsJisbi8NgyjXNmFLlcnJSVFs2bNclt8AADPuNyU7/OsVqvsdvtF1zHlGwAAAKi93DpS6vyPigtHPOXn55ujp+x2u8rKylRQUHDJOhdKTExUYWGhueTm5rozbACAF+3YsUMtW7ZUu3btNHbsWOXn55vrrnbKNyNsAQAAgOrPrUmp8PBw2e12ZWRkmGVlZWXKzMxUdHS0JKlz587y9fV1qZOXl6evvvrKrHMhq9WqRo0auSwAgJovLi5Ob731lj766CPNnTtXWVlZuueee+R0OiVd3ZRv6dwI26CgIHMJDQ2t0nYAAAAAqLxKT987ceKEvvnmG/N1Tk6O9u3bp6ZNm6p169ZKSEhQcnKyIiIiFBERoeTkZPn7+2vkyJGSpKCgID3xxBOaOnWqmjVrpqZNm2ratGmKiooyp2YAAOqGYcOGmf+OjIxUly5dFBYWpk2bNplPcb2YK00LT0xM1JQpU8zXRUVFJKYAAACAaqbSSanPPvtMvXr1Ml+f/0//qFGjtHz5ck2fPl2lpaUaP368CgoK1LVrV23dulWBgYHmNvPnz5ePj4+GDh2q0tJS9e7dW8uXL1f9+vXd0CQAQE0VHByssLAwHTlyRJLrlO9fj5bKz8+/5Oha6dwIW6vVWuXxAgAAALh6lZ6+FxMTI8Mwyi3Lly+XdO4m50lJScrLy9Mvv/yizMxMRUZGuuyjYcOGWrBggY4fP66SkhJt3LiRK9gAAB0/fly5ubkKDg6WdHVTvgEAAADUDFXy9D0AAKTLT/lu2rSpkpKSNHjwYAUHB+vYsWN6/vnn1bx5cz344IOSmPINAKh52szY5O0QAKDGICkFAKgyl5vyvWjRIu3fv18rVqzQzz//rODgYPXq1Utr1qxhyjcAAABQB5CUAgBUmfNTvi9ly5YtV9zH+SnfCxYscGdoAAAAALyMpBQAAFfAVAwAAADA/Sp9o3MAAAAAAADgWpGUAgAAAAAAgMeRlAIAAAAAAIDHkZQCAAAAAACAx5GUAgAAAAAAgMeRlAIAAADgVSkpKbr99tsVGBioli1batCgQTp06JBLHcMwlJSUpJCQEPn5+SkmJkYHDhxwqeN0OjVp0iQ1b95cAQEBuv/++/X99997sikAgEogKQUAAADAqzIzMzVhwgTt2bNHGRkZOn36tGJjY3Xy5Emzzpw5czRv3jwtXLhQWVlZstvt6tu3r4qLi806CQkJWrdunVavXq1du3bpxIkTGjBggM6cOeONZgEArsDH2wEAVa3NjE1ee+9jr/T32nsDAADUFB988IHL62XLlqlly5bKzs7W3XffLcMwlJqaqpkzZyo+Pl6SlJaWJpvNpvT0dI0bN06FhYVaunSpVq5cqT59+kiSVq1apdDQUG3btk39+vXzeLsAAJfHSCkAAAAA1UphYaEkqWnTppKknJwcORwOxcbGmnWsVqt69uyp3bt3S5Kys7N16tQplzohISGKjIw061yM0+lUUVGRywIA8AxGSgFVyJujtAAAAGoiwzA0ZcoU9ejRQ5GRkZIkh8MhSbLZbC51bTabvv32W7NOgwYN1KRJk3J1zm9/MSkpKZo1a5Y7mwAAqCBGSgEAAACoNiZOnKgvv/xSf/vb38qts1gsLq8NwyhXdqEr1UlMTFRhYaG55ObmXl3gAIBKIykFAAAAoFqYNGmSNmzYoO3bt6tVq1Zmud1ul6RyI57y8/PN0VN2u11lZWUqKCi4ZJ2LsVqtatSokcsCAPAMklIAAAAAvMowDE2cOFFr167VRx99pPDwcJf14eHhstvtysjIMMvKysqUmZmp6OhoSVLnzp3l6+vrUicvL09fffWVWQcAUL1wTykAAAAAXjVhwgSlp6fr3XffVWBgoDkiKigoSH5+frJYLEpISFBycrIiIiIUERGh5ORk+fv7a+TIkWbdJ554QlOnTlWzZs3UtGlTTZs2TVFRUebT+AAA1QtJKQAAAABetWjRIklSTEyMS/myZcs0evRoSdL06dNVWlqq8ePHq6CgQF27dtXWrVsVGBho1p8/f758fHw0dOhQlZaWqnfv3lq+fLnq16/vqaYAACqBpBQAAAAArzIM44p1LBaLkpKSlJSUdMk6DRs21IIFC7RgwQI3RgcAqCokpQDUGm1mbPLaex97pb/X3hsAAAAAaiJudA4AAAAAAACPIykFAAAAAAAAjyMpBQAAAAAAAI8jKQUAAAAAAACPc3tS6vTp03rhhRcUHh4uPz8/tW3bVi+99JLOnj1r1jEMQ0lJSQoJCZGfn59iYmJ04MABd4cCAAAAAACAasrtSanZs2frzTff1MKFC3Xw4EHNmTNHr776qstjWefMmaN58+Zp4cKFysrKkt1uV9++fVVcXOzucAAAAAAAAFANuT0p9c9//lMPPPCA+vfvrzZt2uihhx5SbGysPvvsM0nnRkmlpqZq5syZio+PV2RkpNLS0lRSUqL09HR3hwMAAAAAAIBqyO1JqR49eujDDz/U4cOHJUlffPGFdu3apfvuu0+SlJOTI4fDodjYWHMbq9Wqnj17avfu3Rfdp9PpVFFRkcsCAAAAAACAmsvH3Tt87rnnVFhYqJtuukn169fXmTNn9Ic//EEjRoyQJDkcDkmSzWZz2c5ms+nbb7+96D5TUlI0a9Ysd4cKAAAAAAAAL3H7SKk1a9Zo1apVSk9P1+eff660tDT98Y9/VFpamks9i8Xi8towjHJl5yUmJqqwsNBccnNz3R02AAAAAAAAPMjtI6WeffZZzZgxQ8OHD5ckRUVF6dtvv1VKSopGjRolu90u6dyIqeDgYHO7/Pz8cqOnzrNarbJare4OFQAAAAAAAF7i9pFSJSUlqlfPdbf169fX2bNnJUnh4eGy2+3KyMgw15eVlSkzM1PR0dHuDgcAAAAAAADVkNtHSg0cOFB/+MMf1Lp1a3Xo0EF79+7VvHnzNGbMGEnnpu0lJCQoOTlZERERioiIUHJysvz9/TVy5Eh3hwMAAAAAAIBqyO1JqQULFuh3v/udxo8fr/z8fIWEhGjcuHH6/e9/b9aZPn26SktLNX78eBUUFKhr167aunWrAgMD3R0OAAAAAAAAqiG3J6UCAwOVmpqq1NTUS9axWCxKSkpSUlKSu98eAAAAAAAANYDb7ykFAAAAAAAAXAlJKQAAAAAAAHgcSSkAAAAAAAB4HEkpAAAAAAAAeBxJKQAAAAAAAHgcSSkAAAAAAAB4HEkpAAAAAAAAeBxJKQAAAAAAAHgcSSkAAAAAAAB4HEkpAAAAAAAAeBxJKQAAAAAAAHgcSSkAAAAAAAB4HEkpAAAAAF63c+dODRw4UCEhIbJYLFq/fr3L+tGjR8tisbgs3bp1c6njdDo1adIkNW/eXAEBAbr//vv1/fffe7AVAIDKICkFAAAAwOtOnjypjh07auHChZesc++99yovL89c3n//fZf1CQkJWrdunVavXq1du3bpxIkTGjBggM6cOVPV4QMAroKPtwNA3dBmxiZvhwDAC3bu3KlXX31V2dnZysvL07p16zRo0CBzvWEYmjVrlhYvXqyCggJ17dpVr7/+ujp06GDWcTqdmjZtmv72t7+ptLRUvXv31htvvKFWrVp5oUUAgKoSFxenuLi4y9axWq2y2+0XXVdYWKilS5dq5cqV6tOnjyRp1apVCg0N1bZt29SvXz+3xwwAuDaMlAIAVJkrXfWeM2eO5s2bp4ULFyorK0t2u119+/ZVcXGxWYer3gCA83bs2KGWLVuqXbt2Gjt2rPLz88112dnZOnXqlGJjY82ykJAQRUZGavfu3d4IFwBwBYyUAgBUmctd9TYMQ6mpqZo5c6bi4+MlSWlpabLZbEpPT9e4ceO46g0AMMXFxWnIkCEKCwtTTk6Ofve73+mee+5Rdna2rFarHA6HGjRooCZNmrhsZ7PZ5HA4Lrlfp9Mpp9Npvi4qKqqyNgAAXDFSCgDgFTk5OXI4HC5XtK1Wq3r27Gle0b7aq95Op1NFRUUuCwCgZhs2bJj69++vyMhIDRw4UJs3b9bhw4e1adPlbxNhGIYsFssl16ekpCgoKMhcQkND3R06AOASGClVh3BfJwDVyfmr1jabzaXcZrPp22+/NetczVXvlJQUzZo1y80RAwCqk+DgYIWFhenIkSOSJLvdrrKyMhUUFLj0G/n5+YqOjr7kfhITEzVlyhTzdVFREYkpAPAQRkoBALzqwqvXV7qiXZE6iYmJKiwsNJfc3Fy3xAoAqD6OHz+u3NxcBQcHS5I6d+4sX19fZWRkmHXy8vL01VdfXTYpZbVa1ahRI5cFAOAZjJQCAHjF+acnORwO8weFdO6K9vnRU1d71dtqtcpqtVZR5ACAqnDixAl988035uucnBzt27dPTZs2VdOmTZWUlKTBgwcrODhYx44d0/PPP6/mzZvrwQcflCQFBQXpiSee0NSpU9WsWTM1bdpU06ZNU1RUlHlfQgBA9cJIKQCAV4SHh8tut7tc0S4rK1NmZqaZcLraq94AgJrns88+U6dOndSpUydJ0pQpU9SpUyf9/ve/V/369bV//3498MADateunUaNGqV27drpn//8pwIDA819zJ8/X4MGDdLQoUPVvXt3+fv7a+PGjapfv763mgUAuAxGSgEAqszlrnq3bt1aCQkJSk5OVkREhCIiIpScnCx/f3+NHDlSEle9AaAuiYmJkWEYl1y/ZcuWK+6jYcOGWrBggRYsWODO0AAAVYSkFACgynz22Wfq1auX+fr8jWRHjRql5cuXa/r06SotLdX48eNVUFCgrl27auvWreWuevv4+Gjo0KEqLS1V7969tXz5cq56AwAAADVclUzf++GHH/TII4+oWbNm8vf316233qrs7GxzvWEYSkpKUkhIiPz8/BQTE6MDBw5URSgAAC86f9X7wmX58uWSzt3kPCkpSXl5efrll1+UmZmpyMhIl32cv+p9/PhxlZSUaOPGjTwVCQAAAKgF3J6UKigoUPfu3eXr66vNmzfr66+/1ty5c9W4cWOzzpw5czRv3jwtXLhQWVlZstvt6tu3r4qLi90dDgAAAAAAAKoht0/fmz17tkJDQ7Vs2TKzrE2bNua/DcNQamqqZs6cqfj4eElSWlqabDab0tPTNW7cOHeHBAAAAAAAgGrG7SOlNmzYoC5dumjIkCFq2bKlOnXqpCVLlpjrc3Jy5HA4FBsba5ZZrVb17NlTu3fvvug+nU6nioqKXBYAAAAAAADUXG5PSh09elSLFi1SRESEtmzZoqeeekpPP/20VqxYIUlyOBySJJvN5rKdzWYz110oJSVFQUFB5sK9RAAAAAAAAGo2tyelzp49q9tuu03Jycnq1KmTxo0bp7Fjx2rRokUu9SwWi8trwzDKlZ2XmJiowsJCc8nNzXV32AAAAAAAAPAgtyelgoODdfPNN7uUtW/fXt99950kyW63S1K5UVH5+fnlRk+dZ7Va1ahRI5cFAAAAAAAANZfbk1Ldu3fXoUOHXMoOHz6ssLAwSVJ4eLjsdrsyMjLM9WVlZcrMzFR0dLS7wwEAAAAAAEA15Pan7z3zzDOKjo5WcnKyhg4dqk8//VSLFy/W4sWLJZ2btpeQkKDk5GRFREQoIiJCycnJ8vf318iRI90dDgAAAAAAAKohtyelbr/9dq1bt06JiYl66aWXFB4ertTUVD388MNmnenTp6u0tFTjx49XQUGBunbtqq1btyowMNDd4QAAAAAAAKAacntSSpIGDBigAQMGXHK9xWJRUlKSkpKSquLtAQAAAAAAUM25/Z5SAAAAAAAAwJWQlAIAAAAAAIDHkZQCAAAAAACAx5GUAgAAAAAAgMeRlAIAAAAAAIDHkZQCAAAAAACAx5GUAgAAAAAAgMeRlAIAAAAAAIDHkZQCAAAAAACAx5GUAgAAAAAAgMf5eDsAAACA6qLNjE1ee+9jr/T32nsDAAB4AyOlAAAAAAAA4HEkpQAAAAAAAOBxTN8DADdgyg8AAAAAVA4jpQAAAAAAAOBxJKUAAAAAAADgcUzfAwAAqAaYBgygJuMcBuBqMFIKAAAAgNft3LlTAwcOVEhIiCwWi9avX++y3jAMJSUlKSQkRH5+foqJidGBAwdc6jidTk2aNEnNmzdXQECA7r//fn3//fcebAUAoDJISgEAAADwupMnT6pjx45auHDhRdfPmTNH8+bN08KFC5WVlSW73a6+ffuquLjYrJOQkKB169Zp9erV2rVrl06cOKEBAwbozJkznmoGAKASmL4HAAAAwOvi4uIUFxd30XWGYSg1NVUzZ85UfHy8JCktLU02m03p6ekaN26cCgsLtXTpUq1cuVJ9+vSRJK1atUqhoaHatm2b+vXr57G2AAAqhpFSAAAAAKq1nJwcORwOxcbGmmVWq1U9e/bU7t27JUnZ2dk6deqUS52QkBBFRkaadS7G6XSqqKjIZQEAeAZJKQAAAADVmsPhkCTZbDaXcpvNZq5zOBxq0KCBmjRpcsk6F5OSkqKgoCBzCQ0NdXP0AIBLISkFAAAAoEawWCwurw3DKFd2oSvVSUxMVGFhobnk5ua6JVYAwJVxTykAAHBRPN4bQHVht9slnRsNFRwcbJbn5+ebo6fsdrvKyspUUFDgMloqPz9f0dHRl9y31WqV1WqtosgBAJdT5SOlUlJSZLFYlJCQYJZV5HGuAAAAACBJ4eHhstvtysjIMMvKysqUmZlpJpw6d+4sX19flzp5eXn66quvLpuUAgB4T5WOlMrKytLixYt1yy23uJSff5zr8uXL1a5dO7388svq27evDh06pMDAwKoMCQAAAEA1dOLECX3zzTfm65ycHO3bt09NmzZV69atlZCQoOTkZEVERCgiIkLJycny9/fXyJEjJUlBQUF64oknNHXqVDVr1kxNmzbVtGnTFBUVZT6NDwBQvVTZSKkTJ07o4Ycf1pIlS1yGz174ONfIyEilpaWppKRE6enpVRUOAAAAgGrss88+U6dOndSpUydJ0pQpU9SpUyf9/ve/lyRNnz5dCQkJGj9+vLp06aIffvhBW7dudbmoPX/+fA0aNEhDhw5V9+7d5e/vr40bN6p+/fpeaRMA4PKqLCk1YcIE9e/fv9xViYo8zhUAAABA3RITEyPDMMoty5cvl3TuJudJSUnKy8vTL7/8oszMTEVGRrrso2HDhlqwYIGOHz+ukpISbdy4kafpAUA1ViXT91avXq3PP/9cWVlZ5dZd7nGu33777UX353Q65XQ6zddFRUVujBYAAAAAAACe5vaRUrm5uZo8ebJWrVqlhg0bXrJeZR7nmpKSoqCgIHPhagcAAAAAAEDN5vakVHZ2tvLz89W5c2f5+PjIx8dHmZmZeu211+Tj42OOkDo/Yuq8Xz/O9UKJiYkqLCw0l9zcXHeHDQAAAAAAAA9y+/S93r17a//+/S5ljz/+uG666SY999xzatu2rfk41/M3MTz/ONfZs2dfdJ9Wq1VWq9XdoQIAAAAAAMBL3J6UCgwMLHfDwYCAADVr1swsv9LjXAEAAAAAAFC7VcmNzq9k+vTpKi0t1fjx41VQUKCuXbuWe5wrAAAAAAAAai+PJKV27Njh8vr841yTkpI88fbVTpsZm7wdAgAAAAAAgFe5/UbnAAAAAAAAwJWQlAIAAAAAAIDHeeWeUgAASFJSUpJmzZrlUmaz2eRwOCRJhmFo1qxZWrx4sXkPwtdff10dOnTwRrhAreXNWwsce6W/194bAAB4FyOlAABe1aFDB+Xl5ZnL/v37zXVz5szRvHnztHDhQmVlZclut6tv374qLi72YsQAAAAA3IGkFADAq3x8fGS3282lRYsWks6NkkpNTdXMmTMVHx+vyMhIpaWlqaSkROnp6V6OGgAAAMC1IikFAPCqI0eOKCQkROHh4Ro+fLiOHj0qScrJyZHD4VBsbKxZ12q1qmfPntq9e/dl9+l0OlVUVOSyAAAAAKheSEoBALyma9euWrFihbZs2aIlS5bI4XAoOjpax48fN+8rZbPZXLb59T2nLiUlJUVBQUHmEhoaWmVtAAAAAHB1SEoBALwmLi5OgwcPVlRUlPr06aNNm87dbDktLc2sY7FYXLYxDKNc2YUSExNVWFhoLrm5ue4PHgAAAMA14el7AIBqIyAgQFFRUTpy5IgGDRokSXI4HAoODjbr5Ofnlxs9dSGr1Sqr1VqVoaKKefNpcAAAAPAMRkoBAKoNp9OpgwcPKjg4WOHh4bLb7crIyDDXl5WVKTMzU9HR0V6MEgAAAIA7MFIKAOA106ZN08CBA9W6dWvl5+fr5ZdfVlFRkUaNGiWLxaKEhAQlJycrIiJCERERSk5Olr+/v0aOHOnt0AEAAABcI5JSAACv+f777zVixAj99NNPatGihbp166Y9e/YoLCxMkjR9+nSVlpZq/PjxKigoUNeuXbV161YFBgZ6OXIAAAAA14qkFADAa1avXn3Z9RaLRUlJSUpKSvJMQAAAAAA8hntKAQAAAAAAwONISgEAAAAAAMDjSEoBAAAAAADA47inFAAAALymzYxNXnvvY6/099p7AwAARkoBAAAAAADAC0hKAQAAAAAAwONISgEAAAAAAMDjSEoBAAAAAADA47jROQAAAIBqLykpSbNmzXIps9lscjgckiTDMDRr1iwtXrxYBQUF6tq1q15//XV16NDBG+HCg3hgAlBzMVIKAAAAQI3QoUMH5eXlmcv+/fvNdXPmzNG8efO0cOFCZWVlyW63q2/fviouLvZixACAyyEpBQAAAKBG8PHxkd1uN5cWLVpIOjdKKjU1VTNnzlR8fLwiIyOVlpamkpISpaenezlqAMClkJQCAAAAUCMcOXJEISEhCg8P1/Dhw3X06FFJUk5OjhwOh2JjY826VqtVPXv21O7duy+7T6fTqaKiIpcFAOAZbk9KpaSk6Pbbb1dgYKBatmypQYMG6dChQy51DMNQUlKSQkJC5Ofnp5iYGB04cMDdoQAAAACoJbp27aoVK1Zoy5YtWrJkiRwOh6Kjo3X8+HHzvlI2m81lm1/fc+pSUlJSFBQUZC6hoaFV1gYAgCu3J6UyMzM1YcIE7dmzRxkZGTp9+rRiY2N18uRJsw7zvQEAAABURlxcnAYPHqyoqCj16dNHmzadu7l1WlqaWcdisbhsYxhGubILJSYmqrCw0Fxyc3PdHzwA4KLc/vS9Dz74wOX1smXL1LJlS2VnZ+vuu+8uN99bOteR2Gw2paena9y4ce4OCQAAAEAtExAQoKioKB05ckSDBg2SJDkcDgUHB5t18vPzy42eupDVapXVaq3KUAEAl1Dl95QqLCyUJDVt2lTS1c33Zp43AAAAgF9zOp06ePCggoODFR4eLrvdroyMDHN9WVmZMjMzFR0d7cUoAQCXU6VJKcMwNGXKFPXo0UORkZGSdFXzvZnnDQAAANRt06ZNU2ZmpnJycvTJJ5/ooYceUlFRkUaNGiWLxaKEhAQlJydr3bp1+uqrrzR69Gj5+/tr5MiR3g4dAHAJbp++92sTJ07Ul19+qV27dpVbV5n53omJiZoyZYr5uqioiMQUAAAAUId8//33GjFihH766Se1aNFC3bp10549exQWFiZJmj59ukpLSzV+/HgVFBSoa9eu2rp1qwIDA70cOQDgUqosKTVp0iRt2LBBO3fuVKtWrcxyu90uqXLzvZnnDQAAANRtq1evvux6i8WipKQkJSUleSYgAMA1c/v0PcMwNHHiRK1du1YfffSRwsPDXdYz3xsAAAAAAABuHyk1YcIEpaen691331VgYKB5n6igoCD5+fm5zPeOiIhQRESEkpOTme8NAAAAAABQh7g9KbVo0SJJUkxMjEv5smXLNHr0aEnM9wYAAAAAAKjr3J6UMgzjinWY7w0AAAAAAFC3VenT9wAAAIDqqs2MTV5772Ov9PfaewMAUF24/UbnAAAAAAAAwJWQlAIAAAAAAIDHMX0PAAAAAICr4K1pwEwBRm3BSCkAAAAAAAB4HEkpAAAAAAAAeBxJKQAAAAAAAHgcSSkAAAAAAAB4HEkpAAAAAAAAeBxJKQAAAAAAAHgcSSkAAAAAAAB4HEkpAAAAAAAAeJyPtwMAAAAAAAAV12bGJq+997FX+nvtvVH7MFIKAAAAAAAAHkdSCgAAAAAAAB5HUgoAAAAAAAAeR1IKAAAAAAAAHkdSCgAAAAAAAB5XZ5++582nFQAAAAAAUBPx5D+4EyOlAAAAAAAA4HF1dqQUAAAA4C3eGmnAKAMAQHXCSCkAAAAAAAB4HEkpAAAAAAAAeBxJKQAAAAAAAHicV5NSb7zxhsLDw9WwYUN17txZH3/8sTfDAQBUY/QZAICKos8AgJrBazc6X7NmjRISEvTGG2+oe/fu+vOf/6y4uDh9/fXXat26tbfCAgBUQ/QZAICKos8Aai9vPSRC4kERVcVrI6XmzZunJ554Qk8++aTat2+v1NRUhYaGatGiRd4KCQBQTdFnAAAqij4DAGoOr4yUKisrU3Z2tmbMmOFSHhsbq927d3sjJABANUWfAQCoKPoMAFXFm6O0vKmqR4h5JSn1008/6cyZM7LZbC7lNptNDoejXH2n0ymn02m+LiwslCQVFRVddQxnnSVXvS0AVCfXci48v61hGO4Kx+0q22dI7u836DMA1Bb0GeXRZwDApV3tubCifYbX7iklSRaLxeW1YRjlyiQpJSVFs2bNKlceGhpaZbEBQE0RlHrt+yguLlZQUNC176gKVbTPkOg3AOBS6DPKo88AgEu71n7jSn2GV5JSzZs3V/369ctdrcjPzy93VUOSEhMTNWXKFPP12bNn9Z///EfNmjW7ZOdSlYqKihQaGqrc3Fw1atTI4+/vCbSxdqCNtUNVttEwDBUXFyskJMSt+3WnyvYZ0qX7DV9fX7Vu3bpWf188qS78/XkKx9K9OJ7u8+tjGRgYWKf6jIv91qhL36261FapbrWXttZO1a2tFf2d4ZWkVIMGDdS5c2dlZGTowQcfNMszMjL0wAMPlKtvtVpltVpdyho3blzVYV5Ro0aNqsWHXZVoY+1AG2uHqmpjdb/aXdk+Q7p0v3F+GHFd+L54EsfTfTiW7sXxdJ/zx7Iu9RmXU5e+W3WprVLdai9trZ2qU1sr0md4bfrelClT9Oijj6pLly668847tXjxYn333Xd66qmnvBUSAKCaos8AAFQUfQYA1BxeS0oNGzZMx48f10svvaS8vDxFRkbq/fffV1hYmLdCAgBUU/QZAICKos8AgJrDqzc6Hz9+vMaPH+/NEK6K1WrViy++WG6Yb21CG2sH2lg71IU2VoQ7+gyOpXtxPN2HY+leHE/3qanHsqp+Z9TU43E16lJbpbrVXtpaO9XUtlqM6vxMVwAAAAAAANRK9bwdAAAAAAAAAOoeklIAAAAAAADwOJJSAAAAAAAA8DiSUhWUkpIii8WihIQEs8wwDCUlJSkkJER+fn6KiYnRgQMHvBdkJSUlJclisbgsdrvdXF/T23feDz/8oEceeUTNmjWTv7+/br31VmVnZ5vra3o727RpU+5ztFgsmjBhgqSa3z5JOn36tF544QWFh4fLz89Pbdu21UsvvaSzZ8+adWpDO4uLi5WQkKCwsDD5+fkpOjpaWVlZ5vra0EZve+ONNxQeHq6GDRuqc+fO+vjjj70dUo1zpb4Dl7dz504NHDhQISEhslgsWr9+vct6/s4r7krHcvTo0eW+q926dfNOsNVcSkqKbr/9dgUGBqply5YaNGiQDh065FKH7+Y5daEfqc3n+bp0Dq5L58i6dA6rSFtr2mdLUqoCsrKytHjxYt1yyy0u5XPmzNG8efO0cOFCZWVlyW63q2/fviouLvZSpJXXoUMH5eXlmcv+/fvNdbWhfQUFBerevbt8fX21efNmff3115o7d64aN25s1qnp7czKynL5DDMyMiRJQ4YMkVTz2ydJs2fP1ptvvqmFCxfq4MGDmjNnjl599VUtWLDArFMb2vnkk08qIyNDK1eu1P79+xUbG6s+ffrohx9+kFQ72uhNa9asUUJCgmbOnKm9e/fqrrvuUlxcnL777jtvh1bjXK7vwOWdPHlSHTt21MKFCy+6nr/zirvSsZSke++91+W7+v7773swwpojMzNTEyZM0J49e5SRkaHTp08rNjZWJ0+eNOvw3axb/UhtPc/XpXNwXTpH1qVzWEXaKtWwz9bAZRUXFxsRERFGRkaG0bNnT2Py5MmGYRjG2bNnDbvdbrzyyitm3V9++cUICgoy3nzzTS9FWzkvvvii0bFjx4uuqw3tMwzDeO6554wePXpccn1taeevTZ482bjhhhuMs2fP1pr29e/f3xgzZoxLWXx8vPHII48YhlE7PseSkhKjfv36xnvvvedS3rFjR2PmzJm1oo3edscddxhPPfWUS9lNN91kzJgxw0sR1UyX6ztQOZKMdevWma/5O796Fx5LwzCMUaNGGQ888IBX4qnp8vPzDUlGZmamYRh8N8+rK/1IXTnP16VzcF07R9alc9iFbTWMmvfZMlLqCiZMmKD+/furT58+LuU5OTlyOByKjY01y6xWq3r27Kndu3d7OsyrduTIEYWEhCg8PFzDhw/X0aNHJdWe9m3YsEFdunTRkCFD1LJlS3Xq1ElLliwx19eWdp5XVlamVatWacyYMbJYLLWmfT169NCHH36ow4cPS5K++OIL7dq1S/fdd5+k2vE5nj59WmfOnFHDhg1dyv38/LRr165a0UZvKisrU3Z2tsvxk6TY2FiO31W4VN+Ba8Pfufvt2LFDLVu2VLt27TR27Fjl5+d7O6QaobCwUJLUtGlTSXw3pbrXj9TF83xd/J7X1nNkXTqHXdjW82rSZ0tS6jJWr16tzz//XCkpKeXWORwOSZLNZnMpt9ls5rrqrmvXrlqxYoW2bNmiJUuWyOFwKDo6WsePH68V7ZOko0ePatGiRYqIiNCWLVv01FNP6emnn9aKFSsk1Y7P8dfWr1+vn3/+WaNHj5ZUe9r33HPPacSIEbrpppvk6+urTp06KSEhQSNGjJBUO9oZGBioO++8U//93/+tH3/8UWfOnNGqVav0ySefKC8vr1a00Zt++uknnTlzhuPnBpfrO3Bt+Dt3r7i4OL311lv66KOPNHfuXGVlZemee+6R0+n0dmjVmmEYmjJlinr06KHIyEhJfDelutWP1NXzfF37ntfWc2RdOoddrK1SzftsfbwdQHWVm5uryZMna+vWreVGLvyaxWJxeW0YRrmy6iouLs78d1RUlO68807dcMMNSktLM2+EVpPbJ0lnz55Vly5dlJycLEnq1KmTDhw4oEWLFumxxx4z69X0dp63dOlSxcXFKSQkxKW8prdvzZo1WrVqldLT09WhQwft27dPCQkJCgkJ0ahRo8x6Nb2dK1eu1JgxY3T99derfv36uu222zRy5Eh9/vnnZp2a3kZv4/hdu8v1HVOmTPFiZLUH31P3GDZsmPnvyMhIdenSRWFhYdq0aZPi4+O9GFn1NnHiRH355ZfatWtXuXV8N+vGMajr5/m68BlLtfccWZfOYZdqa037bBkpdQnZ2dnKz89X586d5ePjIx8fH2VmZuq1116Tj4+PmWW9MLOan59fLgNbUwQEBCgqKkpHjhwxn7BR09sXHBysm2++2aWsffv25g0pa0s7Jenbb7/Vtm3b9OSTT5pltaV9zz77rGbMmKHhw4crKipKjz76qJ555hlzFGNtaecNN9ygzMxMnThxQrm5ufr000916tQphYeH15o2ekvz5s1Vv359jl8V+HXfgWvD33nVCg4OVlhYGN/Vy5g0aZI2bNig7du3q1WrVmY538263Y/UlfN8Xf+e14ZzZF06h12qrRdT3T9bklKX0Lt3b+3fv1/79u0zly5duujhhx/Wvn371LZtW9ntdvNJZ9K5ueaZmZmKjo72YuRXz+l06uDBgwoODjZ/BNf09nXv3r3cIzIPHz6ssLAwSao17ZSkZcuWqWXLlurfv79ZVlvaV1JSonr1XE9X9evX19mzZyXVnnaeFxAQoODgYBUUFGjLli164IEHal0bPa1Bgwbq3Lmzy/GTpIyMDI7fNfp134Frw9951Tp+/Lhyc3P5rl6EYRiaOHGi1q5dq48++kjh4eEu6/lu1u1+pK6c5+v697wmnyPr0jnsSm29mGr/2Xr6zuo12a+fvmcYhvHKK68YQUFBxtq1a439+/cbI0aMMIKDg42ioiLvBVkJU6dONXbs2GEcPXrU2LNnjzFgwAAjMDDQOHbsmGEYNb99hmEYn376qeHj42P84Q9/MI4cOWK89dZbhr+/v7Fq1SqzTm1o55kzZ4zWrVsbzz33XLl1taF9o0aNMq6//nrjvffeM3Jycoy1a9cazZs3N6ZPn27WqQ3t/OCDD4zNmzcbR48eNbZu3Wp07NjRuOOOO4yysjLDMGpHG71p9erVhq+vr7F06VLj66+/NhISEoyAgADznIeKuVLfgcsrLi429u7da+zdu9eQZMybN8/Yu3ev8e233xqGwd95ZVzuWBYXFxtTp041du/ebeTk5Bjbt2837rzzTuP666/nWF7Eb3/7WyMoKMjYsWOHkZeXZy4lJSVmHb6bdacfqc3n+bp0Dq5L58i6dA67Ultr4mdLUqoSLkxKnT171njxxRcNu91uWK1W4+677zb279/vvQAradiwYUZwcLDh6+trhISEGPHx8caBAwfM9TW9fedt3LjRiIyMNKxWq3HTTTcZixcvdllfG9q5ZcsWQ5Jx6NChcutqQ/uKioqMyZMnG61btzYaNmxotG3b1pg5c6bhdDrNOrWhnWvWrDHatm1rNGjQwLDb7caECROMn3/+2VxfG9roba+//roRFhZmNGjQwLjttttcHp+LirlS34HL2759uyGp3DJq1CjDMPg7r4zLHcuSkhIjNjbWaNGiheHr62u0bt3aGDVqlPHdd995O+xq6WLHUZKxbNkysw7fzXPqQj9Sm8/zdekcXJfOkXXpHHalttbEz9ZiGIZRRYOwAAAAAAAAgIvinlIAAAAAAADwOJJSAAAAAAAA8DiSUgAAAAAAAPA4klIAAAAAAADwOJJSAAAAAAAA8DiSUgAAAAAAAPA4klIAAAAAAADwOJJSAAAAAAAA8DiSUsBlxMTEKCEhwdthAABwSaNHj9agQYO8HQYAoIpxvkdtRFIKqKCYmBhZLBZZLBZZrVZdf/31GjhwoNauXVuu7vl6FotFAQEBioiI0OjRo5WdnW3WGT16tEu9iy0AAAAAUFn5+fkaN26cWrduLavVKrvdrn79+umf//ynWadNmzbm7w4/Pz/ddNNNevXVV2UYhsu+Jk+erM6dO8tqterWW2/1cEtQ25GUAiph7NixysvL0zfffKN33nlHN998s4YPH67f/OY35eouW7ZMeXl5OnDggF5//XWdOHFCXbt21YoVKyRJf/rTn5SXl2cuv97m12UAgJqprKzM2yEAADygOp7vBw8erC+++EJpaWk6fPiwNmzYoJiYGP3nP/9xqffSSy8pLy9PBw8e1LRp0/T8889r8eLFLnUMw9CYMWM0bNgwTzYBdQRJKaAS/P39ZbfbFRoaqm7dumn27Nn685//rCVLlmjbtm0udRs3biy73a42bdooNjZWb7/9th5++GFNnDhRBQUFCgoKkt1uN5dfb/PrMgBAzRATE6OJEydqypQpat68ufr27SuLxaItW7aoU6dO8vPz0z333KP8/Hxt3rxZ7du3V6NGjTRixAiVlJSY+3n77bcVFRUlPz8/NWvWTH369NHJkyclSWfOnNGUKVPUuHFjNWvWTNOnTy93RRsAULWq+/n+559/1q5duzR79mz16tVLYWFhuuOOO5SYmKj+/fu71A0MDDR/szz55JO65ZZbtHXrVpc6r732miZMmKC2bdte45EDyiMpBVyjUaNGqUmTJhedxnehZ555RsXFxcrIyPBAZAAAT0tLS5OPj4/+53/+RyNGjJAkJSUlaeHChdq9e7dyc3M1dOhQpaamKj09XZs2bVJGRoYWLFggScrLy9OIESM0ZswYHTx4UDt27FB8fLz5Q2Tu3Ln661//qqVLl2rXrl36z3/+o3Xr1nmtvQBQV1Xn8/11112n6667TuvXr5fT6azQNoZhaMeOHTp48KB8fX2v4ogAV8fH2wEANV29evXUrl07HTt27Ip1b7rpJkmqUF0AQM1z4403as6cOZJkTsN++eWX1b17d0nSE088ocTERP373/82rzg/9NBD2r59u5577jnl5eXp9OnTio+PV1hYmCQpKirK3H9qaqoSExM1ePBgSdKbb76pLVu2eKx9AIBzqvP53sfHR8uXL9fYsWP15ptv6rbbblPPnj01fPhw3XLLLS51n3vuOb3wwgsqKyvTqVOn1LBhQz399NPXcGSAymGkFOAGhmFU6Mbk5698cBNzAKidunTpUq7s1z8AbDab/P39XaZA2Gw25efnS5I6duyo3r17KyoqSkOGDNGSJUtUUFAgSSosLFReXp7uvPNOc1sfH5+LvicAoGpV9/P94MGD9eOPP2rDhg3q16+fduzYodtuu03Lly93qffss89q3759yszMVK9evTRz5kxFR0dX+H2Aa0VSCrhGZ86c0ZEjRxQeHn7FugcPHpSkCtUFANQ8AQEB5cp+PQ3CYrGUmxZhsVh09uxZSVL9+vWVkZGhzZs36+abb9aCBQv0X//1X8rJyanawAEAlVITzvcNGzZU37599fvf/167d+/W6NGj9eKLL7rUad68uW688UbdeeedeueddzR//vxy98oFqhJJKeAapaWlqaCgwBxaezmpqalq1KiR+vTp44HIAAA1kcViUffu3TVr1izt3btXDRo00Lp16xQUFKTg4GDt2bPHrHv69GllZ2d7MVoAwNXy9Pn+5ptvNm+kfjFNmjTRpEmTNG3aNB6iAY/hnlJAJZSUlMjhcOj06dP64YcftHbtWs2fP1+//e1v1atXL5e6P//8sxwOh5xOpw4fPqw///nPWr9+vVasWKHGjRt7pwEAgGrtk08+0YcffqjY2Fi1bNlSn3zyif7v//5P7du3lyRNnjxZr7zyiiIiItS+fXvNmzdPP//8s3eDBgBUWlWe748fP64hQ4ZozJgxuuWWWxQYGKjPPvtMc+bM0QMPPHDZbSdMmKDZs2frnXfe0UMPPSRJ+uabb3TixAk5HA6VlpZq3759ks4luRo0aHDVxwCQSEoBlbJkyRItWbJEDRo0ULNmzdS5c2etWbNGDz74YLm6jz/+uKRzw2avv/569ejRQ59++qluu+02T4cNAKghGjVqpJ07dyo1NVVFRUUKCwvT3LlzFRcXJ0maOnWq8vLyNHr0aNWrV09jxozRgw8+qMLCQi9HDgCojKo831933XXq2rWr5s+fr3//+986deqUQkNDNXbsWD3//POX3bZFixZ69NFHlZSUpPj4eNWrV09PPvmkMjMzzTqdOnWSJOXk5KhNmzZXfxAASRaDcXkAAAAAAADwMO4pBQAAAAAAAI8jKQUAAAAAQA3x3Xff6brrrrvk8t1333k7RKDCmL4HAAAAAEANcfr0aR07duyS69u0aSMfH24fjZqBpBQAAAAAAAA8jul7AAAAAAAA8DiSUgAAAAAAAPA4klIAAAAAAADwOJJSAAAAAAAA8DiSUgAAAAAAAPA4klIAAAAAAADwOJJSAAAAAAAA8DiSUgAAAAAAAPC4/wc9K5EqHg2oFgAAAABJRU5ErkJggg==",
      "text/plain": [
       "<Figure size 1200x300 with 3 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Plotting lDDT vs rmsd.\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAwQAAAL2CAYAAAD/1lFEAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjguMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy81sbWrAAAACXBIWXMAAA9hAAAPYQGoP6dpAAEAAElEQVR4nOz9eZycZZkv/n+epZ7aq7o7vXc6ewKExIRdE4UgAqKjqCDO4BmWo+PugIzHMaMOYXQScealqOcMP505B5gzB3VGBf2OoySKBlkUEAMkQEiHhHTSS/Vaez1PPcvvj0pVunqtteuprs/79corvVRV3/V0dfd13fd13bdgWZYFIiIiIiJqSGKtB0BERERERLXDhICIiIiIqIExISAiIiIiamBMCIiIiIiIGhgTAiIiIiKiBsaEgIiIiIiogTEhICIiIiJqYEwIiIiIiIgaGBMCIiIiIqIGxoSAiIiIiKiBLZmE4NSpU/hv/+2/YdmyZfB4PNi6dSv+8Ic/5D5vWRZ27dqF7u5uuN1u7NixA4cOHarhiImIiIiIak+u9QAqYWJiAtu3b8fll1+On//852hvb8fRo0fR1NSUu83XvvY1fP3rX8f999+PDRs24Ctf+QquvPJKHD58GH6/P+/xTNPEwMAA/H4/BEFY5GdDREREdmRZFqLRKLq7uyGKS2ZOlQiCZVlWrQdRrs9//vN44okn8Nvf/nbWz1uWhe7ubtx+++3467/+awCAqqro6OjA3XffjY9+9KN5tz958iR6e3urPm4iIiKqP/39/Vi+fHmth0FUMUsiIdi4cSOuvvpqnDx5Evv370dPTw8+8YlP4C/+4i8AAK+99hrWrl2L5557Duedd17uftdeey2amprwwAMP5D1eOBxGU1MTXnrppbzVA6fTCafTuThPioiIiGwlEomgt7cXk5OTCAaDtR4OUcUsiZKh1157Dffeey/uuOMO/M3f/A2efvpp/OVf/iWcTiduuukmDA0NAQA6Ojry7tfR0YHXX399xuNly4Q2btyY9/E777wTu3btqs6TICIiorrAcmJaapZEQmCaJi688ELs3r0bAHDeeefh0KFDuPfee3HTTTflbjf9B9iyrHl/qPv7+xEIBHLvc3WAiIiIiJaaJdER09XVNWM2/5xzzsGJEycAAJ2dnQCQWynICoVCM1YNpgoEAnn/mBAQERER0VKzJBKC7du34/Dhw3kfe/XVV7Fy5UoAwOrVq9HZ2Yl9+/blPq9pGvbv349t27Yt6liJiIiIiOxkSZQMfeYzn8G2bduwe/du3HDDDXj66afx3e9+F9/97ncBZEqFbr/9duzevRvr16/H+vXrsXv3bng8Htx44401Hj0RERERUe0siYTgoosuwkMPPYSdO3fi7/7u77B69Wrcc889+OAHP5i7zec+9zkkk0l84hOfwMTEBC655BLs3bt3xhkERES1tOrzP6v4Yx7/6jsr/phERLR0LIltRystEokgGAwiHA7nNRUTEVUbEwIi+2J8QEvVkughICIiIiKi0iyJkiEiIlo8lV7F4AoGEVFtcYWAiIiIiKiBMSEgIiIiImpgTAiIiIiIiBoYEwIiIiIiogbGhICIiIiIqIExISAiIiIiamBMCIiIiIiIGhgTAiIiIiKiBsaEgIiIiIiogTEhICIiIiJqYEwIiIiIiIgaGBMCIiIiIqIGxoSAiIiIiKiBMSEgIiIiImpgTAiIiIiIiBoYEwIiIiIiogbGhICIiIiIqIExISAiIiIiamBMCIiIiIiIGhgTAiIiIiKiBibXegBERPVs1ed/VushEBERlYUrBEREREREDYwJARERERFRA2NCQERERETUwJgQEBERERE1MCYEREREREQNjAkBEREREVEDY0JARERERNTAeA4BERHVVDXOcjj+1XdW/DGJiJYqrhAQERERETUwJgRERERERA2MCQERERERUQNjQkBERERE1MDYVExEtMRVo2mXiIiWDq4QEBERERE1MCYEREREREQNjCVDRA2qHspIuJc8ERFR9XGFgIiIiIiogTEhICIiIiJqYEwIiIiIiIga2JJLCPbs2QNBEHD77bfnPmZZFnbt2oXu7m643W7s2LEDhw4dqt0giYiIiIhsYkk1FT/zzDP47ne/ize84Q15H//a176Gr3/967j//vuxYcMGfOUrX8GVV16Jw4cPw+/312i0RERULZVummeDOxEtZUtmhSAWi+GDH/wg/vmf/xnNzc25j1uWhXvuuQdf+MIX8L73vQ+bNm3CAw88gEQigQcffHDex4xEInn/VFWt9tMgIiIiIlpUS2aF4JOf/CTe+c534m1vexu+8pWv5D5+7NgxDA0N4aqrrsp9zOl04rLLLsOTTz6Jj370o3M+Zm9vb977d955J3bt2lXxsRPR4qiHrVbJnqrx2uGqAxHZxZJICL7//e/jueeewzPPPDPjc0NDQwCAjo6OvI93dHTg9ddfn/dx+/v7EQgEcu87nc4KjJaIiIiIyD7qPiHo7+/Hbbfdhr1798Llcs15O0EQ8t63LGvGx6YLBAJ5CQERERER0VJT9z0Ef/jDHxAKhXDBBRdAlmXIsoz9+/fjW9/6FmRZzq0MZFcKskKh0IxVAyIiIiKiRlP3CcEVV1yBF198EQcOHMj9u/DCC/HBD34QBw4cwJo1a9DZ2Yl9+/bl7qNpGvbv349t27bVcORERERERLVX9yVDfr8fmzZtyvuY1+vFsmXLch+//fbbsXv3bqxfvx7r16/H7t274fF4cOONN9ZiyEREREREtlH3CUEhPve5zyGZTOITn/gEJiYmcMkll2Dv3r08g4CIiIiIGt6STAh+85vf5L0vCAJ27drFLUOJiIiIiKZZkgkBUbF4qqk98dwAIiKi6qv7pmIiIiIiIiodEwIiIiIiogbGhICIiIiIqIExISAiIiIiamBMCIiIiIiIGhgTAiIiIiKiBsaEgIiIiIiogTEhICIiIiJqYEwIiIiIiIgaGBMCIiIiIqIGxoSAiIiIiKiBMSEgIiIiImpgTAiIiIiIiBoYEwIiIiIiogbGhICIiIiIqIHJtR4AERVm1ed/VushEBER0RLEFQIiIiIiogbGhICIiIiIqIExISAiIiIiamBMCIiIiIiIGhibiqnu1ENzbT2MkYiIiAjgCgERERERUUNjQkBERERE1MCYEBARERERNTAmBEREREREDYwJARERERFRA+MuQ0RERDVQ6d3Ijn/1nRV9PCJqHFwhICIiIiJqYEwIiIiIiIgaGBMCIiIiIqIGxoSAiIiIiKiBMSEgIiIiImpgTAiIiIiIiBoYEwIiIiIiogbGhICIiIiIqIExISAiIiIiamBMCIiIiIiIGhgTAiIiIiKiBsaEgIiIiIiogTEhICIiIiJqYEsiIdizZw8uuugi+P1+tLe34z3veQ8OHz6cdxvLsrBr1y50d3fD7XZjx44dOHToUI1GTERERERkD0siIdi/fz8++clP4ne/+x327dsHXddx1VVXIR6P527zta99DV//+tfxP//n/8QzzzyDzs5OXHnllYhGozUcORERERFRbcm1HkAl/OIXv8h7/7777kN7ezv+8Ic/4NJLL4VlWbjnnnvwhS98Ae973/sAAA888AA6Ojrw4IMP4qMf/Wgthk1EREREVHNLYoVgunA4DABoaWkBABw7dgxDQ0O46qqrcrdxOp247LLL8OSTT875OJFIJO+fqqrVHTgRERER0SJbcgmBZVm444478OY3vxmbNm0CAAwNDQEAOjo68m7b0dGR+9xsent7EQwGc//27NlTvYETEREREdXAkigZmupTn/oUXnjhBTz++OMzPicIQt77lmXN+NhU/f39CAQCufedTmflBkpEREREZANLKiH49Kc/jZ/+9Kd47LHHsHz58tzHOzs7AWRWCrq6unIfD4VCM1YNpgoEAnkJARERERHRUrMkSoYsy8KnPvUp/PjHP8ajjz6K1atX531+9erV6OzsxL59+3If0zQN+/fvx7Zt2xZ7uEREREREtrEkVgg++clP4sEHH8RPfvIT+P3+XF9AMBiE2+2GIAi4/fbbsXv3bqxfvx7r16/H7t274fF4cOONN9Z49EREREREtbMkEoJ7770XALBjx468j99333245ZZbAACf+9znkEwm8YlPfAITExO45JJLsHfvXvj9/kUebeNZ9fmf1XoIRERERDSHJZEQWJa14G0EQcCuXbuwa9eu6g+IiIiIiKhOLIkeAiIiIiIiKs2SWCEgIiJqdJUuzzz+1XdW9PGIyL64QkBERERE1MCYEBARERERNTAmBEREREREDYwJARERERFRA2NCQERERETUwJgQEBERERE1MCYEREREREQNjAkBEREREVEDY0JARERERNTAeFIxzVDp0y6JiIiIyL64QkBERERE1MCYEBARERERNTAmBEREREREDYwJARERERFRA2NCQERERETUwJgQEBERERE1MCYEREREREQNjAkBEREREVEDY0JARERERNTAmBAQERERETUwJgRERERERA2MCQERERERUQNjQkBERERE1MCYEBARERERNTAmBEREREREDYwJARERERFRA2NCQERERETUwJgQEBERERE1MCYEREREREQNjAkBEREREVEDY0JARERERNTAmBAQERERETUwJgRERERERA1MrvUAqDyrPv+zWg+BiIiIiOoYVwiIiIiIiBoYEwIiIiIiogbGhICIiIiIqIExISAiIiIiamBsKl5kbAImIiIiIjvhCsESYOlpTD7+/2Dp6VoPpS7x+pWO1650vHbl4fUrHa9d6VRVzfufaKlouITgn/7pn7B69Wq4XC5ccMEF+O1vf1vrIZXNMtIIP/E9WAZ/uZeC1690vHal47UrD69f6XjtSseEgJaqhkoIfvCDH+D222/HF77wBfzxj3/EW97yFlxzzTU4ceJErYdGRERERFQTDZUQfP3rX8eHPvQhfPjDH8Y555yDe+65B729vbj33ntrPTQiIiIioppomKZiTdPwhz/8AZ///OfzPn7VVVfhySefzPuYZVkAgFOnTiESieQ+7nQ64XQ6yxqHqSbKuv98j1mNx24EvH6l47UrHa9deXj9SlfotZv6948yotEogDNxAtFSIVgN8qoeGBhAT08PnnjiCWzbti338d27d+OBBx7A4cOHcx87efIkent7azFMIiIisrmjR49izZo1tR4GUcU0zApBliAIee9bljXjY93d3Th69CgcDkfe5yqxQkBERET1ybIsRKNRdHd313ooRBXVMAlBa2srJEnC0NBQ3sdDoRA6OjryPiaKIjN/IiIimiEYDNZ6CEQV1zBNxYqi4IILLsC+ffvyPr5v3768EiIiIiIiokbSMCsEAHDHHXfgz//8z3HhhRfiTW96E7773e/ixIkT+NjHPlbroRERERER1URDJQQf+MAHMDY2hr/7u7/D4OAgNm3ahP/6r//CypUraz00IiIiIqKaaJhdhoiIiIiIaKaG6SEgIiIiIqKZmBAQERERETUwJgRERERERA2MCQERERERUQNrqF2GCmWaJgYGBuD3+2ecYkxERESNaepJxaK48Jwq4wmqpaJer5bN7N6927rwwgstn89ntbW1Wddee631yiuv5N3GNE3rzjvvtLq6uiyXy2Vddtll1sGDBxd87B/+8IfWOeecYymKYp1zzjnWj3/841lv19/fbwHgP/7jP/7jP/7jP/6b8e/o0aMFxTSMJ/jPDv/6+/sXfK3aboVg//79+OQnP4mLLroIuq7jC1/4Aq666iq89NJL8Hq9AICvfe1r+PrXv477778fGzZswFe+8hVceeWVOHz4MPx+/6yP+9RTT+EDH/gAvvzlL+O9730vHnroIdxwww14/PHHcckll+TdNvsY/f39CAQC1X3CFRCJRNDb21s347UbXr/S8dqVjteuPHa9fpZlIZ1OAwAcDoctZ4Xteu3qwalTp7Bx40Y4HI6Cbl9v8QQtLdmf9bli46lsfw7ByMgI2tvbsX//flx66aWwLAvd3d24/fbb8dd//dcAAFVV0dHRgbvvvhsf/ehHZ32cD3zgA4hEIvj5z3+e+9jb3/52NDc343vf+17ebSORCILBIMLhcF38ANfbeO2G1690vHal47UrD69f6XjtSnfy5MlcMrV8+fIFb89rTbVUzOvP9k3F4XAYANDS0gIAOHbsGIaGhnDVVVflbuN0OnHZZZfhySefnPNxnnrqqbz7AMDVV189730ikUjeP1VVy3kqREREVEdUVc2LA6LRaEmPw3iC7M7WCYFlWbjjjjvw5je/GZs2bQIADA0NAQA6OjrybtvR0ZH73GyGhoaKvk9vby+CwWDu3549e0p9KkRE1AB0XcfDDz+Mhx9+GLqu13o4VKY9e/bkxQEbN24s6XEYT5Dd2a6HYKpPfepTeOGFF/D444/P+Nz0ukzLshas1Sz2PtNr/pxOZyHDXnROpxN33nmnbcdnd7x+peO1Kx2vXXnsev1M08Tzzz8PAHjHO95R49HMzq7Xzo527tyJO+64I/f+6Ogo1q5dW/S1q5d4ghqXbROCT3/60/jpT3+Kxx57LK9Or7OzE0Bmxr+rqyv38VAoNGMFYKrOzs4ZqwEL3ScQCNRFzZ/T6cSuXbtqPYy6xetXOl670vHalYfXr3S8doVzOp2zBu/FBvT1Ek9Q47JdyZBlWfjUpz6FH//4x3j00UexevXqvM+vXr0anZ2d2LdvX+5jmqZh//792LZt25yP+6Y3vSnvPgCwd+/eee9DRERERLTU2W6F4JOf/CQefPBB/OQnP4Hf78/N6geDQbjdbgiCgNtvvx27d+/G+vXrsX79euzevRsejwc33nhj7nFuuukm9PT05Or0brvtNlx66aW4++67ce211+InP/kJfvnLX85ajkRERERE1ChslxDce++9AIAdO3bkffy+++7DLbfcAgD43Oc+h2QyiU984hOYmJjAJZdcgr179+bts3rixIm8U9m2bduG73//+/jiF7+IL33pS1i7di1+8IMfzDiDgIiIiIiokdj+HIJa4L7BRERUCk3TcivTO3fuhKIoNR4RVVKx8QHjCaqlJXUOARERERERVY/tSoaIiIjqlcPhwGc/+9nc20RE9YAJARERUYUIggCv11vrYRARFYUlQ0REREREDYwrBERERBWi6zoeeeQRAMDVV18NWeafWSKyP64QEBERVYhpmnj22Wfx7LPPwjTNWg+HiKggTAiIiIiIiBoY1zKJaEkxTQunJpOIazq8ioyeJjdEUaj1sIiIiGyLCQERLRl9oSgeOTiMoyMxpHQDLlnC2jYfrt7UgXXt/oUfgIiIqAExISCiJaEvFMV9TxzHeFxDV9AFj+JGQtNxcCCMgXASt25fxaSAiIhoFuwhIKK6Z5oWHjk4jPG4hvXtPvhdDkiiAL/LgfXtPozHNew9NAzTtGo9VCIiItthQkBEde/UZBJHR2LoCrogCPn9AoIgoCvoQl8ohlOTyRqNkIiIyL5YMkRUZWxyrb64piOlG/Ao7lk/71YkDEdSiGv6Io+MGo3D4cBtt92We5sIADbd+QhEpyfvY8e/+s4ajYZoJiYERFXEJtfF4VVkuGQJCU2H3zUzCEtqBpyyBK/CX3lUXYIgoKmpqdbDICIqCkuGiKok2+R6cCCMJo8Da1p9aPI4cHAgjPueOI6+ULTWQ1wyeprcWNvmw2A4BcvK7xOwLAuD4RTWtfvQ0zT7CgIREVEjY0JAVAVscl1coijg6k0daPEqOBKKIZpKQzdNRFNpHAnF0OJVcNW5HSzVoqozDAN79+7F3r17YRhGrYdDRFQQJgREVcAm18W3rt2PW7evwqbuICYTaRwfjWMykcbmniC3HKVFYxgGnnrqKTz11FNMCIiobrCglqgK2ORaG+va/Vizw8cmbiIioiIwISCqAja51o4oCuht8Sx8Q3AHKCIiIoAJAVFVZJtcDw6E4XPKeWVD2SbXzT1BNrnWEHeAIiIiymBCQFQF2SbXgXASR0KZXgK3IiGpGRgMp9jkWmPZHaDG4xq6gi54FDcSmo6DA2EMhJPsOSAioobCpmKiKmGTqz1xBygiIqJ8XCEgqiI2udpPMTtAFdqLQEREVM+YEBBVWTFNrlR93AGKqsnhcODjH/947m0ionrAhICIGgp3gKJqEgQB7e3ttR4GEVFR2ENARA0luwPUYDgFy8rvE8juALWu3ccdoIiIqGFwCoyIGgp3gKJqMgwDv/3tbwEAb3nLWyBJUo1HRES0MCYERNRwsjtAZc8hGI6k4JQlbO4J4qpzeQ4Blc4wDOzfvx8AsG3bNiYERFQXmBAQUUPiDlBEREQZTAiIqGFxBygiIiImBA3NNC3OjhIRERE1OCYEDaovFM3VT6d0Ay5Zwto2H67exPppIiIiokbChKAB9YWiuO+J4xiPa+gKuuBR3EhoOg4OhDEQTuLW7auYFBARERE1CJ5D0GBM08IjB4cxHtewvt0Hv8sBSRTgdzmwvt2H8biGvYeGYZrWwg9GRERERHWPKwQN5tRkEkdHMnuvC0J+v4AgCOgKutAXiuHUZJLNlkRERZJlGR/+8IdzbxMR1QP+tmowcU1HSjfgUWY/hdWtSBiOpBDX9EUeGRFR/RNFET09PbUeBtnY8a++s9ZDIJqBJUMNxqvIcMkSEnME/EnNgFOW4FWYKxIRERE1AiYEDaanyY21bT4MhlOwrPw+AcuyMBhOYV27Dz1Ns68gEBHR3AzDwBNPPIEnnngChmHUejhERAXhNHCDEUUBV2/qwEA4iSOhTC+BW5GQ1AwMhlNo8Sq46twOnkdARFQCwzDwy1/+EgBw0UUXQZKkGo+IiGhhXCFoQOva/bh1+yps6g5iMpHG8dE4JhNpbO4JcstRIiIiogbDFYIGta7djzU7fDypmIiIiKjBMSFoYKIo1GxrUdO0mIwQERER2QATAlp0faEoHjk4jKMjMaR0Ay5Zwto2H67e1MFyJSIiIqJFxoSAFlVfKIr7njiO8biGrqALHsWNhKbj4EAYA+EkexiIiIiIFhkTAlo0pmnhkYPDGI9rWN/uy52U7Hc54HPKOBKKYe+hYaxp9RVVPsTyIyIiIqLSMSGgRXNqMomjI5mtTrPJQJYgCOgKutAXiuHUZLLg3gaWHxGRnciyjJtvvjn3NhFRPeBvK1o0cU1HSjfgUWY/9MytSBiOpBCf4xTl6Vh+RER2I4oiVq1aVethEBEVhecQ0KLxKjJcsoTEHAF/UjPglCV4lYXz1OnlR36XA5IowO9yYH27D+NxDXsPDcM0rQUfi4iIiKiRMSGgRdPT5MbaNh8GwylYVn6gblkWBsMprGv3oadp9hWEqYopPyIiWiyGYeDpp5/G008/DcMwaj0cIqKCsGSIFo0oCrh6UwcGwkkcCWWCebciIakZGAyn0OJVcNW5HQU1BFe6/IiIqBIMw8DPf/5zAMDWrVshSVKNR0REtDCuENCiWtfux63bV2FTdxCTiTSOj8YxmUhjc0+wqJr/SpYfERERETUyRku06Na1+7Fmh6+srUKz5UcHB8LwOeW8sqFs+dHmnmBB5UdEREREjYwJAdWEKAoFby061/0rVX5EVC08I4OIiOqB7UqGHnvsMbzrXe9Cd3c3BEHAww8/nPd5QRBm/fcP//APcz7m/fffP+t9UqlUlZ8NVVOlyo+IqqEvFMW9vzmKb+x7Fd/61RF8Y9+ruPc3R9EXitZ6aERERHlst0IQj8exZcsW3HrrrbjuuutmfH5wcDDv/Z///Of40Ic+NOttpwoEAjh8+HDex1wuV/kDppqqRPkRUaXxjAwiIqontksIrrnmGlxzzTVzfr6zszPv/Z/85Ce4/PLLsWbNmnkfVxCEGfelpaHc8iOiSpp+Rka2v8XvcsDnlHEkFMPeQ8NY0+pj4kpERLZgu4SgGMPDw/jZz36GBx54YMHbxmIxrFy5EoZhYOvWrfjyl7+M8847b977RCKRvPedTiecTmdZYyai2lisev5izshgIjtTvfddyLKMP/uzP8u9TfVNVVWoqpp7f3pcUChTTeQ9BuMJspu6/m31wAMPwO/3433ve9+8tzv77LNx//33Y/PmzYhEIvjmN7+J7du34/nnn8f69evnvF9vb2/e+3feeSd27dpViaHTElPvQcxS1xeK4pGDwzg6EkNKN+CSJaxt8+HqTR0VL93hGRmlW8zvU7WIoogNGzbUehhUIXv27MFdd91V9uOcuveW3NvBexhPkP0I1vQjY21EEAQ89NBDeM973jPr588++2xceeWV+Pa3v13U45qmifPPPx+XXnopvvWtb834fCQSQTAYRH9/PwKBQO7jzOhpNq8ORfHDP/Tj6EgMhgU0ux1Y1+63TRDT6MnKzHp+GQlNz+1GVel6/v7xBL6x71U0eRzwuxwzPh9NpTGZSOMzV27gCsEUi/19IirEbCsEvb29CIfDefHBXLLxRM/H74foPPPzLkgOCPLM3w/THf/qO0sbOBHOvP4Keb3W7QrBb3/7Wxw+fBg/+MEPir6vKIq46KKLcOTIkXlvFwgECvqBp6Wj2OD5Vy8P41u/OoKRqApFFuCUJUSTaYzGNVs0jy6FGddy1KKen2dkFG8p9V0YhoEXX3wRALB582aeVFznKjURKDo9eQkBkd3UbULwv//3/8YFF1yALVu2FH1fy7Jw4MABbN68uQojo3pVbPD86nAE3/rVEQxFUugMOKHIEtKGiXAyDVU3AKCmQQx3uqlNPT/PyCjeUuq7MAwDP/nJTwAAGzduZEJARHXBducQxGIxHDhwAAcOHAAAHDt2DAcOHMCJEydyt4lEIviP//gPfPjDH571MW666Sbs3Lkz9/5dd92FRx55BK+99hoOHDiAD33oQzhw4AA+9rGPVfW5UP3IBs8HB8Jo8jiwptWHJo8DBwfCuO+J4zP2jjdNCz989hRGoiq6Ai64HDJEIbNC0OJVkEqbSGg6jgxHcWoyuejPZ/qMq9/lgCQK8LscWN/uw3hcw95DwzBN21YMVsSZev7Z5z7cigRVNypez88zMopTq+8TERFl2G6F4Nlnn8Xll1+ee/+OO+4AANx88824//77AQDf//73YVlWbieH6U6cOAFRPJPrTE5O4iMf+QiGhoYQDAZx3nnn4bHHHsPFF19cvSdCdaOUcoXsjKYii3DI+Xm1IAjwuWREUzomk+maBDFLaca1HF5FhkuWkND0Wev5k5oBpyzBO0cgWg6ekVG4Wn6fiIjIhgnBjh07sFCf80c+8hF85CMfmfPzv/nNb/Le/8Y3voFvfOMblRheTTV6c2i1lBI8xzUdhmXCKYtIGxaccv79HJIITdcgCqhJEMOdbjJqXc/PMzIKU+vvExFRo7NdQkCza/Tm0GoqJXj2KjKa3QqiSR3hZBqKV8kLYjTdgKZbWNvuq0kQwxnXDNbz1wd+n4iIast2PQQ0U7H17VScqcHzbGYLnnua3FjX7odHkeFySBiPa1B1A6ZlIZU2MBRR0e534vrze2sSxGRnXAfDqRkrbtkZ13U1SlYWG+v56wO/T0REtbO0pweXgKW0HZ9dlVKuMHVGEwASmo6oqkPT09B0E50BFz59xXps6KxNEMMZ13ys568P/D4REdUGEwKbY3No9ZUaPGdnNB85OIy+UBSTSQ2iIGJduw/XXdCDDR21PcNi6viOjsQwHEnBKUvY3BPEVedWvtTM7j0urOevD/X+fZJlGddff33ubSKiesDfVjbH5tDFUWrwbPcZzcUaH3tciDJEUcS5555b62EQERWFCYHNsTl08ZQaPNt9RrPa4+MBaERERPWNUaTNcTu+xWX34N5u2ONSWXYvu6KFmaaJl19+GQBwzjnn5J2JQ0RkV0wIbI7NoWRn7HGpHJZdLQ26ruOHP/whAGDnzp1QFKXGIyIiWhinLuoAt+MjuzrT4zL73IJbkaDqBntcFsCthYmIqJa4QlAn7N68ShmNVvLBHpfyseyKiIhqjX+l60i917cv9WC5EUs+2ONSPpZdERFRrTEhoEWx1IPlRt1phz0u5ePWwkREVGvsIaCqW+r10dNLPvwuByRRgN/lwPp2H8bjGvYeGoZpWrUealWwx6U8U8uuZsOyKyIiqjb+haGqaoT6aJZ8sMelHCy7IiKiWmNCQFXVCMEySz4y6r3HpVZYdrW0SJKEa6+9Nvc2EVE9YEJAVVXNYNkuTcrcaYfKlS27yvbZDEdScMoSNvcEcdW5S6PPplFIkoStW7fWehhEREVhhEJVVa1guRpNyqUmGCz5oEpg2RUREdUKEwKqqmoEy9XY0aecBIMlH1QpLLuqf6Zpoq+vDwCwbt06iCL37iAi++NvKqqqbLDc4lVwJBRDNJWGbpqIptI4EooVHSxXY0efSuyCxJ12iAgAdF3H9773PXzve9+Dri/tviEiWjq4QkBVV8n66Eo3KVdyFySWfBAREVE9YkJAi6JSwXKlm5QrnWCw5IOIiIjqDRMCWjSVCJYr3aTMLUOJiIio0bGHgOpKtkl5MJyCZeX3CWSblNe1+wpuUuYpsURERNTomBBQXal0k3KlEwwiIiKiesOEgOpOJXf0qXSCQURERFRvWAdBdamSO/rwlFgiqhRJknDNNdfk3iYiqgdMCKguzHWKcKV29OGWoURUCZIk4eKLL671MIiIisKEgGyvnFOEi2HXLUPnSoaIiIiIKoEJAdla9hTh8biGrqALHsWNhKbj4EAYpyYTeMfmLrT6nbYOlMsJ6BcrGSKiyjBNEydOnAAArFixAqLIVj0isj8mBGRb850irOkmnj4+juf7J7FimRduhz0D5XIC+vmSoYFwsugGaqoNrvA0Fl3X8cADDwAAdu7cCUVRajwiIqKFMSEg25rrFOHxuIbnT4ahpk0YkoA2nxOyJNguUC4noJ8vGfI5ZRwJxbD30DDWtPoYXNoYV3iIiKgecC2TbMc0LfSPJ3BwIIzxhAq348xOHZZloS8UQ1LT0eZXIAqAYVnwuxxY3+7DeFzD3kPDME1rzsd9ZSiC/vHErLep5HOYGtD7XQ5IolDQOIG5kyEAEAQBXUEX+kIxnJpMVu05UHmyCeHBgTCaPA6safWhyePAwYEw7nviOPpC0VoPkYiICABXCMhmps6ojic0HA3FEU3q2NgdRItXQTSlYyKhwedyQDctSKIIRcrktdMD5akNwos9U1tMQD9bI3Nc05HSDXiU2Q9EcysShiMpxOc4YXmpqpfyG67wEBFRPWFCQLYxvcSmK+hCNJnGyckkVN3EeSuaYVoWdMOE7JQwmdDRHnDB7zrzMp4tUK5FLX65Ab1XkeGSJSQ0HX6XY8bnk5oBpyzBqzTOj3A9ld+UmxBOVy+JEBER1afGiSbI1uaaUd3YHYCqGxiJqXhpMIxzOv2wAIxGVfjdDqxt8+YFXNMD5VrN1JYb0Pc0ubG2zYeDA2H4nHLec7QsC4PhFDb3BNHTNHvCsdTUW4N1JVd46ikRIiKi+sQeArKFuWZUW7xOnLeiGcub3AhFVJycSMApS1BkCVuWB9HideZumw2U17X7coFyrWrxswH9YDgFy8rvE5htnNOJooCrN3WgxavgSCiGaCoN3TQRTaVxJBRDi1fBVed2NMQscbn9GLUwNSG0LAuRZBqjMRWRZBqWZRW8wsM+BCIiWgxcIaCKKaesYb4Z1RavE5esWYaXByO44aJe+Jwyfv7iEMbiGhRZgluRkNQMDIZTMwLlWtXiZwP6gXASR0KZhGS+cc5mXbsft25flZsdHo6k4JQlbO4J4qpzG2d2uNLlN4shmxD+7tgYdN3ERDKT0MmiiGa3A7Is4k1rls27wsM+hPokSRLe9ra35d4mIqoHTAioIsota1ioxCaVNtDsUbCpO4jeFg+6gq6CAuVa1uJXIqBf1+7Hmh2+hqkfny2prMcGa1EUcHaXHw8dOIVoKo1lXgVBtwNJzcBrY3EEXA6c1emf9/tYj4kQZZKA7du313oYRERFYUJAZatEfXexNfOFBsq1rsWvREAvikJDBHxzJZVv6A3WXYO1aVp4ZTCKroALbT4FE4k0wsk0ZFHEmlYvZFHE4aEoLj+rfc7XQj0mQkREVJ/s8xeU6lKlyhpKKbEpJFCuROlOuRoloC/HfEnlqckkmtwODIZTddNgnZ3dX9/hg88pI5rSoRkmFEmE3yUjpuoLzu5zp6n6ZJomBgcHAQBdXV0QRbbqUelWff5nC97m+FffuQgjoaWOv6moLJVs2s2W2GzqDmIykcbx0TgmE2ls7gmWtYtMMY+7mIeXUcZCTcMTCQ0QgGaPo24arM/M7mcSmIDbgVafEwG3A4IgwK1IUHVj3tn9chvTqTZ0Xce//Mu/4F/+5V+g61y9IaL6wKklKkulyxqqVTNfyOPaeXvHpbwPfSFJ5WQijfed34Pn+8N10WBdidl9O6xuERFRY2BCQGWpRllDtUps5ntcO+9zb+dEpRKmJ5WWZeWV2GRm01No9Tvx8R1r6yIxqlTvCneaIiKixcCEYImp9kzy9MfvCriKCnzsONNt5+0d7ZyoVMrUpDJtmDgaimM8oeW26fQqEpq9CryKXDf9GJWc3W+0naaIiGjxMSFYQqo9kzzX45/d5S8o8LHrTLddt3e0c6JSSVP37J+Iq0ilTfhcMhySDE03cHIyCd20kEwXVnZml6SzkrP79ZIIERFRfWJCsERUeyZ5ocd/69nteGUwOmfgY+eZbrtu72jXRKXSRFHAlRs7sPelIYzENHQGXHBIItKGibhqoM3nRMDlwC9fCmFd2/x799st6eTsPhER1QMmBEtAtWeSC3n8w0NRfPTSNRg8HThPDXzsPtNt1+0d7ZqoVINbkdDmd0IWBcS1zO47siii/XRJmkMSFkx+7Jp0iqKAniZ3Lik4NZlkUkBERLbChGAJqPZMcqGPPxhJobfFkyvZeDUUhVeRYVrWgvc/MhzFs6+PI+B2LPosaq0PL5uLXROVaohrOhRZxBvXLENCM/L27BcEAbppzpv82DnptNuqBVWXJEm47LLLcm8TEdWD+o8kqOozycU8/mzBT8AtYzSmonuOgDqZNvDSYATf2X8UToe06AGTXbd3tGuiUg3Z5CeZNhBwF5/82LW8yq6rFlQ9kiRhx44dtR4GEVFReDDZEjB1Jnk25c4kF/r4I1EV9z1xHAcHwmjyOLCm1YcmjwPHRuPoH09gYDIx477jcQ3PvT6BaErHMq8zd5+DA2Hc98Rx9IWiJY25WNU6FK0c2USlxavUzYFcpSr3EK6pB4HNppCDwCptoQPXxuMa9h4atuXhdzygj4iosXCFYAmo9kxyIY+/qTuA5/snZy3ZeENPEIOTKRwciJwuBRJz9+0LRTGZTGNNqxedp2d3a1XmYccG0EbZh77cVRo7llfZddViISxxKo9lWRgZGQEAtLW1zfjeExHZEROCJaDaJS+FPP6W3ib8+LlTswY/oihiU08AfzwxiRdOhbG2zQe3IiEUSeH1sQSaPQ6sm5JEALULmOy4vaMdE5VqKCf5sWN5VT02hbPEqXzpdBr33nsvAGDnzp1QFKXGIyIiWhgTgiWi2jPJCz2+blqzBj/ZU2cVObOLzKoWLyYTaQxHUkilTfhdMs5f0YwWr3PG17RjwFQrdkxUqqHU5MeOfSB2XLWYj50bs4mIqLrs8ZeIKqLaM8nzPX7/eGJG8DMeV3OnzibTOkwT2NLbhPee24M2vxORZBrf+/0JuByz78Rht4CJFkepyY/dyqvsuGoxn3otcSIiovLZrqn4sccew7ve9S50d3dDEAQ8/PDDeZ+/5ZZbIAhC3r83vvGNCz7uj370I2zcuBFOpxMbN27EQw89VKVnUFvZYOrszgB6WzwVn8mb6/GnN4WOx1Uc6J9EKJqCUxYgCyLaA070jyfwi4NDkEUBF65swbp2f8mNpFRbdmw8Xdfux8d3rMVnrtyAT1+xHp+5cgM+dtlaNoUXwI6N2UREtDhsN/Uaj8exZcsW3Hrrrbjuuutmvc3b3/523Hfffbn3F6rRfOqpp/CBD3wAX/7yl/He974XDz30EG644QY8/vjjuOSSSyo6/kY1tWTj1eEYRqIpJFQdPpcDcVWHxyljY1cQzR5HrvTgY5f5bFfmQYWpZeNp9pyLuVbB7FReZbdVi/nUW4kTERFVju1+s19zzTW45ppr5r2N0+lEZ2dnwY95zz334Morr8TOnTsBZBq99u/fj3vuuQff+973yhovnZENfv79mX4cGghDEgWoupk7bbbFm0ncppYe1FPARBm1bDytxx1w6qUpvN5KnIiIqHJslxAU4je/+Q3a29vR1NSEyy67DH//93+P9vb2OW//1FNP4TOf+Uzex66++mrcc889836dSCSS977T6YTTObP5lc5Y1+7He87vwauhKDoDbrgdUu602WyDcTJtYCKhIZpK5+5TDwFTI5lrFr6Wjaf1vAOOnVYt5mLHxmyiWlNVFaqq5t6fHhcUylTzz+ERJAcEeeZKHFGt1F1CcM011+D9738/Vq5ciWPHjuFLX/oS3vrWt+IPf/jDnMH60NAQOjo68j7W0dGBoaGheb9Wb29v3vt33nkndu3aVdb4G4FPkeGUJahpA+7TDcOzNRg//McBKLKIde3+ugiYGsV8s/BOWapJ4yl3wMlYqFyqXFyxK58kSXjTm96Ue5vq2549e3DXXXeV/Tin7r0l7/3g9j9D05s/WPbjElVK3SUEH/jAB3Jvb9q0CRdeeCFWrlyJn/3sZ3jf+9435/2mBy+WZS14YEx/fz8CgUDufa4OLKwvFMUvXhxC/3gS43EVQbcDboeEuGbAsix4nRLSuoimgAMnxuO474njtp7ZbTQLzcJfdlZbTfbW5w44i1cuxRW78kiShKuuuqrWw6AK2blzJ+64447c+5FIZMZkYSF6Pn4/ROeZ302CxNUBspe6Swim6+rqwsqVK3HkyJE5b9PZ2TljNSAUCs1YNZguEAjkJQQ0v6nB5NmdPhweBmKpNIYiKZimhe4mN+KqMWuDcb3M7FZ7hraWTNPCLw4O4eREAj1NblgWIAr5s/DPHh+HUxIXvfG0Hg/5qqTFLpfiih1RRqVKhUWnJy8hILKbuk8IxsbG0N/fj66urjlv86Y3vQn79u3L6yPYu3cvtm3bthhDbAizlXR4nQ4cGghjIq7BMC2Mnf7cunb/rA3GtQpACg3y67GhtRhPHh3Ff704CFU3cWoyCVkU0eJRsLbdixavE11BF0YiKtr8LvRPJBa18bSRd8BhuVR9sSwL4XAYABAMBhdciSYisgPb/fWMxWLo6+vLvX/s2DEcOHAALS0taGlpwa5du3Ddddehq6sLx48fx9/8zd+gtbUV733ve3P3uemmm9DT04M9e/YAAG677TZceumluPvuu3HttdfiJz/5CX75y1/i8ccfX/Tnt1TNVtLR4lVwdqcfYzEVikOEYQBndfgR9JzZJrbcmd1yZuxN08KTR0fxy5dCGAwnIYqA2yHPGuTXW0NrsdelLxTFg0+fwFhcQ0fACacsIW2YCEVTiKppbO1tQsDtwHDExIWrmhHX9EVtPK3lDji1XhViuVR9SafT+OY3vwkgU26y0LbYROVa9fmfzfv541995yKNhOqZ7RKCZ599Fpdffnnu/Wzt3s0334x7770XL774Iv71X/8Vk5OT6OrqwuWXX44f/OAH8PvPBGMnTpyAKJ45c23btm34/ve/jy9+8Yv40pe+hLVr1+IHP/gBzyCooLlKOpyyBLciwymLSFg60tMOrypnZrecGfu+UBQP/v4Efv1KCIm0AZ9TRpvPCXeTNCPIr7cZ2mKvS/b5xVUdzW4HREGEKAhwyhIUr4jxuIajI3Gc1eGDU5ZwTlcAa9q8i9p4WqsdcOywKtTo5VJERFR9tksIduzYMePU2qkeeeSRBR/jN7/5zYyPXX/99bj++uvLGRrNY66SDr9LRrNHwcBkEi6HCEU6k6iVM7Nbzox9XyiK//P4cTx7fByWZaG32Q3dBEZjKuKagS3LgxiLa7kgv55maEu5Ltnnt6bVC023MBJNQfEquZPAfS4ZYzEVr0kCLlm9LDdDvtiNp4u9A45dVoUauVyKiIgWB/+CUEXMVdIhCALWtnlxciKBTJ5nQTfNsmZ2y5mxz9731GQCggA0eRVIoghJBBSvgvG4htdGM7Ph2SC/XmZoS70u2efX7XRjXbsPMVXHeFyDzyXDIYkwLSCcTGNduy/ve1WLxtPF2gHHTqtCPDCMiIiqTVz4JkQLy5Z0tHgVHAnFEE2loZsmoqk0xuIatvQ24c3rWxFO6jg+GsdkIo3NPcGSZlmLmbGf674tHgWGZcExZcUiOxs+HtegmxZU3cgFndkZ2tnYZYa21Osy9fm1eBVs7W1Cm9+FVNrEREJDXNXR4nXiTy9eYYs+iWwicnZnAL0tnqoE5OW8xiptvp+tI6EYDwwjIqKycYWAKmahko5s+U25M7vlzNhn79vqdUIWRaQNE075zOFBDklETNURTem5IL9eZmhLvS7Tn1+LV8FFq5oRTelQdQOnJpO4eFULtq9tXYynYQt2WxXigWFERFRNTAioohYq6ahEiUk5NdXZ+0qigBaPglA0BcUr5oL8tGFCFgRMJLS8evlaNLQWq9TrMtfzEwRgIpHG8mYPrt7UWfPnt5jsWLfPA8OIiKhamBBQxVW7trycGfup913T5kFUzZQ0OeVM6VAkqcOtSOhpcucF+fUwQ1vOdamH57eY7LoqxAPD7E8URVx44YW5t4mI6gETAqo75czYT73vWFxDZ8CFVwYjGI6kMqsDooiOgB9XnDMzCLb7DG25Kxl2f36LqdhrWeuzCsg+ZFnGO9/Jfd+JqL4wIaC6VOyM9vSA7eY3rcL3n8mcQ6CZFlq8CprcCrqCTgiCgEdfCWHlMs+Mx7H7DG25M/12f36LqdBrWcpZBUwgiIjITpgQUN0qdEZ7toBtTasXlmVhxTIPeprccMoS/K5MaYhlWbY7bKwYnOmfW7GB+ELXspSzCuxw2BlVj2VZSCQSAACPxzNjlyoiIjtiQkB1baEZ7bkCtmdeH8eJsQTOW5HZYnMqux02BhQfyC7GTH+9zXKXGojPdS1LOavALoedUfWk02n84z/+IwBg586dUBSlxiMiIloYEwJasuYL2Hqa3HhlKIqBcArLm2fO4tnlsDHAnjPKdhzTfKoRiBd7grWdDjsjIiKailsgUEWYpoX+8QReGYqgfzwB07RqPaR5AzanLMHnlDESVRFNzQz67XLYWDaQPTgQRpPHgTWtPjR5HDg4EMZ9TxxHXyjKMS1geiDudzkgiQL8LgfWt/swHtew99Bw0a/ZM2cVzP4acStS7nA7wF6HnREREU3FFQIqm11ni+c7XMrvktHqU/D6WAKqbgA4s9e8XQ4bs+OMsh3HtJBiZ/ILVexZBXY77IyIiCiLKwRUFjvPFk8N2KYTBAE9TW64HRJOTSYRTaWhmyaiqTSOhGK2OGzMjjPKdhzTQoqdyS9U9qyCwXAKlpW/upBNKte1+3JJ5XyvR8A+q1JERNR4mBBQyapVilHqWKaXLC0UsCXTJt56djsuWtWCyUQax0fjmEyksbknaIvmzmoFsvU6plLL0qoViGfPKmjxKjgSii2YVBabQNQzO5YQEhHR3DgVtUjqbUeWQlSrFKNY85UsLXS41J9dsgJrWu25RWexJSlLeUzllKVV89ThYs59KPfguHph1xJCIiKaGxOCRbBU/0DaoSa6kN1j5grY3raxHU5ZwquhKLyKjA3tflsFY9UMZOtpTOXuEFTtQLyYcx/KPTjO7ritKiCKIrZs2ZJ7m4ioHjAhqLKl/Aey1jPYhTa4fuyytfj4tIAtqRnYd8jeSZodZ5QXe0yVamKudiBezLkPS/XguHpsOK8GWZbxnve8p9bDICIqChOCKlrqfyAXe7Z4etmVaVlFlSxlA7a+UBQPPFUfSZodZ5QXc0yVLEuzUyC+GAfHLTa7lBASEVHxmBBU0VL/AymKAq48tx2vhqJ47sQEuoIutPmdSKXNis8Wz1Z2FXDLGI2p6J4j4ZitZKmYJA2ALYJHOwWyiz2mSpelLcVA3C7sUEJoB5ZlIZ1OAwAcDseM3/1ERHbEhKCKlvofyL5QFPsOhRBXdYQiKl4fS8CjSOht8eD8Fc0Vmy2eq+zq6EgM/eMJtPoU9LZ4Z9xvtpKlQpO0J46O4oX+sG1KimoRyC7UCL8YY6p1WRoVjt+rjHQ6jT179gAAdu7cCUVRajwiWuqOf/WdtR4CLQFL+zdzjS3lP5BTg/QVLR6c1eHHSFTFQDgJryLjbedUJnCeb0b/DT1BDE6mcHAgcjpYPdPAN1fJUiFJWl8ohu89fQKWBduXFFWLXRrh7dhYTbPj94qIqH5xC4QqWqr7js92/oAsiehqcuP8Fc3QDBO/fLky5w/MN6MviiI29QSQ1k28cCpc0OFiC+1Jn1B1jMY0xFW95mcr1IqdDpsrdq9/qh1+r4iI6hcTgipaqn8gF/O02oUOwupqcqO3xYPVy3wFHS62UJL22mgcAiysafXVzUm8lWSnw+aysk3Mm7qDtjxAjs7g94qIqD7VX61KnbHjLjHlWszeiELKrlp9Tvz3N6+CIAgLNrhmk7RTk0k8f3ISzR4FAZcDkggMRVR4nTIsC/A65z6Jt577PhZSiUb4ahzCZ8fGapodv1dERPWHCcEiWGp/IBezN6LQuuTlzZ6irqdLFjES1XAkFIMAIOh24I1rluGtZ7fjx8+dWpJ9H4UoN9mrZu8BdwiqH/xeERHVl6UZ1djQUvoDuZjNg5U+CGtqM/Qlq5thmEAklcZEQkMqbaA94Gzoxshykr2lfAgfERHRUsaEgIq22KfVVqrsaq4di5q9Cla0eHAkFMMvXwrhyo32Oh14MZWa7C31Q/iICiWKIjZu3Jh7m4ioHjAhoJIsdm9EJcquCq2Pf9eW7iXX91GoUpO9pX4IH1GhZFnG+9///loPg4ioKEwIqGTZIL1/IoFjo3EAwJpWL5Y3VyfgE0UBPU3uXFJwajJZVFJQTH382Z2BJdX3UYxSkr2lfggfERHRUsaEgMry2mhs0Q6wKrdhtdj6+KXU91GsYldklvIhfEREREsd/zpTyRazibQSX4snqRanmIRo+rUFgGhKh2aYcIgChiIq3rCc15aWPk3TsGfPHgDAzp07oShKjUdERLQwJgRUksVsIq3U11rsZuhGMvXa/rF/EglVR1TVoeoG0rqFNr8T779wOa8tERGRDXELBCpJMU2kpmmhfzyBV4Yi6B9PFH3KbSVPRuZJqsUr9Pu3rt2Pt57djkgyjcFwCppuwiVL6GpyIeBy4NFXQugLRRd59PMr97VJRES0FHCFgEpSaBPpy4MR/PTAQFk9BpVuWF1qB8VVymwnDBfTI2KaFl4ZjKIr6MJ5vU1ImxYUSYTflfk1Y7etR6t5iBoREVE9YUJAJSmkiVTVTfzsxUFoullWj0E1GlYr1TA8WxBth2C3WLMFx01uB0IxFYZp5b5/cVXH08fHcGgwjD+7eAW2r23NPd/sSk53k3vW75Odth7lIWpERERnMCGgkizUoDswmYKaNiGLBjZ0+MvqMbBrM/BSmWGeLTiOq2k8fnQUSc3AZRta4Xc5MB5XcTQUx1hcRTiZxvHRBN6xqRNv39yJde3+utl6lIeoERER5WNCQCVZqEHX6RChmyK6m9xlH1Rlx2bgas4wL+aqw1zBMSBABCAJwGujCQACnj85iaRmwOeS4XRIiKs6nnl9HIORFG7dvqputh7lIWpERET5mBBQyeY7wGpdhw8P//EUPHMEf6XU/dvl9OBqzjAv9qrDXMGxZpjQTQsBjwPjMRWptIGkZqDFq0AQBJiWhYSWWb0Zj2vYe2gYH3nLGluu5ExXLysZVJ9EUcT69etzbxMR1QMmBFSWuRp0T00m8Qt5qKKzxXZpBq7WDHMt6trnCo4VSYQsiRAgIKVn+kGCHkfu+aYNE7IowilL6ApK6AvFMBhJ2W4lZzb1spKx1CyVfpuFyLKMG2+8sdbDICIqCv/iUdlma9CtVt1/NU4PLjZQqcYM89RVh3VtXsRUAxMJDYokYl2bF30j8arUtc8VHPtdMpo9CgYmkxAAmALgkDKznZZlIZbS0R5wwe+SYVhW7vme3RmwzUrOXOzak1KvCvn5WSr9NkRESxUTAqoKO9b9z6aUQKUaM8zZVQe3Q8Szr09iIqFBN0zIkohmj4KuoLMqde1zBceCIGBtmxcnJxKQJRESLKi6AVEQEEvpcCsy1rZlyqWSqp73fO2ykjOXenlt1oNCfn64oxMRkf0xIaCqsVPd/2yKCVSmzoK6HRLWtHlxaCBSsRnmuKZjNKZiLK5BTRvwuRxwuGSkDQsj0RQiqTSWeZW8VYdKlGDMFxyPxTVs6W1Cm0/B74+NIxRR0eR2oD3gwto2H1q8ypzPtxorOZVk99dmPSjk52dNq6/hdnTSNA3/+I//CAD47Gc/C0VRajwiIqKFMSGgqrLrbHExjcGzHc7V5HZAEgW8OhyD3yVDEgUYpoVoSscyX/EzzG6HhNGYhriqoyPgzI3HKQtQvAqGIyosK3M7oLIlGAsFx2tafXjy6CgefPoE4qqONa1eeJwyoql0Xc+o2/W1WQ8K/fn5kzeIDbmjUzqdrvUQqIGs+vzPSr7v8a++s4IjoXrGhICqzo6zxYU2Bj9xdBS/ODg0YxZ0MJxCQtMRS+k4NKAibZhwSCJWLfPi/RcuLzooz4zAggBrjltkPiegOiUYCwXHb17fhs6gK5c0hKLqkphRt+Nrsx4U+vPz2micOzoREdUBJgTUkAppDB4KJ/Grl2efBVXTOg6cmIBDFrGpO4CA2wHTyqwQPPpKCCuXeYoKkmOqDp9ThqobGI6k0OxR4JBFpA0TsZQOn0vGMq8TMVXHr18ZqUoJxkLBMWfUKavQxnoA3NGJiKgOcJNkakhTG4Nnk9QMGCYwGE7NmAUdi6n47ZExhFNpjMc1vDocw4nxJFwOGRs6/Ll9+U1zrtn+fH2hKB7+4wAGwykk0ybimoGTk0kMR1JIpU20B1w4q8OPVl8mISi0BKMasknD2Z0B9LZ4mAw0qEJ+fpyyhNWtXqxt82EwnIJl5f88ZPtP1rX7uKMTEVGNMSGgumeaFvrHE3hlKIL+8URBgXh2d535ApXuJjdEQcg7XG08ruGZ4+OYSGhwOyTIogCHLGAkmsKB/klMJNJFBeXZ8p8T43G0+51wyxJ6gi54FQmKLOKcLj8uWNGEZNrEunYffC759Mzs3Ae+qbrBEgyqqkJ+fta1+9Db7MHVmzrQ4lVwJBRDNJWGbpqIptI4EorVbf8JEdFSw3VaqmulNtcWsvXkFee048fPncqVO1iWhb5QDAnNgCQKkEQRgAWXLEFxiRiPazg6EsPW3mBBQfnUxswNHX60+V040D+JuKajyaMglkrjxHgCSc3MNSo7ZclWJRiNctgU5Stm61bu6EREZH9MCKhuldtcW8juOs/3h3N79EdTOiYSGvxOGQlNh5o24Hc7oMgiBEGAzyVjPK5h5HTD7UJB+fTGzGaPA+vavHhtNI5YSodhWQhFVFywshk3XNiLde1+mKZlm0O1eNhUYysm0G+k/hNBELBy5crc20RE9YAJAdWlYrYNnS/oWChQmToL6jzd5OtVJFgWYAFo9jhyX9shiYilMjsQbVvbumBQPrUxczyu4mgojvGEhrRhwIKAgMsBl0PCe87ryQVXdjlUi4dNEVBcoN8oOzo5HA7ccssttR4GEVFRmBBQXSp028NC9jefL1CZOgv6wslJpNIGAGDFMg/iqoHk6fIhhyQirupIaAaW+ZwFBeXZxsyByQSOhGJIagZ8Lhl+l4y0YWIiriGa0jEe0+YcU7VLMGYrCQLQcIdN0dwaJdAnIlrKmBBQXSp028NKNNdmZ0H7JxK474ljODYaxxt6gphMpnOz+rFUGgkt0/j7ycvXFhSU9zS5sabNi58eGIBhmljmO3MgmSKJkCURkijg+f5JbFvbmhdcL0YJxqvDEfzw2VM4OhKDYZloditY1+7HG3qDi3rYFPsUiIiIqosJAdWlqdseLkZzrSgKWLnMi//2xpW474nj6BuJoyvowtYVTRiJqhgMJ7HM58Qnd6zDho7CZuhFUcCW3ib86A8nYVqAdvpws+zZAx5Fxvp2L46OxGcNrqs5M/url4fxrV8dwUhUhSKLcMoiokkdozENhwbDiKV0dM9RElXJZIx9ClRvNE3DN7/5TQDAbbfdBkVRajwiIqKF2W7b0cceewzvete70N3dDUEQ8PDDD+c+l06n8dd//dfYvHkzvF4vuru7cdNNN2FgYGDex7z//vshCMKMf6lUqsrPhqql0G0P56rjL2WrUuBMuc6m7iAmE2m8PhaHYVrYtrYVn37rOmzoLC5IbfM70dviQVfQhVTaxERCy509sLW3CV1N7kXfRvTVoSi+9asjGIqk0O53os3vhFuREU6mMZHQEIqkMBrTEFfTs96/mGRsvu9Dtk/h4EAYTR4H1rT60ORx4OBAOJOUhaIVe85ElZRIJJBIJGo9DCKigtluhSAej2PLli249dZbcd111+V9LpFI4LnnnsOXvvQlbNmyBRMTE7j99tvx7ne/G88+++y8jxsIBHD48OG8j7lcroqPnxZHOc215c46V7Jcx6vIaPU5EXTLAARohglFEuF3ZXYQiqbSi76N6A//0I+RqIrOgBNOhwQAcMoCFK+C8bgGRRZgWSZeG41jy3LHnDsddQVc6B9PzHmN5ipJunpTZocn9ikQEREtDtslBNdccw2uueaaWT8XDAaxb9++vI99+9vfxsUXX4wTJ05gxYoVcz6uIAjo7Oys6Fiptkpprq3U7jiVKtfJrnQcHAjnBb7A4m8jCpxp1lZkAYos5X0uu7VqTDXQ5HbAezowny0ZO6vTj+889hr6QlFMJNOQBGBtmw/XX9CLDZ3+eUuSBsJJXLOpc1H7FIiIiBqZ7RKCYoXDYQiCgKampnlvF4vFsHLlShiGga1bt+LLX/4yzjvvvHnvE4lE8t53Op1wOp3lDpkqqJjZ+kptVVpJdtlGNCuu6TAswClLSBsmnNOSAockQtPT8LscuPHiFXi+PzwjGTur049HXwnhxHgCCVVHTNWh6gZeHozi98fGcd35y/Gj505iKJJCV8AFhywibVgIJ9NQdRMA8KuXQ0imDXQvQtM4EdFcVFWFqqq596fHBYUy1fwSMkFyQJBn9r8R1UpdJwSpVAqf//znceONNyIQCMx5u7PPPhv3338/Nm/ejEgkgm9+85vYvn07nn/+eaxfv37O+/X29ua9f+edd2LXrl2VGj5VSKGz9ZXcqrSS7HSSq1eR0ex2IJpMI5xMQ/GKeddK001oemY3pW1rW7FtbWteMtYVcOE7j72GE+MJTMRVpNImfC4ZAbcDmm5gKJzCtx49AgFAd5Nr1pKkRFrHqYkEBFGwzYnMRNSY9uzZg7vuuqvsxzl17y157we3/xma3vzBsh+XqFLq9q9pOp3Gn/7pn8I0TfzTP/3TvLd94xvfiDe+8Y2597dv347zzz8f3/72t/Gtb31rzvv19/fnJRpcHahv1dyqtNytMWt9kmt2/FE1jTa/E6NxFapuYDyuweeST68MGBiKqOgMuHDdBT25sU1NnvrHE+gLRZFQdaTSJlq8Si6hcDlktHhMHB2NwyEJmcPdLCv3+WxJUjSlw++UsKo50zRe6xOZqXTcMpbq3c6dO3HHHXfk3o9EIjMmCwvR8/H7ITrP/K4UJK4OkL3UZUKQTqdxww034NixY3j00UfnXR2YjSiKuOiii3DkyJF5bxcIBIp+bKqt+QKQam1VWmiT8kLBUSX6EkoJwKaPX9NNRJI6nA4JTllCVNWh6Ro03UJXwIVPX7EeGzpm/7mIazomkmnEVB0+V34gn9R0jMU16KYF3bBwfCyBgMuBFq8Ct5JZKciWJEmihCvO6cAvDg7ZopSKiteoW8YKgoDu7u7c21TfKlUqLDo9eQkBkd3UXUKQTQaOHDmCX//611i2bFnRj2FZFg4cOIDNmzdXYYSNxU4zgAsFIFMbeCs161xok/JiBEelfI25xq+mTURSabT5nAi4HRAFYG27D9ef3zvv1qpeRYYkAKpuIOA+k3QlNR1DkRSSaQOSAAhC5rUT13RohonOQCbgn1qStH1tK7qCrkUvpbLTa7peVap5vx45HA78xV/8Ra2HQURUFNslBLFYDH19fbn3jx07hgMHDqClpQXd3d24/vrr8dxzz+E///M/YRgGhoaGAAAtLS25A2Buuukm9PT0YM+ePQCAu+66C2984xuxfv16RCIRfOtb38KBAwfwv/7X/1r8J7iE2GkGsNAApJINvIU2KZsm8MBT1Q2OSgnApo5/XZsXMdXAREKDIonY2hvEkVAMK5d5ce3WbvhdjoIC42zS9fJgFJpuwOWQYVkWxuNpaLoJSRThdACGaZ3evtSCmjYwFlfRCgVD0fySpMUupbLTa7pe2bF5n4iI5me7hODZZ5/F5Zdfnns/W7t38803Y9euXfjpT38KANi6dWve/X79619jx44dAIATJ05AFM+cuTY5OYmPfOQjGBoaQjAYxHnnnYfHHnsMF198cXWfzBJWyxnA6TO4XQFXwQFIpRp4TdPCs6+P47kTE2jxziw/yjYpHxmOYjKhVTU4KjUAyzZZux0inn19EhMJDbphQpZENHsUdAWdGImq8LscBZcyiaKA6y/oxe+PjWMokkJnIBP0x1QdFjLNw25BhglAOX0qcyptYDKhIW1Y6A7OLEmq5onMUzXyrHYl2bV5n4iI5ma7hGDHjh0zTp6dar7PZf3mN7/Je/8b3/gGvvGNb5Q7NDqt3BnAckoyZpvBbfUpeG00jhUtnoICkHJnnbNjeO7EOA4NRBB0O3ByInMycotXyd3OrUg4Nqohkkpj5TJv0cFRodep1AAsrukYjakYi2tQ0wZ8LgccLhlpw8JINIVIKo1lXqXoJusNnX785RXrc+cMAIBumvA6ZTglEUGPgtWtXoxEVUzEVSRlEYm0gUvXt+Ijl64t+rTnSuCsduVUs3m/HqTT6dzq8yc/+Uk4HGweJSL7s11CQPZXzgxgOSUZc83gvjQYwYnxBNr9LvhnOXx6tgCk1FnnV4ei+F+/7sNYXEXA7UDQJUMSBYxEU4ipOrb2NuWSgqRmQBREGJYFzxyNynMFR8Vcp1IDMLdDwmhMQ1zV0RFw5r6X2S1AhyMqLCtzu2JdcU4Helvc+OGzp3BwYBJHQzF4FQmdwUxJUYtXwaplHkRTOiYSGpKagU9fsR4rl3mL/lqVwFntyqlW8369sCwL4XA49zYRUT1Ymr+RqapKDUDLKcmYbwZ3XZsPr43EcXg4glZf64yArlIByKvDEXz5P1/OlNkoIkajKpJpE0ib6Ag4MZFI4+hIDM2eZgDAYDizahCKpIoKjoq9TqUGYJmrZEHAXEFL5nOlzodv6Ajg89f4cXIigf/z+HEcG4vhDT3BXDmfIAjwu2QMRVLY0tuE3ubaBdqNPqu9kGJW9arRvE9E1bHq8z8r6vbHv/rOKo2Eao0JARWtlAC03JKM+WZwA24HuoIuDE6mEEmmEfScKdupVADSF4rif/36KI6OxNDkccDrlJE2TCTSBqIpHYgAPpeM0ZiKwXASMdVAi1fBdRf0YN+hUMHBUSnXqdQALJE20OpzYkxA3nkDacNELJXZNnSZ14lE2ij5uomigBXLvPhvb1qB+544jr6RuC23EK3GrPZS2a2o2FW9xTh9e6lcWyIiu2BCQEUrJQAttyRjvhlcQRBwVqcfIzEVfSMxbOjwVzQAyQbpYzEVHkWC1ylDFAQop5tvNd2EbppIGyaiKR3j8TQuWNmca1IWBaHg4KiU61RqAOZVZLT6nGj1KRgKqxhPZMqHJFFEe8CFzoATgFCR0g47ncY8m0rPar86HMEPnz2FoyMxGJaJZreS2+Wq1s+1GKWu6lXz+82doIiIKo8JARVtrgA0oep4bTQOr1PGG5YH8+5TbknGQjO4LoeEDR1+rGn1YTSmVjQAmRqkj8Yyu+GYpoHxeBrJtAHdNGFZgM/lwKplXnz0sjW4cGVLLgAvJjgq9TqVEoBNDYIvWNmEmGpAM0wokgifU0LfSLyipR3lNnNXc1a4krPav3p5ONdQrcginLKIaFLHaEyrq92Kyl3Vq8aWsdwJioioOpgQUEmmB6B9oRhGYxoEWLAs4MfPncLz/eHcrF25JRmFzOCev6IZH3nLGgyeDpgrFTRGU2mMJzR0+J3wKBJGYipSmg7dBBRZhCJLiKs6xmIq3A4JTR7HjK+ZDY5OTiTw2mgcALC61Tujbr6c61RsADY1CM6W8jR5HEhqBvpG4lUp5Sm1mXsxZoUrMav96lAU3/rVEQxFUugKuOCQRaQNC+FkGqpuAkDd7FY0fbXKsixEU3ouaewMOBdstK7klrHcCYqIqHqYEFDJsgHoE0dH8b2nT0AQgDWtPnid8oxZuzWtvrJKMgqdwZVlcc4ApJQZ5r5QFA//cQBHQzEcH41BgICJmAbDshBwOSAJgGZkkqBWnxMBlwO/fCmEdW3+GY/92mhswaC23NKVYgMwu5fyAIs7Kzw9qXI7JAjI9Fv0jyfmfc2YpoUf/qEfI1EVnQEnnKd3Z8ru2jQe15BI6zgyHK2L3YqmrlaNxzX0hWJ5Z1UE3TKcsrRojdb1shOUIAhoa2vLvU1EVA+YEFDZXugPw7KALcub5py1+9hlvrJLMsoJXkuZYc4GomMxFe1+JyYTaSiyAPP0VoKqbkIQgLRhosWr4KJVLVBkcdagpNCgdjEaMme7rot5GnAxajErnE2q+kJR/OfzgwW/ZrIBqyILUOT8rVoFQYDPJSOa0jGZ1Opit6LsatXAZAJHQnEkNT3vrIqhsApRAEaiKs7urP546mUnKIfDgU984hM1HQMRUbGYEFBZipm1q8RsdCnBaykzzFMD0Q0dfrT5XTjQP4nJhAaHJAAWIMsCXKfLdy5a1YxlPid005wRlBQb1NZi1r7U0o5q7/ZSq1nhUl4zcU2HYQFOWULaMOGclhQ4JBGanoYoiHWxB39PkxtrWr346QsDMEwLy7xK7nugSIAsApIo4oX+MLavba16Atno5xsQEVUTf3PSrAoN9IqdtavEbHQxwWupM8zTA9EWr4KtvU04NBBGOJmGBQta2sSKZg/O7Q7mHUY2PSgpJai166z91NfFaFTFgROTeG00njeDfuW57XA75IqMuxazwqW+ZryKjGa3A9FkGuFkGopXzPt+a7oJTTexrt1XF3vwi6KArSua8KPnTsKyLGiGmbctrccpY327D0dHFqdMh+cbEBFVDxMCmqGY8ppSZu0q2Wi4kFJnmGcLRFu8CravXQZYQCiagiQKOLvTn0sG5gpKSg1qF/M6FWLq62I0pqJ/PAGHLGJTdwBrWn1IaDp+99oY9r40hDa/E4osFtz8O1cCWotZ4VJfMz1Nbqxr92M0rkHVjbyzHTTdwFBERWfAhesu6Kl5YleoVr8TvS0eaLqJyWQ6b1vatW1eBNwOHB+NL0qZTi3K6UqRTqfxz//8zwCAv/iLv4DDMfN1S0RkN0wIKE+xpRJ2n7UrNRifKxAVRRHn9gQQP6YjmtKRNjJnEMwXlCyFUoepr4vOgAsDk0lYlgXDtHAkFIfXmXleEwkNIzEVsiTgjauXIZk2cq+dm9+0Cm5FmhH0z5eAltOMXmo5UzkJXDZgBYCEqiOq6tB0DZpuoSvgwqevWI8NHYGCr3utZc+qCLoz39/sDkN+V+Z7EU2lF/W1Ww9N8JZlYWRkJPc2EVE9sG8EQouulFIJu8/alRqMz5foNHsUtAdcaPcDumHh+Gh83qCknKTJDieyTn9dZBpj02jyKlAk8fQONFEAQCptoDPgRFw1kNAMBNyZ184f+yfx5f98Ca0+Baph5oL+s7v8ePSV0LwJaCmvr3K2KS1369dswNoXimIymYYoAGvbfbj+/F5s6Kx9wFqMqa/dqb8TgNol/HYtpyMiqmdMCCin1FIJO8/alRqML5TorGjxzDnjPV2pSZNdTmSd/rrQTq+KOCQ5t3vOcESFAMDvdsAhCUhoGjQjs+/+REJDKJJCNKWjM7gMPc2Z8qIXT01i70tDCLgdOK93vh2q1hb1+ip3m9JyV72WUsBq14TfbuV0RET1jgkB5ZTTwGnXIKicgKaSiU6xj2WnE1mnvy4USYQsirmddBySiLRpQrAAhyQgbZiQRBGKJMKyLBwNxaEbJjyKBEWWIIkC/C4HOgMWXjgZhizNvPaz7VBVyOurEtuUViIIXkoBq50TfiIiqgwmBJRTbq27XYOgQgOa2cpzKpnoLGZQW0nTXxd+l4wWj4JQNAXFm0kMHKIIAUBaNxFTdbQHXPCf3nd/PKHB6ZBgWplkIittWlBkEbFUph8j4M5/zU1PQAt5fVVqm9KlHASXUoZm14SfiIgqgwkB5di9QbgcCwU0C5XnVCrRWcygtlJme12sbfciqqYxFlOhm0BHIHMGw6nJFNp8Cta2eXPlRWnDACwBHcFMkpClSCIUWYSqG7nyIiDzWoum9NOn4lpwO6TZhjWrSm5TuhSD4HLK0Oya8BMRUfmYEFCOXeuFK2WugKbY8hzTtHByIoHXRuMAgNWtXvQ2eyp2XaYHtdkAObvDi1uRoOqLdyLrbK+LgNuB9e0+HDwVgWlZcDkkKLIDhgkE3A44JBG6mdl3P6mZ8LsdWNt2ZrXDsixYlgVZEDCpGZBPX7pMg3IME3EVE8k0Wr0K/r/nB/D2TZ0FzcpXekenpRQEz/c6PzWZwDs2d6HV71wSiU8tCYKAYDCYe5uIqB4wIaA8S7lUYjbFluf0haJ48Pcn8LvXxhBOpGEJQJNbwRtXt+DGN66oyPWZGtSmDRNHQ3GMJzTohgnLAhRZRItPgaeImfNyzfW6uHZrD97QG0Tb6UAyqRnY99KZ2yiSiLVtPgBAsycToE8N+kdiGgzTwpOvjWPlMjcGwypiKR0CLLT6FJzV4cehgQgGw6mC+iaW8ipXOeZ7nWu6iaePj+P5/kmsWOaF27G4zet22EmrkhwOB26//fZaD4OIqChMCGiGpVgqMZf5ynMAwOeU8IfXJ/Ds6+Nociv41qNH8Hz/JCQBWOZXIEDAZCKNfS8PIxRTcfvb1pcdRGWD2t8dG8NEXEUqbUKWBCTTBuKqjmTawGBYxn1PHMMH37hy0ZK0Ql8X69rzb5NM63jgyddxJBSD2yHi8HAsF/R3Bp3oDrpxfDSOZ49PQhYFLPMpWOZzYm2bFy1eJyzLKrhvYqmvcpVqrtf5eFzD8yfDUNMmDElAm88JWRIWrXm9EjtpLbWEgoioFpgQ0KyWUqnEfOaqOc/OYo/FVUSSafz/ftMHzbDQP56EIglY5nPmAquOgIixuIZXh6N45OAQ1uwor9lXFAVcubEDe18awkhMQ5PbgbGYBk03AUFAwJXZ2vOJvjGouoX//ubF23GokNfFbLe5dfsq/OLFIfzXwSGMx1UE3Y68oL876MLPDw1hmdeJS9a0IOBy5K5vsX0TjbbKVYjZXueWZaEvFENS09HmVxBOpmFYFppdyqI0r1diJy27bM1LRFTvmBBQQ5ut5nw8ruFA/ySSmg5FFhF0O+B2yDg4MI5U2kB3kztvllUQhNyOOi+cDFek2detSGjzOyGJwInxJJKaAZdDhNsho8WrQBQz9fCnJhOLuuNQqda1+/HurSJeHAjjrA4fmjxK7rRbANAtwKPIMC0LAoQZqzXFNANnv16jrHIVYrbXebZx2+dyQDet3FaxQPWb1yuxk5adtuadKp1O4/777wcA3HLLLXA4ZvayEBHZDRMCalimmWlsDbgcODoSwxt6ghAEITdr2uxxYCKRRnvAhaDHAacsndki83TttWFZkAQht5d+Iq1XpNk3fjoZObcriMmEjmVeJ9yOzK48giDAtCzELR3NHmVRdxwqRyJtQBIF9DR7IE0L8ubacSir2GZgoHFWuQoxW2+FZpjQDROyU8Jk4sxWsVlTk7BKl+WUu5OW3bbmncqyLAwMDOTeJiKqB0wIaMmaL4iZWmowGlPRP57AYDiF1cs8GIurUGQRE4k03IqEtW1eyKdnTx2SiGhKx4nxBNKGBdOyIAoCHJIAhyTC45DnDVrnG9PUz0WSaTglEVFVhygAAbcMcUrglD38K+ByYCyuLtqOQ+WYbwcgv0uG3yljMKXDMS2As0MzcL3Xqc/WWyEKgAVgNKqe3gXKmxecZ5OwkaiKX70UqmhZTrnbw9pta14ionrHhICWpPlqiwHklRp0N7nR6lNw8FQEB09FoOomlvkUtAdceY2t7QEnRmIq4momSHErEiQhs71mNKVDFgX0NLnmDFoXGtPUzzklEaMxDSndgCwKSBsWnPKZLTtjqcyMriSi6JnzWplvByAgUzLU5ndiKKJCFAXbNAMvlTr16b0VqbQOpyxBFwRsWR5Ei9eZu202CesOuvDzF4cwkahsWU6528NW8rwJIjrj+FffWeshUI3YP4ogKtJC+627HNKMUoPeFi96mtx4+vg4RqIq3rA8iK6gO6+xdW2bD68MRQEApmXCskQYFqDqJqTTASzm2Hd8vjG9PBQBABimlfe50ZiG8bgG3bSQTGto8yvQzUwy4FYkrGn1YCiiLvrMeamz5QvtALRimQdvPbsdrwxGbdMMXE6duh1XFab3VoxEVfz8xSGMxTUospT3/Wj2OGABmEhUviyn3O1hK33eBBFRo+NvSyqL3YKehWqLnz85idGoiotXL5sxQy2KIjZ2BfD71ARCURVdwfxgRBYFyKKAFq8CC0BCMyAAcDokLG92Y02rF5OJdK5MIXttoqk0Hv7jAMZiKjZ0+PPG5FUkPHJoGBCAqzd2QBTF3OfOW9EEnMj0JYzFNJycSMLrzMyidwddGIun0eJRsHl5EK+Gooty/cudLS9kB6DLz2q3xWuqnDp1O68qTO2tOLsT6Aq6Zv1+vGF5ED9+7lRVynLK3R6W500QEVUWEwIqmR2DnoVqi5s9CvqGYzDM2Zv9PE4ZrT4F3tMB39RApW8kBrdDwvZ1y+CUJUwk0oAANLsdCLgdMCwLx0fjiGt63rUZT2g4Goqh3e9Em9+FFq+S+3ox1YBhWYCVeTvgFvPGu77Dh4m4hg9eshIvnJzEYDgFSRAAZAIyWMBDz51alD3cK7Wry0I7ANmlGbjUOnW77n4zl7m+H6+GolUtyylne1ieN0FEVFlMCKgkdg16Fqot9rtkQACiqTSapwTmWUnNQKvPifee34MX+sN5gcq53UG4HBLcigy/y4GgJ//+SVXPNWH+4uBQ7toosojjozFMJtI40D+Jrb1NuaQgs6OOBUCYdXedTNBlYmN3AO/Z2pML2kajKv7rxUFMJNKLsod7pXd1sUvQP5+FXksuh4iJhIqDA2EAyM1G23X3m/nM9v1YjLKccraHtfN5Ex6PvV/bRETTMSGgotl5yz+3Q4JhWDg5kUDztL3ugUzZT9CtYDyhobfFM2epwfa1rdi+tjUvUOkKuPCdx16bUaZgWRYiyTT6RmI4p8uPAycm866NZQEuhwynLCCu6jg6EkOzpxmCIJze9z3zONk94KeaGnRlgzbTtPCrl0KYSKQXbQ/3RtzVZb6AeDyu4aWBMEJRFT94ph/7PMNY2+bDG3qDS+Y6ZctyXjw1ic6AhbRpQZHE3Naks5XllFJCWE5yaMfzJhRFwf/4H/+jZl+fiKgUTAioaHYNDvtCUfzi4BD6JxIYi2todjvQ7HViXbsvU/dvWRiKqHjTmhYk02ZBpQbTxz+9TCGVNnB4KIrBcAqylAn+JxIazu6c2isgo8WjIBRNweuUMR7XMmcZuB3wOaVMCZAA+JxS3teaqxa6Fnu4N+KuLnPVqY/HNfzxxARGYiqWN7txblcAybSBgwNhHBoMI5bS0T1H7Xo9XSdRFHB2lx97XxrCCyfDUOTMWRF+pwyPImPFMk/ez0qtSgjrYbWJiMjumBBQ0ewYHE6d8T6704/Dw1HEUzoGJpOIpNI4q8OHZNpEi1fBn12yAgBKKjWYWqbwx/4JvDochW5Y6Gpy4ayOABKajr6RGA4PR+F1ymjxOjM7FLV7EVXTiKXSMCwgmdYhCJlZ1g2dma/XNxIvqBa6Fnu4N+KuLrPVqbscIl4aCGMkpqLNp2BjVwCyJMIviWea1mMa4moaAffsJWn1cp36QlE8+koIAbcDsiggpupQdQODKR1tfifeenZ77mfFriWERERUGPv/VSLbsVtwONuMt9cp42gojrG4irGYilcs4J2bO3H1ps5cYFJO7fKqS7342iOHkUobWNfmQ8DtgCAIiCRFNLsdiKd0HB2Jo9mjQBAEtHid2NrbhJcGIghFVQxHUmj2OHNJCFB4glKLPdwbdVeX6XXqEwkVoWhmZWBjVyBv735BELCm1YuRqIrXRuPYstxRt9dp6s/Ueb1NAIBoSodmmHCIAoYiKg4PRXH5We0A6rNvolrS6TT+3//7fwCAD37wg3A4Zv6MEhHZDRMCKprdgsPZZrxbvE40r1IQTemYSGhIpg28a0s3Vizz5u5XTqnBYCSF0dPbiE4Nyv0uGc1eJwYmkxiLqbnSIABo9iho87twwapmvOe8HvidjrwkpNAEpRZ7uDfyri5T69QPDoTxg2f6ce7plYHpMrtUOWfdpWq262S3bXuzZvuZyr6OgczrIbuKBMCWJYS1YlkWXn/99dzbRET1gAkBFc1uweFcM96CICDgdsDjlHB8NI5E2liUr7mu3YdIKo2xmIqJhAaP88y1WeZTcMOFvbOWTxSaoNRqD3c77+pSbVO/N/s8w0imDfjnaAJv9TnxvvN78Py0XaqmXyc7btubVewqkt1KCImIqDhMCBpIJWcj7RQc1qKEab6v2eJVcFaHD69YQDJt4PhovOLXplZ7uNtxV5fFVGgytW1tK7ZN26Vq6nWye819sT9TdiohJCKi4hX1G/pf//VfS/5CN910U8n3pfJVYzbSLsFhoUFaV8CF/vFERcY629e0LAvRVKbxciyu4R2bOvDurT1IpI2qXJtyrv+aVh+u2dSJX74UwqmJJCQxszVqoQlFI5R+zKbYZKpSuzwttmJXkexUQkhERMUrKiG45ZZbZvyyn14zOl32NkwIaqeas5F2CA4LCdLO6vTjO4+9VrGEaPrXdDtEnJpMYjSmIabq8Dgyj68ZJs7uDFThWZ8ZR7HXf2pymEwbgAC0B1y44pwObF/b2jCz/aUqd3WsGtv2VroXodjEx04lhEREVLyiEoL77rtvxsd++MMf4mc/+xmuuOIKvOUtb0FHRweGh4fx2GOP4dFHH8Wf/Mmf4LrrrqvYgKk49TAbmR1nOQHNfEHaWZ1+PPpKqOIJUfZrPvi7E/j14RCSaQNep4zVrV50B10YDKdw3xPHq1r+Uex1m54cdp++FoPhFH5xcAhdQVfN69frQTmrM5XetrdavQjFJD52KiEkIqLiFZUQ3HzzzXnvP/zww9i3bx/27t2Lt73tbTNuv3fvXrz73e/Ghz/84fJGSSWz6yFiU1UqoFnT6sO7toh4bTQOAFjd6kVP0I3vPPZa1RKiNa0+tPgUrFjmQU+TG05Zyp2ObFlWVROuYq9bvSSH9aLU1bFK9rxUuxehmMTHLiWEdsCtRomo3pTV5bV7927ccMMNsyYDAHDVVVfh/e9/P77yla/g3e9+dzlfikpkx0PEpqpUQDNXcLylN1jVhOjUZBKvjcSxts03I7irZsKVvW5jMQ0Bl4yAywHTtPDiqbmv22Ikh3bdRtNOKrVt72IleMUkPnYoIaw1RVHwN3/zN7UeBhFRUcpKCA4dOoQrr7xy3tv09vbixz/+cTlfhspgt0PEpqpUQDNfUnFoMIxYSkf3HMFVuQlRLRKu7HU7MZ6Arps4PhaHbpqQxdOHomn6rNet2mO18zaadlKpbXvrYfWPiIjqw8yNtIvg9/vx2GOPzXubxx57DH4/g4Fayc5GDoZTMw7Jyc5Grmv31WQHkGxA0xlwIprSMRpTEUmmc43oUwOauUxPKvwuByRRgN/lwPp2H+KqjrGYirg6e5BbbkI0NeGq9OObpoX+8QReGYqgfzwB08x8/05NJvHH/gmMRFMYialwOUQ0exS4HCJGYipCkRSeOzEx47pVc6zZpOzgQBhNHgfWtPrQ5HHg4EAY9z1xHH2haNGPuZRla+43dQcxmUjj+Ggck4k0NvcEC14VO5Pgzf79cisSVN3g/v9ERLSgsqaF3/Oe9+Cf//mf8fGPfxx33XUX2tvbc58LhUL427/9Wzz11FP4yEc+UvZAqTR2O0RsqriWSQIGJpOYTKahGyZkKRPcrmv3IeCWF5yxXmiWdE2rFyNRDa+NxrBleVPFt0Ss1qnN8822a7qJE2MJGKaJZT5n7ms6ZQmKV8RYTEX/eAKRRBr9OLPNalfAVZWxsjehNOXW3Nt59a+R6bqOf//3fwcA3HDDDZBlXn8isr+yflPt2bMHTz75JL7zne/g/vvvx7p169De3o5QKIS+vj6oqopNmzZhz549lRovlcCuO4CMRDOBq2kBzV4FDpeMtGFhJJpCTNWxvt27YEAzWxlM9jwAzTAhCkCr1wHv6cC00gmRKAq4cmMHXh2O4rkTE+gKutHmdyKVLv3xF+qruGBlc+akXJc8axLkdEgYj2v4t9+/DlU38xKKs7v8FU8OWbpSunJq7quVjFJ5TNPEkSNHcm8TEdWDshKC5uZm/P73v8fdd9+N//t//y8OHTqEQ4cOAQBWr16NP//zP8fnPvc5eDwMAmrNbjuAmKaF5/sn4ZBEGKYJRRIywawsQPEqGItrODgQwbVbuucNaKbPko7HNfSFYphIaNANExYyM+fv2NKNsahW8YSoLxTFvpeGEdd0hKIqXh9LwK1IWNHiwfkrmot+fF038e/P9OP1sTjWtflygd7U2fbnT07C5RChpk34nNaMQDCa0pFMGzgxHsdZnYEZCcVbz27HK4PRil0LuzeuL6bFbKq28+ofERHVl7LXMt1uN3bt2oVdu3YhFoshHA4jEAiwb8CG7LQDSHZ3nk09ARwJxTAe1+BzyXBIItKGCd0wYZoCtvQ2zRvQTJ0l1XQTz58MI6np8LkckJ0SRqMqdEHAC/1h3LJ9Jd7t6K5YsDZ1Jn9FiwdndfgRiqYwGE7B65Txto3tRQXYfaEo/v2Zk/j5wSFIIjAa09DiUbC23YsWrzM3235qMol2vwujMXXGdYsmdcRUHaIgoDPgnjWhODwUxUcvXYPB00F6udeCpSsZtWiqtuvqHxER1ZeK/oX2+Xzw+XyVfEhaorKzymtaffA6ZRwNxTGe0BBXdUiiiK4mNxRJRKvfOe/jZGdJT00m8PTxcahpE21+BbppYTKhw+92YMvyIMbiGn75Uggfu2xtRWZM56qb727yoCvoxpFQDL98KYR1bf6Cvl42uXh9LA5JFLDMp8AwLYSiKUTVNLb2NqHF64RbkSAJAta3+yEKAnTTxEQijZiqwzAtpNIGEpoOt0PCoYEwBk43jbd4lbzyncFIqmLJIUtXqn8ewHzstvpHRET1p6yE4NSpUzh69CguvPDCXFmQaZr4h3/4B/z0pz+Fx+PBX/3VX+Htb397RQZLS8fUWeUWrxPNq5Rc3b8iiQAshJN6QbPK69r9eMfmLhzon4QhCQgn05BEEe0BF9a2ZWbXFVmqaB17JevmpyYX69p8GI1pMMwzTcLjcQ1HR+Jo9ihIagZcDglv29iO9EETYzEVy5vdiKk6Dg9FoaYNOCQR3U1uKLKU68fIJBRKVcp3Gr10Jfv9G4tp6Aw4oeomLEuH3yVjfbuvIk3VC5Ui2Wn1L4tnUhAR1Y+yEoIvfelLePjhhzE8PJz72N///d/jzjvvzL2/f/9+PPnkk7jwwgvL+VK0xMw2qxxwZ8pNsif8FjOr3Op3YuUyL9p8ThiWBUUS85puKx0IV7Jufmpy4XPKaPYoGImmoJye1fe5ZIzHNUSSaQxHVWzuCWLb2lZ0Bl145OAw+kJRvDYSRyptYuUyLyaTaSiyCKcsQvEqpxOKGJo9zVUp3zFNC05ZwmUb2vDs8cx2qMMRs2FKV7LbwE7EtbwzIbLlXuU2Vdfj+Q71OGYiokZWVlTw1FNP4W1ve1vumHbTNPHtb38bZ599Nvbu3YuhoSG87W1vwz/+4z/i+9//fkUGTEtDpWeVvYoMt0OCLAlodikzPl/pQLiSdfNTkwtBELCu3YeYquf6AyRRQCqto28khpXLvLnrki0Vefb1cXxn/1Es8zrREXDhD69PIBRNQfGKcyYUlSrfmR74OSURbX4nLlzdgnM6AxWfFbbjrPPLgxG8OhSFQxLgdzvgkGSkDTNX7rW5J1jyeQC1LEUqVT2OmYio0ZUVHQ0ODuJd73pX7v3nnnsOo6OjuOuuu7B8+XIsX74c73nPe7B///6yB0rVU6sgq5INkYtdx17Jrzc9uWjxKtja25TbLSmVNmCYwMbuAG64sDfvuohiZmXF6ZDQHnBBFAWsbfciqqanJRTGjISiXHMFfv0TScS1Eaxp9c74OuW81io161zJ17tpWnj2+AR000KLV4FTlgDkl3sdHo5iRbOn6GS0Hs93qMcxV5qiKHmr5ERE9aCshMAwjLx9ln/7299CEAS89a1vzX2sp6cHQ0ND5XwZqqJaL+1XqiGymBWHSgSElVzhmC25aPEquGhVMyLJNPpGYji3O4j/cdVZkOWZh4vPTCic2NrblGvUTqV1GCZwbncQ779weUW+r6UEftnXWl8oiolkGpIArG3z4foLerGhc/4xVWrWudKv91OTSYxEU+gKuhA+XaqVvRaCIMDrlDA4mcIFK5uLTkbr8XyHehwzERGVmRCsWLECTz/9dO79hx9+GF1dXTjrrLNyHxsaGkJTU1M5X4aqxC5L+5VqiCxkxaGSAWGlVjjmSy6GoypWLvPi/RcunzUZAOZKKDKN2tmEYmN3YM6EohTFBn7Z19qJ8QQSamZrVFU38PJgFL8/No6/vGI9rjinY9avValZ52q83uOaDtUwcVZnAC+eCs/YBjaWMiBLAi5c1VJ00lmP5zvU45iJiAgoKzq47rrr8MQTT+D9738//vzP/xyPP/443ve+9+Xd5uDBg1izZk3Bj/nYY4/hXe96F7q7uyEIAh5++OG8z1uWhV27dqG7uxtutxs7duzIHYY2nx/96EfYuHEjnE4nNm7ciIceeqjgMS1F04Msv8sBSczsV7++3YfxuIa9h4Zhmlath1qUde1+fHzHWnzmyg349BXr8ZkrN+Bjl63NJQP3PXEcBwfCaPI4sKbVhyaPAwcHwrjviePoC0Ur+vWKfZxbt6/Cpu4gJhNpHB+NYzKRqT9fKFDNJhQtXgVHQjFEU2nopomYqucSihsu7K1YMgBMDfxmn1NwK1Kubj77WjsxnsBEXEU4mYZbkdDmd6HNr2AoksK3fnUErw5HZn2sYpKPuRTzejdNC/3jCbwyFMmcpD3Pz0B2dcblELG1twltfhdSafN0qZeJJq8DGzr8OKczMO/YZvt6U1d+ZmPH8x3qccyVpus6/uM//gP/8R//AV1n4kNE9aGs38qf/exnsXfvXvzoRz8CAGzevBm7du3Kff7ll1/GM888g507dxb8mPF4HFu2bMGtt96K6667bsbnv/a1r+HrX/867r//fmzYsAFf+cpXcOWVV+Lw4cNzHob21FNP4QMf+AC+/OUv473vfS8eeugh3HDDDXj88cdxySWXFPekl4ilvLQ/24pDJWubZys5qtQKR6nlU4t9QFUxTdWnJpPoC0WRUHWk0mbuTAQAcDlkdAYEjERV/OgPp/DXb595bkMlZp0Lfb0/eXQUz/eHC15Bmro6s77dh4tWNee2z3WIAoYiKbxhedOc5ULzrVitafWV3KdSq74gnkmR2VzjpZdeAgBce+21NR4NEVFhykoIAoEAfve73+HgwYMAgHPOOQeSJOU+73a78dBDDxW15eg111yDa665ZtbPWZaFe+65B1/4whdyKxEPPPAAOjo68OCDD+KjH/3orPe75557cOWVV+YSk507d2L//v2455578L3vfa/gsS0ljba0X6kEqNo9F+WUTy3mAVXFBH6vnu4ZiKk6fC55xvVXZBGKLM55/Suxo1Mhr/e+UAwPPn0CloWCS4pmK/fyOCUIGjAYTmGZzzlnL0khJUyl9KnUsi+o0c+kICKqVxWpIdi0aRM2bdqUlwwAwKpVq3Dttdeip6enEl8Gx44dw9DQEK666qrcx5xOJy677DI8+eSTc97vqaeeyrsPAFx99dXz3gcAIpFI3j9VVct7AjbSaEv7xZS4zKUaJUeVlk0ozu4MoLfFU7XAa64ypWgqjSOhWF7g51VkSAKg6plD06ZLGyYUWYRpmbNe/2zyMRhOwbLyy3eyyce6dt+8s84Lvd4Tqo7RmIq4qhddQldKuVehJUxrWn1FPfbU12jQLaPV64RpWXj6+Bj+z+PHFuU1Wk75G5HdqKo6IxYohakm8v5ZerrCIyUqT11Fe9ndijo68psPOzo68Prrr897v9nus9DuR729vXnv33nnnXklUfXMTkv7i1HeUO4sM7dTnKnQMqXsa+3lwSg03YDLceYaW5aFWEpH0ONAk1uZ9fpXYtZ5odf7a6NxAALWtHpLWkEqdnWmmBWrQh976mt0mVfBK0OZbWt1w4QsChgMp+B2SPjCOzdW/TW6mKtVRNW0Z88e3HXXXWU/zql7b8l7P7j9z9D05g+W/bhElVJ2QvDkk0/iH/7hH/D888/j1KlTszZRCYJQ0eaq6X9ALcua8bFK3Ke/vx+BwJlmQKfTWeRI7csuS/uLVd5QbgJUiZIjOx6qVa5CAj9RFHD9Bb34/bFxDEVS6AwIUOTsLjw6XA4JHoeM9R3+Oa9/uT0SC73efU4ZlgV4nTOTRWDhErpiv7fFluwVUkqWfY26HSKePxlGUtPhczngcMlIGxYm4hoefSWEHWe34y3r2+Z9rEqo1O5hRLW0c+dO3HHHHbn3I5HIjMnCQvR8/H6IzjM/D4I0++8aolopKyH4t3/7N9x8882wLAtr1qzBxRdfDFmu3qJDZ2cngMyMf1dXV+7joVBoxgrA9PtNXw1Y6D5ApkdiakKw1Cx2I+p0i7ntabkJULk9F7U+72E+5SYqhQR+Gzr9+Msr1uNbvzqCkaia6xsIehzwOGSsWOZZMAEtd9Z5vtf75uVBPPTcqZJWkEr53lbypOusuKYjmdYxEU8jqel5zdtOWUCbX8HJiSR+9fIwtq9trftklGgxOJ3OikwGik5PXkJAZDdlRe9f/vKX0dzcjJ///Oe46KKLKjWmOa1evRqdnZ3Yt28fzjvvPACApmnYv38/7r777jnv96Y3vQn79u3DZz7zmdzH9u7di23btlV9zHZXq6X9WpTglJMAlRPA2eW8h9ksZqJyxTkd6G1x44fPnsLRkRhMy0STW8H6Dn/BCWi5s85zvd4B4IX+cNErSKV+bytVsjc1mYsk0zAMCyMxFX6XY8ZKlm5a8DplDIZTdbl7GBERVU9ZCcGJEyfwoQ99qKLJQCwWQ19fX+79Y8eO4cCBA2hpacGKFStw++23Y/fu3Vi/fj3Wr1+P3bt3w+Px4MYbb8zd56abbkJPTw/27NkDALjttttw6aWX4u6778a1116Ln/zkJ/jlL3+Jxx9/vGLjrme1WNqv1banpSRApmnBsiwEXA4cHYnhDT1BiOKZ5tj5AjjTtPCLg0M4OZFAT5MblgWIgj16D2qRqGzoCODz1/hrWjo11+u92BWkcpLaSpTsTU/mnJKI0ZiGyUQazZ78pDXbq9Hmd0IShCWze5gdORyO3I52DgfLQoioPpSVEKxatQqaplVqLACAZ599Fpdffnnu/Wzt3s0334z7778fn/vc55BMJvGJT3wCExMTuOSSS7B37968MwhOnDiRF7Bt27YN3//+9/HFL34RX/rSl7B27Vr84Ac/aNgzCOygltueFpMATQ26RmMq+scTGAynsKk7gK4m94IB3JNHR/FfLw5C1U2cmkxCFgR4nDK6m9xo8znRGXDW5LyHWjZJ27W2vNgVpHKT2nJWrOZK5k5OJKHqBgbCKbT7nVNOTNbhViR0B10AhCWze5gdCYIARVFqPQwioqKU9VfhYx/7GO6++26Mj4+jpaWlIgPasWPHjK0FpxIEAbt27Zp3t5/f/OY3Mz52/fXX4/rrr6/ACKkSqlFDXWnTg67uJjdafQoOnorgjycmMRrT0OpzzhnA9YWiePDpExiLa+gIOGGawFhMxcnJJPpCMSzzKejwu6DI4qLP2C7lg+kKNVvvRDErSJVIaktdsZormXvjmhaEoipiqo6kqiMOQBJFtAdcWNPqwVg8XbODwZZiUz0R0VJRVrR12223oa+vD9u3b8cXv/hFbNmyZc4m3BUrVpTzpWiJsdO2p7OZK+jqbfGip8mNF06FsbrVi1u3r0Zv85n9/rNBT1RN4+HnTiGW0tHsdkDTLYzHNaQNE15FhqYbSKUNDIaTEAUBo1EV6Jx7LMUGUgvdp9hgthrBXC0DxIV6JwpJgiqV1Ba7YjJfMieKIi5Y2YTnTkyi2etEZ9CFgMsBSQSGImrNDgazc1N9pem6jv/8z/8EAPzJn/xJVTfaICKqlLJ/U23duhX/9m//hptuumnO21R621Gqf3bZ9nQuCwVda9t8mEykIQpCboxTg57xhIqjoTjafAqcDgn9EwmYpgWPIkMQAAgS1LQJRRLhkEU83z+JbbPs/FJKIFXIfYoJZqsRzNUyQKxU70StktqFkrmuJjdWxDSsafMiktQxFlcXdfew6ezcVF8Npmni+eefBwC84x3vqPFoiIgKU1ZC8O1vfxu33347HA4HLr/8cnR1dXE2ZAmp9gxurbc9nU+xM+jTgx5FFnF8NIFwMg3DAtKGBVEQYFgWJACwANUwsUyWsKk7gKMj8RnlOYUGUlO/T6NRFf/14iAmEul571NoMJvUDDzwVGWDuVoGiJXsnah2UjvXz9/0ZM6yLERTOjQjk2ACFlp9Tty6fTXE0w3EtSrR4YF+RET1oazo/Rvf+AZ6enrw5JNPYvny5ZUaE9nAYs3g2vVE02Jm0GcLeiwLcDkkuGQRY3ENogB4FRFJzYRxukfGfToZ6Gpy4/hoPK/WvNBAyjSBfS9lvk/JtIHXx+LQTQsXr2rJjXuu4GuhYPZtG9ux79D8Y3jk4BCULSISaQMehwQLQDJtFHSabi0CxEr3TlQrqZ3v529Nqy+XzGm6iaMj8bwTiU0Ab1nXmlfKVknFTBSwV4WIqD6UlRAMDQ3hox/9KJOBJWaxZ3DtuOtMMeUgswU9fpeMZo+CkWgKPqeE8bgGTTeRbZfXTRNBtwyvU5611nyuQCo7G+yURTzRN4pXBiNImxa6gi74DBmvDkehGxaePxnG1t4mtHgzu53MFnwtFMw6ZWneYM7tEPGzF4fwwqkwkpqB0ZgKQECrT0GrzzlrAlnNALGQQLUau1tVOqkt5Ofv6k0deHkogv2vjkAUBDR5HFBkAZFEZkVqOKritdFYxVdaip0oqOVuYkRUvFWf/1nFHuv4V99Zscei6isrIVi3bh0mJycrNBSyg1rP4NpFMeUgswU9giBgXbsPMVXHRExF2jBhWhY8igTBBBwOCYIg4ED/BJq9TrxpzbK8WvPsY7odLkSSaWiGiYSaaUKeTKaR1g2Eopkm0besb4Pf5cBoTIUgAG1+BZOJNI6OxNDsac59D2cLvuYLZl8ZiswZzI3HNRwejmE8rqIr6MJYXEVC1WFBgCAgsxvTLAlktQLEQgPVau1uVamkttCfv4+8ZQ3a/U64HRIEAUhoOiRRRHezJ7ebUKV/TkuZKKiH3cSIiKjMhOAzn/kM/uqv/gqvv/46Vq5cWakxUQ1xif+MQstB5gp6WrwKtiwP4tFXhiEIgCQK0HQTXpeMVq8CSRQxFElBFkW87Zz8WvPMTkQmfvfaGOKagYSmI5LUIQpAeyDTo2ABMEzkVgMUSYQsitBNCz6XjPG4hmhKh98lI5rST5eVWHA7pLznOVcwO9fzsiwLfaEYYikdQZeMiYQGNW2iPeACkEkWhiIqLljRhL6ReF5gWo0AsZhA1e67WxX68/dc/wQmE2lsX9cKALn+Ab8r85wUWaroz2mpEwW1ut7c4pSIqDhlJQRr167FZZddhgsvvBC33XYbtm7dOue2o5deemk5X4oWCZf48xVSDjJf0COfvt2Gdj9WLPNgMJxCQtWR0k1IgoU2vwKXQ8JkUoNpWrnHTWoGRqIqhiIpdAVciCYzpyVbgoCxmApJFOCURXQGnJhMZlYDLljRhBaPglA0hSaPA7ppYiSm4pWhKCbiKiaSabR6Ffx/zw/g7Zs6Fywnmet5RVM6JuIqBFjwuRTEVR0+15nPZ5ORmGrMSCArHSAWG6jafXerQn/+xuIaUrqBbqcb0ixjrfTPaakTBbW43o20xSkRUaWUlRDs2LHjdAOlhb/927+d8YdiKsMwyvlStEi4xD/TQuUg8wU9fSMxyKKIzcuDaPO7sLbNh2hKx0hMxcBkEjFVx/FoHN/ZfxTPrJjINY3ue2kYAbcDhmliIqEhkTbgckiwYCGuGnA7JPhdMnTLygvA17Z7EVXTGImqMC3g8FAUmm5CgIVWn4KzOvw4NBDBYDi1YC/IXM9rIqFlkgufgq4mN14ZisAhnTkZ3CGJiKmZXW+aPI68wLTSAWIpgaqdd7cq9OdvmVdZ1J/TciYKFvN622GLU4fDgc9+9rO5t4mI6kFZfy3uvPPOSo2DbMLuJRV2NVfQc253MLPb0OkyHUEQoJsWXh9LIKnpUGQRQbcDy7zOXNByzaZOHB2JYX27D2nDgxdOhjEe1wBYEAURfrcMt0NCs1tBJJXOrQZoholWnxNblgfx+2MTCCfT0HQDTR4Fy3xOrG3zosXrhGVZBfeCzPa8dMNCqzeTXATdCvpEEWnDhFPOPMe0YUIWRSiSOGtgWskAsdRA1a67WxX683d+bzOeOTaxaD+n85WPzVeOlrUY19su/U+CIMDr9Vbt8YmIqqGshOCuu+7Cn/7pn+LBBx+s1HioxmqxxL9U6n1nC3q6Ai5857HXcoEbAPSFYkhqOpo9Dkwk0mgPuNAZdKETwJFQDL96OYRk2kC3kikHOW9FE2KqDocowOWQIEsCJpNpdDe7kB4zMRJVIUsiRAGIptIYi6exsSuA0biKdp8TTR4lV1sOFN8LMv15uR0S/r/nB3BoIIKeJilXpqR4M6sEsZSO9oALPqeEvpH4rIFppQLEcla07Li7VaE/f7IsLurP6WyJynhcQ18oVnA5WrWvN/ufiIhKV1ZCEAwGsWLFikqNhWxisZf4l1K972xBz9TAzeeUMBZXocgiJhJpuBUJa9u8uQCmK+jCqYkkcHrnGL/LgYDLgQ6/KxN0yyK00zPwbT4XmtwOPH1sArIkYiymwuWQsbkniHUdPjz8x1PoafbMW2MeTaXRP57IBfoCgMQs5whMf15v39SJwXAKfSNxdAadmExqODWRgAUBPqeMDr8TfSPxeQPTSgSIS3FFq9Cfv8X8OZ2eqLgdIg4PZxrLSylHqwa79D/puo5HHnkEAHD11VfzsE4iqgtl/aa6+OKLc0e009KyGEv8dqj3LUS5KxhTA7fnTkwgkkwj6HagPeDKlfFkuRUJkpjZSWgwnMoFudnegLGYCt0EOoMuABbG4mlctKoFb9/ciTa/Mze+U5NJ/EIemnfmXNVNPPzHAYzG1NP/MmVJrT7nnOcIzPac/tg/gYRmIKoasCwLumni8HAUb1yzDDdesqKq30O7NwmXqtCfv8Usfcp+z3/x4hD+6+AQxuNqptytxHK0SrNL/5Npmnj22WcBAFdeeWVVvxYRUaWUXTJ02WWX4YEHHsDNN99cqTGRTVRzib/a9b6VKkOq1ApGNnB79vVxfGf/USzzOtE5S2lDUjPgcsi44pwO/OLgUC7IDbgdWN/uw8FTEZiWBacsIpzU55wNnjpz7lUkxFQjtzWlzynhSCiGSDINWRTgUSSMxTXE1cxs79g85whMf07muRZeDUWxvNmNC1c2w+2QEFUzNeWp9OJsJGDnJuFyFPrzt5ilT+va/Xj3VhEvDoRxVoev7HK0SlqKq0VERIulrIRg79692LFjB/77f//v+Pa3v42LL74YHR0ds9ZvfulLXyproFQ/CgnGq1nvW6kgvtIrGKIo4MKVLXhmRaYZtHPa56cGLdvXtqIr6JoR5F67tQdv6A3mrQbMVY6TPc32kUPDMCwLgAVAgChk/rX7nVjf7sOzr09CTRvoCGRWKuY7R2Aq07Sw71AImm7i/BVnDkBr9ipocjvQNxLDvz/bj89ddTZkWUQhSk3k7NokvBQl0gYkUViwHG2xtyZeqqtFRESLoayEYNeuXbm3n3vuOTz33HOz3o4JQeMoNBiv5om1lQjiq7WCUUzQUrEgV0AmF0DmfmnDRFo3cW63HzHVwERCg8/lKOgcgalmS+rG4yqOhuIYT2hIpnW8PpYALAE3XLR8weteTCI3V+LAZtHilJKA2aU0ZzZLdbWIiKjayvqN/etf/7pS46AloJhgvBpBRSWD+GquYBQTtJQa5GavhWFauHpjR17JUCqt49eHRzAwmYLLIUE3TDhcZ66zQxIRn+McgammJ3XjcRUH+ieR1Az4XDK8TifGYhpeGgzjvif0eZOxYl47S60RvVZKvY4LleYMTKawYpkbUTXTsL7YKzVcLSIiKl5ZCcFll11WqXFQnSs2GK9GvW+xQfx8s6PV3rGk2kHL1GshiiIC7vySHa9TxkhMxfJmN2RJRNqw4JTPrCBI85wjkHuMKUmdzynjaCiOpGagxatAEASoeuYwtXVtPgxH1XlLjwp97bw2GquLRnS7K2clbb5Vrmxvim6a+J+P9tUsWeNqERFRcbgfGlVEscF4qfW+lQriF5od9TgkGKaFUxOJGY2TQGXKIqoZtMx1LSzLggULHkXCYDgFWQSaPQpGoikoXgVAYecIAPkzxR1+J8YTGnynr5NlWbnHCbgdEEWhqNKjrKmvnf6JhC0Onqp3lVhJm22VS9VNRJJpBFwOrGjxwKPIFU/WlsqZJUREdsOEgCqilBn1Yut9FwriCy1DGomq+MXBIYzFVPhdMgIuBwzTxIunJjEQTuKtZ7fj5YEI+seTZ7ZW9Dqxtv3M1oqLtWNJqQHQbNdian1/JKkhqRn4/fEJrG3zIpKSMBxRAVjwuWR0Bgo7RyCb1PWNxJBM6/A6nVB1A7GUDrciY21bJuCcb0Wl0NfOsdE4D56qgEqVw01d5Yqm0nj4jwOQRQEbOvxVSdbqpVTM4XDgtttuy71NRFQPmBBQRZTaE1Bo6UwhJQ5rWn0LliFt6g7g+f5JnBhLQDdNHB9LQDdMyJKIZo8DIzEVB0+F0RV04exOHw4PZ2bMT00mEE5pOKvDj2TaXJQdS2YLgNa0erF1RRNaF9hlaHpJ1kRCy9X3e50S3A4ZTR4Fad3E0VAcy3wKLEuGAAvLvE4AQkGNmNmk7t+f6cfrYwmMxTS4HNLpMxZ8aDm96jDX9980LUSSaahpE8ORJLqC7lm3YnXKEgDY4uCpelfJcrjsKlf/eAKjMRXdTTO/f5VI1urlzBIg83ybmppqPQwioqIwIaCKKKcnYKHSmUJLHD52mW/BMqQtvU24/8njCEVTMEwLPpcDDpeMtGEhFFURTqYhADivtwlBjwKv04G+UAwTcRVjMQ2HrSjesbm76rOSswVAA5MJ/PSFAfzouZPobfHMe3jY1Nn7V4djGImmkFB1+FwOxFUdHqeMrb1NaHLLeOFUGKtbvbh52ypIgjDjpOKFVinWtfvxuavPBiDgpcEw1rX5EHCf2bVoru9/NuHpC0XRP57Ay4M6Vi7zYF27P5dITL3vmlavbXe3qSfVaOivZs9Ntc8sISIiJgRUIdXcA7yYEoeFypA0w8SJ8QQMw8Iyn5J7PKcswDodtMiiCM0wAQAtXgUXrWpGNJU5bCupGfiTLV1Yucxb/kWbw2wB0Hhcw5FQHIZpwbIsaIaJoNsx7wzp1Nn7QwNhSKIAVTdnzN6vbfNhMpGGLIozErNCyzRkWcQNFy3HfU/oGI6qEEVh3u//9ITnwlXNeO7EBF4bjWM8ruH804ecTb3v8mYPD56qgGKT90LK1qq5FWk1d/yqBsMw8Ktf/QoAcMUVV0CSpBqPiIhoYUwIqGKqtQd4sbOP85UhPX1sDEnNyGsStiwLmm4injZgWRYM04Smm7nHFwQBAbcDHqeE46NxJKt8Au/0AMiyLPSFYkhqOpZ5FWiGiclEGgCwvt037wzpunY/3nN+D14NRdEZcMPtkGY0SM81e1tsmUah3//ZEh6/y4GLVrWgbziG18cTeO71CZzT6ceKFg8uWNmcKxniwVPlKyZ5LzQhrOYpwdXe8avSDMPAU/9/9v48SNK8vO9FP++ee2XWXtVdvXfPvjEzDMMgFrFIaCxLN4wdRrYDWWH5SMb2tQmFQoTlEIrje5BPEOfqElJg6VgCJK6wbiAJy0YgYBBgBhhmYZae6Zleq7u6tqyq3Jd3/90/3szs2vclq+f3iZiYruqqzF+++Vb1s3yf7/ODHwDw7ne/WyYEEonkUCATAsmushd2mtupPq4lQ0pZOnFDw/ECUpaO7YUU6i5NL8APQmwvxNCUVYP+/ZKkLA+A2t2J9vKwxXsCFldIbxYbKIqy4rqnLYPehEXC1Na9fglDY6LQoO76xA2Nr52f2bJMYzPv/1oV396kxaMnTY72Jpgo1ImZOvmKzZdfnORr52c6gahcPLVzNpO8bSUh3MsOYTcvQpNIJJLbBfkbVLLr7Lad5m5WH9Mxg2N9CW4WG8yUo6piGIKmKSgK6JpCKODSbI3epLWqln2vJSnLAyA3CJcsD1u8JwCiCunlfI0//t44FdtbUcndzLD1SE+Mv35xiqvzdWw/IAgEE8UGdw6ntyzT2Oj9X6/iqygKuqYwVbbRNZVzQ+lV7St/9d2nV0061pK3vFntKtd73eslb+vp9pOmxsuTZb7ww+v88ydOMpZLdDZr70WytpfdB4lEIpFEyIRA0vXsZvXxSDbOQ2M5HD9g3K7j+FFHQAiFZKt7EApBqRlt2H3sZC+2F+6rJGV5AGRqamd5mKnd2hOQbiUI06UmE4UGihLNA6xWyV3v+mmqQr7qdBKDhBnnZrHBQt3ljdkqSUunN2ktOeNOZBrrVXyFEFycqeIHgjMDqc7frzZAvtl5hztH0rw+Xe16u8rdZjNyn7WSt7W6OG3r2tmqzaV8jemSzf1Hs53H3IsO4V52HyQSiUQSIRMCyaFgt6qP7eDi4myVK6LOSE8MU1cRIThBSMLUOdmfYLLYJF9xuDBdIZew9lWSsjwAGs5Y9MR1ZsoOugoJS+f0QBJFUQjDkPOTFQxd5f4jPahq1DVYGUCfXvX63Tvaw0LNYbpiL6kE5xImubhB3fa5MlcnlzCXBIY7kWmsV/GtNL0oMclGC80Ws15nYi15yw+vLvBXL04y0hPj7ODqydJm3tPD1mHYqU3nal2cQt1ZYl0LUWK4/DG30yHcjJOVlIpJJHvD+O88edBHkHQBMiGQHBp2q/p4ZjDNkw+McGGmggI4foiuqgwtct85mktwYbrCP3p0jHtHe/Y9AFweAFm6hqqApqqcHYxsPau2x5W5Gl4geOjYrWSgzWoOTMuvXygE/59vXlpRCU7HdHJJi6lSk4WaQ9X2OwH6TmUa61V8L8/V0FWFO4ZWSpVg9c7EWvKWlKXjhyFV22MgZXaSj63aVR6WhVhtdsOmc3kXRwjBlXydphvQ2xpsNzSVXGuL906sPzd7ffei+yCRSCSSCJkQSA4VuzWfcNdwhntGMuiagqlrmJq6xH3H9gJyCZN7R3sOzMpweQA0V3V4aaLE1bk64/N1LF3jZH8SIWA0u/oZlwfQy6/f6zOVVfX8iqJwZjBFxfZYqDkUGy4Ja/dkGmtVfO8Z7SFmaMSM1Z1ZVutMrCVviYaxPfqSJsWGtySp2axd5WFaiNVmN2w6l3dxqrZPoeGSasnUFsvWdmL9udXru9vzSRKJRCKJkAmB5E3JkWycM4Npzk+VObtsO243DSouDoDuHIYnTvevWuHfrgPLenr+3qTJHUMpXhfQ9IJOErJbMo3VKr4jmRh/8N2rWxogXWtIuT2M3ZMwom3IfkClGX3e1FTiporjB2vOQRzWhVi7YdO5vItj6SpeEGDqKoW6S9zUOT1w65psZ6bksF7fxawmdTIMg1/91V8FwDBW/kxKJBJJNyITAsmbkq0MKnaTfnx5hTQMxY4cWDZycGl6IU/eN8zPPjC6YoPxXrwe2PqugbWSmvYwdtMN8EPB6zNVGm5kL6trKklTI5c010yWDttCrDa7ZdO5uIvz8mQJ2wsBf4m0bquPuZjDen3brC91Gjzo40kkEsmWkAmB5E3LWrKVe0czPDCWxQ8F37s0x4s3Sh07zm7Tj+/UgWUz3/9T9w5zbA83My9nqwOkayU16ZhOLmFwcbaKrqqoikc6ZmDEdDw/5GapiR8Kmu7qi+YO20KsNrtp09nu4twsNvjj741zbaG2ZHh9O4/Z5rBeXzicUjKJRCJZD5kQSN7UrKXT/8sXJpmvOUwUGhi6yr2jGU71p7ryH/2dOrBs9P2n+lOdhWX71SHZygDpekmNpigIFPxQkLI0DE3BC0Jqjs9AyiQTN/jmhVnODK6UpRzWhVi7bdOpqgrH+pL808eP8dmnx7k8V98V68/Den03lDrNVvirr3yDR07keOc73yk3FUskkkNBd/2mlUgWsV9SnbZs5XK+2tnOO5yJMVVqIoQgCAWX8nWSlkFv0uxKffNOHVjW+v6r8zU+8+0r++aws933fM0h5SM9BELg+iF1N6DhumiqymAmxumBJIamrilL2cuFWHt9b++FTeduP+ZhXTi2kdRpOG3SeOkVvnsDnnjiCZkQSCSSQ4FMCCRdyX5bPS6v+lVtn1LTI5s0MbVokPLKXI1cIte1+uadOrAs//79lkXs9D1fLamp2h4TxQYn+5I03KAzUNx2x/HDcE1Zyl4txNqve3svbDp38zEP68KxzUidmvt8JolEItkpMiGQdB0Hoc9dXvVzgxA/DDG0KHBMxXQKdbdjXdnN+ubdYL8dYHbrPV+e1EwUGsR0jaYXrFh0BhvLUna7Kr7f9/Ze2HTu1mOGocDSNd51boDnxovMVW1mK2HXLxzbjNRJIpFIDhsyIZB0FQdlRbi86mdqKrqq4gVRgGJoKjXHxw1CoHv1zeuxFZnKfjrA7OV7vhuylN2qit8ONpu7xfIuiaWpDKQtHjnZy13Dma5eOLbRPTVbsRk6wPNJJBLJdjg80YzkTcFBWREur/qlYzq9CZN81cZMRomBrqqYmtrV+ua12KpMZT8dYLb7nm8mwdktWcpuVMUPu83mbrFWl2Si2KTuznGqP9m1yQBs7p6SSCSSw4ZMCCRdxUFZEa5W9Ts9mKTqRJt6/RCGe2KA4FK+1rX65tXYjkxlPx1gtvOebyXB2YsB2+1wmG02d4vbpUuy3j31k+d6+fPzB31CiUQi2RoyIZB0FQdlRbha1S8TNzg7mOL8ZIVQCCxdpdz0u1rfvJztBmCrJUhCCKq2j+MHTJaavPVE7650SLb6nm8nwdmLAdutclhtNneT26lLstY95fveQR9NIpFItszt+y+P5FBykFaEa1X9fu7BI9w/1sNA2tqTQHIvLSh3EoDdP9bDq9NlXrpZojdhMlVuMl9zqTk+CSOqyF+dr+04MdrKe76TCvNeDNhuhb2+t/fqPtrNx73duiSr3VO6rvMv/sW/6PxZIpFIDgPyt5WkqzhoK8L9riTvtQXlVgKwduD36lSZ77wxx2zFpub4zNccXr5ZRlMUehIGJ/uTjPbEmC7bfPbp8R0742zlPZ8oNA5thXkv7+29uo92+3HfDF0SVVU5cuTIQR9DIpFItsTh/a0ruW05aM33flWS98OCcrMB2HzV4anX8nzv8hyvTVVwgxBLV+mJG7hBiBDQk9S5/2gPY7lER0K0W5rvzb7nh73CvBf39l7dR3vxuId1GZlEIpHc7siEQNKVdIPmey/Zr+HKzQRgIz0x/uaVaW4Uosq1HwqSpo4fhpSbPm4QVYY9XzBbcRjLRcnSblfkN/Oe3w4V5t28t/fqPtqrxz3oDuB+EAQBP/zhDwF429veJjcVSySSQ0H3/qspedNz0JrvvWS/his3DMASJggo1F3qjofjBSStaO+CKaLdC2EIoRB4YUih5nSWs8HuV+Q3es9vlwrzbt3be3Uf7eX9edAdwL0mCAK++c1vAvDoo4/KhEAikRwKZEIgkRwA+yl9WS8Au+9oD3/1wiTpmM4bMx6qoqCrKgCKAqauYnsBCgqOH2L7QWc5G+x/Rf7NUGHeCnt1H+31/Xm7dwAlEonksCETAonkANhv6ctaAdjFfBXbD8jEDIQATVUIhEBvVYUNVUFVFXwRogSgoGBqUcJwUBX5bqww76VT1Hrs1X20H/fn7dwBlEgkksOGTAgkkgPgIKQvqwVg7cAvCENihortazh+gGZoKAqEgKVrKIDtBaQsnbipUbW9NSvyexkctx/bDwV/74ERFKDhBQdaYd5rp6j12Kv76HaRZkkkEolkc8iEQCI5ALpF+tIO/F6ZLJFLmjS8gCCEphdgagquL4gbCp4vyCYMhnosri/U16zI72VwvN5jH1SleT+cotZjr+6jbrk/JRLJ7jD+O08e9BEkXY5MCCSSA6IbpC+LA7+6ExA3NIQAxwuoOT6KomAInbHeOB/9yTPcOZxZs/K/l8Hxeo89WWrwM/eN0L9Hi+PWYr+cojbiVH+KD947zDdfyzNZbKKpEDP0Hd9H3XB/SiQSiWR/kAmBRHKAdMNw5eLA78cTRSYKDQCSMZ2hTIz7j/bwobeMcW547QBwL4Pj9R7b9UN+NF7gpYkSx/qSxI2tdyS2K3HaL6eo9VjcNWl6ASgwmInx3ruGeOJ0/47vo264PyUSiUSy98iEQCI5YLphuHJx4Fd1PGq2T8rSSceMTQWAexkcr/XYhbrLSzfLOF5IoCkMpCx0TdlSR2InEqeq7VFouJi6ihCQji3V2u/1krTlXZPRVtdkumzztfMzjPTEdqWK3w3352FC13U+8pGPdP4skUgkhwH1oA+wHU6cOIGiKCv+++hHP7rq13/7299e9etff/31fT65RNK9tAO/u0d6eOvJPu4e7WGsN7GpavAtm8rVA6C4GQ0rbyc4Xu2xhRBcztdouj4DaRNVgUAI0jGDs4MpCnWXr786SxiKNR+3HVCfnyqTTRic6k+RTRicnyrz2afHuZyvrvu9X/7xFFfyNX54dZ4fXF3gufEihbrT+Zq9tGRd3jVJxww0VdnS65fsDaqqcuLECU6cOIGqHsp/YiUSyZuQQ1m+ePbZZwmCoPPx+fPnef/7388//If/cN3ve+ONN8hkMp2PBwYG9uyMEkm3shcuQFu1qVztDMCq51rtsau2T7HhkooZ+KFAU9WOHepmOhI7kTi1E4mFmsNg2qLU8LB0hXzVpup4PDiWJZcw99SJZ6sdmYOyRZVIJBLJ4eBQJgTLA/nf+Z3f4fTp07zrXe9a9/sGBwfJZrN7eDKJpLvZKxegrdhUrnaGbNwABUoNb8W5TvWnVjy2G4T4QYhuaZQaPoOZGOnYrV9nG8l1titxWpxInBtKM5CO8eJEibrjk7R0arbHa9MVBlIx+lJ758SzlcVhB2mL+mYkCAKef/55AB5++GG5qVgikRwKDmVCsBjXdfnCF77Axz72sRX/sC/noYcewrZt7r77bn7zN3+T97znPet+faVSWfKxZVlYlrXjM0skB8FeugBt1qby6nxtxRmmSg2+cWEWgEdP5DjVn1pxruWPrSoggPmqQzpucHogueTnfyO5znY38S5PJHqTJg+OZbmcr1FsuAQC8hWHh4/n+EePjO1ZwL3Zjsx81eGr52e6yp3pdicIAr761a8C8OCDD8qE4JDjOA6Oc0sKuDwu2Cyh01jysaIZKPrKn12J5KA49AnBl7/8ZUqlEr/4i7+45teMjIzwh3/4hzz88MM4jsOf/umf8t73vpdvf/vbvPOd71zz+8bGxpZ8/Fu/9Vt84hOf2KWTSyT7x35YZG5kU3mqP8Vnvn1lyRmi7oGDqasgBDMVh6O5xIpz/cq7Ti95bNvzsXQNX1F44GgPvclbifpmFmdtdxPvaolEb9Lk0RM5qrZP0/OZrdj8/ENH9rT6vpmOzL2jPbx4o7SuO9OLN4oMZGLoqsrpgRQfeuQI54Yyaz2tRPKm45Of/CS//du/vePHmfzMLy75uOeJD5N9xz/Z8eNKJLvFoU8I/uiP/ogPfvCDjI6Orvk1d9xxB3fccUfn48cff5yJiQk+9alPrZsQTExMLJk5kN2Bw8ubXUO9XxaZ69lUThQaK87QngWIgnJBoe5StX0ycWPFuZY/9lzV4auvzLBQdzF1bUuLszYKqKdKTY73JanaHhOFxrrzDO1rmI7p1BwfVVGo2T5hKPbsHttMR+b+sR7+6oXJNd2ZarZP0wuw/ZBQwIXpCs9cW+Dfvvcs771raE/OLZEcNj7+8Y/zsY99rPNxpVJZUSzcDEd+9XOo1q3frYomuwOS7uJQJwTXr1/nm9/8Jn/5l3+55e9929vexhe+8IV1vyaTySxJCCSHk8Osod6tRGa7EpntsJZN5WpnaM8CGDEdUKg7Pm4QrnmuxY995zCM9MS2tThrvYD60myNiu3hh4Lf+7vLG84zABTqDpdna1wvNEjHdL74zA2evVbc03tso46MH4oV17vtzlRuuHhBiBuEmLpKOmbg+SHTFZtPP3WJsVxi3b0TEsmbhd2SCqtWYklCIJF0G4c6IfjsZz/L4OAgTz659ZXcP/7xjxkZGdmDU0m6ib3Uze81u5nIbFcis5ssP4MQAscLCEJB3fYxdXWJW9BmzrWTxVmrBdSOH1KxPTJxg2O9CRKmvuE8g+0FPH+9SKnpkU2YPHwsR8xQ9+Ue26gjs5o7U6Hu4IZRMmBpKjFdQ1UULENjOGMxV3X40gsT/MZP3/Wm6qJJJBLJm5lDmxCEYchnP/tZPvKRj6xY/vLxj3+cyclJ/uRP/gSA3/3d3+XEiRPcc889nSHkv/iLv+Av/uIvDuLokn1iP3Tze8VuJzJbcQHaKxafwfUDrs41WKg7lG2PuZqDoSoc70923II2e66dLM5avpDtyy9Moqsq54bWvl8WzzNczld5dbpCzfY51Z/kzGCa3qQJsG/32Fqvf7X33A1CbC/A9UIQkIjp0fxGC1PXMHWFK3u8YVkikUgk3cWhTQi++c1vcuPGDX7pl35pxd9NT09z48aNzseu6/Jrv/ZrTE5OEo/Hueeee/jKV77Cz/zMz+znkSX7zH7p5nebvUhkNusCtJeJUfsMF2YqfOfiPKoCPQmD/pTFVMnG9kNKDY/5mkPM0Pb1XGO9CSYKDeZrLqPZje+XdiLx3PUCf/Cdq/QmDUZ64ku+76DvsdXe82iBG9heQNLS6E0YS87sBSGmrhEK9mzDskQikUi6j0ObEHzgAx9AiNW3cH7uc59b8vGv//qv8+u//uv7cCpJN7GfuvndZLLU5HK+StrSo4FZTSUd0zsbtrcbZG6kOd8P6dSp/hSDKYu4qaECDTdAV1XODCYRAuaqDs9fL3L3SGZfzwVbv19UVSETN7AMlaFMfFXb44O+x5a/57bnE9M1NFWhL2URX7b9uWb79CQMsnFjT+VjtzO6rvPhD3+482eJRCI5DMjfVpLblm7QzW+HC9MVXp2qoCgQCIGuqvQmTE4NJjFUlaYXUGy4VG1vy4+9E839bjBZalJqejxxug+IJCzthAdgutykUPf48GPHeOR4775Kuda7X4QQ5Cs2thdSaXodB6HDcI8tf89nKzb/19cvMlOxsXQVU9fwgpCa7RMzVBKmztmh9J7Kx25nVFXl3LlzB30MiUQi2RLdFQlJJLtIN+jmt8rlfJWvvDJNzfHJJgwyloEXhNwsRpadCUsjFIIwhC//eApTV7dcQd+J5n6ntKvwo1YcbZVgfyBtMV9zWKi7TJaa+5qsrHW/FOoul/NVri+sdBBay3UIuuseW+rOlMEPBJ9+6hJzVQdTVzB1jZ6EQcLUOdab2HOZlkQikUi6C5kQSG5bukE3vxXaswOOF3C8L8Fc1SFlQRhC0wtouAGKIkiaBtmMwY1Cnc8+Pd7VTknLWa+iXqi7vDZVJl91+PNnJ/hGYnZf7WFXu1+aXsALLQehXMLgLcdyxAxtXdehbr7H2rz3riHGcgm+9PwEV+ZqhAKycYOzQ+l9lWndjgRBwCuvvALAfffdJzcVSySSQ4FMCCS3Nd2gm98s7SHo0WycgXSMmhOwUHepOz5eEBI3VOpOQE/c4u6RHnIJY1sDxge5pG29KvyPbxSZqzkczcW5ZyRDww340fgCr06X+fBbj/HE6f49P+fi++Vyvspr0xWqHQehVGcj8npblDe6x7plSd654TS/8cG7DvQs3XItdpMgCPjv//2/A3D33XfLhEAikRwKZEIgue05aN38Zlk81KqpCg+OZXl1qsxc1UFRQACmrnJ2KNWxttzqgPFBL2lbrQofM1RemyozV3MYSJncPZKhYntcydcjW9Kmx/h8g5+5d5ifvm94z8+51EHoCn1Ji+FlTlUbbVFe6x476Ou/nIOUj3XbtZBIJJI3MzIhkLwpOMjAZ7Msl9P0Jk3uHE6zUHNIxYzW7IBgIHVra+ZWXGy6ZUnb8q5NseGQr0adgbtHos3gL06UaLoBqZiOZWjUHZ9nrxeYrtj7cs5bDkIag5mVNqSw/hbl1bicr/LH3xtnstSgN2HSn7TQVOVQLMnbbbrlXpRIJBJJhEwIJJIuYTU5jaVrxE0dQ1WoOQGDmVjHkQc272LTbUvaFlfUz0+V+fNnJ7hnJIOmKjw3XqTpBvQmTRRFIRSChhtdn0Ld3bdz7qaDUBgK/uyZGzw3XkBRInlYxz1qIMHCPr6ug6bb7kWJRCKRgLrxl0gkkv2gLafpTZpcyteo2h5xUyVpakxXbGKGyumBZCeAarvYnBlMbehis5UlbftFu6J+72gPvQmTphdQtX0KDZdU7NZ8gReE6KqKpWv7es52gjZdtlfsPNnKtQf4/pV5/u71PK4fEDc1cgmTmKGSr9q8dLNM3FD3/fofFN14L0okEsmbHdkhkEi6iOVyGscPyCVN/FCQiRsYmoofhlt2senmJW2LOyO5hIEfhhha9KupvSyr3RkJhNi3c+6WS1UYCr75Wp6GFzCWi6OpUR3G0jXMpEqh7jJVtulNmF23JG8v6OZ7USK5nRj/nScP+giSQ4RMCCSSLmO1AdWmG/CN19Z2sdnIraWbF2gtDrxvFhsIAY4foCoKNdsnbuqcHoikJU3H39dz7oZL1WSpyXS5ScrS8UPQFvVlFUUhFdOZqzr0bGM78GF06enme1EikUjerMjfuBJJF7LagOqZwdVdbDbj1tLtS9ragffXzs8wV50mX3HIxg0GMzFOD0SuSgd1zp26VNVdH1WFgVS0dM1szUa00VWFuuMz0hNjJBNjotDY1PMcVpeebr8Xd4qu63zoQx/q/FkikUgOA/K3lURySFgtSdisW8tBLGnbavX6zGCaf/XuFA+OZfmzH92g7kT+/wlLp2p7B7roaycuVUlTJ27oxLMadTegUI9mJAxNxQtCinWPuKFz50iGP/ju1U0F+IfZpeewLQzcKqqqcs899xz0MSQSiWRLyIRAIjmkbNWtZT+XtG23eq2qCu84O8BwT6zz/fmq07XL5DbD4or4A0d7uDJXp9hwqTk+uqJgGSr3jWZ4eaJMsbFxgH87uPQcpoWBEolE8mZAJgQSySFlK24t7ep2W/5ys9jg6nwdgJP9ScZyu7ejYTeq14dlmdxmWFwRX6i73Dmcwg8FVdun2HAZ7YkRN3Wmy/amAvztvO/dyO30Hi8mDEMuXLgAwF133YWqSjM/iUTS/ciEQCI5pGzXreXqfG3PtOe7Wb0+DMvkNstq7lGWrvHYyT7uP9rDX74wuekAv+76NL2AVKBHMwmaSnqRTethcum5nd7jNr7v86UvfQmAj3/845imecAnkkgkko2RCYFEckjZjlvLXmvPb5fq9V6wVkX8Yr66pcRuvupwfaHOxdkqikJnwdnpwSS9SUu69EgkEolky8hepkRySNnq4qzl1ft0zEBTFdIxg7ODqc4W4DAUqz3dprjVtVg9GI2bGo4fHIrq9V7QrojfOZxhrDeBqipLErvVWBzgX85X+ZtXpvFDgR8IsnGjs+DsxYkSCzV7SwvTJBKJRCIBmRBIJIeWxZuNL85WmSo1mK00mSo1uDhbXeHWsh8bYtcLboUQ5CsOthdQaXo7SjxuJzab2A2lLP5/z97kRqHB3SMZ0jGdUsMDIJswqDQ9fnStSG/icLv0SCQSiWT/kT1lieQQc2YwzU/eOcjnnh7n1akKXhBiaCon+pP8wzsHl8h/9mND7Foe84W6w+XZGtcLDdIxnS8+c4NnrxU7cwvdtGBrv8+yGRvOO4bTfOobb/DV8zNoqsJ8zSVmqCRVHdsL8cMQXVPRNZWfvm94U7KvbrrmEolEIjlYZEIgkewB+xVsXc5X+dbreZKWzuOn+lBVhTAUVGyfb72e53hfohMc7teG2PvHenh1usxLN0uc6k/i+CHPXy9SanpkEyYPH8sRM9TO3MJP3jnI69PVrliwdVDLvtaz4bxjOM23Xs9zfaGOpkJf0qTpBizUXOKmxt3DGZIxHVWBhZrDQNrq2tcpkUgkku5EJgQSyS6zX8HW4pmAc0OpJTKgYSFWOPrs9YbYxa+7ZvvM11zmKg4NL8D1Q071JzkzmKY3GbmupCydH0+U+PRTlxjpiTGajR/ogq2DXva12tDxSCbGH3z3KoW6y5mB6O+mSk3cQBCEIaWGi+0GvPNcP4amETP0DRO6g36dEolEIuk+ZEIgkewi+xlsbdXRZy83xC5/3aPZOHXH47XpCuWmx31HMtwxnFlxzobjM1d1eGgs2+laHMSCrW5Z9rXchnOi0Oi8x24Q0nACao5PytKwdB03CCk0XH50rcBQT5zHT/Wtm9B1y+tcfqbbSbqkaRo/93M/1/mzRCKRHAZkQiA5dHRrALHfwdZ2ZgL2YkPsWq87Ezc5PZBifKFB2V45l1C1faqOj6mreMsGjPfborRb7VLb73HciPH6dBVLVwmFhhsITEVgqAqeojBfd0mYOu+7a/2Ertte53500/b794WmaTz44IN79vgSiUSyF8iEQHKo2IsAYrcChv0OtrY7E7DbG2LXe92WrpGydOaqDlXbJxO/dU43CHFa76GprTQ8288FW/sxcL0d2u/xXNWh0HDpTZmEIRTqLk0vwA9CBDCajdGftoib61eku+l17kc3Tc5KSCQSyeaQCYHk0LAXAcRuBgz7HWztZCZgNzfErve60zGd/pTJ9YUGjh8AtxICQ1XwfEF/SicdW/mraD8XbO3XwPVWab/HP7g6jxcEpGM6qq4wasRwvIBCw2MgbfHYyRw3Cs0N761ueZ370U07qFmJMAy5fPkyAGfOnEFVpbu3RCLpfuRvKsmhYC+WarUDhvNTZbIJg1P9KbIJg/NTZT779DiX89UtnXErC6Z2g8V7CC7la1RtDz8Mqdoel/K1Hc0EbIX1XreiRMPMcUNjstRccsaZSuSIkzBWXo/VFqvtJVtd8rZftN/jvqRF0w2pOz6hELhBSN0NyCZM7hntwfHFpu6tbnmde70TYz+W8K2F7/t88Ytf5Itf/CK+/+ZcwCeRSA4fMiGQHAp2O4DYi4DhIIKt9kzAvaM9lBoe4/N1Sg2P+470dCqgYSiYKDR4fabCRKGx60HQRq+76YX85J2DPHqid8kZ7z/aw79971mO9SUONKGB7kmuVuPMYJqPvucMpwdSlBoexbqL7YUMZmI8OJYllzA2fW91y+vc643W+7GETyKRSG4npGRIcijYbTnOXuj999LFZy3CMKoMv+euAR49mSNl6aRjRmcmYD801Jt53R9+7Bin+lefWzjel9jVIeftshcD17vFueE0//Fn7+L3/+4KCzWHkZ4YA2kL2wu5lK+RSxjcf7SHi/nqhjMh3fA691q61E2zEhKJRHIYkAmB5FCw2wHEXgUM+xlsrRfst5OB/dJQb/Z1r5Zc7faQ805Y6ywQWYAe5PnODWX4Nz95pnONry80sHSN0Z4YAvjLFyY3nfQd9DXf650Y3TIrIZFIJIcF+dtQcijY7QBiLwOG/Qi2Ngr2P/L4Cb7x2v76ze/kde/mkPNOWX6WbnKqWX6N56oOX31lhmJj60nfQV7zve6m7XXCIZFIJLcbMiGQHAp2O4DY64BhL4OtzTi0fOmFCeYqzr77zXdTYL8bHORW37XscNvXOAwFT72Wp9joniVjW2Evu2kHId+TSCSSw4xMCCSHht0MIA5zwLCZ+Ycr+RqBEBzJrR6cSw31xhzkVt+LsxW+9NwkV+ZqBCIkFzc5M5he0pXotiVj22Evu2ndMCshkRwkJ37jK5v+2vHfeXIPTyI5DMiEQHKo2M0A4rAGDJuZfwgFaIq6pxrqbt0YvVscVMD91IVZPv3UJeaqDqauYukq1abPfM1d0pW4XQZn97KrdBCzEpqm8cEPfrDzZ4lEIjkMyIRAcujYzQDioIcrt8Nm5h+ycYOBdIyJYmNPJFHdpKvfKw4i4L44U+XTT11ipmIzkolh6CpeICg3PRw/BOh0JW6XwdkwFNwsNrg6XwfgZH+SsVxi134G91vGpmkab33rW/ft+SQSiWQ36O5/KSSSfeCw6d43O//wvrsH+fz3r++6JOogdfX7yX4H3GEo+NLzE8xVHYYzFpYRVZctXcFMmhTqLg3P59JslclS87YYnL2cr/Jnz9zgh1cXKDc8hALZuMnbTvbyC287dlvcR5vldu+4SSSS7kYmBBLJIWOz8w97IYk6SF39frPbAfdGAV9bomTqCqa+VGqiKAqpmE7V9ik1Xequv6M5mIMIPpc/Z9Pz+fRTl3lpooSmQF/aREGh1PD4xoVZ8jWHf/e+s4cuKQjDkBs3bgBw7NgxVHXj/Z9vho6bRCLpbmRCIJEcQjYb7O+2JGotXb0QgqrtY+kqL02UmCg2ON6X3JXXelDs5uD5ZgK+uusTCLB0DS8IsZYlBYam4voeqqJ2uhLbSfoOIvhc/pyWpjJXc5goNDE1hb6U1bmfhjIqC3WXi7NV/vb8DKfefbiSS9/3+fznPw/Axz/+cUzTXPfr3ywdN4lE0t3IhEAiOaRsNtjfTUlUW1cfN2JUmh5uENJwfGYqNsWGhxeE2F7AZ5++xj992/EDCWR2s/q9G12WzQZ8SVMnFzeoNj3KTQ8zqS5Julw/xPVDzgymlnQltpL0HUTwudpz5is2F2dr2F7AaHZpcqkoCulWN+Tlm+WudknaKW+mjptEIuluZEIgkRxi9nv+IWnquH7ID68uUHcDGo5PxfZRVRhMx0haUVX72nydzz49vu/Vzb2ofu+ky7KVgO9INs6ZwTTzdRfHDyjUXVIxvdUZCJipOAxnYvyDh49sK+nbr+BzcUKWMDS+9srMiuc0dBVLV6g7UWcpEzOWJAWGFslsGp7f9S5JO+F2sI6VSCS3BzIhkEgkm6bpBsxVHWYqNsNpi6oQCKIgcKHmEDc0jvYmuP9ID5fn6vta3dzL6vd2E6+tBnxtiRJAw/GpOj6u7+L6gpFMjH/z3rOcG8ps6zXsR/C5PCELQsFEocmdw6klz2lqKqauoas+DTfA9cPOEDWAF0SOSglD78ijbseh29vFOlYikRx+ZEIgkUg2RRgKvvHaLJm4QRCGlJoeDTcgpquAoOYEqIrCqYEkqqrua3VzefUboGr7uEHIUNpipmIfiPRiqwHfYonS5XyVUtNDVeD0YIoPvWWMc8Pb77bsdfC5WkI2WWxQqDu8MQtJy6A3Genp0zGdwbRFoebg+gF+KLBaj9OeR9FUhfuPRkPbt+vQ7e1iHSuRSA4/8reMRHKIOMgq6WSpyeV8leGMRU9c58psnYW6C0KgqiqZuE7M0DBarir7Wd1cXP0uNjwu52sUGy5+EKJrKklT44UbxX2XXmwn4Nur3Rh7GXyuJUfKJkx64gY12+fKXI1cIoeiKCiKwpnBFNNlm3zFplh3iG4bhXLDIxSCB8ay/NS9w1ydr922Q7e3g3WsRCK5PZAJgURySDjoKumFmQqvTldQgCAUhEJgaCo9cYNs3EDXVMpNF7cl99jP6ma7+m17Gq9Mlmm6PqmYgRHT8QJBqeExV3O4MFPZ14RguwHfXsyG7GXwuZYcKR3T6UtaTJYaFGpONC8Qj5KRXMJkrDdBX8qk4fgs1FwAeuImj5/q5cOPHeNUf4rPfPvKbTt0u5tOVhKJRLITZEIgkRwCDtqa8HK+yldemqZm+2QTBpm4geuHlBoehbpL2tJBCdFUFVNT9726mTR1LE3ljZkKTdenN2l2gkdLVxAxjblqwHPjBd535/4FWN0U8O3lWdaSIymKwunBJGXbZaHmUmy4JKxbz3msN8FHHj9BzFBX3VQ8UWgcuqFbTdN43/ve1/nzRuzFvhCJRCLZKjIhkEi6nIO2Jmw/v+OHHO9NMFdzSFkQMzSOZONcLzSYLDdJWzoj2TgguJSv7WuweyQbZyAd45lrBQbT1orqd90JGMnGmKs4+x48dlPAt1dnWU+O1Ju0uGMozeuiStMNGJ+vr/qcx1bZW3EYh241TeOJJ57Y0vfslUxMIpFINotMCCSSLuegrQnbzz+ajTGQNqm5Pgt1F0uPfPJzCYNC3SUwI/eYctPfcoC509kIVVV45ESOr52fpmp7oETWlV4QUrN94q3kZb7ucmWutmvB1mrnBlZ8rpsCvr04y0ZypKYX8uR9o/y9B0ZoesGmn/PNNHS73xbCEolEspjD/1tUcltyO1oMbpeDrpIufn5NVTjZn+TFGyXmqg5BKFCVKBH40MNH+Ol7R7b8fu3WbMRdIxnODacp1l3qbkDd8dFUlVRMBwGvTlVoegFffOYGr05Wdjx7sdq5swkDBJSa3qqvZTO7Avbjvt/t4HMzcqSfundoy9urD+PQbRiGTE9PAzAyMoLaGrKXSCSSbkYmBJKu46CHZ7uNg66SLn5+Lwi5Nl/H0FWO5uIoioLrR8H3tfkG+hYDzd2cjTiSjfPQWI5XJsvcl7HwQkHD9bk0W6Pp+vghHMslGM3Gdjx7sdq5p0oNvvHaLACPnshxqj+1pddy2O/7vZAjddMMxmbxfZ//+l//KwAf//jHMU3zgE8kkUgkGyMTAklXcdDDs93IQVdJ28//ymSZcsOl6Qb0tYZ2hRA4XsDxviSOF2xplmG3ZyNUVeH9dw9xcbbK5ZbEaqrUpGp76Fpki3p2KE0mbpKOGes+/nqV+tXOLYRgpuxgagooCjMVh6O5xKZfy+1y3++FHKmbZjAkEonkdkUmBJKu4aCHZ9c710HKlw66Stp+/ouzVa4XGmQTBgJw/SDS55s6ZwbTGJqyZJZho+u227MRl/NVvvHaLHXXJ191uDJXo2YHZBMGQ5kYpwdSncVY6z3+RpX61c5dtX0KDZd0y1KzUHc7FpsbvZZuve+3y15o4btpBkMikUhuR2RCIOkaDnp4djW6RcZx0FXSM4NpnnxghAszFYJQUGy46KrK4KJA2w/DzizDZq7bbs5GLK6wH+tNcMdQmkuzVZ6/USRmaJzqT3aSgfUefzOVej8UK87tBiF+GGJo0a/UmuN39jEsfq6q4zFRaCwJarvxvu9G5NCtRCKR7B0yIZB0DQc9PLucbpNxHHSV9K7hDPeMZNA1BVPXMDWVdOyWhKk9yzBXdfja+ZkNr9tWZiO2KuEBOJJLcL3QwPZCrs7Xl+wmWP74ix9noeYynLFw/BAhfNIxnbODqU6l/sn7R1ac29RUdDVyNQLQW/sYFj+X44d8+YVJ5mvukiTp7FCqq+57iUQikbz5kAmBpGs46OHZxXSrjOMgq6Rt+8zzU2XO9sRXnWW4dzTDSxOlTV23zc5GNN2Az3z7ypYkPNFzrr0ld7XZi8lSkx9PFCnWXcYX6vhhiK6q9CZMTg8mO5V6BVacOx3T6U2YzFaaoCgMZWKkY3rnuS7N1qjYHrqqMppdmiRdzFdx/XDFfS+EoGr7FBsufihIGBsvuWpz0DI3iUQikRwuZEIg6RoOenh2MVLGsZLNzDI8MJblL1+YZDgTo2pHspnFnYTl122jx7tjOM3nf7B1CQ9svCV3+ezFhekKF2eqGJpCOm5gaDpeEJKvRlKf+4704PgBDS9Y9dzDPRYTxQYgGM5YeEHA9YUGU6UmFTtgNGtxbmhlknRxtorjh0yVbM4NRdepUHe4kq+zUHcoNz16kxZ//eIUP33f8IZdqW6RuS1GJigSSXdz4je+subfjf/Ok/t4EslBcegSgk984hP89m//9pLPDQ0NMTMzs+b3fOc73+FjH/sYr776KqOjo/z6r/86v/Irv7LXR5VskYMenl1Mt8mXuoWNZhn8UDBfc5gqNSk1vRVV9kzcWHLd1nu89909yDdezW/YbVhNwtNms1tyw1Dw3HgRPxT0Jk0sParGW7qGmVQp1F3emK1yLJfoBLQfvHeYb76WZ7LYRFMhZui8/+4hEHBhpsJ3L87TcH1QIAzB9nx6kxYn+1Od8ymKwmg2zo1CA0tXuZSvETdU3pitUrN9QKE/FSUSr05XmK7YfOTxE8RNbdXguttkbu0zdVuCspdomsa73vWuzp8lEonkMHDoEgKAe+65h29+85udj9f7pXvt2jV+5md+hl/+5V/mC1/4Ak8//TT/6l/9KwYGBvgH/+Af7MdxJVvgoIdn23STfKnbWG+W4XuX5pgoNBBCkE2aK6rsZwdTK67bWo+32S7NahKettzG8QMW6i4/c+8IP/vg6JpbcidLTeaqNiM9McpND7O1hbn9XElLY7pk8/DxXEfCdDlfpdhwcQPBSE+Mv//gKD9xZoBvvZ7nW2/kEQiO5OLoqsLNUpNy0+d7l+YBliQFcVPD0lWefGCEizNV/uaVaQp1l2zcoDdldYa2hRD8eKLE//4/X6M/ZeIE4ZLg+lR/qutkbt2YoOw1mqbx7ne/+6CPIZFIJFviUEYzuq4zPDy8qa/9L//lv3Ds2DF+93d/F4C77rqL5557jk996lMyIehSDnp4FrpLvtSNrDbLEIaCF2+UMHSVIBSYWhRUt6vsCzWH85MVfu7BIyuu22qPt9kuzXIJT9xQmSw1ma+51ByfhBEFzV4QcudwZtXHqrs+ThByx3CGVybLFOouqZiOoUWDwjU7QNcUjmTjfP4H49woNGg4PjUnSjrG5+tcztdwvJA//cF1ml7Aqf4kqqrieAGmpqHpUHcDXpwocbw30dlg204u7xrOcMdQmlduljk3lCaXMJcMbRcbLvmKTdX2Ge7p40hu6eKzD9473FUyt26dw5FIJBLJSg7lTvVLly4xOjrKyZMn+cf/+B9z9erVNb/2Bz/4AR/4wAeWfO6nfuqneO655/A8b93nqVQqS/5zHGdXzi/ZmHaAeOdwhrHexL4HDG35Um/S5FK+RtWO5C9V2+NSvtaVG1IPmslSk6vzde4dzZAwdQp1F8cPCIVo2XKCFwjuH+vZ1HVb3KVZjcVdmnZnaSQT48c3SlxfaABwsj/JQ8eyTJdtPvv0OJfz1XWfK2aoPDiWZSAdw/ZCinWXatPH1FWOZuPcLDa5UWhQbGn746bGQDrGQNpkpmLzn7/6OpfyVfqSZifgN3WVuKHhhYK4oVJqeEyXbeBWcnlmMBqybnoBmqZwNJfo7DBof92VfB0/CEmYGqauoakK6ZjB2cEUhbrLUxfyNL2AhKkjhKDS9JivOVSaHkII4qaG4wf7JnNb3OEBlpwHWJKg3E4IIcjn8+TzeYQQB30cyQ5xHGdFLLAdQqex5D/hrx9/SCT7zaHrEDz22GP8yZ/8CefOnWN2dpb/9J/+E29/+9t59dVX6evrW/H1MzMzDA0NLfnc0NAQvu8zPz/PyMjIms81Nja25OPf+q3f4hOf+MSuvA5J99Mt8qXDQruif6o/RdIyuJyvUWxEVXpdVRnuiWHpKgNpa1OPt9Uuzan+FL0pk2N9CY5k41i61qmwCyHWrUgvfq6zgykePZFjotDkynyNmu2xUHfwg5DvXZpHUcD1wyU2pjFDZzijcGOhjhOESzogiqLQmzRxgxDXC/CDkJrjU7W9FbMxa0nV2ovPLEMjFCyxNG1X/yeLTVBgqtRguuxE7kRBiK6p5BImIz3Wvsrc2veD7WlcmC6uOM+J/sS+Jij7hed5fOYznwHg4x//OKZpbvAdkm7mk5/85Iq5xe0w+ZlfXPJxzxMfJvuOf7Ljx5VIdotDlxB88IMf7Pz5vvvu4/HHH+f06dN8/vOf52Mf+9iq37O8fd6u2iz//HImJibIZG5JDCxrc4GM5PahG+RLh4XFwWxv0uTRE7klTkMgKDf9TQekWx0ynyw1uTpX5/RAasXcx0aSmeXPFTdULuZr1GwfBcFA2mI4E+Olm2U8P2AkG1/x+8PUVSxdpeGFVOzIGahN3NQYzsSYKTdxg5BS0yNh6iuSy7WSIDcI8YIAhMJQzy1L08WPr6lg6hrPjhcxdZV0zMCI6XiBIF+1uVls8IG7h5YkK3vp/pM0dVw/5PnrBYJQkFp0nrmqTaHuMNabeFPO4UgODx//+MeXxBaVSmVFsXAzHPnVz6Fat37vKNrK2TSJ5CA59L+Jk8kk9913H5cuXVr174eHh1c4EOXzeXRdX7WjsJhMJrMkIZC8OXmzb0jdbNC4WjC72Pf/Ur625bmLrXRpduoM1X6ur70yw9+cn6FQd+iJG/SlLE4PJNFVlStzdWYdP9ppEDOWJAVeEJKOG9iB6AwFt2VDAJauoCgK9x7p4TefvIueuLniWq6VBLl+SNONHv/0QGpFMtKWT+ntzoEQgACU6P+tIshiActeu/+MZGI4XpT8HMtFCZTrhwRCkDA1ZisOQ5mQkUxsx8+1n0gL1TcXlmXtSjFQtRJLEgKJpNs49AmB4zhcuHCBn/iJn1j17x9//HH+x//4H0s+9/Wvf51HHnkEw5AZukSyHlsJGvfKNnazXZqtOkOtFtidGUzz9x9UeWWqzB1DKbKLBnuFiCxJ56oOdcfH9UOs1rIwIQQ126cnYdCXNJmpONwoNulLmp1rsFB3ycQN/rd3nebeI9l1X+/yJMjUVE4PRM5EucTS19aWTx3rTZCv2Dx6IsdM2aHQcKk7PpqqMtQTZzhjUWp4TJaaOH6w5+4/0xUby1DJJQxmKw5eEOIEIUEoCEJBTNfwgpDpin1oEu43m4WqRCJ583DoEoJf+7Vf42d/9mc5duwY+Xye//Sf/hOVSoWPfOQjQNTem5yc5E/+5E8A+JVf+RV+7/d+j4997GP88i//Mj/4wQ/4oz/6I774xS8e5MuQSLqe7VhG7tXcxWa6NFuZOVgvsPNDgaYqHMkl0BYlHYqicPdIhpvFBuWmR8PzCYXA8UNsLyQd00kYOo+f7uPcUIrPf/864wt1CnUXQ1O5YyjNR95+gvfeNbTa8Vdcx+VJUNPz+fz3r6+ZbD18PMeXX5zkVH+Ko7nEisVwgRCMz9ep2h5/9/rcnrv/1N1oGPvMQIpnx4vUXR9NVdBVhYSpoasKk6UmF6Yr+54QbKfK/2a0UJVIJG8eDl1CcPPmTT784Q8zPz/PwMAAb3vb2/jhD3/I8ePHAZienubGjRudrz958iR/8zd/w7//9/+e3//932d0dJRPf/rT0nJUIlmHnVhGHtTcxWY7FFfna+sGdh+8d3jNTkNfyuL+oz386GqByWLkFKQot4Z8T/QnO4nPu88N8sJEkYW6S1/S5C1jOXR988ZuqyVB6yVblq7xtfMznXO35Vptmo6PpWvUHH9f7EmTpo6lqdwoNEjFdIZ7YoRCoCkKpq7i+CFzVYfnrxd5313759i1nSr/Vn4eJBKJ5DBy6BKC//bf/tu6f/+5z31uxefe9a538cILL+zRiSSS24/NLgVbK2hcHsyGoWCi0NjzBGGjDsWp/hSf+faVdQO7lyZKnBpI8upUZdVOg+MJhntiuIHADwWqArqq3NLvt9B1lbeeXH9OaTuvb61kKwzFpjokqZi+L1u4j2TjDKQtnrlWYCBtEjM0hBC4fkjD9anaAcM9FvmKvW+7EbZb5d/Kz8NQ6tD9syqRSCSHLyGQSCR7z04HdBez37rrdtB8s9jg6nwdiPYRjOUSmwrsrszV+X+85QjTZXtFp2Gq1KRiewz1xHnwaA81J+jIclKWxuW5+p4v21pLPrXZDomla/uyhVtVFR452cvXXp2hZgc4XkjV9qm7AV4YYqgq2bjBQt3dF+vRnXS9tvLzoGkWjz/+OBBtLZZIJJLDgEwIJBLJCrY6oLsWB6W7vjpfWzUJOTuU2lRgN5C2Vu00HO9L4oeCY61Nw5n40q7Afm8DXs5mZjg220nYjS3cdw1nODeUZqrUZKrUxAsEph4lAumYTqnpUrE95qsObG75/LbZSddrKz8Pmqbxvve9n8lSk0tzdelEJJFIDgUyIZBIJCvY6lKw1dhJRXYnrJeEXMxXO5KVjQK7sd7ECnlO1fb4vb+7TGKNRGi35DbrsdFA7EYzHHvlBrUaR7JxHhzLcn2+Tiamk4mb6Go0QwDR5mpNVXhposTbT/fvadC8k67Xbg2sy6FjiUTSrciEQCKRrGA3gsadziFsh42SkIuzVRw/ZKpkc25oaWAXhiFX5mqc6k8hhCAMxQp5zkShsS9ym7XYbLC5kSvTfm3hVlWFB8ay/MXzN1EUlTAUeAhcO7IgTZg6ZweTXJmr73lXZSddry0NrH/vGuVymYG0xUhPhqYXSCciiUTS9ciEQCI5YLp10dFOg8bdnEPYLBslIaPZODcKDSxdXRLYTZeanJ+q4PkhQsDvfvPSqoH2bnROtst25VfL76+RTIzpio0fCn72gREE0PSCPbv3BtIWfSmT2bLNzZJL0LJ1zcYNTo4mGMnGGZ+v7/kcwU7fu80OrBdrTYZufAsA7bF/uOcdMYlEItkNZEIgkRwg3S4v2ImF6G7NIWyFzSQhlq7y5AMjXJqpcWWuxuV8jYlCA0NTeehYltFsYs1Aez/lNovZrvxq+f3l+iGOF2IZKqauLrnf9qo6P191WKi5qAocycVRFYVQCFwv4Np8HV1V9rSr0mY33rv1fh4mCg2uzNUYWmXz8l51xCQSiWS3kAmBRHJAHJZFR5tZCrYaB1FN32wSctdwhvfdOcREscFnn76GosD9R3pQ1Ujbvl6gvVdym/U6RduRXy2/v2xP4/nrBUpNj1zC4C3HcsQMbU/vtzAUvHijhKGrBKFCetF9ICydhZrD+ckKP/fgkW3fB1vpsO3Ge7fWz0M7GR1Jx2is8n37MV8ikUgk20UmBBLJAXBQA7f7yW5UZLcqp9psEjKSiTFRbPCjawUuztY4kl070L40W+W56wUycaNzhtUqxW0pzuszlS3LbzbqFG1VftW+vxZqDsOZGLYX8PpMNXJIysUpNjzGFxo8cjzH2cHUnt1vk6UmV+fr3Dua4VI+2tqciukYmooXhPghhEJw/1jPtp53Ox22vVqctzgZXY29ni+RSCSSnSB/M0kkB8BBDNweBDupyG4n2NtMEnLHcJr/428u8MNrBeZqNtWmzxszGmO9Ce470kNv0uo8nu0FvDpd4Q++cxXLUFecYXE1/g++e3Vb0q/NdIq2Kr+aLDX58USRYt1lfKFB0/VZqLnEDA1DVUhZOoW6S9X2ycSNPbvf2onMqf4UScvgcr5GseFSc3x0VWUoE13rmuMzUWhsOYnabodtu12v9Wgno6/eXGBo2d/t9XyJRCKR7BSZEEgkB8BBDNweFNupyO4k2FsvCbljOM1f/XiSlyZKqIpCf9LC80NcP+TqXJ26E/C2U730Ji0KdYfnrxep2T69SYOhzOpn2MlZN9sp+pc/cWpL8qsLMxUuzlYxVBVDV2i6AbYfYvsBVccnGzewDBU3CIHdv9/anZ2Zsk0QCOqOR2/S5NETOaq2jxuENByf64U6c1WXP392gm8kZjedRHVjh62TjBarnc/5YYjteXs6XyKRSCS7gUwIJJID4CAGbg+SrVRkdyPYW0vS81++c4WLs1VMXaUvaQJQtXXqboAIQxZqDlfydbLHTS7P1ig1PU71JxnpiaMoyooznOhN7uism+0UTVfsDTsf77triMlSk6rj8a0LefxAEDegUHex/RBNVdAUhUAISk2XmNeWt1i7er8t7uw0vYCJYoNrC3XeeqKXvpRFJm5QqLtcyteYqzkczcW5Z2Rr9pzd2mE7M5jmn73tOH9+Pvr4+kIdy7R23c5VIpFIdpvbI9qQSA4ZB2lfuR/sxEp1M8Hearr+5Y+/2g6BVybLBKEgHTM6j92btHADG0cIglAwU2lyZV7neqFBNmFyZjC95ByLA84XJoo7Cky30im6czizbufjG69Fny80oqRGADNlB1WBpKmBELihQFMUvFCgKDBbtjmaje/a/ba8WzJqxokbKs+OF/nOxTkePZFjuCfGa1Nl5moOAymTu0cy6JpKWlM3nfB1c4ftzFCGRx55hJrj86HHzpKJx7rGSlgi2Q4nfuMrB32EVRn/nScP+gi3FTIhkEgOgIOyr9wP1tP+n+rfWDq0UbC3ka5/LequT92LAkRDu/WccVNjOBNjoe5QbnpUmpHePh3TefhYjt5WJ2Ex7YBzoe7uKDDdaqdotc5H0/P5/Pevd4JwU1cZn2/gBQFNL8DUVUzAMjQ8x8cLBCDIxg1mqw4vT5Y5mkvs+H5bq7Mz1pskYWr86FqR12dqFOoO+WrUGbh7JLNkZmOz1f1u7rDpus6TT8pARSKRHC5kQiCRHBD7tS12P1lPT39hpsJgyqLU9NYdvF0v2Nusrh9WdinihkbSiH7leYHA0pcmBf2KiaYqHM3F+YXHjvHUa3lihrrq62wHnH1Jc8uB6eJzJQyNU/1JXp2ubLpTtLjzEYaCz3z7ypIgXAiIGdG+hXLTRwX8ICQUYOoamhoShgInCAm9kBN9Sf7Z48eXvAfb6fCs19npS8V44kwfU2WbnzjbDxfy3NPqDCxnM9X9buuwdetyQYlEItksMiGQSA6QvbJAPAjW0/67fsB3Ls4TNzWeON3HqLV2IL9WsCeE2JSu/1R/iqvztRVdilMDSUZ7YlxbqFO1PcykueSxa7ZPTNd44vQAP333CNfnmxsGnG8Zy/GjawWeHS9wJBvH0jXSMb0VmK8MTFfrnmQTBpqqbKtTtFoQno7p5BImU6Umlh4F3IPpGLqm4HoB+aoDmooQAlVRMJcF5dtdlrdRZydh6eiqwtFcgt6ESdMLSK+SEGymun+QC+KW/6yuuNc0lZNZg/fcNcA9xwZXJEcSiUTSjciEQCI5YPbCAvEgWKtCLITg6lwDVYEo/FPQ1NUDeVVV1gz28hVnU7r+p6/M87XzMyu6FK9OVdDU6OuuztWZrTj0JAxAUG54hAIeGE3zU/cOoevqpgLO8UKdQs3lxkKDN2aqJC2dgZTFaDZG0wuXBKYXZ6r8/t9dZqHuMNoT52RfkqYXPZ6mKoxkYpQa3pY6RasF4YqicGYwRdX2qNoeTS9AUcDzQ6YrDgAjGRM/gGzSaC1nG+efP3ECYNuOSZuV8ZzqT+5KdX+/O2xrJXP5qkMQis71qjdtCs/8BX/xDOj/4t9w55HeXT2HRCKR7AUyIZBIJLvCWhXiqu1TaLj0JAwabtCxuoS1NeOrBXu2F2yo658pN3nqwvquP6cHUpzqS/LMeJGFWhQgZ+MGj53q4xceO9YJJDcKOOFW8PzQsSxTJZu5msP4Qp3Zis177hzsPN7F2Qr/+/+8wJW5GnFTZb7m0pswOT2Y7CwG60tZfOSJEzS9YNOdouVBuBCCqu0TCsHZwTRBGHKj0CRfcQiFQFcVBtIWQShIWDp3j/SQSxhcytf42/MzCNi2Y9JmZTxHc4k1k62pUhPL0DgzGHXNNroG+9VhW00KV3d8vndpnqYX8K5zA50kKB0zaBuPPnUhz7mR3KHs+EkkkjcXMiGQSCS7wloVYjcI8cMQEx1dVVdIVNbSjC8P9ipNjy8+c2NdXX8QwnTZ5kg2vqbrT6nh8e/ed5ZffMdJrs3XATjVn+RoLrEicFsr4ARWaPeP5hJUbR/HD5gsNelLmpzqT3E5X+X3/+4KV+ZqZBMGSUvHC0LyVZuq4/HgWJaRnhhX5mqoisKdw5lNX/PFQbjrh1yZq1NsuPhBiK5GFqPvONNHytL5/pUFTF1FURQGMxanB1KdxGqkJ8bLN8ugsO61W2/YdysyntWSLccPcfwQPxR8+cVJvnZ+ZlNSpb3usK0lhQNQFFAVhavzdXoXSdDaXJk7/MsFJRLJmwOZEEgkkl1hrQqxqaloikK54XEkFycdW/prZz3N+PIB2mevFdetQI9m48xUbBJr6M/byUfDC7hzOMPxvuSGr2u1gHOi0Fghj1IUhUzcAAxihsaVuTo3iw3+9vwsCzWHhKmRtHRURcHSNcykSqHucmWuzoNjWRx/6zaZ7SD8wkyF71ycQ1UUsgkDU1eoNDwCAYau8f57h1louAxn4sRacwVeKKg0PdIxnbip0Wg5MG107dY741ZkPIuTrQszFb7y0jS6GjKajZEw9U1LlfaataRwbhASCEFPwuhsfU7HdKpNr/M1tuffFssFJRLJ7Y9MCCQSya6wVoUYBEJAKASn+pPb1oxvpgL93rsG+csXJvfcjnKzPvhX5+tcmasxnIkxXbKp2h4xXetU6lMxnULdZa7qbOtcYSgwNBVdVVrdF2i4PpqqMppLcKo/wULd44XrRXJxE9cPuL7QuNVF0FRyCYNMzCAIBLqmUnf8VmKzlM1eu63IeFRV4Ug2zl+/OIUbhJwb6o6tw4tZ6702NRVdVQGBH4bM1Rxen6lSrjd5rPU1Nwo2c1WHO4f3/dgbIp2RJBLJYmRCIJFIdo21KsTvONtPvuqwUHcxdXXbjjAbVaBP9ad4caK0adef7bLZAVqA+ZqD4wWUba/1eZWEqdObNLEMlZrtMV1u8vbT/Z1zbSZYaw+5vnyzxCtTZeK6SjJmMpqNM5CyOq/b1DXmKg66qvDDawVMXSUdMzBiOlXb442ZKrYfMpAySZo6M2Wbt57M0ZeKdZ5rq9duKzKebt063Gat9zod0+lNmEyWGqiKwqXZGkEYkrG0ztf4QchXX5lhpCfWVTbC23WSkkgkty8yIZBIJLvKWhXixfaM7UD+3tEMD4xl8UPBRKGxqSrlehXoy/nqpl1/dsJmB2gFgqvzdcJQ0BPTUQA3EFRtD8cPyMR1mm5IX8rqnGszwdriIde4qUU7FiyNStPDC0KycYOq7eMGIZqiYPthx4IUES0ma7ohc1UH2wtQVYVswuRUf4Lnrpf4zsV5Hj2RYyQbbw372liGypmhzQ37boXd3jq825Xvtd5rRVE4NZBgotig5vgkTcFgxiIMbp3zkRNZFhouX391lhO9SaZbr+MgK/Lr7Qo5aHmWRCI5OGRCIJFIdp3VKsTLA/m5qsNLEyX+8oXJLVcpV3v8xYHORq4/u/H6NpIvnRtK8Uf/6xrlhksQChxTQ1dVYoaCH4Q03ADbC3hwLMtH332GM4PpTQVrp/pTS4Zcq7aPoamoikpvUmOmYvPdi3MkTA1fRHItlWiQ+NETOWbKTrQtuObg+lGSlLIMHD8km7B417kBfjReiDoHXogbhDheQNWBP/3BOAlT574jPfz0vcO7ci3XqsC3HZMieZMgbmjrPEpEO5m6nK9SbLpoisrpgRQfeuQI54Y2P6y9mPXe64W6x9mhNNfna6AolJseugLl+Gi0DyIZwwldvntxjhuFBo4X4AThgVXk19sV0g3yLIlEcnDIhEAikewb7UD+cr666q6A5YHvZiu9qwU6a7n+7BbryZfuGE7z1y9NcXWuzmDGotL0cf0QPwgwNIWeuIGpBWiqwi+/8xTnhtObDtb+3v3qEolNexHZXNUmbkQzAI4fkoolyJo6c1UXCLmxUOd4X4JHTuSYKjd5frxI3NRIWToCKDZc3CCkP2XxxOk+pko27zjbzzcvzDJTtonMYgWgcG2uzuszVf7d+87uOKBdrQJfqLtcztco1h2KTY/+pMn/eGlq1SSk3RFoDyYv1F2ark+tdR0uTFd45toC//a9Z3nvXUPbOuN67/WZoRR/9cIkAymLQAhMTSUdG6HY8HjuRoWpcpN8xebKfI1juQR3DKeJGdqBVOS7XZ4lkUgODpkQSCSSfWUzge8Xn7lBLmlyda6+qe7BaoHOaq4/ux3orCZfGsnE+IPvXu04C+WSkTa/UPdoej5NLwTF58xAElPX6E9bHdeilydLjGbiHblPFFzqS4K1q/P1JRKbxYvIJstNAr9lORoKSg2PTNzgdH8yqvrPVulPWVi6hqYpJFsBuOsHSyxhE5aOpsKPrhW4NFvD1BTScQNDU/GCkErD5fnxAr/3rct87P3nVrVs3SzLK/BxQ+WN2Ro120dB0J8yuWMozatTFabL9pIAenFH4NXpMvNVF4CYoTGYtsjEjdZCNptPP3WJsVyCc8PbC77XkqpNlpp8zZhB1xRyscjGtVB3eXGiRMPxaDg+mqrQnzQpNT2ev17k7FCaobTFTMXZ14r8bsuzJBLJ7YNMCCQSyb6yUZUybqh86/U8x/oSnB5IbUrjvBuBzna158vlS4stSedrLl4giJs6o4aG64c0vQA/FBzNxZkuO/zx/7pG1fapuR6XZmu8qpZAVVAUBUNVGUpbnBlKkYkbzFZsgBUSm96kydnBFDMVG1+JkgE/EAz1xDg9kCKXiDYST5dsKk2v45DjtZKOmu0zmIl1LGGbboAfCC7NltEU6EtZnfcqDKM5iLLt8dSFWZpuwP1HszuSv7Qr8F97ZYa/OT9Doe7QEzfoS1mcHkjSm7QQQiyRtFydr3XkVWEYslB1qTs+fiioOR6BEAylLeKmznDGYq7q8KUXJviNn75rR8nL8oRyeYdDCMGVmRKO45A0dWarAamYga6pVG2X2abHfM1hIGWRtHReuBHuW0V+s8PwO3Xhkkgkhw/5Uy+RSPaV9YJ3IaKgvOkFHMnGl2x/XU/jvNNAZzddV9qv72RfklzCZq5qY7aWVlmGhqGrzJabPH1lgaYbcjlfIxXTMTSVStPDDwWqohAzNDRVoVh3mK40OT2QQldVjrcSpeVDrglLJxPT8IOoK/HQWJZM3Oj8/bnhNHM1h8tzNc4OpsjGDabLTXQtcj06PZBa4saUSZjU3Qr96VsLt5puwEzFxgtC4oaG4wUI2BX5y5nBNH//QZVXpsqcG0qia2oncRFCLOmStPc7FOoufUmTZ64VaHgBKGAZKkEgKDc8wlAw0hPDMjRMXeHKHshhlnc4UobgjtmnuAP4lv92VEUhbWnMVhy8ICRmaAghUFWFUsNlruZwYbqyLwnBZofhd+rCJZFIDh+rr/yUSCSSLRK2nIJen6kwUWgQhmLVr1scvC+navvM11ySlt6x7WyzXOO8mHagM122EWLp87YDnTODqVUDnfYg7/mpMtmEwan+FNmEwfmpMp99epzL+eqWrkP79TW9gDODKeJmtGvA9nyars90qUm+Gi2ySlgaY71x0pbOdKmJGwiEUABBKAQq0PRDbhabfP/KAhPFBl95eZo7R9L0Jk0u5WtUbQ8/DHH9ENsTxC2d+49m6Uks3ZwbNzTODaW5eyRDuelj6iqqoqCpCmcHk2TikQ3ppXyN3qTJW45lQQEFpXMdC3WnlQyoaGq0SyFuapwZSHKz2OD/+8Mb3Fior/neb0TDC6Kko+xwfrLCj8YL/ODqAs+OF7hRqNNwA4oNl8tztdZ+B4src7WOLEcIUABdU1EUcP2QQsPD80NMXSMUUcK22Xt1s7Q7HPeO9lCouZ3PZ+I6fUmTqh10kihTUxECNFUhFTPwA8Hz14s7PsNmaCcvy++dxe/7brhwSSSSw4fsEEgkkh2zlQr7elVKxw+oOT4n+5MrNhrD2tKfzbj+rBbo7IXryuLXd3YwxYNjWV6+WYo6H66P7YWoqkJMUcnEdDRVxfN9nCBEAZTW4K7jh/ihQISRU1AQCsZyiY6W/ifvHOT16WpnyNXUIkcdgFxiaZeknRS95ViOf/kTpzr2l/NVhxdvlLg6X2d8vr5kp4OhqXzhhzcoNTyGMmpL7tS2L1VwPB/L0FCA566XyFdtLs/WmCo3uf/I9iRE81WHiUIDIQTZhEkoVKq2x4XpJq9NV0iaGpqq8pWXp5mvOfhhJCPy/MgNyQsEYSgw9Sjo1jWFputTVKA/bZGNG8xXHZ56Lb/rHvxnBtOE9whuLlQ6n/P9ENsXVGyflKWhKOCHUbdDBeqOz0g2Rr5i75tsaCvbpCUSyZsHmRBIJJIdsVVf8/WC98lSk4ShMbrKfAGsL/3ZTqCz3jwDQMrSef56geeu53jkeO+mZwqWD8n6QUhMV1EUHVUJyMYNCg2PhZqLqWk4fkjQCmRDAZoisP0oEdDVaEhWIIiZGsd7E1zK13hjpsr/9s5TS7ztm57P579/fd2kSNfVW4HnMLz9dP+K2QmAm8UG5wZTPHNtgfmag6GphEIQCnD8EAEMpEwu52vYXhAldwgS23TQCUPBizdKGLpK0w2YqzrUXZ+GG3QkQwI4MxBjrmpzJV/v3BNJS8PQFEqhRxCC7YUYWpRe2V5ALqGQMHVySZO/eWWaYsPbdQ/+y/kqn//+dUoNl/Zi4pipIRwfLwixvajb4vgBMVOj5gYkTJ1zQ2kqTW9fB3m3sk1aIpG8OZAJgUQi2TbbrbCvFby/9UTvEunPVjXOWw101ppnKNQdruTrzNcdKk2PP/jOVZ49Vtx0FXnlkKwbDckaGsWGSzZu0PCClqTFJWG25VECBQVD1/CCKNC19CgZaLoBsFQ6NV2xV1SVt5oULR+UXdztKTRcVFVpSYwUvCAkDBU0TWEwbWEZGjXbpzdp4gYhuqaRTZikY1vvrkyWmlydr3OsN84L10tR0iEECqBpCl4AthuQS5jcNZzm1akqTc8nrmuoioJhqGREJDvzQoEfCvwgwNRVckmLY7kECCg2vF334F/8c3BmME1tPPp83YnOW2l6BKGg7vqoikLc0BjKRAPfhqbgeOG+D/JuZZu0RCK5/ZEJgUQi2TY78TVfb6PxZ58e35L0ZzFbCXRWG0Yu1B1enCjRdKNgsidu0Js0tlxFXjwke8dQimzCRAjBD68VWkGhTjXwaLo+aUuPrEKDEE1VCAKBpkWBo6Io1J2AmKF1pEDruSbtpPq7vNszmo3TnzJ5ZbJC0/VJmCp+AMdzcY73J3l9pkaqdd0WOxVtx9O+7vo0vYCaHXUbTD2kUHeJbiuFuKHgh4L5mkvNCUiYGq4fuQr5XkCiNbBt6QpK0E4mFYYzFu8+N8j9Yz381QuTe+LBv9bPQX/KYqEZYOoatudzaiDJqYEkA6lbjk6X8jU5yCuRSA4cmRBIJJIt07boPD9VptBwGOmJrfp1G9l9rrXReL80zsvnGQCu5Os03YBcwqDY8BjMxBjpiYK1zVSRF9uXzpRtVAWO5BKtoVfRWSKWS+g4fkDDDQjCkLSls9Cy0DR1HU1R8EJBEEROPkdycTKt4LvRstecKdurBvzbqf6u1e0Z600SNzS+f2UBRVHQlJDZqovjC2q2BwgWalGAfqo/0fm+rXraJ02dUAjmag69KbOzzdnSVVQlWonmBe3txR6mrpJNmCRMndmKQ8XxUVFQFQVLj2RYpqFy75Es7797iBCxZx78SzpNIuh8/uHjOWoezFSavDRRJpc0GUxHSW7N8Ted5EokEsleIxMCiWSf2K7PfbdxcabKl56f4MpcjbobMFlsUm363D3aQ2/SXPK12/U13y+N8wrLSEtnvu5g6irFhkd8kR0nsGEVeflwdRAKJgpN4obGWG+ys0Ss5kSOQ5m4ThiKaKtuEGLqKpaukrI06m4YdSk0haFMjPuO9KAoCgs1mx9dK6JrKn/+7A3ihr5iKHY799paVe5C3eHlyTKhiGYcHjqWZbLY5Np8nartUairxEwNXVO4MtdAUVR6k+am3vvF52zLaF6aKEWdEKFiaCqqqqAp0PRCEqaGqrSshABT13j8dB/lhsdr09GwdRCCrkYbnB8+lqNq+/z+313msVO9kWzH8VsL6yKEaCcZLn4YzUBslcWdppSpoveNAaCoKpl4NEzsBYJT/Unma64c5JVIJF2HTAgkkn1gN33uD5KnLszy6acuMVd1UJQo8HL9gPGFBo4f8tCxXCcp2Kmv+X5onMNQYGoq94xmeP56kZvFBpWmR0/cYLCl8V6c5KxXRV5tuLru+Fybq/PseJGEqdOXsuhNmjw4luVyvsrluRqKAkJANmHQmzRJGjpCAdsLmK04JAyNe49kyMQNbhTqPHO1QCgEDxzt4VR/iqYXLJEzAdu611abpxBCdDomA2mLUtMjHTO4a8SgakcSH11TONmXJBAwV7WpOT4PHO1hoe6u+96v9jOhqZEd51zVJZswiOkqNcdHVRWM1tZmUMjGdLRW2yBt6WRiBjMVG9sLSVlR9f1INk5/2uLKbI3rhQYXZspoisq1uTpvPZmjLxXrzIos1B3KTY/epMVfvzjFT983vKWfy+XOUok73rHkGq7m8LTZRM33Q16YKLLQ2rnwlrEcui4dwyWSE7/xlYM+Qtcw/jtP7vgxZEIgkewxW3Xh6VYuzlb49FOXuFlsYqh0Bjcjq8eQm0WBZag8drIX2wu7Ug6xuCI9V3X41oVZnrlWiJZYIbA0FV1VONGX4I7hzAqt+VpV77XkNpm4wVtP5vjOxXl+NF7gidN9JCwdQ4v8/5OmTm/C5PRgksF0jKYXMFVqYhkaT943QtzUeGmixNW5Oq/cLHNhutJyzTG5XmhSsaNdB2cHU1zK1/izZ27QdH2myja5hEl/0kJTN7c4bEmV29Kp2j6Fhsts1SZt6fihQFdVDFXhjdkaoRAc640zW3GYrznkkibZhMFc1eVH4wUeOZ5b8d63r/+F6QpfeWUaxwsYzcY7PxNTJZuUpWN7IY4XoGnRxmaFyErV80OySZPZqsu5oeh1XJ6rk7I0FuouSSuSCmXiBv1pi5dasyDZhEEQRud9bbrKdy7Oc+dwmqlyk5rtAwr9KYtzQylena4wXbG39HO5WdvbJQ5Pm+CpC7N87ulxxhfqeEGIoamc6Evyi0+c4L13DW36cSQSiWQjZEIgkewhe+FzfxCEoeBLz00yVW4ihMAJaMlbFIIwpGILHD+SD12IVcglrAOTQ6wll1lckZ6vOVyZq1F1fFKmHm3jJdocW3V8XrpZZiBt0Ze6NRuxXsdjveHqvlSMR0/keH2mylTJRtcUTE1FQWEsl+ChY9lb94Wmcm4oui8u52v8yrtO88Tpfr5/ZZ4/+l/XUBQ4mouTihl4gehU5B8cyzKcsfjOG3kCATFdZbIYbSHOJUxO9Sc6i8P++RMnOJpLrLjf2lXuH15bwPdDii0rzFLdpRnT0TWVo7kEQgjyFRtDVzA1jUwsJJswsb2QIAzRNQVDVfiZ+0aWvPft6385X+XVqQo1x+d4X4KBdAxNVUjHDM4N6dRdn3LDJZsw6U2aBEIwWWgwU3HQNYVcwuT+o9G9BVE35IUbxSWdnVP9Sa7ORZ2N3qSJAIoNl/50jHelYzxzbYEXb5ZQFYVc3KA3ZXW6QUKIbf1c7vbsy1MXZvnkV1+nanv0Jc1OgnExX+WTX30dQCYFEolk15AJgUSyh+zEhaebmCw1uZyv4vkhAAlTp/1ydE0lEzOoOR6GpvKhh8e4/2j2QGYk1pJm3TmS5luv5ynUXYYzFpOlBnUnwPdDPC0EoRAzIw17KKDS9HjmaoF3nO0nYekbuhytZV/aZiQbx/ZC/tGjYwz3xKg0Pb74zA1yyaXbhNt6dktXeelmiZvFBkdzCV6aKOOFIemYTipmtAZnFcykSaHucmWuRl/SZLrikI0bZBMmhqbgBYLJYoOr8zVMTeXSTLTI7FR/ivfdPcjbT/d3XouqKtw5kuavXpzsBKE9MYNKw6PU8IiZGqam8KPxItPlJrqmoCkKmqbylv4kuURkPaopCvM1h/60teR9aXfJ0lZ070TdBIeaE/DgWJbe1rU4O5jiRqHBqf4U87VoM/LxviSPnuzj4eM57hrJLLm3Tr07xXPXC/zBd67Ql7QYysSYrjSZLjeJt+xcvSBEV1VMTSUTN3jgaA9PX17gvtZ92nZGgp39XJ4ZTHP07Sb/+T//DgD/7Ff+n5wY6Fn352C1BDYMBZ97epyq7XEsF0dVI4lQOqaSNDVuFJt8/vvjvOvsgJQPSSSSXUEmBBLJHrJRoLgTZ5P9pO761L1oSNYyNJblNmhq5O7iBSEDaetAkpu1pFmvTJb4+mszZOIGD41lqdo+81UXgHRMxw0EhbpDn2IRCkHa0gGBQGGqbKOryoaV3tXsSxfTdCPb0NMDKcZ6E7w2XabYdLEMDSGicxQbLlfydQoNFy8IsL2QP/7eOO+7e5ArczVGe+LM11y8IMTSo0BXURRSMZ1CzWG2bBMKQV/SbG0ThjAMaXqRk1HcUAkEXJuv88ZslW9fzPPoiV7ef88Qdw1nGMnEeH26ykgmxkAqSjTqno+igKGpKELw7HgRgcALQoIQUFRiAi7nazx0LEt/yqJqe8SMW7Kq5V2yhbpLIAQZyyBl0UlocokciqIQNzUsXeXnHxolHTM21NurqsIjx3t59liRH15bYKLQYKZik686xAyVsq6haQpHc4mO1aeuqQigN2kuGTBuEzM0io0656fKAFtKbhd/3eJOzGqB/9X52qoJ7Eg2xvhCnb6k2UkGbj2+Sl/S5Np8nRcmirz1ZN+mziWRSCTrIRMCiWQP2UyguB0Xnv0maeokDRVNjbzgzWVLw/ww2lwbNzVSsf1/LetJs4YzgpdvltG16HNuEOIFISDQVA01DFmoR972itIxsGEwZfHhtx5juCe24QDocvvS9RaqXc5X+fILk1zJ1xmfbxAzNGKGSt3xEQJSMR1TVwGfaws1/uxHNjXb574jPfQmTPJVGzOpRpt7hSAMBWXbo+74mKoaVem9AENTKNQd/FBg6SoVJ8BQFdIZHUWFyUKTr52f4YdXF7hjOB3JbObrnB1K4fohjl/B9UMSpk7F9qg2A4SAXFxvbdwNUUSIqmpUmh5X5upk48aS1+r7IV97bZrvXsrTl7IQAkxNRVfVTmKTiukU6i5VO3L/af9MpGPGphPL5d2NTEzH0hVCAaWmh2Vo9KeszvviBwJFiZKRuKEt6RAU6g6vTVXIVx3+/NkbfCNh7dgAYLXOVTZhkK86BKGIZg6MGPmqzQ+uzmP7AQ3HZzBjrfp4cVOjUHdZqLvbOo9EIpEsp7ujEInkkLOVQLGbOZKNc2YwzSuTFUIhaHqRRaamQBBGG1njpsap/iRpa2Xis9esJ83ywsgus2b7VG0fU4vsLEHB8QOaboAfCBKmgqVHAXXDDZitOmiqwp3DmQ2ff7NDpVfna/zx98aZLDVIWTp1x8PSdG4UGviB4HhvAlNTKdRdhjIx7j/Sw8uTZRZqDg0nYKjHIl+zma3YxA2NquNTs30ark8YgqbB+EKduKFhaQpOAJauUHcDgiAkGzPRNIXZio0ANBUUJdLXvzrlM1FsYukaNwoNmq5PT8JEV6E5FyBa/vpOINC1qGqtqQphKHCDgNlyk5cno0r8B+4Z4u/eyPO5p8e5mK9SqrvoqsIL8SL3jGbIteRCZjJ6L2qOjxuEK34mNmufGoZiSXejWHdRUHCDkGxcR1NV5msOJ/oSFOoOL02U8AN46WaJ6wt1+pIWpweTAPz4RpG5msvRXJy7hzPM1Rx+cHWei/kqH33Pac4NZbZk67qW+9T3Ls3T9ALedW4ALxBcmC5F1qdBSLHhUnOi+Y3+9ModH003wNCiToFEIpHsBjIhkEj2kM0Git08UAzR6/jQw2M8c63A9UIDTVFwvKhiHAhBytI5movz8PHeA0lu1pNmmZqKqas4foAbhPQlTQYzFsVmFHQhQNcUdFUBBEEoiOkqcUPj5YkyTyzS2a/HRkOlp/pT/O//81V+cGUBIULcIPLEL9s+YShQVIW5ukPS1UhYBqcHUqiqyqn+JDeLTb59cQ5TV3D9kGrTZ7biAFGnRlFAKOAHkdjJD3warUVoQmiRY48KuaRJse7hBYKEpWF7IXFDo+4EnMgluDhb5UfXFkiYGoNpC1VVW++ziJIHou3JR7LRrEWx4VJ3A2pOdC+c7E/yT992nOsLjc5ArKUpqGoUnM9VHb57aZ6j2Th6K/ExdRVViexrL+VrS5KnxVV1S1MZSMd45MTKOYJ2Qnh2KNVxSJqr2VyareEHIZahMV9zeGOmwmvTVQAeGMswXXao2T6TpQbFhoPnCwpNl/6kxWhPnB9PlDtB+pV8jalSkyfvG2Gq1GSu6uAE4bq2rmt1riBKxFRF4fxUGT8Q2F60oTnUFAKhU7F9Jks2cVMjuSjJDlsdrTuG0rxlLLeNnxaJRCJZiUwIJJI9Zj837+4lqgqn+5NcX2hQczxUNaqoD6djDGdiHO9PHlhys540Kx3TSVs607aPoSqt5WBppss2laaHEGBpCgJB3Q0RAoYykUvSlbmtDZaut1Dtz5+9wV/9eAo3iKq7uqpg6Cq2F80LxAyVhhMw2hPnnkVL3hw/pOFGcqJQaGTjJo4X0PBEK4lRCESUzAiijb5CQBAKQkFHCpVLmMQMjflaFISHgtZwskqp6XFpvobrC6q2hxPTCEJaDj0C0doFZujRdVIUhYSpETc1bC9gvuZyoi/JP3/iJEd64vz2X79G1fYYSJnMVhxCEUl0DBVcP6quj/TEycZ15mou6ZiOH4jOzwSwpKpuexpvzFR45lqBr52f5txwmofGcp0gfHFCqCgKmbhBJm6QS5hcydeZrztUmh6X8jXipsZbT/TSl7IY6XG5nK8xU25yvdDA9UNSlk7DDfjReAFLU+lNWfiqQrnp8fLNEq9OVUhZOmO5OHcMp4kZ2hJb12PZWzKfqfLqnSs3CKM5irjOzWKTpKmTiUXvTdMLCEWICvih4MpcnRN9ic5w+0LdJRMz+MjbT8iBYolEsmvIhEAi2Qf2a/PuXtGWPTT9kPfc0c+1+QbzdRfbC/CDkHPDaX7hsWMHltysJ82CyBVpIG0xU3FQVYVMXOf0QJJ8xSYQ0dc03QDL0Diai3PfkR4ycYPx+fqWB75XW6h2cabKH33vGk3PpydmoGsqgRC4fiS98oNoB0IqpnPncHrJcrc3ZqooKDx2spfZqkO+YlN3AxKGjhsENL0AVVEwtWi+IwyjpIDW6woEqALCEBpuQCgEqqJgeyFJS8fxQypNv+P1X3N8grCVGPgBmZhOICKZkKZEuvxACETr/LYf4och9x7JMJZL8Nz1AuMLdXoTBqWGh+OHaKqC7UUJCoAbCKbKDWqOwf1Hs3z4sWPcNZzpdJc+8+0rnap6seHxymSZpuszmI6GlosNl1cmbwXhayWEvUmL3AmT6bLNzWKDIBCMZGMYmooQgt6kyemBJIW6Q8LUCYXHkWychbpHzfYJTY2G41NqerhBdGGFEKiKoNSMzvXgWLazB+Lrr87yS4+PdZ6/4Qardq4Wz1FEnQGN2aqDF0TyNiE0Ela0f6HhBszVXPRG5OJ1x1Caj7xd7iGQSCS7i0wIJJJ9Yj827+4Fq8kexnqTVG0/2j1QatKXNDnVn9q386yWWK0nzTrWl+An7xzk9elqp0ujayrH+5KM5eLETR0UyLUqy4qiRHKXNZaQrfb8633+S89PUGpEA6yqGkl8dEVBM7TOtt+mF9CTMDoOQhDZn06XbUayMY71JTjWl4gq2ddDeuIa1xeahAIMVaBrGooicEVIKKJ8QGn9ZxoKddcnCKPKdMOJ3KKycZ2pso2qwGDaotZyQ0qYGqEQNNzWIHHCxA9DHC/EDULqjh+5EDk+DTfA0lUKjajavlCPnJA0TafqBHhBSNjOBBZfRxElYYoCp/qTnZ+NiUKjU1WHyMGo6fodW1KUaGblvlGL2arD11+d5V/+xClO9Sd59nqBI9l4ayj5VmI4U7ax/YDZss1Cw8Vo7Wc4PZDkylw9GuzNxBhfiBIsLwhJWhpeIJgpN1HVSFImiIJ52wvJJRQabsCVfJ1zw0rHKnaqPMDZs2cBSFrGqolKOhYtpLs2XwOi6+AFgrgRvfdNPyBp6QxnLCYKDR4+nuPJ+0cZSFs8eCTLbM3h9ZnKoSssSCSS7kUmBBKJZF1WG9htyzLAIGZoXJmr78suhbX2DLSlIxtJs95zx2AnaE8YGn/94hSvTlc43pfovDYhBOWGy+W5GveM9jCSiW34/HeOpDvJxvJzWbrGlbkacSMaoG1bgLa1/6auEoQhvojmF0DghyFNN+DyXA1dU7hjKNNxFTJUhVAIJssOTTca9HUD8MIABTpVeK3lmKRrKiM9MYp1l4YXBeBxXSOXMKjaAWEIw6331vUC+pImhqaSsrRW9T+S8lyarTJZtjE0lXzFjgL9luwobmp8//ICl2Zr/PS9wxiaSr2VMIr26xKgK9H5QiCmR/aijhcsWQK2WP5TtX2KDZdUzOi8P4YWOTJ5LXeey/kaP7i2QKHhcmOhweszVVKWTn/K5Eg2zlzVZbpik0sYxE2dhKnih3Cz2GC2YhOKkGzCAhElSZFkR2CpGmEYUvdDMjGj83nbCwgFTJejpLJQc5mt2KAIbC/kT354k3/6vp/lzGCaMBScGkjy7PjKROXUQIJr8zWEgKYXJWLtrpGhqfQmTPxQkIoZBCHcfzSL4wf839+7tub9L5FIJNtFJgQSiWRdumWXwlp7BhbrtzeSZi3v0vz0fcNMV+xOV6HpBVycqUbBnqoQMzT+4LtX+al7V+ra28//w2sL/OWPb9KbMDmSi9OftNBUOud617mBaHuwES3k8oKlLk2IaBg4Ezd568k+yk2ffNXB0jXuHs10bEkL9Vt69/mWvKRdd2/PDbQervP/QEBcU3jnmX4qts/rM1Xma9HG36rt0xM3CIXA1BUKdZeEpXOyP8m1+QY118dQVVzfJ9/aiKwIcPwALxQoIvLrP5KNY+oajuczWWry1VdmGEiZXFtotDoDAr910HayohAF9n4o6IkbS5aALZb/uEGIH4QYi6xsvSBEay0Zi5sal/M1vvijGwgBDx3LMlWymas5XF9oMFtx6EuZjPTEePBoD//r0jw3Cg2UVmLieAEhYKgKti8YTFv4gaBqN9FaQ+ZRp8THbx1eUUBXIzlPpekRCkE2YbQ6AJFV7GefHuefP3ECgEItSlTemKmStHQGUhaj2RhNL+Qtx3NcnatzbaGOqkSvK2lF3YP2e96fiu6nCzMVvvPG3Ib3v0QikWwHmRBIJJJ16YZdCuvtGUhZeke/3a4yb2UIuN1V+PFEkYuzVfyWzvyOoVsDo5OlJjFdXfH8KUun0vCYqdiUm5G2XV8kR1mouzx/vUg2blBtepSbHkNpi2LDpemFeCLS1WsqvPuOAf6Pn7+P6VZylTR1RjIx/uC7V/nh1YXW9wSdZEIQ0GoQdGgnCKoazQwoQMzUyMQNskmLkWyMC9MVfuLcIDMlm/H5Gq9OV6g7AUOZGKcHkvQmo6HYF2+UmG3YuH7IfM0lYercPZphqtSk1PCw/Wg4uthwcX1BIAQIwbX5GvcfzZI0NcoNj3CV664oUHc8EpZOOhbNLbQTysXzIENpC11T8QKBpUcdkprtM5iJkY7pVG2P+ZqLosADR7MoSmR72pazXZ2rU2l63H+kh1LTp+4G+KFAASxDJQgVqnbA1YUGMV1D1xR6YgZpS6fq+CjcymDay95sP8RS6TyOoig0XB9VVTpWsZfn6nzxmRs0vchCdHGiMr5QZ7Zi8547B/mFx44xWWryH/7qPGEoyMYNEpaGH4poR4KpcyQbQwh47lph0/f/VmxRJRKJBGRCIJFINqAbdim0ZUvDmRhVO6ocm5rakV+0pSPbkS2dGUxz4p1J/s+/fR3bCzgzkOrMEUAU9L90s8Rc1eWxk7klr//GQoOrC3UQ4PohcVNDU1XmWhX1s4PR4PJg2mKh7kZ7D7yAvpSJEGD7AYW6x7HeJP/iJ06i6+qK87//nkG+/toMczWH3oRBMYiGnwGCMOgMRbe7AtBKBpRI754wNKbLNpah4foB2bjJ++8a4kg2zkSxwWefvsa1+Tr3H+lBVVUKdYdr83V0FXpiOn4YyY9ihspMJQoyQwSmplCxPQTRPERb0hQIwaV8lb93/whfeOYGbrs90EKl5YJEVKUPwnBJQrl4HmSmYpMwoh0CMVPF8ULSsWggHODqfB0Fwan+1KpytoYbcKMY7Xi4PFdHCMHx3jjFhk/N8Wi6QWcAO2aoZCydmuN3lsNVHR9N1Wi6HrquEQg6W7pdP0QQDVvX3YBsa4GZIkKGr30VLxRMHflJ7j/WvyJRWTx3c6o/xQfuGuJ7V+YJRTSsrKsqg5kYp/qjpPJYb4J8xV51z8by+9/xA/72/CyX81WKTRdNUTk9kOJDjxzh3NDGOzUkEsmbE5kQSLoOWd3qLrphl0Ld9ZmvOVF1uunhhyG6GumsTw8mycSNHcmWpis28zWXc0PpFV0QRVHIJUwu5WsEi8rdCzWHH15boOH4aCq4gUK+4jCUidGbNCnUXSZLNn1Jg0dO9lJvlfMbjk/V8XH9ANcXHM3G+TfvPbtmsBY3IockXY1kPU7LorQnYdKbhKmSTSgEhhbNFrS7BjFDpT9pUXV8nr9eRFOjIdjTgymaboCqKhzvi/YGfPbpcS7P1RnOWFyarVFpeugt1yM3COhpaegnS01sL4jcdogkSQqgGgqaqhCEEAaCctPjxkKdnphBqekRtORNqkJnG3Q7GZitODx+un9JQnlmMM1H3n6c//s7Vzlfc5mv2YgQkpZONmFSd3zmay5JS289jmC+5ixJEgEyMQMFmKnYnVkES1eJmzpX5wNQFHQ1SljKTQ9FURjKWNRsHz8QmJpK3fdxAnCCAE2JOguhiBbe6aqCqUUbvM8OpSKb1sCHMEAjGsbezNzNL7ztGHYrUcglTNIxHV1VmKk49CZNHj6e48svTpJYowvXlu1dmK7wnYtz3Fho0HB9ao6P44dcmK7wzLUF/u17z0p3IolEsioyIZB0FRsNjUoOhoPepTBfdZgoNBBCkE2aGJqOF4TkqzZVx+PsYApL10gYGhOFxhLJzWIJzlrJ5UZzEu3AsmJ75FrB/rPjRSq2j6ZGwXAowPYCZio2w5kYqZjOfM0hGze4azjDqf5kp3JbanqoCpweTPGht4xxbnjt61d3fUxd5W2n+pit2jw/XiRuaqQsHdcPKTU9Gi2r0LAVoOutoLvU0rjrWiQjycSjX/mf/8F4R3O++L19ebLERLFBzNA6ic3LN0tMlZqU7eg5gvBWJ6KNEKKTJJh65FD06lSVUAjSlo4fitaAcTSELABVCPxAASOKsgAAUhVJREFUoGnKioTycr7Kf/vRBM9eL2J7UVcDBQwl2rJcaXr85J2DnBtO8/nvjzNVara0/beSxN7WLEdP3KDQWi7WnkWoND2qdmsHhaERb3UD6o7P9YWAmK7i+iFHe+OYmooX2J3dDp4fomsquqbSnzToSZiEoWAgZbGctbZ2L5+7OTOY5pfecbLz81Wou0t+vixd46uvTJOv2Bi6uiLxaboBpqby3HiRGwsNio3IEjgVM0jHIlemiWKDT/3tGxzJxrlzRHYKJBLJUmRCIOkaNjs0KjkYDmqXQhgKXrxRwtBVgjCq2iqKgqVrmEmVhZrD+ckKT5zp569fnOLqfB3bD3D9yCbTMqJNxesll0lTx2q556wWcLUDy2LDZSwX53K+RsP1oy28KDi+IGaoJC2NphdSaLgMZyzqjs9oNt65Ttu5fu0ZjqYXLS2b6rHJV22wIociBQU/jKrvAKaukDQ06m6A7fkkLQ1NURjoiWYE2t2OxZrz9nv73Utz/NfvXeVUX4pswmCi0KTc8KKugAKWruAG4C8baPbD6ON2cpQydRw/qsAPpE3qTkjNifT3YSjQVAVdU1BQ+Km7h5e8H5fzVf74e+M8N15AhCEjPRauD03PJ6ar3D+UpuZEsqUXbxTxwyixGEhHrjztJPGBoz0s1D3edqqPuarNRKFBKARJS2eu5kQdAF0laUbD3nFDQ291USqOjwgEhXrk/d+XtGi4fmuRXEjciOYabC8k5kXzF0JEXQpDRG5PtK6HEGKFzG3x3E27I+qHgp99YARB5Dq0+P64OFthvuZGblWmiqFpncQnl4j2LIz1xsmXmzRcH9sL6E2a2F7ITNWh6YUEgWB8oc5//O+v8P/6+fvXTUIlEsmbD5kQSLqCrQ6NSg6Gg9ilMFlqcnW+zr2jGS7l6xTqLqmYjqFFi538EGzf5/pCnZmWztr2VF64UaTY8MjGDR4+3kvMUNdMLpuev27ANVNxeNupPmwv4OXJMvmqTaqlN0dR0NRIEhKIW7aY06WQuKHx3rsG13Q52gyLZzjODqY4PZik6ngU6i6aAo1WlTmyGFVImFH3JGjJiEZ64rztdF/U5Wj9XK02c6GqCqcHUgymYuha9HXT5SZAx4kHFHQVTE2h7t7STwUh6FpUoY8ZGj1xnVJDENM1HC9kNBvD9aOAPRACTYFi3SUTNzk5kOx46o9kYvzt+VkmSw3cIMQJBVMlp7VMDWo2JKwmDx7t4ZmrBQbSFm890ctLN8uUGh6pmE42YTBXdfjRtSKPnujliTP9fO/SHEJE8ipVEXhhFKwnTA1djbY5e0FIoCrRfdXqylRsn7ih0Zcy8UKBF0QD3V4oSMd05qoOFVshZnh8/+oCrh8SeB7va12XK3M1vLkGxYaHH4StgXMDXVV5/HQfTTfgM9++smpHtP2+XM5X+fz3rwN0NjqrCsxWmizUHQYzMY71JnjkRC9/9sz11gyEESUDLXtYS1cxNY3QFYwvNPj9v7vMv3nvGVlgkUgkHQ7d3vNPfvKTPProo6TTaQYHB/n5n/953njjjXW/59vf/nbL83vpf6+//vo+nVqyEat53bdZPjQneXPRlvOMZhM8OJZlIB3Dbrm32F7IcI+Foam4QcjZwRQpS2d8PhokPZaLE4RRZTRl6ZwdTFGou3z91dnOsqzlARdC6QRcP7pW4McTJXqTJr/w2DF+6R0nOdmXwnYDXD+SC5m6ylhvnEzMwA8EXhDg+QG6pvGeOwd5++n+Tb3OMBRMFBq8PlOJqtmt87VnOHqTUWXf0NRok3JM52bJJggFcUMlYenEDC2yHW39jktaems5mbLk5ypuajh+sGLmop18TJVsJotNZit21CkBgkBEQa2qYGjaku8TgAijKvtQ2sLxQvqTJm891UsgonkLiORCYSioND0CAQlT489/NMGnn7rE//sbF/k///YNfjxR7MxL2F6IoSmdHQ6BEIzPN7hZjFydsnEDQ1M53pcgEzew3SAaym1Jeu4b6+Fbr+eZqTi840w/x3oTJGMGqhJ1fRw/pOb4NFsbt90gpOn6UcdFjRI9zw9oOD5DGSuq6AuB44XRZuekScrUKDU8XD9a0DaUvbW34kYxSmZVBXoSBipwKV/jeqGOH4Z87vvXOD9VJpswONUfdWXOT5WjmY58dUmh5KFjWd56so/BTCyShimRdawCfOTxE9w1nEFTVBw/RFehUHfwgqiboalKlNS1hswX6s6SnwGJRCI5dB2C73znO3z0ox/l0Ucfxfd9/sN/+A984AMf4LXXXiOZTK77vW+88QaZzC3t5MDAwF4fV7JJusXrXtJ9LLY97U2aPHI8y3TZpuEFJAwNIQRTJZvRnjiKolBpehQaURdBVaPB2ELdpWr7ZOLGkuTySDa+JOAqNjwu52sUG24n4BrKRAFXu5r6S+84wXSlSaLl9HMpX8PuOAcJao5PzQl4cCzLLzx2bFMdrYuzFb703CRX5moEIiQXNzkzmO7Im5bPcDh+QH/KYr7m0vQDRjJxYkakew9a2vzpcpQs2H6AGyw1/1zLKlZVFe4cSfP112aYKjWpOT6Gdmthm9KalXC9aMA2EJEDkdpasOaHIaWGC4rCY6f6ePcdA9xYaDC+0CBfdfCDyP5IVZROFyKXNEmYOg3X57XpMtfn6yRjOkKIzsBu5GgqUInkNG/MVHA8n+sLDd6YreEHYavir3MkFyeXMJiv2rw+XVnSdUzFDF6dKnPJreIF0RIw0bJ+NfVIiuYEkfNQwwlQ1CiJ8eouKUtnNBuLEgg34M6RNNcXGvSnLE71J/FacraUAbW56HoqwMm+BKWmz1zVoeEGIAR1x+cLP7xOT9zgsZN9nUH25R3Rv3e/2nLXsqg0PSpNj4G0yWiPhWVoUYcsEMRNrZPMXWjZyDa9qDMQuT9FrkgxQyVuaIz0xLftyiWRSG5PDl1C8LWvfW3Jx5/97GcZHBzk+eef553vfOe63zs4OEg2m93D00m2Szd43Uu6k8WSGdcPuTJXp9gaEtVVhaYXoKkKA+loqNMNQvwwxNCie8XQVGqO3wmKFyeXyztTvUmTR0/kOppv1w86AVebo7kE9x/JdiQ8qZjRSSL8IMT2Qu4cTvPR92xOkvHUhVk+/dQl5qoOpq5i6SrVZiRhWixvWj7DMVO2+dzT15gu26jLer2mppAwVUoNH0RkP9pmPavYy/kq33o9TyZmEGRC7IVoOy9EFenhjIXtR9t6NVOlavsIorkBVQHbCwGFe0bTlJse//lrF5ksNqjYfsdlyNKV1ryDguOHeC0JjBDQmzS5MF2h5gYkLR3HD4GQuhPg+EFnoHmyZEePIRoM9MTIJgz8EKq2x/UFgaklCYXCdNnmSDbe6Y70Jk2eON2HEILJYoO6Gw06ewGEQiDC6B7RWufRlGgBmeeH5KsORmvYeLgnFgX3KJweSJKJm7eub+BDqp9Sw0PXFO4cyVBuerwyWY66ColI23+z1MBQVV66WebBsSy9SbNzndtJ69X5OvM1hytztVsOT9xaCHf3aIaG61Fv7UH40CNHeObaAhPFBkEgMLVop4Hrtzs7Kr0pi4G0xfWFuiywSCSSDoc+uiqXywD09vZu+LUPPfQQtm1z991385u/+Zu85z3vWffrK5XKko8ty8KyVjpJSHZON3jdS7qTtmTmwkxkqagqCtmEgakrVBoerh+iqgrT5SZjvUlMTUVX1ZZ2Oqqi6q3NtrA0uVytM3XLHhL8MGR8/lbg1B4APTuU4mK+ysXZKqPZOA8d62Gu6jBdtulLWXz0Pac5N7RxMnBxpsqnn7oUzT5kYhh6NKxabnqtYJglszOLZxCSpk4uYVK1o+pzKKINyJHeXkFpbdlVVBUQ+GG4rlXscnmKEILvXZ5npmxHw9t1l1LTR0HB0qOdAAlTxzK0ViW6NcegRrKVZ8eL+EGIE4SotPz7W90EPwiJt/YYvDJZIq5rFJteJLcKogr6WC6O64cU61609Kz9GCKaaQgF0fe0nIx6kxa9SZOFusv5qQpvP9XHbNVZYdVZakbuQg0vpOmGWEaUoJgt2ZmmRrsn2jIiFBVFjTY03yw0GMzEIktWS0cISC5zElI0HffUO3n5yjx+GCUSM+XIpWgoEyWeXhgiRCQjaro+V+Zq5BK3dly0k9apVsJabnpoSjTzoKBg+yHX5uuUmx5nBlOdQsm5oQz/9r1n+dTfvsH4Qp3QFRiaSqwlueqJG5weSGJ7ssCyWRzHwXGczsfL44LNEjqNJR8rmoGir+5CJbk9Gf+dJw/6COtyqH8bCCH42Mc+xjve8Q7uvffeNb9uZGSEP/zDP+Thhx/GcRz+9E//lPe+9718+9vfXrerMDY2tuTj3/qt3+ITn/jEbh1fsohu8LqXdC+n+lMMpi3ihobSGqTVVJXRXIKTfXGevV7i/GSFI9k46ZhOb8IkX7UxEsqSzbbLk8toA/HmOlPLLXHbLkY3Cg0sXcXSNd5+un/TNqxhKPjS8xPMVR2GM1Zn2ZilK5gta9OG53NptrqqtONINs6ZwTQ3ig0qtofX6mS0LTOrjk/M0HngaA/lpk++6qxrFbu8W1JseC3Hm5CKHSVE7Qq160ezEyf6k9w72tOZ4VAVeObqQpREaCquHxCEAstUI+lNEOKFApVIetR0A64162TjBrlk5L8ftmRXMxW7MzcgWnaqKLcWsLV7HrYbIISg4QZk4gaeH2DoGg+MZfnuxTlmK01MXWvZhwa8dLNMww1IWXrU6VAUfBHS8AJ0RSEdi2YxNFWhantA5KAUCnADQV/K4vFTfdx3tIe/emFy1Xun4QQUGx5uEPLSRImq7ZM0o/sobmqI1lCzgBWStvZ9Z2qRXMj2ImemTOJWoSSpqjTckLmaw1guzkjm1tzCe1tL5/7jl19hvNAgYWjEDY3elLXEZUoWWDbHJz/5SX77t397x48z+ZlfXPJxzxMfJvuOf7Ljx5VIdotDnRD863/9r3n55Zf53ve+t+7X3XHHHdxxxx2djx9//HEmJib41Kc+tW5CMDExsWTmQHYH9paD9rqXdC+TpSalhscTZ6IB3eWbiu8dDfnxjRIvT5Y5PZDiRH+ChbrDjWKTbMLkeF+CmuOvSC4325lqugGf/8FKS9ypko2lqzz5wAh3DWe2ZMPaDsBNXcHUlw7pRnp3nartU2q6q0o7VFXh/XcP8fXXZtBUlZih4PohDT/qEmTjJkd745wdTPGzD4zSWGZluZzF3ZJC3eXFiRJN12e4J0a1GclSXF+gqpGk5f6xLGO5xJJrNlVqUrUje05LV5mvR0mCyq3ugdeSdIW6QrkZzWqksnGs1jUwVbWTYPz/27vz6MrKMtH/3z2eOXNSSWqesAaKAqtkVKGFhos49G3bAbEbRHpdlKuUs8C9F7tbW+9aatOunxf7Koo2KHJFbdtWEGgpQGUswGKsec48nXmP7++PfXIqqUpSSWpKkeezVq1VOWef1D5v7ZO8z37f53nKflTCczhHQSmFFygMovKqfhCtgpTdgEAF5B2fTMzAMg0GS261clRUTUij4EYJxDoaOSdKyNU0jUzcouBGfRaGS4VGW5x0YpZOwtRBAy8IaauNc8mqFpY1Z/jT3qHDrp3+gsuWrixOEGLpOrXxqBrVcNWfOZkYThBSl7BwvYCUbeCHYXVL2/B1t6Ahya7eAklbB6JytrapY1SqWQ3nVIQqaqw3MmBc0VbDl//rGXzrd9voKzi01SZozsQoewFbu/Nyg2UKbrrpJj71qU9Vv85ms4fdLJyMuR+9Ez128P9IM2R1QMwsp2xA8PGPf5xf/vKXPProo8ybN2/Krz/33HO56667JjympqZmVEAgjr+TVetezGzVSkOxBMYY10JbXYLevMviphSDRQ/HD5jfkKSl0ocgW/JwvPCw4HIyK1OXrGrhwZfGLol72pwoAXRbV55LVkxtglVwfQJFdVtT7JCgwDJ0XN9D1/Rxt3YkbKPaxbjg+JT9g9VvVrXVYJs623sKaJrGitaJf5YN5/EUnCixulRJ4tY0jZrKpLZQ9ggVGLrOvBF782F4IlvCtqJJq6YzqhFZNPmmOtkuOAGBAlNX9BVcDF1HqZCOrEPc1LENjbwboBMFBGGl8Vq0ahDiB+CFVAMG04i6BtcnYwyVPX7wh900pm1qEiaeH1LyQjqHytVu06YR7alXRCsOMdOg5EYTd0PXcLwATddIx0wa0zFCFeUy9OYdfvCH3Xz4gkWHXTtxS+eV/f2sHXyMs3SNp9PnUnBDNDQsAxwvYP9QifbaBKvba9jZW6An52AaOroW5UAMX3frFtbzcscQtqlTm7AZLHqUvABPhdWAUdc0dJ0xA8bTWjN8/OJl1Rssu/sKcoNlGo7VVmE9lhwVEAgx05xyAYFSio9//OP8/Oc/55FHHmHx4sXT+j7PPfccbW1tx/jsxLFwMmrdi5ltMknnTekYH75gMbqmTalT8ZFWpmKmMemSuFO5blO2SX3CIlfyGCp52Cl91Pd3/RDXD1nWkh53a0fB9bEMnVVtNWSdKIG4PmlRk4h6DvhhOOnqXMOrJU/t6mOg4JAe0bdg+HzmNqSYk4nx/N6DqzEjA6jGdAyloDNbRoVg6hqB0ghChYYiUFFlIh0NtKi1salrlL2w0pwrmq0vaEji+AE7ewuVO/VatHUISNsG+bLCH1Ey09RBhYpyGJBzPOKWXt12s3JhAy8eGGJbd74aQFiGRjpm4IfR+9KIJtmGrmEbGnEzqm6UsHTaaqPtOP2FqAHZGXNr2dZT4LcvdXH9hUtHXTsDRYeenMMaovE+e1ED2/qimxsFN8Ayoq1Ty1vSLGhMURM3eWrnAKYRNdiLW+ao6y7Kf4gqMkW9HMJKH4coyMo7PknLHDdglBssQojJOuUCghtuuIEf/ehH/Nu//RuZTIbOzk4AamtrSSSiX5o33XQT+/fv54c//CEAt912G4sWLWL16tW4rstdd93Ffffdx3333XfS3ocQYvImu7Vnfn3ysMnOZCbpE02cXu3MTrsk7nAS8liTseEcgN6Ci+MHoxquuX5AZ9ahtSbOe9bNHXcC15tz2N1XYEtXDk2LtuQMN1RrSMWmVJ1reLXkpY4hBkoecyydUEX7/vNln4RtVCrqWPQVDq7GjAygLlk5h9++1Mkv/3QAx48as/mhQmnR3XG/Moc3TVBKQ2lRJRyIuv4qFPPrEyRjJroe7ef3/Kjcqa5BwfGjHgAaozole1HBn2qycNI2yaQNBiqVfhKWQco2cLygclddqybb5lSUZKzCaKuVoYFlGmi6oiFl4x7y/nVdHxUEjrx2XjwwxL1P7oR90bnVp2zelEnQWhPnxQND0aqDFuVfHBgs0TFUYkVbDX+1bi4tNfFR10gYKtbMrWVnT4FcyaMxHcOu5If4YUiu7GObBmfMmzgXQG6wCCEm45QLCG6//XYALrroolGPf//73+eaa64BoKOjgz179lSfc12Xz3zmM+zfv59EIsHq1av5j//4D97+9refqNMW4nVroknvsTKdpPOpntd4E6fplsQ9NAk5bhosaUpx5oI6mjJRk6s/XxW9J4Ci45NzfFzfxfUVbTVxPn7xck6bM/ZWn23dOX69uQM/jPoONGeiTsDduTLZsseylhS9eZdV7TWjkk4nsqwlwwfPXhBVVnICSm6Aoeu01MRZ2hwFGbmyN+ZqzPD46jq82pXjhb2D1a1C7ohgAMDxFXFLZ0FdglApMnGLonsw8bns+vQXPVprE7heQHfejSomETUTqxQsGpVcHG1PiioYOX5IQ9omW/IYKEZ/6pI2QyWfUIVRMEJURSokyg2Ith3pnDG/nqRt8syufgqOh2kYo94/HB4Ejrx2HkweLEEK0SrSgsYU6bjJywey7B8s8dSufvxAkbQN4pbB5n1ZLjs9Mer603WN/3J6K692RmO5byBqVleqrBxpmsb8+gQr2mom/Xmb6DNxIj7HQoiZ65QLCJQ6cmfFO++8c9TXn/vc5/jc5z53nM5IiNlrrEnv0uZ0taHWsTSVpPNjeV7TKYm7rTvH938/Ogn5wGCRX/7pAPdt2sf8hiRN6RhLm9O8bUULr3bk2NadY7DkoWuwtCXNX71xPqe1HjzXkRO2hGVw/4udDBQ9zl7UwAv7hhgseqTjJjFLZ+9Akd19BeqSNnHL4F8e3THp937+0ibevqaNp3f1M7eS7DucvH2k1Zjh/6cNlyznR0/sYePWHnry5VHBgE60f1/XtEpDMIOyF2CbOmGo2NdfxAsVWqV5WW3CwjR0OobK1XKj0WpIFBIYuh7d2dc1UrZBwQsougElJwpmIJr4Jy0z2krkKQIV1eZ3A0UQRFV84jGD2oSFG4QkUCxqTtGSjlGXtKvvf9h4QWBbTZzGlM3wWpGqBDEAdQkLTYuCifpKZ+I5NXFKXsCLB4ZG9Zw4dCy/9btt/O7Vnqgqkq6Rsk1aamM0JmP856vdLGxMHvH/dqLPBHDCPsdCiJnplAsIhBAzw1iT3qLrjzu5ORYmsyd6Ouc10d3Rqa5OjKznP5yE3F9w2dpdIAgVSincIKQ2YVXP6erzF/KuM9snfE8jJ2xBoNg7UGRFa4bGdIwz59exrTtPZ7ZEX97FDxVxU+esBfU0pe0p/Z8M35nuGCpXxtAgUIrSGFWaJvp/uvntKxm6z6U7W8bQFLoeNfiyDJ1UzMQPQnJljwUNSRK2wb6BYqUCUbRVpykdwzQ0+vMuZS9A16ExZVNyAsphSMzQ8IIotyBu6iRsE00DO1BoGnTnHZa3pKlPWph6lLSbsqNmZ7ahUfaCSj4DWKZOOmYxvyHFGXNr2dpdQCdqejc/Pvkg8IEXu9jVV2C4zMUfd/RxWlsdZS9g8/4svXkHo9Lz4MBgmYRt0JCKjepOvKQpDTAq+FvSlKJvnkt9wgJdoz5hVUuUjnzdeP8nE30mXumM6uoHoTphn2MhxMwjAYEQYsrGmvRCVHnn0MnN8dg+NN6e6Omc12RWE6ayOnFoPX+lVLVqT2NlT/pg0QNgeUuard15Hnq5m+svXFp9/ZbuXDUw2NGbP2wyt2+gSF/B5bWuHKlKU671C+t4fFvUWbkuaVJyAzJxk0zcImUb/Gn/EHc9sZsPX7B43Lv7w45FCeA/7uzjyV0DhCqqtW/pUSMyN1DViW6oIOf4nDGvls4hh+bKVqpQKUwjKiuad3wKjk8yZnLWgnq2defZ01+M8iVSNnnHJwzD6takhB01d0vaBqahoxHdne8YKqHrGgnbJGZqFJzobrsfREFKY8qu5gi018Wj/hKWMakgcOSEe159Eio7VruGyuzPdqNrUc5CyjbIxC1Cpdg/WCRbdjlrQT0NqVg1L+EP23t5Ye/QmMHf/IbUYeN8pKT2iT4TKdvggZe6QIPLVs1Br6yonIjPsRBiZpGAQAgxZYdOekc6mso7J/q8prKaMNmKLYd2P86VfQaKbrVqj2XoFBwfNwhHndOhE8HhnIO+gsO+gSJz6xLVffL1SZv6hEWh7LO9p0B90ibvBJS8gMZ0DFCYRnQnur/gsK0rz/6hEi93ZNnRXeDsxQ38lzWtE07sj6ZCTRgqHn6li5LrY1XKgWpa1NU3rDRR0wgxNUW+7PPMngG8MOScxY2k4xbbuvMMFBy6cw5lP8Ayoh4Lr3RmKxWLoOQFxO2A1toY/XmXnONXJ94NSZN3njmXriGHA4MlrMoWpZil84bWNF1Zh2ypgB9Etf0XNCZZM7d2VI5AzNS5Yk0bW7vyYwZFS5rS7O0vkit7/OK5A/Tlnag7dRhQSDUA8LYlLTyyrR9d01jRmubJnQMU3ahZGxoMFj0MXefC05pJ2AbbuvP86Kk9KMWEwd9IR0pqf2Z3P5v2DNCQOjz/Je8EURdoFf29JqFXnzuZn2MhxIknAYEQYsoOnfQeaqJJykw5r+msJkymYsuhSchuEEZ3oePRj1svCDF0HdvQq+c01kSw6Pps3NLD1u48mbjB/sFStYrQkuYk9akYBwZL9OUdcmW/+u+YMYPBYtSd2QtCntzZR18lKdcLFDv7CnTmyrzalWPDJctZ1pIZd8vUoe83DBV7+4tHDBD2D5boGCqTtk2yjk+gFKamYRmVJmBO1CvCUeCEHqYRdQHuGCqzPG7xpkX1HBgs8ccdfYRKgVIoTSMdszB1yJWiBmNDRY+YqVObsGivSzCnJsa+gRKGrvP8nkH0Svbx0pY071jbTvdQmRcPZImZUb5BYzrGirbMqAZrSim6sw5lL6AuafHf3rrksNK1O3rz3P7Idrb35OkvumzvztOSidGcidOQskmvvQyAbMnDNKLKQK915Sn7USUlFUbdl0Pgtc4cTWmbBQ1JevMOmgZr59VVz2es4G8y+QzDK1+b9vTz0oEstQmLfQNllrWkaUhFic9RM7Qo9Xu4Mdp4nxchxOubBARCiCmbbuWdmXRex2uV49AkZNvQMY1oG4ttQL4cTdYzlQCh6PhjTgS9QDFQdCl5PnFLo7UmgReGdOfK5ByPxU0psmWPvrzDQNElEzdRRKVIMwmLJc0pXtw3RFfWwdC1SvMrRSZm4gQhL+wd5MdP7uH9b1rAgy8fOaF0KonaBTe6W99aFyfXncf1QwzLQNOiHgAJy6g2AJtbm2DN3Ayb92fpGCpTcAPOnF+HbUa9BABilokfKixdI2YZURff/iJBqFjclGZ+fZJc2WPT3kH68y6modGbK9OYibG4MUVX1qE330tz2gYN4pZBbdLCDxUp26hu7do7UGR7d57OrENd0uTHT+7h6Z0DXLyihazjsbO3wNO7+nl6Zz+DJY+22ji2qbOrN89g0eP5vYOcOb9u1IRbKUXB8bHNqFObHyhsM1oxCUOF40ddtgeLLqCxpCk16nrMxM3Dgr/h/IFD8xmGA7tXOrP8xwsdOH5IYypGbcLC0DV6cmXyjl89xygojf6t4QB1vM+LEOL1TT7lQogpm07lnZl2Xlu6c8dllePQJOTWmhi1CZPOIQdTh2TMZGlzqjoJ3dFb4NCJ4HDegReElcmziroZWwZ2Sqe/4NKbdzmtJc1rKto+4/rR5M3XNNbOq8XUNPYNltA0iJs6ZT8kFTNJx03SQFfW4ZHXeujJO3jBxAmlU03UTtkmCctgbl2C/kLUq6Dg+MSsqOfAUMkjUBA3NLww5NXOPF6gCMOQoZLiT/sGaauJVTo467hBdO62eXBVpSEZ9UMoOD67egvs6S8wWPSBaEtR1vEZKHl0DJZZ2ZZhe0+BhG1wwdJGUrEMBwaLPL1rgI1belnRmqErV2ZXTwGn0jU6YcbRNPjP17r41yd2A1FCeNkLMQ2d85c2kIlbKAVxazgvwWd7T566RB15JyBf9im7PmUvIFTRtWESJfAaugYaGHrUYGyg6DG3LkEqdnCyP7zy01oTZ6jk0l9wGSi6JGOH5zPs6M3zwItdbOvO8VJHlnzZZ2FDkqaMTWMqRneuTH3SYqDosb0nT32ynnTMwNCi80jHRnfKPpmfYyHEiScBgRBiyqbTF2CmndfxXOU4NCk3ZhroGhi6zvKWNDUJi1zZi7bVxEyUojoRhIN5B3XJqLfAQNGl6PrVLrWpmEFf3sEyNK5Y08o717ZT9AJ6cg6/2dxJX8HFD0LKXvQeypXqPg0jtpvUJEx29xY4MGhx/tKmcbdMLWpITXlr1cjA7JzF9by4P8u+wRIl169UEgJLh6a0TW2lHOdQOeoXoAF9eZfevBNtgwoV6ZhZPfeS69OXdxkqeyil0LUoMdm2DMCL8ghMg7imEYQhOcfnqZ39pGwTy9BQCgxdY35DiqRt8tjWXp7eNYBX2T7TlLLJJGwKrs+zewbpr5xHbcKiOWOzt79M0fV5fFsfmqaxqDFFQ9KmO1cmFTPpGszT37mRUMEL9ll05z2CMOqfYBp6VC5Vg7CyWmBWVj1qEhZJ26To+nhBFBAOFKP/x2h1Jwq0Sl7Art7CqHwGoBqwpWNGlEidtOjJO+Rdn8VNKXJONL62qdObd+gYKpF3gmpp2209hRnzORZCnHgSEAghpuVYVKI5med1vFc5Dk3K7ck5vLB3kB09hVETujXzavn5pv2jApPhfIBMPLorHoSwb7CMZWiYle0/QRCyrCXNZae3sqAxqj6zojXKQXjgxS5+v72nMuEMScctGpI2CfvgXWAvCPFCRUMqNuGWqU17B6a8tWpkYNZXcDlrQR1r5tXSlS3zzO4B/NAjaZtkywF5p4ypU0k0BkMDRTRZjv4vKn0LdCg4URBV9qJk4NqERUPK5vm9gxTdADcIqUtY1QmsqrQvLnohXuBS8gM27R1k7bxoy0xDyqYxZZF3fNIxi0zCIlO5FsIw5MUDWfxAUZ80CULwA6p9ALLlgOf3DrKwIcXSlmjC3Z93yJYcbL0MwEDRRdc0AiBQYIQhQaWtmmloxC2DhpSFoevUJSzaahNs7YoCgbIXkI5bWHETzw/pyJZprYlz1TkLRnU1Brj9ke3VgK2v4BKEipqERTpGdTVp7bw6dvQU6C04ZEse/QWPdQvrqwHFTPscCyFOLAkIhBDTdjSVaE72eZ2IVY6RSbkrWuGCpU2HnRPAn/YOjQpMhvMOciWPvrxL3NJJWAauH+IHirIX7Ul/y2nNh03Yht/7+kV13PSzF3H8gPZDJvNKKYaKHqauMadmdNWaYcNbpvoK7rS2Vo0MzIabrg0U3WovAsvUsQ2NIFQMlaMk74SlEyqoiVusX1TP3v5itcRoyfHpzbs4QUhdwsTQddrrEtQlbSxDx/FdFFGzM50owMg7Pn54sDmYrmkMFd3qXn9T18i7AbapESqFrkXVjGxTJ+8G+JVKUGgaKlTV76HQSNgGg0WPjmyJuXVJ1s6v4z9f6YoqCFW24/sKErYJboAbKNwQbE2hAFvTmVsXp+xFgV990uZtK5t5bu8APXmH1poYpg4FxydX9qmNm9QkTF7cn+X6Zc3V63Jvf3FUwDYyZyVm6qTjJv0FlzfMybB+UX2lv4TDf7twCesXNlS/z0z8HAshThwJCIQQR2UylXdOhsmc14le5RjvnA4PTHSSls5r3XkMDRbUJ0nYZhQQhIpsySVumfTlXMJQHTZx03WNNy1q5KI3NPPgy1305aNEY8vQ8YKQXCnqg9BSE8fUD08mhYNbphpT9rS3Vi1ryRCujnIGhkoePbkyrh9gVyrv2IYJlao7QagouNFe+ygB1mF+Q5KCGzBYdJlTEyPv+NQlo3NIxiyWNqcx9aikqKFrBCFR8KJFgV1Y2avvBYpQRcFLSyZW3Ue/qDFZ2ecf4oUhBTfA1HUSlo6hR9uLdJ0oqNC0qAGapVNwA2KmRilU1cRnU4vu/C9tSsNg9P5jpo5tGdi6Rn8xypsAjaQdbevJlaNeEUnLZPmcDIsaUzRnYpi6Rn/RJVuKSrRahoZpWBSdgE17Btg3UETTNAquT+dQmZLn014J2IaDi55cGTtljypzC1G+wrqFDaOCgYmuTSHE7CABgRBiVpsJqxyHBiaOH5CwTSxdJxkzMAydylySshdQm7RZ3pJme8/4VZB0XeOD5yygO+ewpTNHrhwl3IKGoeu8cWENc2ridAyVyUzQjfeN8+t5eufAtLZWbevO8YM/7Ka/4NKYton1GcQtk0ApPD8kj4+mgVcpw4kCS9doTMfoyTn05l00LZp0b+/OU/JC0jHFvIYEa+ZG236UUtSnbHryDroW5VgUvQA3CNA1CMOod4OmUdlOdPCueco2GCx5WHq0fcfxArRKki/VBG/wA0VN3CRmRZ2F3SBKktY0sE2dXNljW08e09BY1V5TDQiSpoHSNQzdoCYBQ0WPIFSU3QDTiAKD+qTNgsYkl66eQ9ELsE2d0+Zk2LRngKRtkombpGwTLwwZLLocGCrxjQe3oGta1LgsVOztK6JCaMzEsA2dpc0p8o5Pf8GtVDSKVj62duclL0AIMSYJCIQQs95MuDt6aGDSOVTmjsd34vkhgyWPguNj6DotNXGWNqeoSVjs6i1MWAVpWUuGDZcs5/4XO9m8f4iiG5C0Dc6YW8dlpx9MRp1oy5Rp6tPaWnVon4e+QrSfPh2LEmf1SpUlxw9RqlL8UoNMwqQxZVPyAnb3FdE0aM7EWN6S5uUDWRRU8wsgymNY1VbDnv4i+XK0gpB3fIqOT4CGoUPSNrANHT9UOH7Uobjk+rzWma3M+xV+EOL4IWU/xDY0dE2v/lt2XKMhFSU1R6sMNjv7fFJ2VD51sOixqr2GuGXghwfr+YdQ3a5kmzqZhImhaZS8gIRlML8hwZnz66urUXv7i8QMnS1dOYJQjcrbiOkGJSNgYKjM0zv7OH9ZE4sbUxwYKtGVc9jVX6QxZZGKRfkii5uS9OQcdvcVycRN/CCUvAAhxLgkIBBCiBliZGCSsk2a01ENeYgSjW1Dr97Nz5W9SVVBWtaS4WMTrIBMZsvUdLZWDfd5aK2Jkyv75B0fpaIJvxeGhF4I2nD9+yg3wtChOR3lNAwUPFDRVp+6pM1pczLkygFd2RIlL6iWztS0aLK+sCHJ/sESZS/E0jVsUydm6sRNg4a0zZLmNL05l/6iS9nzK1uWdDIxM+pyrBTpmIHjh7iBIlQ+uq4RN40oMAhD/FCj5Ab0FVzaauL87VuXsHZ+HSnbpK0mzr88uoMntnWyrDIGrh9i6wf/nolbzMnY7O4vccHSJj596WnMq0+Oqs7UnInx5M5+mjOjG5AVHI89/UWCSn+KTbsHySRMCk5A0tJx/ICCE5COmXRlS/QVHJozMd60uIEr1rSxsq1G8gKEEOOSgEAIIWagkVWQRpb7hKlXQZpoBWSyW6amurWq4EYN1w4MlhgseXhBECUPK0VjyqZoBAwU3aiOv6lTRhG3dGKWQdkLom07uoamoL0uga7r1Wo+2VLU22Cg6GIZOh1DZVa01XD9RUt5eucA27pz7OorUHIDFjalKt15YyxqVAwVXV7syBKEIdu7C9WeAH6oouZnhk7KoJLcbHLtmxfzu1d72NVXoL8Q/XtvmJPh6vMXcfHKOaPe82Wnz2FL5wB5LRnVElKQLXtoaMQtnXTMoDfvkomZvHf9vGp1qJH/T+sXN3D/S53kywGapmFVksv3DZbwAkUqZqBrUZWiPX1F/FCxsDFJKmbRW3Bw/CiJeqDoko6ZvOesebx5eZMEAkKICUlAIIQQM9CJ7PUw2S1TU9la1Ztz2NtfRClFXcomEzexDJ0Dg2W6sg7NaZvGlI0XKkxNp9mK9vaXvZCi6+OHUTOypG1UVw0aUjHOnF/H1q48eweK7Oor0JyOj1qpuHjFnMO69VqGjh+GdAyWKr0FolUCL4wKAiVsg6Rt4AdRgvacmhg6GjnH5/S5tfzNuYvYtHeAvoJLY8rmjfPrMc3Dk7GXtWS44eI3cNOgy5auHH4YElSqHOk61b4Qb17exPlLm8Yct5WtNZw2J8NAwaXgRs3NBksehhblHMQqZWh1TWM4Ibs759CSsamNmyxsTNJXcPHCKGi8+6ndbN4/NGZHaSGEGCYBgRBCzFAztdfDkYSh4vk9g1imThAqbCNKbK1N2Fi6zr7BEt05h7baBBkjujt/9qIGGlI2uXKUDLt5/yB+oKJj4gd/VTWkYqxs06hNWlx59gKWNqdHrVQMBy3zG5IsaUpVx25bd57tPXlyjg+awtKjUqOBiiolRbX7zahcaTkq65q0DdIxE9PUOXtx46Teu65ptNbGOTBURiNaZVAosmUf11esnpvmg+csGDeQm1uX4Kz59WzeP8iamjgDRY/N+4ewzajykuOFZBIWXhBS9KL8i3LBpej4UeM2LwqA6hIWRdcnaRnjdpQWYqp2ffWKk30K4jiRgEAIIWawmVAFaar2D5bY0Vvg9PYatnZHW23SlRUCw9DIxExCBddcsIgFDUl+vbmDvkpFnGTMACxMXccLApY0pQ7bLtWZdVg7r463Lm+ecByGx27vQJE7Ht/J3v4StXEougGaaRCUPQiivgBeEFZLqA6WPFoyMeY3JMcstRqGasz/D98PuffpfQyVPM5Z1EBXrsxApbJQbdwiBFozcZY0pSccv7Xza3mpY4idfQXqEhaaprAMA6WiOlGmBj15hyBQGAYQKspeQKCivgUJS6fomqRjJnXJaHVmrI7SQggxTAKCCbiui+u6hz2u6zqmaY46bjyapmFZ1rSO9TwPpdQJPRbAtu1pHev7PuGIChtHc6xlWdVJwPE6NggCgiA4Jseapoleqec+E44NwxDfH7/6jGEYGIYxY45VSuF53jE5duTn83gdCxN/lo/Hz4g5aRNNs06JnxFDhRJl12FxY5pUzGJbd9R9t1B2sXSN9hqLmKlzRnua0+ZkaEoaPPRyNzv6y9VVkAuW1tOTK9OfK2JpYXW7VFc22i7156taqhPbI33uNaArW0bXIWXolMoucVMnMBSOCgkrycueF2LoYBsadUmbs+bV0Jw0Rv2f7OjJR+fam6ccQNwyWdqc5rSWJE/t6OWhlw7wJvdPaBpozeewsiVFMmZgmxZoMFD02NOXp7328IZw1e/dVyJfjnIwOgYKFEsuugppzZj0ZB26hgpRZaZQ4YY6Cg1DA02F6Cg8L8DzPBzPoDdbIGOnaU2bbOvKVsvUzoTP/an4M2Kiz7IQpzIJCCbw9a9/nXg8ftjjy5cv54Mf/GD166997Wvj/kBZuHAh11xzTfXrf/7nf6ZYLI55bHt7O3/7t39b/fpb3/oWQ0NDYx7b3NzMxz72serX3/nOd+jp6Rnz2NraWjZs2FD9+s477+TAgQNjHptMJvnsZz9b/fruu+9m9+7dYx5rWRY333xz9et7772XrVu3jnkswK233lr9+89//nNefvnlcY+96aabqpOOX/3qV7zwwgvjHvuZz3yGVCpKznvggQd45plnxj32xhtvpK6uDoCHH36YP/7xj+Me+9GPfpSWlhYAHnvsMTZu3Djusddddx1z584F4IknnuChhx4a99irr76aRYsWAfDss8/ym9/8Ztxjr7zySk477TQANm/ezL/927+Ne+xf/dVfsXr1agBeeeUVfvrTn4577Lvf/W7OPPNMALZt28aPf/zjcY+9/PLLOfvsswHYs2cPP/jBD8Y99pJLLuGCCy4AoKOjg+9+97vjHnvhhRdy0UUXAdDT08Ptt98+7rHnnXcel156KQBDQ0P88z//87jHrl+/niuuiJa1i8UiX/va18Y9du3atfzFX/wFEE1sv/KVr4x77KpVq3jve99b/XqiY+VnBMwBirsMGs59H29aVE+u7BNueww92wXZ6Jj7vjf6c/LJj3+2etf9jw/9Cra/Un2uNOL7Aix8503V5470M+Ldf/PfKLpRH4al7nbOYz/4RAkE9uhjf2+vJ0+c+Q1JmnNb+d//+54xv2ct0HrG5ZRNiyd29vHsk4+z0N3NRcMHKHhD9yPQHX2ZWnMpKlVPd87h+Wef5gdPPjbu+dYufyvt89opOD69O1+iKXgFCiMOGHHODzrL2BfWoYAlRj9vtneN/mbbIbe98vfWN1JwoxpI8jNiej8jvv71r497rBCnsrFbVAohhBBHKVTRXVVN06hJWNiVO7jjmd+QZEVrDfMbkqO2CR2tpG2StE1A40jf1QlClrWkueGiZdQl7QmP1XWNlG2SL3uU3fFX7wAUqrolyR4jIfnQ8zX0aMzm1k+cxG3q0QpIMP6Czohj9SOWqRVCzE6ammhdeJbKZrPU1tbS09NDTU3NYc/LlqGxj5UtQ7Jl6FTfDjCTtwyNdexM/hmxoyfPvz6xm/5SWK2QVCy7dA2VqE/Z/PW5C1nSPHov/fH6GWEYJrdv3M4vnz9AGPikYgbdWQcvCLEMDcePtg3VJWzWLmjkhouXcdqcmlGfuX0DRf6//9xGbcKq5hUMlHxe6sixrTsPKsBAUWNpvC14GoBXmy/ECTXKXsjZi5voyrusmVvL3755EUodPN/Dvreuo2nRZ1mFIR2DBf6wrZeGtE2u5NNXdLB0nbipM+SGlFxVaYIWEnVNICpzapu4YUhd3GJBY5I3LWnmoxctR9e1GfG5PxV/RvT19dHc3MzQ0NCY84NDDc8n5m+4Fz12cpsfHguSVHxqGb7+JnO9yq2CCdi2PeoX1ETHTeV7TtbIX9CnwrEjJ0CnwrEjf4G83o7VdX3S19pMOFbTtFPqWDh+n/vXy8+IFXMb+PBbrMMqJJ0+v3FSFZLG+iyPTub1q8m8k/nc/5fTW3m1M8cLewfxyiGZVIyhkk+27BMqjYZkjLetbuWD5yyontvIz5yrypRDjfZEHE3X6C84PL9viMGih6ZBwrYoeSH+iCUIJ9TQDJNiyWVbb4GFjSkuXT0Hyxp9vod+b4gmn7myjxuElAKNcqCBbqKboOshTqjQQo2yHwUDABo6w9PrggduEBCzDLIepBJxLju9bUQ1ppP/uX+9/4wQ4lQiAYEQQojjYrIVksar2jPStu5cNbgo+wFx02Bpc3rS9fWXtWTYcMlyfvTkHp7Y0cdQycPQoL0uzunttbxn3TzOXzp+A6+UbRI3DYquTzpmsr07anzWkLIpugF+GG3JaclYMBC9puwGFIOAIITV7bW8Z91cYqbBq53ZUe9z5PfOxC36C241EdsPQgpuFLgYQyXa65MkbZMDgyWGSt6orUKaBgagaxCE4IVghIp56RgfOHuBlBwVQoxLAgIhhBBHbbxJ/ZGamU1mor+tO8f3f7+L/oJLW22cpJ2g6PpTrq+/rCXDzZev5Jk9/WztyhO3DN60qJ4FDakjluIc2Tl6TiZGfzEqpWob0dadwZJHfdLCtg6u4q1sz9CVD1jVXsN/PWsuD77UzfaePCXPJwyhrTbBJataOHdxY/V7u37IC/uGKLk+6biFGTPozTmgQclXBIEiFTNpycSjQMQ/uPVI18A0dDQUmqYIlYZt6LxleRMXjNMITQghQAICIYQQR2m6d+8nM9Ff0pTmgRe76C+4LG9JV/MGMnGLdGxq9fW3dGX56TP72d6TJ1Ah9QmbrqEyZy6ooykTm7DHw8jO0dsqk/pULIYbhBiGRswy0DUNNwhx9TiBUnRnXRY213DO4kZ++Mdd7B8sYeoafQWXXMnj+X2DbNzSzZ+taOGCZU3sHyzy1K5+HC+kOWPjh4qenAuaxty6OP0Fr1I2NIFhaJi6hk6UVGwYGkopQqXQNQ3LiBqvJWyD9YsapPeAEGJCEhAIIYSYtunevQ9DNamJ/hVn6GzvydNWGz+s8pCmabTVxtnWna/W1x/Pw6908c2Ht9KTc7BNnZip05112LR3kPs27WN+Q5KmdGzCQGa4c/S9T+9ld1+RvrxL3DKYVx+9tidXpjvr8DtzPQrFO9rrePNpzfx8036e3zsIQH/BRSlFKmbRlLYplH0e39pL2QtZO7+O5/cOEhhatKVJ16lL2iigORMnZhh05x368h6KEKUUtqlhGjqtNXHyZZ+SH1DJK8bQNZY2p1nZduTkVyHE7CYBgRBCiGmZ7KR+rLv3+wdLk5ro7+wtUPYDknZizHNI2AZd2TIFd/xqNVs6c3zz4a10Zsu01cSxTJ1c2WffQNTdIB0zcIOQ2oR1xEBmWUuGz122AtB4uWOIZc1pahIWA0WXnmxUXtTxA1Ixk5cPZHlm9wCvdeVBqWgLlRads+MH9OQUDSkLpWDfQAHHD6hPWszJJIjbOjHTQKF4Ykc/ubJH3vXxgqgikm0a1CQhX/YJVbSNqCFl4/ohfhiSK/vYpsH5SxuZWzf22AkhxDDpQyCEEGJaJjup3z9YOuy1BdevTPTHvi81PGkGqgm3Yxmu7T9eff0wVPz02b305Bxaa2LELAONaCKta9Fd9BAYKETlXpe3pOkvuPz2pS7CcOzSq6ap8743zWNhY4qunMO+gSKb9gxwYKhEEEJ9KoauaWztiQIapRRxS8fxQ9wg2vOfsAy8IJq4l9yAnpzLH7b3sq27wAv7B9nZW8QPFTVxi7ipc2CwTM7xMXWdprRNXdLG1HX8ENA0BosuZT8gBHLlgFDBG1ozXHZ665S3C4WhYm9/kVc7s+ztL447DkKI1w9ZIRBCCDEtByf1U797f2hlnUMNT/SXNKWqCbfpmDkq8FBK0TFUZs3c2nHvgg8HLbapYZuVEqJ+SMmLSnICeH6I4we4QTjpbUjD24fuf7GTX2/uoL/gUpewaEgazOt5ilDB1tqz2NpbrvRp0DA0DRUqSl5IJqZjV1YqANAgZuhk0ha5kkd3rkze8Vk7rzZKEkDh+4rapIVt6vihwjY0ahPRyoBSir58FNTUJmzOW9LAledMvrLQcFL4Sx1DPPJqD925MqGC+oTFspbMpKs5CSFOTRIQCCGEmJbJTurHuns/smrPRBP9efXJajLv1u58tclZyQ3oGCrTkLK5dPWcce+CF1yfQEHMjO7Ix0yDoJJ8a2g6oCiH0YTdNqJF88lsQ4IoKHjnWp3N+4Y4bU6G+qSN8j20zlzlCA1L13HCAM8P0bRo5cQPQoIwqgRUcgOSMYP6hIUTKJa1pNnWnafo+GRLHi8eGCJb8kjHLYJAEbeMan7BnNoErTUxlIIPnD2fkhetPixuSjG/PjnplYHhpPDHt/XwckcW1w+JGTp1KYtcKkZvwZ1SNSchxKlHAgIhhBDTMtlJ/Vh370dW7TnSRH/4bvyhTc7WzK09YpOzlG1Sn4juug+VPOyUjqFp6JpGoBQoCEJFQ8omE49+JR5pG9JIJS/AMDTm1ScxdI3erMtw6ypD0zANjVDpGLqG5ytUqACFH4a4XggoGlM2BTegpSbO/Ppktc9BV67M/sESOrC4Oc2y5jSWoeMGIbahk4mbBEqxq7dAbdLmnNapJw8PJ4Xv7i2wtSuPHyiStkEYRo3RQgU1la1bk63mJIQ49UhAIIQQYlqmMqkfy1Qm+pNtcnaouXUJlrVk6C24OH5Af8ElHTOJWzq5SmOvdNxkVVsGTdMmtQ1ppENXSWz9YGqeZepYpo7rh8zJxBgs+QyUPIJAVVcM4pVtSwnbYGlzCk3TaEjFqF9kM1j0eKUzi2VovGFODTWJMVZhHH/SwcuhhpPC+/JOdftXOmZiGhpKRcFOWNniFHN9tnbljljNSQhxapKAQAghxLQdzd374ddPdqJ/pCZnYxkZtAAUHZ+c46OUwg/BMjTWzK2hNmmTK3uTCmRGOnSVJB03yVeeU0ph6zoqZuIraK2No2vRv6tpYGpRQnN9ymZVWw0NqVj1+2qV1YV5dQmaM3H2DhTJxKeeQzGR4fyKTNzk1U4PQ9eo7JpC08A2dUpeSE3CIlf2GSx5R9xGJYQ4NUlAIIQQ4qhM9+79sOlM9Kd6fsNBy7buHIMlD11L0JCOkbYNhko+u3oLUwpkRp77yFWS1vTBX6v7BkpkknHWtNTSOVSmY6iMZRqsakmzrCXDWQvq2LR7gD39JeqT9qjvO3Kyf8mqFn7wh91s6Yom74auEVS29DSmJx+8HGp4VaAmbqGUwtQ1QgVG5VsZmoanokRr14+qMk1nJUIIMfPJJ1sIIcRRO96T+qM1XtACTDuQGfm9q6skXYPUVh9PY9vRRH9BQ5L1ixpYt7CelW011X9nWUua7/9+14Rbrpa1ZHjbihbu/P0uXjowhBeEWIbOosYU710/b9qJvsPbnYIwJG4ZOEGI44UkLB2tkmOhaRphGOL6iqUtaelpIMTrlAQEQgghZoXxgpZjEcgMBxy7eoa4d08CDfj2VevoK4cTBhuT2XK1rTvHf77aTSpmcO6SBgxdJ6g0H/vPV7tZ2JicVlAwvN1p8/5BGlI2JTcg0KOcAdvQcfyAuGXQX/Boq43zV2+cLwnFQrxOSUAghBBCHAO6rrFkTh1f+Pznqo8lJxFrTLTlamQ36NPmZA7LIZioG/Rkznd4u1PBCUjYBkpB2Q8oOD5oEChFe22Cj1+8nNNapeSoEK9XEhAIIYQQJ9l4qxdT6QY9nZWOkSsUz+0dYG9/EYUiEzdpycRZO7+O96yby2lzpl7SVAhx6pCAQAghhJihjqYb9GSNXKHIOR75sk86ZpKJW9PKqRCHe/HvLqOmRoIqMXNJQCCEEEIcI57ncffddwNw1VVXYVmH9w6YiqPpBj0VMz0pXAhxfElAIIQQQhwjSil2795d/fvROppu0EIIMVn6kQ8RQgghxMkwnPjbkLLZ2p0nV/bww5Bc2WNrd35KTdSEEGI8EhAIIYQQM9hw4u/p7bUMFj129RYYLHqsmVvLhy9YNO0+BEIIMUy2DAkhhBAz3NF2gxZCiIlIQCCEEEKcAiTxVwhxvMiWISGEEEIIIWYxWSEQQgghjqGjLTUqhBAnmgQEQgghxDFi2zY333zzyT4NIYSYEtkyJIQQQgghxCwmAYEQQgghhBCzmGwZEkIIIY4R3/e59957AXjf+96HacqvWSHEzCc/qYQQQohjJAxDtm7dWv27EEKcCmTLkBBCCCGEELOYBASvA47j8MUvfhHHcU72qZySZPymT8Zu+mTsjo6M3/TJ2E3f8JhNduymevzrnVx7o82k8ThlA4L/83/+D4sXLyYej7Nu3Toee+yxCY/fuHEj69atIx6Ps2TJEr797W+foDM9/hzH4e/+7u9mxAV1KpLxmz4Zu+mTsTs6Mn7TJ2M3fRIQHB259kabSeNxSgYEP/nJT9iwYQO33HILzz33HG95y1u4/PLL2bNnz5jH79y5k7e//e285S1v4bnnnuPmm2/mE5/4BPfdd98JPnMhhBBCCCFmllMyIPjGN77BRz7yEa677jpWrlzJbbfdxvz587n99tvHPP7b3/42CxYs4LbbbmPlypVcd911XHvttXzta187wWcuhBBCCCHEzHLKVRlyXZdnn32WL3zhC6Mev/TSS/nDH/4w5mv++Mc/cumll4567LLLLuOOO+7A87zD2swrpQDYv38/2Wy2+ngsFiMWix2Lt3FMDZ/jyHMVkyfjN30ydtMnY3d0Zur4ua5LuVwGonOzbfskn9HhZurYzUSO44zaztHR0QEcnCccyfBxw68bNlPnE8ebXHujHe/xGP6+k7leT7mAoLe3lyAImDNnzqjH58yZQ2dn55iv6ezsHPN43/fp7e2lra1t1HO5XA6AVatWHcMzP/7mz59/sk/hlCbjN30ydtMnY3d0ZvL4ffWrXz3ZpzChmTx2M53neVM67uyzzz6ep3PKkWtvtOM9Hrlcjtra2gmPOeUCgmGapo36Wil12GNHOn6sxwHa29vZvn07lmWNen62RvRCCCHEbHToCoFSCs/zWLRo0aRev2jRIplPiJNGKUUul6O9vf2Ix55yAUFTUxOGYRy2GtDd3X3YKsCw1tbWMY83TZPGxsbDjtd1nSVLlhy7kxZCCCHErCPzCXGyHWllYNgpl1Rs2zbr1q3jwQcfHPX4gw8+yPnnnz/ma84777zDjv/tb3/L+vXrD8sfEEIIIYQQYjY55QICgE996lN897vf5Xvf+x6vvPIKn/zkJ9mzZw/XX389ADfddBN/8zd/Uz3++uuvZ/fu3XzqU5/ilVde4Xvf+x533HEHn/nMZ07WWxBCCCGEEGJGOOW2DAG8//3vp6+vj7//+7+no6OD008/nV//+tcsXLgQiLL5R/YkWLx4Mb/+9a/55Cc/ybe+9S3a29v55je/yXve856T9RaEEEIIIYSYEU7JFQKAj33sY+zatQvHcXj22Wd561vfWn3uzjvv5JFHHhl1/IUXXsimTZtwHIedO3dWVxNOVV/5ylfQNI0NGzZUH1NK8cUvfpH29nYSiQQXXXQRL7300sk7yRnki1/8IpqmjfrT2tpafV7GbmL79+/nQx/6EI2NjSSTSc4880yeffbZ6vMyfuNbtGjRYdeepmnccMMNgIzdRHzf53/8j//B4sWLSSQSLFmyhL//+78nDMPqMTJ+48vlcmzYsIGFCxeSSCQ4//zzefrpp6vPy9gd9Oijj/LOd76T9vZ2NE3jF7/4xajnJzNWjuPw8Y9/nKamJlKpFO9617v48pe/zOLFi4nH46xbt47HHnvsBL6rk2e2/849XtfTvn37jt9JK3HKeeqpp9SiRYvUGWecoW688cbq41/96ldVJpNR9913n9q8ebN6//vfr9ra2lQ2mz15JztD3HrrrWr16tWqo6Oj+qe7u7v6vIzd+Pr7+9XChQvVNddco5588km1c+dO9dBDD6lt27ZVj5HxG193d/eo6+7BBx9UgPrd736nlJKxm8iXvvQl1djYqH71q1+pnTt3qv/3//6fSqfT6rbbbqseI+M3vve9731q1apVauPGjWrr1q3q1ltvVTU1NWrfvn1KKRm7kX7961+rW265Rd13330KUD//+c9HPT+Zsbr++uvV3Llz1YMPPqg2bdqkVq1apTRNU//yL/+iXn75ZXXjjTeqVCqldu/efYLf3Yk323/nHo/r6c/+7M/U2rVrle/7x+WcJSA4xeRyObV8+XL14IMPqgsvvLAaEIRhqFpbW9VXv/rV6rHlclnV1taqb3/72yfpbGeOW2+9Va1du3bM52TsJvb5z39evfnNbx73eRm/qbnxxhvV0qVLVRiGMnZHcMUVV6hrr7121GN/+Zd/qT70oQ8ppeTam0ixWFSGYahf/epXox5fu3atuuWWW2TsJnDoBG4yYzU4OKgsy1L33HNP9ZizzjpLAer++++vPrZixQr1hS984fi/iZNMfucedKyup/379ytd10ddT8fSKbtlaLa64YYbuOKKK7jkkktGPb5z5046OztHdWSOxWJceOGF43Zwnm22bt1Ke3s7ixcv5gMf+AA7duwAZOyO5Je//CXr16/nve99Ly0tLZx11ll85zvfqT4v4zd5ruty1113ce2116JpmozdEbz5zW/m4YcfZsuWLQC88MILPP7447z97W8H5NqbiO/7BEFAPB4f9XgikeDxxx+XsZuCyYzVs88+i+d51WNc1+VPf/oTCxcuHDWel1566awZX/mdO7bpXE8Q9cg6/fTTj9sYSUBwCrnnnnvYtGkTX/nKVw57brjPwlQ6OM8m55xzDj/84Q954IEH+M53vkNnZyfnn38+fX19MnZHsGPHDm6//XaWL1/OAw88wPXXX88nPvEJfvjDHwJy7U3FL37xCwYHB7nmmmsAGbsj+fznP8+VV17JihUrsCyLs846iw0bNnDllVcCMn4TyWQynHfeefzDP/wDBw4cIAgC7rrrLp588kk6Ojpk7KZgMmPV2dmJbdvU19cD0NvbSxAEtLS0jBrP2TK+8jt3fNO5nsY65lg7JasMzUZ79+7lxhtv5Le//e1hd3xGmmoH59ni8ssvr/59zZo1nHfeeSxdupQf/OAHnHvuuYCM3XjCMGT9+vX84z/+IwBnnXUWL730Erfffvuo8r4yfkd2xx13cPnllx/WNVLGbmw/+clPuOuuu/jRj37E6tWref7559mwYQPt7e1cffXV1eNk/Mb2r//6r1x77bXMnTsXwzB44xvfyAc/+EE2bdpUPUbGbvKmM1aHHjNbxld+5x7ZsbiejiVZIThFPPvss3R3d7Nu3TpM08Q0TTZu3Mg3v/lNTNOsRppT6eA8m6VSKdasWcPWrVurlQ9k7MbW1tbGqlWrRj22cuXKamlfGb/J2b17Nw899BDXXXdd9TEZu4l99rOf5Qtf+AIf+MAHWLNmDX/913/NJz/5yeoqqYzfxJYuXcrGjRvJ5/Ps3buXp556Cs/zWLx4sYzdFExmrFpbW3Fdl4GBAQCampowDIOenp5R4zlbx1d+5x40netprGOONQkIThEXX3wxmzdv5vnnn6/+Wb9+PVdddRXPP/88S5YsobW1dVRHZtd12bhx47gdnGczx3F45ZVXaGtrq/5ylLEb2wUXXMBrr7026rEtW7ZU+37I+E3O97//fVpaWrjiiiuqj8nYTaxYLKLro39NGYZRLTsq4zc5qVSKtrY2BgYGeOCBB3j3u98tYzcFkxmrdevWYVlW9RjbtjnjjDPYvXv3qPF88MEHZ+X4yu/cg6ZzPUHUY+vFF188fmN0XFKVxQkxssqQUlEZq9raWvWzn/1Mbd68WV155ZWvqzJeR+PTn/60euSRR9SOHTvUE088od7xjneoTCajdu3apZSSsZvIU089pUzTVF/+8pfV1q1b1d13362SyaS66667qsfI+E0sCAK1YMEC9fnPf/6w52Tsxnf11VeruXPnVsuO/uxnP1NNTU3qc5/7XPUYGb/x3X///eo3v/mN2rFjh/rtb3+r1q5dq84++2zluq5SSsZupFwup5577jn13HPPKUB94xvfUM8991y1ROhkxur6669X8+bNUw899JDatGmTWr16tdI0TX3nO99RL7/8stqwYYNKpVLV3zuvZ7P9d+7xuJ7e9ra3SdlRMbZDA4IwDNWtt96qWltbVSwWU29961vV5s2bT94JziDDNX4ty1Lt7e3qL//yL9VLL71UfV7GbmL//u//rk4//XQVi8XUihUr1P/9v/931PMyfhN74IEHFKBee+21w56TsRtfNptVN954o1qwYIGKx+NqyZIl6pZbblGO41SPkfEb309+8hO1ZMkSZdu2am1tVTfccIMaHBysPi9jd9Dvfvc7BRz25+qrr1ZKTW6sSqWS+u///b+rhoYGlUgk1Dve8Q71pS99SS1cuFDZtq3e+MY3qo0bN56Ed3fizfbfucfretqzZ89xO2dNKaWOz9qDEEIIIYQQYqaTHAIhhBBCCCFmMQkIhBBCCCGEmMUkIBBCCCGEEGIWk4BACCGEEEKIWUwCAiGEEEIIIWYxCQiEEEIIIYSYxSQgEEIIIYQQYhaTgEAIIYQQQohZTAICIcTryq5du9A0jWuuueZkn4oQQghxSpCAQAjxunfNNdegaVr1j2ma1NfXs2rVKq666ip++tOf4rrumK+96KKLRr3WsiwaGxs588wz+chHPsL9999PGIYTvuZIfx555JETMApCCCHE2MyTfQJCCHGifOQjH2HevHkopchms2zZsoV///d/50c/+hErV67knnvu4YwzzhjztZ/+9KdJp9OEYcjg4CCvvPIKd999N9/73vc4//zz+fGPf8yCBQuAKAC56KKLRr3+zjvvZPfu3dx4443U1dWNem7RokXH4d0KIYQQkyMBgRBi1rjuuus499xzRz2Wy+X4X//rf3Hbbbdx6aWXsmnTJtrb2w977Wc+8xlaW1tHPdbT08MnPvEJ7rnnHi677DKeeeYZUqnUmNuVHnnkEXbv3s2GDRskABBCCDGjyJYhIcSslslk+Kd/+ic+/OEP09XVxZe+9KVJv7a5uZm7776biy++mFdffZVvfetbx/FMhRBCiONDAgIhhAD+5//8nwDce++9KKUm/Tpd17nlllsA+MlPfnJczk0IIYQ4niQgEEIIYPHixcyfP5++vj527tw5pdeef/75WJbF888/j+/7x+kMhRBCiONDAgIhhKgYzh3o7e2d0utisRgNDQ2EYUh/f//xODUhhBDiuJGAQAghKqayVehYvlYIIYQ4mSQgEEKIio6ODiBKFp4Kx3Ho7+/HMAwaGhqOx6kJIYQQx40EBEIIAezYsYO9e/fS1NQ05bKgv//97/F9nzPPPBPTlGrOQgghTi0SEAghBPAP//APAHzgAx9A07RJvy4MQ/7xH/8RgCuvvPK4nJsQQghxPElAIISY1XK5HJ/61Ke48847aWtr4+abb570a3t6evjQhz7Eww8/zKpVq/joRz96HM9UCCGEOD5kbVsIMWt897vf5f7770cpRS6XY8uWLTz66KPkcjlWr17NPffcQ1tb25iv/drXvkY6nSYMQ7LZLC+//DKPPvoojuNwwQUXcM8995BMJk/wOxJCCCGOngQEQohZ44477gDAMAwymQzt7e28613v4i/+4i9497vfjWVZ477261//OgCmaZLJZFiwYAFXXXUV73vf+/jzP/9zdF0WXIUQQpyaNCW18oQQQgghhJi15JaWEEIIIYQQs5gEBEIIIYQQQsxiEhAIIYQQQggxi0lAIIQQQgghxCwmAYEQQgghhBCzmAQEQgghhBBCzGISEAghhBBCCDGLSUAghBBCCCHELCYBgRBCCCGEELOYBARCCCGEEELMYhIQCCGEEEIIMYtJQCCEEEIIIcQs9v8Dgd3Uy8UMKugAAAAASUVORK5CYII=",
      "text/plain": [
       "<Figure size 800x800 with 3 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "scores_af2 = pd.read_csv(\"scores.sc\", sep=\"\\s+\", header=0)\n",
    "\n",
    "## Filtering AF2 scores based on lddt and rmsd\n",
    "# Define your desired cutoffs here:\n",
    "AF2_filters = {\"lDDT\": [85.0, \">=\"],\n",
    "               \"rmsd\": [1.5, \"<=\"],\n",
    "               \"rmsd_SR1\": [1.0, \"<=\"]}  # 1st catalytic residue sc-rmsd\n",
    "\n",
    "\n",
    "scores_af2_filtered = utils.filter_scores(scores_af2, AF2_filters)\n",
    "utils.dump_scorefile(scores_af2_filtered, \"filtered_scores.sc\")\n",
    "\n",
    "## Plotting AF2 scores\n",
    "plt.figure(figsize=(12, 3))\n",
    "for i,k in enumerate(AF2_filters):\n",
    "    plt.subplot(1, 3, i+1)\n",
    "    plt.hist(scores_af2[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "    \n",
    "utils.plot_score_pairs(scores_af2, \"lDDT\", \"rmsd\", AF2_filters[\"lDDT\"][0], AF2_filters[\"rmsd\"][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "9aeb0952",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "┌──────────────────────────────────────────────────────────────────────────────┐\n",
      "│                                 PyRosetta-4                                  │\n",
      "│              Created in JHU by Sergey Lyskov and PyRosetta Team              │\n",
      "│              (C) Copyright Rosetta Commons Member Institutions               │\n",
      "│                                                                              │\n",
      "│ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE │\n",
      "│              See LICENSE.md or email license@uw.edu for details              │\n",
      "└──────────────────────────────────────────────────────────────────────────────┘\n",
      "PyRosetta-4 2024 [Rosetta PyRosetta4.Release.python39.ubuntu.cxx11thread.serialization 2024.10+release.2c36cbc7108d85646ca5b8ddc89c29ac1ccde88e 2024-03-01T16:53:36] retrieved from: http://www.pyrosetta.org\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/2_af2/good/7o2g_HBA_dif_130_traj2_T0.3_0_5_model_4.0_r3_af2.pdb: alignment RMSD = 0.944\n",
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/2_af2/good/7o2g_HBA_dif_130_traj2_T0.3_0_5_model_4.0_r3_af2.pdb: fixing CYS99 with reference CYS\n"
     ]
    }
   ],
   "source": [
    "### Copying good predictions to a separate directory\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "if len(scores_af2_filtered) > 0:\n",
    "    os.makedirs(\"good\", exist_ok=True)\n",
    "    good_af2_models = [row[\"Output_PDB\"]+\".pdb\" for idx,row in scores_af2_filtered.iterrows()]\n",
    "    for pdb in good_af2_models:\n",
    "        copy2(pdb, f\"good/{pdb}\")\n",
    "    good_af2_models = glob.glob(f\"{AF2_DIR}/good/*.pdb\")\n",
    "else:\n",
    "    sys.exit(\"No good models to continue this pipeline with\")\n",
    "\n",
    "os.chdir(f\"{AF2_DIR}/good\")\n",
    "\n",
    "\n",
    "### Aligning the ligand back into the AF2 predictions.\n",
    "### This is done by aligning the AF2 model to diffusion output and copying over the ligand using PyRosetta.\n",
    "### --fix_catres option will readjust the rotamer and tautomer of \n",
    "### any catalytic residue to be the same as in the reference model.\n",
    "\n",
    "align_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/utils/place_ligand_after_af2.py \"\\\n",
    "            f\"--outdir with_heme --params {' '.join(params)} --fix_catres \"\\\n",
    "            f\"--pdb {' '.join(good_af2_models)} \"\\\n",
    "            f\"--ref {' '.join(glob.glob(DIFFUSION_DIR+'/filtered_structures/*.pdb'))}\"\n",
    "\n",
    "p = subprocess.Popen(align_cmd, shell=True)\n",
    "(output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cddf39ab",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "aac54ca5",
   "metadata": {},
   "source": [
    "## 3.1: Performing binding site design with ligandMPNN / FastRelax\n",
    "(see 3.2 down below for design with [Rosetta FastDesign](#Rosetta_design))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "b8e730b3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/ubuntu/data/heme_binder_diffusion_edit/output/2_af2/good/with_heme\n",
      "Design command:\n",
      "#!/bin/bash\n",
      "/home/ubuntu/anaconda3/envs/SE3nv_ln_pr/bin/python /home/ubuntu/data/heme_binder_diffusion_edit/scripts/design/heme_pocket_ligMPNN.py --pdb /home/ubuntu/data/heme_binder_diffusion_edit/output/2_af2/good/with_heme/7o2g_HBA_dif_130_traj2_T0.3_0_5_model_4.0_r3_af2_HBA.pdb --nstruct 10 --scoring /home/ubuntu/data/heme_binder_diffusion_edit/scripts/design/scoring/heme_scoring.py --params /home/ubuntu/data/heme_binder_diffusion_edit/theozyme/HBA/HBA.params --cstfile /home/ubuntu/data/heme_binder_diffusion_edit/theozyme/HBA/HBA_CYS_UPO.cst > logs/7o2g_HBA_dif_130_traj2_T0.3_0_5_model_4.0_r3_af2_HBA.log\n",
      "\n",
      "Running\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[28], line 34\u001b[0m\n\u001b[1;32m     32\u001b[0m     \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mRunning\u001b[39m\u001b[38;5;124m'\u001b[39m)\n\u001b[1;32m     33\u001b[0m     p \u001b[38;5;241m=\u001b[39m subprocess\u001b[38;5;241m.\u001b[39mPopen([\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mbash\u001b[39m\u001b[38;5;124m'\u001b[39m, cmds_filename_des], stdout\u001b[38;5;241m=\u001b[39msubprocess\u001b[38;5;241m.\u001b[39mPIPE, stderr\u001b[38;5;241m=\u001b[39msubprocess\u001b[38;5;241m.\u001b[39mPIPE)\n\u001b[0;32m---> 34\u001b[0m     (output, err) \u001b[38;5;241m=\u001b[39m p\u001b[38;5;241m.\u001b[39mcommunicate()\n\u001b[1;32m     35\u001b[0m     \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mDone\u001b[39m\u001b[38;5;124m'\u001b[39m)\n\u001b[1;32m     36\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "File \u001b[0;32m~/anaconda3/lib/python3.11/subprocess.py:1209\u001b[0m, in \u001b[0;36mPopen.communicate\u001b[0;34m(self, input, timeout)\u001b[0m\n\u001b[1;32m   1206\u001b[0m     endtime \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m\n\u001b[1;32m   1208\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1209\u001b[0m     stdout, stderr \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_communicate(\u001b[38;5;28minput\u001b[39m, endtime, timeout)\n\u001b[1;32m   1210\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mKeyboardInterrupt\u001b[39;00m:\n\u001b[1;32m   1211\u001b[0m     \u001b[38;5;66;03m# https://bugs.python.org/issue25942\u001b[39;00m\n\u001b[1;32m   1212\u001b[0m     \u001b[38;5;66;03m# See the detailed comment in .wait().\u001b[39;00m\n\u001b[1;32m   1213\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m timeout \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n",
      "File \u001b[0;32m~/anaconda3/lib/python3.11/subprocess.py:2108\u001b[0m, in \u001b[0;36mPopen._communicate\u001b[0;34m(self, input, endtime, orig_timeout)\u001b[0m\n\u001b[1;32m   2101\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_check_timeout(endtime, orig_timeout,\n\u001b[1;32m   2102\u001b[0m                         stdout, stderr,\n\u001b[1;32m   2103\u001b[0m                         skip_check_and_raise\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m)\n\u001b[1;32m   2104\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mRuntimeError\u001b[39;00m(  \u001b[38;5;66;03m# Impossible :)\u001b[39;00m\n\u001b[1;32m   2105\u001b[0m         \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m_check_timeout(..., skip_check_and_raise=True) \u001b[39m\u001b[38;5;124m'\u001b[39m\n\u001b[1;32m   2106\u001b[0m         \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mfailed to raise TimeoutExpired.\u001b[39m\u001b[38;5;124m'\u001b[39m)\n\u001b[0;32m-> 2108\u001b[0m ready \u001b[38;5;241m=\u001b[39m selector\u001b[38;5;241m.\u001b[39mselect(timeout)\n\u001b[1;32m   2109\u001b[0m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_check_timeout(endtime, orig_timeout, stdout, stderr)\n\u001b[1;32m   2111\u001b[0m \u001b[38;5;66;03m# XXX Rewrite these to use non-blocking I/O on the file\u001b[39;00m\n\u001b[1;32m   2112\u001b[0m \u001b[38;5;66;03m# objects; they are no longer using C stdio!\u001b[39;00m\n",
      "File \u001b[0;32m~/anaconda3/lib/python3.11/selectors.py:415\u001b[0m, in \u001b[0;36m_PollLikeSelector.select\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m    413\u001b[0m ready \u001b[38;5;241m=\u001b[39m []\n\u001b[1;32m    414\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 415\u001b[0m     fd_event_list \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_selector\u001b[38;5;241m.\u001b[39mpoll(timeout)\n\u001b[1;32m    416\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mInterruptedError\u001b[39;00m:\n\u001b[1;32m    417\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m ready\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "### Setting up design directory and commands\n",
    "os.chdir(WDIR)\n",
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/3.1_design_pocket_ligandMPNN\"\n",
    "os.makedirs(DESIGN_DIR_ligMPNN, exist_ok=True)\n",
    "os.chdir(DESIGN_DIR_ligMPNN)\n",
    "\n",
    "AF2_DIR = f\"{WDIR}/2_af2\"\n",
    "os.makedirs(DESIGN_DIR_ligMPNN+\"/logs\", exist_ok=True)\n",
    "\n",
    "### Performing 5 design iterations on each input structure\n",
    "NSTRUCT = 10\n",
    "cstfile = f\"{SCRIPT_DIR}/theozyme/HBA/HBA_CYS_UPO.cst\"\n",
    "\n",
    "commands_design = \"#!/bin/bash\\n\"\n",
    "cmds_filename_des = \"commands_design\"\n",
    "\n",
    "print(f\"{AF2_DIR}/good/with_heme\")\n",
    "\n",
    "\n",
    "with open(cmds_filename_des, \"w\") as file:\n",
    "    for pdb in glob.glob(f\"{AF2_DIR}/good/with_heme/*.pdb\"):\n",
    "        commands_design += f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/heme_pocket_ligMPNN.py \"\n",
    "        commands_design += f\"--pdb {pdb} --nstruct {NSTRUCT} \"\n",
    "        commands_design += f\"--scoring {SCRIPT_DIR}/scripts/design/scoring/heme_scoring.py \"\n",
    "        commands_design += f\"--params {' '.join(params)} --cstfile {cstfile} > logs/{os.path.basename(pdb).replace('.pdb', '.log')}\\n\"\n",
    "    file.write(commands_design)\n",
    "\n",
    "print(\"Design command:\")\n",
    "print(commands_design)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_ligMPNN+\"/.done\"):\n",
    "    print('Running')\n",
    "    p = subprocess.Popen(['bash', cmds_filename_des], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n",
    "    print('Done')\n",
    "else:\n",
    "    print('Found done file - Skipping')\n",
    "\n",
    "\n",
    "# ### Running design jobs with Slurm.\n",
    "# submit_script = \"submit_design.sh\"\n",
    "# utils.create_slurm_submit_script(filename=submit_script, name=\"3.1_design_pocket_ligMPNN\", mem=\"4g\", \n",
    "#                                  N_cores=1, time=\"3:00:00\", email=EMAIL, array=len(commands_design),\n",
    "#                                  array_commandfile=cmds_filename_des)\n",
    "\n",
    "# if not os.path.exists(DESIGN_DIR_ligMPNN+\"/.done\"):\n",
    "#     p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "#     (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "43c2821f",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with design and happy with the outputs then mark it as done\n",
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/3.1_design_pocket_ligandMPNN\"\n",
    "os.chdir(DESIGN_DIR_ligMPNN)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_ligMPNN+\"/.done\"):\n",
    "    with open(f\"{DESIGN_DIR_ligMPNN}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ef0d7f43",
   "metadata": {},
   "source": [
    "### Analyzing ligMPNN / FastRelax design results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e2364e89",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Analyzing ligMPNN / FastRelax designs\n",
    "if not os.path.exists(\"scorefile.txt\"):\n",
    "    sys.exit(\"Design job failed, or no successful outputs were produced.\")\n",
    "\n",
    "scores = pd.read_csv(\"scorefile.txt\", sep=\"\\s+\", header=0)\n",
    "\n",
    "filters = {'all_cst': [1.0, '<='],\n",
    " 'nlr_SR1_rms': [0.8, '<='],\n",
    " 'nlr_totrms': [1.0, '<='],\n",
    " 'L_SASA': [0.2, '<='],\n",
    " 'COO_hbond': [1.0, '='],\n",
    " 'heme_angle_wrst': [80.0, '>='],\n",
    " 'score_per_res': [0.0, '<='],\n",
    " 'corrected_ddg': [-50.0, '<='],\n",
    " 'cms_per_atom': [4.8, '>=']}\n",
    "\n",
    "filtered_scores = utils.filter_scores(scores, filters)\n",
    "\n",
    "## Plotting AF2 scores\n",
    "plt.figure(figsize=(12, 9))\n",
    "for i,k in enumerate(filters):\n",
    "    plt.subplot(3, 3, i+1)\n",
    "    plt.hist(scores[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "\n",
    "### Copying good designs over to a new directory\n",
    "if len(filtered_scores) > 0:\n",
    "    os.makedirs(f\"{DESIGN_DIR_ligMPNN}/good\")\n",
    "    for idx, row in filtered_scores.iterrows():\n",
    "        copy2(row[\"description\"]+\".pdb\", \"good/\"+row[\"description\"]+\".pdb\")\n",
    "else:\n",
    "    print(\"No good designs created, bummer...\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f2c82f60",
   "metadata": {},
   "source": [
    "## 4.1 Performing ligandMPNN redesign on the 2nd layer residues\n",
    "\n",
    "Resampling residues that are not in the pocket, but also not very far from the pocket"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1f470af9",
   "metadata": {},
   "outputs": [],
   "source": [
    "if len(glob.glob(DESIGN_DIR_ligMPNN+'/good/*.pdb')) == 0:\n",
    "    sys.exit(\"No designs to run 2nd MPNN on.\")\n",
    "\n",
    "os.chdir(WDIR)\n",
    "DESIGN_DIR_2nd_mpnn = f\"{WDIR}/4.1_2nd_mpnn\"\n",
    "os.makedirs(DESIGN_DIR_2nd_mpnn, exist_ok=True)\n",
    "os.chdir(DESIGN_DIR_2nd_mpnn)\n",
    "\n",
    "### Making a JSON file specifiying designable positions for each structure.\n",
    "### Will also make non-pocket ALA positions as designable.\n",
    "### This is to fix any surface ALA-patches that previous MPNN may have introduced.\n",
    "\n",
    "make_json_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/setup_ligand_mpnn_2nd_layer.py \"\\\n",
    "                f\"--params {' '.join(params)} --ligand {LIGAND} --output_path parsed_pdbs_lig.jsonl \"\\\n",
    "                 \"--output_path masked_pos.jsonl --dist_bb 6.0 --dist_sc 5.0 \"\\\n",
    "                f\"--pdb {' '.join(glob.glob(DESIGN_DIR_ligMPNN+'/good/*.pdb'))}\"\n",
    "\n",
    "p = subprocess.Popen(make_json_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n",
    "\n",
    "\n",
    "### Setting up ligandMPNN run commands\n",
    "## We're doing design with 2 temperatures (more conservative than before), and 5 sequences each.\n",
    "\n",
    "MPNN_temperatures = [0.1, 0.2]\n",
    "MPNN_outputs_per_temperature = 5\n",
    "MPNN_omit_AAs = \"CM\"\n",
    "\n",
    "commands_mpnn = []\n",
    "cmds_filename_mpnn = \"commands_mpnn\"\n",
    "with open(cmds_filename_mpnn, \"w\") as file:\n",
    "    for T in MPNN_temperatures:\n",
    "        for f in glob.glob(DESIGN_DIR_ligMPNN+'/good/*.pdb'):\n",
    "            commands_mpnn.append(f\"{PYTHON['proteinMPNN']} {proteinMPNN_script} \"\n",
    "                                 f\"--model_type ligand_mpnn --ligand_mpnn_use_atom_context 1 \"\n",
    "                                 \"--fixed_residues_multi masked_pos.jsonl --out_folder ./ \"\n",
    "                                 f\"--number_of_batches {MPNN_outputs_per_temperature} --temperature {T} \"\n",
    "                                 f\"--omit_AA {MPNN_omit_AAs} --pdb_path {f}\"\n",
    "                                 f\"--checkpoint_protein_mpnn {SCRIPT_DIR}/lib/LigandMPNN/model_params/ligandmpnn_v_32_010_25.pt\\n\")\n",
    "            file.write(commands_mpnn[-1])\n",
    "\n",
    "print(\"Example MPNN command:\")\n",
    "print(commands_mpnn[-1])\n",
    "\n",
    "### Running ligandMPNN with Slurm.\n",
    "### Grouping jobs with 10 commands per one array job.\n",
    "\n",
    "submit_script = \"submit_mpnn.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"4.1_2nd_mpnn\", mem=\"8g\", \n",
    "                                 N_cores=1, time=\"0:15:00\", email=EMAIL, array=len(commands_mpnn),\n",
    "                                 array_commandfile=cmds_filename_mpnn, group=10)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_2nd_mpnn+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ffd97aea",
   "metadata": {},
   "source": [
    "## 5.1 AlphaFold2 predictions on the 2nd MPNN run"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "15e96fe9",
   "metadata": {},
   "outputs": [],
   "source": [
    "os.chdir(WDIR)\n",
    "DESIGN_DIR_2nd_mpnn = f\"{WDIR}/4.1_2nd_mpnn\"\n",
    "assert os.path.exists(DESIGN_DIR_2nd_mpnn+\"/.done\"), \"2nd MPNN has not been performed!\"\n",
    "\n",
    "AF2_DIR = f\"{WDIR}/5.1_2nd_af2\"\n",
    "os.makedirs(AF2_DIR, exist_ok=True)\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "### First collecting MPNN outputs and creating FASTA files for AF2 input\n",
    "mpnn_fasta = utils.parse_fasta_files(glob.glob(f\"{DESIGN_DIR_2nd_mpnn}/seqs/*.fa\"))\n",
    "# Giving sequences unique names based on input PDB name, temperature, and sequence identifier\n",
    "_mpnn_fasta = {}\n",
    "for k, seq in mpnn_fasta.items():\n",
    "    if \"model_path\" in k:\n",
    "        _mpnn_fasta[k.split(\",\")[0]+\"_native\"] = seq.strip()\n",
    "    else:\n",
    "        _mpnn_fasta[k.split(\",\")[0]+\"_\"+k.split(\",\")[2].replace(\" T=\", \"T\")+\"_0_\"+k.split(\",\")[1].replace(\" id=\", \"\")] = seq.strip()\n",
    "mpnn_fasta = {k:v for k,v in _mpnn_fasta.items()}\n",
    "\n",
    "print(f\"A total on {len(mpnn_fasta)} sequences will be predicted.\")\n",
    "\n",
    "## Splitting the MPNN sequences based on length\n",
    "## and grouping them in smaller batches for each AF2 job\n",
    "## Use group size of >40 when running on GPU. Also depends on how many sequences and resources you have.\n",
    "SEQUENCES_PER_AF2_JOB = 5  # CPU\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    SEQUENCES_PER_AF2_JOB = 100  # GPU\n",
    "mpnn_fasta_split = utils.split_fasta_based_on_length(mpnn_fasta, SEQUENCES_PER_AF2_JOB, write_files=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f6d082ac",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Setting up AlphaFold2 run\n",
    "\n",
    "AF2_recycles = 3\n",
    "AF2_models = \"4\"  # add other models to this string if needed\n",
    "\n",
    "commands_af2 = []\n",
    "cmds_filename_af2 = \"commands_af2\"\n",
    "with open(cmds_filename_af2, \"w\") as file:\n",
    "    for ff in glob.glob(\"*.fasta\"):\n",
    "        commands_af2.append(f\"{PYTHON['af2']} {AF2_script} \"\n",
    "                             f\"--af-nrecycles {AF2_recycles} --af-models {AF2_models} \"\n",
    "                             f\"--fasta {ff} --scorefile {ff.replace('.fasta', '.csv')}\\n\")\n",
    "        file.write(commands_af2[-1])\n",
    "\n",
    "print(\"Example AF2 command:\")\n",
    "print(commands_af2[-1])\n",
    "\n",
    "### Running AF2 with Slurm.\n",
    "### Running jobs on the CPU. It takes ~10 minutes per sequence\n",
    "\n",
    "submit_script = \"submit_af2.sh\"\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"5.1_2nd_af2\", mem=\"6g\", \n",
    "                                     N_cores=2, gpu=True, gres=\"gpu:rtx2080:1\", time=\"00:12:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "else:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"5.1_2nd_af2\", mem=\"6g\", \n",
    "                                     N_cores=4, time=\"01:00:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "98095a31",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with AF2 and happy with the outputs then mark it as done\n",
    "AF2_DIR = f\"{WDIR}/5.1_2nd_af2\"\n",
    "os.chdir(AF2_DIR)\n",
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/3.1_design_pocket_ligandMPNN\"\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    with open(f\"{AF2_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c770263c",
   "metadata": {},
   "source": [
    "### Analyzing AlphaFold2 predictions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c2fd05f9",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Combining all CSV scorefiles into one\n",
    "os.system(\"head -n 1 $(ls *aa*.csv | shuf -n 1) > scores.csv ; for f in *aa*.csv ; do tail -n +2 ${f} >> scores.csv ; done\")\n",
    "assert os.path.exists(\"scores.csv\"), \"Could not combine scorefiles\"\n",
    "\n",
    "### Calculating the RMSDs of AF2 predictions relative to the diffusion outputs\n",
    "### Catalytic residue sidechain RMSDs are calculated in the reference PDB has REMARK 666 line present\n",
    "\n",
    "analysis_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/utils/analyze_af2.py --scorefile scores.csv \"\\\n",
    "               f\"--ref_path {DESIGN_DIR_ligMPNN}/good/ --mpnn --params {' '.join(params)}\"\n",
    "\n",
    "p = subprocess.Popen(analysis_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8512c33f",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Visualizing and filtering AF2 results\n",
    "AF2_DIR = f\"{WDIR}/5.1_2nd_af2\"\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "scores_af2 = pd.read_csv(\"scores.sc\", sep=\"\\s+\", header=0)\n",
    "\n",
    "### Filtering AF2 scores based on lddt and rmsd\n",
    "# Define your desired cutoffs here:\n",
    "AF2_filters = {\"lDDT\": [85.0, \">=\"],\n",
    "               \"rmsd\": [1.2, \"<=\"],\n",
    "               \"rmsd_SR1\": [1.0, \"<=\"]}  # 1st catalytic residue sc-rmsd\n",
    "\n",
    "scores_af2_filtered = utils.filter_scores(scores_af2, AF2_filters)\n",
    "utils.dump_scorefile(scores_af2_filtered, \"filtered_scores.sc\")\n",
    "\n",
    "## Plotting AF2 scores\n",
    "plt.figure(figsize=(12, 3))\n",
    "for i,k in enumerate(AF2_filters):\n",
    "    plt.subplot(1, 3, i+1)\n",
    "    plt.hist(scores_af2[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "\n",
    "utils.plot_score_pairs(scores_af2, \"lDDT\", \"rmsd\", AF2_filters[\"lDDT\"][0], AF2_filters[\"rmsd\"][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "20de13ad",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Copying good predictions to a separate directory\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "if len(scores_af2_filtered) > 0:\n",
    "    os.makedirs(\"good\", exist_ok=True)\n",
    "    good_af2_models = [row[\"Output_PDB\"]+\".pdb\" for idx,row in scores_af2_filtered.iterrows()]\n",
    "    for pdb in good_af2_models:\n",
    "        copy2(pdb, f\"good/{pdb}\")\n",
    "    good_af2_models = glob.glob(f\"{AF2_DIR}/good/*.pdb\")\n",
    "else:\n",
    "    sys.exit(\"No good models to continue this pipeline with\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9bc2ddec",
   "metadata": {},
   "source": [
    "## 6.1: Final FastRelax with the ligand\n",
    "Relaxing good AF2 models together with the ligand"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "268bceef",
   "metadata": {},
   "outputs": [],
   "source": [
    "AF2_DIR = f\"{WDIR}/5.1_2nd_af2\"\n",
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/3.1_design_pocket_ligandMPNN\"\n",
    "assert len(glob.glob(f\"{AF2_DIR}/good/*.pdb\")) > 0, \"No good AF2 models to relax with\"\n",
    "\n",
    "os.chdir(WDIR)\n",
    "RELAX_DIR = f\"{WDIR}/6.1_final_relax\"\n",
    "os.makedirs(RELAX_DIR, exist_ok=True)\n",
    "os.chdir(RELAX_DIR)\n",
    "\n",
    "os.makedirs(RELAX_DIR+\"/logs\", exist_ok=True)\n",
    "\n",
    "## First matching up the AF2 output filenames of step 5 with pocket design filenames from step 3\n",
    "ref_and_model_pairs = []\n",
    "for r in glob.glob(f\"{DESIGN_DIR_ligMPNN}/good/*.pdb\"):\n",
    "    for pdbfile in glob.glob(f\"{AF2_DIR}/good/*.pdb\"):\n",
    "        if os.path.basename(r).replace(\".pdb\", \"_\") in pdbfile:\n",
    "            ref_and_model_pairs.append((r, pdbfile))\n",
    "\n",
    "assert len(ref_and_model_pairs) == len(glob.glob(f\"{AF2_DIR}/good/*.pdb\")), \"Was not able to match all models with reference structures\"\n",
    "\n",
    "\n",
    "## Generating commands for relax jobs\n",
    "### Performing 1 relax iteration on each input structure\n",
    "NSTRUCT = 1\n",
    "cstfile = f\"{SCRIPT_DIR}/theozyme/HBA/HBA_CYS_UPO.cst\"\n",
    "\n",
    "commands_relax = []\n",
    "cmds_filename_rlx = \"commands_relax\"\n",
    "with open(cmds_filename_rlx, \"w\") as file:\n",
    "    for r_m in ref_and_model_pairs:\n",
    "        commands_relax.append(f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/align_add_ligand_relax.py \"\n",
    "                              f\"--outdir ./ --ligand {LIGAND} --ref_pdb {r_m[0]} \"\n",
    "                              f\"--pdb {r_m[1]} --nstruct {NSTRUCT} \"\n",
    "                              f\"--params {' '.join(params)} --cstfile {cstfile} > logs/{os.path.basename(r_m[1]).replace('.pdb', '.log')}\\n\")\n",
    "        file.write(commands_relax[-1])\n",
    "\n",
    "print(\"Example design command:\")\n",
    "print(commands_relax[-1])\n",
    "\n",
    "\n",
    "### Running design jobs with Slurm.\n",
    "submit_script = \"submit_relax.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"6.1_final_relax\", mem=\"4g\", \n",
    "                                 N_cores=1, time=\"0:30:00\", email=EMAIL, array=len(commands_relax),\n",
    "                                 array_commandfile=cmds_filename_rlx)\n",
    "\n",
    "if not os.path.exists(RELAX_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "366ce521",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with final relax and happy with the outputs then mark it as done\n",
    "RELAX_DIR = f\"{WDIR}/6.1_final_relax\"\n",
    "os.chdir(RELAX_DIR)\n",
    "\n",
    "if not os.path.exists(RELAX_DIR+\"/.done\"):\n",
    "    with open(f\"{RELAX_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d848dbbf",
   "metadata": {},
   "source": [
    "### Analyzing final relaxed structures\n",
    "Filtering them based on the same metrics as was used for the initial design"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3744d0a4",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Analyzing Rosetta designs\n",
    "RELAX_DIR = f\"{WDIR}/6.1_final_relax\"\n",
    "os.chdir(RELAX_DIR)\n",
    "\n",
    "scores = pd.read_csv(\"scorefile.txt\", sep=\"\\s+\", header=0)\n",
    "\n",
    "filters = {'all_cst': [1.0, '<='],\n",
    " 'nlr_SR1_rms': [0.8, '<='],\n",
    " 'nlr_totrms': [1.0, '<='],\n",
    " 'L_SASA': [0.2, '<='],\n",
    " 'COO_hbond': [1.0, '='],\n",
    " 'heme_angle_wrst': [80.0, '>='],\n",
    " 'score_per_res': [0.0, '<='],\n",
    " 'corrected_ddg': [-50.0, '<='],\n",
    " 'cms_per_atom': [4.8, '>='],\n",
    " 'rmsd_CA_rlx_in': [1.0, \"<=\"]}  # rmsd_CA_rlx_in is rmsd between relaxed structure and AF2 prediction\n",
    "\n",
    "filtered_scores = utils.filter_scores(scores, filters)\n",
    "\n",
    "## Plotting relax scores\n",
    "plt.figure(figsize=(12, 9))\n",
    "for i,k in enumerate(filters):\n",
    "    if k not in scores.keys():\n",
    "        continue\n",
    "    plt.subplot(4, 3, i+1)\n",
    "    plt.hist(scores[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "\n",
    "### Copying good designs over to a new directory\n",
    "if len(filtered_scores) > 0:\n",
    "    os.makedirs(f\"{RELAX_DIR}/good\", exist_ok=True)\n",
    "    for idx, row in filtered_scores.iterrows():\n",
    "        copy2(row[\"description\"]+\".pdb\", \"good/\"+row[\"description\"]+\".pdb\")\n",
    "else:\n",
    "    print(\"No good designs created, bummer...\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "76e448ea",
   "metadata": {},
   "outputs": [],
   "source": [
    "if len(filtered_scores) > 0:\n",
    "    print(f\"CONGRATULATIONS! You have successfully designed {len(filtered_scores)} proteins against ligand {LIGAND}\")\n",
    "    print(\"You can find the design models in the directory:\\n\"\n",
    "          f\"    {RELAX_DIR}/good\")\n",
    "    print(\"\\nIt is advised you manually inspect them before ordering.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ad026b16",
   "metadata": {},
   "source": [
    "## \n",
    "## --------------------------------------------------------------------------------------------------------------------"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "02646039",
   "metadata": {},
   "source": [
    "<i>The following part of the pipeline branches off after the first AlphaFold2 step (2)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "37afe717",
   "metadata": {},
   "source": [
    "## 3.2: Performing binding site design with Rosetta FastDesign\n",
    "<a id='Rosetta_design'></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d77879a7",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Setting up design directory and commands\n",
    "os.chdir(WDIR)\n",
    "DESIGN_DIR_ROSETTA = f\"{WDIR}/3.2_design_pocket_Rosetta\"\n",
    "os.makedirs(DESIGN_DIR_ROSETTA, exist_ok=True)\n",
    "os.chdir(DESIGN_DIR_ROSETTA)\n",
    "AF2_DIR = f\"{WDIR}/2_af2\"\n",
    "\n",
    "os.makedirs(DESIGN_DIR_ROSETTA+\"/logs\", exist_ok=True)\n",
    "\n",
    "### Performing 5 design iterations on each input structure\n",
    "NSTRUCT = 5\n",
    "cstfile = f\"{SCRIPT_DIR}/theozyme/HBA/HBA_CYS_UPO.cst\"\n",
    "\n",
    "commands_design = []\n",
    "cmds_filename_des = \"commands_design\"\n",
    "with open(cmds_filename_des, \"w\") as file:\n",
    "    for pdb in glob.glob(f\"{AF2_DIR}/good/with_heme/*.pdb\"):\n",
    "        commands_design.append(f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/heme_pocket_FastDesign.py \"\n",
    "                             f\"--pdb {pdb} --norelax --nstruct {NSTRUCT} \"\n",
    "                             f\"--params {' '.join(params)} --cstfile {cstfile} > logs/{os.path.basename(pdb).replace('.pdb', '.log')}\\n\")\n",
    "        file.write(commands_design[-1])\n",
    "\n",
    "print(\"Example design command:\")\n",
    "print(commands_design[-1])\n",
    "\n",
    "\n",
    "### Running design jobs with Slurm.\n",
    "submit_script = \"submit_design.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"3.2_design_pocket_Rosetta\", mem=\"4g\", \n",
    "                                 N_cores=1, time=\"3:00:00\", email=EMAIL, array=len(commands_design),\n",
    "                                 array_commandfile=cmds_filename_des)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_ROSETTA+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "85082bc5",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with design and happy with the outputs then mark it as done\n",
    "DESIGN_DIR_ROSETTA = f\"{WDIR}/3.2_design_pocket_Rosetta\"\n",
    "os.chdir(DESIGN_DIR_ROSETTA)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_ROSETTA+\"/.done\"):\n",
    "    with open(f\"{DESIGN_DIR_ROSETTA}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d8512980",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Analyzing Rosetta designs\n",
    "scores = pd.read_csv(\"scorefile.txt\", sep=\"\\s+\", header=0)\n",
    "\n",
    "filters = {'all_cst': [1.0, '<='],\n",
    " 'nlr_SR1_rms': [0.8, '<='],\n",
    " 'nlr_totrms': [1.0, '<='],\n",
    " 'L_SASA': [0.2, '<='],\n",
    " 'COO_hbond': [1.0, '='],\n",
    " 'heme_angle_wrst': [80.0, '>='],\n",
    " 'score_per_res': [0.0, '<='],\n",
    " 'corrected_ddg': [-50.0, '<='],\n",
    " 'cms_per_atom': [4.8, '>=']}\n",
    "\n",
    "filtered_scores = utils.filter_scores(scores, filters)\n",
    "\n",
    "## Plotting AF2 scores\n",
    "plt.figure(figsize=(12, 9))\n",
    "for i,k in enumerate(filters):\n",
    "    plt.subplot(3, 3, i+1)\n",
    "    plt.hist(scores[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "\n",
    "### Copying good designs over to a new directory\n",
    "if len(filtered_scores) > 0:\n",
    "    os.makedirs(f\"{DESIGN_DIR_ROSETTA}/good\", exist_ok=True)\n",
    "    for idx, row in filtered_scores.iterrows():\n",
    "        copy2(row[\"description\"]+\".pdb\", \"good/\"+row[\"description\"]+\".pdb\")\n",
    "else:\n",
    "    print(\"No good designs created, bummer...\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d049a3a8",
   "metadata": {},
   "source": [
    "## 4.2 Performing ligandMPNN redesign on the 2nd layer residues\n",
    "\n",
    "Resampling residues that are not in the pocket, but also not very far from the pocket"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "af9fe184",
   "metadata": {},
   "outputs": [],
   "source": [
    "if len(glob.glob(DESIGN_DIR_ROSETTA+'/good/*.pdb')) == 0:\n",
    "    sys.exit(\"No designs to run 2nd MPNN on.\")\n",
    "\n",
    "os.chdir(WDIR)\n",
    "DESIGN_DIR_2nd_mpnn = f\"{WDIR}/4.2_2nd_mpnn\"\n",
    "os.makedirs(DESIGN_DIR_2nd_mpnn, exist_ok=True)\n",
    "os.chdir(DESIGN_DIR_2nd_mpnn)\n",
    "\n",
    "### Making a JSON file specifiying designable positions for each structure.\n",
    "### Will also make non-pocket ALA positions as designable.\n",
    "### This is to fix any surface ALA-patches that previous MPNN may have introduced.\n",
    "\n",
    "make_json_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/setup_ligand_mpnn_2nd_layer.py \"\\\n",
    "                f\"--params {' '.join(params)} --ligand {LIGAND} --output_path parsed_pdbs_lig.jsonl \"\\\n",
    "                 \"--output_path masked_pos.jsonl --dist_bb 6.0 --dist_sc 5.0 \"\\\n",
    "                f\"--pdb {' '.join(glob.glob(DESIGN_DIR_ROSETTA+'/good/*.pdb'))}\"\n",
    "\n",
    "p = subprocess.Popen(make_json_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n",
    "\n",
    "\n",
    "### Setting up ligandMPNN run commands\n",
    "## We're doing design with 2 temperatures (more conservative than before), and 5 sequences each.\n",
    "\n",
    "MPNN_temperatures = [0.1, 0.2]\n",
    "MPNN_outputs_per_temperature = 5\n",
    "MPNN_omit_AAs = \"CM\"\n",
    "\n",
    "commands_mpnn = []\n",
    "cmds_filename_mpnn = \"commands_mpnn\"\n",
    "with open(cmds_filename_mpnn, \"w\") as file:\n",
    "    for T in MPNN_temperatures:\n",
    "        for f in glob.glob(DESIGN_DIR_ROSETTA+'/good/*.pdb'):\n",
    "            commands_mpnn.append(f\"{PYTHON['proteinMPNN']} {proteinMPNN_script} \"\n",
    "                                 f\"--model_type ligand_mpnn --ligand_mpnn_use_atom_context 1 \"\n",
    "                                 \"--fixed_residues_multi masked_pos.jsonl --out_folder ./ \"\n",
    "                                 f\"--number_of_batches {MPNN_outputs_per_temperature} --temperature {T} \"\n",
    "                                 f\"--omit_AA {MPNN_omit_AAs} --pdb_path {f} \"\n",
    "                                 f\"--checkpoint_protein_mpnn {SCRIPT_DIR}/lib/LigandMPNN/model_params/ligandmpnn_v_32_010_25.pt\\n\")\n",
    "            file.write(commands_mpnn[-1])\n",
    "\n",
    "print(\"Example MPNN command:\")\n",
    "print(commands_mpnn[-1])\n",
    "\n",
    "\n",
    "### Running ligandMPNN with Slurm.\n",
    "### Grouping jobs with 10 commands per one array job.\n",
    "submit_script = \"submit_mpnn.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"4.2_2nd_mpnn\", mem=\"8g\", \n",
    "                                 N_cores=1, time=\"0:15:00\", email=EMAIL, array=len(commands_mpnn),\n",
    "                                 array_commandfile=cmds_filename_mpnn, group=10)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_2nd_mpnn+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6d4592e7",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with design and happy with the outputs then mark it as done\n",
    "DESIGN_DIR_2nd_mpnn = f\"{WDIR}/4.2_2nd_mpnn\"\n",
    "os.chdir(DESIGN_DIR_2nd_mpnn)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_2nd_mpnn+\"/.done\"):\n",
    "    with open(f\"{DESIGN_DIR_2nd_mpnn}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5b40798b",
   "metadata": {},
   "source": [
    "## 5.2 AlphaFold2 predictions on the 2nd MPNN run"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a97ed25d",
   "metadata": {},
   "outputs": [],
   "source": [
    "os.chdir(WDIR)\n",
    "DESIGN_DIR_2nd_mpnn = f\"{WDIR}/4.2_2nd_mpnn\"\n",
    "assert os.path.exists(DESIGN_DIR_2nd_mpnn+\"/.done\"), \"2nd MPNN has not been performed!\"\n",
    "\n",
    "AF2_DIR = f\"{WDIR}/5.2_2nd_af2\"\n",
    "os.makedirs(AF2_DIR, exist_ok=True)\n",
    "os.chdir(AF2_DIR)\n",
    "DESIGN_DIR_2nd_mpnn = f\"{WDIR}/4.2_2nd_mpnn\"\n",
    "\n",
    "### First collecting MPNN outputs and creating FASTA files for AF2 input\n",
    "mpnn_fasta = utils.parse_fasta_files(glob.glob(f\"{DESIGN_DIR_2nd_mpnn}/seqs/*.fa\"))\n",
    "# Giving sequences unique names based on input PDB name, temperature, and sequence identifier\n",
    "_mpnn_fasta = {}\n",
    "for k, seq in mpnn_fasta.items():\n",
    "    if \"model_path\" in k:\n",
    "        _mpnn_fasta[k.split(\",\")[0]+\"_native\"] = seq.strip()\n",
    "    else:\n",
    "        _mpnn_fasta[k.split(\",\")[0]+\"_\"+k.split(\",\")[2].replace(\" T=\", \"T\")+\"_0_\"+k.split(\",\")[1].replace(\" id=\", \"\")] = seq.strip()\n",
    "mpnn_fasta = {k:v for k,v in _mpnn_fasta.items()}\n",
    "\n",
    "print(f\"A total on {len(mpnn_fasta)} sequences will be predicted.\")\n",
    "\n",
    "## Splitting the MPNN sequences based on length\n",
    "## and grouping them in smaller batches for each AF2 job\n",
    "## Use group size of >40 when running on GPU. Also depends on how many sequences and resources you have.\n",
    "\n",
    "SEQUENCES_PER_AF2_JOB = 5  # CPU\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    SEQUENCES_PER_AF2_JOB = 100  # GPUSEQUENCES_PER_AF2_JOBSEQUENCES_PER_AF2_JOBSEQUENCES_PER_AF2_JOB\n",
    "mpnn_fasta_split = utils.split_fasta_based_on_length(mpnn_fasta, SEQUENCES_PER_AF2_JOB, write_files=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8e01021a",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Setting up AlphaFold2 run\n",
    "\n",
    "AF2_recycles = 3\n",
    "AF2_models = \"4\"  # add other models to this string if needed\n",
    "\n",
    "commands_af2 = []\n",
    "cmds_filename_af2 = \"commands_af2\"\n",
    "with open(cmds_filename_af2, \"w\") as file:\n",
    "    for ff in glob.glob(\"*.fasta\"):\n",
    "        commands_af2.append(f\"{PYTHON['af2']} {AF2_script} \"\n",
    "                             f\"--af-nrecycles {AF2_recycles} --af-models {AF2_models} \"\n",
    "                             f\"--fasta {ff} --scorefile {ff.replace('.fasta', '.csv')}\\n\")\n",
    "        file.write(commands_af2[-1])\n",
    "\n",
    "print(\"Example AF2 command:\")\n",
    "print(commands_af2[-1])\n",
    "\n",
    "\n",
    "### Running AF2 with Slurm.\n",
    "\n",
    "submit_script = \"submit_af2.sh\"\n",
    "\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"5.2_2nd_af2\", mem=\"6g\", \n",
    "                                     N_cores=2, gpu=True, gres=\"gpu:rtx2080:1\", time=\"00:12:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "else:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"5.2_2nd_af2\", mem=\"6g\", \n",
    "                                     N_cores=4, time=\"01:00:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "25adc9d7",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with AF2 and happy with the outputs then mark it as done\n",
    "AF2_DIR = f\"{WDIR}/5.2_2nd_af2\"\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    with open(f\"{AF2_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "08bcb115",
   "metadata": {},
   "source": [
    "### Analyzing AlphaFold2 predictions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2b116b5e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Combining all CSV scorefiles into one\n",
    "DESIGN_DIR_ROSETTA = f\"{WDIR}/3.2_design_pocket_Rosetta\"\n",
    "\n",
    "os.system(\"head -n 1 $(ls *aa*.csv | shuf -n 1) > scores.csv ; for f in *aa*.csv ; do tail -n +2 ${f} >> scores.csv ; done\")\n",
    "assert os.path.exists(\"scores.csv\"), \"Could not combine scorefiles\"\n",
    "\n",
    "### Calculating the RMSDs of AF2 predictions relative to the diffusion outputs\n",
    "### Catalytic residue sidechain RMSDs are calculated in the reference PDB has REMARK 666 line present\n",
    "\n",
    "analysis_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/utils/analyze_af2.py --scorefile scores.csv \"\\\n",
    "               f\"--ref_path {DESIGN_DIR_ROSETTA}/good/ --mpnn --params {' '.join(params)}\"\n",
    "\n",
    "p = subprocess.Popen(analysis_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7d0021fd",
   "metadata": {},
   "outputs": [],
   "source": [
    "scores_af2 = pd.read_csv(\"scores.sc\", sep=\"\\s+\", header=0)\n",
    "\n",
    "### Filtering AF2 scores based on lddt and rmsd\n",
    "# Define your desired cutoffs here:\n",
    "AF2_filters = {\"lDDT\": [85.0, \">=\"],\n",
    "               \"rmsd\": [1.2, \"<=\"],\n",
    "               \"rmsd_SR1\": [1.0, \"<=\"]}  # 1st catalytic residue sc-rmsd\n",
    "\n",
    "scores_af2_filtered = utils.filter_scores(scores_af2, AF2_filters)\n",
    "utils.dump_scorefile(scores_af2_filtered, \"filtered_scores.sc\")\n",
    "\n",
    "## Plotting AF2 scores\n",
    "plt.figure(figsize=(12, 3))\n",
    "for i,k in enumerate(AF2_filters):\n",
    "    plt.subplot(1, 3, i+1)\n",
    "    plt.hist(scores_af2[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "    \n",
    "utils.plot_score_pairs(scores_af2, \"lDDT\", \"rmsd\", AF2_filters[\"lDDT\"][0], AF2_filters[\"rmsd\"][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b2f27212",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "### Copying good predictions to a separate directory\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "if len(scores_af2_filtered) > 0:\n",
    "    os.makedirs(\"good\", exist_ok=True)\n",
    "    good_af2_models = [row[\"Output_PDB\"]+\".pdb\" for idx,row in scores_af2_filtered.iterrows()]\n",
    "    for pdb in good_af2_models:\n",
    "        copy2(pdb, f\"good/{pdb}\")\n",
    "    good_af2_models = glob.glob(f\"{AF2_DIR}/good/*.pdb\")\n",
    "else:\n",
    "    sys.exit(\"No good models to continue this pipeline with\")\n",
    "\n",
    "os.chdir(f\"{AF2_DIR}/good\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "be2d775a",
   "metadata": {},
   "source": [
    "## 6.2: Final FastRelax with the ligand\n",
    "Relaxing good AF2 models together with the ligand"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1c7118b6",
   "metadata": {},
   "outputs": [],
   "source": [
    "assert len(glob.glob(f\"{AF2_DIR}/good/*.pdb\")) > 0, \"No good AF2 models to relax with\"\n",
    "\n",
    "os.chdir(WDIR)\n",
    "RELAX_DIR = f\"{WDIR}/6.2_final_relax\"\n",
    "os.makedirs(RELAX_DIR, exist_ok=True)\n",
    "os.chdir(RELAX_DIR)\n",
    "\n",
    "DESIGN_DIR_ROSETTA = f\"{WDIR}/3.2_design_pocket_Rosetta\"\n",
    "AF2_DIR = f\"{WDIR}/5.2_2nd_af2\"\n",
    "os.makedirs(RELAX_DIR+\"/logs\", exist_ok=True)\n",
    "\n",
    "## First matching up the AF2 output filenames of step 5 with pocket design filenames from step 3\n",
    "ref_and_model_pairs = []\n",
    "for r in glob.glob(f\"{DESIGN_DIR_ROSETTA}/good/*.pdb\"):\n",
    "    for pdbfile in glob.glob(f\"{AF2_DIR}/good/*.pdb\"):\n",
    "        if os.path.basename(r).replace(\".pdb\", \"_\") in pdbfile:\n",
    "            ref_and_model_pairs.append((r, pdbfile))\n",
    "\n",
    "assert len(ref_and_model_pairs) == len(glob.glob(f\"{AF2_DIR}/good/*.pdb\")), \"Was not able to match all models with reference structures\"\n",
    "\n",
    "\n",
    "## Generating commands for relax jobs\n",
    "### Performing 1 relax iteration on each input structure\n",
    "NSTRUCT = 1\n",
    "cstfile = f\"{SCRIPT_DIR}/theozyme/HBA/HBA_CYS_UPO.cst\"\n",
    "\n",
    "commands_relax = []\n",
    "cmds_filename_rlx = \"commands_relax\"\n",
    "with open(cmds_filename_rlx, \"w\") as file:\n",
    "    for r_m in ref_and_model_pairs:\n",
    "        commands_relax.append(f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/align_add_ligand_relax.py \"\n",
    "                              f\"--outdir ./ --ligand {LIGAND} --ref_pdb {r_m[0]} \"\n",
    "                              f\"--pdb {r_m[1]} --nstruct {NSTRUCT} \"\n",
    "                              f\"--params {' '.join(params)} --cstfile {cstfile} > logs/{os.path.basename(r_m[1]).replace('.pdb', '.log')}\\n\")\n",
    "        file.write(commands_relax[-1])\n",
    "\n",
    "print(\"Example design command:\")\n",
    "print(commands_relax[-1])\n",
    "\n",
    "\n",
    "### Running design jobs with Slurm.\n",
    "submit_script = \"submit_relax.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"6.2_final_relax\", mem=\"4g\", \n",
    "                                 N_cores=1, time=\"0:30:00\", email=EMAIL, array=len(commands_relax),\n",
    "                                 array_commandfile=cmds_filename_rlx)\n",
    "\n",
    "if not os.path.exists(RELAX_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4a8778db",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with final relax and happy with the outputs then mark it as done\n",
    "RELAX_DIR = f\"{WDIR}/6.2_final_relax\"\n",
    "os.chdir(RELAX_DIR)\n",
    "\n",
    "if not os.path.exists(RELAX_DIR+\"/.done\"):\n",
    "    with open(f\"{RELAX_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "be98d5d9",
   "metadata": {},
   "source": [
    "### Analyzing final relaxed structures\n",
    "Filtering them based on the same metrics as was used for the initial design"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6d86fca0",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "## Analyzing Rosetta designs\n",
    "RELAX_DIR = f\"{WDIR}/6.2_final_relax\"\n",
    "os.chdir(RELAX_DIR)\n",
    "\n",
    "scores = pd.read_csv(\"scorefile.txt\", sep=\"\\s+\", header=0)\n",
    "\n",
    "filters = {'all_cst': [1.0, '<='],\n",
    " 'nlr_SR1_rms': [0.8, '<='],\n",
    " 'nlr_totrms': [1.0, '<='],\n",
    " 'L_SASA': [0.2, '<='],\n",
    " 'COO_hbond': [1.0, '='],\n",
    " 'heme_angle_wrst': [80.0, '>='],\n",
    " 'score_per_res': [0.0, '<='],\n",
    " 'corrected_ddg': [-50.0, '<='],\n",
    " 'cms_per_atom': [4.8, '>='],\n",
    " 'rmsd_CA_rlx_in': [1.0, \"<=\"]}  # rmsd_CA_rlx_in is rmsd between relaxed structure and AF2 prediction\n",
    "\n",
    "filtered_scores = utils.filter_scores(scores, filters)\n",
    "\n",
    "## Plotting relax scores\n",
    "plt.figure(figsize=(12, 9))\n",
    "for i,k in enumerate(filters):\n",
    "    if k not in scores.keys():\n",
    "        continue\n",
    "    plt.subplot(3, 3, i+1)\n",
    "    plt.hist(scores[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "\n",
    "### Copying good designs over to a new directory\n",
    "if len(filtered_scores) > 0:\n",
    "    os.makedirs(f\"{RELAX_DIR}/good\", exist_ok=True)\n",
    "    for idx, row in filtered_scores.iterrows():\n",
    "        copy2(row[\"description\"]+\".pdb\", \"good/\"+row[\"description\"]+\".pdb\")\n",
    "else:\n",
    "    print(\"No good designs created, bummer...\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "69f0c05b",
   "metadata": {},
   "outputs": [],
   "source": [
    "if len(filtered_scores) > 0:\n",
    "    print(f\"CONGRATULATIONS! You have successfully designed {len(filtered_scores)} proteins against ligand {LIGAND}\")\n",
    "    print(\"You can find the design models in the directory:\\n\"\n",
    "          f\"    {RELAX_DIR}/good\")\n",
    "    print(\"\\nIt is advised you manually inspect them before ordering.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ddb6b45e",
   "metadata": {},
   "source": [
    "## \n",
    "## --------------------------------------------------------------------------------------------------------------------\n",
    "<a id='inpainting'></a>"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c5fd5f9e",
   "metadata": {},
   "source": [
    "## RFjoint Inpainting diversification of diffusion outputs\n",
    "This part of the pipeline branches off right after diffusion.\n",
    "\n",
    "The purpose of inpainting diversification is to resample the loop regions of diffused backbones. Any region that is close to the ligand will not be touched because inpainting model does not see the ligand. Resampling is done with variable sequence length.\n",
    "\n",
    "For some reason the AF2 success rates may go significantly up when performing inpainting on diffused backbones.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "012ef367",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Identifying loop regions in diffused backbones and setting up input JSON files for inpainting\n",
    "assert os.path.exists(f\"{DIFFUSION_DIR}/filtered_structures\"), \"No diffused backbones to run inpainting on\"\n",
    "INPAINT_DIR = f\"{WDIR}/inpainting/0_inpainting\"\n",
    "os.makedirs(INPAINT_DIR, exist_ok=True)\n",
    "os.chdir(INPAINT_DIR)\n",
    "\n",
    "\n",
    "inpaint_inputs_per_job = 2  # How many structures will go into each arrayed job\n",
    "inpaint_catalytic_residues = \"A15\"  # Catalytic residues that were used as motif during diffusion, space-separated string\n",
    "\n",
    "inpaint_setup_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/diffusion_analysis/setup_inpaint_from_diffusion.py \"\\\n",
    "               f\"--group {inpaint_inputs_per_job} --var --design_full --params {' '.join(params)} \"\\\n",
    "               f\"--ref_catres {inpaint_catalytic_residues} \"\\\n",
    "               f\"--pdb {' '.join(glob.glob(f'{DIFFUSION_DIR}/filtered_structures/*.pdb'))}\"\n",
    "\n",
    "p = subprocess.Popen(inpaint_setup_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "888fd826",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Setting up inpainting run commands\n",
    "assert os.path.exists(f\"{DIFFUSION_DIR}/filtered_structures\"), \"No diffused backbones to run inpainting on\"\n",
    "\n",
    "commands_inpaint = []\n",
    "cmds_filename = \"commands_inpaint\"\n",
    "with open(cmds_filename, \"w\") as file:\n",
    "    for jsn in glob.glob(f\"{INPAINT_DIR}/*.json\"):\n",
    "        commands_inpaint.append(f\"{PYTHON['diffusion']} {inpaint_script} \"\n",
    "                             f\"--input_json {jsn} > {jsn.replace('.json', '.log')}\\n\")\n",
    "        file.write(commands_inpaint[-1])\n",
    "\n",
    "print(\"Example inpainting command:\")\n",
    "print(commands_inpaint[-1])\n",
    "\n",
    "\n",
    "### Running inpainting jobs with Slurm.\n",
    "submit_script = \"submit_inpaint.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"00_inpainting\", mem=\"6g\", gpu=True, gres=\"gpu:a4000:1\", \n",
    "                                 N_cores=2, time=\"0:30:00\", email=EMAIL, array=len(commands_inpaint),\n",
    "                                 array_commandfile=cmds_filename)\n",
    "\n",
    "if not os.path.exists(INPAINT_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n",
    "    print(output)\n",
    "    print(err)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "04886d58",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with inpainting and happy with the outputs then mark it as done\n",
    "INPAINT_DIR = f\"{WDIR}/inpainting/0_inpainting\"\n",
    "os.chdir(INPAINT_DIR)\n",
    "\n",
    "if not os.path.exists(INPAINT_DIR+\"/.done\"):\n",
    "    with open(f\"{INPAINT_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6daa91b7",
   "metadata": {},
   "source": [
    "### Aligning the ligand into inpainting outputs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d14a1eb3",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Inpainting outputs will be aligned to diffusion outputs based on constant regions defined in the inpainting TRB file.\n",
    "### Ligand will be placed into the inpaining model, \n",
    "### and REMARK 666 lines defining any catalytic residues will be added to the PDB file.\n",
    "\n",
    "assert len(glob.glob(f\"{INPAINT_DIR}/*.pdb\")) > 0, \"No inpainted backbones to process\"\n",
    "os.makedirs(INPAINT_DIR+\"/alignment\", exist_ok=True)\n",
    "\n",
    "\n",
    "inpaint_align_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/diffusion_analysis/align_dif_inpaint_add_ligand.py \"\\\n",
    "               f\"--outdir alignment --params {' '.join(params)} \"\\\n",
    "               f\"--ref {' '.join(glob.glob(f'{DIFFUSION_DIR}/filtered_structures/*.pdb'))} \"\\\n",
    "               f\"--pdb {' '.join(glob.glob(f'{INPAINT_DIR}/*.pdb'))}\"\n",
    "\n",
    "p = subprocess.Popen(inpaint_align_cmd, shell=True)\n",
    "(output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "21f1bb51",
   "metadata": {},
   "source": [
    "## 1: Running ProteinMPNN on inpainted backbones\n",
    "\n",
    "We are first trying to just design a sequence on the backbone, without considering the ligand.\n",
    "The goal is to first find backbones that fold well.\n",
    "\n",
    "Alternatively, one could also go ahead and run the [LigandMPNN/FastRelax step](#ligmpnn_fr) already on the inpainted backbones.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "def0617c",
   "metadata": {},
   "outputs": [],
   "source": [
    "INPAINT_DIR = f\"{WDIR}/inpainting/0_inpainting\"\n",
    "assert len(glob.glob(INPAINT_DIR+\"/alignment/*.pdb\")) > 0, \"No good backbones found!\"\n",
    "\n",
    "os.chdir(WDIR)\n",
    "\n",
    "MPNN_DIR = f\"{WDIR}/inpainting/1_proteinmpnn\"\n",
    "os.makedirs(MPNN_DIR, exist_ok=True)\n",
    "os.chdir(MPNN_DIR)\n",
    "\n",
    "### Parsing diffusion output TRB files to extract fixed motif residues\n",
    "## These residues will not be redesigned with proteinMPNN\n",
    "mask_json_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/make_maskdict_from_trb.py \"\\\n",
    "                f\"--out masked_pos.jsonl --trb {' '.join(glob.glob(INPAINT_DIR+'/*.trb'))}\"\n",
    "p = subprocess.Popen(mask_json_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n",
    "\n",
    "assert os.path.exists(\"masked_pos.jsonl\"), \"Failed to create masked positions JSONL file\"\n",
    "\n",
    "\n",
    "### Setting up proteinMPNN run commands\n",
    "## We're doing design with 3 temperatures, and 5 sequences each.\n",
    "## This usually gives decent success with designable backbones. For more complicated cases consider doing >100 sequences.\n",
    "\n",
    "MPNN_temperatures = [0.1, 0.2, 0.3]\n",
    "MPNN_outputs_per_temperature = 5\n",
    "MPNN_omit_AAs = \"CM\"\n",
    "\n",
    "commands_mpnn = []\n",
    "cmds_filename_mpnn = \"commands_mpnn\"\n",
    "with open(cmds_filename_mpnn, \"w\") as file:\n",
    "    for T in MPNN_temperatures:\n",
    "        for f in glob.glob(INPAINT_DIR+\"/alignment/*.pdb\"):\n",
    "            commands_mpnn.append(f\"{PYTHON['proteinMPNN']} {proteinMPNN_script} \"\n",
    "                                 f\"--model_type protein_mpnn --ligand_mpnn_use_atom_context 0 \"\n",
    "                                 \"--fixed_residues_multi masked_pos.jsonl --out_folder ./ \"\n",
    "                                 f\"--number_of_batches {MPNN_outputs_per_temperature} --temperature {T} \"\n",
    "                                 f\"--omit_AA {MPNN_omit_AAs} --pdb_path {f} \"\n",
    "                                 f\"--checkpoint_protein_mpnn {SCRIPT_DIR}/lib/LigandMPNN/model_params/proteinmpnn_v_48_020.pt\\n\")\n",
    "            file.write(commands_mpnn[-1])\n",
    "\n",
    "            \n",
    "            \n",
    "print(f\"{len(commands_mpnn)} MPNN jobs to run\")\n",
    "print(\"Example MPNN command:\")\n",
    "print(commands_mpnn[-1])\n",
    "\n",
    "### Running proteinMPNN with Slurm.\n",
    "### Grouping jobs with 10 commands per one array job.\n",
    "\n",
    "submit_script = \"submit_mpnn.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"1_proteinmpnn\", mem=\"4g\", \n",
    "                                 N_cores=1, time=\"0:15:00\", email=EMAIL, array=len(commands_mpnn),\n",
    "                                 array_commandfile=cmds_filename_mpnn, group=10)\n",
    "\n",
    "if not os.path.exists(MPNN_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a57ab8d0",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with MPNN and happy with the outputs then mark it as done\n",
    "MPNN_DIR = f\"{WDIR}/inpainting/1_proteinmpnn\"\n",
    "os.chdir(MPNN_DIR)\n",
    "\n",
    "if not os.path.exists(MPNN_DIR+\"/.done\"):\n",
    "    with open(f\"{MPNN_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4f19d9ce",
   "metadata": {},
   "source": [
    "## 2: Running AlphaFold2\n",
    "Performing AF2 single sequence predictions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fa8e267d",
   "metadata": {},
   "outputs": [],
   "source": [
    "os.chdir(WDIR)\n",
    "assert len(glob.glob(MPNN_DIR+\"/seqs/*.fa\")) > 0, \"No MPNN outputs to run AF2 on\"\n",
    "\n",
    "AF2_DIR = f\"{WDIR}/inpainting/2_af2\"\n",
    "os.makedirs(AF2_DIR, exist_ok=True)\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "### First collecting MPNN outputs and creating FASTA files for AF2 input\n",
    "mpnn_fasta = utils.parse_fasta_files(glob.glob(f\"{MPNN_DIR}/seqs/*.fa\"))\n",
    "mpnn_fasta = {k: seq.strip() for k, seq in mpnn_fasta.items() if \"model_path\" not in k}  # excluding the diffused poly-A sequence\n",
    "# Giving sequences unique names based on input PDB name, temperature, and sequence identifier\n",
    "mpnn_fasta = {k.split(\",\")[0]+\"_\"+k.split(\",\")[2].replace(\" T=\", \"T\")+\"_0_\"+k.split(\",\")[1].replace(\" id=\", \"\"): seq for k, seq in mpnn_fasta.items()}\n",
    "\n",
    "print(f\"A total on {len(mpnn_fasta)} sequences will be predicted.\")\n",
    "\n",
    "## Splitting the MPNN sequences based on length\n",
    "## and grouping them in smaller batches for each AF2 job\n",
    "## Use group size of >40 when running on GPU. Also depends on how many sequences and resources you have.\n",
    "SEQUENCES_PER_AF2_JOB = 4  # CPU\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    SEQUENCES_PER_AF2_JOB = 100  # GPU\n",
    "mpnn_fasta_split = utils.split_fasta_based_on_length(mpnn_fasta, SEQUENCES_PER_AF2_JOB, write_files=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "411d5b39",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Setting up AlphaFold2 run\n",
    "\n",
    "AF2_recycles = 3\n",
    "AF2_models = \"4\"  # add other models to this string if needed, i.e. \"3 4 5\"\n",
    "\n",
    "commands_af2 = []\n",
    "cmds_filename_af2 = \"commands_af2\"\n",
    "with open(cmds_filename_af2, \"w\") as file:\n",
    "    for ff in glob.glob(\"*.fasta\"):\n",
    "        commands_af2.append(f\"{PYTHON['af2']} {AF2_script} \"\n",
    "                             f\"--af-nrecycles {AF2_recycles} --af-models {AF2_models} \"\n",
    "                             f\"--fasta {ff} --scorefile {ff.replace('.fasta', '.csv')}\\n\")\n",
    "        file.write(commands_af2[-1])\n",
    "\n",
    "print(\"Example AF2 command:\")\n",
    "print(commands_af2[-1])\n",
    "\n",
    "### Running AF2 with Slurm.\n",
    "### Running jobs on the CPU. It takes ~10 minutes per sequence\n",
    "### \n",
    "\n",
    "submit_script = \"submit_af2.sh\"\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"2_af2\", mem=\"6g\", \n",
    "                                     N_cores=2, gpu=True, gres=\"gpu:rtx2080:1\", time=\"00:12:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "else:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"2_af2\", mem=\"6g\", \n",
    "                                     N_cores=4, time=\"01:00:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "bb0c7b76",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with AF2 and happy with the outputs then mark it as done\n",
    "AF2_DIR = f\"{WDIR}/inpainting/2_af2\"\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    with open(f\"{AF2_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "184907b7",
   "metadata": {},
   "source": [
    "### Analyzing AlphaFold2 predictions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4c85aeef",
   "metadata": {},
   "outputs": [],
   "source": [
    "INPAINT_DIR = f\"{WDIR}/inpainting/0_inpainting\"\n",
    "\n",
    "# Combining all CSV scorefiles into one\n",
    "os.system(\"head -n 1 $(ls *aa*.csv | shuf -n 1) > scores.csv ; for f in *aa*.csv ; do tail -n +2 ${f} >> scores.csv ; done\")\n",
    "assert os.path.exists(\"scores.csv\"), \"Could not combine scorefiles\"\n",
    "\n",
    "### Calculating the RMSDs of AF2 predictions relative to the diffusion outputs\n",
    "### Catalytic residue sidechain RMSDs are calculated in the reference PDB has REMARK 666 line present\n",
    "\n",
    "analysis_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/utils/analyze_af2.py --scorefile scores.csv \"\\\n",
    "               f\"--ref_path {INPAINT_DIR}/alignment/ --mpnn --params {' '.join(params)}\"\n",
    "\n",
    "p = subprocess.Popen(analysis_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "146e5a18",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Visualizing and filtering AF2 predictions\n",
    "AF2_DIR = f\"{WDIR}/inpainting/2_af2\"\n",
    "\n",
    "scores_af2 = pd.read_csv(\"scores.sc\", sep=\"\\s+\", header=0)\n",
    "\n",
    "### Filtering AF2 scores based on lddt and rmsd\n",
    "# Define your desired cutoffs here:\n",
    "AF2_filters = {\"lDDT\": [85.0, \">=\"],\n",
    "               \"rmsd\": [1.5, \"<=\"],\n",
    "               \"rmsd_SR1\": [2.0, \"<=\"]}  # 1st catalytic residue sc-rmsd\n",
    "\n",
    "scores_af2_filtered = utils.filter_scores(scores_af2, AF2_filters)\n",
    "utils.dump_scorefile(scores_af2_filtered, \"filtered_scores.sc\")\n",
    "\n",
    "## Plotting AF2 scores\n",
    "plt.figure(figsize=(12, 3))\n",
    "for i,k in enumerate(AF2_filters):\n",
    "    plt.subplot(1, 3, i+1)\n",
    "    plt.hist(scores_af2[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "    \n",
    "utils.plot_score_pairs(scores_af2, \"lDDT\", \"rmsd\", AF2_filters[\"lDDT\"][0], AF2_filters[\"rmsd\"][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d2ac5be2",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Copying good predictions to a separate directory\n",
    "os.chdir(AF2_DIR)\n",
    "INPAINT_DIR = f\"{WDIR}/inpainting/0_inpainting\"\n",
    "\n",
    "if len(scores_af2_filtered) > 0:\n",
    "    os.makedirs(\"good\", exist_ok=True)\n",
    "    good_af2_models = [row[\"Output_PDB\"]+\".pdb\" for idx,row in scores_af2_filtered.iterrows()]\n",
    "    for pdb in good_af2_models:\n",
    "        copy2(pdb, f\"good/{pdb}\")\n",
    "    good_af2_models = glob.glob(f\"{AF2_DIR}/good/*.pdb\")\n",
    "else:\n",
    "    sys.exit(\"No good models to continue this pipeline with\")\n",
    "\n",
    "os.chdir(f\"{AF2_DIR}/good\")\n",
    "\n",
    "\n",
    "### Aligning the ligand back into the AF2 predictions.\n",
    "### This is done by aligning the AF2 model to diffusion output and copying over the ligand using PyRosetta.\n",
    "### --fix_catres option will re-adjust the rotamer and tautomer of \n",
    "### any catalytic residue to be the same as in the reference model.\n",
    "\n",
    "align_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/utils/place_ligand_after_af2.py \"\\\n",
    "            f\"--outdir with_heme --params {' '.join(params)} --fix_catres \"\\\n",
    "            f\"--pdb {' '.join(good_af2_models)} \"\\\n",
    "            f\"--ref {' '.join(glob.glob(INPAINT_DIR+'/alignment/*.pdb'))}\"\n",
    "\n",
    "p = subprocess.Popen(align_cmd, shell=True)\n",
    "(output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fa9ad826",
   "metadata": {},
   "source": [
    "## 3.1: Performing binding site design with LigandMPNN / FastRelax\n",
    "<a id='ligmpnn_fr'></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d7ad0fc2",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Setting up design directory and commands\n",
    "os.chdir(WDIR)\n",
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/inpainting/3.1_design_pocket_ligandMPNN\"\n",
    "os.makedirs(DESIGN_DIR_ligMPNN, exist_ok=True)\n",
    "os.chdir(DESIGN_DIR_ligMPNN)\n",
    "\n",
    "AF2_DIR = f\"{WDIR}/inpainting/2_af2\"  # change this if you want to run it on inpainting outputs\n",
    "os.makedirs(DESIGN_DIR_ligMPNN+\"/logs\", exist_ok=True)\n",
    "\n",
    "### Performing 5 design iterations on each input structure\n",
    "NSTRUCT = 10\n",
    "cstfile = f\"{SCRIPT_DIR}/theozyme/HBA/HBA_CYS_UPO.cst\"\n",
    "\n",
    "commands_design = []\n",
    "cmds_filename_des = \"commands_design\"\n",
    "with open(cmds_filename_des, \"w\") as file:\n",
    "    for pdb in glob.glob(f\"{AF2_DIR}/good/with_heme/*.pdb\"):\n",
    "        commands_design.append(f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/heme_pocket_ligMPNN.py \"\n",
    "                             f\"--pdb {pdb} --nstruct {NSTRUCT} \"\n",
    "                             f\"--scoring {SCRIPT_DIR}/scripts/design/scoring/heme_scoring.py \"\n",
    "                             f\"--params {' '.join(params)} --cstfile {cstfile} > logs/{os.path.basename(pdb).replace('.pdb', '.log')}\\n\")\n",
    "        file.write(commands_design[-1])\n",
    "\n",
    "print(\"Example design command:\")\n",
    "print(commands_design[-1])\n",
    "\n",
    "\n",
    "### Running design jobs with Slurm.\n",
    "submit_script = \"submit_design.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"3.1_design_pocket_ligMPNN\", mem=\"4g\", \n",
    "                                 N_cores=1, time=\"3:00:00\", email=EMAIL, array=len(commands_design),\n",
    "                                 array_commandfile=cmds_filename_des)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_ligMPNN+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c77afc69",
   "metadata": {},
   "outputs": [],
   "source": [
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/inpainting/3.1_design_pocket_ligandMPNN\"\n",
    "os.chdir(DESIGN_DIR_ligMPNN)\n",
    "\n",
    "## If you're done with design and happy with the outputs then mark it as done\n",
    "if not os.path.exists(DESIGN_DIR_ligMPNN+\"/.done\"):\n",
    "    with open(f\"{DESIGN_DIR_ligMPNN}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4fcf6697",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "## Analyzing Rosetta designs\n",
    "scores = pd.read_csv(\"scorefile.txt\", sep=\"\\s+\", header=0)\n",
    "\n",
    "filters = {'all_cst': [1.0, '<='],\n",
    " 'nlr_SR1_rms': [0.8, '<='],\n",
    " 'nlr_totrms': [1.0, '<='],\n",
    " 'L_SASA': [0.2, '<='],\n",
    " 'COO_hbond': [1.0, '='],\n",
    " 'heme_angle_wrst': [80.0, '>='],\n",
    " 'score_per_res': [0.0, '<='],\n",
    " 'corrected_ddg': [-50.0, '<='],\n",
    " 'cms_per_atom': [4.8, '>=']}\n",
    "\n",
    "filtered_scores = utils.filter_scores(scores, filters)\n",
    "\n",
    "## Plotting design scores\n",
    "plt.figure(figsize=(12, 9))\n",
    "for i,k in enumerate(filters):\n",
    "    plt.subplot(3, 3, i+1)\n",
    "    plt.hist(scores[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "\n",
    "### Copying good designs over to a new directory\n",
    "if len(filtered_scores) > 0:\n",
    "    os.makedirs(f\"{DESIGN_DIR_ligMPNN}/good\", exist_ok=True)\n",
    "    for idx, row in filtered_scores.iterrows():\n",
    "        copy2(row[\"description\"]+\".pdb\", \"good/\"+row[\"description\"]+\".pdb\")\n",
    "else:\n",
    "    print(\"No good designs created, bummer...\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f56c82b3",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "8397b389",
   "metadata": {},
   "source": [
    "## 4.1 Performing ligandMPNN redesign on the 2nd layer residues\n",
    "\n",
    "Resampling residues that are not in the pocket, but also not very far from the pocket"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7ac14eee",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "if len(glob.glob(DESIGN_DIR_ligMPNN+'/good/*.pdb')) == 0:\n",
    "    sys.exit(\"No designs to run 2nd MPNN on.\")\n",
    "\n",
    "os.chdir(WDIR)\n",
    "DESIGN_DIR_2nd_mpnn = f\"{WDIR}/inpainting/4.1_2nd_mpnn\"\n",
    "os.makedirs(DESIGN_DIR_2nd_mpnn, exist_ok=True)\n",
    "os.chdir(DESIGN_DIR_2nd_mpnn)\n",
    "\n",
    "### Making a JSON file specifiying designable positions for each structure.\n",
    "### Will also make non-pocket ALA positions as designable.\n",
    "### This is to fix any surface ALA-patches that previous MPNN may have introduced.\n",
    "\n",
    "make_json_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/setup_ligand_mpnn_2nd_layer.py \"\\\n",
    "                f\"--params {' '.join(params)} --ligand {LIGAND} --output_path parsed_pdbs_lig.jsonl \"\\\n",
    "                 \"--output_path masked_pos.jsonl --dist_bb 6.0 --dist_sc 5.0 \"\\\n",
    "                f\"--pdb {' '.join(glob.glob(DESIGN_DIR_ligMPNN+'/good/*.pdb'))}\"\n",
    "\n",
    "p = subprocess.Popen(make_json_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n",
    "\n",
    "if not os.path.exists(\"masked_pos.jsonl\"):\n",
    "    sys.exit()\n",
    "\n",
    "### Setting up ligandMPNN run commands\n",
    "## We're doing design with 2 temperatures (more conservative than before), and 5 sequences each.\n",
    "\n",
    "MPNN_temperatures = [0.1, 0.2]\n",
    "MPNN_outputs_per_temperature = 5\n",
    "MPNN_omit_AAs = \"CM\"\n",
    "\n",
    "commands_mpnn = []\n",
    "cmds_filename_mpnn = \"commands_mpnn\"\n",
    "with open(cmds_filename_mpnn, \"w\") as file:\n",
    "    for T in MPNN_temperatures:\n",
    "        for f in glob.glob(DESIGN_DIR_ligMPNN+'/good/*.pdb'):\n",
    "            commands_mpnn.append(f\"{PYTHON['proteinMPNN']} {proteinMPNN_script} \"\n",
    "                                 f\"--model_type ligand_mpnn --ligand_mpnn_use_atom_context 1 \"\n",
    "                                 \"--fixed_residues_multi masked_pos.jsonl --out_folder ./ \"\n",
    "                                 f\"--number_of_batches {MPNN_outputs_per_temperature} --temperature {T} \"\n",
    "                                 f\"--omit_AA {MPNN_omit_AAs} --pdb_path {f} \"\n",
    "                                 f\"--checkpoint_protein_mpnn {SCRIPT_DIR}/lib/LigandMPNN/model_params/ligandmpnn_v_32_010_25.pt\\n\")\n",
    "            file.write(commands_mpnn[-1])\n",
    "\n",
    "print(\"Example MPNN command:\")\n",
    "print(commands_mpnn[-1])\n",
    "\n",
    "### Running ligandMPNN with Slurm.\n",
    "### Grouping jobs with 10 commands per one array job.\n",
    "\n",
    "submit_script = \"submit_mpnn.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"4.1_2nd_mpnn\", mem=\"8g\", \n",
    "                                 N_cores=1, time=\"0:15:00\", email=EMAIL, array=len(commands_mpnn),\n",
    "                                 array_commandfile=cmds_filename_mpnn, group=10)\n",
    "\n",
    "if not os.path.exists(DESIGN_DIR_2nd_mpnn+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0233f278",
   "metadata": {},
   "outputs": [],
   "source": [
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/inpainting/3.1_design_pocket_ligandMPNN\"\n",
    "DESIGN_DIR_2nd_mpnn = f\"{WDIR}/inpainting/4.1_2nd_mpnn\"\n",
    "os.chdir(DESIGN_DIR_2nd_mpnn)\n",
    "if len(glob.glob(DESIGN_DIR_ligMPNN+'/good/*.pdb')) == 0:\n",
    "    sys.exit(\"No designs to run 2nd MPNN on.\")\n",
    "\n",
    "## If you're done with design and happy with the outputs then mark it as done\n",
    "if not os.path.exists(DESIGN_DIR_2nd_mpnn+\"/.done\"):\n",
    "    with open(f\"{DESIGN_DIR_2nd_mpnn}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d495bf96",
   "metadata": {},
   "source": [
    "## 5.1 AlphaFold2 predictions on the 2nd MPNN run"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8cba4381",
   "metadata": {},
   "outputs": [],
   "source": [
    "os.chdir(WDIR)\n",
    "assert os.path.exists(DESIGN_DIR_2nd_mpnn+\"/.done\"), \"2nd MPNN has not been performed!\"\n",
    "\n",
    "AF2_DIR = f\"{WDIR}/inpainting/5.1_2nd_af2\"\n",
    "os.makedirs(AF2_DIR, exist_ok=True)\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "### First collecting MPNN outputs and creating FASTA files for AF2 input\n",
    "mpnn_fasta = utils.parse_fasta_files(glob.glob(f\"{DESIGN_DIR_2nd_mpnn}/seqs/*.fa\"))\n",
    "# Giving sequences unique names based on input PDB name, temperature, and sequence identifier\n",
    "_mpnn_fasta = {}\n",
    "for k, seq in mpnn_fasta.items():\n",
    "    if \"model_path\" in k:\n",
    "        _mpnn_fasta[k.split(\",\")[0]+\"_native\"] = seq.strip()\n",
    "    else:\n",
    "        _mpnn_fasta[k.split(\",\")[0]+\"_\"+k.split(\",\")[2].replace(\" T=\", \"T\")+\"_0_\"+k.split(\",\")[1].replace(\" id=\", \"\")] = seq.strip()\n",
    "mpnn_fasta = {k:v for k,v in _mpnn_fasta.items()}\n",
    "\n",
    "print(f\"A total on {len(mpnn_fasta)} sequences will be predicted.\")\n",
    "\n",
    "## Splitting the MPNN sequences based on length\n",
    "## and grouping them in smaller batches for each AF2 job\n",
    "## Use group size of >40 when running on GPU. Also depends on how many sequences and resources you have.\n",
    "SEQUENCES_PER_AF2_JOB = 5  # CPU\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    SEQUENCES_PER_AF2_JOB = 100  # GPU\n",
    "mpnn_fasta_split = utils.split_fasta_based_on_length(mpnn_fasta, SEQUENCES_PER_AF2_JOB, write_files=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b309a47d",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Setting up AlphaFold2 run\n",
    "\n",
    "AF2_recycles = 3\n",
    "AF2_models = \"4\"  # add other models to this string if needed\n",
    "\n",
    "commands_af2 = []\n",
    "cmds_filename_af2 = \"commands_af2\"\n",
    "with open(cmds_filename_af2, \"w\") as file:\n",
    "    for ff in glob.glob(\"*.fasta\"):\n",
    "        commands_af2.append(f\"{PYTHON['af2']} {AF2_script} \"\n",
    "                             f\"--af-nrecycles {AF2_recycles} --af-models {AF2_models} \"\n",
    "                             f\"--fasta {ff} --scorefile {ff.replace('.fasta', '.csv')}\\n\")\n",
    "        file.write(commands_af2[-1])\n",
    "\n",
    "print(\"Example AF2 command:\")\n",
    "print(commands_af2[-1])\n",
    "\n",
    "### Running AF2 with Slurm.\n",
    "### Running jobs on the CPU. It takes ~10 minutes per sequence\n",
    "\n",
    "submit_script = \"submit_af2.sh\"\n",
    "if USE_GPU_for_AF2 is True:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"5.1_2nd_af2\", mem=\"6g\", \n",
    "                                     N_cores=2, gpu=True, gres=\"gpu:rtx2080:1\", time=\"00:12:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "else:\n",
    "    utils.create_slurm_submit_script(filename=submit_script, name=\"5.1_2nd_af2\", mem=\"6g\", \n",
    "                                     N_cores=4, time=\"01:00:00\", email=EMAIL, array=len(commands_af2),\n",
    "                                     array_commandfile=cmds_filename_af2)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0915fdf9",
   "metadata": {},
   "outputs": [],
   "source": [
    "## If you're done with AF2 and happy with the outputs then mark it as done\n",
    "AF2_DIR = f\"{WDIR}/inpainting/5.1_2nd_af2\"\n",
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/inpainting/3.1_design_pocket_ligandMPNN\"\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "if not os.path.exists(AF2_DIR+\"/.done\"):\n",
    "    with open(f\"{AF2_DIR}/.done\", \"w\") as file:\n",
    "        file.write(f\"Run user: {username}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8d53a010",
   "metadata": {},
   "source": [
    "### Analyzing AlphaFold2 predictions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1266a22e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Combining all CSV scorefiles into one\n",
    "os.system(\"head -n 1 $(ls *aa*.csv | shuf -n 1) > scores.csv ; for f in *aa*.csv ; do tail -n +2 ${f} >> scores.csv ; done\")\n",
    "assert os.path.exists(\"scores.csv\"), \"Could not combine scorefiles\"\n",
    "\n",
    "### Calculating the RMSDs of AF2 predictions relative to the diffusion outputs\n",
    "### Catalytic residue sidechain RMSDs are calculated in the reference PDB has REMARK 666 line present\n",
    "\n",
    "analysis_cmd = f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/utils/analyze_af2.py --scorefile scores.csv \"\\\n",
    "               f\"--ref_path {DESIGN_DIR_ligMPNN}/good/ --mpnn --params {' '.join(params)}\"\n",
    "\n",
    "p = subprocess.Popen(analysis_cmd, shell=True)\n",
    "(output, err) = p.communicate()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "981cc7b4",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Visualizing and filtering AF2 results\n",
    "AF2_DIR = f\"{WDIR}/inpainting/5.1_2nd_af2\"\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "scores_af2 = pd.read_csv(\"scores.sc\", sep=\"\\s+\", header=0)\n",
    "\n",
    "### Filtering AF2 scores based on lddt and rmsd\n",
    "# Define your desired cutoffs here:\n",
    "AF2_filters = {\"lDDT\": [85.0, \">=\"],\n",
    "               \"rmsd\": [1.2, \"<=\"],\n",
    "               \"rmsd_SR1\": [1.0, \"<=\"]}  # 1st catalytic residue sc-rmsd\n",
    "\n",
    "scores_af2_filtered = utils.filter_scores(scores_af2, AF2_filters)\n",
    "utils.dump_scorefile(scores_af2_filtered, \"filtered_scores.sc\")\n",
    "\n",
    "## Plotting AF2 scores\n",
    "plt.figure(figsize=(12, 3))\n",
    "for i,k in enumerate(AF2_filters):\n",
    "    plt.subplot(1, 3, i+1)\n",
    "    plt.hist(scores_af2[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "    \n",
    "utils.plot_score_pairs(scores_af2, \"lDDT\", \"rmsd\", AF2_filters[\"lDDT\"][0], AF2_filters[\"rmsd\"][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "afad819b",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Copying good predictions to a separate directory\n",
    "os.chdir(AF2_DIR)\n",
    "\n",
    "if len(scores_af2_filtered) > 0:\n",
    "    os.makedirs(\"good\", exist_ok=True)\n",
    "    good_af2_models = [row[\"Output_PDB\"]+\".pdb\" for idx,row in scores_af2_filtered.iterrows()]\n",
    "    for pdb in good_af2_models:\n",
    "        copy2(pdb, f\"good/{pdb}\")\n",
    "    good_af2_models = glob.glob(f\"{AF2_DIR}/good/*.pdb\")\n",
    "else:\n",
    "    sys.exit(\"No good models to continue this pipeline with\")\n",
    "\n",
    "os.chdir(f\"{AF2_DIR}/good\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d059ff96",
   "metadata": {},
   "source": [
    "## 6.1: Final FastRelax with the ligand\n",
    "Relaxing good AF2 models together with the ligand"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7c7accf7",
   "metadata": {},
   "outputs": [],
   "source": [
    "AF2_DIR = f\"{WDIR}/inpainting/5.1_2nd_af2\"\n",
    "DESIGN_DIR_ligMPNN = f\"{WDIR}/inpainting/3.1_design_pocket_ligandMPNN\"\n",
    "assert len(glob.glob(f\"{AF2_DIR}/good/*.pdb\")) > 0, \"No good AF2 models to relax with\"\n",
    "\n",
    "os.chdir(WDIR)\n",
    "RELAX_DIR = f\"{WDIR}/inpainting/6.1_final_relax\"\n",
    "os.makedirs(RELAX_DIR, exist_ok=True)\n",
    "os.chdir(RELAX_DIR)\n",
    "\n",
    "\n",
    "## First matching up the AF2 output filenames of step 5 with pocket design filenames from step 3\n",
    "ref_and_model_pairs = []\n",
    "for r in glob.glob(f\"{DESIGN_DIR_ligMPNN}/good/*.pdb\"):\n",
    "    for pdbfile in glob.glob(f\"{AF2_DIR}/good/*.pdb\"):\n",
    "        if os.path.basename(r).replace(\".pdb\", \"_\") in pdbfile:\n",
    "            ref_and_model_pairs.append((r, pdbfile))\n",
    "\n",
    "assert len(ref_and_model_pairs) == len(glob.glob(f\"{AF2_DIR}/good/*.pdb\")), \"Was not able to match all models with reference structures\"\n",
    "\n",
    "\n",
    "## Generating commands for relax jobs\n",
    "### Performing 1 relax iteration on each input structure\n",
    "NSTRUCT = 1\n",
    "cstfile = f\"{SCRIPT_DIR}/theozyme/HBA/HBA_CYS_UPO.cst\"\n",
    "\n",
    "commands_relax = []\n",
    "cmds_filename_rlx = \"commands_design\"\n",
    "with open(cmds_filename_rlx, \"w\") as file:\n",
    "    for r_m in ref_and_model_pairs:\n",
    "        commands_relax.append(f\"{PYTHON['general']} {SCRIPT_DIR}/scripts/design/align_add_ligand_relax.py \"\n",
    "                              f\"--outdir ./ --ligand {LIGAND} --ref_pdb {r_m[0]}\"\n",
    "                              f\"--pdb {r_m[1]} --nstruct {NSTRUCT} \"\n",
    "                              f\"--params {' '.join(params)} --cstfile {cstfile} > logs/{os.path.basename(pdb).replace('.pdb', '.log')}\\n\")\n",
    "        file.write(commands_relax[-1])\n",
    "\n",
    "print(\"Example design command:\")\n",
    "print(commands_design[-1])\n",
    "\n",
    "\n",
    "### Running design jobs with Slurm.\n",
    "submit_script = \"submit_relax.sh\"\n",
    "utils.create_slurm_submit_script(filename=submit_script, name=\"6.1_final_relax\", mem=\"4g\", \n",
    "                                 N_cores=1, time=\"0:30:00\", email=EMAIL, array=len(commands_relax),\n",
    "                                 array_commandfile=cmds_filename_rlx)\n",
    "\n",
    "if not os.path.exists(RELAX_DIR+\"/.done\"):\n",
    "    p = subprocess.Popen(['sbatch', submit_script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n",
    "    (output, err) = p.communicate()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "19f57db8",
   "metadata": {},
   "source": [
    "### Analyzing final relaxed structures\n",
    "Filtering them based on the same metrics as was used for the initial design"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0b11d1f7",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Analyzing Rosetta designs\n",
    "RELAX_DIR = f\"{WDIR}/inpainting/6.1_final_relax\"\n",
    "os.chdir(RELAX_DIR)\n",
    "\n",
    "scores = pd.read_csv(\"scorefile.txt\", sep=\"\\s+\", header=0)\n",
    "\n",
    "filters = {'all_cst': [1.0, '<='],\n",
    " 'nlr_SR1_rms': [0.8, '<='],\n",
    " 'nlr_totrms': [1.0, '<='],\n",
    " 'L_SASA': [0.2, '<='],\n",
    " 'COO_hbond': [1.0, '='],\n",
    " 'heme_angle_wrst': [80.0, '>='],\n",
    " 'score_per_res': [0.0, '<='],\n",
    " 'corrected_ddg': [-50.0, '<='],\n",
    " 'cms_per_atom': [4.8, '>='],\n",
    " 'rmsd_CA_rlx_in': [1.0, \"<=\"]}  # rmsd_CA_rlx_in is rmsd between relaxed structure and AF2 prediction\n",
    "\n",
    "filtered_scores = utils.filter_scores(scores, filters)\n",
    "\n",
    "## Plotting relax scores\n",
    "plt.figure(figsize=(12, 9))\n",
    "for i,k in enumerate(filters):\n",
    "    if k not in scores.keys():\n",
    "        continue\n",
    "    plt.subplot(4, 3, i+1)\n",
    "    plt.hist(scores[k])\n",
    "    plt.title(k)\n",
    "    plt.xlabel(k)\n",
    "plt.tight_layout()\n",
    "plt.show()\n",
    "\n",
    "### Copying good designs over to a new directory\n",
    "if len(filtered_scores) > 0:\n",
    "    os.makedirs(f\"{RELAX_DIR}/good\", exist_ok=True)\n",
    "    for idx, row in filtered_scores.iterrows():\n",
    "        copy2(row[\"description\"]+\".pdb\", \"good/\"+row[\"description\"]+\".pdb\")\n",
    "else:\n",
    "    print(\"No good designs created, bummer...\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fba174b7",
   "metadata": {},
   "outputs": [],
   "source": [
    "if len(filtered_scores) > 0:\n",
    "    print(f\"CONGRATULATIONS! You have successfully designed {len(filtered_scores)} proteins against ligand {LIGAND}\")\n",
    "    print(\"You can find the design models in the directory:\\n\"\n",
    "          f\"    {RELAX_DIR}/good\")\n",
    "    print(\"\\nIt is advised you manually inspect them before ordering.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "780be3c9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
